{
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/NaDaGyeong/ml-for-AI/blob/main/Final_project/novozymes.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "id": "28bdf740",
      "metadata": {
        "papermill": {
          "duration": 0.012144,
          "end_time": "2022-12-08T02:17:19.592311",
          "exception": false,
          "start_time": "2022-12-08T02:17:19.580167",
          "status": "completed"
        },
        "tags": [],
        "id": "28bdf740"
      },
      "source": [
        "# XGBoost - LB 0.30+ CV 0.30+ Train With Kaggle Data!\n",
        "# Download 5000 Single Point Mutations and 200 PDB Files!\n",
        "In this notebook version 14, we show you how to process Kaggle's train data and train a model using Kaggle's train data. This is an exciting accomplishment. Currently, as of today, all public notebooks better than LB 0.200 **do not use** Kaggle train data. Instead they use models and techniques from the internet to predict mutation stability. In this notebook we use only Kaggle train and achieve LB 0.30+\n",
        "\n",
        "The notebook you are reading is the first notebook to use only Kaggle train data and score over LB 0.200! (It achieves LB 0.307 single model). Many Kagglers have contributed to making this a success! Including Robert Hatch, GreySnow, Rope on Mars, Vladimir Slaykovskiy, Jinyuan Sun, Kaggleqrdl, Lucas Morin, Moth! It is quite tricky to utilize Kaggle's train data. The process is explained in detail [here][5] and summarized in the diagram below:\n",
        "\n",
        "![](https://raw.githubusercontent.com/cdeotte/Kaggle_Images/main/Oct-2022/k_train3.png)\n",
        "\n",
        "In previous notebook versions, we built a pipeline to train XGB using features extracted from protein sequence and structure. Also in previous notebook versions, we demonstrated how to download and use external data. In notebook version 13 we added transformer embedding features. In notebook version 11, we download 5000 Single Point Mutations and 200 PDB files from Jinyuan Sun's GitHub [here][1] and discussion [here][4]. We then train a model to predict `dTm` from single point mutations with `dTm` targets and achieve 5-Fold CV Spearman correlation coefficient 0.26 on `dTm`. Next we use this model to predict a holdout dataset's `ddG` and achieve Spearman correlation coefficient 0.26 on `ddG`. Finally we predict Kaggle's test data using this model and achieve LB 0.192. In notebook version 12, we train with both dTm and ddG targets (mixed together) and achieve dTm CV 0.194, ddG CV 0.460, LB 0.209. Notebook version 13 achieves single model LB 0.3! In notebook version 13, we submit the ensemble of versions 8, 11, 12, and 13 and achieve LB 0.34!\n",
        "\n",
        "An advantage of dowloading data is that we can train with PDB files and then utilize Kaggle's PDB test file during inference. But note we train with PDB files from Protein Data Bank and Kaggle provides PDB file from Alpha Fold for inference. These PDB files were **not** created in the same way. See discussion [here][2]. In order to have inference match training, we load our own test data wild type PDB file created from the internet as described [here][3]. This new PDB file has `b_factor` more similar to the real `b_factor` that we train on. However `atom_df` rows containing Hydrogen atoms are still different (between AlphaFold PDBs and Protein Data Bank PDBs), so be careful with feature engineering. If we submit this real `b_factor` (which isn't Alpha Fold's `pLDDT`) it scores LB 0.139 by itself. Our XGB model achieves a better LB, so it means that our model is learning something!\n",
        "\n",
        "![](https://raw.githubusercontent.com/cdeotte/Kaggle_Images/main/Oct-2022/t_model.png)\n",
        "\n",
        "# Notes\n",
        "This notebook is a work in progress. The most helpful aspects of this notebook is that it shows (1) how to use Kaggle's train data, (2) how to extract features from ESM transformer, and (3) how to download 5000 more mutations from the internet and their corresponding 200 PDB files. Specifically 4000 of these mutations, have `ddG` targets and 1000 of these mutations have `dTm` targets. We must decide how to best use this external data and the two types of targets. Furthermore we must be careful training with Protein Data Bank PDB and inferring with AlphaFold PDB. \n",
        "\n",
        "* **Versions 1-6** use Kaggle's PDB with `b_factor` column different than Protein Data Bank's PDB `b_factor` column, so we can ignore those `submission.csv`.\n",
        "* **Version 7 LB 0.146** trains/KFold-validates on `ddG` and holdout-validates on `dTM` and uses an updated PDB for inference with a `b_factor` column that is similar to training data. However many features still do not transfer well from train PDB to infer PDB because the Protein Data Bank's PDB `atom_df` dataframe excludes many hydrogen atoms. Therefore feature engineering needs to be updated. We need to only use rows with `atom_name` isin `['N','H','CA','O']` when counting atoms and/or creating features from `x_coord`, `y_coord`, `z_coord`. (Discussion [here][2])\n",
        "* **Version 8 LB 0.173** trains/Kfold-validates on `dTm` and holdout-validates on `ddG`. Uses correct `b_factor` in both train and infer. Uses features that don't work well for train and infer (because of atom PDB differences). Furthermore Version 8 model scores LB 0.173 by itself, but Version 8 notebook submits a 50/50 ensemble of version 7 and version 8 which has benefits of train on both `ddG` and `ddT` and achieves LB 0.196.\n",
        "* **Version 9-11 LB LB 0.192**  trains/Kfold-validates on `dTm` and holdout-validates on `ddG`. Achieves CV 0.260 on both targets. Uses features that exclude hydrogen atoms in PDB since Alpha Fold PDB `atom_df` contains more hydrogen atom rows than corresponding Protein Data Bank PDBs. Also uses features that should be stable between train and test. \n",
        "* **Version 12 LB 0.209** trains/Kfold-validates on all 5000 mutations! It trains with mixture of rank normalized `dTm` and `ddG` targets. Achieves CV 0.194 on `dTM` proteins and CV 0.460 on `ddG` proteins. Uses same features as versions 9-11 (but more train data and both targets). Furthermore Version 12 model scores LB 0.209 by itself, but Version 12 notebook submits a 33/33/33 ensemble of version 8, 11, 12 and achieves LB 0.298 woohoo!\n",
        "* **Version 13 LB 0.300** trains/Kfold-validates on all 5000 mutations! It trains with mixture of rank normalized `dTm` and `ddG` targets. Achieves CV 0.233 on `dTM` proteins and CV 0.619 on `ddG` proteins. Amazing `ddG` CV score! Uses same features as versions 12-13 and adds additional transformer embedding features. Furthermore Version 14 model scores LB 0.300 by itself! but Version 14 notebook submits a 50/50 ensemble of version 12 and 14 and achieves LB ???\n",
        "* **Version 14 LB 0.307** Woohoo! We finally trained a model and achieved a good CV LB using only Kaggle train data. This is a huge milestone which took collaboration from many Kagglers and 2 weeks of time. This single model achieves CV 0.34 LB 0.31. The LB score posted on version 14 is better than 0.31 because it is the ensemble of notebook version 14 with previous notebook versions 13, 12, 11, 8, and 7. Each previous version adds diversity by training with different data and/or features.\n",
        "* **Version 15** In version 15 which has not been created yet, we plan to demonstrate how to use ESM transformer to create mutation probability features and entropy features! Stay tuned...\n",
        "\n",
        "[1]: https://github.com/JinyuanSun/mutation-stability-data\n",
        "[2]: https://www.kaggle.com/competitions/novozymes-enzyme-stability-prediction/discussion/356920\n",
        "[3]: https://www.kaggle.com/competitions/novozymes-enzyme-stability-prediction/discussion/356182#1968210\n",
        "[4]: https://www.kaggle.com/competitions/novozymes-enzyme-stability-prediction/discussion/356182\n",
        "[5]: https://www.kaggle.com/competitions/novozymes-enzyme-stability-prediction/discussion/358320"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "0bf7e219",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:17:19.617384Z",
          "iopub.status.busy": "2022-12-08T02:17:19.616609Z",
          "iopub.status.idle": "2022-12-08T02:17:19.627325Z",
          "shell.execute_reply": "2022-12-08T02:17:19.626483Z"
        },
        "papermill": {
          "duration": 0.026338,
          "end_time": "2022-12-08T02:17:19.629319",
          "exception": false,
          "start_time": "2022-12-08T02:17:19.602981",
          "status": "completed"
        },
        "tags": [],
        "id": "0bf7e219"
      },
      "outputs": [],
      "source": [
        "# DEFINE WHAT TO TRAIN WITH (and KFOLD VALIDATE) VERSUS HOLDOUT VALIDATE WITH\n",
        "# ADD WORDS \"kaggle.csv\", \"jin_tm.csv\", \"jin_train.csv\", \"jin_test.csv\" to lists below\n",
        "# IF YOU ADD MORE DATASETS, ADD THOSE WORDS TOO\n",
        "\n",
        "KFOLD_SOURCES = ['kaggle.csv']\n",
        "HOLDOUT_SOURCES = ['jin_tm.csv','jin_train.csv','jin_test.csv']\n",
        "\n",
        "# IF WILD TYPE GROUP HAS FEWER THAN THIS MANY MUTATION ROWS REMOVE THEM\n",
        "EXCLUDE_CT_UNDER = 25 \n",
        "\n",
        "# IF WE TRAIN WITH ALPHA FOLD'S PDBS WE MUST INFER WITH PLDDT = TRUE\n",
        "# KAGGLE.CSV USES ALPHA FOLD PDB, SO SET BELOW TO TRUE WITH KAGGLE.CSV\n",
        "# JIN.CSV EXTERNAL DATA USES PROTEIN DATA BANK, SO SET BELOW TO FALSE WITH JIN DATA\n",
        "USE_PLDDT_INFER = True"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "78d6e46c",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:17:19.654469Z",
          "iopub.status.busy": "2022-12-08T02:17:19.653102Z",
          "iopub.status.idle": "2022-12-08T02:17:20.499774Z",
          "shell.execute_reply": "2022-12-08T02:17:20.498822Z"
        },
        "papermill": {
          "duration": 0.862444,
          "end_time": "2022-12-08T02:17:20.502512",
          "exception": false,
          "start_time": "2022-12-08T02:17:19.640068",
          "status": "completed"
        },
        "tags": [],
        "id": "78d6e46c"
      },
      "outputs": [],
      "source": [
        "import os, numpy as np, gc, seaborn as sns\n",
        "from scipy.stats import spearmanr, pearsonr, rankdata\n",
        "import matplotlib.pyplot as plt\n",
        "import pandas as pd\n",
        "pd.set_option('display.max_columns', 500)"
      ]
    },
    {
      "cell_type": "markdown",
      "id": "66439ef6",
      "metadata": {
        "papermill": {
          "duration": 0.010145,
          "end_time": "2022-12-08T02:17:20.523705",
          "exception": false,
          "start_time": "2022-12-08T02:17:20.513560",
          "status": "completed"
        },
        "tags": [],
        "id": "66439ef6"
      },
      "source": [
        "# Transform Kaggle Train Data into Mutation CSV\n",
        "Kaggle's `train.csv` cannot be used as is. Each row has one protein. Instead we need to convert the dataframe where each row has one wild type protein and one single point mutation protein and target `dTm` (i.e. delta `Tm`). Then we can more readily train with the data to predict Kaggle's `test.csv`. This is explained in discussion [here][1].\n",
        "\n",
        "Robert Hatch has processed the train data and found pairs of wild type and mutation in his notebook [here][2] with discussion [here][3]. We will load his output. GreySnow has made 73 of 78 train's wild type protein PDB's available in his dataset [here][7] with discussion [here][4] using code from Vladimir [here][5] and [here][6]. We will load his output.\n",
        "\n",
        "Note that Robert finds 4195 pairs (in Kaggle train data) which include 78 wild types. We will filter this below. From each group involving one wild type, we will only keep the rows from the most common `data_source` and `pH` pair. Furthermore, we remove rows belonging to a single `data_source` `pH` pair where all `Tm` targets are the same. And we remove 4 out of 78 groups where GreySnow and Vladimir did not find the PDB file. We saved our processed mutation CSV as `kaggle_train.csv` so you can use it in other notebooks.\n",
        "\n",
        "[1]: https://www.kaggle.com/competitions/novozymes-enzyme-stability-prediction/discussion/358320\n",
        "[2]: https://www.kaggle.com/code/roberthatch/novo-train-data-contains-wildtype-groups/notebook\n",
        "[3]: https://www.kaggle.com/competitions/novozymes-enzyme-stability-prediction/discussion/358156\n",
        "[4]: https://www.kaggle.com/competitions/novozymes-enzyme-stability-prediction/discussion/359284\n",
        "[5]: https://www.kaggle.com/code/vslaykovsky/nesp-alphafold-v2-exact-match-data\n",
        "[6]: https://www.kaggle.com/code/vslaykovsky/nesp-alphafold2-all-close-matches\n",
        "[7]: https://www.kaggle.com/datasets/shlomoron/train-wildtypes-af"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "1bbb7bee",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:17:20.546400Z",
          "iopub.status.busy": "2022-12-08T02:17:20.546060Z",
          "iopub.status.idle": "2022-12-08T02:17:20.626883Z",
          "shell.execute_reply": "2022-12-08T02:17:20.625654Z"
        },
        "papermill": {
          "duration": 0.095433,
          "end_time": "2022-12-08T02:17:20.629745",
          "exception": false,
          "start_time": "2022-12-08T02:17:20.534312",
          "status": "completed"
        },
        "tags": [],
        "id": "1bbb7bee",
        "outputId": "d170dbdf-1e15-4f70-e5b3-a8141693848c"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Before processing Robert dataframe shape: (4195, 8)\n"
          ]
        },
        {
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>seq_id</th>\n",
              "      <th>protein_sequence</th>\n",
              "      <th>pH</th>\n",
              "      <th>data_source</th>\n",
              "      <th>tm</th>\n",
              "      <th>x</th>\n",
              "      <th>group</th>\n",
              "      <th>wildtype</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>18020</td>\n",
              "      <td>MNAFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...</td>\n",
              "      <td>2.0</td>\n",
              "      <td>10.1021/bi00535a054</td>\n",
              "      <td>38.1</td>\n",
              "      <td>164</td>\n",
              "      <td>0</td>\n",
              "      <td>MNIFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>18021</td>\n",
              "      <td>MNAFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...</td>\n",
              "      <td>4.2</td>\n",
              "      <td>NaN</td>\n",
              "      <td>53.3</td>\n",
              "      <td>164</td>\n",
              "      <td>0</td>\n",
              "      <td>MNIFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>18022</td>\n",
              "      <td>MNAFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...</td>\n",
              "      <td>2.0</td>\n",
              "      <td>10.1038/334406a0</td>\n",
              "      <td>38.1</td>\n",
              "      <td>164</td>\n",
              "      <td>0</td>\n",
              "      <td>MNIFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>18023</td>\n",
              "      <td>MNAFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...</td>\n",
              "      <td>6.5</td>\n",
              "      <td>10.1038/334406a0</td>\n",
              "      <td>62.9</td>\n",
              "      <td>164</td>\n",
              "      <td>0</td>\n",
              "      <td>MNIFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>18060</td>\n",
              "      <td>MNCFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...</td>\n",
              "      <td>2.0</td>\n",
              "      <td>NaN</td>\n",
              "      <td>41.9</td>\n",
              "      <td>164</td>\n",
              "      <td>0</td>\n",
              "      <td>MNIFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "   seq_id                                   protein_sequence   pH  \\\n",
              "0   18020  MNAFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...  2.0   \n",
              "1   18021  MNAFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...  4.2   \n",
              "2   18022  MNAFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...  2.0   \n",
              "3   18023  MNAFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...  6.5   \n",
              "4   18060  MNCFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...  2.0   \n",
              "\n",
              "           data_source    tm    x  group  \\\n",
              "0  10.1021/bi00535a054  38.1  164      0   \n",
              "1                  NaN  53.3  164      0   \n",
              "2     10.1038/334406a0  38.1  164      0   \n",
              "3     10.1038/334406a0  62.9  164      0   \n",
              "4                  NaN  41.9  164      0   \n",
              "\n",
              "                                            wildtype  \n",
              "0  MNIFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...  \n",
              "1  MNIFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...  \n",
              "2  MNIFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...  \n",
              "3  MNIFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...  \n",
              "4  MNIFEMLRIDERLRLKIYKDTEGYYTIGIGHLLTKSPSLNAAKSEL...  "
            ]
          },
          "execution_count": 3,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# https://www.kaggle.com/code/roberthatch/novo-train-data-contains-wildtype-groups/notebook\n",
        "kaggle = pd.read_csv('../input/novo-train-data-contains-wildtype-groups/train_wildtype_groups.csv')\n",
        "print('Before processing Robert dataframe shape:', kaggle.shape )\n",
        "kaggle.head()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "9d3cea0f",
      "metadata": {
        "_kg_hide-input": true,
        "execution": {
          "iopub.execute_input": "2022-12-08T02:17:20.654403Z",
          "iopub.status.busy": "2022-12-08T02:17:20.653585Z",
          "iopub.status.idle": "2022-12-08T02:17:25.237521Z",
          "shell.execute_reply": "2022-12-08T02:17:25.236460Z"
        },
        "papermill": {
          "duration": 4.598184,
          "end_time": "2022-12-08T02:17:25.239757",
          "exception": false,
          "start_time": "2022-12-08T02:17:20.641573",
          "status": "completed"
        },
        "tags": [],
        "id": "9d3cea0f",
        "outputId": "9caf0a45-5032-4899-f365-524f40946bca"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "After processing Robert dataframe shape: (2556, 8)\n"
          ]
        },
        {
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>PDB</th>\n",
              "      <th>WT</th>\n",
              "      <th>position</th>\n",
              "      <th>MUT</th>\n",
              "      <th>dTm</th>\n",
              "      <th>sequence</th>\n",
              "      <th>mutant_seq</th>\n",
              "      <th>CIF</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>GP01</td>\n",
              "      <td>L</td>\n",
              "      <td>89</td>\n",
              "      <td>A</td>\n",
              "      <td>2.28642</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>AF-P00644-F1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>GP01</td>\n",
              "      <td>T</td>\n",
              "      <td>95</td>\n",
              "      <td>C</td>\n",
              "      <td>1.48642</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>AF-P00644-F1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>GP01</td>\n",
              "      <td>T</td>\n",
              "      <td>95</td>\n",
              "      <td>C</td>\n",
              "      <td>0.28642</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>AF-P00644-F1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>GP01</td>\n",
              "      <td>T</td>\n",
              "      <td>95</td>\n",
              "      <td>S</td>\n",
              "      <td>2.48642</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>AF-P00644-F1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>GP01</td>\n",
              "      <td>T</td>\n",
              "      <td>95</td>\n",
              "      <td>S</td>\n",
              "      <td>3.88642</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>AF-P00644-F1</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "    PDB WT  position MUT      dTm  \\\n",
              "0  GP01  L        89   A  2.28642   \n",
              "1  GP01  T        95   C  1.48642   \n",
              "2  GP01  T        95   C  0.28642   \n",
              "3  GP01  T        95   S  2.48642   \n",
              "4  GP01  T        95   S  3.88642   \n",
              "\n",
              "                                            sequence  \\\n",
              "0  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...   \n",
              "1  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...   \n",
              "2  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...   \n",
              "3  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...   \n",
              "4  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...   \n",
              "\n",
              "                                          mutant_seq           CIF  \n",
              "0  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...  AF-P00644-F1  \n",
              "1  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...  AF-P00644-F1  \n",
              "2  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...  AF-P00644-F1  \n",
              "3  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...  AF-P00644-F1  \n",
              "4  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...  AF-P00644-F1  "
            ]
          },
          "execution_count": 4,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "kaggle['id'] = kaggle.pH.astype('str') + '_' + kaggle.group.astype('str')\n",
        "kaggle['ct'] = kaggle.groupby('id').id.transform('count')\n",
        "#kaggle['n'] = kaggle.groupby('id').tm.transform('nunique')\n",
        "#kaggle = kaggle.loc[kaggle.n>1]\n",
        "kaggle = kaggle.sort_values(['group','ct'],ascending=[True,False])\n",
        "KEEP = kaggle.groupby('group').id.agg('first').values\n",
        "kaggle = kaggle.loc[kaggle.id.isin(KEEP)]\n",
        "\n",
        "def find_mut(row):\n",
        "    mut = row.protein_sequence\n",
        "    seq = row.wildtype\n",
        "    same = True\n",
        "    for i,(x,y) in enumerate(zip(seq,mut)):\n",
        "        if x!=y: \n",
        "            same = False\n",
        "            break\n",
        "    if not same:\n",
        "        row['WT'] = seq[i]\n",
        "        row['position'] = i+1\n",
        "        row['MUT'] = mut[i]\n",
        "    else:\n",
        "        row['WT'] = 'X'\n",
        "        row['position'] = -1   \n",
        "        row['MUT'] = 'X'     \n",
        "    return row\n",
        "\n",
        "grp = [f'GP{g:02d}' for g in kaggle.group.values]\n",
        "kaggle['PDB'] = grp\n",
        "kaggle = kaggle.apply(find_mut,axis=1)\n",
        "kaggle = kaggle.loc[kaggle.position!=-1]\n",
        "kaggle['base'] = kaggle.groupby('group').tm.transform('mean')\n",
        "kaggle['dTm'] = kaggle.tm - kaggle.base\n",
        "kaggle = kaggle.rename({'wildtype':'sequence','protein_sequence':'mutant_seq'},axis=1)\n",
        "COLS = ['PDB','WT','position','MUT','dTm','sequence','mutant_seq']\n",
        "kaggle = kaggle[COLS]\n",
        "\n",
        "# https://www.kaggle.com/datasets/shlomoron/train-wildtypes-af\n",
        "alphafold = pd.read_csv('../input/train-wildtypes-af/alpha_fold_df.csv')\n",
        "dd = {}\n",
        "for s in kaggle.sequence.unique():\n",
        "    tmp = alphafold.loc[alphafold.af2_sequence==s,'af2id']\n",
        "    if len(tmp)>0: c = tmp.values[0].split(':')[1]\n",
        "    else: c = np.nan\n",
        "    dd[s] = c\n",
        "    \n",
        "kaggle['CIF'] = kaggle.sequence.map(dd)\n",
        "kaggle = kaggle.loc[kaggle.CIF.notnull()].reset_index(drop=True)\n",
        "kaggle.to_csv('kaggle_train.csv',index=False)\n",
        "print('After processing Robert dataframe shape:', kaggle.shape )\n",
        "kaggle.head()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "79a36e16",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:17:25.264150Z",
          "iopub.status.busy": "2022-12-08T02:17:25.263850Z",
          "iopub.status.idle": "2022-12-08T02:17:25.285701Z",
          "shell.execute_reply": "2022-12-08T02:17:25.284547Z"
        },
        "papermill": {
          "duration": 0.036504,
          "end_time": "2022-12-08T02:17:25.288136",
          "exception": false,
          "start_time": "2022-12-08T02:17:25.251632",
          "status": "completed"
        },
        "tags": [],
        "id": "79a36e16",
        "outputId": "2f4ce635-1176-428c-f9c7-6990601fa2cc"
      },
      "outputs": [
        {
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>PDB</th>\n",
              "      <th>wildtype</th>\n",
              "      <th>position</th>\n",
              "      <th>mutation</th>\n",
              "      <th>dTm</th>\n",
              "      <th>sequence</th>\n",
              "      <th>mutant_seq</th>\n",
              "      <th>CIF</th>\n",
              "      <th>source</th>\n",
              "      <th>ddG</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>GP01</td>\n",
              "      <td>L</td>\n",
              "      <td>89</td>\n",
              "      <td>A</td>\n",
              "      <td>2.28642</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>AF-P00644-F1</td>\n",
              "      <td>kaggle.csv</td>\n",
              "      <td>NaN</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>GP01</td>\n",
              "      <td>T</td>\n",
              "      <td>95</td>\n",
              "      <td>C</td>\n",
              "      <td>1.48642</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>AF-P00644-F1</td>\n",
              "      <td>kaggle.csv</td>\n",
              "      <td>NaN</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>GP01</td>\n",
              "      <td>T</td>\n",
              "      <td>95</td>\n",
              "      <td>C</td>\n",
              "      <td>0.28642</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>AF-P00644-F1</td>\n",
              "      <td>kaggle.csv</td>\n",
              "      <td>NaN</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>GP01</td>\n",
              "      <td>T</td>\n",
              "      <td>95</td>\n",
              "      <td>S</td>\n",
              "      <td>2.48642</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>AF-P00644-F1</td>\n",
              "      <td>kaggle.csv</td>\n",
              "      <td>NaN</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>GP01</td>\n",
              "      <td>T</td>\n",
              "      <td>95</td>\n",
              "      <td>S</td>\n",
              "      <td>3.88642</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...</td>\n",
              "      <td>AF-P00644-F1</td>\n",
              "      <td>kaggle.csv</td>\n",
              "      <td>NaN</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "    PDB wildtype  position mutation      dTm  \\\n",
              "0  GP01        L        89        A  2.28642   \n",
              "1  GP01        T        95        C  1.48642   \n",
              "2  GP01        T        95        C  0.28642   \n",
              "3  GP01        T        95        S  2.48642   \n",
              "4  GP01        T        95        S  3.88642   \n",
              "\n",
              "                                            sequence  \\\n",
              "0  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...   \n",
              "1  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...   \n",
              "2  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...   \n",
              "3  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...   \n",
              "4  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...   \n",
              "\n",
              "                                          mutant_seq           CIF  \\\n",
              "0  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...  AF-P00644-F1   \n",
              "1  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...  AF-P00644-F1   \n",
              "2  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...  AF-P00644-F1   \n",
              "3  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...  AF-P00644-F1   \n",
              "4  MLVMTEYLLSAGICMAIVSILLIGMAISNVSKGQYAKRFFFFATSC...  AF-P00644-F1   \n",
              "\n",
              "       source  ddG  \n",
              "0  kaggle.csv  NaN  \n",
              "1  kaggle.csv  NaN  \n",
              "2  kaggle.csv  NaN  \n",
              "3  kaggle.csv  NaN  \n",
              "4  kaggle.csv  NaN  "
            ]
          },
          "execution_count": 5,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "kaggle['source'] = 'kaggle.csv'\n",
        "kaggle['ddG'] = np.nan\n",
        "kaggle = kaggle.rename({'WT':'wildtype','MUT':'mutation'},axis=1)\n",
        "kaggle = kaggle.sort_values(['PDB','position']).reset_index(drop=True)\n",
        "kaggle.head()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "5d3706cf",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:17:25.315411Z",
          "iopub.status.busy": "2022-12-08T02:17:25.313734Z",
          "iopub.status.idle": "2022-12-08T02:17:25.321711Z",
          "shell.execute_reply": "2022-12-08T02:17:25.320670Z"
        },
        "papermill": {
          "duration": 0.023219,
          "end_time": "2022-12-08T02:17:25.323793",
          "exception": false,
          "start_time": "2022-12-08T02:17:25.300574",
          "status": "completed"
        },
        "tags": [],
        "id": "5d3706cf",
        "outputId": "0bab3e40-c02d-473e-ece7-c008ade83126"
      },
      "outputs": [
        {
          "data": {
            "text/plain": [
              "array(['GP01', 'GP02', 'GP03', 'GP04', 'GP06', 'GP07', 'GP08', 'GP09',\n",
              "       'GP10', 'GP11', 'GP12', 'GP13', 'GP14', 'GP15', 'GP16', 'GP17',\n",
              "       'GP18', 'GP19', 'GP20', 'GP21', 'GP22', 'GP23', 'GP24', 'GP25',\n",
              "       'GP26', 'GP27', 'GP28', 'GP29', 'GP30', 'GP31', 'GP32', 'GP33',\n",
              "       'GP34', 'GP35', 'GP36', 'GP37', 'GP38', 'GP39', 'GP40', 'GP41',\n",
              "       'GP42', 'GP43', 'GP44', 'GP45', 'GP46', 'GP48', 'GP49', 'GP50',\n",
              "       'GP51', 'GP52', 'GP53', 'GP54', 'GP55', 'GP57', 'GP58', 'GP59',\n",
              "       'GP60', 'GP62', 'GP63', 'GP64', 'GP65', 'GP66', 'GP67', 'GP68',\n",
              "       'GP69', 'GP70', 'GP71', 'GP72', 'GP73', 'GP74', 'GP75', 'GP76',\n",
              "       'GP77'], dtype=object)"
            ]
          },
          "execution_count": 6,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "kaggle.PDB.unique()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "f244b382",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:17:25.350459Z",
          "iopub.status.busy": "2022-12-08T02:17:25.349017Z",
          "iopub.status.idle": "2022-12-08T02:17:25.354204Z",
          "shell.execute_reply": "2022-12-08T02:17:25.353254Z"
        },
        "papermill": {
          "duration": 0.020148,
          "end_time": "2022-12-08T02:17:25.356194",
          "exception": false,
          "start_time": "2022-12-08T02:17:25.336046",
          "status": "completed"
        },
        "tags": [],
        "id": "f244b382"
      },
      "outputs": [],
      "source": [
        "# THE FOLLOWING PROTEINS SEQUENCES CANNOT BE ALIGNED BETWEEN CSV AND PDB FILE (not sure why)\n",
        "#bad = [f for f in df.PDB.unique() if len(f)>4]\n",
        "#bad += ['1LVE', '2IMM', '2RPN', '1BKS', '1BLC', '1D5G', '1KDX', '1OTR', '3BN0', '3D3B', '3HHR', '3O39']\n",
        "#bad += ['3BDC','1AMQ','1X0J','1TPK','1GLM','1RHG','3DVI','1RN1','1QGV'] \n",
        "#bad += ['1SVX','4E5K'] \n",
        "#print(f'We will ignore mutations from {len(bad)} PDB files')\n",
        "bad = []"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "cbd207dd",
      "metadata": {
        "_kg_hide-output": true,
        "execution": {
          "iopub.execute_input": "2022-12-08T02:17:25.380831Z",
          "iopub.status.busy": "2022-12-08T02:17:25.380568Z",
          "iopub.status.idle": "2022-12-08T02:17:25.386412Z",
          "shell.execute_reply": "2022-12-08T02:17:25.385396Z"
        },
        "papermill": {
          "duration": 0.020638,
          "end_time": "2022-12-08T02:17:25.388534",
          "exception": false,
          "start_time": "2022-12-08T02:17:25.367896",
          "status": "completed"
        },
        "tags": [],
        "id": "cbd207dd",
        "outputId": "3ba8f523-3b9d-43ee-becc-5510ff8e098c"
      },
      "outputs": [
        {
          "data": {
            "text/plain": [
              "\"\\nos.system('mkdir downloaded_pdb')\\nfor p in [f for f in df.PDB.unique() if f not in bad]:\\n    if p[:2]=='GP': continue # skip kaggle CIFba\\n    os.system(f'cd downloaded_pdb; wget https://files.rcsb.org/download/{p}.pdb') \\n\""
            ]
          },
          "execution_count": 8,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "\"\"\"\n",
        "os.system('mkdir downloaded_pdb')\n",
        "for p in [f for f in df.PDB.unique() if f not in bad]:\n",
        "    if p[:2]=='GP': continue # skip kaggle CIFba\n",
        "    os.system(f'cd downloaded_pdb; wget https://files.rcsb.org/download/{p}.pdb') \n",
        "\"\"\""
      ]
    },
    {
      "cell_type": "markdown",
      "id": "1edadaba",
      "metadata": {
        "papermill": {
          "duration": 0.011844,
          "end_time": "2022-12-08T02:17:25.412152",
          "exception": false,
          "start_time": "2022-12-08T02:17:25.400308",
          "status": "completed"
        },
        "tags": [],
        "id": "1edadaba"
      },
      "source": [
        "# Feature Engineer\n",
        "Below is 3D plot of Kaggle's test data protein structure from PDB file rendered by py3Dmol (with docs [here][1]). For each of our 5000 mutation downloaded train data, we also have protein structure since we downloaded 200 PDB files from Protein Data Bank. And when using Kaggle train data, we now have PDB files! Therefore when we feature engineer, we can create features from both protein sequence, protein structure, and amino acids. In this notebook, we use the following features:\n",
        "* Protein Structure features (from atom positions in PDB file)\n",
        "* Protein Sequence features\n",
        "* Amino Acid features \n",
        "* Transformer ESM embeddings\n",
        "\n",
        "thanks @kaggleqrdl for his ESM starter notebook [here][2]\n",
        "\n",
        "[1]: https://3dmol.csb.pitt.edu/doc/$3Dmol.GLViewer.html\n",
        "[2]: https://www.kaggle.com/code/kaggleqrdl/esm-quick-start-lb237"
      ]
    },
    {
      "cell_type": "markdown",
      "id": "412548dc",
      "metadata": {
        "papermill": {
          "duration": 0.011849,
          "end_time": "2022-12-08T02:17:25.435661",
          "exception": false,
          "start_time": "2022-12-08T02:17:25.423812",
          "status": "completed"
        },
        "tags": [],
        "id": "412548dc"
      },
      "source": [
        "## Protein Structure Features"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "ecdd820b",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:17:25.460759Z",
          "iopub.status.busy": "2022-12-08T02:17:25.460453Z",
          "iopub.status.idle": "2022-12-08T02:17:36.666304Z",
          "shell.execute_reply": "2022-12-08T02:17:36.665171Z"
        },
        "papermill": {
          "duration": 11.226561,
          "end_time": "2022-12-08T02:17:36.674024",
          "exception": false,
          "start_time": "2022-12-08T02:17:25.447463",
          "status": "completed"
        },
        "tags": [],
        "id": "ecdd820b",
        "outputId": "239f7235-f49d-4077-cde9-5e29ed796874"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\r\n",
            "\u001b[0m"
          ]
        },
        {
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_16704658566527212\"  style=\"position: relative; width: 800px; height: 600px\">\n        <p id=\"3dmolwarning_16704658566527212\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n      resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdn.jsdelivr.net/npm/3dmol@latest/build/3Dmol-min.min.js');\n}\n\nvar viewer_16704658566527212 = null;\nvar warn = document.getElementById(\"3dmolwarning_16704658566527212\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_16704658566527212 = $3Dmol.createViewer($(\"#3dmolviewer_16704658566527212\"),{backgroundColor:\"white\"});\nviewer_16704658566527212.zoomTo();\n\tviewer_16704658566527212.addModelsAsFrames(\"ATOM      1  N   VAL A   1      34.064  -6.456  50.464  1.00 45.11           N  \\nATOM      2  H   VAL A   1      33.576  -6.009  51.228  1.00 45.11           H  \\nATOM      3  H2  VAL A   1      33.882  -7.449  50.477  1.00 45.11           H  \\nATOM      4  H3  VAL A   1      35.060  -6.323  50.566  1.00 45.11           H  \\nATOM      5  CA  VAL A   1      33.643  -5.877  49.162  1.00 45.11           C  \\nATOM      6  HA  VAL A   1      32.602  -6.130  48.959  1.00 45.11           H  \\nATOM      7  C   VAL A   1      34.525  -6.533  48.112  1.00 45.11           C  \\nATOM      8  CB  VAL A   1      33.798  -4.340  49.169  1.00 45.11           C  \\nATOM      9  HB  VAL A   1      34.856  -4.086  49.236  1.00 45.11           H  \\nATOM     10  O   VAL A   1      35.722  -6.569  48.382  1.00 45.11           O  \\nATOM     11  CG1 VAL A   1      33.211  -3.674  47.923  1.00 45.11           C  \\nATOM     12 HG11 VAL A   1      32.153  -3.914  47.814  1.00 45.11           H  \\nATOM     13 HG12 VAL A   1      33.309  -2.592  48.006  1.00 45.11           H  \\nATOM     14 HG13 VAL A   1      33.768  -3.972  47.035  1.00 45.11           H  \\nATOM     15  CG2 VAL A   1      33.081  -3.719  50.378  1.00 45.11           C  \\nATOM     16 HG21 VAL A   1      32.027  -3.996  50.385  1.00 45.11           H  \\nATOM     17 HG22 VAL A   1      33.147  -2.632  50.323  1.00 45.11           H  \\nATOM     18 HG23 VAL A   1      33.551  -4.014  51.317  1.00 45.11           H  \\nATOM     19  N   PRO A   2      33.995  -7.139  47.031  1.00 41.54           N  \\nATOM     20  CA  PRO A   2      32.926  -6.593  46.177  1.00 41.54           C  \\nATOM     21  HA  PRO A   2      32.395  -5.757  46.632  1.00 41.54           H  \\nATOM     22  C   PRO A   2      31.880  -7.669  45.766  1.00 41.54           C  \\nATOM     23  CB  PRO A   2      33.799  -6.106  45.023  1.00 41.54           C  \\nATOM     24  HB2 PRO A   2      33.209  -5.785  44.165  1.00 41.54           H  \\nATOM     25  HB3 PRO A   2      34.406  -5.259  45.344  1.00 41.54           H  \\nATOM     26  O   PRO A   2      31.961  -8.783  46.261  1.00 41.54           O  \\nATOM     27  CG  PRO A   2      34.730  -7.298  44.763  1.00 41.54           C  \\nATOM     28  HG2 PRO A   2      35.700  -6.963  44.393  1.00 41.54           H  \\nATOM     29  HG3 PRO A   2      34.275  -8.001  44.065  1.00 41.54           H  \\nATOM     30  CD  PRO A   2      34.847  -7.942  46.145  1.00 41.54           C  \\nATOM     31  HD2 PRO A   2      34.493  -8.972  46.107  1.00 41.54           H  \\nATOM     32  HD3 PRO A   2      35.893  -7.940  46.454  1.00 41.54           H  \\nATOM     33  N   VAL A   3      30.848  -7.474  44.941  1.00 40.19           N  \\nATOM     34  H   VAL A   3      30.300  -8.316  44.831  1.00 40.19           H  \\nATOM     35  CA  VAL A   3      30.251  -6.338  44.219  1.00 40.19           C  \\nATOM     36  HA  VAL A   3      30.525  -5.393  44.687  1.00 40.19           H  \\nATOM     37  C   VAL A   3      28.733  -6.578  44.318  1.00 40.19           C  \\nATOM     38  CB  VAL A   3      30.580  -6.315  42.698  1.00 40.19           C  \\nATOM     39  HB  VAL A   3      29.870  -6.972  42.195  1.00 40.19           H  \\nATOM     40  O   VAL A   3      28.286  -7.721  44.233  1.00 40.19           O  \\nATOM     41  CG1 VAL A   3      30.389  -4.904  42.129  1.00 40.19           C  \\nATOM     42 HG11 VAL A   3      29.360  -4.574  42.271  1.00 40.19           H  \\nATOM     43 HG12 VAL A   3      31.071  -4.199  42.605  1.00 40.19           H  \\nATOM     44 HG13 VAL A   3      30.579  -4.914  41.056  1.00 40.19           H  \\nATOM     45  CG2 VAL A   3      31.962  -6.800  42.240  1.00 40.19           C  \\nATOM     46 HG21 VAL A   3      31.865  -7.147  41.211  1.00 40.19           H  \\nATOM     47 HG22 VAL A   3      32.309  -7.650  42.827  1.00 40.19           H  \\nATOM     48 HG23 VAL A   3      32.692  -5.991  42.210  1.00 40.19           H  \\nATOM     49  N   ASN A   4      27.952  -5.519  44.501  1.00 42.11           N  \\nATOM     50  H   ASN A   4      28.379  -4.604  44.470  1.00 42.11           H  \\nATOM     51  CA  ASN A   4      26.494  -5.559  44.400  1.00 42.11           C  \\nATOM     52  HA  ASN A   4      26.116  -6.300  45.106  1.00 42.11           H  \\nATOM     53  C   ASN A   4      26.105  -5.946  42.955  1.00 42.11           C  \\nATOM     54  CB  ASN A   4      25.986  -4.143  44.763  1.00 42.11           C  \\nATOM     55  HB2 ASN A   4      25.812  -3.578  43.848  1.00 42.11           H  \\nATOM     56  HB3 ASN A   4      26.739  -3.606  45.340  1.00 42.11           H  \\nATOM     57  O   ASN A   4      26.673  -5.347  42.040  1.00 42.11           O  \\nATOM     58  CG  ASN A   4      24.728  -4.117  45.604  1.00 42.11           C  \\nATOM     59  ND2 ASN A   4      24.038  -3.002  45.624  1.00 42.11           N  \\nATOM     60 HD21 ASN A   4      24.264  -2.251  44.988  1.00 42.11           H  \\nATOM     61 HD22 ASN A   4      23.178  -3.029  46.153  1.00 42.11           H  \\nATOM     62  OD1 ASN A   4      24.388  -5.051  46.307  1.00 42.11           O  \\nATOM     63  N   PRO A   5      25.188  -6.900  42.714  1.00 42.39           N  \\nATOM     64  CA  PRO A   5      24.717  -7.167  41.362  1.00 42.39           C  \\nATOM     65  HA  PRO A   5      25.566  -7.322  40.696  1.00 42.39           H  \\nATOM     66  C   PRO A   5      23.911  -5.962  40.864  1.00 42.39           C  \\nATOM     67  CB  PRO A   5      23.912  -8.466  41.441  1.00 42.39           C  \\nATOM     68  HB2 PRO A   5      23.071  -8.471  40.747  1.00 42.39           H  \\nATOM     69  HB3 PRO A   5      24.572  -9.310  41.240  1.00 42.39           H  \\nATOM     70  O   PRO A   5      22.948  -5.533  41.504  1.00 42.39           O  \\nATOM     71  CG  PRO A   5      23.450  -8.520  42.897  1.00 42.39           C  \\nATOM     72  HG2 PRO A   5      22.504  -7.985  42.992  1.00 42.39           H  \\nATOM     73  HG3 PRO A   5      23.340  -9.546  43.247  1.00 42.39           H  \\nATOM     74  CD  PRO A   5      24.532  -7.769  43.675  1.00 42.39           C  \\nATOM     75  HD2 PRO A   5      24.080  -7.194  44.484  1.00 42.39           H  \\nATOM     76  HD3 PRO A   5      25.259  -8.477  44.074  1.00 42.39           H  \\nATOM     77  N   GLU A   6      24.357  -5.389  39.746  1.00 41.49           N  \\nATOM     78  H   GLU A   6      25.171  -5.776  39.291  1.00 41.49           H  \\nATOM     79  CA  GLU A   6      23.604  -4.383  39.002  1.00 41.49           C  \\nATOM     80  HA  GLU A   6      23.469  -3.516  39.649  1.00 41.49           H  \\nATOM     81  C   GLU A   6      22.232  -4.955  38.616  1.00 41.49           C  \\nATOM     82  CB  GLU A   6      24.367  -3.945  37.742  1.00 41.49           C  \\nATOM     83  HB2 GLU A   6      24.714  -4.830  37.209  1.00 41.49           H  \\nATOM     84  HB3 GLU A   6      23.686  -3.393  37.094  1.00 41.49           H  \\nATOM     85  O   GLU A   6      22.136  -6.130  38.252  1.00 41.49           O  \\nATOM     86  CG  GLU A   6      25.551  -3.028  38.079  1.00 41.49           C  \\nATOM     87  HG2 GLU A   6      25.171  -2.157  38.612  1.00 41.49           H  \\nATOM     88  HG3 GLU A   6      26.229  -3.557  38.749  1.00 41.49           H  \\nATOM     89  CD  GLU A   6      26.338  -2.574  36.838  1.00 41.49           C  \\nATOM     90  OE1 GLU A   6      26.865  -1.438  36.883  1.00 41.49           O  \\nATOM     91  OE2 GLU A   6      26.449  -3.368  35.879  1.00 41.49           O  \\nATOM     92  N   PRO A   7      21.156  -4.158  38.723  1.00 42.58           N  \\nATOM     93  CA  PRO A   7      19.829  -4.625  38.379  1.00 42.58           C  \\nATOM     94  HA  PRO A   7      19.631  -5.579  38.867  1.00 42.58           H  \\nATOM     95  C   PRO A   7      19.735  -4.802  36.866  1.00 42.58           C  \\nATOM     96  CB  PRO A   7      18.869  -3.567  38.927  1.00 42.58           C  \\nATOM     97  HB2 PRO A   7      18.610  -3.816  39.956  1.00 42.58           H  \\nATOM     98  HB3 PRO A   7      17.968  -3.476  38.320  1.00 42.58           H  \\nATOM     99  O   PRO A   7      20.021  -3.871  36.109  1.00 42.58           O  \\nATOM    100  CG  PRO A   7      19.700  -2.283  38.910  1.00 42.58           C  \\nATOM    101  HG2 PRO A   7      19.384  -1.588  39.688  1.00 42.58           H  \\nATOM    102  HG3 PRO A   7      19.623  -1.814  37.929  1.00 42.58           H  \\nATOM    103  CD  PRO A   7      21.133  -2.764  39.129  1.00 42.58           C  \\nATOM    104  HD2 PRO A   7      21.823  -2.169  38.531  1.00 42.58           H  \\nATOM    105  HD3 PRO A   7      21.385  -2.693  40.187  1.00 42.58           H  \\nATOM    106  N   ASP A   8      19.295  -5.991  36.453  1.00 40.36           N  \\nATOM    107  H   ASP A   8      19.207  -6.736  37.128  1.00 40.36           H  \\nATOM    108  CA  ASP A   8      18.903  -6.287  35.082  1.00 40.36           C  \\nATOM    109  HA  ASP A   8      19.789  -6.277  34.446  1.00 40.36           H  \\nATOM    110  C   ASP A   8      17.941  -5.203  34.596  1.00 40.36           C  \\nATOM    111  CB  ASP A   8      18.237  -7.673  34.990  1.00 40.36           C  \\nATOM    112  HB2 ASP A   8      17.594  -7.689  34.109  1.00 40.36           H  \\nATOM    113  HB3 ASP A   8      17.603  -7.834  35.862  1.00 40.36           H  \\nATOM    114  O   ASP A   8      16.776  -5.124  34.998  1.00 40.36           O  \\nATOM    115  CG  ASP A   8      19.233  -8.825  34.835  1.00 40.36           C  \\nATOM    116  OD1 ASP A   8      20.206  -8.650  34.070  1.00 40.36           O  \\nATOM    117  OD2 ASP A   8      18.973  -9.890  35.438  1.00 40.36           O  \\nATOM    118  N   ALA A   9      18.451  -4.345  33.716  1.00 40.70           N  \\nATOM    119  H   ALA A   9      19.444  -4.395  33.539  1.00 40.70           H  \\nATOM    120  CA  ALA A   9      17.648  -3.463  32.904  1.00 40.70           C  \\nATOM    121  HA  ALA A   9      16.956  -2.915  33.544  1.00 40.70           H  \\nATOM    122  C   ALA A   9      16.840  -4.347  31.949  1.00 40.70           C  \\nATOM    123  CB  ALA A   9      18.570  -2.462  32.194  1.00 40.70           C  \\nATOM    124  HB1 ALA A   9      17.975  -1.799  31.565  1.00 40.70           H  \\nATOM    125  HB2 ALA A   9      19.290  -2.992  31.572  1.00 40.70           H  \\nATOM    126  HB3 ALA A   9      19.106  -1.868  32.935  1.00 40.70           H  \\nATOM    127  O   ALA A   9      17.230  -4.577  30.805  1.00 40.70           O  \\nATOM    128  N   THR A  10      15.705  -4.865  32.421  1.00 39.44           N  \\nATOM    129  H   THR A  10      15.523  -4.803  33.413  1.00 39.44           H  \\nATOM    130  CA  THR A  10      14.665  -5.376  31.538  1.00 39.44           C  \\nATOM    131  HA  THR A  10      15.058  -6.215  30.964  1.00 39.44           H  \\nATOM    132  C   THR A  10      14.315  -4.243  30.595  1.00 39.44           C  \\nATOM    133  CB  THR A  10      13.410  -5.836  32.293  1.00 39.44           C  \\nATOM    134  HB  THR A  10      12.581  -5.903  31.588  1.00 39.44           H  \\nATOM    135  O   THR A  10      13.726  -3.240  31.005  1.00 39.44           O  \\nATOM    136  CG2 THR A  10      13.611  -7.210  32.923  1.00 39.44           C  \\nATOM    137 HG21 THR A  10      12.704  -7.502  33.452  1.00 39.44           H  \\nATOM    138 HG22 THR A  10      13.830  -7.946  32.149  1.00 39.44           H  \\nATOM    139 HG23 THR A  10      14.441  -7.173  33.630  1.00 39.44           H  \\nATOM    140  OG1 THR A  10      13.070  -4.923  33.311  1.00 39.44           O  \\nATOM    141  HG1 THR A  10      13.120  -4.049  32.916  1.00 39.44           H  \\nATOM    142  N   SER A  11      14.729  -4.390  29.340  1.00 40.63           N  \\nATOM    143  H   SER A  11      15.296  -5.192  29.108  1.00 40.63           H  \\nATOM    144  CA  SER A  11      14.310  -3.551  28.236  1.00 40.63           C  \\nATOM    145  HA  SER A  11      14.615  -2.520  28.415  1.00 40.63           H  \\nATOM    146  C   SER A  11      12.793  -3.631  28.169  1.00 40.63           C  \\nATOM    147  CB  SER A  11      14.958  -4.045  26.937  1.00 40.63           C  \\nATOM    148  HB2 SER A  11      14.525  -3.513  26.090  1.00 40.63           H  \\nATOM    149  HB3 SER A  11      16.029  -3.842  26.968  1.00 40.63           H  \\nATOM    150  O   SER A  11      12.226  -4.579  27.626  1.00 40.63           O  \\nATOM    151  OG  SER A  11      14.763  -5.439  26.780  1.00 40.63           O  \\nATOM    152  HG  SER A  11      13.817  -5.593  26.732  1.00 40.63           H  \\nATOM    153  N   VAL A  12      12.129  -2.651  28.773  1.00 41.68           N  \\nATOM    154  H   VAL A  12      12.647  -2.004  29.350  1.00 41.68           H  \\nATOM    155  CA  VAL A  12      10.720  -2.401  28.524  1.00 41.68           C  \\nATOM    156  HA  VAL A  12      10.173  -3.341  28.591  1.00 41.68           H  \\nATOM    157  C   VAL A  12      10.690  -1.906  27.087  1.00 41.68           C  \\nATOM    158  CB  VAL A  12      10.102  -1.401  29.528  1.00 41.68           C  \\nATOM    159  HB  VAL A  12      10.367  -0.379  29.255  1.00 41.68           H  \\nATOM    160  O   VAL A  12      10.948  -0.734  26.812  1.00 41.68           O  \\nATOM    161  CG1 VAL A  12       8.575  -1.538  29.518  1.00 41.68           C  \\nATOM    162 HG11 VAL A  12       8.198  -1.356  28.511  1.00 41.68           H  \\nATOM    163 HG12 VAL A  12       8.134  -0.800  30.188  1.00 41.68           H  \\nATOM    164 HG13 VAL A  12       8.281  -2.538  29.838  1.00 41.68           H  \\nATOM    165  CG2 VAL A  12      10.582  -1.626  30.970  1.00 41.68           C  \\nATOM    166 HG21 VAL A  12      11.642  -1.388  31.058  1.00 41.68           H  \\nATOM    167 HG22 VAL A  12      10.422  -2.664  31.260  1.00 41.68           H  \\nATOM    168 HG23 VAL A  12      10.034  -0.974  31.650  1.00 41.68           H  \\nATOM    169  N   GLU A  13      10.495  -2.828  26.144  1.00 42.60           N  \\nATOM    170  H   GLU A  13      10.466  -3.801  26.415  1.00 42.60           H  \\nATOM    171  CA  GLU A  13      10.066  -2.469  24.803  1.00 42.60           C  \\nATOM    172  HA  GLU A  13      10.838  -1.881  24.306  1.00 42.60           H  \\nATOM    173  C   GLU A  13       8.820  -1.608  24.997  1.00 42.60           C  \\nATOM    174  CB  GLU A  13       9.750  -3.711  23.951  1.00 42.60           C  \\nATOM    175  HB2 GLU A  13       9.167  -4.416  24.543  1.00 42.60           H  \\nATOM    176  HB3 GLU A  13       9.145  -3.389  23.103  1.00 42.60           H  \\nATOM    177  O   GLU A  13       7.762  -2.098  25.394  1.00 42.60           O  \\nATOM    178  CG  GLU A  13      11.002  -4.408  23.394  1.00 42.60           C  \\nATOM    179  HG2 GLU A  13      11.609  -3.664  22.878  1.00 42.60           H  \\nATOM    180  HG3 GLU A  13      11.586  -4.800  24.227  1.00 42.60           H  \\nATOM    181  CD  GLU A  13      10.658  -5.554  22.420  1.00 42.60           C  \\nATOM    182  OE1 GLU A  13      11.542  -5.907  21.606  1.00 42.60           O  \\nATOM    183  OE2 GLU A  13       9.516  -6.067  22.475  1.00 42.60           O  \\nATOM    184  N   ASN A  14       8.973  -0.297  24.802  1.00 38.11           N  \\nATOM    185  H   ASN A  14       9.914   0.045  24.669  1.00 38.11           H  \\nATOM    186  CA  ASN A  14       7.879   0.660  24.783  1.00 38.11           C  \\nATOM    187  HA  ASN A  14       7.242   0.497  25.651  1.00 38.11           H  \\nATOM    188  C   ASN A  14       7.031   0.379  23.537  1.00 38.11           C  \\nATOM    189  CB  ASN A  14       8.435   2.099  24.828  1.00 38.11           C  \\nATOM    190  HB2 ASN A  14       9.329   2.173  24.209  1.00 38.11           H  \\nATOM    191  HB3 ASN A  14       7.691   2.783  24.420  1.00 38.11           H  \\nATOM    192  O   ASN A  14       7.023   1.144  22.574  1.00 38.11           O  \\nATOM    193  CG  ASN A  14       8.719   2.596  26.233  1.00 38.11           C  \\nATOM    194  ND2 ASN A  14       9.809   3.298  26.433  1.00 38.11           N  \\nATOM    195 HD21 ASN A  14      10.489   3.415  25.696  1.00 38.11           H  \\nATOM    196 HD22 ASN A  14       9.979   3.566  27.392  1.00 38.11           H  \\nATOM    197  OD1 ASN A  14       7.943   2.420  27.154  1.00 38.11           O  \\nATOM    198  N   VAL A  15       6.327  -0.750  23.535  1.00 48.38           N  \\nATOM    199  H   VAL A  15       6.467  -1.387  24.307  1.00 48.38           H  \\nATOM    200  CA  VAL A  15       5.195  -0.975  22.654  1.00 48.38           C  \\nATOM    201  HA  VAL A  15       5.479  -0.737  21.629  1.00 48.38           H  \\nATOM    202  C   VAL A  15       4.148   0.011  23.142  1.00 48.38           C  \\nATOM    203  CB  VAL A  15       4.697  -2.432  22.710  1.00 48.38           C  \\nATOM    204  HB  VAL A  15       4.300  -2.653  23.701  1.00 48.38           H  \\nATOM    205  O   VAL A  15       3.560  -0.181  24.206  1.00 48.38           O  \\nATOM    206  CG1 VAL A  15       3.587  -2.647  21.672  1.00 48.38           C  \\nATOM    207 HG11 VAL A  15       3.969  -2.450  20.670  1.00 48.38           H  \\nATOM    208 HG12 VAL A  15       3.232  -3.676  21.726  1.00 48.38           H  \\nATOM    209 HG13 VAL A  15       2.750  -1.980  21.876  1.00 48.38           H  \\nATOM    210  CG2 VAL A  15       5.824  -3.432  22.415  1.00 48.38           C  \\nATOM    211 HG21 VAL A  15       6.293  -3.207  21.457  1.00 48.38           H  \\nATOM    212 HG22 VAL A  15       5.429  -4.448  22.391  1.00 48.38           H  \\nATOM    213 HG23 VAL A  15       6.582  -3.389  23.197  1.00 48.38           H  \\nATOM    214  N   ALA A  16       3.982   1.114  22.411  1.00 55.85           N  \\nATOM    215  H   ALA A  16       4.594   1.265  21.621  1.00 55.85           H  \\nATOM    216  CA  ALA A  16       2.934   2.087  22.676  1.00 55.85           C  \\nATOM    217  HA  ALA A  16       3.193   2.646  23.575  1.00 55.85           H  \\nATOM    218  C   ALA A  16       1.619   1.329  22.912  1.00 55.85           C  \\nATOM    219  CB  ALA A  16       2.848   3.059  21.493  1.00 55.85           C  \\nATOM    220  HB1 ALA A  16       2.608   2.517  20.578  1.00 55.85           H  \\nATOM    221  HB2 ALA A  16       3.800   3.575  21.363  1.00 55.85           H  \\nATOM    222  HB3 ALA A  16       2.068   3.797  21.680  1.00 55.85           H  \\nATOM    223  O   ALA A  16       1.166   0.566  22.053  1.00 55.85           O  \\nATOM    224  N   LEU A  17       1.071   1.461  24.123  1.00 55.23           N  \\nATOM    225  H   LEU A  17       1.538   2.047  24.800  1.00 55.23           H  \\nATOM    226  CA  LEU A  17      -0.118   0.729  24.536  1.00 55.23           C  \\nATOM    227  HA  LEU A  17       0.104  -0.338  24.490  1.00 55.23           H  \\nATOM    228  C   LEU A  17      -1.253   1.028  23.557  1.00 55.23           C  \\nATOM    229  CB  LEU A  17      -0.509   1.113  25.975  1.00 55.23           C  \\nATOM    230  HB2 LEU A  17      -0.491   2.200  26.052  1.00 55.23           H  \\nATOM    231  HB3 LEU A  17      -1.535   0.794  26.162  1.00 55.23           H  \\nATOM    232  O   LEU A  17      -1.531   2.185  23.237  1.00 55.23           O  \\nATOM    233  CG  LEU A  17       0.398   0.507  27.062  1.00 55.23           C  \\nATOM    234  HG  LEU A  17       1.444   0.630  26.779  1.00 55.23           H  \\nATOM    235  CD1 LEU A  17       0.171   1.230  28.388  1.00 55.23           C  \\nATOM    236 HD11 LEU A  17      -0.869   1.128  28.701  1.00 55.23           H  \\nATOM    237 HD12 LEU A  17       0.412   2.288  28.277  1.00 55.23           H  \\nATOM    238 HD13 LEU A  17       0.822   0.808  29.153  1.00 55.23           H  \\nATOM    239  CD2 LEU A  17       0.102  -0.981  27.278  1.00 55.23           C  \\nATOM    240 HD21 LEU A  17       0.326  -1.539  26.369  1.00 55.23           H  \\nATOM    241 HD22 LEU A  17      -0.943  -1.122  27.555  1.00 55.23           H  \\nATOM    242 HD23 LEU A  17       0.741  -1.363  28.074  1.00 55.23           H  \\nATOM    243  N   LYS A  18      -1.919  -0.034  23.103  1.00 69.25           N  \\nATOM    244  H   LYS A  18      -1.588  -0.948  23.379  1.00 69.25           H  \\nATOM    245  CA  LYS A  18      -3.141   0.076  22.310  1.00 69.25           C  \\nATOM    246  HA  LYS A  18      -2.900   0.530  21.349  1.00 69.25           H  \\nATOM    247  C   LYS A  18      -4.152   0.951  23.030  1.00 69.25           C  \\nATOM    248  CB  LYS A  18      -3.775  -1.293  22.106  1.00 69.25           C  \\nATOM    249  HB2 LYS A  18      -4.759  -1.165  21.653  1.00 69.25           H  \\nATOM    250  HB3 LYS A  18      -3.902  -1.772  23.077  1.00 69.25           H  \\nATOM    251  O   LYS A  18      -4.375   0.783  24.228  1.00 69.25           O  \\nATOM    252  CG  LYS A  18      -2.925  -2.172  21.202  1.00 69.25           C  \\nATOM    253  HG2 LYS A  18      -1.907  -2.235  21.587  1.00 69.25           H  \\nATOM    254  HG3 LYS A  18      -2.900  -1.726  20.207  1.00 69.25           H  \\nATOM    255  CD  LYS A  18      -3.566  -3.556  21.126  1.00 69.25           C  \\nATOM    256  HD2 LYS A  18      -4.440  -3.617  21.774  1.00 69.25           H  \\nATOM    257  HD3 LYS A  18      -3.893  -3.738  20.102  1.00 69.25           H  \\nATOM    258  CE  LYS A  18      -2.546  -4.600  21.543  1.00 69.25           C  \\nATOM    259  HE2 LYS A  18      -1.741  -4.618  20.808  1.00 69.25           H  \\nATOM    260  HE3 LYS A  18      -2.104  -4.320  22.499  1.00 69.25           H  \\nATOM    261  NZ  LYS A  18      -3.217  -5.911  21.642  1.00 69.25           N  \\nATOM    262  HZ1 LYS A  18      -3.666  -6.003  22.542  1.00 69.25           H  \\nATOM    263  HZ2 LYS A  18      -3.911  -5.990  20.913  1.00 69.25           H  \\nATOM    264  HZ3 LYS A  18      -2.517  -6.629  21.514  1.00 69.25           H  \\nATOM    265  N   THR A  19      -4.825   1.806  22.274  1.00 85.36           N  \\nATOM    266  H   THR A  19      -4.600   1.847  21.290  1.00 85.36           H  \\nATOM    267  CA  THR A  19      -6.026   2.502  22.746  1.00 85.36           C  \\nATOM    268  HA  THR A  19      -6.276   2.147  23.746  1.00 85.36           H  \\nATOM    269  C   THR A  19      -7.215   2.149  21.861  1.00 85.36           C  \\nATOM    270  CB  THR A  19      -5.812   4.018  22.863  1.00 85.36           C  \\nATOM    271  HB  THR A  19      -6.740   4.479  23.199  1.00 85.36           H  \\nATOM    272  O   THR A  19      -7.031   1.762  20.710  1.00 85.36           O  \\nATOM    273  CG2 THR A  19      -4.705   4.368  23.858  1.00 85.36           C  \\nATOM    274 HG21 THR A  19      -4.936   3.930  24.829  1.00 85.36           H  \\nATOM    275 HG22 THR A  19      -3.743   3.986  23.518  1.00 85.36           H  \\nATOM    276 HG23 THR A  19      -4.640   5.450  23.967  1.00 85.36           H  \\nATOM    277  OG1 THR A  19      -5.440   4.589  21.634  1.00 85.36           O  \\nATOM    278  HG1 THR A  19      -4.545   4.288  21.461  1.00 85.36           H  \\nATOM    279  N   GLY A  20      -8.421   2.258  22.413  1.00 90.73           N  \\nATOM    280  H   GLY A  20      -8.488   2.558  23.376  1.00 90.73           H  \\nATOM    281  CA  GLY A  20      -9.677   1.904  21.750  1.00 90.73           C  \\nATOM    282  HA2 GLY A  20     -10.473   2.452  22.253  1.00 90.73           H  \\nATOM    283  HA3 GLY A  20      -9.665   2.217  20.706  1.00 90.73           H  \\nATOM    284  C   GLY A  20     -10.029   0.415  21.817  1.00 90.73           C  \\nATOM    285  O   GLY A  20      -9.157  -0.452  21.910  1.00 90.73           O  \\nATOM    286  N   SER A  21     -11.329   0.118  21.816  1.00 93.49           N  \\nATOM    287  H   SER A  21     -11.994   0.877  21.762  1.00 93.49           H  \\nATOM    288  CA  SER A  21     -11.857  -1.249  21.947  1.00 93.49           C  \\nATOM    289  HA  SER A  21     -11.307  -1.764  22.735  1.00 93.49           H  \\nATOM    290  C   SER A  21     -11.710  -2.081  20.672  1.00 93.49           C  \\nATOM    291  CB  SER A  21     -13.337  -1.208  22.347  1.00 93.49           C  \\nATOM    292  HB2 SER A  21     -13.699  -2.228  22.477  1.00 93.49           H  \\nATOM    293  HB3 SER A  21     -13.449  -0.672  23.290  1.00 93.49           H  \\nATOM    294  O   SER A  21     -11.608  -3.305  20.738  1.00 93.49           O  \\nATOM    295  OG  SER A  21     -14.104  -0.566  21.348  1.00 93.49           O  \\nATOM    296  HG  SER A  21     -13.995   0.377  21.492  1.00 93.49           H  \\nATOM    297  N   GLY A  22     -11.710  -1.433  19.505  1.00 95.33           N  \\nATOM    298  H   GLY A  22     -11.842  -0.432  19.525  1.00 95.33           H  \\nATOM    299  CA  GLY A  22     -11.819  -2.096  18.208  1.00 95.33           C  \\nATOM    300  HA2 GLY A  22     -11.172  -2.973  18.200  1.00 95.33           H  \\nATOM    301  HA3 GLY A  22     -11.496  -1.405  17.429  1.00 95.33           H  \\nATOM    302  C   GLY A  22     -13.228  -2.569  17.861  1.00 95.33           C  \\nATOM    303  O   GLY A  22     -13.411  -3.198  16.818  1.00 95.33           O  \\nATOM    304  N   ASP A  23     -14.225  -2.278  18.698  1.00 94.12           N  \\nATOM    305  H   ASP A  23     -14.011  -1.770  19.544  1.00 94.12           H  \\nATOM    306  CA  ASP A  23     -15.595  -2.762  18.515  1.00 94.12           C  \\nATOM    307  HA  ASP A  23     -15.565  -3.794  18.167  1.00 94.12           H  \\nATOM    308  C   ASP A  23     -16.373  -1.975  17.459  1.00 94.12           C  \\nATOM    309  CB  ASP A  23     -16.335  -2.706  19.858  1.00 94.12           C  \\nATOM    310  HB2 ASP A  23     -17.404  -2.778  19.661  1.00 94.12           H  \\nATOM    311  HB3 ASP A  23     -16.163  -1.746  20.345  1.00 94.12           H  \\nATOM    312  O   ASP A  23     -17.347  -2.477  16.896  1.00 94.12           O  \\nATOM    313  CG  ASP A  23     -15.981  -3.857  20.799  1.00 94.12           C  \\nATOM    314  OD1 ASP A  23     -15.345  -4.838  20.350  1.00 94.12           O  \\nATOM    315  OD2 ASP A  23     -16.635  -3.965  21.854  1.00 94.12           O  \\nATOM    316  N   SER A  24     -15.940  -0.749  17.169  1.00 95.40           N  \\nATOM    317  H   SER A  24     -15.090  -0.429  17.610  1.00 95.40           H  \\nATOM    318  CA  SER A  24     -16.583   0.135  16.202  1.00 95.40           C  \\nATOM    319  HA  SER A  24     -17.167  -0.466  15.505  1.00 95.40           H  \\nATOM    320  C   SER A  24     -15.552   0.910  15.392  1.00 95.40           C  \\nATOM    321  CB  SER A  24     -17.541   1.095  16.917  1.00 95.40           C  \\nATOM    322  HB2 SER A  24     -18.212   0.531  17.565  1.00 95.40           H  \\nATOM    323  HB3 SER A  24     -18.141   1.619  16.173  1.00 95.40           H  \\nATOM    324  O   SER A  24     -14.416   1.105  15.818  1.00 95.40           O  \\nATOM    325  OG  SER A  24     -16.829   2.053  17.678  1.00 95.40           O  \\nATOM    326  HG  SER A  24     -16.436   1.610  18.433  1.00 95.40           H  \\nATOM    327  N   GLN A  25     -15.972   1.416  14.235  1.00 96.15           N  \\nATOM    328  H   GLN A  25     -16.903   1.196  13.910  1.00 96.15           H  \\nATOM    329  CA  GLN A  25     -15.136   2.283  13.409  1.00 96.15           C  \\nATOM    330  HA  GLN A  25     -14.209   1.757  13.180  1.00 96.15           H  \\nATOM    331  C   GLN A  25     -14.764   3.597  14.120  1.00 96.15           C  \\nATOM    332  CB  GLN A  25     -15.892   2.550  12.104  1.00 96.15           C  \\nATOM    333  HB2 GLN A  25     -16.197   1.596  11.674  1.00 96.15           H  \\nATOM    334  HB3 GLN A  25     -16.787   3.136  12.315  1.00 96.15           H  \\nATOM    335  O   GLN A  25     -13.702   4.152  13.857  1.00 96.15           O  \\nATOM    336  CG  GLN A  25     -15.030   3.290  11.074  1.00 96.15           C  \\nATOM    337  HG2 GLN A  25     -14.754   4.273  11.457  1.00 96.15           H  \\nATOM    338  HG3 GLN A  25     -14.119   2.721  10.894  1.00 96.15           H  \\nATOM    339  CD  GLN A  25     -15.740   3.493   9.743  1.00 96.15           C  \\nATOM    340  NE2 GLN A  25     -15.088   4.114   8.786  1.00 96.15           N  \\nATOM    341 HE21 GLN A  25     -14.149   4.453   8.942  1.00 96.15           H  \\nATOM    342 HE22 GLN A  25     -15.604   4.270   7.932  1.00 96.15           H  \\nATOM    343  OE1 GLN A  25     -16.879   3.114   9.526  1.00 96.15           O  \\nATOM    344  N   SER A  26     -15.617   4.095  15.023  1.00 96.88           N  \\nATOM    345  H   SER A  26     -16.479   3.603  15.210  1.00 96.88           H  \\nATOM    346  CA  SER A  26     -15.353   5.292  15.836  1.00 96.88           C  \\nATOM    347  HA  SER A  26     -14.879   6.047  15.210  1.00 96.88           H  \\nATOM    348  C   SER A  26     -14.406   5.041  17.013  1.00 96.88           C  \\nATOM    349  CB  SER A  26     -16.670   5.875  16.361  1.00 96.88           C  \\nATOM    350  HB2 SER A  26     -17.186   6.370  15.538  1.00 96.88           H  \\nATOM    351  HB3 SER A  26     -16.466   6.617  17.132  1.00 96.88           H  \\nATOM    352  O   SER A  26     -13.843   5.995  17.543  1.00 96.88           O  \\nATOM    353  OG  SER A  26     -17.519   4.865  16.878  1.00 96.88           O  \\nATOM    354  HG  SER A  26     -17.069   4.362  17.561  1.00 96.88           H  \\nATOM    355  N   ASP A  27     -14.216   3.780  17.406  1.00 96.80           N  \\nATOM    356  H   ASP A  27     -14.675   3.039  16.896  1.00 96.80           H  \\nATOM    357  CA  ASP A  27     -13.320   3.372  18.490  1.00 96.80           C  \\nATOM    358  HA  ASP A  27     -12.660   4.204  18.736  1.00 96.80           H  \\nATOM    359  C   ASP A  27     -12.410   2.204  18.054  1.00 96.80           C  \\nATOM    360  CB  ASP A  27     -14.153   3.068  19.745  1.00 96.80           C  \\nATOM    361  HB2 ASP A  27     -14.784   2.197  19.565  1.00 96.80           H  \\nATOM    362  HB3 ASP A  27     -14.797   3.919  19.962  1.00 96.80           H  \\nATOM    363  O   ASP A  27     -12.517   1.094  18.584  1.00 96.80           O  \\nATOM    364  CG  ASP A  27     -13.254   2.818  20.954  1.00 96.80           C  \\nATOM    365  OD1 ASP A  27     -12.219   3.505  21.070  1.00 96.80           O  \\nATOM    366  OD2 ASP A  27     -13.574   1.905  21.747  1.00 96.80           O  \\nATOM    367  N   PRO A  28     -11.549   2.405  17.036  1.00 97.78           N  \\nATOM    368  CA  PRO A  28     -10.664   1.361  16.538  1.00 97.78           C  \\nATOM    369  HA  PRO A  28     -11.220   0.437  16.381  1.00 97.78           H  \\nATOM    370  C   PRO A  28      -9.506   1.111  17.509  1.00 97.78           C  \\nATOM    371  CB  PRO A  28     -10.160   1.888  15.195  1.00 97.78           C  \\nATOM    372  HB2 PRO A  28     -10.900   1.662  14.427  1.00 97.78           H  \\nATOM    373  HB3 PRO A  28      -9.186   1.480  14.925  1.00 97.78           H  \\nATOM    374  O   PRO A  28      -9.089   2.020  18.226  1.00 97.78           O  \\nATOM    375  CG  PRO A  28     -10.097   3.399  15.415  1.00 97.78           C  \\nATOM    376  HG2 PRO A  28     -10.182   3.949  14.478  1.00 97.78           H  \\nATOM    377  HG3 PRO A  28      -9.173   3.660  15.930  1.00 97.78           H  \\nATOM    378  CD  PRO A  28     -11.293   3.658  16.332  1.00 97.78           C  \\nATOM    379  HD2 PRO A  28     -12.160   3.918  15.725  1.00 97.78           H  \\nATOM    380  HD3 PRO A  28     -11.067   4.465  17.030  1.00 97.78           H  \\nATOM    381  N   ILE A  29      -8.902  -0.079  17.450  1.00 97.52           N  \\nATOM    382  H   ILE A  29      -9.287  -0.794  16.849  1.00 97.52           H  \\nATOM    383  CA  ILE A  29      -7.587  -0.307  18.062  1.00 97.52           C  \\nATOM    384  HA  ILE A  29      -7.646  -0.029  19.114  1.00 97.52           H  \\nATOM    385  C   ILE A  29      -6.569   0.598  17.369  1.00 97.52           C  \\nATOM    386  CB  ILE A  29      -7.143  -1.784  17.981  1.00 97.52           C  \\nATOM    387  HB  ILE A  29      -7.152  -2.101  16.938  1.00 97.52           H  \\nATOM    388  O   ILE A  29      -6.280   0.423  16.185  1.00 97.52           O  \\nATOM    389  CG1 ILE A  29      -8.135  -2.666  18.765  1.00 97.52           C  \\nATOM    390 HG12 ILE A  29      -9.092  -2.629  18.243  1.00 97.52           H  \\nATOM    391 HG13 ILE A  29      -8.278  -2.263  19.768  1.00 97.52           H  \\nATOM    392  CG2 ILE A  29      -5.707  -1.947  18.519  1.00 97.52           C  \\nATOM    393 HG21 ILE A  29      -5.394  -2.989  18.442  1.00 97.52           H  \\nATOM    394 HG22 ILE A  29      -4.998  -1.358  17.937  1.00 97.52           H  \\nATOM    395 HG23 ILE A  29      -5.670  -1.628  19.561  1.00 97.52           H  \\nATOM    396  CD1 ILE A  29      -7.738  -4.140  18.881  1.00 97.52           C  \\nATOM    397 HD11 ILE A  29      -7.411  -4.520  17.913  1.00 97.52           H  \\nATOM    398 HD12 ILE A  29      -6.933  -4.247  19.609  1.00 97.52           H  \\nATOM    399 HD13 ILE A  29      -8.593  -4.716  19.236  1.00 97.52           H  \\nATOM    400  N   LYS A  30      -5.999   1.538  18.111  1.00 97.27           N  \\nATOM    401  H   LYS A  30      -6.349   1.647  19.052  1.00 97.27           H  \\nATOM    402  CA  LYS A  30      -4.934   2.435  17.663  1.00 97.27           C  \\nATOM    403  HA  LYS A  30      -4.919   2.491  16.574  1.00 97.27           H  \\nATOM    404  C   LYS A  30      -3.596   1.902  18.137  1.00 97.27           C  \\nATOM    405  CB  LYS A  30      -5.195   3.830  18.227  1.00 97.27           C  \\nATOM    406  HB2 LYS A  30      -5.355   3.736  19.301  1.00 97.27           H  \\nATOM    407  HB3 LYS A  30      -4.322   4.462  18.063  1.00 97.27           H  \\nATOM    408  O   LYS A  30      -3.406   1.724  19.341  1.00 97.27           O  \\nATOM    409  CG  LYS A  30      -6.423   4.476  17.584  1.00 97.27           C  \\nATOM    410  HG2 LYS A  30      -6.220   4.674  16.531  1.00 97.27           H  \\nATOM    411  HG3 LYS A  30      -7.278   3.804  17.659  1.00 97.27           H  \\nATOM    412  CD  LYS A  30      -6.767   5.783  18.294  1.00 97.27           C  \\nATOM    413  HD2 LYS A  30      -5.956   6.499  18.161  1.00 97.27           H  \\nATOM    414  HD3 LYS A  30      -6.908   5.583  19.356  1.00 97.27           H  \\nATOM    415  CE  LYS A  30      -8.065   6.325  17.697  1.00 97.27           C  \\nATOM    416  HE2 LYS A  30      -8.818   5.538  17.754  1.00 97.27           H  \\nATOM    417  HE3 LYS A  30      -7.902   6.549  16.643  1.00 97.27           H  \\nATOM    418  NZ  LYS A  30      -8.533   7.524  18.429  1.00 97.27           N  \\nATOM    419  HZ1 LYS A  30      -8.703   7.280  19.395  1.00 97.27           H  \\nATOM    420  HZ2 LYS A  30      -7.841   8.259  18.390  1.00 97.27           H  \\nATOM    421  HZ3 LYS A  30      -9.403   7.851  18.035  1.00 97.27           H  \\nATOM    422  N   ALA A  31      -2.695   1.642  17.199  1.00 96.27           N  \\nATOM    423  H   ALA A  31      -2.926   1.820  16.232  1.00 96.27           H  \\nATOM    424  CA  ALA A  31      -1.361   1.146  17.496  1.00 96.27           C  \\nATOM    425  HA  ALA A  31      -1.033   1.548  18.455  1.00 96.27           H  \\nATOM    426  C   ALA A  31      -0.352   1.594  16.438  1.00 96.27           C  \\nATOM    427  CB  ALA A  31      -1.413  -0.382  17.598  1.00 96.27           C  \\nATOM    428  HB1 ALA A  31      -2.059  -0.668  18.428  1.00 96.27           H  \\nATOM    429  HB2 ALA A  31      -1.812  -0.805  16.677  1.00 96.27           H  \\nATOM    430  HB3 ALA A  31      -0.413  -0.777  17.778  1.00 96.27           H  \\nATOM    431  O   ALA A  31      -0.685   1.752  15.261  1.00 96.27           O  \\nATOM    432  N   ASP A  32       0.898   1.738  16.860  1.00 95.56           N  \\nATOM    433  H   ASP A  32       1.108   1.588  17.837  1.00 95.56           H  \\nATOM    434  CA  ASP A  32       2.011   2.000  15.958  1.00 95.56           C  \\nATOM    435  HA  ASP A  32       1.672   2.669  15.167  1.00 95.56           H  \\nATOM    436  C   ASP A  32       2.478   0.702  15.281  1.00 95.56           C  \\nATOM    437  CB  ASP A  32       3.143   2.713  16.713  1.00 95.56           C  \\nATOM    438  HB2 ASP A  32       4.022   2.742  16.070  1.00 95.56           H  \\nATOM    439  HB3 ASP A  32       3.399   2.139  17.604  1.00 95.56           H  \\nATOM    440  O   ASP A  32       2.321  -0.402  15.812  1.00 95.56           O  \\nATOM    441  CG  ASP A  32       2.810   4.158  17.115  1.00 95.56           C  \\nATOM    442  OD1 ASP A  32       1.749   4.684  16.695  1.00 95.56           O  \\nATOM    443  OD2 ASP A  32       3.673   4.776  17.775  1.00 95.56           O  \\nATOM    444  N   LEU A  33       3.061   0.840  14.092  1.00 97.05           N  \\nATOM    445  H   LEU A  33       3.151   1.772  13.714  1.00 97.05           H  \\nATOM    446  CA  LEU A  33       3.641  -0.256  13.325  1.00 97.05           C  \\nATOM    447  HA  LEU A  33       3.576  -1.173  13.911  1.00 97.05           H  \\nATOM    448  C   LEU A  33       5.123   0.024  13.073  1.00 97.05           C  \\nATOM    449  CB  LEU A  33       2.809  -0.465  12.047  1.00 97.05           C  \\nATOM    450  HB2 LEU A  33       1.778  -0.630  12.360  1.00 97.05           H  \\nATOM    451  HB3 LEU A  33       2.833   0.441  11.442  1.00 97.05           H  \\nATOM    452  O   LEU A  33       5.487   1.019  12.447  1.00 97.05           O  \\nATOM    453  CG  LEU A  33       3.267  -1.667  11.198  1.00 97.05           C  \\nATOM    454  HG  LEU A  33       3.678  -2.440  11.848  1.00 97.05           H  \\nATOM    455  CD1 LEU A  33       2.086  -2.271  10.434  1.00 97.05           C  \\nATOM    456 HD11 LEU A  33       1.284  -2.529  11.126  1.00 97.05           H  \\nATOM    457 HD12 LEU A  33       2.409  -3.175   9.916  1.00 97.05           H  \\nATOM    458 HD13 LEU A  33       1.721  -1.556   9.698  1.00 97.05           H  \\nATOM    459  CD2 LEU A  33       4.311  -1.266  10.155  1.00 97.05           C  \\nATOM    460 HD21 LEU A  33       3.907  -0.493   9.501  1.00 97.05           H  \\nATOM    461 HD22 LEU A  33       4.575  -2.128   9.542  1.00 97.05           H  \\nATOM    462 HD23 LEU A  33       5.221  -0.897  10.629  1.00 97.05           H  \\nATOM    463  N   GLU A  34       5.981  -0.880  13.536  1.00 97.46           N  \\nATOM    464  H   GLU A  34       5.631  -1.667  14.063  1.00 97.46           H  \\nATOM    465  CA  GLU A  34       7.415  -0.824  13.267  1.00 97.46           C  \\nATOM    466  HA  GLU A  34       7.738   0.217  13.277  1.00 97.46           H  \\nATOM    467  C   GLU A  34       7.719  -1.410  11.881  1.00 97.46           C  \\nATOM    468  CB  GLU A  34       8.175  -1.557  14.378  1.00 97.46           C  \\nATOM    469  HB2 GLU A  34       7.810  -1.226  15.350  1.00 97.46           H  \\nATOM    470  HB3 GLU A  34       7.983  -2.625  14.279  1.00 97.46           H  \\nATOM    471  O   GLU A  34       7.550  -2.606  11.634  1.00 97.46           O  \\nATOM    472  CG  GLU A  34       9.683  -1.277  14.300  1.00 97.46           C  \\nATOM    473  HG2 GLU A  34      10.009  -1.370  13.264  1.00 97.46           H  \\nATOM    474  HG3 GLU A  34       9.868  -0.250  14.616  1.00 97.46           H  \\nATOM    475  CD  GLU A  34      10.522  -2.243  15.148  1.00 97.46           C  \\nATOM    476  OE1 GLU A  34      11.758  -2.199  14.982  1.00 97.46           O  \\nATOM    477  OE2 GLU A  34       9.960  -3.165  15.789  1.00 97.46           O  \\nATOM    478  N   VAL A  35       8.178  -0.559  10.965  1.00 98.17           N  \\nATOM    479  H   VAL A  35       8.313   0.406  11.231  1.00 98.17           H  \\nATOM    480  CA  VAL A  35       8.574  -0.938   9.608  1.00 98.17           C  \\nATOM    481  HA  VAL A  35       7.866  -1.674   9.227  1.00 98.17           H  \\nATOM    482  C   VAL A  35       9.966  -1.564   9.637  1.00 98.17           C  \\nATOM    483  CB  VAL A  35       8.539   0.288   8.681  1.00 98.17           C  \\nATOM    484  HB  VAL A  35       9.202   1.059   9.074  1.00 98.17           H  \\nATOM    485  O   VAL A  35      10.909  -0.984  10.180  1.00 98.17           O  \\nATOM    486  CG1 VAL A  35       9.021  -0.041   7.271  1.00 98.17           C  \\nATOM    487 HG11 VAL A  35      10.065  -0.354   7.279  1.00 98.17           H  \\nATOM    488 HG12 VAL A  35       8.956   0.870   6.676  1.00 98.17           H  \\nATOM    489 HG13 VAL A  35       8.406  -0.825   6.827  1.00 98.17           H  \\nATOM    490  CG2 VAL A  35       7.121   0.865   8.578  1.00 98.17           C  \\nATOM    491 HG21 VAL A  35       7.109   1.710   7.890  1.00 98.17           H  \\nATOM    492 HG22 VAL A  35       6.788   1.214   9.556  1.00 98.17           H  \\nATOM    493 HG23 VAL A  35       6.430   0.101   8.220  1.00 98.17           H  \\nATOM    494  N   LYS A  36      10.114  -2.736   9.007  1.00 95.80           N  \\nATOM    495  H   LYS A  36       9.303  -3.120   8.543  1.00 95.80           H  \\nATOM    496  CA  LYS A  36      11.365  -3.515   9.003  1.00 95.80           C  \\nATOM    497  HA  LYS A  36      12.149  -2.914   9.462  1.00 95.80           H  \\nATOM    498  C   LYS A  36      11.825  -3.847   7.590  1.00 95.80           C  \\nATOM    499  CB  LYS A  36      11.212  -4.793   9.854  1.00 95.80           C  \\nATOM    500  HB2 LYS A  36      10.402  -5.406   9.458  1.00 95.80           H  \\nATOM    501  HB3 LYS A  36      12.139  -5.364   9.808  1.00 95.80           H  \\nATOM    502  O   LYS A  36      11.052  -4.370   6.787  1.00 95.80           O  \\nATOM    503  CG  LYS A  36      10.921  -4.440  11.317  1.00 95.80           C  \\nATOM    504  HG2 LYS A  36      11.686  -3.742  11.656  1.00 95.80           H  \\nATOM    505  HG3 LYS A  36       9.957  -3.933  11.365  1.00 95.80           H  \\nATOM    506  CD  LYS A  36      10.882  -5.633  12.286  1.00 95.80           C  \\nATOM    507  HD2 LYS A  36      11.855  -6.123  12.324  1.00 95.80           H  \\nATOM    508  HD3 LYS A  36      10.097  -6.334  12.004  1.00 95.80           H  \\nATOM    509  CE  LYS A  36      10.559  -4.957  13.621  1.00 95.80           C  \\nATOM    510  HE2 LYS A  36       9.591  -4.462  13.538  1.00 95.80           H  \\nATOM    511  HE3 LYS A  36      11.286  -4.160  13.770  1.00 95.80           H  \\nATOM    512  NZ  LYS A  36      10.566  -5.789  14.843  1.00 95.80           N  \\nATOM    513  HZ1 LYS A  36       9.846  -6.496  14.851  1.00 95.80           H  \\nATOM    514  HZ2 LYS A  36      10.367  -5.132  15.584  1.00 95.80           H  \\nATOM    515  HZ3 LYS A  36      11.484  -6.160  15.045  1.00 95.80           H  \\nATOM    516  N   GLY A  37      13.103  -3.604   7.307  1.00 92.85           N  \\nATOM    517  H   GLY A  37      13.660  -3.127   8.001  1.00 92.85           H  \\nATOM    518  CA  GLY A  37      13.760  -4.027   6.069  1.00 92.85           C  \\nATOM    519  HA2 GLY A  37      13.884  -5.110   6.091  1.00 92.85           H  \\nATOM    520  HA3 GLY A  37      14.751  -3.575   6.014  1.00 92.85           H  \\nATOM    521  C   GLY A  37      12.993  -3.631   4.803  1.00 92.85           C  \\nATOM    522  O   GLY A  37      12.586  -2.485   4.631  1.00 92.85           O  \\nATOM    523  N   GLN A  38      12.769  -4.599   3.913  1.00 95.67           N  \\nATOM    524  H   GLN A  38      13.084  -5.535   4.122  1.00 95.67           H  \\nATOM    525  CA  GLN A  38      12.152  -4.351   2.607  1.00 95.67           C  \\nATOM    526  HA  GLN A  38      12.588  -3.437   2.206  1.00 95.67           H  \\nATOM    527  C   GLN A  38      10.639  -4.079   2.651  1.00 95.67           C  \\nATOM    528  CB  GLN A  38      12.509  -5.489   1.640  1.00 95.67           C  \\nATOM    529  HB2 GLN A  38      11.923  -5.382   0.727  1.00 95.67           H  \\nATOM    530  HB3 GLN A  38      12.271  -6.453   2.090  1.00 95.67           H  \\nATOM    531  O   GLN A  38      10.088  -3.691   1.618  1.00 95.67           O  \\nATOM    532  CG  GLN A  38      13.995  -5.427   1.254  1.00 95.67           C  \\nATOM    533  HG2 GLN A  38      14.621  -5.642   2.120  1.00 95.67           H  \\nATOM    534  HG3 GLN A  38      14.230  -4.421   0.905  1.00 95.67           H  \\nATOM    535  CD  GLN A  38      14.345  -6.411   0.148  1.00 95.67           C  \\nATOM    536  NE2 GLN A  38      15.236  -6.074  -0.758  1.00 95.67           N  \\nATOM    537 HE21 GLN A  38      15.770  -5.220  -0.686  1.00 95.67           H  \\nATOM    538 HE22 GLN A  38      15.498  -6.795  -1.415  1.00 95.67           H  \\nATOM    539  OE1 GLN A  38      13.834  -7.512   0.082  1.00 95.67           O  \\nATOM    540  N   SER A  39       9.975  -4.203   3.810  1.00 97.70           N  \\nATOM    541  H   SER A  39      10.465  -4.547   4.623  1.00 97.70           H  \\nATOM    542  CA  SER A  39       8.582  -3.751   3.979  1.00 97.70           C  \\nATOM    543  HA  SER A  39       7.986  -4.181   3.174  1.00 97.70           H  \\nATOM    544  C   SER A  39       8.425  -2.228   3.882  1.00 97.70           C  \\nATOM    545  CB  SER A  39       7.973  -4.238   5.298  1.00 97.70           C  \\nATOM    546  HB2 SER A  39       6.911  -3.994   5.294  1.00 97.70           H  \\nATOM    547  HB3 SER A  39       8.070  -5.321   5.373  1.00 97.70           H  \\nATOM    548  O   SER A  39       7.309  -1.744   3.721  1.00 97.70           O  \\nATOM    549  OG  SER A  39       8.579  -3.633   6.425  1.00 97.70           O  \\nATOM    550  HG  SER A  39       9.470  -3.983   6.502  1.00 97.70           H  \\nATOM    551  N   ALA A  40       9.528  -1.470   3.924  1.00 98.30           N  \\nATOM    552  H   ALA A  40      10.416  -1.911   4.118  1.00 98.30           H  \\nATOM    553  CA  ALA A  40       9.503  -0.014   3.836  1.00 98.30           C  \\nATOM    554  HA  ALA A  40       8.927   0.376   4.675  1.00 98.30           H  \\nATOM    555  C   ALA A  40       8.805   0.511   2.579  1.00 98.30           C  \\nATOM    556  CB  ALA A  40      10.942   0.501   3.938  1.00 98.30           C  \\nATOM    557  HB1 ALA A  40      11.536   0.147   3.096  1.00 98.30           H  \\nATOM    558  HB2 ALA A  40      11.395   0.152   4.866  1.00 98.30           H  \\nATOM    559  HB3 ALA A  40      10.940   1.591   3.942  1.00 98.30           H  \\nATOM    560  O   ALA A  40       7.970   1.399   2.688  1.00 98.30           O  \\nATOM    561  N   LEU A  41       9.103  -0.056   1.407  1.00 98.54           N  \\nATOM    562  H   LEU A  41       9.761  -0.821   1.402  1.00 98.54           H  \\nATOM    563  CA  LEU A  41       8.507   0.378   0.142  1.00 98.54           C  \\nATOM    564  HA  LEU A  41       8.696   1.446   0.037  1.00 98.54           H  \\nATOM    565  C   LEU A  41       6.973   0.222   0.113  1.00 98.54           C  \\nATOM    566  CB  LEU A  41       9.222  -0.334  -1.021  1.00 98.54           C  \\nATOM    567  HB2 LEU A  41      10.237   0.056  -1.095  1.00 98.54           H  \\nATOM    568  HB3 LEU A  41       9.285  -1.398  -0.792  1.00 98.54           H  \\nATOM    569  O   LEU A  41       6.300   1.216  -0.149  1.00 98.54           O  \\nATOM    570  CG  LEU A  41       8.526  -0.184  -2.384  1.00 98.54           C  \\nATOM    571  HG  LEU A  41       7.534  -0.628  -2.305  1.00 98.54           H  \\nATOM    572  CD1 LEU A  41       8.378   1.255  -2.869  1.00 98.54           C  \\nATOM    573 HD11 LEU A  41       7.768   1.270  -3.773  1.00 98.54           H  \\nATOM    574 HD12 LEU A  41       9.358   1.672  -3.100  1.00 98.54           H  \\nATOM    575 HD13 LEU A  41       7.889   1.875  -2.118  1.00 98.54           H  \\nATOM    576  CD2 LEU A  41       9.314  -0.953  -3.444  1.00 98.54           C  \\nATOM    577 HD21 LEU A  41       9.367  -2.007  -3.170  1.00 98.54           H  \\nATOM    578 HD22 LEU A  41       8.817  -0.852  -4.409  1.00 98.54           H  \\nATOM    579 HD23 LEU A  41      10.319  -0.537  -3.517  1.00 98.54           H  \\nATOM    580  N   PRO A  42       6.392  -0.968   0.361  1.00 98.55           N  \\nATOM    581  CA  PRO A  42       4.945  -1.127   0.300  1.00 98.55           C  \\nATOM    582  HA  PRO A  42       4.584  -0.780  -0.669  1.00 98.55           H  \\nATOM    583  C   PRO A  42       4.224  -0.308   1.375  1.00 98.55           C  \\nATOM    584  CB  PRO A  42       4.695  -2.630   0.401  1.00 98.55           C  \\nATOM    585  HB2 PRO A  42       4.639  -3.051  -0.603  1.00 98.55           H  \\nATOM    586  HB3 PRO A  42       3.790  -2.857   0.965  1.00 98.55           H  \\nATOM    587  O   PRO A  42       3.261   0.376   1.052  1.00 98.55           O  \\nATOM    588  CG  PRO A  42       5.932  -3.173   1.092  1.00 98.55           C  \\nATOM    589  HG2 PRO A  42       6.107  -4.219   0.838  1.00 98.55           H  \\nATOM    590  HG3 PRO A  42       5.829  -3.041   2.169  1.00 98.55           H  \\nATOM    591  CD  PRO A  42       7.023  -2.261   0.545  1.00 98.55           C  \\nATOM    592  HD2 PRO A  42       7.338  -2.613  -0.437  1.00 98.55           H  \\nATOM    593  HD3 PRO A  42       7.869  -2.232   1.232  1.00 98.55           H  \\nATOM    594  N   PHE A  43       4.748  -0.230   2.606  1.00 98.63           N  \\nATOM    595  H   PHE A  43       5.562  -0.775   2.852  1.00 98.63           H  \\nATOM    596  CA  PHE A  43       4.164   0.680   3.597  1.00 98.63           C  \\nATOM    597  HA  PHE A  43       3.101   0.453   3.679  1.00 98.63           H  \\nATOM    598  C   PHE A  43       4.227   2.147   3.162  1.00 98.63           C  \\nATOM    599  CB  PHE A  43       4.813   0.485   4.972  1.00 98.63           C  \\nATOM    600  HB2 PHE A  43       4.521   1.319   5.610  1.00 98.63           H  \\nATOM    601  HB3 PHE A  43       5.897   0.521   4.864  1.00 98.63           H  \\nATOM    602  O   PHE A  43       3.289   2.895   3.413  1.00 98.63           O  \\nATOM    603  CG  PHE A  43       4.397  -0.787   5.688  1.00 98.63           C  \\nATOM    604  CD1 PHE A  43       3.034  -1.085   5.875  1.00 98.63           C  \\nATOM    605  HD1 PHE A  43       2.273  -0.430   5.478  1.00 98.63           H  \\nATOM    606  CD2 PHE A  43       5.370  -1.662   6.202  1.00 98.63           C  \\nATOM    607  HD2 PHE A  43       6.418  -1.439   6.066  1.00 98.63           H  \\nATOM    608  CE1 PHE A  43       2.650  -2.242   6.573  1.00 98.63           C  \\nATOM    609  HE1 PHE A  43       1.600  -2.465   6.696  1.00 98.63           H  \\nATOM    610  CE2 PHE A  43       4.989  -2.822   6.894  1.00 98.63           C  \\nATOM    611  HE2 PHE A  43       5.747  -3.483   7.288  1.00 98.63           H  \\nATOM    612  CZ  PHE A  43       3.629  -3.111   7.085  1.00 98.63           C  \\nATOM    613  HZ  PHE A  43       3.340  -4.001   7.624  1.00 98.63           H  \\nATOM    614  N   ASP A  44       5.280   2.579   2.467  1.00 98.66           N  \\nATOM    615  H   ASP A  44       6.053   1.954   2.287  1.00 98.66           H  \\nATOM    616  CA  ASP A  44       5.356   3.954   1.968  1.00 98.66           C  \\nATOM    617  HA  ASP A  44       5.123   4.620   2.799  1.00 98.66           H  \\nATOM    618  C   ASP A  44       4.324   4.248   0.866  1.00 98.66           C  \\nATOM    619  CB  ASP A  44       6.784   4.254   1.509  1.00 98.66           C  \\nATOM    620  HB2 ASP A  44       7.488   3.713   2.141  1.00 98.66           H  \\nATOM    621  HB3 ASP A  44       6.924   3.931   0.477  1.00 98.66           H  \\nATOM    622  O   ASP A  44       3.785   5.356   0.812  1.00 98.66           O  \\nATOM    623  CG  ASP A  44       7.126   5.728   1.670  1.00 98.66           C  \\nATOM    624  OD1 ASP A  44       7.004   6.235   2.809  1.00 98.66           O  \\nATOM    625  OD2 ASP A  44       7.626   6.345   0.709  1.00 98.66           O  \\nATOM    626  N   VAL A  45       3.980   3.247   0.042  1.00 98.61           N  \\nATOM    627  H   VAL A  45       4.439   2.357   0.173  1.00 98.61           H  \\nATOM    628  CA  VAL A  45       2.832   3.312  -0.884  1.00 98.61           C  \\nATOM    629  HA  VAL A  45       2.957   4.172  -1.541  1.00 98.61           H  \\nATOM    630  C   VAL A  45       1.533   3.513  -0.098  1.00 98.61           C  \\nATOM    631  CB  VAL A  45       2.715   2.039  -1.753  1.00 98.61           C  \\nATOM    632  HB  VAL A  45       2.613   1.169  -1.103  1.00 98.61           H  \\nATOM    633  O   VAL A  45       0.757   4.412  -0.433  1.00 98.61           O  \\nATOM    634  CG1 VAL A  45       1.475   2.065  -2.659  1.00 98.61           C  \\nATOM    635 HG11 VAL A  45       1.510   2.920  -3.335  1.00 98.61           H  \\nATOM    636 HG12 VAL A  45       1.420   1.135  -3.225  1.00 98.61           H  \\nATOM    637 HG13 VAL A  45       0.568   2.114  -2.057  1.00 98.61           H  \\nATOM    638  CG2 VAL A  45       3.934   1.808  -2.656  1.00 98.61           C  \\nATOM    639 HG21 VAL A  45       3.933   0.771  -2.992  1.00 98.61           H  \\nATOM    640 HG22 VAL A  45       4.861   1.995  -2.115  1.00 98.61           H  \\nATOM    641 HG23 VAL A  45       3.873   2.459  -3.528  1.00 98.61           H  \\nATOM    642  N   ASP A  46       1.314   2.729   0.960  1.00 98.60           N  \\nATOM    643  H   ASP A  46       1.942   1.954   1.117  1.00 98.60           H  \\nATOM    644  CA  ASP A  46       0.122   2.817   1.817  1.00 98.60           C  \\nATOM    645  HA  ASP A  46      -0.765   2.623   1.214  1.00 98.60           H  \\nATOM    646  C   ASP A  46      -0.000   4.210   2.445  1.00 98.60           C  \\nATOM    647  CB  ASP A  46       0.180   1.776   2.952  1.00 98.60           C  \\nATOM    648  HB2 ASP A  46       0.975   2.033   3.651  1.00 98.60           H  \\nATOM    649  HB3 ASP A  46      -0.754   1.831   3.512  1.00 98.60           H  \\nATOM    650  O   ASP A  46      -1.045   4.863   2.388  1.00 98.60           O  \\nATOM    651  CG  ASP A  46       0.383   0.334   2.492  1.00 98.60           C  \\nATOM    652  OD1 ASP A  46      -0.039   0.029   1.361  1.00 98.60           O  \\nATOM    653  OD2 ASP A  46       0.910  -0.452   3.316  1.00 98.60           O  \\nATOM    654  N   CYS A  47       1.104   4.709   2.999  1.00 98.65           N  \\nATOM    655  H   CYS A  47       1.914   4.108   3.039  1.00 98.65           H  \\nATOM    656  CA  CYS A  47       1.178   6.024   3.615  1.00 98.65           C  \\nATOM    657  HA  CYS A  47       0.421   6.085   4.397  1.00 98.65           H  \\nATOM    658  C   CYS A  47       0.906   7.146   2.608  1.00 98.65           C  \\nATOM    659  CB  CYS A  47       2.556   6.178   4.253  1.00 98.65           C  \\nATOM    660  HB2 CYS A  47       3.314   5.973   3.497  1.00 98.65           H  \\nATOM    661  HB3 CYS A  47       2.671   7.210   4.585  1.00 98.65           H  \\nATOM    662  O   CYS A  47       0.199   8.099   2.941  1.00 98.65           O  \\nATOM    663  SG  CYS A  47       2.837   5.080   5.669  1.00 98.65           S  \\nATOM    664  N   TRP A  48       1.399   7.034   1.370  1.00 98.62           N  \\nATOM    665  H   TRP A  48       1.989   6.244   1.150  1.00 98.62           H  \\nATOM    666  CA  TRP A  48       1.050   7.968   0.300  1.00 98.62           C  \\nATOM    667  HA  TRP A  48       1.273   8.983   0.629  1.00 98.62           H  \\nATOM    668  C   TRP A  48      -0.442   7.916  -0.043  1.00 98.62           C  \\nATOM    669  CB  TRP A  48       1.878   7.679  -0.954  1.00 98.62           C  \\nATOM    670  HB2 TRP A  48       2.928   7.890  -0.753  1.00 98.62           H  \\nATOM    671  HB3 TRP A  48       1.804   6.620  -1.204  1.00 98.62           H  \\nATOM    672  O   TRP A  48      -1.080   8.964  -0.166  1.00 98.62           O  \\nATOM    673  CG  TRP A  48       1.432   8.471  -2.142  1.00 98.62           C  \\nATOM    674  CD1 TRP A  48       1.646   9.789  -2.336  1.00 98.62           C  \\nATOM    675  HD1 TRP A  48       2.201  10.415  -1.653  1.00 98.62           H  \\nATOM    676  CD2 TRP A  48       0.584   8.039  -3.243  1.00 98.62           C  \\nATOM    677  CE2 TRP A  48       0.341   9.162  -4.091  1.00 98.62           C  \\nATOM    678  CE3 TRP A  48      -0.024   6.820  -3.600  1.00 98.62           C  \\nATOM    679  HE3 TRP A  48       0.129   5.950  -2.979  1.00 98.62           H  \\nATOM    680  NE1 TRP A  48       1.008  10.204  -3.487  1.00 98.62           N  \\nATOM    681  HE1 TRP A  48       1.003  11.160  -3.811  1.00 98.62           H  \\nATOM    682  CH2 TRP A  48      -1.022   7.853  -5.585  1.00 98.62           C  \\nATOM    683  HH2 TRP A  48      -1.631   7.764  -6.473  1.00 98.62           H  \\nATOM    684  CZ2 TRP A  48      -0.432   9.080  -5.256  1.00 98.62           C  \\nATOM    685  HZ2 TRP A  48      -0.573   9.946  -5.886  1.00 98.62           H  \\nATOM    686  CZ3 TRP A  48      -0.818   6.737  -4.757  1.00 98.62           C  \\nATOM    687  HZ3 TRP A  48      -1.263   5.793  -5.035  1.00 98.62           H  \\nATOM    688  N   ALA A  49      -1.020   6.719  -0.165  1.00 98.56           N  \\nATOM    689  H   ALA A  49      -0.449   5.893  -0.050  1.00 98.56           H  \\nATOM    690  CA  ALA A  49      -2.440   6.543  -0.449  1.00 98.56           C  \\nATOM    691  HA  ALA A  49      -2.667   7.023  -1.401  1.00 98.56           H  \\nATOM    692  C   ALA A  49      -3.313   7.208   0.632  1.00 98.56           C  \\nATOM    693  CB  ALA A  49      -2.720   5.041  -0.584  1.00 98.56           C  \\nATOM    694  HB1 ALA A  49      -3.786   4.869  -0.730  1.00 98.56           H  \\nATOM    695  HB2 ALA A  49      -2.167   4.635  -1.431  1.00 98.56           H  \\nATOM    696  HB3 ALA A  49      -2.412   4.510   0.316  1.00 98.56           H  \\nATOM    697  O   ALA A  49      -4.234   7.967   0.316  1.00 98.56           O  \\nATOM    698  N   ILE A  50      -2.986   7.001   1.910  1.00 98.42           N  \\nATOM    699  H   ILE A  50      -2.248   6.340   2.105  1.00 98.42           H  \\nATOM    700  CA  ILE A  50      -3.678   7.637   3.040  1.00 98.42           C  \\nATOM    701  HA  ILE A  50      -4.748   7.439   2.965  1.00 98.42           H  \\nATOM    702  C   ILE A  50      -3.484   9.157   3.000  1.00 98.42           C  \\nATOM    703  CB  ILE A  50      -3.170   7.049   4.376  1.00 98.42           C  \\nATOM    704  HB  ILE A  50      -2.089   7.180   4.410  1.00 98.42           H  \\nATOM    705  O   ILE A  50      -4.450   9.923   2.994  1.00 98.42           O  \\nATOM    706  CG1 ILE A  50      -3.482   5.540   4.499  1.00 98.42           C  \\nATOM    707 HG12 ILE A  50      -4.538   5.392   4.725  1.00 98.42           H  \\nATOM    708 HG13 ILE A  50      -3.275   5.037   3.555  1.00 98.42           H  \\nATOM    709  CG2 ILE A  50      -3.787   7.806   5.567  1.00 98.42           C  \\nATOM    710 HG21 ILE A  50      -3.439   8.839   5.589  1.00 98.42           H  \\nATOM    711 HG22 ILE A  50      -4.873   7.799   5.479  1.00 98.42           H  \\nATOM    712 HG23 ILE A  50      -3.491   7.341   6.508  1.00 98.42           H  \\nATOM    713  CD1 ILE A  50      -2.623   4.863   5.577  1.00 98.42           C  \\nATOM    714 HD11 ILE A  50      -2.970   5.137   6.573  1.00 98.42           H  \\nATOM    715 HD12 ILE A  50      -1.579   5.155   5.462  1.00 98.42           H  \\nATOM    716 HD13 ILE A  50      -2.675   3.780   5.470  1.00 98.42           H  \\nATOM    717  N   LEU A  51      -2.229   9.615   2.960  1.00 98.12           N  \\nATOM    718  H   LEU A  51      -1.475   8.944   2.926  1.00 98.12           H  \\nATOM    719  CA  LEU A  51      -1.891  11.026   3.124  1.00 98.12           C  \\nATOM    720  HA  LEU A  51      -2.415  11.380   4.011  1.00 98.12           H  \\nATOM    721  C   LEU A  51      -2.349  11.875   1.933  1.00 98.12           C  \\nATOM    722  CB  LEU A  51      -0.372  11.149   3.346  1.00 98.12           C  \\nATOM    723  HB2 LEU A  51       0.133  10.801   2.445  1.00 98.12           H  \\nATOM    724  HB3 LEU A  51      -0.069  10.493   4.162  1.00 98.12           H  \\nATOM    725  O   LEU A  51      -2.881  12.967   2.129  1.00 98.12           O  \\nATOM    726  CG  LEU A  51       0.119  12.573   3.657  1.00 98.12           C  \\nATOM    727  HG  LEU A  51      -0.235  13.260   2.888  1.00 98.12           H  \\nATOM    728  CD1 LEU A  51      -0.358  13.079   5.021  1.00 98.12           C  \\nATOM    729 HD11 LEU A  51       0.183  12.545   5.803  1.00 98.12           H  \\nATOM    730 HD12 LEU A  51      -0.123  14.136   5.141  1.00 98.12           H  \\nATOM    731 HD13 LEU A  51      -1.416  12.880   5.193  1.00 98.12           H  \\nATOM    732  CD2 LEU A  51       1.643  12.584   3.656  1.00 98.12           C  \\nATOM    733 HD21 LEU A  51       2.011  12.333   2.661  1.00 98.12           H  \\nATOM    734 HD22 LEU A  51       2.030  11.856   4.369  1.00 98.12           H  \\nATOM    735 HD23 LEU A  51       2.009  13.574   3.927  1.00 98.12           H  \\nATOM    736  N   CYS A  52      -2.141  11.397   0.710  1.00 98.22           N  \\nATOM    737  H   CYS A  52      -1.706  10.489   0.626  1.00 98.22           H  \\nATOM    738  CA  CYS A  52      -2.271  12.202  -0.502  1.00 98.22           C  \\nATOM    739  HA  CYS A  52      -2.374  13.252  -0.230  1.00 98.22           H  \\nATOM    740  C   CYS A  52      -3.501  11.860  -1.346  1.00 98.22           C  \\nATOM    741  CB  CYS A  52      -0.984  12.063  -1.318  1.00 98.22           C  \\nATOM    742  HB2 CYS A  52      -1.073  12.686  -2.207  1.00 98.22           H  \\nATOM    743  HB3 CYS A  52      -0.885  11.027  -1.641  1.00 98.22           H  \\nATOM    744  O   CYS A  52      -3.909  12.681  -2.165  1.00 98.22           O  \\nATOM    745  SG  CYS A  52       0.534  12.537  -0.440  1.00 98.22           S  \\nATOM    746  N   LYS A  53      -4.102  10.677  -1.165  1.00 97.65           N  \\nATOM    747  H   LYS A  53      -3.688  10.019  -0.520  1.00 97.65           H  \\nATOM    748  CA  LYS A  53      -5.299  10.252  -1.914  1.00 97.65           C  \\nATOM    749  HA  LYS A  53      -5.581  11.039  -2.614  1.00 97.65           H  \\nATOM    750  C   LYS A  53      -6.541  10.087  -1.042  1.00 97.65           C  \\nATOM    751  CB  LYS A  53      -4.997   8.983  -2.732  1.00 97.65           C  \\nATOM    752  HB2 LYS A  53      -5.889   8.703  -3.294  1.00 97.65           H  \\nATOM    753  HB3 LYS A  53      -4.784   8.155  -2.057  1.00 97.65           H  \\nATOM    754  O   LYS A  53      -7.606   9.808  -1.582  1.00 97.65           O  \\nATOM    755  CG  LYS A  53      -3.816   9.124  -3.707  1.00 97.65           C  \\nATOM    756  HG2 LYS A  53      -2.909   9.361  -3.151  1.00 97.65           H  \\nATOM    757  HG3 LYS A  53      -3.651   8.158  -4.186  1.00 97.65           H  \\nATOM    758  CD  LYS A  53      -4.011  10.171  -4.812  1.00 97.65           C  \\nATOM    759  HD2 LYS A  53      -3.112  10.165  -5.429  1.00 97.65           H  \\nATOM    760  HD3 LYS A  53      -4.117  11.165  -4.378  1.00 97.65           H  \\nATOM    761  CE  LYS A  53      -5.225   9.847  -5.690  1.00 97.65           C  \\nATOM    762  HE2 LYS A  53      -6.134  10.094  -5.142  1.00 97.65           H  \\nATOM    763  HE3 LYS A  53      -5.231   8.775  -5.889  1.00 97.65           H  \\nATOM    764  NZ  LYS A  53      -5.161  10.585  -6.974  1.00 97.65           N  \\nATOM    765  HZ1 LYS A  53      -5.069  11.578  -6.819  1.00 97.65           H  \\nATOM    766  HZ2 LYS A  53      -5.980  10.402  -7.537  1.00 97.65           H  \\nATOM    767  HZ3 LYS A  53      -4.356  10.272  -7.498  1.00 97.65           H  \\nATOM    768  N   GLY A  54      -6.425  10.288   0.273  1.00 96.51           N  \\nATOM    769  H   GLY A  54      -5.505  10.445   0.660  1.00 96.51           H  \\nATOM    770  CA  GLY A  54      -7.548  10.159   1.201  1.00 96.51           C  \\nATOM    771  HA2 GLY A  54      -8.382  10.772   0.859  1.00 96.51           H  \\nATOM    772  HA3 GLY A  54      -7.233  10.506   2.185  1.00 96.51           H  \\nATOM    773  C   GLY A  54      -8.036   8.719   1.350  1.00 96.51           C  \\nATOM    774  O   GLY A  54      -9.211   8.504   1.640  1.00 96.51           O  \\nATOM    775  N   ALA A  55      -7.159   7.734   1.121  1.00 97.99           N  \\nATOM    776  H   ALA A  55      -6.202   7.967   0.899  1.00 97.99           H  \\nATOM    777  CA  ALA A  55      -7.492   6.335   1.350  1.00 97.99           C  \\nATOM    778  HA  ALA A  55      -8.363   6.104   0.737  1.00 97.99           H  \\nATOM    779  C   ALA A  55      -7.855   6.099   2.833  1.00 97.99           C  \\nATOM    780  CB  ALA A  55      -6.328   5.456   0.883  1.00 97.99           C  \\nATOM    781  HB1 ALA A  55      -6.059   5.710  -0.142  1.00 97.99           H  \\nATOM    782  HB2 ALA A  55      -5.467   5.601   1.536  1.00 97.99           H  \\nATOM    783  HB3 ALA A  55      -6.617   4.405   0.918  1.00 97.99           H  \\nATOM    784  O   ALA A  55      -7.325   6.787   3.714  1.00 97.99           O  \\nATOM    785  N   PRO A  56      -8.754   5.143   3.130  1.00 97.71           N  \\nATOM    786  CA  PRO A  56      -9.218   4.912   4.493  1.00 97.71           C  \\nATOM    787  HA  PRO A  56      -9.638   5.832   4.899  1.00 97.71           H  \\nATOM    788  C   PRO A  56      -8.068   4.444   5.388  1.00 97.71           C  \\nATOM    789  CB  PRO A  56     -10.319   3.854   4.379  1.00 97.71           C  \\nATOM    790  HB2 PRO A  56     -10.343   3.189   5.243  1.00 97.71           H  \\nATOM    791  HB3 PRO A  56     -11.283   4.347   4.255  1.00 97.71           H  \\nATOM    792  O   PRO A  56      -7.343   3.529   5.023  1.00 97.71           O  \\nATOM    793  CG  PRO A  56      -9.967   3.104   3.095  1.00 97.71           C  \\nATOM    794  HG2 PRO A  56     -10.851   2.670   2.629  1.00 97.71           H  \\nATOM    795  HG3 PRO A  56      -9.222   2.338   3.313  1.00 97.71           H  \\nATOM    796  CD  PRO A  56      -9.357   4.188   2.211  1.00 97.71           C  \\nATOM    797  HD2 PRO A  56     -10.141   4.682   1.639  1.00 97.71           H  \\nATOM    798  HD3 PRO A  56      -8.621   3.743   1.541  1.00 97.71           H  \\nATOM    799  N   ASN A  57      -7.939   5.041   6.574  1.00 97.24           N  \\nATOM    800  H   ASN A  57      -8.538   5.826   6.787  1.00 97.24           H  \\nATOM    801  CA  ASN A  57      -6.915   4.699   7.570  1.00 97.24           C  \\nATOM    802  HA  ASN A  57      -6.176   4.047   7.104  1.00 97.24           H  \\nATOM    803  C   ASN A  57      -7.463   3.931   8.786  1.00 97.24           C  \\nATOM    804  CB  ASN A  57      -6.194   5.986   7.998  1.00 97.24           C  \\nATOM    805  HB2 ASN A  57      -5.808   6.487   7.110  1.00 97.24           H  \\nATOM    806  HB3 ASN A  57      -5.349   5.716   8.630  1.00 97.24           H  \\nATOM    807  O   ASN A  57      -6.689   3.477   9.626  1.00 97.24           O  \\nATOM    808  CG  ASN A  57      -7.076   6.963   8.760  1.00 97.24           C  \\nATOM    809  ND2 ASN A  57      -6.486   7.954   9.384  1.00 97.24           N  \\nATOM    810 HD21 ASN A  57      -5.485   8.072   9.320  1.00 97.24           H  \\nATOM    811 HD22 ASN A  57      -7.097   8.598   9.864  1.00 97.24           H  \\nATOM    812  OD1 ASN A  57      -8.293   6.877   8.789  1.00 97.24           O  \\nATOM    813  N   VAL A  58      -8.786   3.783   8.879  1.00 98.30           N  \\nATOM    814  H   VAL A  58      -9.353   4.212   8.162  1.00 98.30           H  \\nATOM    815  CA  VAL A  58      -9.461   2.919   9.848  1.00 98.30           C  \\nATOM    816  HA  VAL A  58      -8.735   2.563  10.579  1.00 98.30           H  \\nATOM    817  C   VAL A  58     -10.010   1.719   9.090  1.00 98.30           C  \\nATOM    818  CB  VAL A  58     -10.562   3.663  10.622  1.00 98.30           C  \\nATOM    819  HB  VAL A  58     -11.338   3.997   9.933  1.00 98.30           H  \\nATOM    820  O   VAL A  58     -10.854   1.864   8.203  1.00 98.30           O  \\nATOM    821  CG1 VAL A  58     -11.186   2.724  11.656  1.00 98.30           C  \\nATOM    822 HG11 VAL A  58     -10.403   2.265  12.260  1.00 98.30           H  \\nATOM    823 HG12 VAL A  58     -11.758   1.950  11.144  1.00 98.30           H  \\nATOM    824 HG13 VAL A  58     -11.864   3.278  12.305  1.00 98.30           H  \\nATOM    825  CG2 VAL A  58     -10.013   4.886  11.364  1.00 98.30           C  \\nATOM    826 HG21 VAL A  58     -10.808   5.355  11.944  1.00 98.30           H  \\nATOM    827 HG22 VAL A  58      -9.209   4.590  12.038  1.00 98.30           H  \\nATOM    828 HG23 VAL A  58      -9.632   5.619  10.654  1.00 98.30           H  \\nATOM    829  N   LEU A  59      -9.495   0.541   9.420  1.00 98.45           N  \\nATOM    830  H   LEU A  59      -8.861   0.503  10.205  1.00 98.45           H  \\nATOM    831  CA  LEU A  59      -9.694  -0.687   8.664  1.00 98.45           C  \\nATOM    832  HA  LEU A  59     -10.381  -0.495   7.840  1.00 98.45           H  \\nATOM    833  C   LEU A  59     -10.314  -1.775   9.537  1.00 98.45           C  \\nATOM    834  CB  LEU A  59      -8.344  -1.148   8.089  1.00 98.45           C  \\nATOM    835  HB2 LEU A  59      -8.495  -2.086   7.555  1.00 98.45           H  \\nATOM    836  HB3 LEU A  59      -7.681  -1.339   8.933  1.00 98.45           H  \\nATOM    837  O   LEU A  59     -10.194  -1.750  10.759  1.00 98.45           O  \\nATOM    838  CG  LEU A  59      -7.633  -0.154   7.151  1.00 98.45           C  \\nATOM    839  HG  LEU A  59      -7.394   0.762   7.691  1.00 98.45           H  \\nATOM    840  CD1 LEU A  59      -6.330  -0.790   6.667  1.00 98.45           C  \\nATOM    841 HD11 LEU A  59      -6.551  -1.698   6.106  1.00 98.45           H  \\nATOM    842 HD12 LEU A  59      -5.809  -0.093   6.011  1.00 98.45           H  \\nATOM    843 HD13 LEU A  59      -5.691  -1.017   7.520  1.00 98.45           H  \\nATOM    844  CD2 LEU A  59      -8.466   0.192   5.920  1.00 98.45           C  \\nATOM    845 HD21 LEU A  59      -8.733  -0.721   5.388  1.00 98.45           H  \\nATOM    846 HD22 LEU A  59      -7.891   0.843   5.261  1.00 98.45           H  \\nATOM    847 HD23 LEU A  59      -9.368   0.726   6.217  1.00 98.45           H  \\nATOM    848  N   GLN A  60     -10.936  -2.768   8.906  1.00 97.65           N  \\nATOM    849  H   GLN A  60     -11.038  -2.706   7.904  1.00 97.65           H  \\nATOM    850  CA  GLN A  60     -11.417  -3.969   9.592  1.00 97.65           C  \\nATOM    851  HA  GLN A  60     -11.028  -3.983  10.610  1.00 97.65           H  \\nATOM    852  C   GLN A  60     -10.901  -5.209   8.869  1.00 97.65           C  \\nATOM    853  CB  GLN A  60     -12.948  -3.942   9.663  1.00 97.65           C  \\nATOM    854  HB2 GLN A  60     -13.366  -4.270   8.712  1.00 97.65           H  \\nATOM    855  HB3 GLN A  60     -13.283  -2.921   9.848  1.00 97.65           H  \\nATOM    856  O   GLN A  60     -11.083  -5.329   7.660  1.00 97.65           O  \\nATOM    857  CG  GLN A  60     -13.470  -4.819  10.809  1.00 97.65           C  \\nATOM    858  HG2 GLN A  60     -13.102  -4.410  11.750  1.00 97.65           H  \\nATOM    859  HG3 GLN A  60     -13.100  -5.840  10.708  1.00 97.65           H  \\nATOM    860  CD  GLN A  60     -14.992  -4.849  10.875  1.00 97.65           C  \\nATOM    861  NE2 GLN A  60     -15.575  -5.147  12.016  1.00 97.65           N  \\nATOM    862 HE21 GLN A  60     -15.060  -5.225  12.881  1.00 97.65           H  \\nATOM    863 HE22 GLN A  60     -16.584  -5.143  11.992  1.00 97.65           H  \\nATOM    864  OE1 GLN A  60     -15.695  -4.618   9.906  1.00 97.65           O  \\nATOM    865  N   ARG A  61     -10.235  -6.127   9.576  1.00 97.32           N  \\nATOM    866  H   ARG A  61     -10.171  -6.010  10.577  1.00 97.32           H  \\nATOM    867  CA  ARG A  61      -9.587  -7.288   8.945  1.00 97.32           C  \\nATOM    868  HA  ARG A  61      -9.070  -6.935   8.052  1.00 97.32           H  \\nATOM    869  C   ARG A  61     -10.618  -8.328   8.503  1.00 97.32           C  \\nATOM    870  CB  ARG A  61      -8.543  -7.883   9.898  1.00 97.32           C  \\nATOM    871  HB2 ARG A  61      -7.862  -7.092  10.213  1.00 97.32           H  \\nATOM    872  HB3 ARG A  61      -9.048  -8.275  10.781  1.00 97.32           H  \\nATOM    873  O   ARG A  61     -11.447  -8.739   9.303  1.00 97.32           O  \\nATOM    874  CG  ARG A  61      -7.724  -9.001   9.239  1.00 97.32           C  \\nATOM    875  HG2 ARG A  61      -8.384  -9.797   8.895  1.00 97.32           H  \\nATOM    876  HG3 ARG A  61      -7.175  -8.595   8.389  1.00 97.32           H  \\nATOM    877  CD  ARG A  61      -6.736  -9.585  10.250  1.00 97.32           C  \\nATOM    878  HD2 ARG A  61      -7.296  -9.965  11.104  1.00 97.32           H  \\nATOM    879  HD3 ARG A  61      -6.069  -8.794  10.592  1.00 97.32           H  \\nATOM    880  NE  ARG A  61      -5.944 -10.671   9.648  1.00 97.32           N  \\nATOM    881  HE  ARG A  61      -6.061 -10.826   8.657  1.00 97.32           H  \\nATOM    882  NH1 ARG A  61      -4.921 -11.371  11.579  1.00 97.32           N  \\nATOM    883 HH11 ARG A  61      -5.500 -10.737  12.111  1.00 97.32           H  \\nATOM    884 HH12 ARG A  61      -4.257 -11.966  12.053  1.00 97.32           H  \\nATOM    885  NH2 ARG A  61      -4.328 -12.274   9.624  1.00 97.32           N  \\nATOM    886 HH21 ARG A  61      -4.406 -12.339   8.619  1.00 97.32           H  \\nATOM    887 HH22 ARG A  61      -3.597 -12.789  10.094  1.00 97.32           H  \\nATOM    888  CZ  ARG A  61      -5.072 -11.432  10.284  1.00 97.32           C  \\nATOM    889  N   VAL A  62     -10.535  -8.818   7.271  1.00 97.85           N  \\nATOM    890  H   VAL A  62      -9.786  -8.488   6.681  1.00 97.85           H  \\nATOM    891  CA  VAL A  62     -11.456  -9.830   6.728  1.00 97.85           C  \\nATOM    892  HA  VAL A  62     -12.301  -9.933   7.409  1.00 97.85           H  \\nATOM    893  C   VAL A  62     -10.819 -11.213   6.636  1.00 97.85           C  \\nATOM    894  CB  VAL A  62     -12.061  -9.404   5.380  1.00 97.85           C  \\nATOM    895  HB  VAL A  62     -12.740 -10.192   5.055  1.00 97.85           H  \\nATOM    896  O   VAL A  62      -9.597 -11.365   6.679  1.00 97.85           O  \\nATOM    897  CG1 VAL A  62     -12.902  -8.140   5.557  1.00 97.85           C  \\nATOM    898 HG11 VAL A  62     -12.282  -7.323   5.926  1.00 97.85           H  \\nATOM    899 HG12 VAL A  62     -13.352  -7.858   4.605  1.00 97.85           H  \\nATOM    900 HG13 VAL A  62     -13.693  -8.328   6.282  1.00 97.85           H  \\nATOM    901  CG2 VAL A  62     -11.012  -9.182   4.285  1.00 97.85           C  \\nATOM    902 HG21 VAL A  62     -10.395 -10.072   4.158  1.00 97.85           H  \\nATOM    903 HG22 VAL A  62     -10.382  -8.331   4.541  1.00 97.85           H  \\nATOM    904 HG23 VAL A  62     -11.505  -8.960   3.339  1.00 97.85           H  \\nATOM    905  N   ASN A  63     -11.669 -12.231   6.517  1.00 96.74           N  \\nATOM    906  H   ASN A  63     -12.656 -12.027   6.583  1.00 96.74           H  \\nATOM    907  CA  ASN A  63     -11.267 -13.616   6.308  1.00 96.74           C  \\nATOM    908  HA  ASN A  63     -10.521 -13.855   7.066  1.00 96.74           H  \\nATOM    909  C   ASN A  63     -10.595 -13.848   4.940  1.00 96.74           C  \\nATOM    910  CB  ASN A  63     -12.489 -14.534   6.535  1.00 96.74           C  \\nATOM    911  HB2 ASN A  63     -12.960 -14.269   7.482  1.00 96.74           H  \\nATOM    912  HB3 ASN A  63     -12.144 -15.564   6.620  1.00 96.74           H  \\nATOM    913  O   ASN A  63     -10.656 -13.028   4.022  1.00 96.74           O  \\nATOM    914  CG  ASN A  63     -13.560 -14.487   5.447  1.00 96.74           C  \\nATOM    915  ND2 ASN A  63     -14.588 -15.291   5.569  1.00 96.74           N  \\nATOM    916 HD21 ASN A  63     -14.713 -15.851   6.400  1.00 96.74           H  \\nATOM    917 HD22 ASN A  63     -15.370 -15.147   4.947  1.00 96.74           H  \\nATOM    918  OD1 ASN A  63     -13.496 -13.768   4.463  1.00 96.74           O  \\nATOM    919  N   GLU A  64      -9.999 -15.030   4.789  1.00 97.09           N  \\nATOM    920  H   GLU A  64      -9.912 -15.623   5.602  1.00 97.09           H  \\nATOM    921  CA  GLU A  64      -9.263 -15.435   3.587  1.00 97.09           C  \\nATOM    922  HA  GLU A  64      -8.419 -14.757   3.456  1.00 97.09           H  \\nATOM    923  C   GLU A  64     -10.105 -15.383   2.303  1.00 97.09           C  \\nATOM    924  CB  GLU A  64      -8.745 -16.864   3.825  1.00 97.09           C  \\nATOM    925  HB2 GLU A  64      -8.246 -16.908   4.793  1.00 97.09           H  \\nATOM    926  HB3 GLU A  64      -9.594 -17.546   3.868  1.00 97.09           H  \\nATOM    927  O   GLU A  64      -9.640 -14.911   1.267  1.00 97.09           O  \\nATOM    928  CG  GLU A  64      -7.774 -17.360   2.747  1.00 97.09           C  \\nATOM    929  HG2 GLU A  64      -8.286 -17.384   1.785  1.00 97.09           H  \\nATOM    930  HG3 GLU A  64      -7.484 -18.382   2.991  1.00 97.09           H  \\nATOM    931  CD  GLU A  64      -6.514 -16.497   2.651  1.00 97.09           C  \\nATOM    932  OE1 GLU A  64      -5.862 -16.499   1.583  1.00 97.09           O  \\nATOM    933  OE2 GLU A  64      -6.186 -15.768   3.619  1.00 97.09           O  \\nATOM    934  N   LYS A  65     -11.367 -15.824   2.374  1.00 97.72           N  \\nATOM    935  H   LYS A  65     -11.697 -16.160   3.268  1.00 97.72           H  \\nATOM    936  CA  LYS A  65     -12.283 -15.817   1.225  1.00 97.72           C  \\nATOM    937  HA  LYS A  65     -11.831 -16.385   0.412  1.00 97.72           H  \\nATOM    938  C   LYS A  65     -12.494 -14.398   0.697  1.00 97.72           C  \\nATOM    939  CB  LYS A  65     -13.611 -16.478   1.628  1.00 97.72           C  \\nATOM    940  HB2 LYS A  65     -13.413 -17.491   1.977  1.00 97.72           H  \\nATOM    941  HB3 LYS A  65     -14.061 -15.913   2.444  1.00 97.72           H  \\nATOM    942  O   LYS A  65     -12.369 -14.162  -0.503  1.00 97.72           O  \\nATOM    943  CG  LYS A  65     -14.598 -16.540   0.451  1.00 97.72           C  \\nATOM    944  HG2 LYS A  65     -14.790 -15.534   0.077  1.00 97.72           H  \\nATOM    945  HG3 LYS A  65     -14.167 -17.137  -0.353  1.00 97.72           H  \\nATOM    946  CD  LYS A  65     -15.935 -17.152   0.881  1.00 97.72           C  \\nATOM    947  HD2 LYS A  65     -16.341 -16.571   1.709  1.00 97.72           H  \\nATOM    948  HD3 LYS A  65     -15.777 -18.181   1.206  1.00 97.72           H  \\nATOM    949  CE  LYS A  65     -16.915 -17.118  -0.298  1.00 97.72           C  \\nATOM    950  HE2 LYS A  65     -16.518 -17.735  -1.104  1.00 97.72           H  \\nATOM    951  HE3 LYS A  65     -16.974 -16.093  -0.664  1.00 97.72           H  \\nATOM    952  NZ  LYS A  65     -18.265 -17.588   0.100  1.00 97.72           N  \\nATOM    953  HZ1 LYS A  65     -18.233 -18.545   0.421  1.00 97.72           H  \\nATOM    954  HZ2 LYS A  65     -18.901 -17.532  -0.683  1.00 97.72           H  \\nATOM    955  HZ3 LYS A  65     -18.632 -17.009   0.841  1.00 97.72           H  \\nATOM    956  N   THR A  66     -12.787 -13.457   1.593  1.00 98.08           N  \\nATOM    957  H   THR A  66     -12.852 -13.711   2.568  1.00 98.08           H  \\nATOM    958  CA  THR A  66     -12.984 -12.052   1.224  1.00 98.08           C  \\nATOM    959  HA  THR A  66     -13.698 -11.991   0.403  1.00 98.08           H  \\nATOM    960  C   THR A  66     -11.676 -11.431   0.758  1.00 98.08           C  \\nATOM    961  CB  THR A  66     -13.538 -11.239   2.392  1.00 98.08           C  \\nATOM    962  HB  THR A  66     -12.778 -11.183   3.171  1.00 98.08           H  \\nATOM    963  O   THR A  66     -11.669 -10.746  -0.256  1.00 98.08           O  \\nATOM    964  CG2 THR A  66     -13.939  -9.829   1.964  1.00 98.08           C  \\nATOM    965 HG21 THR A  66     -13.089  -9.302   1.531  1.00 98.08           H  \\nATOM    966 HG22 THR A  66     -14.743  -9.874   1.230  1.00 98.08           H  \\nATOM    967 HG23 THR A  66     -14.276  -9.265   2.835  1.00 98.08           H  \\nATOM    968  OG1 THR A  66     -14.680 -11.879   2.905  1.00 98.08           O  \\nATOM    969  HG1 THR A  66     -14.368 -12.548   3.519  1.00 98.08           H  \\nATOM    970  N   LYS A  67     -10.553 -11.733   1.423  1.00 98.10           N  \\nATOM    971  H   LYS A  67     -10.624 -12.286   2.265  1.00 98.10           H  \\nATOM    972  CA  LYS A  67      -9.226 -11.276   0.996  1.00 98.10           C  \\nATOM    973  HA  LYS A  67      -9.212 -10.187   1.048  1.00 98.10           H  \\nATOM    974  C   LYS A  67      -8.937 -11.641  -0.458  1.00 98.10           C  \\nATOM    975  CB  LYS A  67      -8.160 -11.851   1.938  1.00 98.10           C  \\nATOM    976  HB2 LYS A  67      -8.410 -11.591   2.967  1.00 98.10           H  \\nATOM    977  HB3 LYS A  67      -8.168 -12.938   1.853  1.00 98.10           H  \\nATOM    978  O   LYS A  67      -8.499 -10.790  -1.224  1.00 98.10           O  \\nATOM    979  CG  LYS A  67      -6.746 -11.328   1.628  1.00 98.10           C  \\nATOM    980  HG2 LYS A  67      -6.543 -11.350   0.557  1.00 98.10           H  \\nATOM    981  HG3 LYS A  67      -6.678 -10.291   1.958  1.00 98.10           H  \\nATOM    982  CD  LYS A  67      -5.663 -12.138   2.344  1.00 98.10           C  \\nATOM    983  HD2 LYS A  67      -4.737 -11.562   2.350  1.00 98.10           H  \\nATOM    984  HD3 LYS A  67      -5.968 -12.337   3.372  1.00 98.10           H  \\nATOM    985  CE  LYS A  67      -5.442 -13.448   1.586  1.00 98.10           C  \\nATOM    986  HE2 LYS A  67      -6.389 -13.982   1.513  1.00 98.10           H  \\nATOM    987  HE3 LYS A  67      -5.126 -13.239   0.564  1.00 98.10           H  \\nATOM    988  NZ  LYS A  67      -4.469 -14.334   2.255  1.00 98.10           N  \\nATOM    989  HZ1 LYS A  67      -4.498 -15.218   1.766  1.00 98.10           H  \\nATOM    990  HZ2 LYS A  67      -3.531 -13.975   2.356  1.00 98.10           H  \\nATOM    991  HZ3 LYS A  67      -4.869 -14.646   3.128  1.00 98.10           H  \\nATOM    992  N   ASN A  68      -9.207 -12.881  -0.857  1.00 98.26           N  \\nATOM    993  H   ASN A  68      -9.525 -13.551  -0.171  1.00 98.26           H  \\nATOM    994  CA  ASN A  68      -8.971 -13.320  -2.232  1.00 98.26           C  \\nATOM    995  HA  ASN A  68      -7.964 -13.024  -2.525  1.00 98.26           H  \\nATOM    996  C   ASN A  68      -9.905 -12.615  -3.224  1.00 98.26           C  \\nATOM    997  CB  ASN A  68      -9.045 -14.854  -2.277  1.00 98.26           C  \\nATOM    998  HB2 ASN A  68      -9.991 -15.191  -1.853  1.00 98.26           H  \\nATOM    999  HB3 ASN A  68      -8.987 -15.189  -3.313  1.00 98.26           H  \\nATOM   1000  O   ASN A  68      -9.431 -12.129  -4.246  1.00 98.26           O  \\nATOM   1001  CG  ASN A  68      -7.886 -15.483  -1.515  1.00 98.26           C  \\nATOM   1002  ND2 ASN A  68      -8.020 -16.704  -1.060  1.00 98.26           N  \\nATOM   1003 HD21 ASN A  68      -7.266 -17.020  -0.467  1.00 98.26           H  \\nATOM   1004 HD22 ASN A  68      -8.874 -17.225  -1.201  1.00 98.26           H  \\nATOM   1005  OD1 ASN A  68      -6.843 -14.878  -1.305  1.00 98.26           O  \\nATOM   1006  N   SER A  69     -11.182 -12.433  -2.869  1.00 98.23           N  \\nATOM   1007  H   SER A  69     -11.511 -12.845  -2.008  1.00 98.23           H  \\nATOM   1008  CA  SER A  69     -12.106 -11.611  -3.664  1.00 98.23           C  \\nATOM   1009  HA  SER A  69     -12.151 -12.024  -4.672  1.00 98.23           H  \\nATOM   1010  C   SER A  69     -11.643 -10.153  -3.784  1.00 98.23           C  \\nATOM   1011  CB  SER A  69     -13.506 -11.665  -3.047  1.00 98.23           C  \\nATOM   1012  HB2 SER A  69     -13.858 -12.696  -3.053  1.00 98.23           H  \\nATOM   1013  HB3 SER A  69     -13.468 -11.313  -2.016  1.00 98.23           H  \\nATOM   1014  O   SER A  69     -11.788  -9.543  -4.841  1.00 98.23           O  \\nATOM   1015  OG  SER A  69     -14.411 -10.856  -3.773  1.00 98.23           O  \\nATOM   1016  HG  SER A  69     -14.308 -11.046  -4.708  1.00 98.23           H  \\nATOM   1017  N   ASN A  70     -11.040  -9.588  -2.738  1.00 98.55           N  \\nATOM   1018  H   ASN A  70     -11.011 -10.094  -1.864  1.00 98.55           H  \\nATOM   1019  CA  ASN A  70     -10.481  -8.241  -2.784  1.00 98.55           C  \\nATOM   1020  HA  ASN A  70     -11.258  -7.557  -3.126  1.00 98.55           H  \\nATOM   1021  C   ASN A  70      -9.345  -8.153  -3.809  1.00 98.55           C  \\nATOM   1022  CB  ASN A  70     -10.035  -7.798  -1.380  1.00 98.55           C  \\nATOM   1023  HB2 ASN A  70      -9.527  -6.838  -1.462  1.00 98.55           H  \\nATOM   1024  HB3 ASN A  70      -9.325  -8.510  -0.960  1.00 98.55           H  \\nATOM   1025  O   ASN A  70      -9.348  -7.231  -4.614  1.00 98.55           O  \\nATOM   1026  CG  ASN A  70     -11.182  -7.623  -0.405  1.00 98.55           C  \\nATOM   1027  ND2 ASN A  70     -10.894  -7.310   0.827  1.00 98.55           N  \\nATOM   1028 HD21 ASN A  70     -11.628  -6.984   1.440  1.00 98.55           H  \\nATOM   1029 HD22 ASN A  70      -9.939  -7.321   1.155  1.00 98.55           H  \\nATOM   1030  OD1 ASN A  70     -12.348  -7.744  -0.733  1.00 98.55           O  \\nATOM   1031  N   ARG A  71      -8.439  -9.137  -3.861  1.00 98.31           N  \\nATOM   1032  H   ARG A  71      -8.511  -9.888  -3.190  1.00 98.31           H  \\nATOM   1033  CA  ARG A  71      -7.352  -9.163  -4.860  1.00 98.31           C  \\nATOM   1034  HA  ARG A  71      -6.789  -8.233  -4.792  1.00 98.31           H  \\nATOM   1035  C   ARG A  71      -7.846  -9.255  -6.306  1.00 98.31           C  \\nATOM   1036  CB  ARG A  71      -6.405 -10.336  -4.597  1.00 98.31           C  \\nATOM   1037  HB2 ARG A  71      -5.638 -10.315  -5.371  1.00 98.31           H  \\nATOM   1038  HB3 ARG A  71      -6.949 -11.278  -4.669  1.00 98.31           H  \\nATOM   1039  O   ARG A  71      -7.167  -8.786  -7.221  1.00 98.31           O  \\nATOM   1040  CG  ARG A  71      -5.713 -10.258  -3.239  1.00 98.31           C  \\nATOM   1041  HG2 ARG A  71      -6.419 -10.493  -2.443  1.00 98.31           H  \\nATOM   1042  HG3 ARG A  71      -5.335  -9.247  -3.092  1.00 98.31           H  \\nATOM   1043  CD  ARG A  71      -4.554 -11.253  -3.195  1.00 98.31           C  \\nATOM   1044  HD2 ARG A  71      -4.948 -12.256  -3.360  1.00 98.31           H  \\nATOM   1045  HD3 ARG A  71      -3.863 -11.029  -4.007  1.00 98.31           H  \\nATOM   1046  NE  ARG A  71      -3.838 -11.216  -1.906  1.00 98.31           N  \\nATOM   1047  HE  ARG A  71      -4.037 -11.974  -1.269  1.00 98.31           H  \\nATOM   1048  NH1 ARG A  71      -2.526  -9.350  -2.302  1.00 98.31           N  \\nATOM   1049 HH11 ARG A  71      -3.253  -8.860  -2.805  1.00 98.31           H  \\nATOM   1050 HH12 ARG A  71      -1.791  -8.719  -2.017  1.00 98.31           H  \\nATOM   1051  NH2 ARG A  71      -2.158 -10.564  -0.504  1.00 98.31           N  \\nATOM   1052 HH21 ARG A  71      -2.285 -11.388   0.065  1.00 98.31           H  \\nATOM   1053 HH22 ARG A  71      -1.429  -9.896  -0.297  1.00 98.31           H  \\nATOM   1054  CZ  ARG A  71      -2.861 -10.378  -1.585  1.00 98.31           C  \\nATOM   1055  N   ASP A  72      -9.008  -9.865  -6.525  1.00 98.03           N  \\nATOM   1056  H   ASP A  72      -9.461 -10.332  -5.753  1.00 98.03           H  \\nATOM   1057  CA  ASP A  72      -9.653  -9.879  -7.841  1.00 98.03           C  \\nATOM   1058  HA  ASP A  72      -8.909 -10.110  -8.603  1.00 98.03           H  \\nATOM   1059  C   ASP A  72     -10.233  -8.501  -8.180  1.00 98.03           C  \\nATOM   1060  CB  ASP A  72     -10.743 -10.960  -7.899  1.00 98.03           C  \\nATOM   1061  HB2 ASP A  72     -11.278 -10.862  -8.844  1.00 98.03           H  \\nATOM   1062  HB3 ASP A  72     -11.460 -10.804  -7.093  1.00 98.03           H  \\nATOM   1063  O   ASP A  72      -9.939  -7.949  -9.243  1.00 98.03           O  \\nATOM   1064  CG  ASP A  72     -10.182 -12.380  -7.820  1.00 98.03           C  \\nATOM   1065  OD1 ASP A  72      -9.076 -12.602  -8.368  1.00 98.03           O  \\nATOM   1066  OD2 ASP A  72     -10.861 -13.240  -7.219  1.00 98.03           O  \\nATOM   1067  N   ARG A  73     -10.998  -7.913  -7.249  1.00 97.96           N  \\nATOM   1068  H   ARG A  73     -11.168  -8.432  -6.399  1.00 97.96           H  \\nATOM   1069  CA  ARG A  73     -11.666  -6.610  -7.425  1.00 97.96           C  \\nATOM   1070  HA  ARG A  73     -12.215  -6.617  -8.367  1.00 97.96           H  \\nATOM   1071  C   ARG A  73     -10.700  -5.430  -7.522  1.00 97.96           C  \\nATOM   1072  CB  ARG A  73     -12.657  -6.369  -6.279  1.00 97.96           C  \\nATOM   1073  HB2 ARG A  73     -12.143  -6.490  -5.325  1.00 97.96           H  \\nATOM   1074  HB3 ARG A  73     -13.018  -5.343  -6.351  1.00 97.96           H  \\nATOM   1075  O   ARG A  73     -11.031  -4.428  -8.148  1.00 97.96           O  \\nATOM   1076  CG  ARG A  73     -13.874  -7.304  -6.343  1.00 97.96           C  \\nATOM   1077  HG2 ARG A  73     -13.564  -8.337  -6.182  1.00 97.96           H  \\nATOM   1078  HG3 ARG A  73     -14.318  -7.236  -7.336  1.00 97.96           H  \\nATOM   1079  CD  ARG A  73     -14.950  -6.930  -5.314  1.00 97.96           C  \\nATOM   1080  HD2 ARG A  73     -15.350  -5.950  -5.576  1.00 97.96           H  \\nATOM   1081  HD3 ARG A  73     -15.769  -7.643  -5.412  1.00 97.96           H  \\nATOM   1082  NE  ARG A  73     -14.462  -6.954  -3.915  1.00 97.96           N  \\nATOM   1083  HE  ARG A  73     -14.234  -7.860  -3.530  1.00 97.96           H  \\nATOM   1084  NH1 ARG A  73     -14.559  -4.682  -3.476  1.00 97.96           N  \\nATOM   1085 HH11 ARG A  73     -14.838  -4.472  -4.423  1.00 97.96           H  \\nATOM   1086 HH12 ARG A  73     -14.528  -3.950  -2.781  1.00 97.96           H  \\nATOM   1087  NH2 ARG A  73     -13.956  -6.032  -1.879  1.00 97.96           N  \\nATOM   1088 HH21 ARG A  73     -13.647  -6.925  -1.522  1.00 97.96           H  \\nATOM   1089 HH22 ARG A  73     -13.643  -5.225  -1.358  1.00 97.96           H  \\nATOM   1090  CZ  ARG A  73     -14.337  -5.910  -3.108  1.00 97.96           C  \\nATOM   1091  N   SER A  74      -9.520  -5.527  -6.918  1.00 98.12           N  \\nATOM   1092  H   SER A  74      -9.328  -6.344  -6.356  1.00 98.12           H  \\nATOM   1093  CA  SER A  74      -8.446  -4.535  -7.056  1.00 98.12           C  \\nATOM   1094  HA  SER A  74      -8.871  -3.532  -7.053  1.00 98.12           H  \\nATOM   1095  C   SER A  74      -7.642  -4.705  -8.348  1.00 98.12           C  \\nATOM   1096  CB  SER A  74      -7.492  -4.659  -5.870  1.00 98.12           C  \\nATOM   1097  HB2 SER A  74      -8.049  -4.638  -4.933  1.00 98.12           H  \\nATOM   1098  HB3 SER A  74      -6.794  -3.823  -5.873  1.00 98.12           H  \\nATOM   1099  O   SER A  74      -6.851  -3.839  -8.712  1.00 98.12           O  \\nATOM   1100  OG  SER A  74      -6.770  -5.866  -5.969  1.00 98.12           O  \\nATOM   1101  HG  SER A  74      -6.198  -5.893  -5.198  1.00 98.12           H  \\nATOM   1102  N   GLY A  75      -7.795  -5.838  -9.038  1.00 98.02           N  \\nATOM   1103  H   GLY A  75      -8.477  -6.509  -8.712  1.00 98.02           H  \\nATOM   1104  CA  GLY A  75      -6.964  -6.210 -10.175  1.00 98.02           C  \\nATOM   1105  HA2 GLY A  75      -6.851  -5.348 -10.832  1.00 98.02           H  \\nATOM   1106  HA3 GLY A  75      -7.475  -7.000 -10.724  1.00 98.02           H  \\nATOM   1107  C   GLY A  75      -5.566  -6.716  -9.800  1.00 98.02           C  \\nATOM   1108  O   GLY A  75      -4.801  -7.048 -10.705  1.00 98.02           O  \\nATOM   1109  N   ALA A  76      -5.211  -6.848  -8.516  1.00 97.89           N  \\nATOM   1110  H   ALA A  76      -5.872  -6.596  -7.795  1.00 97.89           H  \\nATOM   1111  CA  ALA A  76      -3.901  -7.360  -8.100  1.00 97.89           C  \\nATOM   1112  HA  ALA A  76      -3.130  -6.702  -8.501  1.00 97.89           H  \\nATOM   1113  C   ALA A  76      -3.625  -8.775  -8.645  1.00 97.89           C  \\nATOM   1114  CB  ALA A  76      -3.815  -7.315  -6.572  1.00 97.89           C  \\nATOM   1115  HB1 ALA A  76      -2.851  -7.696  -6.236  1.00 97.89           H  \\nATOM   1116  HB2 ALA A  76      -3.918  -6.285  -6.230  1.00 97.89           H  \\nATOM   1117  HB3 ALA A  76      -4.610  -7.913  -6.127  1.00 97.89           H  \\nATOM   1118  O   ALA A  76      -2.535  -9.041  -9.161  1.00 97.89           O  \\nATOM   1119  N   ASN A  77      -4.630  -9.659  -8.650  1.00 97.40           N  \\nATOM   1120  H   ASN A  77      -5.498  -9.389  -8.209  1.00 97.40           H  \\nATOM   1121  CA  ASN A  77      -4.502 -11.018  -9.197  1.00 97.40           C  \\nATOM   1122  HA  ASN A  77      -3.627 -11.488  -8.748  1.00 97.40           H  \\nATOM   1123  C   ASN A  77      -4.248 -11.056 -10.716  1.00 97.40           C  \\nATOM   1124  CB  ASN A  77      -5.745 -11.841  -8.813  1.00 97.40           C  \\nATOM   1125  HB2 ASN A  77      -5.832 -12.704  -9.473  1.00 97.40           H  \\nATOM   1126  HB3 ASN A  77      -6.648 -11.244  -8.931  1.00 97.40           H  \\nATOM   1127  O   ASN A  77      -3.739 -12.051 -11.232  1.00 97.40           O  \\nATOM   1128  CG  ASN A  77      -5.692 -12.387  -7.396  1.00 97.40           C  \\nATOM   1129  ND2 ASN A  77      -6.792 -12.895  -6.908  1.00 97.40           N  \\nATOM   1130 HD21 ASN A  77      -6.801 -13.256  -5.965  1.00 97.40           H  \\nATOM   1131 HD22 ASN A  77      -7.641 -12.874  -7.456  1.00 97.40           H  \\nATOM   1132  OD1 ASN A  77      -4.672 -12.383  -6.722  1.00 97.40           O  \\nATOM   1133  N   LYS A  78      -4.513  -9.961 -11.442  1.00 97.17           N  \\nATOM   1134  H   LYS A  78      -4.869  -9.146 -10.962  1.00 97.17           H  \\nATOM   1135  CA  LYS A  78      -4.176  -9.843 -12.872  1.00 97.17           C  \\nATOM   1136  HA  LYS A  78      -4.436 -10.776 -13.371  1.00 97.17           H  \\nATOM   1137  C   LYS A  78      -2.678  -9.647 -13.105  1.00 97.17           C  \\nATOM   1138  CB  LYS A  78      -4.974  -8.702 -13.519  1.00 97.17           C  \\nATOM   1139  HB2 LYS A  78      -4.735  -7.754 -13.037  1.00 97.17           H  \\nATOM   1140  HB3 LYS A  78      -4.685  -8.617 -14.567  1.00 97.17           H  \\nATOM   1141  O   LYS A  78      -2.211  -9.828 -14.227  1.00 97.17           O  \\nATOM   1142  CG  LYS A  78      -6.478  -8.978 -13.451  1.00 97.17           C  \\nATOM   1143  HG2 LYS A  78      -6.764  -9.199 -12.422  1.00 97.17           H  \\nATOM   1144  HG3 LYS A  78      -6.703  -9.849 -14.067  1.00 97.17           H  \\nATOM   1145  CD  LYS A  78      -7.318  -7.790 -13.920  1.00 97.17           C  \\nATOM   1146  HD2 LYS A  78      -7.128  -6.943 -13.261  1.00 97.17           H  \\nATOM   1147  HD3 LYS A  78      -7.073  -7.501 -14.941  1.00 97.17           H  \\nATOM   1148  CE  LYS A  78      -8.786  -8.202 -13.782  1.00 97.17           C  \\nATOM   1149  HE2 LYS A  78      -8.859  -9.028 -13.075  1.00 97.17           H  \\nATOM   1150  HE3 LYS A  78      -9.328  -7.372 -13.330  1.00 97.17           H  \\nATOM   1151  NZ  LYS A  78      -9.413  -8.572 -15.073  1.00 97.17           N  \\nATOM   1152  HZ1 LYS A  78      -8.935  -9.333 -15.533  1.00 97.17           H  \\nATOM   1153  HZ2 LYS A  78      -9.435  -7.769 -15.685  1.00 97.17           H  \\nATOM   1154  HZ3 LYS A  78     -10.383  -8.812 -14.927  1.00 97.17           H  \\nATOM   1155  N   GLY A  79      -1.931  -9.284 -12.059  1.00 97.17           N  \\nATOM   1156  H   GLY A  79      -2.394  -9.170 -11.169  1.00 97.17           H  \\nATOM   1157  CA  GLY A  79      -0.497  -9.021 -12.118  1.00 97.17           C  \\nATOM   1158  HA2 GLY A  79       0.022  -9.931 -12.420  1.00 97.17           H  \\nATOM   1159  HA3 GLY A  79      -0.151  -8.731 -11.126  1.00 97.17           H  \\nATOM   1160  C   GLY A  79      -0.137  -7.907 -13.105  1.00 97.17           C  \\nATOM   1161  O   GLY A  79       0.672  -8.168 -13.996  1.00 97.17           O  \\nATOM   1162  N   PRO A  80      -0.695  -6.685 -12.970  1.00 97.69           N  \\nATOM   1163  CA  PRO A  80      -0.499  -5.607 -13.945  1.00 97.69           C  \\nATOM   1164  HA  PRO A  80      -0.958  -5.894 -14.891  1.00 97.69           H  \\nATOM   1165  C   PRO A  80       0.974  -5.281 -14.203  1.00 97.69           C  \\nATOM   1166  CB  PRO A  80      -1.249  -4.397 -13.382  1.00 97.69           C  \\nATOM   1167  HB2 PRO A  80      -0.726  -3.457 -13.561  1.00 97.69           H  \\nATOM   1168  HB3 PRO A  80      -2.239  -4.358 -13.837  1.00 97.69           H  \\nATOM   1169  O   PRO A  80       1.368  -5.002 -15.328  1.00 97.69           O  \\nATOM   1170  CG  PRO A  80      -1.374  -4.689 -11.887  1.00 97.69           C  \\nATOM   1171  HG2 PRO A  80      -2.242  -4.199 -11.444  1.00 97.69           H  \\nATOM   1172  HG3 PRO A  80      -0.461  -4.383 -11.376  1.00 97.69           H  \\nATOM   1173  CD  PRO A  80      -1.499  -6.208 -11.850  1.00 97.69           C  \\nATOM   1174  HD2 PRO A  80      -2.541  -6.491 -12.003  1.00 97.69           H  \\nATOM   1175  HD3 PRO A  80      -1.142  -6.590 -10.893  1.00 97.69           H  \\nATOM   1176  N   PHE A  81       1.812  -5.402 -13.174  1.00 98.05           N  \\nATOM   1177  H   PHE A  81       1.412  -5.582 -12.264  1.00 98.05           H  \\nATOM   1178  CA  PHE A  81       3.238  -5.091 -13.237  1.00 98.05           C  \\nATOM   1179  HA  PHE A  81       3.380  -4.263 -13.931  1.00 98.05           H  \\nATOM   1180  C   PHE A  81       4.112  -6.241 -13.766  1.00 98.05           C  \\nATOM   1181  CB  PHE A  81       3.676  -4.614 -11.847  1.00 98.05           C  \\nATOM   1182  HB2 PHE A  81       3.649  -5.462 -11.164  1.00 98.05           H  \\nATOM   1183  HB3 PHE A  81       4.704  -4.253 -11.891  1.00 98.05           H  \\nATOM   1184  O   PHE A  81       5.312  -6.054 -13.946  1.00 98.05           O  \\nATOM   1185  CG  PHE A  81       2.809  -3.508 -11.282  1.00 98.05           C  \\nATOM   1186  CD1 PHE A  81       2.750  -2.259 -11.923  1.00 98.05           C  \\nATOM   1187  HD1 PHE A  81       3.361  -2.074 -12.794  1.00 98.05           H  \\nATOM   1188  CD2 PHE A  81       2.041  -3.735 -10.127  1.00 98.05           C  \\nATOM   1189  HD2 PHE A  81       2.101  -4.687  -9.620  1.00 98.05           H  \\nATOM   1190  CE1 PHE A  81       1.906  -1.250 -11.428  1.00 98.05           C  \\nATOM   1191  HE1 PHE A  81       1.857  -0.291 -11.923  1.00 98.05           H  \\nATOM   1192  CE2 PHE A  81       1.209  -2.720  -9.622  1.00 98.05           C  \\nATOM   1193  HE2 PHE A  81       0.633  -2.879  -8.722  1.00 98.05           H  \\nATOM   1194  CZ  PHE A  81       1.135  -1.480 -10.277  1.00 98.05           C  \\nATOM   1195  HZ  PHE A  81       0.501  -0.698  -9.887  1.00 98.05           H  \\nATOM   1196  N   LYS A  82       3.543  -7.426 -14.037  1.00 97.38           N  \\nATOM   1197  H   LYS A  82       2.545  -7.514 -13.914  1.00 97.38           H  \\nATOM   1198  CA  LYS A  82       4.271  -8.524 -14.703  1.00 97.38           C  \\nATOM   1199  HA  LYS A  82       5.288  -8.573 -14.314  1.00 97.38           H  \\nATOM   1200  C   LYS A  82       4.416  -8.285 -16.204  1.00 97.38           C  \\nATOM   1201  CB  LYS A  82       3.576  -9.871 -14.467  1.00 97.38           C  \\nATOM   1202  HB2 LYS A  82       2.550  -9.815 -14.829  1.00 97.38           H  \\nATOM   1203  HB3 LYS A  82       4.095 -10.629 -15.055  1.00 97.38           H  \\nATOM   1204  O   LYS A  82       5.411  -8.709 -16.780  1.00 97.38           O  \\nATOM   1205  CG  LYS A  82       3.585 -10.310 -12.997  1.00 97.38           C  \\nATOM   1206  HG2 LYS A  82       4.614 -10.377 -12.643  1.00 97.38           H  \\nATOM   1207  HG3 LYS A  82       3.046  -9.581 -12.392  1.00 97.38           H  \\nATOM   1208  CD  LYS A  82       2.904 -11.679 -12.865  1.00 97.38           C  \\nATOM   1209  HD2 LYS A  82       1.878 -11.593 -13.224  1.00 97.38           H  \\nATOM   1210  HD3 LYS A  82       3.433 -12.406 -13.480  1.00 97.38           H  \\nATOM   1211  CE  LYS A  82       2.898 -12.143 -11.405  1.00 97.38           C  \\nATOM   1212  HE2 LYS A  82       2.442 -11.361 -10.798  1.00 97.38           H  \\nATOM   1213  HE3 LYS A  82       3.928 -12.263 -11.070  1.00 97.38           H  \\nATOM   1214  NZ  LYS A  82       2.145 -13.414 -11.240  1.00 97.38           N  \\nATOM   1215  HZ1 LYS A  82       2.547 -14.152 -11.799  1.00 97.38           H  \\nATOM   1216  HZ2 LYS A  82       2.139 -13.705 -10.272  1.00 97.38           H  \\nATOM   1217  HZ3 LYS A  82       1.184 -13.291 -11.525  1.00 97.38           H  \\nATOM   1218  N   ASP A  83       3.432  -7.619 -16.802  1.00 95.39           N  \\nATOM   1219  H   ASP A  83       2.663  -7.296 -16.233  1.00 95.39           H  \\nATOM   1220  CA  ASP A  83       3.415  -7.229 -18.211  1.00 95.39           C  \\nATOM   1221  HA  ASP A  83       4.433  -7.190 -18.600  1.00 95.39           H  \\nATOM   1222  C   ASP A  83       2.797  -5.825 -18.344  1.00 95.39           C  \\nATOM   1223  CB  ASP A  83       2.665  -8.294 -19.020  1.00 95.39           C  \\nATOM   1224  HB2 ASP A  83       3.131  -9.262 -18.835  1.00 95.39           H  \\nATOM   1225  HB3 ASP A  83       1.632  -8.352 -18.680  1.00 95.39           H  \\nATOM   1226  O   ASP A  83       1.602  -5.687 -18.626  1.00 95.39           O  \\nATOM   1227  CG  ASP A  83       2.686  -8.031 -20.529  1.00 95.39           C  \\nATOM   1228  OD1 ASP A  83       3.119  -6.930 -20.953  1.00 95.39           O  \\nATOM   1229  OD2 ASP A  83       2.203  -8.943 -21.235  1.00 95.39           O  \\nATOM   1230  N   PRO A  84       3.591  -4.768 -18.084  1.00 97.10           N  \\nATOM   1231  CA  PRO A  84       3.106  -3.396 -18.118  1.00 97.10           C  \\nATOM   1232  HA  PRO A  84       2.382  -3.307 -17.308  1.00 97.10           H  \\nATOM   1233  C   PRO A  84       2.438  -2.995 -19.439  1.00 97.10           C  \\nATOM   1234  CB  PRO A  84       4.353  -2.541 -17.841  1.00 97.10           C  \\nATOM   1235  HB2 PRO A  84       4.903  -2.370 -18.767  1.00 97.10           H  \\nATOM   1236  HB3 PRO A  84       4.149  -1.584 -17.361  1.00 97.10           H  \\nATOM   1237  O   PRO A  84       1.416  -2.305 -19.448  1.00 97.10           O  \\nATOM   1238  CG  PRO A  84       5.181  -3.432 -16.931  1.00 97.10           C  \\nATOM   1239  HG2 PRO A  84       4.792  -3.393 -15.913  1.00 97.10           H  \\nATOM   1240  HG3 PRO A  84       6.236  -3.160 -16.969  1.00 97.10           H  \\nATOM   1241  CD  PRO A  84       4.940  -4.811 -17.532  1.00 97.10           C  \\nATOM   1242  HD2 PRO A  84       5.653  -4.997 -18.335  1.00 97.10           H  \\nATOM   1243  HD3 PRO A  84       5.046  -5.566 -16.753  1.00 97.10           H  \\nATOM   1244  N   GLN A  85       2.996  -3.463 -20.559  1.00 96.84           N  \\nATOM   1245  H   GLN A  85       3.722  -4.160 -20.477  1.00 96.84           H  \\nATOM   1246  CA  GLN A  85       2.545  -3.100 -21.900  1.00 96.84           C  \\nATOM   1247  HA  GLN A  85       2.482  -2.015 -21.974  1.00 96.84           H  \\nATOM   1248  C   GLN A  85       1.151  -3.651 -22.192  1.00 96.84           C  \\nATOM   1249  CB  GLN A  85       3.551  -3.612 -22.939  1.00 96.84           C  \\nATOM   1250  HB2 GLN A  85       3.726  -4.676 -22.780  1.00 96.84           H  \\nATOM   1251  HB3 GLN A  85       3.125  -3.478 -23.933  1.00 96.84           H  \\nATOM   1252  O   GLN A  85       0.302  -2.906 -22.680  1.00 96.84           O  \\nATOM   1253  CG  GLN A  85       4.883  -2.850 -22.878  1.00 96.84           C  \\nATOM   1254  HG2 GLN A  85       4.690  -1.789 -23.039  1.00 96.84           H  \\nATOM   1255  HG3 GLN A  85       5.337  -2.973 -21.894  1.00 96.84           H  \\nATOM   1256  CD  GLN A  85       5.880  -3.333 -23.926  1.00 96.84           C  \\nATOM   1257  NE2 GLN A  85       6.895  -2.556 -24.234  1.00 96.84           N  \\nATOM   1258 HE21 GLN A  85       7.511  -2.924 -24.945  1.00 96.84           H  \\nATOM   1259 HE22 GLN A  85       7.002  -1.641 -23.821  1.00 96.84           H  \\nATOM   1260  OE1 GLN A  85       5.789  -4.408 -24.487  1.00 96.84           O  \\nATOM   1261  N   LYS A  86       0.878  -4.908 -21.815  1.00 97.07           N  \\nATOM   1262  H   LYS A  86       1.630  -5.475 -21.450  1.00 97.07           H  \\nATOM   1263  CA  LYS A  86      -0.455  -5.520 -21.947  1.00 97.07           C  \\nATOM   1264  HA  LYS A  86      -0.706  -5.603 -23.005  1.00 97.07           H  \\nATOM   1265  C   LYS A  86      -1.562  -4.693 -21.297  1.00 97.07           C  \\nATOM   1266  CB  LYS A  86      -0.393  -6.921 -21.330  1.00 97.07           C  \\nATOM   1267  HB2 LYS A  86       0.252  -7.532 -21.962  1.00 97.07           H  \\nATOM   1268  HB3 LYS A  86       0.044  -6.858 -20.334  1.00 97.07           H  \\nATOM   1269  O   LYS A  86      -2.689  -4.684 -21.786  1.00 97.07           O  \\nATOM   1270  CG  LYS A  86      -1.760  -7.603 -21.216  1.00 97.07           C  \\nATOM   1271  HG2 LYS A  86      -2.405  -7.038 -20.543  1.00 97.07           H  \\nATOM   1272  HG3 LYS A  86      -2.222  -7.659 -22.201  1.00 97.07           H  \\nATOM   1273  CD  LYS A  86      -1.596  -9.007 -20.643  1.00 97.07           C  \\nATOM   1274  HD2 LYS A  86      -0.971  -9.594 -21.317  1.00 97.07           H  \\nATOM   1275  HD3 LYS A  86      -1.113  -8.946 -19.668  1.00 97.07           H  \\nATOM   1276  CE  LYS A  86      -2.982  -9.636 -20.510  1.00 97.07           C  \\nATOM   1277  HE2 LYS A  86      -3.478  -9.578 -21.479  1.00 97.07           H  \\nATOM   1278  HE3 LYS A  86      -3.563  -9.048 -19.799  1.00 97.07           H  \\nATOM   1279  NZ  LYS A  86      -2.872 -11.044 -20.071  1.00 97.07           N  \\nATOM   1280  HZ1 LYS A  86      -3.784 -11.473 -20.007  1.00 97.07           H  \\nATOM   1281  HZ2 LYS A  86      -2.406 -11.091 -19.176  1.00 97.07           H  \\nATOM   1282  HZ3 LYS A  86      -2.315 -11.558 -20.739  1.00 97.07           H  \\nATOM   1283  N   TRP A  87      -1.253  -4.028 -20.189  1.00 96.58           N  \\nATOM   1284  H   TRP A  87      -0.304  -4.097 -19.850  1.00 96.58           H  \\nATOM   1285  CA  TRP A  87      -2.227  -3.276 -19.400  1.00 96.58           C  \\nATOM   1286  HA  TRP A  87      -3.230  -3.604 -19.672  1.00 96.58           H  \\nATOM   1287  C   TRP A  87      -2.213  -1.770 -19.669  1.00 96.58           C  \\nATOM   1288  CB  TRP A  87      -2.020  -3.627 -17.927  1.00 96.58           C  \\nATOM   1289  HB2 TRP A  87      -0.987  -3.414 -17.650  1.00 96.58           H  \\nATOM   1290  HB3 TRP A  87      -2.662  -2.996 -17.312  1.00 96.58           H  \\nATOM   1291  O   TRP A  87      -2.955  -1.038 -19.014  1.00 96.58           O  \\nATOM   1292  CG  TRP A  87      -2.332  -5.055 -17.611  1.00 96.58           C  \\nATOM   1293  CD1 TRP A  87      -1.424  -6.008 -17.315  1.00 96.58           C  \\nATOM   1294  HD1 TRP A  87      -0.359  -5.835 -17.251  1.00 96.58           H  \\nATOM   1295  CD2 TRP A  87      -3.630  -5.721 -17.598  1.00 96.58           C  \\nATOM   1296  CE2 TRP A  87      -3.428  -7.088 -17.240  1.00 96.58           C  \\nATOM   1297  CE3 TRP A  87      -4.958  -5.311 -17.851  1.00 96.58           C  \\nATOM   1298  HE3 TRP A  87      -5.150  -4.287 -18.136  1.00 96.58           H  \\nATOM   1299  NE1 TRP A  87      -2.067  -7.201 -17.051  1.00 96.58           N  \\nATOM   1300  HE1 TRP A  87      -1.581  -8.018 -16.708  1.00 96.58           H  \\nATOM   1301  CH2 TRP A  87      -5.797  -7.559 -17.394  1.00 96.58           C  \\nATOM   1302  HH2 TRP A  87      -6.628  -8.245 -17.326  1.00 96.58           H  \\nATOM   1303  CZ2 TRP A  87      -4.487  -8.002 -17.142  1.00 96.58           C  \\nATOM   1304  HZ2 TRP A  87      -4.291  -9.027 -16.862  1.00 96.58           H  \\nATOM   1305  CZ3 TRP A  87      -6.031  -6.216 -17.741  1.00 96.58           C  \\nATOM   1306  HZ3 TRP A  87      -7.035  -5.873 -17.940  1.00 96.58           H  \\nATOM   1307  N   GLY A  88      -1.386  -1.296 -20.608  1.00 95.66           N  \\nATOM   1308  H   GLY A  88      -0.821  -1.948 -21.134  1.00 95.66           H  \\nATOM   1309  CA  GLY A  88      -1.234   0.133 -20.889  1.00 95.66           C  \\nATOM   1310  HA2 GLY A  88      -2.201   0.555 -21.162  1.00 95.66           H  \\nATOM   1311  HA3 GLY A  88      -0.548   0.253 -21.728  1.00 95.66           H  \\nATOM   1312  C   GLY A  88      -0.678   0.928 -19.704  1.00 95.66           C  \\nATOM   1313  O   GLY A  88      -0.892   2.136 -19.617  1.00 95.66           O  \\nATOM   1314  N   ILE A  89       0.012   0.262 -18.775  1.00 96.52           N  \\nATOM   1315  H   ILE A  89       0.237  -0.706 -18.957  1.00 96.52           H  \\nATOM   1316  CA  ILE A  89       0.690   0.926 -17.666  1.00 96.52           C  \\nATOM   1317  HA  ILE A  89       0.189   1.876 -17.483  1.00 96.52           H  \\nATOM   1318  C   ILE A  89       2.125   1.264 -18.078  1.00 96.52           C  \\nATOM   1319  CB  ILE A  89       0.589   0.133 -16.346  1.00 96.52           C  \\nATOM   1320  HB  ILE A  89       1.076   0.731 -15.576  1.00 96.52           H  \\nATOM   1321  O   ILE A  89       2.719   0.643 -18.963  1.00 96.52           O  \\nATOM   1322  CG1 ILE A  89       1.297  -1.217 -16.431  1.00 96.52           C  \\nATOM   1323 HG12 ILE A  89       0.661  -1.942 -16.939  1.00 96.52           H  \\nATOM   1324 HG13 ILE A  89       2.203  -1.071 -17.020  1.00 96.52           H  \\nATOM   1325  CG2 ILE A  89      -0.873  -0.081 -15.925  1.00 96.52           C  \\nATOM   1326 HG21 ILE A  89      -1.404   0.870 -15.929  1.00 96.52           H  \\nATOM   1327 HG22 ILE A  89      -1.375  -0.755 -16.620  1.00 96.52           H  \\nATOM   1328 HG23 ILE A  89      -0.919  -0.501 -14.920  1.00 96.52           H  \\nATOM   1329  CD1 ILE A  89       1.717  -1.770 -15.072  1.00 96.52           C  \\nATOM   1330 HD11 ILE A  89       2.287  -1.009 -14.539  1.00 96.52           H  \\nATOM   1331 HD12 ILE A  89       0.839  -2.043 -14.486  1.00 96.52           H  \\nATOM   1332 HD13 ILE A  89       2.355  -2.643 -15.206  1.00 96.52           H  \\nATOM   1333  N   LYS A  90       2.701   2.279 -17.439  1.00 95.85           N  \\nATOM   1334  H   LYS A  90       2.223   2.690 -16.649  1.00 95.85           H  \\nATOM   1335  CA  LYS A  90       4.046   2.737 -17.782  1.00 95.85           C  \\nATOM   1336  HA  LYS A  90       4.083   2.838 -18.866  1.00 95.85           H  \\nATOM   1337  C   LYS A  90       5.099   1.691 -17.401  1.00 95.85           C  \\nATOM   1338  CB  LYS A  90       4.278   4.121 -17.171  1.00 95.85           C  \\nATOM   1339  HB2 LYS A  90       5.122   4.589 -17.678  1.00 95.85           H  \\nATOM   1340  HB3 LYS A  90       3.397   4.738 -17.345  1.00 95.85           H  \\nATOM   1341  O   LYS A  90       4.868   0.852 -16.529  1.00 95.85           O  \\nATOM   1342  CG  LYS A  90       4.582   4.059 -15.674  1.00 95.85           C  \\nATOM   1343  HG2 LYS A  90       5.591   3.663 -15.562  1.00 95.85           H  \\nATOM   1344  HG3 LYS A  90       3.898   3.394 -15.147  1.00 95.85           H  \\nATOM   1345  CD  LYS A  90       4.512   5.468 -15.080  1.00 95.85           C  \\nATOM   1346  HD2 LYS A  90       4.974   6.174 -15.769  1.00 95.85           H  \\nATOM   1347  HD3 LYS A  90       3.481   5.759 -14.883  1.00 95.85           H  \\nATOM   1348  CE  LYS A  90       5.312   5.415 -13.800  1.00 95.85           C  \\nATOM   1349  HE2 LYS A  90       6.198   4.821 -14.028  1.00 95.85           H  \\nATOM   1350  HE3 LYS A  90       4.735   4.915 -13.022  1.00 95.85           H  \\nATOM   1351  NZ  LYS A  90       5.779   6.734 -13.353  1.00 95.85           N  \\nATOM   1352  HZ1 LYS A  90       6.194   7.259 -14.109  1.00 95.85           H  \\nATOM   1353  HZ2 LYS A  90       4.981   7.231 -12.984  1.00 95.85           H  \\nATOM   1354  HZ3 LYS A  90       6.441   6.612 -12.600  1.00 95.85           H  \\nATOM   1355  N   ALA A  91       6.270   1.776 -18.032  1.00 97.39           N  \\nATOM   1356  H   ALA A  91       6.390   2.490 -18.736  1.00 97.39           H  \\nATOM   1357  CA  ALA A  91       7.397   0.895 -17.736  1.00 97.39           C  \\nATOM   1358  HA  ALA A  91       7.123  -0.118 -18.028  1.00 97.39           H  \\nATOM   1359  C   ALA A  91       7.730   0.886 -16.235  1.00 97.39           C  \\nATOM   1360  CB  ALA A  91       8.606   1.331 -18.571  1.00 97.39           C  \\nATOM   1361  HB1 ALA A  91       8.362   1.295 -19.632  1.00 97.39           H  \\nATOM   1362  HB2 ALA A  91       8.900   2.346 -18.300  1.00 97.39           H  \\nATOM   1363  HB3 ALA A  91       9.445   0.661 -18.380  1.00 97.39           H  \\nATOM   1364  O   ALA A  91       7.590   1.909 -15.559  1.00 97.39           O  \\nATOM   1365  N   LEU A  92       8.166  -0.269 -15.725  1.00 97.75           N  \\nATOM   1366  H   LEU A  92       8.299  -1.054 -16.347  1.00 97.75           H  \\nATOM   1367  CA  LEU A  92       8.520  -0.442 -14.315  1.00 97.75           C  \\nATOM   1368  HA  LEU A  92       7.628  -0.240 -13.721  1.00 97.75           H  \\nATOM   1369  C   LEU A  92       9.601   0.557 -13.876  1.00 97.75           C  \\nATOM   1370  CB  LEU A  92       8.991  -1.886 -14.054  1.00 97.75           C  \\nATOM   1371  HB2 LEU A  92       9.326  -1.967 -13.020  1.00 97.75           H  \\nATOM   1372  HB3 LEU A  92       9.855  -2.091 -14.686  1.00 97.75           H  \\nATOM   1373  O   LEU A  92      10.411   0.988 -14.702  1.00 97.75           O  \\nATOM   1374  CG  LEU A  92       7.931  -2.968 -14.301  1.00 97.75           C  \\nATOM   1375  HG  LEU A  92       7.726  -3.009 -15.371  1.00 97.75           H  \\nATOM   1376  CD1 LEU A  92       8.481  -4.330 -13.879  1.00 97.75           C  \\nATOM   1377 HD11 LEU A  92       7.758  -5.107 -14.125  1.00 97.75           H  \\nATOM   1378 HD12 LEU A  92       8.665  -4.339 -12.804  1.00 97.75           H  \\nATOM   1379 HD13 LEU A  92       9.411  -4.534 -14.409  1.00 97.75           H  \\nATOM   1380  CD2 LEU A  92       6.621  -2.692 -13.561  1.00 97.75           C  \\nATOM   1381 HD21 LEU A  92       6.112  -1.835 -14.000  1.00 97.75           H  \\nATOM   1382 HD22 LEU A  92       5.968  -3.559 -13.658  1.00 97.75           H  \\nATOM   1383 HD23 LEU A  92       6.817  -2.502 -12.505  1.00 97.75           H  \\nATOM   1384  N   PRO A  93       9.625   0.937 -12.587  1.00 96.92           N  \\nATOM   1385  CA  PRO A  93      10.683   1.794 -12.078  1.00 96.92           C  \\nATOM   1386  HA  PRO A  93      10.691   2.722 -12.650  1.00 96.92           H  \\nATOM   1387  C   PRO A  93      12.054   1.111 -12.210  1.00 96.92           C  \\nATOM   1388  CB  PRO A  93      10.318   2.097 -10.626  1.00 96.92           C  \\nATOM   1389  HB2 PRO A  93       9.753   3.027 -10.572  1.00 96.92           H  \\nATOM   1390  HB3 PRO A  93      11.205   2.156  -9.995  1.00 96.92           H  \\nATOM   1391  O   PRO A  93      12.137  -0.123 -12.172  1.00 96.92           O  \\nATOM   1392  CG  PRO A  93       9.417   0.935 -10.214  1.00 96.92           C  \\nATOM   1393  HG2 PRO A  93       8.676   1.239  -9.474  1.00 96.92           H  \\nATOM   1394  HG3 PRO A  93      10.032   0.123  -9.826  1.00 96.92           H  \\nATOM   1395  CD  PRO A  93       8.747   0.495 -11.510  1.00 96.92           C  \\nATOM   1396  HD2 PRO A  93       7.775   0.981 -11.596  1.00 96.92           H  \\nATOM   1397  HD3 PRO A  93       8.616  -0.587 -11.502  1.00 96.92           H  \\nATOM   1398  N   PRO A  94      13.137   1.896 -12.355  1.00 94.10           N  \\nATOM   1399  CA  PRO A  94      14.483   1.344 -12.380  1.00 94.10           C  \\nATOM   1400  HA  PRO A  94      14.574   0.625 -13.194  1.00 94.10           H  \\nATOM   1401  C   PRO A  94      14.793   0.651 -11.052  1.00 94.10           C  \\nATOM   1402  CB  PRO A  94      15.413   2.533 -12.636  1.00 94.10           C  \\nATOM   1403  HB2 PRO A  94      16.388   2.405 -12.165  1.00 94.10           H  \\nATOM   1404  HB3 PRO A  94      15.527   2.681 -13.710  1.00 94.10           H  \\nATOM   1405  O   PRO A  94      14.376   1.101  -9.985  1.00 94.10           O  \\nATOM   1406  CG  PRO A  94      14.643   3.710 -12.038  1.00 94.10           C  \\nATOM   1407  HG2 PRO A  94      14.804   3.745 -10.960  1.00 94.10           H  \\nATOM   1408  HG3 PRO A  94      14.927   4.657 -12.498  1.00 94.10           H  \\nATOM   1409  CD  PRO A  94      13.186   3.352 -12.328  1.00 94.10           C  \\nATOM   1410  HD2 PRO A  94      12.544   3.762 -11.549  1.00 94.10           H  \\nATOM   1411  HD3 PRO A  94      12.896   3.742 -13.304  1.00 94.10           H  \\nATOM   1412  N   LYS A  95      15.557  -0.441 -11.118  1.00 93.95           N  \\nATOM   1413  H   LYS A  95      15.888  -0.734 -12.026  1.00 93.95           H  \\nATOM   1414  CA  LYS A  95      16.084  -1.110  -9.924  1.00 93.95           C  \\nATOM   1415  HA  LYS A  95      15.262  -1.344  -9.248  1.00 93.95           H  \\nATOM   1416  C   LYS A  95      17.049  -0.189  -9.186  1.00 93.95           C  \\nATOM   1417  CB  LYS A  95      16.786  -2.421 -10.288  1.00 93.95           C  \\nATOM   1418  HB2 LYS A  95      17.210  -2.857  -9.383  1.00 93.95           H  \\nATOM   1419  HB3 LYS A  95      17.598  -2.215 -10.985  1.00 93.95           H  \\nATOM   1420  O   LYS A  95      17.809   0.538  -9.820  1.00 93.95           O  \\nATOM   1421  CG  LYS A  95      15.814  -3.427 -10.908  1.00 93.95           C  \\nATOM   1422  HG2 LYS A  95      15.001  -3.624 -10.209  1.00 93.95           H  \\nATOM   1423  HG3 LYS A  95      15.400  -3.016 -11.829  1.00 93.95           H  \\nATOM   1424  CD  LYS A  95      16.545  -4.733 -11.225  1.00 93.95           C  \\nATOM   1425  HD2 LYS A  95      17.393  -4.544 -11.884  1.00 93.95           H  \\nATOM   1426  HD3 LYS A  95      16.887  -5.193 -10.297  1.00 93.95           H  \\nATOM   1427  CE  LYS A  95      15.530  -5.620 -11.923  1.00 93.95           C  \\nATOM   1428  HE2 LYS A  95      15.216  -5.146 -12.853  1.00 93.95           H  \\nATOM   1429  HE3 LYS A  95      14.645  -5.675 -11.290  1.00 93.95           H  \\nATOM   1430  NZ  LYS A  95      15.991  -6.998 -12.204  1.00 93.95           N  \\nATOM   1431  HZ1 LYS A  95      16.724  -7.004 -12.899  1.00 93.95           H  \\nATOM   1432  HZ2 LYS A  95      15.202  -7.509 -12.572  1.00 93.95           H  \\nATOM   1433  HZ3 LYS A  95      16.302  -7.443 -11.353  1.00 93.95           H  \\nATOM   1434  N   ASN A  96      17.099  -0.329  -7.866  1.00 94.49           N  \\nATOM   1435  H   ASN A  96      16.415  -0.920  -7.415  1.00 94.49           H  \\nATOM   1436  CA  ASN A  96      18.237   0.112  -7.070  1.00 94.49           C  \\nATOM   1437  HA  ASN A  96      18.942   0.657  -7.697  1.00 94.49           H  \\nATOM   1438  C   ASN A  96      18.952  -1.123  -6.496  1.00 94.49           C  \\nATOM   1439  CB  ASN A  96      17.783   1.095  -5.980  1.00 94.49           C  \\nATOM   1440  HB2 ASN A  96      17.281   1.944  -6.444  1.00 94.49           H  \\nATOM   1441  HB3 ASN A  96      17.092   0.611  -5.291  1.00 94.49           H  \\nATOM   1442  O   ASN A  96      18.536  -1.609  -5.442  1.00 94.49           O  \\nATOM   1443  CG  ASN A  96      18.980   1.618  -5.201  1.00 94.49           C  \\nATOM   1444  ND2 ASN A  96      18.782   2.408  -4.183  1.00 94.49           N  \\nATOM   1445 HD21 ASN A  96      17.871   2.534  -3.765  1.00 94.49           H  \\nATOM   1446 HD22 ASN A  96      19.601   2.730  -3.686  1.00 94.49           H  \\nATOM   1447  OD1 ASN A  96      20.127   1.316  -5.493  1.00 94.49           O  \\nATOM   1448  N   PRO A  97      20.006  -1.644  -7.156  1.00 92.85           N  \\nATOM   1449  CA  PRO A  97      20.705  -2.854  -6.718  1.00 92.85           C  \\nATOM   1450  HA  PRO A  97      20.013  -3.694  -6.778  1.00 92.85           H  \\nATOM   1451  C   PRO A  97      21.260  -2.783  -5.291  1.00 92.85           C  \\nATOM   1452  CB  PRO A  97      21.834  -3.064  -7.730  1.00 92.85           C  \\nATOM   1453  HB2 PRO A  97      22.737  -2.549  -7.400  1.00 92.85           H  \\nATOM   1454  HB3 PRO A  97      22.038  -4.123  -7.890  1.00 92.85           H  \\nATOM   1455  O   PRO A  97      21.342  -3.817  -4.635  1.00 92.85           O  \\nATOM   1456  CG  PRO A  97      21.306  -2.391  -8.993  1.00 92.85           C  \\nATOM   1457  HG2 PRO A  97      22.113  -2.078  -9.656  1.00 92.85           H  \\nATOM   1458  HG3 PRO A  97      20.621  -3.064  -9.509  1.00 92.85           H  \\nATOM   1459  CD  PRO A  97      20.535  -1.200  -8.437  1.00 92.85           C  \\nATOM   1460  HD2 PRO A  97      19.745  -0.923  -9.136  1.00 92.85           H  \\nATOM   1461  HD3 PRO A  97      21.210  -0.359  -8.279  1.00 92.85           H  \\nATOM   1462  N   SER A  98      21.577  -1.583  -4.791  1.00 94.02           N  \\nATOM   1463  H   SER A  98      21.519  -0.764  -5.380  1.00 94.02           H  \\nATOM   1464  CA  SER A  98      22.026  -1.375  -3.407  1.00 94.02           C  \\nATOM   1465  HA  SER A  98      22.860  -2.046  -3.200  1.00 94.02           H  \\nATOM   1466  C   SER A  98      20.934  -1.663  -2.370  1.00 94.02           C  \\nATOM   1467  CB  SER A  98      22.512   0.065  -3.223  1.00 94.02           C  \\nATOM   1468  HB2 SER A  98      22.920   0.182  -2.219  1.00 94.02           H  \\nATOM   1469  HB3 SER A  98      21.673   0.750  -3.345  1.00 94.02           H  \\nATOM   1470  O   SER A  98      21.243  -1.891  -1.206  1.00 94.02           O  \\nATOM   1471  OG  SER A  98      23.512   0.374  -4.176  1.00 94.02           O  \\nATOM   1472  HG  SER A  98      23.820   1.269  -4.017  1.00 94.02           H  \\nATOM   1473  N   TRP A  99      19.662  -1.659  -2.780  1.00 94.17           N  \\nATOM   1474  H   TRP A  99      19.487  -1.464  -3.755  1.00 94.17           H  \\nATOM   1475  CA  TRP A  99      18.512  -1.945  -1.920  1.00 94.17           C  \\nATOM   1476  HA  TRP A  99      18.838  -2.026  -0.883  1.00 94.17           H  \\nATOM   1477  C   TRP A  99      17.821  -3.272  -2.272  1.00 94.17           C  \\nATOM   1478  CB  TRP A  99      17.549  -0.759  -2.014  1.00 94.17           C  \\nATOM   1479  HB2 TRP A  99      17.204  -0.653  -3.043  1.00 94.17           H  \\nATOM   1480  HB3 TRP A  99      18.092   0.150  -1.754  1.00 94.17           H  \\nATOM   1481  O   TRP A  99      17.395  -4.021  -1.389  1.00 94.17           O  \\nATOM   1482  CG  TRP A  99      16.361  -0.862  -1.117  1.00 94.17           C  \\nATOM   1483  CD1 TRP A  99      16.334  -0.530   0.191  1.00 94.17           C  \\nATOM   1484  HD1 TRP A  99      17.172  -0.128   0.741  1.00 94.17           H  \\nATOM   1485  CD2 TRP A  99      15.025  -1.341  -1.432  1.00 94.17           C  \\nATOM   1486  CE2 TRP A  99      14.221  -1.250  -0.256  1.00 94.17           C  \\nATOM   1487  CE3 TRP A  99      14.419  -1.843  -2.600  1.00 94.17           C  \\nATOM   1488  HE3 TRP A  99      15.005  -1.921  -3.504  1.00 94.17           H  \\nATOM   1489  NE1 TRP A  99      15.072  -0.750   0.704  1.00 94.17           N  \\nATOM   1490  HE1 TRP A  99      14.821  -0.508   1.652  1.00 94.17           H  \\nATOM   1491  CH2 TRP A  99      12.291  -2.095  -1.428  1.00 94.17           C  \\nATOM   1492  HH2 TRP A  99      11.243  -2.355  -1.455  1.00 94.17           H  \\nATOM   1493  CZ2 TRP A  99      12.867  -1.610  -0.244  1.00 94.17           C  \\nATOM   1494  HZ2 TRP A  99      12.278  -1.498   0.655  1.00 94.17           H  \\nATOM   1495  CZ3 TRP A  99      13.071  -2.231  -2.586  1.00 94.17           C  \\nATOM   1496  HZ3 TRP A  99      12.629  -2.638  -3.483  1.00 94.17           H  \\nATOM   1497  N   SER A 100      17.697  -3.588  -3.565  1.00 93.61           N  \\nATOM   1498  H   SER A 100      18.050  -2.932  -4.247  1.00 93.61           H  \\nATOM   1499  CA  SER A 100      17.191  -4.871  -4.058  1.00 93.61           C  \\nATOM   1500  HA  SER A 100      17.604  -5.668  -3.439  1.00 93.61           H  \\nATOM   1501  C   SER A 100      17.579  -5.112  -5.516  1.00 93.61           C  \\nATOM   1502  CB  SER A 100      15.664  -4.911  -3.955  1.00 93.61           C  \\nATOM   1503  HB2 SER A 100      15.235  -4.190  -4.651  1.00 93.61           H  \\nATOM   1504  HB3 SER A 100      15.359  -4.635  -2.946  1.00 93.61           H  \\nATOM   1505  O   SER A 100      17.405  -4.249  -6.376  1.00 93.61           O  \\nATOM   1506  OG  SER A 100      15.171  -6.208  -4.253  1.00 93.61           O  \\nATOM   1507  HG  SER A 100      14.245  -6.214  -3.998  1.00 93.61           H  \\nATOM   1508  N   ALA A 101      18.011  -6.336  -5.825  1.00 93.43           N  \\nATOM   1509  H   ALA A 101      18.183  -6.986  -5.072  1.00 93.43           H  \\nATOM   1510  CA  ALA A 101      18.237  -6.785  -7.201  1.00 93.43           C  \\nATOM   1511  HA  ALA A 101      18.656  -5.957  -7.774  1.00 93.43           H  \\nATOM   1512  C   ALA A 101      16.942  -7.228  -7.927  1.00 93.43           C  \\nATOM   1513  CB  ALA A 101      19.283  -7.905  -7.160  1.00 93.43           C  \\nATOM   1514  HB1 ALA A 101      19.522  -8.222  -8.175  1.00 93.43           H  \\nATOM   1515  HB2 ALA A 101      18.901  -8.759  -6.600  1.00 93.43           H  \\nATOM   1516  HB3 ALA A 101      20.195  -7.542  -6.686  1.00 93.43           H  \\nATOM   1517  O   ALA A 101      16.956  -7.496  -9.134  1.00 93.43           O  \\nATOM   1518  N   GLN A 102      15.819  -7.341  -7.206  1.00 95.66           N  \\nATOM   1519  H   GLN A 102      15.827  -7.011  -6.251  1.00 95.66           H  \\nATOM   1520  CA  GLN A 102      14.559  -7.844  -7.756  1.00 95.66           C  \\nATOM   1521  HA  GLN A 102      14.785  -8.677  -8.422  1.00 95.66           H  \\nATOM   1522  C   GLN A 102      13.812  -6.787  -8.577  1.00 95.66           C  \\nATOM   1523  CB  GLN A 102      13.644  -8.370  -6.642  1.00 95.66           C  \\nATOM   1524  HB2 GLN A 102      12.697  -8.662  -7.096  1.00 95.66           H  \\nATOM   1525  HB3 GLN A 102      13.455  -7.569  -5.928  1.00 95.66           H  \\nATOM   1526  O   GLN A 102      13.802  -5.607  -8.241  1.00 95.66           O  \\nATOM   1527  CG  GLN A 102      14.216  -9.582  -5.894  1.00 95.66           C  \\nATOM   1528  HG2 GLN A 102      15.103  -9.279  -5.338  1.00 95.66           H  \\nATOM   1529  HG3 GLN A 102      14.510 -10.345  -6.616  1.00 95.66           H  \\nATOM   1530  CD  GLN A 102      13.217 -10.207  -4.919  1.00 95.66           C  \\nATOM   1531  NE2 GLN A 102      13.637 -11.203  -4.171  1.00 95.66           N  \\nATOM   1532 HE21 GLN A 102      12.980 -11.584  -3.507  1.00 95.66           H  \\nATOM   1533 HE22 GLN A 102      14.602 -11.499  -4.198  1.00 95.66           H  \\nATOM   1534  OE1 GLN A 102      12.050  -9.844  -4.826  1.00 95.66           O  \\nATOM   1535  N   ASP A 103      13.126  -7.236  -9.630  1.00 95.83           N  \\nATOM   1536  H   ASP A 103      13.195  -8.211  -9.883  1.00 95.83           H  \\nATOM   1537  CA  ASP A 103      12.181  -6.408 -10.383  1.00 95.83           C  \\nATOM   1538  HA  ASP A 103      12.684  -5.492 -10.693  1.00 95.83           H  \\nATOM   1539  C   ASP A 103      10.961  -6.013  -9.545  1.00 95.83           C  \\nATOM   1540  CB  ASP A 103      11.708  -7.148 -11.641  1.00 95.83           C  \\nATOM   1541  HB2 ASP A 103      11.262  -8.102 -11.359  1.00 95.83           H  \\nATOM   1542  HB3 ASP A 103      10.945  -6.554 -12.144  1.00 95.83           H  \\nATOM   1543  O   ASP A 103      10.408  -6.819  -8.789  1.00 95.83           O  \\nATOM   1544  CG  ASP A 103      12.864  -7.378 -12.602  1.00 95.83           C  \\nATOM   1545  OD1 ASP A 103      13.306  -6.400 -13.225  1.00 95.83           O  \\nATOM   1546  OD2 ASP A 103      13.424  -8.494 -12.608  1.00 95.83           O  \\nATOM   1547  N   PHE A 104      10.480  -4.791  -9.764  1.00 97.73           N  \\nATOM   1548  H   PHE A 104      10.992  -4.179 -10.383  1.00 97.73           H  \\nATOM   1549  CA  PHE A 104       9.274  -4.244  -9.147  1.00 97.73           C  \\nATOM   1550  HA  PHE A 104       9.209  -4.630  -8.130  1.00 97.73           H  \\nATOM   1551  C   PHE A 104       8.006  -4.690  -9.889  1.00 97.73           C  \\nATOM   1552  CB  PHE A 104       9.409  -2.723  -9.058  1.00 97.73           C  \\nATOM   1553  HB2 PHE A 104       8.509  -2.319  -8.594  1.00 97.73           H  \\nATOM   1554  HB3 PHE A 104       9.489  -2.308 -10.063  1.00 97.73           H  \\nATOM   1555  O   PHE A 104       7.292  -3.886 -10.475  1.00 97.73           O  \\nATOM   1556  CG  PHE A 104      10.586  -2.273  -8.226  1.00 97.73           C  \\nATOM   1557  CD1 PHE A 104      10.532  -2.430  -6.834  1.00 97.73           C  \\nATOM   1558  HD1 PHE A 104       9.655  -2.859  -6.375  1.00 97.73           H  \\nATOM   1559  CD2 PHE A 104      11.704  -1.656  -8.815  1.00 97.73           C  \\nATOM   1560  HD2 PHE A 104      11.747  -1.494  -9.882  1.00 97.73           H  \\nATOM   1561  CE1 PHE A 104      11.581  -1.972  -6.030  1.00 97.73           C  \\nATOM   1562  HE1 PHE A 104      11.464  -2.008  -4.957  1.00 97.73           H  \\nATOM   1563  CE2 PHE A 104      12.747  -1.186  -8.000  1.00 97.73           C  \\nATOM   1564  HE2 PHE A 104      13.554  -0.616  -8.436  1.00 97.73           H  \\nATOM   1565  CZ  PHE A 104      12.707  -1.368  -6.609  1.00 97.73           C  \\nATOM   1566  HZ  PHE A 104      13.492  -0.962  -5.988  1.00 97.73           H  \\nATOM   1567  N   LYS A 105       7.759  -6.001  -9.932  1.00 97.47           N  \\nATOM   1568  H   LYS A 105       8.412  -6.595  -9.442  1.00 97.47           H  \\nATOM   1569  CA  LYS A 105       6.662  -6.609 -10.711  1.00 97.47           C  \\nATOM   1570  HA  LYS A 105       6.230  -5.839 -11.351  1.00 97.47           H  \\nATOM   1571  C   LYS A 105       5.491  -7.130  -9.871  1.00 97.47           C  \\nATOM   1572  CB  LYS A 105       7.233  -7.667 -11.658  1.00 97.47           C  \\nATOM   1573  HB2 LYS A 105       6.444  -8.001 -12.332  1.00 97.47           H  \\nATOM   1574  HB3 LYS A 105       8.014  -7.205 -12.262  1.00 97.47           H  \\nATOM   1575  O   LYS A 105       4.568  -7.741 -10.413  1.00 97.47           O  \\nATOM   1576  CG  LYS A 105       7.805  -8.881 -10.913  1.00 97.47           C  \\nATOM   1577  HG2 LYS A 105       7.011  -9.391 -10.369  1.00 97.47           H  \\nATOM   1578  HG3 LYS A 105       8.571  -8.558 -10.208  1.00 97.47           H  \\nATOM   1579  CD  LYS A 105       8.431  -9.847 -11.920  1.00 97.47           C  \\nATOM   1580  HD2 LYS A 105       7.667 -10.191 -12.617  1.00 97.47           H  \\nATOM   1581  HD3 LYS A 105       9.203  -9.316 -12.478  1.00 97.47           H  \\nATOM   1582  CE  LYS A 105       9.055 -11.040 -11.193  1.00 97.47           C  \\nATOM   1583  HE2 LYS A 105       8.265 -11.610 -10.704  1.00 97.47           H  \\nATOM   1584  HE3 LYS A 105       9.724 -10.659 -10.421  1.00 97.47           H  \\nATOM   1585  NZ  LYS A 105       9.815 -11.894 -12.140  1.00 97.47           N  \\nATOM   1586  HZ1 LYS A 105       9.225 -12.218 -12.893  1.00 97.47           H  \\nATOM   1587  HZ2 LYS A 105      10.222 -12.690 -11.670  1.00 97.47           H  \\nATOM   1588  HZ3 LYS A 105      10.568 -11.362 -12.552  1.00 97.47           H  \\nATOM   1589  N   SER A 106       5.539  -6.941  -8.557  1.00 97.62           N  \\nATOM   1590  H   SER A 106       6.279  -6.364  -8.182  1.00 97.62           H  \\nATOM   1591  CA  SER A 106       4.496  -7.364  -7.622  1.00 97.62           C  \\nATOM   1592  HA  SER A 106       3.932  -8.184  -8.066  1.00 97.62           H  \\nATOM   1593  C   SER A 106       3.567  -6.185  -7.322  1.00 97.62           C  \\nATOM   1594  CB  SER A 106       5.106  -7.887  -6.320  1.00 97.62           C  \\nATOM   1595  HB2 SER A 106       5.653  -7.086  -5.823  1.00 97.62           H  \\nATOM   1596  HB3 SER A 106       4.307  -8.224  -5.659  1.00 97.62           H  \\nATOM   1597  O   SER A 106       4.068  -5.094  -7.062  1.00 97.62           O  \\nATOM   1598  OG  SER A 106       5.978  -8.975  -6.577  1.00 97.62           O  \\nATOM   1599  HG  SER A 106       6.512  -9.092  -5.788  1.00 97.62           H  \\nATOM   1600  N   PRO A 107       2.236  -6.359  -7.352  1.00 97.59           N  \\nATOM   1601  CA  PRO A 107       1.314  -5.287  -6.998  1.00 97.59           C  \\nATOM   1602  HA  PRO A 107       1.649  -4.334  -7.408  1.00 97.59           H  \\nATOM   1603  C   PRO A 107       1.220  -5.139  -5.479  1.00 97.59           C  \\nATOM   1604  CB  PRO A 107      -0.020  -5.686  -7.638  1.00 97.59           C  \\nATOM   1605  HB2 PRO A 107      -0.060  -5.318  -8.663  1.00 97.59           H  \\nATOM   1606  HB3 PRO A 107      -0.872  -5.314  -7.070  1.00 97.59           H  \\nATOM   1607  O   PRO A 107       0.711  -6.036  -4.813  1.00 97.59           O  \\nATOM   1608  CG  PRO A 107       0.028  -7.214  -7.653  1.00 97.59           C  \\nATOM   1609  HG2 PRO A 107      -0.543  -7.616  -8.491  1.00 97.59           H  \\nATOM   1610  HG3 PRO A 107      -0.353  -7.605  -6.710  1.00 97.59           H  \\nATOM   1611  CD  PRO A 107       1.513  -7.545  -7.788  1.00 97.59           C  \\nATOM   1612  HD2 PRO A 107       1.748  -7.749  -8.833  1.00 97.59           H  \\nATOM   1613  HD3 PRO A 107       1.761  -8.406  -7.167  1.00 97.59           H  \\nATOM   1614  N   GLU A 108       1.712  -4.026  -4.939  1.00 98.03           N  \\nATOM   1615  H   GLU A 108       2.139  -3.345  -5.550  1.00 98.03           H  \\nATOM   1616  CA  GLU A 108       1.227  -3.504  -3.656  1.00 98.03           C  \\nATOM   1617  HA  GLU A 108       1.193  -4.315  -2.929  1.00 98.03           H  \\nATOM   1618  C   GLU A 108      -0.196  -2.993  -3.828  1.00 98.03           C  \\nATOM   1619  CB  GLU A 108       2.151  -2.392  -3.131  1.00 98.03           C  \\nATOM   1620  HB2 GLU A 108       2.185  -1.571  -3.847  1.00 98.03           H  \\nATOM   1621  HB3 GLU A 108       3.155  -2.806  -3.034  1.00 98.03           H  \\nATOM   1622  O   GLU A 108      -0.501  -2.484  -4.905  1.00 98.03           O  \\nATOM   1623  CG  GLU A 108       1.727  -1.842  -1.759  1.00 98.03           C  \\nATOM   1624  HG2 GLU A 108       2.512  -1.180  -1.394  1.00 98.03           H  \\nATOM   1625  HG3 GLU A 108       0.817  -1.254  -1.875  1.00 98.03           H  \\nATOM   1626  CD  GLU A 108       1.506  -2.987  -0.763  1.00 98.03           C  \\nATOM   1627  OE1 GLU A 108       0.487  -2.988  -0.048  1.00 98.03           O  \\nATOM   1628  OE2 GLU A 108       2.279  -3.967  -0.848  1.00 98.03           O  \\nATOM   1629  N   GLU A 109      -1.056  -3.136  -2.817  1.00 96.89           N  \\nATOM   1630  H   GLU A 109      -0.718  -3.405  -1.904  1.00 96.89           H  \\nATOM   1631  CA  GLU A 109      -2.437  -2.664  -2.894  1.00 96.89           C  \\nATOM   1632  HA  GLU A 109      -2.458  -1.864  -3.634  1.00 96.89           H  \\nATOM   1633  C   GLU A 109      -2.936  -2.066  -1.572  1.00 96.89           C  \\nATOM   1634  CB  GLU A 109      -3.377  -3.756  -3.437  1.00 96.89           C  \\nATOM   1635  HB2 GLU A 109      -2.952  -4.151  -4.360  1.00 96.89           H  \\nATOM   1636  HB3 GLU A 109      -4.333  -3.294  -3.684  1.00 96.89           H  \\nATOM   1637  O   GLU A 109      -2.950  -2.733  -0.535  1.00 96.89           O  \\nATOM   1638  CG  GLU A 109      -3.628  -4.924  -2.469  1.00 96.89           C  \\nATOM   1639  HG2 GLU A 109      -4.230  -4.583  -1.626  1.00 96.89           H  \\nATOM   1640  HG3 GLU A 109      -2.674  -5.290  -2.090  1.00 96.89           H  \\nATOM   1641  CD  GLU A 109      -4.378  -6.080  -3.123  1.00 96.89           C  \\nATOM   1642  OE1 GLU A 109      -4.073  -7.245  -2.782  1.00 96.89           O  \\nATOM   1643  OE2 GLU A 109      -5.295  -5.844  -3.936  1.00 96.89           O  \\nATOM   1644  N   TYR A 110      -3.433  -0.826  -1.634  1.00 98.70           N  \\nATOM   1645  H   TYR A 110      -3.350  -0.321  -2.505  1.00 98.70           H  \\nATOM   1646  CA  TYR A 110      -4.021  -0.141  -0.484  1.00 98.70           C  \\nATOM   1647  HA  TYR A 110      -4.096  -0.873   0.320  1.00 98.70           H  \\nATOM   1648  C   TYR A 110      -5.412   0.446  -0.782  1.00 98.70           C  \\nATOM   1649  CB  TYR A 110      -3.085   0.940   0.050  1.00 98.70           C  \\nATOM   1650  HB2 TYR A 110      -2.067   0.549   0.067  1.00 98.70           H  \\nATOM   1651  HB3 TYR A 110      -3.095   1.806  -0.612  1.00 98.70           H  \\nATOM   1652  O   TYR A 110      -5.584   1.173  -1.754  1.00 98.70           O  \\nATOM   1653  CG  TYR A 110      -3.488   1.342   1.453  1.00 98.70           C  \\nATOM   1654  CD1 TYR A 110      -4.359   2.428   1.663  1.00 98.70           C  \\nATOM   1655  HD1 TYR A 110      -4.686   3.018   0.820  1.00 98.70           H  \\nATOM   1656  CD2 TYR A 110      -3.076   0.550   2.541  1.00 98.70           C  \\nATOM   1657  HD2 TYR A 110      -2.407  -0.280   2.369  1.00 98.70           H  \\nATOM   1658  CE1 TYR A 110      -4.801   2.734   2.965  1.00 98.70           C  \\nATOM   1659  HE1 TYR A 110      -5.467   3.565   3.150  1.00 98.70           H  \\nATOM   1660  CE2 TYR A 110      -3.497   0.865   3.844  1.00 98.70           C  \\nATOM   1661  HE2 TYR A 110      -3.136   0.275   4.674  1.00 98.70           H  \\nATOM   1662  OH  TYR A 110      -4.730   2.276   5.325  1.00 98.70           O  \\nATOM   1663  HH  TYR A 110      -4.292   1.707   5.963  1.00 98.70           H  \\nATOM   1664  CZ  TYR A 110      -4.353   1.964   4.059  1.00 98.70           C  \\nATOM   1665  N   ALA A 111      -6.442   0.249   0.042  1.00 98.44           N  \\nATOM   1666  H   ALA A 111      -7.323   0.645  -0.254  1.00 98.44           H  \\nATOM   1667  CA  ALA A 111      -6.422  -0.468   1.315  1.00 98.44           C  \\nATOM   1668  HA  ALA A 111      -5.647  -0.037   1.949  1.00 98.44           H  \\nATOM   1669  C   ALA A 111      -6.115  -1.962   1.150  1.00 98.44           C  \\nATOM   1670  CB  ALA A 111      -7.759  -0.245   2.023  1.00 98.44           C  \\nATOM   1671  HB1 ALA A 111      -8.570  -0.642   1.414  1.00 98.44           H  \\nATOM   1672  HB2 ALA A 111      -7.911   0.823   2.182  1.00 98.44           H  \\nATOM   1673  HB3 ALA A 111      -7.754  -0.748   2.990  1.00 98.44           H  \\nATOM   1674  O   ALA A 111      -6.421  -2.558   0.116  1.00 98.44           O  \\nATOM   1675  N   PHE A 112      -5.517  -2.553   2.189  1.00 97.95           N  \\nATOM   1676  H   PHE A 112      -5.227  -1.976   2.966  1.00 97.95           H  \\nATOM   1677  CA  PHE A 112      -5.054  -3.937   2.163  1.00 97.95           C  \\nATOM   1678  HA  PHE A 112      -4.198  -4.001   1.491  1.00 97.95           H  \\nATOM   1679  C   PHE A 112      -6.145  -4.886   1.674  1.00 97.95           C  \\nATOM   1680  CB  PHE A 112      -4.640  -4.384   3.576  1.00 97.95           C  \\nATOM   1681  HB2 PHE A 112      -5.525  -4.406   4.213  1.00 97.95           H  \\nATOM   1682  HB3 PHE A 112      -4.255  -5.402   3.511  1.00 97.95           H  \\nATOM   1683  O   PHE A 112      -7.282  -4.820   2.156  1.00 97.95           O  \\nATOM   1684  CG  PHE A 112      -3.578  -3.538   4.239  1.00 97.95           C  \\nATOM   1685  CD1 PHE A 112      -2.354  -3.327   3.583  1.00 97.95           C  \\nATOM   1686  HD1 PHE A 112      -2.157  -3.740   2.605  1.00 97.95           H  \\nATOM   1687  CD2 PHE A 112      -3.800  -2.969   5.509  1.00 97.95           C  \\nATOM   1688  HD2 PHE A 112      -4.736  -3.140   6.020  1.00 97.95           H  \\nATOM   1689  CE1 PHE A 112      -1.368  -2.542   4.190  1.00 97.95           C  \\nATOM   1690  HE1 PHE A 112      -0.440  -2.372   3.664  1.00 97.95           H  \\nATOM   1691  CE2 PHE A 112      -2.807  -2.176   6.114  1.00 97.95           C  \\nATOM   1692  HE2 PHE A 112      -2.976  -1.725   7.081  1.00 97.95           H  \\nATOM   1693  CZ  PHE A 112      -1.588  -1.959   5.448  1.00 97.95           C  \\nATOM   1694  HZ  PHE A 112      -0.819  -1.337   5.881  1.00 97.95           H  \\nATOM   1695  N   ALA A 113      -5.797  -5.861   0.831  1.00 97.70           N  \\nATOM   1696  H   ALA A 113      -4.903  -5.810   0.362  1.00 97.70           H  \\nATOM   1697  CA  ALA A 113      -6.735  -6.929   0.487  1.00 97.70           C  \\nATOM   1698  HA  ALA A 113      -7.599  -6.482  -0.006  1.00 97.70           H  \\nATOM   1699  C   ALA A 113      -7.257  -7.670   1.724  1.00 97.70           C  \\nATOM   1700  CB  ALA A 113      -6.087  -7.908  -0.488  1.00 97.70           C  \\nATOM   1701  HB1 ALA A 113      -5.065  -8.146  -0.191  1.00 97.70           H  \\nATOM   1702  HB2 ALA A 113      -6.678  -8.822  -0.546  1.00 97.70           H  \\nATOM   1703  HB3 ALA A 113      -6.096  -7.451  -1.478  1.00 97.70           H  \\nATOM   1704  O   ALA A 113      -8.386  -8.143   1.718  1.00 97.70           O  \\nATOM   1705  N   SER A 114      -6.489  -7.737   2.814  1.00 97.06           N  \\nATOM   1706  H   SER A 114      -5.590  -7.275   2.801  1.00 97.06           H  \\nATOM   1707  CA  SER A 114      -6.924  -8.317   4.089  1.00 97.06           C  \\nATOM   1708  HA  SER A 114      -7.458  -9.245   3.885  1.00 97.06           H  \\nATOM   1709  C   SER A 114      -7.896  -7.435   4.884  1.00 97.06           C  \\nATOM   1710  CB  SER A 114      -5.703  -8.657   4.949  1.00 97.06           C  \\nATOM   1711  HB2 SER A 114      -5.106  -9.419   4.448  1.00 97.06           H  \\nATOM   1712  HB3 SER A 114      -6.021  -9.039   5.919  1.00 97.06           H  \\nATOM   1713  O   SER A 114      -8.224  -7.798   6.011  1.00 97.06           O  \\nATOM   1714  OG  SER A 114      -4.923  -7.490   5.119  1.00 97.06           O  \\nATOM   1715  HG  SER A 114      -4.450  -7.496   5.954  1.00 97.06           H  \\nATOM   1716  N   SER A 115      -8.370  -6.310   4.341  1.00 98.21           N  \\nATOM   1717  H   SER A 115      -8.082  -6.069   3.404  1.00 98.21           H  \\nATOM   1718  CA  SER A 115      -9.331  -5.407   4.987  1.00 98.21           C  \\nATOM   1719  HA  SER A 115      -9.517  -5.780   5.994  1.00 98.21           H  \\nATOM   1720  C   SER A 115     -10.681  -5.349   4.264  1.00 98.21           C  \\nATOM   1721  CB  SER A 115      -8.737  -4.004   5.137  1.00 98.21           C  \\nATOM   1722  HB2 SER A 115      -9.359  -3.433   5.827  1.00 98.21           H  \\nATOM   1723  HB3 SER A 115      -7.732  -4.073   5.554  1.00 98.21           H  \\nATOM   1724  O   SER A 115     -10.764  -5.612   3.068  1.00 98.21           O  \\nATOM   1725  OG  SER A 115      -8.698  -3.317   3.903  1.00 98.21           O  \\nATOM   1726  HG  SER A 115      -8.184  -3.819   3.266  1.00 98.21           H  \\nATOM   1727  N   LEU A 116     -11.745  -4.961   4.971  1.00 97.74           N  \\nATOM   1728  H   LEU A 116     -11.624  -4.805   5.962  1.00 97.74           H  \\nATOM   1729  CA  LEU A 116     -13.079  -4.742   4.397  1.00 97.74           C  \\nATOM   1730  HA  LEU A 116     -13.411  -5.660   3.913  1.00 97.74           H  \\nATOM   1731  C   LEU A 116     -13.061  -3.640   3.327  1.00 97.74           C  \\nATOM   1732  CB  LEU A 116     -14.043  -4.383   5.546  1.00 97.74           C  \\nATOM   1733  HB2 LEU A 116     -14.034  -5.186   6.283  1.00 97.74           H  \\nATOM   1734  HB3 LEU A 116     -13.675  -3.481   6.035  1.00 97.74           H  \\nATOM   1735  O   LEU A 116     -13.811  -3.696   2.358  1.00 97.74           O  \\nATOM   1736  CG  LEU A 116     -15.499  -4.133   5.100  1.00 97.74           C  \\nATOM   1737  HG  LEU A 116     -15.525  -3.380   4.312  1.00 97.74           H  \\nATOM   1738  CD1 LEU A 116     -16.150  -5.415   4.575  1.00 97.74           C  \\nATOM   1739 HD11 LEU A 116     -15.650  -5.738   3.662  1.00 97.74           H  \\nATOM   1740 HD12 LEU A 116     -17.197  -5.222   4.340  1.00 97.74           H  \\nATOM   1741 HD13 LEU A 116     -16.087  -6.201   5.328  1.00 97.74           H  \\nATOM   1742  CD2 LEU A 116     -16.317  -3.580   6.266  1.00 97.74           C  \\nATOM   1743 HD21 LEU A 116     -16.265  -4.240   7.133  1.00 97.74           H  \\nATOM   1744 HD22 LEU A 116     -15.925  -2.605   6.556  1.00 97.74           H  \\nATOM   1745 HD23 LEU A 116     -17.358  -3.458   5.967  1.00 97.74           H  \\nATOM   1746  N   GLN A 117     -12.184  -2.655   3.507  1.00 98.02           N  \\nATOM   1747  H   GLN A 117     -11.618  -2.688   4.343  1.00 98.02           H  \\nATOM   1748  CA  GLN A 117     -11.989  -1.507   2.628  1.00 98.02           C  \\nATOM   1749  HA  GLN A 117     -12.959  -1.143   2.289  1.00 98.02           H  \\nATOM   1750  C   GLN A 117     -11.170  -1.849   1.375  1.00 98.02           C  \\nATOM   1751  CB  GLN A 117     -11.283  -0.399   3.427  1.00 98.02           C  \\nATOM   1752  HB2 GLN A 117     -10.321  -0.768   3.781  1.00 98.02           H  \\nATOM   1753  HB3 GLN A 117     -11.085   0.425   2.742  1.00 98.02           H  \\nATOM   1754  O   GLN A 117     -11.069  -1.023   0.470  1.00 98.02           O  \\nATOM   1755  CG  GLN A 117     -12.095   0.161   4.610  1.00 98.02           C  \\nATOM   1756  HG2 GLN A 117     -11.603   1.066   4.969  1.00 98.02           H  \\nATOM   1757  HG3 GLN A 117     -13.083   0.447   4.250  1.00 98.02           H  \\nATOM   1758  CD  GLN A 117     -12.249  -0.779   5.810  1.00 98.02           C  \\nATOM   1759  NE2 GLN A 117     -13.295  -0.612   6.587  1.00 98.02           N  \\nATOM   1760 HE21 GLN A 117     -13.378  -1.203   7.403  1.00 98.02           H  \\nATOM   1761 HE22 GLN A 117     -13.932   0.153   6.421  1.00 98.02           H  \\nATOM   1762  OE1 GLN A 117     -11.469  -1.690   6.068  1.00 98.02           O  \\nATOM   1763  N   GLY A 118     -10.560  -3.036   1.330  1.00 97.99           N  \\nATOM   1764  H   GLY A 118     -10.689  -3.674   2.102  1.00 97.99           H  \\nATOM   1765  CA  GLY A 118      -9.782  -3.499   0.192  1.00 97.99           C  \\nATOM   1766  HA2 GLY A 118      -9.090  -2.714  -0.113  1.00 97.99           H  \\nATOM   1767  HA3 GLY A 118      -9.194  -4.368   0.490  1.00 97.99           H  \\nATOM   1768  C   GLY A 118     -10.640  -3.890  -1.013  1.00 97.99           C  \\nATOM   1769  O   GLY A 118     -11.863  -4.071  -0.946  1.00 97.99           O  \\nATOM   1770  N   GLY A 119      -9.961  -4.095  -2.138  1.00 97.62           N  \\nATOM   1771  H   GLY A 119      -8.972  -3.891  -2.145  1.00 97.62           H  \\nATOM   1772  CA  GLY A 119     -10.579  -4.496  -3.395  1.00 97.62           C  \\nATOM   1773  HA2 GLY A 119      -9.877  -5.142  -3.921  1.00 97.62           H  \\nATOM   1774  HA3 GLY A 119     -11.499  -5.054  -3.218  1.00 97.62           H  \\nATOM   1775  C   GLY A 119     -10.844  -3.319  -4.318  1.00 97.62           C  \\nATOM   1776  O   GLY A 119      -9.908  -2.647  -4.740  1.00 97.62           O  \\nATOM   1777  N   THR A 120     -12.105  -3.097  -4.683  1.00 97.23           N  \\nATOM   1778  H   THR A 120     -12.834  -3.692  -4.316  1.00 97.23           H  \\nATOM   1779  CA  THR A 120     -12.495  -2.025  -5.611  1.00 97.23           C  \\nATOM   1780  HA  THR A 120     -12.092  -2.261  -6.596  1.00 97.23           H  \\nATOM   1781  C   THR A 120     -11.932  -0.676  -5.156  1.00 97.23           C  \\nATOM   1782  CB  THR A 120     -14.024  -1.929  -5.716  1.00 97.23           C  \\nATOM   1783  HB  THR A 120     -14.422  -1.581  -4.763  1.00 97.23           H  \\nATOM   1784  O   THR A 120     -12.043  -0.321  -3.985  1.00 97.23           O  \\nATOM   1785  CG2 THR A 120     -14.486  -0.983  -6.822  1.00 97.23           C  \\nATOM   1786 HG21 THR A 120     -14.197   0.041  -6.587  1.00 97.23           H  \\nATOM   1787 HG22 THR A 120     -14.043  -1.271  -7.775  1.00 97.23           H  \\nATOM   1788 HG23 THR A 120     -15.572  -1.022  -6.905  1.00 97.23           H  \\nATOM   1789  OG1 THR A 120     -14.580  -3.203  -5.988  1.00 97.23           O  \\nATOM   1790  HG1 THR A 120     -14.623  -3.302  -6.942  1.00 97.23           H  \\nATOM   1791  N   ASN A 121     -11.357   0.085  -6.091  1.00 96.14           N  \\nATOM   1792  H   ASN A 121     -11.336  -0.268  -7.037  1.00 96.14           H  \\nATOM   1793  CA  ASN A 121     -10.685   1.371  -5.856  1.00 96.14           C  \\nATOM   1794  HA  ASN A 121     -10.353   1.721  -6.834  1.00 96.14           H  \\nATOM   1795  C   ASN A 121      -9.403   1.313  -5.008  1.00 96.14           C  \\nATOM   1796  CB  ASN A 121     -11.674   2.427  -5.325  1.00 96.14           C  \\nATOM   1797  HB2 ASN A 121     -11.960   2.179  -4.303  1.00 96.14           H  \\nATOM   1798  HB3 ASN A 121     -11.183   3.399  -5.300  1.00 96.14           H  \\nATOM   1799  O   ASN A 121      -8.845   2.373  -4.705  1.00 96.14           O  \\nATOM   1800  CG  ASN A 121     -12.911   2.572  -6.181  1.00 96.14           C  \\nATOM   1801  ND2 ASN A 121     -14.066   2.714  -5.575  1.00 96.14           N  \\nATOM   1802 HD21 ASN A 121     -14.873   2.824  -6.171  1.00 96.14           H  \\nATOM   1803 HD22 ASN A 121     -14.113   2.699  -4.566  1.00 96.14           H  \\nATOM   1804  OD1 ASN A 121     -12.864   2.559  -7.397  1.00 96.14           O  \\nATOM   1805  N   ALA A 122      -8.893   0.126  -4.662  1.00 98.45           N  \\nATOM   1806  H   ALA A 122      -9.382  -0.730  -4.883  1.00 98.45           H  \\nATOM   1807  CA  ALA A 122      -7.557   0.026  -4.093  1.00 98.45           C  \\nATOM   1808  HA  ALA A 122      -7.546   0.616  -3.177  1.00 98.45           H  \\nATOM   1809  C   ALA A 122      -6.528   0.627  -5.063  1.00 98.45           C  \\nATOM   1810  CB  ALA A 122      -7.226  -1.420  -3.718  1.00 98.45           C  \\nATOM   1811  HB1 ALA A 122      -6.237  -1.465  -3.262  1.00 98.45           H  \\nATOM   1812  HB2 ALA A 122      -7.959  -1.795  -3.004  1.00 98.45           H  \\nATOM   1813  HB3 ALA A 122      -7.232  -2.041  -4.614  1.00 98.45           H  \\nATOM   1814  O   ALA A 122      -6.595   0.452  -6.282  1.00 98.45           O  \\nATOM   1815  N   ILE A 123      -5.593   1.375  -4.496  1.00 98.62           N  \\nATOM   1816  H   ILE A 123      -5.607   1.436  -3.488  1.00 98.62           H  \\nATOM   1817  CA  ILE A 123      -4.429   1.928  -5.165  1.00 98.62           C  \\nATOM   1818  HA  ILE A 123      -4.704   2.275  -6.161  1.00 98.62           H  \\nATOM   1819  C   ILE A 123      -3.401   0.821  -5.293  1.00 98.62           C  \\nATOM   1820  CB  ILE A 123      -3.899   3.112  -4.336  1.00 98.62           C  \\nATOM   1821  HB  ILE A 123      -3.928   2.838  -3.281  1.00 98.62           H  \\nATOM   1822  O   ILE A 123      -3.010   0.235  -4.290  1.00 98.62           O  \\nATOM   1823  CG1 ILE A 123      -4.832   4.320  -4.572  1.00 98.62           C  \\nATOM   1824 HG12 ILE A 123      -4.672   4.701  -5.580  1.00 98.62           H  \\nATOM   1825 HG13 ILE A 123      -5.871   4.001  -4.500  1.00 98.62           H  \\nATOM   1826  CG2 ILE A 123      -2.444   3.467  -4.681  1.00 98.62           C  \\nATOM   1827 HG21 ILE A 123      -2.094   4.215  -3.971  1.00 98.62           H  \\nATOM   1828 HG22 ILE A 123      -2.369   3.818  -5.710  1.00 98.62           H  \\nATOM   1829 HG23 ILE A 123      -1.783   2.611  -4.540  1.00 98.62           H  \\nATOM   1830  CD1 ILE A 123      -4.645   5.464  -3.576  1.00 98.62           C  \\nATOM   1831 HD11 ILE A 123      -3.613   5.815  -3.565  1.00 98.62           H  \\nATOM   1832 HD12 ILE A 123      -5.300   6.287  -3.863  1.00 98.62           H  \\nATOM   1833 HD13 ILE A 123      -4.938   5.125  -2.582  1.00 98.62           H  \\nATOM   1834  N   LEU A 124      -2.945   0.577  -6.516  1.00 98.50           N  \\nATOM   1835  H   LEU A 124      -3.350   1.093  -7.284  1.00 98.50           H  \\nATOM   1836  CA  LEU A 124      -1.891  -0.371  -6.819  1.00 98.50           C  \\nATOM   1837  HA  LEU A 124      -1.701  -0.941  -5.910  1.00 98.50           H  \\nATOM   1838  C   LEU A 124      -0.601   0.342  -7.196  1.00 98.50           C  \\nATOM   1839  CB  LEU A 124      -2.308  -1.350  -7.926  1.00 98.50           C  \\nATOM   1840  HB2 LEU A 124      -1.396  -1.796  -8.324  1.00 98.50           H  \\nATOM   1841  HB3 LEU A 124      -2.794  -0.810  -8.738  1.00 98.50           H  \\nATOM   1842  O   LEU A 124      -0.628   1.266  -8.008  1.00 98.50           O  \\nATOM   1843  CG  LEU A 124      -3.203  -2.497  -7.437  1.00 98.50           C  \\nATOM   1844  HG  LEU A 124      -2.792  -2.927  -6.524  1.00 98.50           H  \\nATOM   1845  CD1 LEU A 124      -4.638  -2.051  -7.186  1.00 98.50           C  \\nATOM   1846 HD11 LEU A 124      -4.679  -1.403  -6.311  1.00 98.50           H  \\nATOM   1847 HD12 LEU A 124      -5.047  -1.533  -8.054  1.00 98.50           H  \\nATOM   1848 HD13 LEU A 124      -5.246  -2.928  -6.967  1.00 98.50           H  \\nATOM   1849  CD2 LEU A 124      -3.227  -3.590  -8.505  1.00 98.50           C  \\nATOM   1850 HD21 LEU A 124      -2.223  -3.985  -8.657  1.00 98.50           H  \\nATOM   1851 HD22 LEU A 124      -3.885  -4.394  -8.174  1.00 98.50           H  \\nATOM   1852 HD23 LEU A 124      -3.607  -3.173  -9.437  1.00 98.50           H  \\nATOM   1853  N   ALA A 125       0.526  -0.150  -6.689  1.00 98.50           N  \\nATOM   1854  H   ALA A 125       0.459  -0.911  -6.028  1.00 98.50           H  \\nATOM   1855  CA  ALA A 125       1.855   0.301  -7.091  1.00 98.50           C  \\nATOM   1856  HA  ALA A 125       1.766   0.792  -8.060  1.00 98.50           H  \\nATOM   1857  C   ALA A 125       2.830  -0.882  -7.242  1.00 98.50           C  \\nATOM   1858  CB  ALA A 125       2.365   1.337  -6.085  1.00 98.50           C  \\nATOM   1859  HB1 ALA A 125       3.292   1.788  -6.439  1.00 98.50           H  \\nATOM   1860  HB2 ALA A 125       2.558   0.844  -5.132  1.00 98.50           H  \\nATOM   1861  HB3 ALA A 125       1.622   2.123  -5.947  1.00 98.50           H  \\nATOM   1862  O   ALA A 125       2.664  -1.912  -6.585  1.00 98.50           O  \\nATOM   1863  N   PRO A 126       3.851  -0.768  -8.106  1.00 98.19           N  \\nATOM   1864  CA  PRO A 126       4.847  -1.806  -8.298  1.00 98.19           C  \\nATOM   1865  HA  PRO A 126       4.362  -2.777  -8.405  1.00 98.19           H  \\nATOM   1866  C   PRO A 126       5.826  -1.849  -7.126  1.00 98.19           C  \\nATOM   1867  CB  PRO A 126       5.556  -1.454  -9.604  1.00 98.19           C  \\nATOM   1868  HB2 PRO A 126       5.043  -1.920 -10.445  1.00 98.19           H  \\nATOM   1869  HB3 PRO A 126       6.609  -1.735  -9.592  1.00 98.19           H  \\nATOM   1870  O   PRO A 126       6.538  -0.886  -6.840  1.00 98.19           O  \\nATOM   1871  CG  PRO A 126       5.431   0.058  -9.692  1.00 98.19           C  \\nATOM   1872  HG2 PRO A 126       5.434   0.388 -10.730  1.00 98.19           H  \\nATOM   1873  HG3 PRO A 126       6.240   0.532  -9.135  1.00 98.19           H  \\nATOM   1874  CD  PRO A 126       4.100   0.343  -9.003  1.00 98.19           C  \\nATOM   1875  HD2 PRO A 126       4.158   1.282  -8.453  1.00 98.19           H  \\nATOM   1876  HD3 PRO A 126       3.305   0.387  -9.747  1.00 98.19           H  \\nATOM   1877  N   VAL A 127       5.922  -3.017  -6.506  1.00 98.19           N  \\nATOM   1878  H   VAL A 127       5.265  -3.738  -6.767  1.00 98.19           H  \\nATOM   1879  CA  VAL A 127       6.939  -3.366  -5.513  1.00 98.19           C  \\nATOM   1880  HA  VAL A 127       7.713  -2.599  -5.527  1.00 98.19           H  \\nATOM   1881  C   VAL A 127       7.595  -4.690  -5.906  1.00 98.19           C  \\nATOM   1882  CB  VAL A 127       6.356  -3.372  -4.089  1.00 98.19           C  \\nATOM   1883  HB  VAL A 127       7.175  -3.525  -3.386  1.00 98.19           H  \\nATOM   1884  O   VAL A 127       7.198  -5.331  -6.883  1.00 98.19           O  \\nATOM   1885  CG1 VAL A 127       5.698  -2.028  -3.764  1.00 98.19           C  \\nATOM   1886 HG11 VAL A 127       5.467  -1.969  -2.701  1.00 98.19           H  \\nATOM   1887 HG12 VAL A 127       6.356  -1.206  -4.045  1.00 98.19           H  \\nATOM   1888 HG13 VAL A 127       4.774  -1.910  -4.332  1.00 98.19           H  \\nATOM   1889  CG2 VAL A 127       5.333  -4.482  -3.880  1.00 98.19           C  \\nATOM   1890 HG21 VAL A 127       5.796  -5.455  -4.043  1.00 98.19           H  \\nATOM   1891 HG22 VAL A 127       4.955  -4.422  -2.859  1.00 98.19           H  \\nATOM   1892 HG23 VAL A 127       4.492  -4.356  -4.562  1.00 98.19           H  \\nATOM   1893  N   ASN A 128       8.639  -5.115  -5.197  1.00 97.05           N  \\nATOM   1894  H   ASN A 128       8.905  -4.609  -4.365  1.00 97.05           H  \\nATOM   1895  CA  ASN A 128       9.236  -6.430  -5.437  1.00 97.05           C  \\nATOM   1896  HA  ASN A 128       8.997  -6.743  -6.454  1.00 97.05           H  \\nATOM   1897  C   ASN A 128       8.608  -7.503  -4.525  1.00 97.05           C  \\nATOM   1898  CB  ASN A 128      10.768  -6.329  -5.413  1.00 97.05           C  \\nATOM   1899  HB2 ASN A 128      11.087  -5.544  -6.099  1.00 97.05           H  \\nATOM   1900  HB3 ASN A 128      11.181  -7.270  -5.774  1.00 97.05           H  \\nATOM   1901  O   ASN A 128       7.882  -7.202  -3.575  1.00 97.05           O  \\nATOM   1902  CG  ASN A 128      11.342  -6.073  -4.039  1.00 97.05           C  \\nATOM   1903  ND2 ASN A 128      12.268  -5.158  -3.901  1.00 97.05           N  \\nATOM   1904 HD21 ASN A 128      12.479  -4.906  -2.945  1.00 97.05           H  \\nATOM   1905 HD22 ASN A 128      12.523  -4.575  -4.685  1.00 97.05           H  \\nATOM   1906  OD1 ASN A 128      10.986  -6.719  -3.080  1.00 97.05           O  \\nATOM   1907  N   LEU A 129       8.844  -8.778  -4.837  1.00 95.72           N  \\nATOM   1908  H   LEU A 129       9.518  -8.983  -5.561  1.00 95.72           H  \\nATOM   1909  CA  LEU A 129       8.254  -9.892  -4.087  1.00 95.72           C  \\nATOM   1910  HA  LEU A 129       7.175  -9.746  -4.036  1.00 95.72           H  \\nATOM   1911  C   LEU A 129       8.758  -9.946  -2.639  1.00 95.72           C  \\nATOM   1912  CB  LEU A 129       8.552 -11.198  -4.841  1.00 95.72           C  \\nATOM   1913  HB2 LEU A 129       8.121 -11.130  -5.841  1.00 95.72           H  \\nATOM   1914  HB3 LEU A 129       9.632 -11.301  -4.944  1.00 95.72           H  \\nATOM   1915  O   LEU A 129       7.980 -10.227  -1.730  1.00 95.72           O  \\nATOM   1916  CG  LEU A 129       8.009 -12.466  -4.152  1.00 95.72           C  \\nATOM   1917  HG  LEU A 129       8.468 -12.575  -3.169  1.00 95.72           H  \\nATOM   1918  CD1 LEU A 129       6.488 -12.437  -3.995  1.00 95.72           C  \\nATOM   1919 HD11 LEU A 129       6.142 -13.386  -3.585  1.00 95.72           H  \\nATOM   1920 HD12 LEU A 129       6.006 -12.259  -4.956  1.00 95.72           H  \\nATOM   1921 HD13 LEU A 129       6.199 -11.658  -3.289  1.00 95.72           H  \\nATOM   1922  CD2 LEU A 129       8.388 -13.691  -4.982  1.00 95.72           C  \\nATOM   1923 HD21 LEU A 129       7.929 -13.635  -5.969  1.00 95.72           H  \\nATOM   1924 HD22 LEU A 129       8.043 -14.594  -4.478  1.00 95.72           H  \\nATOM   1925 HD23 LEU A 129       9.472 -13.746  -5.085  1.00 95.72           H  \\nATOM   1926  N   ALA A 130      10.040  -9.662  -2.414  1.00 95.72           N  \\nATOM   1927  H   ALA A 130      10.635  -9.402  -3.187  1.00 95.72           H  \\nATOM   1928  CA  ALA A 130      10.610  -9.677  -1.073  1.00 95.72           C  \\nATOM   1929  HA  ALA A 130      10.386 -10.638  -0.611  1.00 95.72           H  \\nATOM   1930  C   ALA A 130      10.001  -8.593  -0.163  1.00 95.72           C  \\nATOM   1931  CB  ALA A 130      12.124  -9.566  -1.213  1.00 95.72           C  \\nATOM   1932  HB1 ALA A 130      12.502 -10.394  -1.812  1.00 95.72           H  \\nATOM   1933  HB2 ALA A 130      12.390  -8.622  -1.689  1.00 95.72           H  \\nATOM   1934  HB3 ALA A 130      12.575  -9.627  -0.223  1.00 95.72           H  \\nATOM   1935  O   ALA A 130       9.683  -8.871   0.993  1.00 95.72           O  \\nATOM   1936  N   SER A 131       9.728  -7.406  -0.706  1.00 97.47           N  \\nATOM   1937  H   SER A 131      10.074  -7.220  -1.636  1.00 97.47           H  \\nATOM   1938  CA  SER A 131       8.980  -6.340  -0.043  1.00 97.47           C  \\nATOM   1939  HA  SER A 131       9.503  -6.047   0.867  1.00 97.47           H  \\nATOM   1940  C   SER A 131       7.580  -6.781   0.362  1.00 97.47           C  \\nATOM   1941  CB  SER A 131       8.859  -5.131  -0.968  1.00 97.47           C  \\nATOM   1942  HB2 SER A 131       8.553  -5.444  -1.966  1.00 97.47           H  \\nATOM   1943  HB3 SER A 131       8.100  -4.460  -0.566  1.00 97.47           H  \\nATOM   1944  O   SER A 131       7.211  -6.603   1.520  1.00 97.47           O  \\nATOM   1945  OG  SER A 131      10.088  -4.449  -1.035  1.00 97.47           O  \\nATOM   1946  HG  SER A 131      10.235  -4.093  -0.156  1.00 97.47           H  \\nATOM   1947  N   GLN A 132       6.836  -7.410  -0.554  1.00 95.22           N  \\nATOM   1948  H   GLN A 132       7.201  -7.519  -1.489  1.00 95.22           H  \\nATOM   1949  CA  GLN A 132       5.491  -7.913  -0.261  1.00 95.22           C  \\nATOM   1950  HA  GLN A 132       4.906  -7.076   0.120  1.00 95.22           H  \\nATOM   1951  C   GLN A 132       5.491  -8.974   0.848  1.00 95.22           C  \\nATOM   1952  CB  GLN A 132       4.852  -8.488  -1.546  1.00 95.22           C  \\nATOM   1953  HB2 GLN A 132       5.628  -8.908  -2.186  1.00 95.22           H  \\nATOM   1954  HB3 GLN A 132       4.176  -9.301  -1.281  1.00 95.22           H  \\nATOM   1955  O   GLN A 132       4.715  -8.909   1.801  1.00 95.22           O  \\nATOM   1956  CG  GLN A 132       4.044  -7.455  -2.345  1.00 95.22           C  \\nATOM   1957  HG2 GLN A 132       4.671  -6.578  -2.505  1.00 95.22           H  \\nATOM   1958  HG3 GLN A 132       3.765  -7.880  -3.309  1.00 95.22           H  \\nATOM   1959  CD  GLN A 132       2.779  -7.020  -1.609  1.00 95.22           C  \\nATOM   1960  NE2 GLN A 132       1.726  -6.600  -2.251  1.00 95.22           N  \\nATOM   1961 HE21 GLN A 132       1.180  -6.027  -1.623  1.00 95.22           H  \\nATOM   1962 HE22 GLN A 132       1.705  -6.399  -3.241  1.00 95.22           H  \\nATOM   1963  OE1 GLN A 132       2.672  -7.138  -0.411  1.00 95.22           O  \\nATOM   1964  N   ASN A 133       6.415  -9.934   0.774  1.00 96.35           N  \\nATOM   1965  H   ASN A 133       7.009  -9.968  -0.042  1.00 96.35           H  \\nATOM   1966  CA  ASN A 133       6.554 -10.963   1.805  1.00 96.35           C  \\nATOM   1967  HA  ASN A 133       5.602 -11.481   1.923  1.00 96.35           H  \\nATOM   1968  C   ASN A 133       6.906 -10.349   3.170  1.00 96.35           C  \\nATOM   1969  CB  ASN A 133       7.627 -11.968   1.360  1.00 96.35           C  \\nATOM   1970  HB2 ASN A 133       7.848 -12.640   2.189  1.00 96.35           H  \\nATOM   1971  HB3 ASN A 133       8.541 -11.432   1.103  1.00 96.35           H  \\nATOM   1972  O   ASN A 133       6.361 -10.754   4.198  1.00 96.35           O  \\nATOM   1973  CG  ASN A 133       7.206 -12.826   0.181  1.00 96.35           C  \\nATOM   1974  ND2 ASN A 133       8.144 -13.503  -0.437  1.00 96.35           N  \\nATOM   1975 HD21 ASN A 133       9.104 -13.444  -0.127  1.00 96.35           H  \\nATOM   1976 HD22 ASN A 133       7.830 -14.095  -1.192  1.00 96.35           H  \\nATOM   1977  OD1 ASN A 133       6.049 -12.944  -0.184  1.00 96.35           O  \\nATOM   1978  N   SER A 134       7.795  -9.351   3.177  1.00 97.41           N  \\nATOM   1979  H   SER A 134       8.206  -9.071   2.298  1.00 97.41           H  \\nATOM   1980  CA  SER A 134       8.192  -8.635   4.387  1.00 97.41           C  \\nATOM   1981  HA  SER A 134       8.515  -9.369   5.126  1.00 97.41           H  \\nATOM   1982  C   SER A 134       7.022  -7.855   4.999  1.00 97.41           C  \\nATOM   1983  CB  SER A 134       9.382  -7.725   4.066  1.00 97.41           C  \\nATOM   1984  HB2 SER A 134       9.081  -6.964   3.346  1.00 97.41           H  \\nATOM   1985  HB3 SER A 134      10.178  -8.324   3.623  1.00 97.41           H  \\nATOM   1986  O   SER A 134       6.790  -7.979   6.201  1.00 97.41           O  \\nATOM   1987  OG  SER A 134       9.882  -7.096   5.230  1.00 97.41           O  \\nATOM   1988  HG  SER A 134       9.956  -7.744   5.935  1.00 97.41           H  \\nATOM   1989  N   GLN A 135       6.241  -7.124   4.192  1.00 97.82           N  \\nATOM   1990  H   GLN A 135       6.465  -7.074   3.209  1.00 97.82           H  \\nATOM   1991  CA  GLN A 135       5.044  -6.404   4.651  1.00 97.82           C  \\nATOM   1992  HA  GLN A 135       5.330  -5.695   5.428  1.00 97.82           H  \\nATOM   1993  C   GLN A 135       4.015  -7.371   5.249  1.00 97.82           C  \\nATOM   1994  CB  GLN A 135       4.433  -5.623   3.476  1.00 97.82           C  \\nATOM   1995  HB2 GLN A 135       5.231  -5.085   2.963  1.00 97.82           H  \\nATOM   1996  HB3 GLN A 135       4.028  -6.346   2.767  1.00 97.82           H  \\nATOM   1997  O   GLN A 135       3.521  -7.133   6.350  1.00 97.82           O  \\nATOM   1998  CG  GLN A 135       3.347  -4.596   3.887  1.00 97.82           C  \\nATOM   1999  HG2 GLN A 135       3.827  -3.645   4.118  1.00 97.82           H  \\nATOM   2000  HG3 GLN A 135       2.823  -4.935   4.781  1.00 97.82           H  \\nATOM   2001  CD  GLN A 135       2.293  -4.383   2.798  1.00 97.82           C  \\nATOM   2002  NE2 GLN A 135       1.799  -3.190   2.546  1.00 97.82           N  \\nATOM   2003 HE21 GLN A 135       1.203  -3.127   1.733  1.00 97.82           H  \\nATOM   2004 HE22 GLN A 135       1.976  -2.332   3.049  1.00 97.82           H  \\nATOM   2005  OE1 GLN A 135       1.859  -5.346   2.194  1.00 97.82           O  \\nATOM   2006  N   GLY A 136       3.763  -8.509   4.592  1.00 96.60           N  \\nATOM   2007  H   GLY A 136       4.149  -8.617   3.665  1.00 96.60           H  \\nATOM   2008  CA  GLY A 136       2.874  -9.549   5.111  1.00 96.60           C  \\nATOM   2009  HA2 GLY A 136       2.854 -10.375   4.399  1.00 96.60           H  \\nATOM   2010  HA3 GLY A 136       1.867  -9.143   5.208  1.00 96.60           H  \\nATOM   2011  C   GLY A 136       3.317 -10.095   6.474  1.00 96.60           C  \\nATOM   2012  O   GLY A 136       2.487 -10.271   7.367  1.00 96.60           O  \\nATOM   2013  N   GLY A 137       4.622 -10.313   6.669  1.00 97.41           N  \\nATOM   2014  H   GLY A 137       5.251 -10.159   5.894  1.00 97.41           H  \\nATOM   2015  CA  GLY A 137       5.187 -10.713   7.961  1.00 97.41           C  \\nATOM   2016  HA2 GLY A 137       4.733 -11.648   8.286  1.00 97.41           H  \\nATOM   2017  HA3 GLY A 137       6.259 -10.866   7.842  1.00 97.41           H  \\nATOM   2018  C   GLY A 137       4.976  -9.659   9.053  1.00 97.41           C  \\nATOM   2019  O   GLY A 137       4.493  -9.986  10.139  1.00 97.41           O  \\nATOM   2020  N   VAL A 138       5.268  -8.391   8.749  1.00 98.10           N  \\nATOM   2021  H   VAL A 138       5.638  -8.191   7.831  1.00 98.10           H  \\nATOM   2022  CA  VAL A 138       5.087  -7.269   9.686  1.00 98.10           C  \\nATOM   2023  HA  VAL A 138       5.604  -7.507  10.616  1.00 98.10           H  \\nATOM   2024  C   VAL A 138       3.610  -7.075  10.043  1.00 98.10           C  \\nATOM   2025  CB  VAL A 138       5.707  -5.975   9.123  1.00 98.10           C  \\nATOM   2026  HB  VAL A 138       5.297  -5.779   8.132  1.00 98.10           H  \\nATOM   2027  O   VAL A 138       3.280  -6.976  11.223  1.00 98.10           O  \\nATOM   2028  CG1 VAL A 138       5.418  -4.771  10.024  1.00 98.10           C  \\nATOM   2029 HG11 VAL A 138       5.721  -4.983  11.050  1.00 98.10           H  \\nATOM   2030 HG12 VAL A 138       4.357  -4.526  10.006  1.00 98.10           H  \\nATOM   2031 HG13 VAL A 138       5.980  -3.904   9.677  1.00 98.10           H  \\nATOM   2032  CG2 VAL A 138       7.234  -6.089   9.015  1.00 98.10           C  \\nATOM   2033 HG21 VAL A 138       7.637  -5.184   8.561  1.00 98.10           H  \\nATOM   2034 HG22 VAL A 138       7.514  -6.940   8.393  1.00 98.10           H  \\nATOM   2035 HG23 VAL A 138       7.670  -6.213  10.007  1.00 98.10           H  \\nATOM   2036  N   LEU A 139       2.704  -7.097   9.060  1.00 97.39           N  \\nATOM   2037  H   LEU A 139       3.030  -7.173   8.108  1.00 97.39           H  \\nATOM   2038  CA  LEU A 139       1.260  -6.989   9.293  1.00 97.39           C  \\nATOM   2039  HA  LEU A 139       1.066  -6.082   9.865  1.00 97.39           H  \\nATOM   2040  C   LEU A 139       0.724  -8.150  10.134  1.00 97.39           C  \\nATOM   2041  CB  LEU A 139       0.507  -6.925   7.953  1.00 97.39           C  \\nATOM   2042  HB2 LEU A 139       0.926  -7.669   7.276  1.00 97.39           H  \\nATOM   2043  HB3 LEU A 139      -0.536  -7.191   8.125  1.00 97.39           H  \\nATOM   2044  O   LEU A 139      -0.077  -7.929  11.037  1.00 97.39           O  \\nATOM   2045  CG  LEU A 139       0.530  -5.547   7.270  1.00 97.39           C  \\nATOM   2046  HG  LEU A 139       1.563  -5.223   7.142  1.00 97.39           H  \\nATOM   2047  CD1 LEU A 139      -0.118  -5.657   5.892  1.00 97.39           C  \\nATOM   2048 HD11 LEU A 139      -0.048  -4.698   5.379  1.00 97.39           H  \\nATOM   2049 HD12 LEU A 139      -1.168  -5.937   5.981  1.00 97.39           H  \\nATOM   2050 HD13 LEU A 139       0.414  -6.391   5.288  1.00 97.39           H  \\nATOM   2051  CD2 LEU A 139      -0.240  -4.497   8.080  1.00 97.39           C  \\nATOM   2052 HD21 LEU A 139       0.282  -4.272   9.010  1.00 97.39           H  \\nATOM   2053 HD22 LEU A 139      -1.247  -4.855   8.299  1.00 97.39           H  \\nATOM   2054 HD23 LEU A 139      -0.312  -3.576   7.503  1.00 97.39           H  \\nATOM   2055  N   ASN A 140       1.180  -9.382   9.892  1.00 96.70           N  \\nATOM   2056  H   ASN A 140       1.818  -9.522   9.122  1.00 96.70           H  \\nATOM   2057  CA  ASN A 140       0.786 -10.527  10.715  1.00 96.70           C  \\nATOM   2058  HA  ASN A 140      -0.302 -10.579  10.746  1.00 96.70           H  \\nATOM   2059  C   ASN A 140       1.245 -10.371  12.172  1.00 96.70           C  \\nATOM   2060  CB  ASN A 140       1.330 -11.818  10.089  1.00 96.70           C  \\nATOM   2061  HB2 ASN A 140       2.368 -11.670   9.790  1.00 96.70           H  \\nATOM   2062  HB3 ASN A 140       1.306 -12.616  10.830  1.00 96.70           H  \\nATOM   2063  O   ASN A 140       0.459 -10.637  13.086  1.00 96.70           O  \\nATOM   2064  CG  ASN A 140       0.517 -12.292   8.898  1.00 96.70           C  \\nATOM   2065  ND2 ASN A 140       1.117 -13.079   8.038  1.00 96.70           N  \\nATOM   2066 HD21 ASN A 140       2.114 -13.224   8.113  1.00 96.70           H  \\nATOM   2067 HD22 ASN A 140       0.586 -13.355   7.224  1.00 96.70           H  \\nATOM   2068  OD1 ASN A 140      -0.672 -12.032   8.758  1.00 96.70           O  \\nATOM   2069  N   GLY A 141       2.475  -9.893  12.388  1.00 96.80           N  \\nATOM   2070  H   GLY A 141       3.070  -9.713  11.592  1.00 96.80           H  \\nATOM   2071  CA  GLY A 141       2.981  -9.540  13.715  1.00 96.80           C  \\nATOM   2072  HA2 GLY A 141       3.999  -9.164  13.618  1.00 96.80           H  \\nATOM   2073  HA3 GLY A 141       2.988 -10.423  14.355  1.00 96.80           H  \\nATOM   2074  C   GLY A 141       2.132  -8.455  14.378  1.00 96.80           C  \\nATOM   2075  O   GLY A 141       1.679  -8.640  15.508  1.00 96.80           O  \\nATOM   2076  N   PHE A 142       1.823  -7.381  13.643  1.00 97.09           N  \\nATOM   2077  H   PHE A 142       2.245  -7.290  12.730  1.00 97.09           H  \\nATOM   2078  CA  PHE A 142       0.941  -6.308  14.099  1.00 97.09           C  \\nATOM   2079  HA  PHE A 142       1.382  -5.849  14.984  1.00 97.09           H  \\nATOM   2080  C   PHE A 142      -0.433  -6.847  14.502  1.00 97.09           C  \\nATOM   2081  CB  PHE A 142       0.813  -5.226  13.013  1.00 97.09           C  \\nATOM   2082  HB2 PHE A 142       0.502  -5.685  12.075  1.00 97.09           H  \\nATOM   2083  HB3 PHE A 142       1.793  -4.780  12.844  1.00 97.09           H  \\nATOM   2084  O   PHE A 142      -0.897  -6.553  15.597  1.00 97.09           O  \\nATOM   2085  CG  PHE A 142      -0.179  -4.127  13.354  1.00 97.09           C  \\nATOM   2086  CD1 PHE A 142      -1.539  -4.268  13.014  1.00 97.09           C  \\nATOM   2087  HD1 PHE A 142      -1.867  -5.145  12.475  1.00 97.09           H  \\nATOM   2088  CD2 PHE A 142       0.253  -2.964  14.018  1.00 97.09           C  \\nATOM   2089  HD2 PHE A 142       1.295  -2.828  14.270  1.00 97.09           H  \\nATOM   2090  CE1 PHE A 142      -2.468  -3.271  13.364  1.00 97.09           C  \\nATOM   2091  HE1 PHE A 142      -3.510  -3.382  13.104  1.00 97.09           H  \\nATOM   2092  CE2 PHE A 142      -0.670  -1.958  14.347  1.00 97.09           C  \\nATOM   2093  HE2 PHE A 142      -0.325  -1.058  14.834  1.00 97.09           H  \\nATOM   2094  CZ  PHE A 142      -2.031  -2.113  14.033  1.00 97.09           C  \\nATOM   2095  HZ  PHE A 142      -2.731  -1.331  14.287  1.00 97.09           H  \\nATOM   2096  N   TYR A 143      -1.087  -7.662  13.672  1.00 96.69           N  \\nATOM   2097  H   TYR A 143      -0.686  -7.854  12.765  1.00 96.69           H  \\nATOM   2098  CA  TYR A 143      -2.410  -8.195  13.997  1.00 96.69           C  \\nATOM   2099  HA  TYR A 143      -3.074  -7.358  14.217  1.00 96.69           H  \\nATOM   2100  C   TYR A 143      -2.394  -9.081  15.243  1.00 96.69           C  \\nATOM   2101  CB  TYR A 143      -2.976  -8.998  12.821  1.00 96.69           C  \\nATOM   2102  HB2 TYR A 143      -3.937  -9.404  13.137  1.00 96.69           H  \\nATOM   2103  HB3 TYR A 143      -2.309  -9.837  12.626  1.00 96.69           H  \\nATOM   2104  O   TYR A 143      -3.296  -8.967  16.073  1.00 96.69           O  \\nATOM   2105  CG  TYR A 143      -3.208  -8.244  11.525  1.00 96.69           C  \\nATOM   2106  CD1 TYR A 143      -3.720  -6.931  11.531  1.00 96.69           C  \\nATOM   2107  HD1 TYR A 143      -3.939  -6.426  12.460  1.00 96.69           H  \\nATOM   2108  CD2 TYR A 143      -2.926  -8.877  10.299  1.00 96.69           C  \\nATOM   2109  HD2 TYR A 143      -2.495  -9.867  10.283  1.00 96.69           H  \\nATOM   2110  CE1 TYR A 143      -3.925  -6.245  10.321  1.00 96.69           C  \\nATOM   2111  HE1 TYR A 143      -4.291  -5.229  10.336  1.00 96.69           H  \\nATOM   2112  CE2 TYR A 143      -3.156  -8.204   9.086  1.00 96.69           C  \\nATOM   2113  HE2 TYR A 143      -2.908  -8.672   8.144  1.00 96.69           H  \\nATOM   2114  OH  TYR A 143      -3.857  -6.249   7.913  1.00 96.69           O  \\nATOM   2115  HH  TYR A 143      -4.028  -5.312   8.032  1.00 96.69           H  \\nATOM   2116  CZ  TYR A 143      -3.648  -6.884   9.094  1.00 96.69           C  \\nATOM   2117  N   SER A 144      -1.379  -9.940  15.381  1.00 96.30           N  \\nATOM   2118  H   SER A 144      -0.668  -9.961  14.664  1.00 96.30           H  \\nATOM   2119  CA  SER A 144      -1.225 -10.829  16.537  1.00 96.30           C  \\nATOM   2120  HA  SER A 144      -2.135 -11.416  16.661  1.00 96.30           H  \\nATOM   2121  C   SER A 144      -1.002 -10.031  17.820  1.00 96.30           C  \\nATOM   2122  CB  SER A 144      -0.062 -11.795  16.298  1.00 96.30           C  \\nATOM   2123  HB2 SER A 144      -0.212 -12.317  15.353  1.00 96.30           H  \\nATOM   2124  HB3 SER A 144       0.877 -11.244  16.252  1.00 96.30           H  \\nATOM   2125  O   SER A 144      -1.800 -10.124  18.757  1.00 96.30           O  \\nATOM   2126  OG  SER A 144      -0.017 -12.735  17.350  1.00 96.30           O  \\nATOM   2127  HG  SER A 144       0.760 -13.290  17.249  1.00 96.30           H  \\nATOM   2128  N   ALA A 145       0.005  -9.151  17.818  1.00 96.20           N  \\nATOM   2129  H   ALA A 145       0.592  -9.077  17.000  1.00 96.20           H  \\nATOM   2130  CA  ALA A 145       0.307  -8.271  18.941  1.00 96.20           C  \\nATOM   2131  HA  ALA A 145       0.571  -8.876  19.808  1.00 96.20           H  \\nATOM   2132  C   ALA A 145      -0.908  -7.415  19.306  1.00 96.20           C  \\nATOM   2133  CB  ALA A 145       1.503  -7.390  18.567  1.00 96.20           C  \\nATOM   2134  HB1 ALA A 145       2.368  -8.013  18.339  1.00 96.20           H  \\nATOM   2135  HB2 ALA A 145       1.755  -6.737  19.402  1.00 96.20           H  \\nATOM   2136  HB3 ALA A 145       1.266  -6.778  17.697  1.00 96.20           H  \\nATOM   2137  O   ALA A 145      -1.187  -7.207  20.488  1.00 96.20           O  \\nATOM   2138  N   ASN A 146      -1.681  -6.984  18.300  1.00 95.55           N  \\nATOM   2139  H   ASN A 146      -1.357  -7.147  17.358  1.00 95.55           H  \\nATOM   2140  CA  ASN A 146      -2.826  -6.108  18.475  1.00 95.55           C  \\nATOM   2141  HA  ASN A 146      -2.615  -5.570  19.399  1.00 95.55           H  \\nATOM   2142  C   ASN A 146      -4.168  -6.796  18.750  1.00 95.55           C  \\nATOM   2143  CB  ASN A 146      -2.828  -4.952  17.473  1.00 95.55           C  \\nATOM   2144  HB2 ASN A 146      -2.809  -5.315  16.446  1.00 95.55           H  \\nATOM   2145  HB3 ASN A 146      -3.727  -4.349  17.599  1.00 95.55           H  \\nATOM   2146  O   ASN A 146      -5.096  -6.104  19.151  1.00 95.55           O  \\nATOM   2147  CG  ASN A 146      -1.635  -4.053  17.755  1.00 95.55           C  \\nATOM   2148  ND2 ASN A 146      -0.662  -4.002  16.889  1.00 95.55           N  \\nATOM   2149 HD21 ASN A 146       0.088  -3.352  17.076  1.00 95.55           H  \\nATOM   2150 HD22 ASN A 146      -0.714  -4.543  16.038  1.00 95.55           H  \\nATOM   2151  OD1 ASN A 146      -1.536  -3.455  18.809  1.00 95.55           O  \\nATOM   2152  N   LYS A 147      -4.247  -8.134  18.714  1.00 94.61           N  \\nATOM   2153  H   LYS A 147      -3.421  -8.641  18.431  1.00 94.61           H  \\nATOM   2154  CA  LYS A 147      -5.510  -8.896  18.823  1.00 94.61           C  \\nATOM   2155  HA  LYS A 147      -5.275  -9.934  18.588  1.00 94.61           H  \\nATOM   2156  C   LYS A 147      -6.543  -8.474  17.765  1.00 94.61           C  \\nATOM   2157  CB  LYS A 147      -6.096  -8.869  20.252  1.00 94.61           C  \\nATOM   2158  HB2 LYS A 147      -6.429  -7.858  20.488  1.00 94.61           H  \\nATOM   2159  HB3 LYS A 147      -6.975  -9.513  20.267  1.00 94.61           H  \\nATOM   2160  O   LYS A 147      -7.740  -8.448  18.030  1.00 94.61           O  \\nATOM   2161  CG  LYS A 147      -5.138  -9.353  21.351  1.00 94.61           C  \\nATOM   2162  HG2 LYS A 147      -4.268  -8.699  21.399  1.00 94.61           H  \\nATOM   2163  HG3 LYS A 147      -4.809 -10.366  21.121  1.00 94.61           H  \\nATOM   2164  CD  LYS A 147      -5.855  -9.343  22.711  1.00 94.61           C  \\nATOM   2165  HD2 LYS A 147      -6.729  -9.993  22.652  1.00 94.61           H  \\nATOM   2166  HD3 LYS A 147      -6.188  -8.329  22.934  1.00 94.61           H  \\nATOM   2167  CE  LYS A 147      -4.925  -9.840  23.825  1.00 94.61           C  \\nATOM   2168  HE2 LYS A 147      -4.056  -9.185  23.880  1.00 94.61           H  \\nATOM   2169  HE3 LYS A 147      -4.568 -10.835  23.557  1.00 94.61           H  \\nATOM   2170  NZ  LYS A 147      -5.615  -9.888  25.142  1.00 94.61           N  \\nATOM   2171  HZ1 LYS A 147      -4.989 -10.221  25.861  1.00 94.61           H  \\nATOM   2172  HZ2 LYS A 147      -5.950  -8.975  25.412  1.00 94.61           H  \\nATOM   2173  HZ3 LYS A 147      -6.406 -10.516  25.107  1.00 94.61           H  \\nATOM   2174  N   VAL A 148      -6.079  -8.139  16.559  1.00 95.99           N  \\nATOM   2175  H   VAL A 148      -5.088  -8.244  16.394  1.00 95.99           H  \\nATOM   2176  CA  VAL A 148      -6.953  -7.768  15.437  1.00 95.99           C  \\nATOM   2177  HA  VAL A 148      -7.753  -7.133  15.816  1.00 95.99           H  \\nATOM   2178  C   VAL A 148      -7.585  -9.032  14.859  1.00 95.99           C  \\nATOM   2179  CB  VAL A 148      -6.199  -6.964  14.364  1.00 95.99           C  \\nATOM   2180  HB  VAL A 148      -5.368  -7.561  13.992  1.00 95.99           H  \\nATOM   2181  O   VAL A 148      -6.947  -9.778  14.104  1.00 95.99           O  \\nATOM   2182  CG1 VAL A 148      -7.106  -6.600  13.182  1.00 95.99           C  \\nATOM   2183 HG11 VAL A 148      -6.537  -6.033  12.445  1.00 95.99           H  \\nATOM   2184 HG12 VAL A 148      -7.946  -5.995  13.524  1.00 95.99           H  \\nATOM   2185 HG13 VAL A 148      -7.489  -7.500  12.701  1.00 95.99           H  \\nATOM   2186  CG2 VAL A 148      -5.644  -5.658  14.946  1.00 95.99           C  \\nATOM   2187 HG21 VAL A 148      -5.133  -5.086  14.171  1.00 95.99           H  \\nATOM   2188 HG22 VAL A 148      -6.455  -5.055  15.354  1.00 95.99           H  \\nATOM   2189 HG23 VAL A 148      -4.935  -5.878  15.743  1.00 95.99           H  \\nATOM   2190  N   ALA A 149      -8.846  -9.271  15.218  1.00 95.62           N  \\nATOM   2191  H   ALA A 149      -9.306  -8.639  15.858  1.00 95.62           H  \\nATOM   2192  CA  ALA A 149      -9.611 -10.409  14.743  1.00 95.62           C  \\nATOM   2193  HA  ALA A 149      -8.953 -11.279  14.748  1.00 95.62           H  \\nATOM   2194  C   ALA A 149     -10.109 -10.191  13.310  1.00 95.62           C  \\nATOM   2195  CB  ALA A 149     -10.763 -10.705  15.708  1.00 95.62           C  \\nATOM   2196  HB1 ALA A 149     -11.264 -11.628  15.415  1.00 95.62           H  \\nATOM   2197  HB2 ALA A 149     -10.383 -10.811  16.724  1.00 95.62           H  \\nATOM   2198  HB3 ALA A 149     -11.493  -9.896  15.681  1.00 95.62           H  \\nATOM   2199  O   ALA A 149     -10.575  -9.111  12.939  1.00 95.62           O  \\nATOM   2200  N   GLN A 150     -10.026 -11.258  12.518  1.00 95.95           N  \\nATOM   2201  H   GLN A 150      -9.695 -12.120  12.927  1.00 95.95           H  \\nATOM   2202  CA  GLN A 150     -10.679 -11.334  11.217  1.00 95.95           C  \\nATOM   2203  HA  GLN A 150     -10.581 -10.360  10.738  1.00 95.95           H  \\nATOM   2204  C   GLN A 150     -12.171 -11.644  11.378  1.00 95.95           C  \\nATOM   2205  CB  GLN A 150      -9.971 -12.366  10.324  1.00 95.95           C  \\nATOM   2206  HB2 GLN A 150     -10.391 -12.305   9.320  1.00 95.95           H  \\nATOM   2207  HB3 GLN A 150      -8.914 -12.106  10.264  1.00 95.95           H  \\nATOM   2208  O   GLN A 150     -12.558 -12.353  12.307  1.00 95.95           O  \\nATOM   2209  CG  GLN A 150     -10.115 -13.814  10.832  1.00 95.95           C  \\nATOM   2210  HG2 GLN A 150     -11.164 -14.106  10.804  1.00 95.95           H  \\nATOM   2211  HG3 GLN A 150      -9.770 -13.880  11.864  1.00 95.95           H  \\nATOM   2212  CD  GLN A 150      -9.326 -14.825  10.009  1.00 95.95           C  \\nATOM   2213  NE2 GLN A 150      -9.467 -16.101  10.294  1.00 95.95           N  \\nATOM   2214 HE21 GLN A 150      -8.899 -16.744   9.762  1.00 95.95           H  \\nATOM   2215 HE22 GLN A 150     -10.063 -16.399  11.053  1.00 95.95           H  \\nATOM   2216  OE1 GLN A 150      -8.575 -14.504   9.104  1.00 95.95           O  \\nATOM   2217  N   PHE A 151     -12.992 -11.205  10.428  1.00 96.13           N  \\nATOM   2218  H   PHE A 151     -12.633 -10.578   9.722  1.00 96.13           H  \\nATOM   2219  CA  PHE A 151     -14.394 -11.616  10.335  1.00 96.13           C  \\nATOM   2220  HA  PHE A 151     -14.534 -12.517  10.933  1.00 96.13           H  \\nATOM   2221  C   PHE A 151     -14.795 -11.984   8.909  1.00 96.13           C  \\nATOM   2222  CB  PHE A 151     -15.306 -10.541  10.931  1.00 96.13           C  \\nATOM   2223  HB2 PHE A 151     -16.266 -10.999  11.173  1.00 96.13           H  \\nATOM   2224  HB3 PHE A 151     -14.863 -10.232  11.878  1.00 96.13           H  \\nATOM   2225  O   PHE A 151     -14.073 -11.707   7.951  1.00 96.13           O  \\nATOM   2226  CG  PHE A 151     -15.565  -9.335  10.050  1.00 96.13           C  \\nATOM   2227  CD1 PHE A 151     -14.519  -8.453   9.737  1.00 96.13           C  \\nATOM   2228  HD1 PHE A 151     -13.556  -8.590  10.205  1.00 96.13           H  \\nATOM   2229  CD2 PHE A 151     -16.850  -9.100   9.527  1.00 96.13           C  \\nATOM   2230  HD2 PHE A 151     -17.670  -9.758   9.777  1.00 96.13           H  \\nATOM   2231  CE1 PHE A 151     -14.721  -7.393   8.841  1.00 96.13           C  \\nATOM   2232  HE1 PHE A 151     -13.906  -6.736   8.576  1.00 96.13           H  \\nATOM   2233  CE2 PHE A 151     -17.073  -7.996   8.688  1.00 96.13           C  \\nATOM   2234  HE2 PHE A 151     -18.065  -7.787   8.316  1.00 96.13           H  \\nATOM   2235  CZ  PHE A 151     -16.005  -7.161   8.328  1.00 96.13           C  \\nATOM   2236  HZ  PHE A 151     -16.181  -6.315   7.679  1.00 96.13           H  \\nATOM   2237  N   ASP A 152     -15.941 -12.645   8.780  1.00 96.66           N  \\nATOM   2238  H   ASP A 152     -16.498 -12.811   9.606  1.00 96.66           H  \\nATOM   2239  CA  ASP A 152     -16.601 -12.923   7.514  1.00 96.66           C  \\nATOM   2240  HA  ASP A 152     -15.875 -12.881   6.702  1.00 96.66           H  \\nATOM   2241  C   ASP A 152     -17.703 -11.876   7.278  1.00 96.66           C  \\nATOM   2242  CB  ASP A 152     -17.155 -14.355   7.541  1.00 96.66           C  \\nATOM   2243  HB2 ASP A 152     -17.942 -14.434   8.292  1.00 96.66           H  \\nATOM   2244  HB3 ASP A 152     -16.352 -15.032   7.832  1.00 96.66           H  \\nATOM   2245  O   ASP A 152     -18.758 -11.941   7.918  1.00 96.66           O  \\nATOM   2246  CG  ASP A 152     -17.699 -14.808   6.183  1.00 96.66           C  \\nATOM   2247  OD1 ASP A 152     -17.927 -13.955   5.293  1.00 96.66           O  \\nATOM   2248  OD2 ASP A 152     -17.825 -16.036   5.989  1.00 96.66           O  \\nATOM   2249  N   PRO A 153     -17.513 -10.921   6.349  1.00 95.35           N  \\nATOM   2250  CA  PRO A 153     -18.533  -9.925   6.034  1.00 95.35           C  \\nATOM   2251  HA  PRO A 153     -18.766  -9.365   6.940  1.00 95.35           H  \\nATOM   2252  C   PRO A 153     -19.835 -10.523   5.495  1.00 95.35           C  \\nATOM   2253  CB  PRO A 153     -17.900  -8.987   5.003  1.00 95.35           C  \\nATOM   2254  HB2 PRO A 153     -18.162  -9.304   3.993  1.00 95.35           H  \\nATOM   2255  HB3 PRO A 153     -18.199  -7.951   5.164  1.00 95.35           H  \\nATOM   2256  O   PRO A 153     -20.872  -9.874   5.575  1.00 95.35           O  \\nATOM   2257  CG  PRO A 153     -16.402  -9.178   5.208  1.00 95.35           C  \\nATOM   2258  HG2 PRO A 153     -16.059  -8.559   6.037  1.00 95.35           H  \\nATOM   2259  HG3 PRO A 153     -15.844  -8.950   4.300  1.00 95.35           H  \\nATOM   2260  CD  PRO A 153     -16.301 -10.647   5.595  1.00 95.35           C  \\nATOM   2261  HD2 PRO A 153     -15.395 -10.804   6.179  1.00 95.35           H  \\nATOM   2262  HD3 PRO A 153     -16.298 -11.269   4.700  1.00 95.35           H  \\nATOM   2263  N   SER A 154     -19.814 -11.758   4.976  1.00 95.31           N  \\nATOM   2264  H   SER A 154     -18.948 -12.277   4.987  1.00 95.31           H  \\nATOM   2265  CA  SER A 154     -21.033 -12.459   4.553  1.00 95.31           C  \\nATOM   2266  HA  SER A 154     -21.691 -11.730   4.080  1.00 95.31           H  \\nATOM   2267  C   SER A 154     -21.829 -13.050   5.724  1.00 95.31           C  \\nATOM   2268  CB  SER A 154     -20.713 -13.518   3.491  1.00 95.31           C  \\nATOM   2269  HB2 SER A 154     -20.092 -13.067   2.718  1.00 95.31           H  \\nATOM   2270  HB3 SER A 154     -21.650 -13.831   3.029  1.00 95.31           H  \\nATOM   2271  O   SER A 154     -22.958 -13.502   5.537  1.00 95.31           O  \\nATOM   2272  OG  SER A 154     -20.063 -14.669   4.001  1.00 95.31           O  \\nATOM   2273  HG  SER A 154     -19.257 -14.411   4.454  1.00 95.31           H  \\nATOM   2274  N   LYS A 155     -21.263 -13.040   6.940  1.00 93.44           N  \\nATOM   2275  H   LYS A 155     -20.341 -12.635   7.019  1.00 93.44           H  \\nATOM   2276  CA  LYS A 155     -21.862 -13.584   8.167  1.00 93.44           C  \\nATOM   2277  HA  LYS A 155     -22.937 -13.692   8.021  1.00 93.44           H  \\nATOM   2278  C   LYS A 155     -21.660 -12.637   9.366  1.00 93.44           C  \\nATOM   2279  CB  LYS A 155     -21.275 -14.975   8.439  1.00 93.44           C  \\nATOM   2280  HB2 LYS A 155     -20.203 -14.877   8.611  1.00 93.44           H  \\nATOM   2281  HB3 LYS A 155     -21.736 -15.371   9.344  1.00 93.44           H  \\nATOM   2282  O   LYS A 155     -21.136 -13.058  10.401  1.00 93.44           O  \\nATOM   2283  CG  LYS A 155     -21.518 -15.986   7.314  1.00 93.44           C  \\nATOM   2284  HG2 LYS A 155     -21.125 -15.617   6.366  1.00 93.44           H  \\nATOM   2285  HG3 LYS A 155     -22.588 -16.171   7.218  1.00 93.44           H  \\nATOM   2286  CD  LYS A 155     -20.788 -17.280   7.659  1.00 93.44           C  \\nATOM   2287  HD2 LYS A 155     -21.085 -17.604   8.656  1.00 93.44           H  \\nATOM   2288  HD3 LYS A 155     -19.714 -17.094   7.660  1.00 93.44           H  \\nATOM   2289  CE  LYS A 155     -21.119 -18.366   6.643  1.00 93.44           C  \\nATOM   2290  HE2 LYS A 155     -20.697 -18.077   5.681  1.00 93.44           H  \\nATOM   2291  HE3 LYS A 155     -22.202 -18.429   6.543  1.00 93.44           H  \\nATOM   2292  NZ  LYS A 155     -20.566 -19.659   7.104  1.00 93.44           N  \\nATOM   2293  HZ1 LYS A 155     -19.562 -19.587   7.182  1.00 93.44           H  \\nATOM   2294  HZ2 LYS A 155     -20.933 -19.871   8.022  1.00 93.44           H  \\nATOM   2295  HZ3 LYS A 155     -20.808 -20.407   6.470  1.00 93.44           H  \\nATOM   2296  N   PRO A 156     -22.110 -11.370   9.287  1.00 87.07           N  \\nATOM   2297  CA  PRO A 156     -21.802 -10.348  10.295  1.00 87.07           C  \\nATOM   2298  HA  PRO A 156     -20.722 -10.229  10.376  1.00 87.07           H  \\nATOM   2299  C   PRO A 156     -22.344 -10.675  11.695  1.00 87.07           C  \\nATOM   2300  CB  PRO A 156     -22.409  -9.053   9.741  1.00 87.07           C  \\nATOM   2301  HB2 PRO A 156     -22.768  -8.393  10.531  1.00 87.07           H  \\nATOM   2302  HB3 PRO A 156     -21.664  -8.539   9.133  1.00 87.07           H  \\nATOM   2303  O   PRO A 156     -21.780 -10.233  12.687  1.00 87.07           O  \\nATOM   2304  CG  PRO A 156     -23.548  -9.535   8.843  1.00 87.07           C  \\nATOM   2305  HG2 PRO A 156     -24.428  -9.757   9.447  1.00 87.07           H  \\nATOM   2306  HG3 PRO A 156     -23.786  -8.809   8.066  1.00 87.07           H  \\nATOM   2307  CD  PRO A 156     -22.984 -10.825   8.256  1.00 87.07           C  \\nATOM   2308  HD2 PRO A 156     -22.398 -10.590   7.367  1.00 87.07           H  \\nATOM   2309  HD3 PRO A 156     -23.792 -11.510   8.001  1.00 87.07           H  \\nATOM   2310  N   GLN A 157     -23.402 -11.487  11.790  1.00 88.01           N  \\nATOM   2311  H   GLN A 157     -23.835 -11.808  10.936  1.00 88.01           H  \\nATOM   2312  CA  GLN A 157     -23.974 -11.930  13.070  1.00 88.01           C  \\nATOM   2313  HA  GLN A 157     -23.919 -11.108  13.784  1.00 88.01           H  \\nATOM   2314  C   GLN A 157     -23.211 -13.102  13.716  1.00 88.01           C  \\nATOM   2315  CB  GLN A 157     -25.455 -12.302  12.864  1.00 88.01           C  \\nATOM   2316  HB2 GLN A 157     -25.874 -12.623  13.818  1.00 88.01           H  \\nATOM   2317  HB3 GLN A 157     -25.518 -13.143  12.174  1.00 88.01           H  \\nATOM   2318  O   GLN A 157     -23.451 -13.422  14.873  1.00 88.01           O  \\nATOM   2319  CG  GLN A 157     -26.318 -11.150  12.317  1.00 88.01           C  \\nATOM   2320  HG2 GLN A 157     -27.348 -11.495  12.224  1.00 88.01           H  \\nATOM   2321  HG3 GLN A 157     -25.972 -10.876  11.320  1.00 88.01           H  \\nATOM   2322  CD  GLN A 157     -26.314  -9.909  13.207  1.00 88.01           C  \\nATOM   2323  NE2 GLN A 157     -26.559  -8.739  12.661  1.00 88.01           N  \\nATOM   2324 HE21 GLN A 157     -26.800  -8.646  11.684  1.00 88.01           H  \\nATOM   2325 HE22 GLN A 157     -26.559  -7.960  13.305  1.00 88.01           H  \\nATOM   2326  OE1 GLN A 157     -26.093  -9.957  14.401  1.00 88.01           O  \\nATOM   2327  N   GLN A 158     -22.318 -13.770  12.977  1.00 88.94           N  \\nATOM   2328  H   GLN A 158     -22.111 -13.428  12.050  1.00 88.94           H  \\nATOM   2329  CA  GLN A 158     -21.595 -14.965  13.444  1.00 88.94           C  \\nATOM   2330  HA  GLN A 158     -22.045 -15.333  14.366  1.00 88.94           H  \\nATOM   2331  C   GLN A 158     -20.124 -14.675  13.754  1.00 88.94           C  \\nATOM   2332  CB  GLN A 158     -21.689 -16.082  12.392  1.00 88.94           C  \\nATOM   2333  HB2 GLN A 158     -21.193 -15.751  11.480  1.00 88.94           H  \\nATOM   2334  HB3 GLN A 158     -21.159 -16.958  12.767  1.00 88.94           H  \\nATOM   2335  O   GLN A 158     -19.499 -15.399  14.524  1.00 88.94           O  \\nATOM   2336  CG  GLN A 158     -23.137 -16.485  12.075  1.00 88.94           C  \\nATOM   2337  HG2 GLN A 158     -23.613 -16.858  12.982  1.00 88.94           H  \\nATOM   2338  HG3 GLN A 158     -23.693 -15.610  11.738  1.00 88.94           H  \\nATOM   2339  CD  GLN A 158     -23.254 -17.553  10.991  1.00 88.94           C  \\nATOM   2340  NE2 GLN A 158     -24.425 -17.706  10.412  1.00 88.94           N  \\nATOM   2341 HE21 GLN A 158     -25.222 -17.178  10.736  1.00 88.94           H  \\nATOM   2342 HE22 GLN A 158     -24.516 -18.471   9.758  1.00 88.94           H  \\nATOM   2343  OE1 GLN A 158     -22.326 -18.257  10.621  1.00 88.94           O  \\nATOM   2344  N   THR A 159     -19.556 -13.630  13.150  1.00 88.51           N  \\nATOM   2345  H   THR A 159     -20.112 -13.087  12.505  1.00 88.51           H  \\nATOM   2346  CA  THR A 159     -18.152 -13.252  13.335  1.00 88.51           C  \\nATOM   2347  HA  THR A 159     -17.742 -13.775  14.199  1.00 88.51           H  \\nATOM   2348  C   THR A 159     -18.041 -11.756  13.597  1.00 88.51           C  \\nATOM   2349  CB  THR A 159     -17.301 -13.637  12.116  1.00 88.51           C  \\nATOM   2350  HB  THR A 159     -16.282 -13.290  12.286  1.00 88.51           H  \\nATOM   2351  O   THR A 159     -18.427 -10.959  12.742  1.00 88.51           O  \\nATOM   2352  CG2 THR A 159     -17.262 -15.142  11.854  1.00 88.51           C  \\nATOM   2353 HG21 THR A 159     -18.254 -15.503  11.581  1.00 88.51           H  \\nATOM   2354 HG22 THR A 159     -16.924 -15.662  12.750  1.00 88.51           H  \\nATOM   2355 HG23 THR A 159     -16.571 -15.349  11.036  1.00 88.51           H  \\nATOM   2356  OG1 THR A 159     -17.797 -13.042  10.941  1.00 88.51           O  \\nATOM   2357  HG1 THR A 159     -18.293 -12.262  11.198  1.00 88.51           H  \\nATOM   2358  N   LYS A 160     -17.475 -11.372  14.747  1.00 88.42           N  \\nATOM   2359  H   LYS A 160     -17.173 -12.082  15.398  1.00 88.42           H  \\nATOM   2360  CA  LYS A 160     -17.197  -9.971  15.087  1.00 88.42           C  \\nATOM   2361  HA  LYS A 160     -17.903  -9.336  14.552  1.00 88.42           H  \\nATOM   2362  C   LYS A 160     -15.783  -9.605  14.638  1.00 88.42           C  \\nATOM   2363  CB  LYS A 160     -17.405  -9.741  16.595  1.00 88.42           C  \\nATOM   2364  HB2 LYS A 160     -18.358 -10.179  16.892  1.00 88.42           H  \\nATOM   2365  HB3 LYS A 160     -16.610 -10.240  17.150  1.00 88.42           H  \\nATOM   2366  O   LYS A 160     -14.815 -10.202  15.100  1.00 88.42           O  \\nATOM   2367  CG  LYS A 160     -17.427  -8.244  16.957  1.00 88.42           C  \\nATOM   2368  HG2 LYS A 160     -16.523  -7.759  16.587  1.00 88.42           H  \\nATOM   2369  HG3 LYS A 160     -18.291  -7.781  16.481  1.00 88.42           H  \\nATOM   2370  CD  LYS A 160     -17.510  -8.031  18.479  1.00 88.42           C  \\nATOM   2371  HD2 LYS A 160     -18.321  -8.634  18.887  1.00 88.42           H  \\nATOM   2372  HD3 LYS A 160     -16.569  -8.355  18.924  1.00 88.42           H  \\nATOM   2373  CE  LYS A 160     -17.759  -6.555  18.829  1.00 88.42           C  \\nATOM   2374  HE2 LYS A 160     -18.766  -6.286  18.509  1.00 88.42           H  \\nATOM   2375  HE3 LYS A 160     -17.057  -5.932  18.275  1.00 88.42           H  \\nATOM   2376  NZ  LYS A 160     -17.596  -6.281  20.281  1.00 88.42           N  \\nATOM   2377  HZ1 LYS A 160     -16.620  -6.126  20.489  1.00 88.42           H  \\nATOM   2378  HZ2 LYS A 160     -17.964  -6.998  20.890  1.00 88.42           H  \\nATOM   2379  HZ3 LYS A 160     -17.966  -5.382  20.555  1.00 88.42           H  \\nATOM   2380  N   GLY A 161     -15.672  -8.639  13.732  1.00 90.51           N  \\nATOM   2381  H   GLY A 161     -16.513  -8.195  13.392  1.00 90.51           H  \\nATOM   2382  CA  GLY A 161     -14.381  -8.128  13.271  1.00 90.51           C  \\nATOM   2383  HA2 GLY A 161     -13.649  -8.934  13.234  1.00 90.51           H  \\nATOM   2384  HA3 GLY A 161     -14.489  -7.713  12.269  1.00 90.51           H  \\nATOM   2385  C   GLY A 161     -13.847  -7.032  14.174  1.00 90.51           C  \\nATOM   2386  O   GLY A 161     -14.626  -6.275  14.748  1.00 90.51           O  \\nATOM   2387  N   THR A 162     -12.526  -6.900  14.232  1.00 95.20           N  \\nATOM   2388  H   THR A 162     -11.940  -7.575  13.762  1.00 95.20           H  \\nATOM   2389  CA  THR A 162     -11.868  -5.819  14.971  1.00 95.20           C  \\nATOM   2390  HA  THR A 162     -12.540  -5.440  15.740  1.00 95.20           H  \\nATOM   2391  C   THR A 162     -11.511  -4.674  14.035  1.00 95.20           C  \\nATOM   2392  CB  THR A 162     -10.610  -6.333  15.671  1.00 95.20           C  \\nATOM   2393  HB  THR A 162      -9.879  -6.629  14.919  1.00 95.20           H  \\nATOM   2394  O   THR A 162     -10.818  -4.872  13.031  1.00 95.20           O  \\nATOM   2395  CG2 THR A 162      -9.981  -5.313  16.605  1.00 95.20           C  \\nATOM   2396 HG21 THR A 162      -9.572  -4.478  16.035  1.00 95.20           H  \\nATOM   2397 HG22 THR A 162      -9.172  -5.794  17.154  1.00 95.20           H  \\nATOM   2398 HG23 THR A 162     -10.723  -4.945  17.314  1.00 95.20           H  \\nATOM   2399  OG1 THR A 162     -10.908  -7.452  16.469  1.00 95.20           O  \\nATOM   2400  HG1 THR A 162     -11.419  -7.142  17.220  1.00 95.20           H  \\nATOM   2401  N   TRP A 163     -11.966  -3.473  14.379  1.00 97.62           N  \\nATOM   2402  H   TRP A 163     -12.535  -3.393  15.209  1.00 97.62           H  \\nATOM   2403  CA  TRP A 163     -11.527  -2.233  13.754  1.00 97.62           C  \\nATOM   2404  HA  TRP A 163     -11.451  -2.390  12.678  1.00 97.62           H  \\nATOM   2405  C   TRP A 163     -10.155  -1.830  14.287  1.00 97.62           C  \\nATOM   2406  CB  TRP A 163     -12.563  -1.137  13.996  1.00 97.62           C  \\nATOM   2407  HB2 TRP A 163     -12.743  -1.036  15.066  1.00 97.62           H  \\nATOM   2408  HB3 TRP A 163     -12.160  -0.190  13.638  1.00 97.62           H  \\nATOM   2409  O   TRP A 163      -9.899  -1.913  15.488  1.00 97.62           O  \\nATOM   2410  CG  TRP A 163     -13.861  -1.370  13.296  1.00 97.62           C  \\nATOM   2411  CD1 TRP A 163     -14.941  -2.001  13.809  1.00 97.62           C  \\nATOM   2412  HD1 TRP A 163     -14.994  -2.419  14.804  1.00 97.62           H  \\nATOM   2413  CD2 TRP A 163     -14.227  -0.977  11.943  1.00 97.62           C  \\nATOM   2414  CE2 TRP A 163     -15.573  -1.379  11.704  1.00 97.62           C  \\nATOM   2415  CE3 TRP A 163     -13.552  -0.315  10.898  1.00 97.62           C  \\nATOM   2416  HE3 TRP A 163     -12.517  -0.042  11.041  1.00 97.62           H  \\nATOM   2417  NE1 TRP A 163     -15.957  -2.002  12.874  1.00 97.62           N  \\nATOM   2418  HE1 TRP A 163     -16.862  -2.419  13.043  1.00 97.62           H  \\nATOM   2419  CH2 TRP A 163     -15.541  -0.407   9.486  1.00 97.62           C  \\nATOM   2420  HH2 TRP A 163     -16.046  -0.162   8.564  1.00 97.62           H  \\nATOM   2421  CZ2 TRP A 163     -16.225  -1.114  10.491  1.00 97.62           C  \\nATOM   2422  HZ2 TRP A 163     -17.246  -1.431  10.339  1.00 97.62           H  \\nATOM   2423  CZ3 TRP A 163     -14.207  -0.006   9.692  1.00 97.62           C  \\nATOM   2424  HZ3 TRP A 163     -13.683   0.557   8.934  1.00 97.62           H  \\nATOM   2425  N   PHE A 164      -9.269  -1.381  13.406  1.00 97.99           N  \\nATOM   2426  H   PHE A 164      -9.528  -1.339  12.430  1.00 97.99           H  \\nATOM   2427  CA  PHE A 164      -7.923  -0.949  13.763  1.00 97.99           C  \\nATOM   2428  HA  PHE A 164      -7.936  -0.626  14.804  1.00 97.99           H  \\nATOM   2429  C   PHE A 164      -7.470   0.248  12.925  1.00 97.99           C  \\nATOM   2430  CB  PHE A 164      -6.938  -2.124  13.654  1.00 97.99           C  \\nATOM   2431  HB2 PHE A 164      -6.001  -1.830  14.127  1.00 97.99           H  \\nATOM   2432  HB3 PHE A 164      -7.329  -2.966  14.224  1.00 97.99           H  \\nATOM   2433  O   PHE A 164      -7.944   0.473  11.813  1.00 97.99           O  \\nATOM   2434  CG  PHE A 164      -6.643  -2.579  12.235  1.00 97.99           C  \\nATOM   2435  CD1 PHE A 164      -7.496  -3.497  11.593  1.00 97.99           C  \\nATOM   2436  HD1 PHE A 164      -8.371  -3.867  12.107  1.00 97.99           H  \\nATOM   2437  CD2 PHE A 164      -5.525  -2.069  11.547  1.00 97.99           C  \\nATOM   2438  HD2 PHE A 164      -4.876  -1.351  12.026  1.00 97.99           H  \\nATOM   2439  CE1 PHE A 164      -7.227  -3.910  10.276  1.00 97.99           C  \\nATOM   2440  HE1 PHE A 164      -7.895  -4.598   9.779  1.00 97.99           H  \\nATOM   2441  CE2 PHE A 164      -5.258  -2.478  10.228  1.00 97.99           C  \\nATOM   2442  HE2 PHE A 164      -4.407  -2.070   9.703  1.00 97.99           H  \\nATOM   2443  CZ  PHE A 164      -6.109  -3.400   9.592  1.00 97.99           C  \\nATOM   2444  HZ  PHE A 164      -5.920  -3.695   8.571  1.00 97.99           H  \\nATOM   2445  N   GLN A 165      -6.520   0.996  13.468  1.00 97.82           N  \\nATOM   2446  H   GLN A 165      -6.220   0.769  14.405  1.00 97.82           H  \\nATOM   2447  CA  GLN A 165      -5.852   2.118  12.833  1.00 97.82           C  \\nATOM   2448  HA  GLN A 165      -5.989   2.071  11.753  1.00 97.82           H  \\nATOM   2449  C   GLN A 165      -4.360   2.032  13.154  1.00 97.82           C  \\nATOM   2450  CB  GLN A 165      -6.471   3.422  13.354  1.00 97.82           C  \\nATOM   2451  HB2 GLN A 165      -7.527   3.441  13.085  1.00 97.82           H  \\nATOM   2452  HB3 GLN A 165      -6.393   3.446  14.441  1.00 97.82           H  \\nATOM   2453  O   GLN A 165      -3.974   1.978  14.323  1.00 97.82           O  \\nATOM   2454  CG  GLN A 165      -5.779   4.665  12.775  1.00 97.82           C  \\nATOM   2455  HG2 GLN A 165      -5.839   4.643  11.687  1.00 97.82           H  \\nATOM   2456  HG3 GLN A 165      -4.725   4.661  13.055  1.00 97.82           H  \\nATOM   2457  CD  GLN A 165      -6.372   5.980  13.264  1.00 97.82           C  \\nATOM   2458  NE2 GLN A 165      -5.875   7.088  12.760  1.00 97.82           N  \\nATOM   2459 HE21 GLN A 165      -6.272   7.954  13.096  1.00 97.82           H  \\nATOM   2460 HE22 GLN A 165      -5.124   7.039  12.086  1.00 97.82           H  \\nATOM   2461  OE1 GLN A 165      -7.247   6.058  14.114  1.00 97.82           O  \\nATOM   2462  N   ILE A 166      -3.525   2.062  12.116  1.00 96.90           N  \\nATOM   2463  H   ILE A 166      -3.906   2.161  11.186  1.00 96.90           H  \\nATOM   2464  CA  ILE A 166      -2.087   2.270  12.294  1.00 96.90           C  \\nATOM   2465  HA  ILE A 166      -1.768   1.750  13.198  1.00 96.90           H  \\nATOM   2466  C   ILE A 166      -1.873   3.771  12.487  1.00 96.90           C  \\nATOM   2467  CB  ILE A 166      -1.264   1.685  11.129  1.00 96.90           C  \\nATOM   2468  HB  ILE A 166      -1.590   2.152  10.200  1.00 96.90           H  \\nATOM   2469  O   ILE A 166      -2.141   4.558  11.578  1.00 96.90           O  \\nATOM   2470  CG1 ILE A 166      -1.465   0.154  11.027  1.00 96.90           C  \\nATOM   2471 HG12 ILE A 166      -2.513  -0.103  11.178  1.00 96.90           H  \\nATOM   2472 HG13 ILE A 166      -0.888  -0.340  11.809  1.00 96.90           H  \\nATOM   2473  CG2 ILE A 166       0.226   2.006  11.357  1.00 96.90           C  \\nATOM   2474 HG21 ILE A 166       0.400   3.081  11.303  1.00 96.90           H  \\nATOM   2475 HG22 ILE A 166       0.548   1.644  12.333  1.00 96.90           H  \\nATOM   2476 HG23 ILE A 166       0.842   1.542  10.587  1.00 96.90           H  \\nATOM   2477  CD1 ILE A 166      -1.057  -0.415   9.664  1.00 96.90           C  \\nATOM   2478 HD11 ILE A 166      -1.160  -1.500   9.677  1.00 96.90           H  \\nATOM   2479 HD12 ILE A 166      -1.703  -0.006   8.887  1.00 96.90           H  \\nATOM   2480 HD13 ILE A 166      -0.023  -0.159   9.431  1.00 96.90           H  \\nATOM   2481  N   THR A 167      -1.456   4.180  13.683  1.00 93.30           N  \\nATOM   2482  H   THR A 167      -1.273   3.492  14.400  1.00 93.30           H  \\nATOM   2483  CA  THR A 167      -1.315   5.600  14.048  1.00 93.30           C  \\nATOM   2484  HA  THR A 167      -2.115   6.170  13.575  1.00 93.30           H  \\nATOM   2485  C   THR A 167      -0.017   6.202  13.546  1.00 93.30           C  \\nATOM   2486  CB  THR A 167      -1.412   5.797  15.567  1.00 93.30           C  \\nATOM   2487  HB  THR A 167      -0.971   6.750  15.859  1.00 93.30           H  \\nATOM   2488  O   THR A 167      -0.003   7.338  13.070  1.00 93.30           O  \\nATOM   2489  CG2 THR A 167      -2.884   5.782  15.979  1.00 93.30           C  \\nATOM   2490 HG21 THR A 167      -3.428   6.564  15.451  1.00 93.30           H  \\nATOM   2491 HG22 THR A 167      -3.327   4.812  15.752  1.00 93.30           H  \\nATOM   2492 HG23 THR A 167      -2.947   5.973  17.050  1.00 93.30           H  \\nATOM   2493  OG1 THR A 167      -0.819   4.741  16.286  1.00 93.30           O  \\nATOM   2494  HG1 THR A 167       0.123   4.871  16.419  1.00 93.30           H  \\nATOM   2495  N   LYS A 168       1.068   5.434  13.619  1.00 93.75           N  \\nATOM   2496  H   LYS A 168       0.989   4.546  14.093  1.00 93.75           H  \\nATOM   2497  CA  LYS A 168       2.384   5.844  13.151  1.00 93.75           C  \\nATOM   2498  HA  LYS A 168       2.256   6.575  12.353  1.00 93.75           H  \\nATOM   2499  C   LYS A 168       3.147   4.651  12.603  1.00 93.75           C  \\nATOM   2500  CB  LYS A 168       3.129   6.499  14.319  1.00 93.75           C  \\nATOM   2501  HB2 LYS A 168       2.461   7.208  14.810  1.00 93.75           H  \\nATOM   2502  HB3 LYS A 168       3.404   5.730  15.040  1.00 93.75           H  \\nATOM   2503  O   LYS A 168       3.053   3.540  13.117  1.00 93.75           O  \\nATOM   2504  CG  LYS A 168       4.399   7.238  13.893  1.00 93.75           C  \\nATOM   2505  HG2 LYS A 168       4.154   7.995  13.148  1.00 93.75           H  \\nATOM   2506  HG3 LYS A 168       5.118   6.534  13.476  1.00 93.75           H  \\nATOM   2507  CD  LYS A 168       5.010   7.906  15.126  1.00 93.75           C  \\nATOM   2508  HD2 LYS A 168       4.295   8.620  15.534  1.00 93.75           H  \\nATOM   2509  HD3 LYS A 168       5.213   7.144  15.878  1.00 93.75           H  \\nATOM   2510  CE  LYS A 168       6.305   8.625  14.753  1.00 93.75           C  \\nATOM   2511  HE2 LYS A 168       7.001   7.895  14.339  1.00 93.75           H  \\nATOM   2512  HE3 LYS A 168       6.086   9.361  13.979  1.00 93.75           H  \\nATOM   2513  NZ  LYS A 168       6.887   9.284  15.946  1.00 93.75           N  \\nATOM   2514  HZ1 LYS A 168       6.238   9.964  16.315  1.00 93.75           H  \\nATOM   2515  HZ2 LYS A 168       7.056   8.600  16.669  1.00 93.75           H  \\nATOM   2516  HZ3 LYS A 168       7.755   9.749  15.720  1.00 93.75           H  \\nATOM   2517  N   PHE A 169       3.946   4.928  11.585  1.00 97.29           N  \\nATOM   2518  H   PHE A 169       3.962   5.875  11.235  1.00 97.29           H  \\nATOM   2519  CA  PHE A 169       4.922   4.003  11.033  1.00 97.29           C  \\nATOM   2520  HA  PHE A 169       4.697   2.985  11.348  1.00 97.29           H  \\nATOM   2521  C   PHE A 169       6.304   4.402  11.560  1.00 97.29           C  \\nATOM   2522  CB  PHE A 169       4.823   4.055   9.510  1.00 97.29           C  \\nATOM   2523  HB2 PHE A 169       4.931   5.098   9.212  1.00 97.29           H  \\nATOM   2524  HB3 PHE A 169       5.646   3.487   9.076  1.00 97.29           H  \\nATOM   2525  O   PHE A 169       6.750   5.534  11.353  1.00 97.29           O  \\nATOM   2526  CG  PHE A 169       3.507   3.532   8.962  1.00 97.29           C  \\nATOM   2527  CD1 PHE A 169       3.412   2.206   8.503  1.00 97.29           C  \\nATOM   2528  HD1 PHE A 169       4.277   1.560   8.526  1.00 97.29           H  \\nATOM   2529  CD2 PHE A 169       2.374   4.368   8.910  1.00 97.29           C  \\nATOM   2530  HD2 PHE A 169       2.438   5.393   9.245  1.00 97.29           H  \\nATOM   2531  CE1 PHE A 169       2.195   1.722   7.992  1.00 97.29           C  \\nATOM   2532  HE1 PHE A 169       2.127   0.706   7.633  1.00 97.29           H  \\nATOM   2533  CE2 PHE A 169       1.157   3.883   8.399  1.00 97.29           C  \\nATOM   2534  HE2 PHE A 169       0.292   4.529   8.350  1.00 97.29           H  \\nATOM   2535  CZ  PHE A 169       1.070   2.562   7.930  1.00 97.29           C  \\nATOM   2536  HZ  PHE A 169       0.144   2.193   7.516  1.00 97.29           H  \\nATOM   2537  N   THR A 170       6.959   3.504  12.288  1.00 96.26           N  \\nATOM   2538  H   THR A 170       6.530   2.600  12.428  1.00 96.26           H  \\nATOM   2539  CA  THR A 170       8.263   3.731  12.935  1.00 96.26           C  \\nATOM   2540  HA  THR A 170       8.586   4.756  12.752  1.00 96.26           H  \\nATOM   2541  C   THR A 170       9.340   2.825  12.327  1.00 96.26           C  \\nATOM   2542  CB  THR A 170       8.153   3.554  14.460  1.00 96.26           C  \\nATOM   2543  HB  THR A 170       9.119   3.777  14.915  1.00 96.26           H  \\nATOM   2544  O   THR A 170       9.073   2.110  11.364  1.00 96.26           O  \\nATOM   2545  CG2 THR A 170       7.107   4.479  15.087  1.00 96.26           C  \\nATOM   2546 HG21 THR A 170       7.309   5.510  14.800  1.00 96.26           H  \\nATOM   2547 HG22 THR A 170       7.148   4.396  16.173  1.00 96.26           H  \\nATOM   2548 HG23 THR A 170       6.107   4.200  14.757  1.00 96.26           H  \\nATOM   2549  OG1 THR A 170       7.803   2.232  14.767  1.00 96.26           O  \\nATOM   2550  HG1 THR A 170       7.546   2.184  15.690  1.00 96.26           H  \\nATOM   2551  N   GLY A 171      10.577   2.870  12.831  1.00 96.19           N  \\nATOM   2552  H   GLY A 171      10.767   3.456  13.631  1.00 96.19           H  \\nATOM   2553  CA  GLY A 171      11.678   2.051  12.309  1.00 96.19           C  \\nATOM   2554  HA2 GLY A 171      11.349   1.014  12.240  1.00 96.19           H  \\nATOM   2555  HA3 GLY A 171      12.522   2.086  12.998  1.00 96.19           H  \\nATOM   2556  C   GLY A 171      12.158   2.520  10.933  1.00 96.19           C  \\nATOM   2557  O   GLY A 171      12.394   3.712  10.725  1.00 96.19           O  \\nATOM   2558  N   ALA A 172      12.277   1.594   9.980  1.00 96.38           N  \\nATOM   2559  H   ALA A 172      11.978   0.654  10.199  1.00 96.38           H  \\nATOM   2560  CA  ALA A 172      12.730   1.850   8.608  1.00 96.38           C  \\nATOM   2561  HA  ALA A 172      13.571   2.542   8.659  1.00 96.38           H  \\nATOM   2562  C   ALA A 172      11.659   2.538   7.730  1.00 96.38           C  \\nATOM   2563  CB  ALA A 172      13.254   0.535   8.012  1.00 96.38           C  \\nATOM   2564  HB1 ALA A 172      12.441  -0.185   7.925  1.00 96.38           H  \\nATOM   2565  HB2 ALA A 172      14.035   0.125   8.652  1.00 96.38           H  \\nATOM   2566  HB3 ALA A 172      13.672   0.720   7.023  1.00 96.38           H  \\nATOM   2567  O   ALA A 172      11.520   2.237   6.547  1.00 96.38           O  \\nATOM   2568  N   ALA A 173      10.863   3.440   8.308  1.00 97.63           N  \\nATOM   2569  H   ALA A 173      11.042   3.671   9.275  1.00 97.63           H  \\nATOM   2570  CA  ALA A 173       9.774   4.116   7.619  1.00 97.63           C  \\nATOM   2571  HA  ALA A 173       9.099   3.352   7.234  1.00 97.63           H  \\nATOM   2572  C   ALA A 173      10.281   4.950   6.428  1.00 97.63           C  \\nATOM   2573  CB  ALA A 173       8.993   4.964   8.630  1.00 97.63           C  \\nATOM   2574  HB1 ALA A 173       8.163   5.464   8.131  1.00 97.63           H  \\nATOM   2575  HB2 ALA A 173       8.596   4.324   9.417  1.00 97.63           H  \\nATOM   2576  HB3 ALA A 173       9.647   5.715   9.073  1.00 97.63           H  \\nATOM   2577  O   ALA A 173      11.243   5.726   6.540  1.00 97.63           O  \\nATOM   2578  N   GLY A 174       9.589   4.810   5.297  1.00 98.28           N  \\nATOM   2579  H   GLY A 174       8.824   4.151   5.270  1.00 98.28           H  \\nATOM   2580  CA  GLY A 174       9.815   5.614   4.104  1.00 98.28           C  \\nATOM   2581  HA2 GLY A 174       9.247   5.180   3.281  1.00 98.28           H  \\nATOM   2582  HA3 GLY A 174      10.872   5.578   3.841  1.00 98.28           H  \\nATOM   2583  C   GLY A 174       9.389   7.083   4.272  1.00 98.28           C  \\nATOM   2584  O   GLY A 174       8.844   7.474   5.314  1.00 98.28           O  \\nATOM   2585  N   PRO A 175       9.684   7.934   3.276  1.00 98.33           N  \\nATOM   2586  CA  PRO A 175       9.400   9.365   3.337  1.00 98.33           C  \\nATOM   2587  HA  PRO A 175       9.904   9.796   4.202  1.00 98.33           H  \\nATOM   2588  C   PRO A 175       7.907   9.714   3.449  1.00 98.33           C  \\nATOM   2589  CB  PRO A 175      10.027   9.944   2.067  1.00 98.33           C  \\nATOM   2590  HB2 PRO A 175       9.455  10.790   1.688  1.00 98.33           H  \\nATOM   2591  HB3 PRO A 175      11.052  10.250   2.276  1.00 98.33           H  \\nATOM   2592  O   PRO A 175       7.573  10.609   4.226  1.00 98.33           O  \\nATOM   2593  CG  PRO A 175      10.043   8.764   1.093  1.00 98.33           C  \\nATOM   2594  HG2 PRO A 175       9.057   8.635   0.646  1.00 98.33           H  \\nATOM   2595  HG3 PRO A 175      10.807   8.876   0.324  1.00 98.33           H  \\nATOM   2596  CD  PRO A 175      10.334   7.592   2.019  1.00 98.33           C  \\nATOM   2597  HD2 PRO A 175      11.409   7.500   2.174  1.00 98.33           H  \\nATOM   2598  HD3 PRO A 175       9.937   6.673   1.588  1.00 98.33           H  \\nATOM   2599  N   TYR A 176       6.997   9.025   2.751  1.00 98.61           N  \\nATOM   2600  H   TYR A 176       7.281   8.206   2.233  1.00 98.61           H  \\nATOM   2601  CA  TYR A 176       5.557   9.273   2.898  1.00 98.61           C  \\nATOM   2602  HA  TYR A 176       5.378  10.346   2.822  1.00 98.61           H  \\nATOM   2603  C   TYR A 176       5.036   8.829   4.258  1.00 98.61           C  \\nATOM   2604  CB  TYR A 176       4.748   8.574   1.804  1.00 98.61           C  \\nATOM   2605  HB2 TYR A 176       3.691   8.645   2.059  1.00 98.61           H  \\nATOM   2606  HB3 TYR A 176       4.999   7.514   1.778  1.00 98.61           H  \\nATOM   2607  O   TYR A 176       4.200   9.513   4.844  1.00 98.61           O  \\nATOM   2608  CG  TYR A 176       4.922   9.183   0.435  1.00 98.61           C  \\nATOM   2609  CD1 TYR A 176       4.435  10.479   0.184  1.00 98.61           C  \\nATOM   2610  HD1 TYR A 176       3.947  11.026   0.977  1.00 98.61           H  \\nATOM   2611  CD2 TYR A 176       5.529   8.447  -0.596  1.00 98.61           C  \\nATOM   2612  HD2 TYR A 176       5.870   7.439  -0.413  1.00 98.61           H  \\nATOM   2613  CE1 TYR A 176       4.577  11.045  -1.097  1.00 98.61           C  \\nATOM   2614  HE1 TYR A 176       4.207  12.040  -1.296  1.00 98.61           H  \\nATOM   2615  CE2 TYR A 176       5.665   9.004  -1.877  1.00 98.61           C  \\nATOM   2616  HE2 TYR A 176       6.139   8.439  -2.666  1.00 98.61           H  \\nATOM   2617  OH  TYR A 176       5.297  10.813  -3.379  1.00 98.61           O  \\nATOM   2618  HH  TYR A 176       5.770  10.206  -3.954  1.00 98.61           H  \\nATOM   2619  CZ  TYR A 176       5.184  10.298  -2.130  1.00 98.61           C  \\nATOM   2620  N   CYS A 177       5.544   7.727   4.801  1.00 98.53           N  \\nATOM   2621  H   CYS A 177       6.160   7.158   4.238  1.00 98.53           H  \\nATOM   2622  CA  CYS A 177       5.188   7.295   6.148  1.00 98.53           C  \\nATOM   2623  HA  CYS A 177       4.103   7.278   6.249  1.00 98.53           H  \\nATOM   2624  C   CYS A 177       5.670   8.260   7.230  1.00 98.53           C  \\nATOM   2625  CB  CYS A 177       5.736   5.892   6.372  1.00 98.53           C  \\nATOM   2626  HB2 CYS A 177       5.722   5.685   7.442  1.00 98.53           H  \\nATOM   2627  HB3 CYS A 177       6.774   5.852   6.042  1.00 98.53           H  \\nATOM   2628  O   CYS A 177       4.920   8.576   8.157  1.00 98.53           O  \\nATOM   2629  SG  CYS A 177       4.820   4.580   5.541  1.00 98.53           S  \\nATOM   2630  N   LYS A 178       6.890   8.791   7.087  1.00 97.95           N  \\nATOM   2631  H   LYS A 178       7.459   8.467   6.318  1.00 97.95           H  \\nATOM   2632  CA  LYS A 178       7.395   9.871   7.946  1.00 97.95           C  \\nATOM   2633  HA  LYS A 178       7.355   9.551   8.988  1.00 97.95           H  \\nATOM   2634  C   LYS A 178       6.502  11.111   7.850  1.00 97.95           C  \\nATOM   2635  CB  LYS A 178       8.849  10.189   7.570  1.00 97.95           C  \\nATOM   2636  HB2 LYS A 178       8.927  10.277   6.486  1.00 97.95           H  \\nATOM   2637  HB3 LYS A 178       9.135  11.145   8.009  1.00 97.95           H  \\nATOM   2638  O   LYS A 178       6.108  11.644   8.884  1.00 97.95           O  \\nATOM   2639  CG  LYS A 178       9.819   9.112   8.078  1.00 97.95           C  \\nATOM   2640  HG2 LYS A 178       9.417   8.119   7.873  1.00 97.95           H  \\nATOM   2641  HG3 LYS A 178       9.934   9.222   9.157  1.00 97.95           H  \\nATOM   2642  CD  LYS A 178      11.186   9.234   7.395  1.00 97.95           C  \\nATOM   2643  HD2 LYS A 178      11.566  10.251   7.488  1.00 97.95           H  \\nATOM   2644  HD3 LYS A 178      11.054   8.996   6.339  1.00 97.95           H  \\nATOM   2645  CE  LYS A 178      12.175   8.251   8.031  1.00 97.95           C  \\nATOM   2646  HE2 LYS A 178      11.624   7.367   8.351  1.00 97.95           H  \\nATOM   2647  HE3 LYS A 178      12.614   8.713   8.916  1.00 97.95           H  \\nATOM   2648  NZ  LYS A 178      13.226   7.825   7.078  1.00 97.95           N  \\nATOM   2649  HZ1 LYS A 178      13.933   7.274   7.545  1.00 97.95           H  \\nATOM   2650  HZ2 LYS A 178      13.656   8.604   6.602  1.00 97.95           H  \\nATOM   2651  HZ3 LYS A 178      12.796   7.207   6.405  1.00 97.95           H  \\nATOM   2652  N   ALA A 179       6.127  11.512   6.634  1.00 97.97           N  \\nATOM   2653  H   ALA A 179       6.517  11.041   5.830  1.00 97.97           H  \\nATOM   2654  CA  ALA A 179       5.238  12.647   6.393  1.00 97.97           C  \\nATOM   2655  HA  ALA A 179       5.645  13.523   6.898  1.00 97.97           H  \\nATOM   2656  C   ALA A 179       3.815  12.425   6.932  1.00 97.97           C  \\nATOM   2657  CB  ALA A 179       5.222  12.928   4.890  1.00 97.97           C  \\nATOM   2658  HB1 ALA A 179       4.603  13.803   4.690  1.00 97.97           H  \\nATOM   2659  HB2 ALA A 179       6.235  13.128   4.542  1.00 97.97           H  \\nATOM   2660  HB3 ALA A 179       4.813  12.072   4.354  1.00 97.97           H  \\nATOM   2661  O   ALA A 179       3.204  13.356   7.448  1.00 97.97           O  \\nATOM   2662  N   LEU A 180       3.285  11.199   6.878  1.00 97.55           N  \\nATOM   2663  H   LEU A 180       3.785  10.482   6.372  1.00 97.55           H  \\nATOM   2664  CA  LEU A 180       1.994  10.860   7.477  1.00 97.55           C  \\nATOM   2665  HA  LEU A 180       1.255  11.571   7.108  1.00 97.55           H  \\nATOM   2666  C   LEU A 180       2.015  11.051   8.994  1.00 97.55           C  \\nATOM   2667  CB  LEU A 180       1.572   9.442   7.050  1.00 97.55           C  \\nATOM   2668  HB2 LEU A 180       1.516   9.404   5.962  1.00 97.55           H  \\nATOM   2669  HB3 LEU A 180       2.343   8.740   7.369  1.00 97.55           H  \\nATOM   2670  O   LEU A 180       1.101  11.665   9.539  1.00 97.55           O  \\nATOM   2671  CG  LEU A 180       0.223   8.978   7.637  1.00 97.55           C  \\nATOM   2672  HG  LEU A 180       0.275   8.999   8.726  1.00 97.55           H  \\nATOM   2673  CD1 LEU A 180      -0.958   9.841   7.181  1.00 97.55           C  \\nATOM   2674 HD11 LEU A 180      -1.877   9.449   7.616  1.00 97.55           H  \\nATOM   2675 HD12 LEU A 180      -1.038   9.821   6.094  1.00 97.55           H  \\nATOM   2676 HD13 LEU A 180      -0.827  10.865   7.531  1.00 97.55           H  \\nATOM   2677  CD2 LEU A 180      -0.055   7.539   7.209  1.00 97.55           C  \\nATOM   2678 HD21 LEU A 180      -0.163   7.485   6.125  1.00 97.55           H  \\nATOM   2679 HD22 LEU A 180       0.771   6.897   7.515  1.00 97.55           H  \\nATOM   2680 HD23 LEU A 180      -0.968   7.179   7.681  1.00 97.55           H  \\nATOM   2681  N   GLY A 181       3.085  10.603   9.655  1.00 93.71           N  \\nATOM   2682  H   GLY A 181       3.799  10.117   9.132  1.00 93.71           H  \\nATOM   2683  CA  GLY A 181       3.265  10.776  11.097  1.00 93.71           C  \\nATOM   2684  HA2 GLY A 181       2.416  10.332  11.619  1.00 93.71           H  \\nATOM   2685  HA3 GLY A 181       4.173  10.254  11.397  1.00 93.71           H  \\nATOM   2686  C   GLY A 181       3.389  12.236  11.552  1.00 93.71           C  \\nATOM   2687  O   GLY A 181       3.064  12.535  12.698  1.00 93.71           O  \\nATOM   2688  N   SER A 182       3.833  13.144  10.678  1.00 94.94           N  \\nATOM   2689  H   SER A 182       4.159  12.830   9.775  1.00 94.94           H  \\nATOM   2690  CA  SER A 182       3.905  14.591  10.940  1.00 94.94           C  \\nATOM   2691  HA  SER A 182       3.814  14.762  12.012  1.00 94.94           H  \\nATOM   2692  C   SER A 182       2.780  15.401  10.281  1.00 94.94           C  \\nATOM   2693  CB  SER A 182       5.273  15.130  10.512  1.00 94.94           C  \\nATOM   2694  HB2 SER A 182       5.324  16.198  10.724  1.00 94.94           H  \\nATOM   2695  HB3 SER A 182       6.054  14.623  11.079  1.00 94.94           H  \\nATOM   2696  O   SER A 182       2.767  16.625  10.396  1.00 94.94           O  \\nATOM   2697  OG  SER A 182       5.483  14.910   9.131  1.00 94.94           O  \\nATOM   2698  HG  SER A 182       6.073  15.588   8.793  1.00 94.94           H  \\nATOM   2699  N   ASN A 183       1.842  14.742   9.590  1.00 94.78           N  \\nATOM   2700  H   ASN A 183       1.925  13.737   9.544  1.00 94.78           H  \\nATOM   2701  CA  ASN A 183       0.819  15.363   8.742  1.00 94.78           C  \\nATOM   2702  HA  ASN A 183       0.398  14.557   8.141  1.00 94.78           H  \\nATOM   2703  C   ASN A 183       1.387  16.378   7.718  1.00 94.78           C  \\nATOM   2704  CB  ASN A 183      -0.310  15.908   9.633  1.00 94.78           C  \\nATOM   2705  HB2 ASN A 183       0.062  16.779  10.172  1.00 94.78           H  \\nATOM   2706  HB3 ASN A 183      -0.595  15.151  10.364  1.00 94.78           H  \\nATOM   2707  O   ASN A 183       0.740  17.370   7.375  1.00 94.78           O  \\nATOM   2708  CG  ASN A 183      -1.557  16.293   8.854  1.00 94.78           C  \\nATOM   2709  ND2 ASN A 183      -2.221  17.347   9.270  1.00 94.78           N  \\nATOM   2710 HD21 ASN A 183      -1.844  17.895  10.030  1.00 94.78           H  \\nATOM   2711 HD22 ASN A 183      -3.042  17.620   8.750  1.00 94.78           H  \\nATOM   2712  OD1 ASN A 183      -1.977  15.649   7.897  1.00 94.78           O  \\nATOM   2713  N   ASP A 184       2.595  16.135   7.204  1.00 97.47           N  \\nATOM   2714  H   ASP A 184       3.069  15.289   7.484  1.00 97.47           H  \\nATOM   2715  CA  ASP A 184       3.228  16.982   6.193  1.00 97.47           C  \\nATOM   2716  HA  ASP A 184       3.018  18.024   6.435  1.00 97.47           H  \\nATOM   2717  C   ASP A 184       2.643  16.699   4.801  1.00 97.47           C  \\nATOM   2718  CB  ASP A 184       4.758  16.828   6.222  1.00 97.47           C  \\nATOM   2719  HB2 ASP A 184       5.126  17.144   7.199  1.00 97.47           H  \\nATOM   2720  HB3 ASP A 184       5.020  15.780   6.083  1.00 97.47           H  \\nATOM   2721  O   ASP A 184       3.099  15.827   4.066  1.00 97.47           O  \\nATOM   2722  CG  ASP A 184       5.448  17.669   5.135  1.00 97.47           C  \\nATOM   2723  OD1 ASP A 184       4.784  18.524   4.498  1.00 97.47           O  \\nATOM   2724  OD2 ASP A 184       6.649  17.454   4.884  1.00 97.47           O  \\nATOM   2725  N   LYS A 185       1.632  17.474   4.399  1.00 97.59           N  \\nATOM   2726  H   LYS A 185       1.260  18.124   5.077  1.00 97.59           H  \\nATOM   2727  CA  LYS A 185       1.024  17.380   3.059  1.00 97.59           C  \\nATOM   2728  HA  LYS A 185       0.866  16.327   2.827  1.00 97.59           H  \\nATOM   2729  C   LYS A 185       1.924  17.903   1.940  1.00 97.59           C  \\nATOM   2730  CB  LYS A 185      -0.322  18.120   3.035  1.00 97.59           C  \\nATOM   2731  HB2 LYS A 185      -0.153  19.162   3.307  1.00 97.59           H  \\nATOM   2732  HB3 LYS A 185      -0.726  18.104   2.023  1.00 97.59           H  \\nATOM   2733  O   LYS A 185       1.594  17.715   0.769  1.00 97.59           O  \\nATOM   2734  CG  LYS A 185      -1.376  17.537   3.983  1.00 97.59           C  \\nATOM   2735  HG2 LYS A 185      -0.980  17.499   4.998  1.00 97.59           H  \\nATOM   2736  HG3 LYS A 185      -2.233  18.211   3.985  1.00 97.59           H  \\nATOM   2737  CD  LYS A 185      -1.858  16.142   3.570  1.00 97.59           C  \\nATOM   2738  HD2 LYS A 185      -2.276  16.175   2.564  1.00 97.59           H  \\nATOM   2739  HD3 LYS A 185      -1.023  15.441   3.590  1.00 97.59           H  \\nATOM   2740  CE  LYS A 185      -2.936  15.719   4.573  1.00 97.59           C  \\nATOM   2741  HE2 LYS A 185      -2.526  15.798   5.580  1.00 97.59           H  \\nATOM   2742  HE3 LYS A 185      -3.768  16.420   4.497  1.00 97.59           H  \\nATOM   2743  NZ  LYS A 185      -3.412  14.342   4.336  1.00 97.59           N  \\nATOM   2744  HZ1 LYS A 185      -3.606  14.173   3.359  1.00 97.59           H  \\nATOM   2745  HZ2 LYS A 185      -4.225  14.126   4.895  1.00 97.59           H  \\nATOM   2746  HZ3 LYS A 185      -2.677  13.689   4.571  1.00 97.59           H  \\nATOM   2747  N   SER A 186       3.052  18.543   2.249  1.00 97.63           N  \\nATOM   2748  H   SER A 186       3.361  18.575   3.210  1.00 97.63           H  \\nATOM   2749  CA  SER A 186       3.924  19.115   1.225  1.00 97.63           C  \\nATOM   2750  HA  SER A 186       3.315  19.763   0.594  1.00 97.63           H  \\nATOM   2751  C   SER A 186       4.529  18.039   0.323  1.00 97.63           C  \\nATOM   2752  CB  SER A 186       5.012  19.981   1.866  1.00 97.63           C  \\nATOM   2753  HB2 SER A 186       4.618  20.493   2.744  1.00 97.63           H  \\nATOM   2754  HB3 SER A 186       5.315  20.740   1.145  1.00 97.63           H  \\nATOM   2755  O   SER A 186       4.853  18.337  -0.826  1.00 97.63           O  \\nATOM   2756  OG  SER A 186       6.143  19.212   2.219  1.00 97.63           O  \\nATOM   2757  HG  SER A 186       5.982  18.866   3.100  1.00 97.63           H  \\nATOM   2758  N   VAL A 187       4.653  16.794   0.801  1.00 97.99           N  \\nATOM   2759  H   VAL A 187       4.444  16.635   1.777  1.00 97.99           H  \\nATOM   2760  CA  VAL A 187       5.100  15.655  -0.011  1.00 97.99           C  \\nATOM   2761  HA  VAL A 187       5.985  15.966  -0.566  1.00 97.99           H  \\nATOM   2762  C   VAL A 187       4.064  15.230  -1.048  1.00 97.99           C  \\nATOM   2763  CB  VAL A 187       5.513  14.439   0.843  1.00 97.99           C  \\nATOM   2764  HB  VAL A 187       5.938  13.686   0.179  1.00 97.99           H  \\nATOM   2765  O   VAL A 187       4.413  14.459  -1.927  1.00 97.99           O  \\nATOM   2766  CG1 VAL A 187       6.600  14.805   1.857  1.00 97.99           C  \\nATOM   2767 HG11 VAL A 187       6.969  13.899   2.339  1.00 97.99           H  \\nATOM   2768 HG12 VAL A 187       6.206  15.470   2.625  1.00 97.99           H  \\nATOM   2769 HG13 VAL A 187       7.434  15.293   1.353  1.00 97.99           H  \\nATOM   2770  CG2 VAL A 187       4.325  13.789   1.554  1.00 97.99           C  \\nATOM   2771 HG21 VAL A 187       4.680  12.943   2.144  1.00 97.99           H  \\nATOM   2772 HG22 VAL A 187       3.843  14.506   2.217  1.00 97.99           H  \\nATOM   2773 HG23 VAL A 187       3.593  13.423   0.834  1.00 97.99           H  \\nATOM   2774  N   CYS A 188       2.814  15.703  -0.998  1.00 98.19           N  \\nATOM   2775  H   CYS A 188       2.560  16.332  -0.249  1.00 98.19           H  \\nATOM   2776  CA  CYS A 188       1.762  15.321  -1.946  1.00 98.19           C  \\nATOM   2777  HA  CYS A 188       1.898  14.270  -2.202  1.00 98.19           H  \\nATOM   2778  C   CYS A 188       1.801  16.094  -3.272  1.00 98.19           C  \\nATOM   2779  CB  CYS A 188       0.393  15.458  -1.274  1.00 98.19           C  \\nATOM   2780  HB2 CYS A 188       0.229  16.503  -1.009  1.00 98.19           H  \\nATOM   2781  HB3 CYS A 188      -0.376  15.177  -1.994  1.00 98.19           H  \\nATOM   2782  O   CYS A 188       1.099  15.713  -4.207  1.00 98.19           O  \\nATOM   2783  SG  CYS A 188       0.174  14.440   0.208  1.00 98.19           S  \\nATOM   2784  N   ASP A 189       2.624  17.140  -3.378  1.00 97.19           N  \\nATOM   2785  H   ASP A 189       3.179  17.403  -2.577  1.00 97.19           H  \\nATOM   2786  CA  ASP A 189       2.802  17.903  -4.616  1.00 97.19           C  \\nATOM   2787  HA  ASP A 189       1.839  18.326  -4.899  1.00 97.19           H  \\nATOM   2788  C   ASP A 189       3.286  16.992  -5.762  1.00 97.19           C  \\nATOM   2789  CB  ASP A 189       3.765  19.071  -4.348  1.00 97.19           C  \\nATOM   2790  HB2 ASP A 189       3.403  19.619  -3.478  1.00 97.19           H  \\nATOM   2791  HB3 ASP A 189       4.746  18.666  -4.098  1.00 97.19           H  \\nATOM   2792  O   ASP A 189       4.241  16.220  -5.609  1.00 97.19           O  \\nATOM   2793  CG  ASP A 189       3.912  20.064  -5.511  1.00 97.19           C  \\nATOM   2794  OD1 ASP A 189       3.581  19.712  -6.668  1.00 97.19           O  \\nATOM   2795  OD2 ASP A 189       4.398  21.178  -5.244  1.00 97.19           O  \\nATOM   2796  N   LYS A 190       2.618  17.098  -6.917  1.00 94.69           N  \\nATOM   2797  H   LYS A 190       1.929  17.835  -6.972  1.00 94.69           H  \\nATOM   2798  CA  LYS A 190       2.934  16.357  -8.146  1.00 94.69           C  \\nATOM   2799  HA  LYS A 190       2.952  15.294  -7.905  1.00 94.69           H  \\nATOM   2800  C   LYS A 190       4.313  16.687  -8.720  1.00 94.69           C  \\nATOM   2801  CB  LYS A 190       1.830  16.582  -9.192  1.00 94.69           C  \\nATOM   2802  HB2 LYS A 190       0.874  16.264  -8.776  1.00 94.69           H  \\nATOM   2803  HB3 LYS A 190       2.050  15.948 -10.051  1.00 94.69           H  \\nATOM   2804  O   LYS A 190       4.925  15.851  -9.376  1.00 94.69           O  \\nATOM   2805  CG  LYS A 190       1.722  18.037  -9.680  1.00 94.69           C  \\nATOM   2806  HG2 LYS A 190       1.478  18.693  -8.844  1.00 94.69           H  \\nATOM   2807  HG3 LYS A 190       2.674  18.349 -10.108  1.00 94.69           H  \\nATOM   2808  CD  LYS A 190       0.644  18.168 -10.758  1.00 94.69           C  \\nATOM   2809  HD2 LYS A 190      -0.327  17.916 -10.330  1.00 94.69           H  \\nATOM   2810  HD3 LYS A 190       0.869  17.474 -11.567  1.00 94.69           H  \\nATOM   2811  CE  LYS A 190       0.636  19.603 -11.291  1.00 94.69           C  \\nATOM   2812  HE2 LYS A 190       0.413  20.281 -10.467  1.00 94.69           H  \\nATOM   2813  HE3 LYS A 190       1.638  19.835 -11.652  1.00 94.69           H  \\nATOM   2814  NZ  LYS A 190      -0.354  19.763 -12.383  1.00 94.69           N  \\nATOM   2815  HZ1 LYS A 190      -1.284  19.554 -12.050  1.00 94.69           H  \\nATOM   2816  HZ2 LYS A 190      -0.335  20.707 -12.743  1.00 94.69           H  \\nATOM   2817  HZ3 LYS A 190      -0.133  19.127 -13.136  1.00 94.69           H  \\nATOM   2818  N   ASN A 191       4.823  17.886  -8.445  1.00 95.55           N  \\nATOM   2819  H   ASN A 191       4.273  18.520  -7.882  1.00 95.55           H  \\nATOM   2820  CA  ASN A 191       6.120  18.346  -8.932  1.00 95.55           C  \\nATOM   2821  HA  ASN A 191       6.280  17.960  -9.939  1.00 95.55           H  \\nATOM   2822  C   ASN A 191       7.286  17.846  -8.066  1.00 95.55           C  \\nATOM   2823  CB  ASN A 191       6.098  19.877  -9.002  1.00 95.55           C  \\nATOM   2824  HB2 ASN A 191       7.055  20.237  -9.380  1.00 95.55           H  \\nATOM   2825  HB3 ASN A 191       5.961  20.277  -7.997  1.00 95.55           H  \\nATOM   2826  O   ASN A 191       8.443  17.943  -8.475  1.00 95.55           O  \\nATOM   2827  CG  ASN A 191       5.011  20.420  -9.912  1.00 95.55           C  \\nATOM   2828  ND2 ASN A 191       4.317  21.440  -9.468  1.00 95.55           N  \\nATOM   2829 HD21 ASN A 191       3.634  21.855 -10.086  1.00 95.55           H  \\nATOM   2830 HD22 ASN A 191       4.407  21.700  -8.496  1.00 95.55           H  \\nATOM   2831  OD1 ASN A 191       4.760  19.947 -11.010  1.00 95.55           O  \\nATOM   2832  N   LYS A 192       7.010  17.302  -6.873  1.00 96.66           N  \\nATOM   2833  H   LYS A 192       6.041  17.182  -6.614  1.00 96.66           H  \\nATOM   2834  CA  LYS A 192       8.044  16.753  -5.993  1.00 96.66           C  \\nATOM   2835  HA  LYS A 192       8.975  17.285  -6.183  1.00 96.66           H  \\nATOM   2836  C   LYS A 192       8.295  15.281  -6.288  1.00 96.66           C  \\nATOM   2837  CB  LYS A 192       7.686  16.959  -4.521  1.00 96.66           C  \\nATOM   2838  HB2 LYS A 192       6.692  16.558  -4.321  1.00 96.66           H  \\nATOM   2839  HB3 LYS A 192       8.409  16.406  -3.923  1.00 96.66           H  \\nATOM   2840  O   LYS A 192       7.365  14.489  -6.401  1.00 96.66           O  \\nATOM   2841  CG  LYS A 192       7.750  18.437  -4.114  1.00 96.66           C  \\nATOM   2842  HG2 LYS A 192       6.953  18.994  -4.607  1.00 96.66           H  \\nATOM   2843  HG3 LYS A 192       8.712  18.861  -4.403  1.00 96.66           H  \\nATOM   2844  CD  LYS A 192       7.599  18.525  -2.596  1.00 96.66           C  \\nATOM   2845  HD2 LYS A 192       8.464  18.062  -2.121  1.00 96.66           H  \\nATOM   2846  HD3 LYS A 192       6.709  17.961  -2.315  1.00 96.66           H  \\nATOM   2847  CE  LYS A 192       7.471  19.972  -2.117  1.00 96.66           C  \\nATOM   2848  HE2 LYS A 192       8.382  20.516  -2.365  1.00 96.66           H  \\nATOM   2849  HE3 LYS A 192       6.642  20.438  -2.651  1.00 96.66           H  \\nATOM   2850  NZ  LYS A 192       7.216  20.011  -0.657  1.00 96.66           N  \\nATOM   2851  HZ1 LYS A 192       7.961  19.587  -0.123  1.00 96.66           H  \\nATOM   2852  HZ2 LYS A 192       7.081  20.956  -0.327  1.00 96.66           H  \\nATOM   2853  HZ3 LYS A 192       6.378  19.484  -0.455  1.00 96.66           H  \\nATOM   2854  N   ASN A 193       9.575  14.921  -6.342  1.00 96.82           N  \\nATOM   2855  H   ASN A 193      10.272  15.640  -6.208  1.00 96.82           H  \\nATOM   2856  CA  ASN A 193      10.034  13.543  -6.449  1.00 96.82           C  \\nATOM   2857  HA  ASN A 193       9.222  12.909  -6.807  1.00 96.82           H  \\nATOM   2858  C   ASN A 193      10.457  13.029  -5.070  1.00 96.82           C  \\nATOM   2859  CB  ASN A 193      11.182  13.498  -7.465  1.00 96.82           C  \\nATOM   2860  HB2 ASN A 193      10.856  13.934  -8.410  1.00 96.82           H  \\nATOM   2861  HB3 ASN A 193      12.025  14.083  -7.097  1.00 96.82           H  \\nATOM   2862  O   ASN A 193      11.546  13.351  -4.600  1.00 96.82           O  \\nATOM   2863  CG  ASN A 193      11.626  12.073  -7.722  1.00 96.82           C  \\nATOM   2864  ND2 ASN A 193      12.825  11.708  -7.344  1.00 96.82           N  \\nATOM   2865 HD21 ASN A 193      13.102  10.750  -7.499  1.00 96.82           H  \\nATOM   2866 HD22 ASN A 193      13.391  12.333  -6.788  1.00 96.82           H  \\nATOM   2867  OD1 ASN A 193      10.893  11.292  -8.312  1.00 96.82           O  \\nATOM   2868  N   ILE A 194       9.594  12.245  -4.434  1.00 98.19           N  \\nATOM   2869  H   ILE A 194       8.727  12.000  -4.890  1.00 98.19           H  \\nATOM   2870  CA  ILE A 194       9.796  11.746  -3.073  1.00 98.19           C  \\nATOM   2871  HA  ILE A 194      10.578  12.323  -2.579  1.00 98.19           H  \\nATOM   2872  C   ILE A 194      10.279  10.295  -3.145  1.00 98.19           C  \\nATOM   2873  CB  ILE A 194       8.498  11.929  -2.261  1.00 98.19           C  \\nATOM   2874  HB  ILE A 194       7.743  11.273  -2.694  1.00 98.19           H  \\nATOM   2875  O   ILE A 194       9.488   9.358  -3.082  1.00 98.19           O  \\nATOM   2876  CG1 ILE A 194       7.935  13.368  -2.317  1.00 98.19           C  \\nATOM   2877 HG12 ILE A 194       7.671  13.618  -3.344  1.00 98.19           H  \\nATOM   2878 HG13 ILE A 194       7.005  13.393  -1.749  1.00 98.19           H  \\nATOM   2879  CG2 ILE A 194       8.703  11.502  -0.803  1.00 98.19           C  \\nATOM   2880 HG21 ILE A 194       8.878  10.426  -0.772  1.00 98.19           H  \\nATOM   2881 HG22 ILE A 194       9.553  12.021  -0.359  1.00 98.19           H  \\nATOM   2882 HG23 ILE A 194       7.801  11.701  -0.224  1.00 98.19           H  \\nATOM   2883  CD1 ILE A 194       8.866  14.462  -1.774  1.00 98.19           C  \\nATOM   2884 HD11 ILE A 194       9.132  14.259  -0.737  1.00 98.19           H  \\nATOM   2885 HD12 ILE A 194       9.772  14.525  -2.377  1.00 98.19           H  \\nATOM   2886 HD13 ILE A 194       8.349  15.421  -1.808  1.00 98.19           H  \\nATOM   2887  N   ALA A 195      11.581  10.108  -3.357  1.00 97.43           N  \\nATOM   2888  H   ALA A 195      12.192  10.912  -3.365  1.00 97.43           H  \\nATOM   2889  CA  ALA A 195      12.186   8.780  -3.429  1.00 97.43           C  \\nATOM   2890  HA  ALA A 195      11.428   8.078  -3.774  1.00 97.43           H  \\nATOM   2891  C   ALA A 195      12.654   8.294  -2.049  1.00 97.43           C  \\nATOM   2892  CB  ALA A 195      13.315   8.790  -4.466  1.00 97.43           C  \\nATOM   2893  HB1 ALA A 195      14.093   9.489  -4.159  1.00 97.43           H  \\nATOM   2894  HB2 ALA A 195      13.748   7.793  -4.538  1.00 97.43           H  \\nATOM   2895  HB3 ALA A 195      12.921   9.082  -5.439  1.00 97.43           H  \\nATOM   2896  O   ALA A 195      13.210   9.070  -1.273  1.00 97.43           O  \\nATOM   2897  N   GLY A 196      12.444   7.007  -1.763  1.00 97.18           N  \\nATOM   2898  H   GLY A 196      11.954   6.442  -2.442  1.00 97.18           H  \\nATOM   2899  CA  GLY A 196      13.068   6.309  -0.635  1.00 97.18           C  \\nATOM   2900  HA2 GLY A 196      12.348   5.617  -0.200  1.00 97.18           H  \\nATOM   2901  HA3 GLY A 196      13.361   7.019   0.139  1.00 97.18           H  \\nATOM   2902  C   GLY A 196      14.309   5.520  -1.060  1.00 97.18           C  \\nATOM   2903  O   GLY A 196      14.778   5.641  -2.194  1.00 97.18           O  \\nATOM   2904  N   ASP A 197      14.797   4.647  -0.176  1.00 96.42           N  \\nATOM   2905  H   ASP A 197      14.439   4.666   0.768  1.00 96.42           H  \\nATOM   2906  CA  ASP A 197      16.004   3.833  -0.406  1.00 96.42           C  \\nATOM   2907  HA  ASP A 197      16.852   4.499  -0.568  1.00 96.42           H  \\nATOM   2908  C   ASP A 197      15.896   2.913  -1.633  1.00 96.42           C  \\nATOM   2909  CB  ASP A 197      16.297   2.974   0.834  1.00 96.42           C  \\nATOM   2910  HB2 ASP A 197      17.155   2.336   0.620  1.00 96.42           H  \\nATOM   2911  HB3 ASP A 197      15.439   2.330   1.027  1.00 96.42           H  \\nATOM   2912  O   ASP A 197      16.898   2.530  -2.227  1.00 96.42           O  \\nATOM   2913  CG  ASP A 197      16.617   3.779   2.096  1.00 96.42           C  \\nATOM   2914  OD1 ASP A 197      16.955   4.976   1.969  1.00 96.42           O  \\nATOM   2915  OD2 ASP A 197      16.470   3.186   3.186  1.00 96.42           O  \\nATOM   2916  N   TRP A 198      14.683   2.590  -2.079  1.00 96.97           N  \\nATOM   2917  H   TRP A 198      13.890   2.946  -1.564  1.00 96.97           H  \\nATOM   2918  CA  TRP A 198      14.424   1.836  -3.310  1.00 96.97           C  \\nATOM   2919  HA  TRP A 198      15.090   0.973  -3.335  1.00 96.97           H  \\nATOM   2920  C   TRP A 198      14.686   2.631  -4.605  1.00 96.97           C  \\nATOM   2921  CB  TRP A 198      12.984   1.330  -3.260  1.00 96.97           C  \\nATOM   2922  HB2 TRP A 198      12.739   0.821  -4.193  1.00 96.97           H  \\nATOM   2923  HB3 TRP A 198      12.900   0.602  -2.453  1.00 96.97           H  \\nATOM   2924  O   TRP A 198      14.674   2.038  -5.681  1.00 96.97           O  \\nATOM   2925  CG  TRP A 198      11.988   2.409  -3.006  1.00 96.97           C  \\nATOM   2926  CD1 TRP A 198      11.478   3.240  -3.932  1.00 96.97           C  \\nATOM   2927  HD1 TRP A 198      11.712   3.214  -4.987  1.00 96.97           H  \\nATOM   2928  CD2 TRP A 198      11.429   2.841  -1.732  1.00 96.97           C  \\nATOM   2929  CE2 TRP A 198      10.546   3.934  -1.976  1.00 96.97           C  \\nATOM   2930  CE3 TRP A 198      11.597   2.423  -0.397  1.00 96.97           C  \\nATOM   2931  HE3 TRP A 198      12.263   1.603  -0.175  1.00 96.97           H  \\nATOM   2932  NE1 TRP A 198      10.626   4.132  -3.330  1.00 96.97           N  \\nATOM   2933  HE1 TRP A 198      10.202   4.895  -3.839  1.00 96.97           H  \\nATOM   2934  CH2 TRP A 198      10.018   4.116   0.361  1.00 96.97           C  \\nATOM   2935  HH2 TRP A 198       9.476   4.586   1.169  1.00 96.97           H  \\nATOM   2936  CZ2 TRP A 198       9.842   4.572  -0.953  1.00 96.97           C  \\nATOM   2937  HZ2 TRP A 198       9.170   5.392  -1.159  1.00 96.97           H  \\nATOM   2938  CZ3 TRP A 198      10.894   3.056   0.642  1.00 96.97           C  \\nATOM   2939  HZ3 TRP A 198      11.015   2.723   1.662  1.00 96.97           H  \\nATOM   2940  N   GLY A 199      14.945   3.942  -4.529  1.00 96.97           N  \\nATOM   2941  H   GLY A 199      14.932   4.369  -3.614  1.00 96.97           H  \\nATOM   2942  CA  GLY A 199      15.490   4.745  -5.632  1.00 96.97           C  \\nATOM   2943  HA2 GLY A 199      16.141   4.119  -6.243  1.00 96.97           H  \\nATOM   2944  HA3 GLY A 199      16.105   5.534  -5.199  1.00 96.97           H  \\nATOM   2945  C   GLY A 199      14.481   5.418  -6.573  1.00 96.97           C  \\nATOM   2946  O   GLY A 199      14.886   5.995  -7.579  1.00 96.97           O  \\nATOM   2947  N   PHE A 200      13.179   5.388  -6.279  1.00 97.32           N  \\nATOM   2948  H   PHE A 200      12.891   4.939  -5.421  1.00 97.32           H  \\nATOM   2949  CA  PHE A 200      12.151   6.069  -7.083  1.00 97.32           C  \\nATOM   2950  HA  PHE A 200      12.603   6.942  -7.555  1.00 97.32           H  \\nATOM   2951  C   PHE A 200      10.994   6.578  -6.216  1.00 97.32           C  \\nATOM   2952  CB  PHE A 200      11.650   5.137  -8.201  1.00 97.32           C  \\nATOM   2953  HB2 PHE A 200      11.023   5.714  -8.879  1.00 97.32           H  \\nATOM   2954  HB3 PHE A 200      12.505   4.791  -8.782  1.00 97.32           H  \\nATOM   2955  O   PHE A 200      10.845   6.158  -5.076  1.00 97.32           O  \\nATOM   2956  CG  PHE A 200      10.863   3.930  -7.726  1.00 97.32           C  \\nATOM   2957  CD1 PHE A 200      11.534   2.740  -7.389  1.00 97.32           C  \\nATOM   2958  HD1 PHE A 200      12.609   2.675  -7.474  1.00 97.32           H  \\nATOM   2959  CD2 PHE A 200       9.458   3.983  -7.630  1.00 97.32           C  \\nATOM   2960  HD2 PHE A 200       8.929   4.882  -7.912  1.00 97.32           H  \\nATOM   2961  CE1 PHE A 200      10.806   1.637  -6.912  1.00 97.32           C  \\nATOM   2962  HE1 PHE A 200      11.326   0.738  -6.617  1.00 97.32           H  \\nATOM   2963  CE2 PHE A 200       8.734   2.864  -7.182  1.00 97.32           C  \\nATOM   2964  HE2 PHE A 200       7.655   2.879  -7.137  1.00 97.32           H  \\nATOM   2965  CZ  PHE A 200       9.409   1.695  -6.803  1.00 97.32           C  \\nATOM   2966  HZ  PHE A 200       8.854   0.839  -6.449  1.00 97.32           H  \\nATOM   2967  N   ASP A 201      10.155   7.475  -6.737  1.00 98.05           N  \\nATOM   2968  H   ASP A 201      10.310   7.809  -7.677  1.00 98.05           H  \\nATOM   2969  CA  ASP A 201       8.926   7.898  -6.048  1.00 98.05           C  \\nATOM   2970  HA  ASP A 201       9.107   7.903  -4.973  1.00 98.05           H  \\nATOM   2971  C   ASP A 201       7.774   6.926  -6.367  1.00 98.05           C  \\nATOM   2972  CB  ASP A 201       8.589   9.341  -6.448  1.00 98.05           C  \\nATOM   2973  HB2 ASP A 201       8.405   9.385  -7.522  1.00 98.05           H  \\nATOM   2974  HB3 ASP A 201       9.457   9.965  -6.236  1.00 98.05           H  \\nATOM   2975  O   ASP A 201       7.321   6.906  -7.519  1.00 98.05           O  \\nATOM   2976  CG  ASP A 201       7.379   9.931  -5.711  1.00 98.05           C  \\nATOM   2977  OD1 ASP A 201       6.573   9.183  -5.114  1.00 98.05           O  \\nATOM   2978  OD2 ASP A 201       7.237  11.173  -5.760  1.00 98.05           O  \\nATOM   2979  N   PRO A 202       7.278   6.124  -5.402  1.00 97.85           N  \\nATOM   2980  CA  PRO A 202       6.229   5.142  -5.668  1.00 97.85           C  \\nATOM   2981  HA  PRO A 202       6.552   4.462  -6.456  1.00 97.85           H  \\nATOM   2982  C   PRO A 202       4.915   5.775  -6.132  1.00 97.85           C  \\nATOM   2983  CB  PRO A 202       6.058   4.363  -4.363  1.00 97.85           C  \\nATOM   2984  HB2 PRO A 202       6.693   3.478  -4.378  1.00 97.85           H  \\nATOM   2985  HB3 PRO A 202       5.014   4.092  -4.204  1.00 97.85           H  \\nATOM   2986  O   PRO A 202       4.198   5.173  -6.934  1.00 97.85           O  \\nATOM   2987  CG  PRO A 202       6.540   5.323  -3.279  1.00 97.85           C  \\nATOM   2988  HG2 PRO A 202       6.919   4.792  -2.405  1.00 97.85           H  \\nATOM   2989  HG3 PRO A 202       5.730   5.995  -2.996  1.00 97.85           H  \\nATOM   2990  CD  PRO A 202       7.640   6.108  -3.988  1.00 97.85           C  \\nATOM   2991  HD2 PRO A 202       8.592   5.594  -3.864  1.00 97.85           H  \\nATOM   2992  HD3 PRO A 202       7.704   7.114  -3.572  1.00 97.85           H  \\nATOM   2993  N   ALA A 203       4.637   7.022  -5.739  1.00 98.09           N  \\nATOM   2994  H   ALA A 203       5.312   7.519  -5.175  1.00 98.09           H  \\nATOM   2995  CA  ALA A 203       3.440   7.747  -6.155  1.00 98.09           C  \\nATOM   2996  HA  ALA A 203       2.558   7.175  -5.868  1.00 98.09           H  \\nATOM   2997  C   ALA A 203       3.385   7.987  -7.666  1.00 98.09           C  \\nATOM   2998  CB  ALA A 203       3.414   9.089  -5.431  1.00 98.09           C  \\nATOM   2999  HB1 ALA A 203       2.492   9.611  -5.685  1.00 98.09           H  \\nATOM   3000  HB2 ALA A 203       3.467   8.927  -4.355  1.00 98.09           H  \\nATOM   3001  HB3 ALA A 203       4.253   9.705  -5.754  1.00 98.09           H  \\nATOM   3002  O   ALA A 203       2.311   8.197  -8.224  1.00 98.09           O  \\nATOM   3003  N   LYS A 204       4.528   7.942  -8.360  1.00 97.78           N  \\nATOM   3004  H   LYS A 204       5.384   7.764  -7.854  1.00 97.78           H  \\nATOM   3005  CA  LYS A 204       4.559   8.054  -9.818  1.00 97.78           C  \\nATOM   3006  HA  LYS A 204       3.858   8.831 -10.125  1.00 97.78           H  \\nATOM   3007  C   LYS A 204       4.071   6.780 -10.506  1.00 97.78           C  \\nATOM   3008  CB  LYS A 204       5.960   8.472 -10.281  1.00 97.78           C  \\nATOM   3009  HB2 LYS A 204       6.002   8.426 -11.369  1.00 97.78           H  \\nATOM   3010  HB3 LYS A 204       6.701   7.791  -9.864  1.00 97.78           H  \\nATOM   3011  O   LYS A 204       3.803   6.835 -11.695  1.00 97.78           O  \\nATOM   3012  CG  LYS A 204       6.313   9.902  -9.866  1.00 97.78           C  \\nATOM   3013  HG2 LYS A 204       5.572  10.585 -10.280  1.00 97.78           H  \\nATOM   3014  HG3 LYS A 204       6.304   9.989  -8.779  1.00 97.78           H  \\nATOM   3015  CD  LYS A 204       7.703  10.276 -10.392  1.00 97.78           C  \\nATOM   3016  HD2 LYS A 204       8.452   9.648  -9.911  1.00 97.78           H  \\nATOM   3017  HD3 LYS A 204       7.739  10.127 -11.472  1.00 97.78           H  \\nATOM   3018  CE  LYS A 204       7.967  11.747 -10.072  1.00 97.78           C  \\nATOM   3019  HE2 LYS A 204       7.826  11.900  -9.002  1.00 97.78           H  \\nATOM   3020  HE3 LYS A 204       7.227  12.357 -10.589  1.00 97.78           H  \\nATOM   3021  NZ  LYS A 204       9.337  12.149 -10.462  1.00 97.78           N  \\nATOM   3022  HZ1 LYS A 204       9.528  11.925 -11.428  1.00 97.78           H  \\nATOM   3023  HZ2 LYS A 204       9.998  11.676  -9.862  1.00 97.78           H  \\nATOM   3024  HZ3 LYS A 204       9.451  13.143 -10.323  1.00 97.78           H  \\nATOM   3025  N   TRP A 205       3.971   5.632  -9.836  1.00 97.75           N  \\nATOM   3026  H   TRP A 205       4.171   5.633  -8.846  1.00 97.75           H  \\nATOM   3027  CA  TRP A 205       3.513   4.365 -10.431  1.00 97.75           C  \\nATOM   3028  HA  TRP A 205       3.273   4.519 -11.483  1.00 97.75           H  \\nATOM   3029  C   TRP A 205       2.197   3.868  -9.816  1.00 97.75           C  \\nATOM   3030  CB  TRP A 205       4.618   3.302 -10.378  1.00 97.75           C  \\nATOM   3031  HB2 TRP A 205       4.155   2.350 -10.636  1.00 97.75           H  \\nATOM   3032  HB3 TRP A 205       5.007   3.236  -9.362  1.00 97.75           H  \\nATOM   3033  O   TRP A 205       1.960   2.665  -9.754  1.00 97.75           O  \\nATOM   3034  CG  TRP A 205       5.758   3.420 -11.335  1.00 97.75           C  \\nATOM   3035  CD1 TRP A 205       5.903   2.685 -12.462  1.00 97.75           C  \\nATOM   3036  HD1 TRP A 205       5.213   1.928 -12.806  1.00 97.75           H  \\nATOM   3037  CD2 TRP A 205       6.933   4.276 -11.263  1.00 97.75           C  \\nATOM   3038  CE2 TRP A 205       7.734   4.042 -12.420  1.00 97.75           C  \\nATOM   3039  CE3 TRP A 205       7.390   5.250 -10.357  1.00 97.75           C  \\nATOM   3040  HE3 TRP A 205       6.837   5.410  -9.442  1.00 97.75           H  \\nATOM   3041  NE1 TRP A 205       7.083   3.029 -13.086  1.00 97.75           N  \\nATOM   3042  HE1 TRP A 205       7.399   2.599 -13.943  1.00 97.75           H  \\nATOM   3043  CH2 TRP A 205       9.308   5.760 -11.782  1.00 97.75           C  \\nATOM   3044  HH2 TRP A 205      10.223   6.306 -11.955  1.00 97.75           H  \\nATOM   3045  CZ2 TRP A 205       8.884   4.787 -12.702  1.00 97.75           C  \\nATOM   3046  HZ2 TRP A 205       9.460   4.577 -13.591  1.00 97.75           H  \\nATOM   3047  CZ3 TRP A 205       8.567   5.980 -10.608  1.00 97.75           C  \\nATOM   3048  HZ3 TRP A 205       8.912   6.696  -9.876  1.00 97.75           H  \\nATOM   3049  N   ALA A 206       1.336   4.782  -9.371  1.00 98.26           N  \\nATOM   3050  H   ALA A 206       1.580   5.761  -9.415  1.00 98.26           H  \\nATOM   3051  CA  ALA A 206       0.067   4.422  -8.757  1.00 98.26           C  \\nATOM   3052  HA  ALA A 206       0.186   3.455  -8.268  1.00 98.26           H  \\nATOM   3053  C   ALA A 206      -1.082   4.309  -9.771  1.00 98.26           C  \\nATOM   3054  CB  ALA A 206      -0.236   5.419  -7.652  1.00 98.26           C  \\nATOM   3055  HB1 ALA A 206      -0.281   6.427  -8.066  1.00 98.26           H  \\nATOM   3056  HB2 ALA A 206      -1.199   5.171  -7.207  1.00 98.26           H  \\nATOM   3057  HB3 ALA A 206       0.543   5.374  -6.891  1.00 98.26           H  \\nATOM   3058  O   ALA A 206      -1.307   5.206 -10.592  1.00 98.26           O  \\nATOM   3059  N   TYR A 207      -1.851   3.228  -9.660  1.00 98.58           N  \\nATOM   3060  H   TYR A 207      -1.598   2.544  -8.961  1.00 98.58           H  \\nATOM   3061  CA  TYR A 207      -2.975   2.896 -10.533  1.00 98.58           C  \\nATOM   3062  HA  TYR A 207      -3.249   3.781 -11.107  1.00 98.58           H  \\nATOM   3063  C   TYR A 207      -4.205   2.462  -9.733  1.00 98.58           C  \\nATOM   3064  CB  TYR A 207      -2.552   1.796 -11.516  1.00 98.58           C  \\nATOM   3065  HB2 TYR A 207      -3.428   1.480 -12.082  1.00 98.58           H  \\nATOM   3066  HB3 TYR A 207      -2.195   0.932 -10.955  1.00 98.58           H  \\nATOM   3067  O   TYR A 207      -4.086   1.993  -8.609  1.00 98.58           O  \\nATOM   3068  CG  TYR A 207      -1.482   2.240 -12.492  1.00 98.58           C  \\nATOM   3069  CD1 TYR A 207      -1.857   2.819 -13.717  1.00 98.58           C  \\nATOM   3070  HD1 TYR A 207      -2.902   2.892 -13.981  1.00 98.58           H  \\nATOM   3071  CD2 TYR A 207      -0.119   2.115 -12.160  1.00 98.58           C  \\nATOM   3072  HD2 TYR A 207       0.163   1.669 -11.217  1.00 98.58           H  \\nATOM   3073  CE1 TYR A 207      -0.878   3.308 -14.597  1.00 98.58           C  \\nATOM   3074  HE1 TYR A 207      -1.178   3.750 -15.536  1.00 98.58           H  \\nATOM   3075  CE2 TYR A 207       0.870   2.598 -13.039  1.00 98.58           C  \\nATOM   3076  HE2 TYR A 207       1.916   2.518 -12.782  1.00 98.58           H  \\nATOM   3077  OH  TYR A 207       1.407   3.728 -15.099  1.00 98.58           O  \\nATOM   3078  HH  TYR A 207       0.965   4.224 -15.792  1.00 98.58           H  \\nATOM   3079  CZ  TYR A 207       0.485   3.208 -14.252  1.00 98.58           C  \\nATOM   3080  N   GLN A 208      -5.388   2.577 -10.328  1.00 98.39           N  \\nATOM   3081  H   GLN A 208      -5.420   3.023 -11.233  1.00 98.39           H  \\nATOM   3082  CA  GLN A 208      -6.619   1.946  -9.847  1.00 98.39           C  \\nATOM   3083  HA  GLN A 208      -6.402   1.297  -8.998  1.00 98.39           H  \\nATOM   3084  C   GLN A 208      -7.205   1.075 -10.952  1.00 98.39           C  \\nATOM   3085  CB  GLN A 208      -7.635   3.004  -9.397  1.00 98.39           C  \\nATOM   3086  HB2 GLN A 208      -7.616   3.846 -10.089  1.00 98.39           H  \\nATOM   3087  HB3 GLN A 208      -8.635   2.570  -9.412  1.00 98.39           H  \\nATOM   3088  O   GLN A 208      -7.228   1.474 -12.119  1.00 98.39           O  \\nATOM   3089  CG  GLN A 208      -7.350   3.479  -7.968  1.00 98.39           C  \\nATOM   3090  HG2 GLN A 208      -7.460   2.632  -7.291  1.00 98.39           H  \\nATOM   3091  HG3 GLN A 208      -6.323   3.836  -7.891  1.00 98.39           H  \\nATOM   3092  CD  GLN A 208      -8.294   4.583  -7.507  1.00 98.39           C  \\nATOM   3093  NE2 GLN A 208      -8.474   4.754  -6.218  1.00 98.39           N  \\nATOM   3094 HE21 GLN A 208      -8.197   4.023  -5.578  1.00 98.39           H  \\nATOM   3095 HE22 GLN A 208      -9.164   5.442  -5.954  1.00 98.39           H  \\nATOM   3096  OE1 GLN A 208      -8.882   5.322  -8.278  1.00 98.39           O  \\nATOM   3097  N   TYR A 209      -7.679  -0.113 -10.593  1.00 98.41           N  \\nATOM   3098  H   TYR A 209      -7.651  -0.392  -9.622  1.00 98.41           H  \\nATOM   3099  CA  TYR A 209      -8.332  -1.001 -11.543  1.00 98.41           C  \\nATOM   3100  HA  TYR A 209      -7.814  -0.928 -12.498  1.00 98.41           H  \\nATOM   3101  C   TYR A 209      -9.790  -0.592 -11.765  1.00 98.41           C  \\nATOM   3102  CB  TYR A 209      -8.201  -2.445 -11.069  1.00 98.41           C  \\nATOM   3103  HB2 TYR A 209      -7.142  -2.683 -10.975  1.00 98.41           H  \\nATOM   3104  HB3 TYR A 209      -8.659  -2.523 -10.082  1.00 98.41           H  \\nATOM   3105  O   TYR A 209     -10.580  -0.473 -10.830  1.00 98.41           O  \\nATOM   3106  CG  TYR A 209      -8.834  -3.461 -11.994  1.00 98.41           C  \\nATOM   3107  CD1 TYR A 209      -9.853  -4.296 -11.505  1.00 98.41           C  \\nATOM   3108  HD1 TYR A 209     -10.164  -4.222 -10.473  1.00 98.41           H  \\nATOM   3109  CD2 TYR A 209      -8.426  -3.560 -13.339  1.00 98.41           C  \\nATOM   3110  HD2 TYR A 209      -7.625  -2.936 -13.709  1.00 98.41           H  \\nATOM   3111  CE1 TYR A 209     -10.508  -5.190 -12.364  1.00 98.41           C  \\nATOM   3112  HE1 TYR A 209     -11.307  -5.806 -11.977  1.00 98.41           H  \\nATOM   3113  CE2 TYR A 209      -9.078  -4.456 -14.207  1.00 98.41           C  \\nATOM   3114  HE2 TYR A 209      -8.786  -4.522 -15.245  1.00 98.41           H  \\nATOM   3115  OH  TYR A 209     -10.846  -6.051 -14.559  1.00 98.41           O  \\nATOM   3116  HH  TYR A 209     -11.700  -6.273 -14.181  1.00 98.41           H  \\nATOM   3117  CZ  TYR A 209     -10.152  -5.234 -13.726  1.00 98.41           C  \\nATOM   3118  N   ASP A 210     -10.135  -0.389 -13.029  1.00 97.33           N  \\nATOM   3119  H   ASP A 210      -9.427  -0.521 -13.737  1.00 97.33           H  \\nATOM   3120  CA  ASP A 210     -11.491  -0.172 -13.507  1.00 97.33           C  \\nATOM   3121  HA  ASP A 210     -12.116   0.229 -12.709  1.00 97.33           H  \\nATOM   3122  C   ASP A 210     -12.060  -1.516 -13.957  1.00 97.33           C  \\nATOM   3123  CB  ASP A 210     -11.420   0.861 -14.629  1.00 97.33           C  \\nATOM   3124  HB2 ASP A 210     -10.984   1.763 -14.198  1.00 97.33           H  \\nATOM   3125  HB3 ASP A 210     -10.751   0.503 -15.411  1.00 97.33           H  \\nATOM   3126  O   ASP A 210     -11.778  -1.996 -15.057  1.00 97.33           O  \\nATOM   3127  CG  ASP A 210     -12.752   1.238 -15.271  1.00 97.33           C  \\nATOM   3128  OD1 ASP A 210     -13.757   0.505 -15.142  1.00 97.33           O  \\nATOM   3129  OD2 ASP A 210     -12.731   2.265 -15.992  1.00 97.33           O  \\nATOM   3130  N   GLU A 211     -12.841  -2.139 -13.075  1.00 94.77           N  \\nATOM   3131  H   GLU A 211     -12.949  -1.722 -12.161  1.00 94.77           H  \\nATOM   3132  CA  GLU A 211     -13.420  -3.464 -13.296  1.00 94.77           C  \\nATOM   3133  HA  GLU A 211     -12.614  -4.168 -13.500  1.00 94.77           H  \\nATOM   3134  C   GLU A 211     -14.354  -3.497 -14.509  1.00 94.77           C  \\nATOM   3135  CB  GLU A 211     -14.140  -3.894 -12.010  1.00 94.77           C  \\nATOM   3136  HB2 GLU A 211     -14.973  -3.215 -11.827  1.00 94.77           H  \\nATOM   3137  HB3 GLU A 211     -13.437  -3.812 -11.182  1.00 94.77           H  \\nATOM   3138  O   GLU A 211     -14.328  -4.456 -15.279  1.00 94.77           O  \\nATOM   3139  CG  GLU A 211     -14.658  -5.338 -12.066  1.00 94.77           C  \\nATOM   3140  HG2 GLU A 211     -13.830  -5.991 -12.340  1.00 94.77           H  \\nATOM   3141  HG3 GLU A 211     -15.421  -5.413 -12.841  1.00 94.77           H  \\nATOM   3142  CD  GLU A 211     -15.260  -5.807 -10.730  1.00 94.77           C  \\nATOM   3143  OE1 GLU A 211     -15.359  -7.041 -10.558  1.00 94.77           O  \\nATOM   3144  OE2 GLU A 211     -15.596  -4.946  -9.884  1.00 94.77           O  \\nATOM   3145  N   LYS A 212     -15.117  -2.418 -14.723  1.00 95.05           N  \\nATOM   3146  H   LYS A 212     -14.960  -1.616 -14.129  1.00 95.05           H  \\nATOM   3147  CA  LYS A 212     -16.083  -2.306 -15.819  1.00 95.05           C  \\nATOM   3148  HA  LYS A 212     -16.728  -3.184 -15.808  1.00 95.05           H  \\nATOM   3149  C   LYS A 212     -15.389  -2.291 -17.178  1.00 95.05           C  \\nATOM   3150  CB  LYS A 212     -16.928  -1.042 -15.603  1.00 95.05           C  \\nATOM   3151  HB2 LYS A 212     -16.276  -0.169 -15.587  1.00 95.05           H  \\nATOM   3152  HB3 LYS A 212     -17.430  -1.117 -14.638  1.00 95.05           H  \\nATOM   3153  O   LYS A 212     -15.867  -2.925 -18.113  1.00 95.05           O  \\nATOM   3154  CG  LYS A 212     -17.982  -0.860 -16.704  1.00 95.05           C  \\nATOM   3155  HG2 LYS A 212     -18.629  -1.736 -16.733  1.00 95.05           H  \\nATOM   3156  HG3 LYS A 212     -17.490  -0.750 -17.671  1.00 95.05           H  \\nATOM   3157  CD  LYS A 212     -18.828   0.390 -16.453  1.00 95.05           C  \\nATOM   3158  HD2 LYS A 212     -18.174   1.259 -16.396  1.00 95.05           H  \\nATOM   3159  HD3 LYS A 212     -19.365   0.277 -15.510  1.00 95.05           H  \\nATOM   3160  CE  LYS A 212     -19.819   0.557 -17.608  1.00 95.05           C  \\nATOM   3161  HE2 LYS A 212     -20.434  -0.341 -17.670  1.00 95.05           H  \\nATOM   3162  HE3 LYS A 212     -19.254   0.635 -18.537  1.00 95.05           H  \\nATOM   3163  NZ  LYS A 212     -20.673   1.755 -17.422  1.00 95.05           N  \\nATOM   3164  HZ1 LYS A 212     -21.205   1.678 -16.566  1.00 95.05           H  \\nATOM   3165  HZ2 LYS A 212     -20.103   2.587 -17.375  1.00 95.05           H  \\nATOM   3166  HZ3 LYS A 212     -21.318   1.849 -18.194  1.00 95.05           H  \\nATOM   3167  N   ASN A 213     -14.284  -1.557 -17.288  1.00 96.35           N  \\nATOM   3168  H   ASN A 213     -13.987  -1.035 -16.476  1.00 96.35           H  \\nATOM   3169  CA  ASN A 213     -13.555  -1.398 -18.547  1.00 96.35           C  \\nATOM   3170  HA  ASN A 213     -14.204  -1.690 -19.373  1.00 96.35           H  \\nATOM   3171  C   ASN A 213     -12.318  -2.307 -18.656  1.00 96.35           C  \\nATOM   3172  CB  ASN A 213     -13.226   0.088 -18.738  1.00 96.35           C  \\nATOM   3173  HB2 ASN A 213     -12.602   0.420 -17.908  1.00 96.35           H  \\nATOM   3174  HB3 ASN A 213     -12.671   0.217 -19.667  1.00 96.35           H  \\nATOM   3175  O   ASN A 213     -11.599  -2.229 -19.649  1.00 96.35           O  \\nATOM   3176  CG  ASN A 213     -14.470   0.954 -18.818  1.00 96.35           C  \\nATOM   3177  ND2 ASN A 213     -14.582   1.962 -17.991  1.00 96.35           N  \\nATOM   3178 HD21 ASN A 213     -15.437   2.498 -18.029  1.00 96.35           H  \\nATOM   3179 HD22 ASN A 213     -13.953   2.035 -17.204  1.00 96.35           H  \\nATOM   3180  OD1 ASN A 213     -15.352   0.765 -19.636  1.00 96.35           O  \\nATOM   3181  N   ASN A 214     -12.058  -3.148 -17.648  1.00 95.45           N  \\nATOM   3182  H   ASN A 214     -12.662  -3.103 -16.841  1.00 95.45           H  \\nATOM   3183  CA  ASN A 214     -10.912  -4.056 -17.562  1.00 95.45           C  \\nATOM   3184  HA  ASN A 214     -10.846  -4.363 -16.518  1.00 95.45           H  \\nATOM   3185  C   ASN A 214      -9.566  -3.369 -17.878  1.00 95.45           C  \\nATOM   3186  CB  ASN A 214     -11.214  -5.317 -18.390  1.00 95.45           C  \\nATOM   3187  HB2 ASN A 214     -11.311  -5.029 -19.437  1.00 95.45           H  \\nATOM   3188  HB3 ASN A 214     -12.164  -5.743 -18.067  1.00 95.45           H  \\nATOM   3189  O   ASN A 214      -8.762  -3.870 -18.666  1.00 95.45           O  \\nATOM   3190  CG  ASN A 214     -10.152  -6.397 -18.265  1.00 95.45           C  \\nATOM   3191  ND2 ASN A 214      -9.725  -6.954 -19.372  1.00 95.45           N  \\nATOM   3192 HD21 ASN A 214      -8.938  -7.585 -19.311  1.00 95.45           H  \\nATOM   3193 HD22 ASN A 214     -10.027  -6.558 -20.251  1.00 95.45           H  \\nATOM   3194  OD1 ASN A 214      -9.703  -6.789 -17.193  1.00 95.45           O  \\nATOM   3195  N   LYS A 215      -9.318  -2.209 -17.258  1.00 97.03           N  \\nATOM   3196  H   LYS A 215      -9.998  -1.889 -16.583  1.00 97.03           H  \\nATOM   3197  CA  LYS A 215      -8.100  -1.405 -17.462  1.00 97.03           C  \\nATOM   3198  HA  LYS A 215      -7.318  -2.057 -17.850  1.00 97.03           H  \\nATOM   3199  C   LYS A 215      -7.573  -0.820 -16.155  1.00 97.03           C  \\nATOM   3200  CB  LYS A 215      -8.356  -0.312 -18.516  1.00 97.03           C  \\nATOM   3201  HB2 LYS A 215      -7.413   0.183 -18.746  1.00 97.03           H  \\nATOM   3202  HB3 LYS A 215      -8.721  -0.782 -19.429  1.00 97.03           H  \\nATOM   3203  O   LYS A 215      -8.303  -0.717 -15.175  1.00 97.03           O  \\nATOM   3204  CG  LYS A 215      -9.372   0.740 -18.047  1.00 97.03           C  \\nATOM   3205  HG2 LYS A 215      -9.024   1.198 -17.121  1.00 97.03           H  \\nATOM   3206  HG3 LYS A 215     -10.330   0.254 -17.861  1.00 97.03           H  \\nATOM   3207  CD  LYS A 215      -9.558   1.844 -19.088  1.00 97.03           C  \\nATOM   3208  HD2 LYS A 215      -9.914   1.414 -20.024  1.00 97.03           H  \\nATOM   3209  HD3 LYS A 215      -8.600   2.337 -19.250  1.00 97.03           H  \\nATOM   3210  CE  LYS A 215     -10.577   2.850 -18.547  1.00 97.03           C  \\nATOM   3211  HE2 LYS A 215     -11.560   2.381 -18.507  1.00 97.03           H  \\nATOM   3212  HE3 LYS A 215     -10.305   3.100 -17.521  1.00 97.03           H  \\nATOM   3213  NZ  LYS A 215     -10.611   4.081 -19.366  1.00 97.03           N  \\nATOM   3214  HZ1 LYS A 215      -9.713   4.540 -19.306  1.00 97.03           H  \\nATOM   3215  HZ2 LYS A 215     -11.312   4.708 -18.997  1.00 97.03           H  \\nATOM   3216  HZ3 LYS A 215     -10.825   3.864 -20.329  1.00 97.03           H  \\nATOM   3217  N   PHE A 216      -6.311  -0.400 -16.156  1.00 98.25           N  \\nATOM   3218  H   PHE A 216      -5.774  -0.476 -17.008  1.00 98.25           H  \\nATOM   3219  CA  PHE A 216      -5.684   0.276 -15.022  1.00 98.25           C  \\nATOM   3220  HA  PHE A 216      -6.298   0.150 -14.130  1.00 98.25           H  \\nATOM   3221  C   PHE A 216      -5.582   1.775 -15.305  1.00 98.25           C  \\nATOM   3222  CB  PHE A 216      -4.327  -0.374 -14.727  1.00 98.25           C  \\nATOM   3223  HB2 PHE A 216      -3.787   0.241 -14.007  1.00 98.25           H  \\nATOM   3224  HB3 PHE A 216      -3.735  -0.404 -15.642  1.00 98.25           H  \\nATOM   3225  O   PHE A 216      -4.862   2.203 -16.204  1.00 98.25           O  \\nATOM   3226  CG  PHE A 216      -4.461  -1.777 -14.166  1.00 98.25           C  \\nATOM   3227  CD1 PHE A 216      -4.560  -1.977 -12.776  1.00 98.25           C  \\nATOM   3228  HD1 PHE A 216      -4.513  -1.136 -12.100  1.00 98.25           H  \\nATOM   3229  CD2 PHE A 216      -4.559  -2.879 -15.036  1.00 98.25           C  \\nATOM   3230  HD2 PHE A 216      -4.510  -2.729 -16.104  1.00 98.25           H  \\nATOM   3231  CE1 PHE A 216      -4.753  -3.272 -12.265  1.00 98.25           C  \\nATOM   3232  HE1 PHE A 216      -4.865  -3.424 -11.201  1.00 98.25           H  \\nATOM   3233  CE2 PHE A 216      -4.751  -4.173 -14.521  1.00 98.25           C  \\nATOM   3234  HE2 PHE A 216      -4.835  -5.019 -15.188  1.00 98.25           H  \\nATOM   3235  CZ  PHE A 216      -4.843  -4.370 -13.134  1.00 98.25           C  \\nATOM   3236  HZ  PHE A 216      -5.000  -5.361 -12.734  1.00 98.25           H  \\nATOM   3237  N   ASN A 217      -6.310   2.582 -14.538  1.00 98.01           N  \\nATOM   3238  H   ASN A 217      -6.836   2.176 -13.777  1.00 98.01           H  \\nATOM   3239  CA  ASN A 217      -6.286   4.035 -14.652  1.00 98.01           C  \\nATOM   3240  HA  ASN A 217      -6.114   4.311 -15.693  1.00 98.01           H  \\nATOM   3241  C   ASN A 217      -5.142   4.599 -13.812  1.00 98.01           C  \\nATOM   3242  CB  ASN A 217      -7.641   4.608 -14.216  1.00 98.01           C  \\nATOM   3243  HB2 ASN A 217      -7.862   4.259 -13.207  1.00 98.01           H  \\nATOM   3244  HB3 ASN A 217      -7.585   5.696 -14.189  1.00 98.01           H  \\nATOM   3245  O   ASN A 217      -4.996   4.248 -12.642  1.00 98.01           O  \\nATOM   3246  CG  ASN A 217      -8.781   4.228 -15.146  1.00 98.01           C  \\nATOM   3247  ND2 ASN A 217      -9.952   4.010 -14.598  1.00 98.01           N  \\nATOM   3248 HD21 ASN A 217     -10.048   4.032 -13.593  1.00 98.01           H  \\nATOM   3249 HD22 ASN A 217     -10.707   3.673 -15.177  1.00 98.01           H  \\nATOM   3250  OD1 ASN A 217      -8.651   4.154 -16.361  1.00 98.01           O  \\nATOM   3251  N   TYR A 218      -4.340   5.485 -14.399  1.00 98.06           N  \\nATOM   3252  H   TYR A 218      -4.514   5.747 -15.359  1.00 98.06           H  \\nATOM   3253  CA  TYR A 218      -3.300   6.195 -13.664  1.00 98.06           C  \\nATOM   3254  HA  TYR A 218      -2.686   5.466 -13.137  1.00 98.06           H  \\nATOM   3255  C   TYR A 218      -3.920   7.142 -12.631  1.00 98.06           C  \\nATOM   3256  CB  TYR A 218      -2.415   6.969 -14.647  1.00 98.06           C  \\nATOM   3257  HB2 TYR A 218      -3.018   7.712 -15.168  1.00 98.06           H  \\nATOM   3258  HB3 TYR A 218      -2.018   6.278 -15.390  1.00 98.06           H  \\nATOM   3259  O   TYR A 218      -4.766   7.964 -12.978  1.00 98.06           O  \\nATOM   3260  CG  TYR A 218      -1.251   7.650 -13.964  1.00 98.06           C  \\nATOM   3261  CD1 TYR A 218      -1.152   9.056 -13.892  1.00 98.06           C  \\nATOM   3262  HD1 TYR A 218      -1.916   9.679 -14.335  1.00 98.06           H  \\nATOM   3263  CD2 TYR A 218      -0.281   6.846 -13.345  1.00 98.06           C  \\nATOM   3264  HD2 TYR A 218      -0.377   5.770 -13.345  1.00 98.06           H  \\nATOM   3265  CE1 TYR A 218      -0.056   9.657 -13.233  1.00 98.06           C  \\nATOM   3266  HE1 TYR A 218       0.032  10.731 -13.165  1.00 98.06           H  \\nATOM   3267  CE2 TYR A 218       0.798   7.442 -12.685  1.00 98.06           C  \\nATOM   3268  HE2 TYR A 218       1.519   6.816 -12.182  1.00 98.06           H  \\nATOM   3269  OH  TYR A 218       2.022   9.341 -12.009  1.00 98.06           O  \\nATOM   3270  HH  TYR A 218       2.074  10.298 -11.966  1.00 98.06           H  \\nATOM   3271  CZ  TYR A 218       0.934   8.841 -12.645  1.00 98.06           C  \\nATOM   3272  N   VAL A 219      -3.491   7.040 -11.372  1.00 97.74           N  \\nATOM   3273  H   VAL A 219      -2.820   6.316 -11.156  1.00 97.74           H  \\nATOM   3274  CA  VAL A 219      -3.950   7.913 -10.269  1.00 97.74           C  \\nATOM   3275  HA  VAL A 219      -4.498   8.750 -10.699  1.00 97.74           H  \\nATOM   3276  C   VAL A 219      -2.804   8.541  -9.480  1.00 97.74           C  \\nATOM   3277  CB  VAL A 219      -4.909   7.178  -9.317  1.00 97.74           C  \\nATOM   3278  HB  VAL A 219      -5.259   7.888  -8.568  1.00 97.74           H  \\nATOM   3279  O   VAL A 219      -3.048   9.206  -8.463  1.00 97.74           O  \\nATOM   3280  CG1 VAL A 219      -6.144   6.674 -10.064  1.00 97.74           C  \\nATOM   3281 HG11 VAL A 219      -6.589   7.483 -10.643  1.00 97.74           H  \\nATOM   3282 HG12 VAL A 219      -5.882   5.856 -10.735  1.00 97.74           H  \\nATOM   3283 HG13 VAL A 219      -6.878   6.316  -9.343  1.00 97.74           H  \\nATOM   3284  CG2 VAL A 219      -4.236   6.013  -8.583  1.00 97.74           C  \\nATOM   3285 HG21 VAL A 219      -3.940   5.243  -9.296  1.00 97.74           H  \\nATOM   3286 HG22 VAL A 219      -3.358   6.355  -8.035  1.00 97.74           H  \\nATOM   3287 HG23 VAL A 219      -4.937   5.586  -7.866  1.00 97.74           H  \\nATOM   3288  N   GLY A 220      -1.577   8.289  -9.937  1.00 96.13           N  \\nATOM   3289  H   GLY A 220      -1.493   7.734 -10.777  1.00 96.13           H  \\nATOM   3290  CA  GLY A 220      -0.348   8.825  -9.385  1.00 96.13           C  \\nATOM   3291  HA2 GLY A 220      -0.306   8.635  -8.313  1.00 96.13           H  \\nATOM   3292  HA3 GLY A 220       0.488   8.314  -9.862  1.00 96.13           H  \\nATOM   3293  C   GLY A 220      -0.175  10.325  -9.600  1.00 96.13           C  \\nATOM   3294  O   GLY A 220      -1.114  11.042  -9.949  1.00 96.13           O  \\nATOM   3295  N   LYS A 221       1.046  10.784  -9.341  1.00 93.80           N  \\nATOM   3296  H   LYS A 221       1.751  10.098  -9.111  1.00 93.80           H  \\nATOM   3297  CA  LYS A 221       1.462  12.185  -9.469  1.00 93.80           C  \\nATOM   3298  HA  LYS A 221       0.636  12.840  -9.192  1.00 93.80           H  \\nATOM   3299  C   LYS A 221       1.824  12.601 -10.888  1.00 93.80           C  \\nATOM   3300  CB  LYS A 221       2.641  12.435  -8.544  1.00 93.80           C  \\nATOM   3301  HB2 LYS A 221       3.377  11.643  -8.685  1.00 93.80           H  \\nATOM   3302  HB3 LYS A 221       3.111  13.374  -8.834  1.00 93.80           H  \\nATOM   3303  O   LYS A 221       2.440  11.781 -11.603  1.00 93.80           O  \\nATOM   3304  CG  LYS A 221       2.208  12.507  -7.077  1.00 93.80           C  \\nATOM   3305  HG2 LYS A 221       1.512  13.333  -6.931  1.00 93.80           H  \\nATOM   3306  HG3 LYS A 221       1.752  11.570  -6.759  1.00 93.80           H  \\nATOM   3307  CD  LYS A 221       3.492  12.761  -6.306  1.00 93.80           C  \\nATOM   3308  HD2 LYS A 221       4.176  11.928  -6.465  1.00 93.80           H  \\nATOM   3309  HD3 LYS A 221       3.965  13.658  -6.706  1.00 93.80           H  \\nATOM   3310  CE  LYS A 221       3.292  12.952  -4.819  1.00 93.80           C  \\nATOM   3311  HE2 LYS A 221       2.573  13.754  -4.648  1.00 93.80           H  \\nATOM   3312  HE3 LYS A 221       2.949  12.023  -4.364  1.00 93.80           H  \\nATOM   3313  NZ  LYS A 221       4.616  13.323  -4.301  1.00 93.80           N  \\nATOM   3314  HZ1 LYS A 221       5.270  12.565  -4.432  1.00 93.80           H  \\nATOM   3315  HZ2 LYS A 221       4.585  13.517  -3.310  1.00 93.80           H  \\nATOM   3316  HZ3 LYS A 221       4.965  14.143  -4.776  1.00 93.80           H  \\nATOM   3317  OXT LYS A 221       1.511  13.763 -11.203  1.00 93.80           O  \\nTER    3318      LYS A 221                                                       \\nEND   \\n\");\n\tviewer_16704658566527212.setStyle({\"model\": -1},{\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {}});\n\tviewer_16704658566527212.zoom(0.12);\n\tviewer_16704658566527212.rotate(235,{\"x\": 0, \"y\": 1, \"z\": 1});\n\tviewer_16704658566527212.spin({\"x\": -0.2, \"y\": 0.5, \"z\": 1},1);\nviewer_16704658566527212.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_16704658566527212\"  style=\"position: relative; width: 800px; height: 600px\">\n",
              "        <p id=\"3dmolwarning_16704658566527212\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
              "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "      resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://cdn.jsdelivr.net/npm/3dmol@latest/build/3Dmol-min.min.js');\n",
              "}\n",
              "\n",
              "var viewer_16704658566527212 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_16704658566527212\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_16704658566527212 = $3Dmol.createViewer($(\"#3dmolviewer_16704658566527212\"),{backgroundColor:\"white\"});\n",
              "viewer_16704658566527212.zoomTo();\n",
              "\tviewer_16704658566527212.addModelsAsFrames(\"ATOM      1  N   VAL A   1      34.064  -6.456  50.464  1.00 45.11           N  \\nATOM      2  H   VAL A   1      33.576  -6.009  51.228  1.00 45.11           H  \\nATOM      3  H2  VAL A   1      33.882  -7.449  50.477  1.00 45.11           H  \\nATOM      4  H3  VAL A   1      35.060  -6.323  50.566  1.00 45.11           H  \\nATOM      5  CA  VAL A   1      33.643  -5.877  49.162  1.00 45.11           C  \\nATOM      6  HA  VAL A   1      32.602  -6.130  48.959  1.00 45.11           H  \\nATOM      7  C   VAL A   1      34.525  -6.533  48.112  1.00 45.11           C  \\nATOM      8  CB  VAL A   1      33.798  -4.340  49.169  1.00 45.11           C  \\nATOM      9  HB  VAL A   1      34.856  -4.086  49.236  1.00 45.11           H  \\nATOM     10  O   VAL A   1      35.722  -6.569  48.382  1.00 45.11           O  \\nATOM     11  CG1 VAL A   1      33.211  -3.674  47.923  1.00 45.11           C  \\nATOM     12 HG11 VAL A   1      32.153  -3.914  47.814  1.00 45.11           H  \\nATOM     13 HG12 VAL A   1      33.309  -2.592  48.006  1.00 45.11           H  \\nATOM     14 HG13 VAL A   1      33.768  -3.972  47.035  1.00 45.11           H  \\nATOM     15  CG2 VAL A   1      33.081  -3.719  50.378  1.00 45.11           C  \\nATOM     16 HG21 VAL A   1      32.027  -3.996  50.385  1.00 45.11           H  \\nATOM     17 HG22 VAL A   1      33.147  -2.632  50.323  1.00 45.11           H  \\nATOM     18 HG23 VAL A   1      33.551  -4.014  51.317  1.00 45.11           H  \\nATOM     19  N   PRO A   2      33.995  -7.139  47.031  1.00 41.54           N  \\nATOM     20  CA  PRO A   2      32.926  -6.593  46.177  1.00 41.54           C  \\nATOM     21  HA  PRO A   2      32.395  -5.757  46.632  1.00 41.54           H  \\nATOM     22  C   PRO A   2      31.880  -7.669  45.766  1.00 41.54           C  \\nATOM     23  CB  PRO A   2      33.799  -6.106  45.023  1.00 41.54           C  \\nATOM     24  HB2 PRO A   2      33.209  -5.785  44.165  1.00 41.54           H  \\nATOM     25  HB3 PRO A   2      34.406  -5.259  45.344  1.00 41.54           H  \\nATOM     26  O   PRO A   2      31.961  -8.783  46.261  1.00 41.54           O  \\nATOM     27  CG  PRO A   2      34.730  -7.298  44.763  1.00 41.54           C  \\nATOM     28  HG2 PRO A   2      35.700  -6.963  44.393  1.00 41.54           H  \\nATOM     29  HG3 PRO A   2      34.275  -8.001  44.065  1.00 41.54           H  \\nATOM     30  CD  PRO A   2      34.847  -7.942  46.145  1.00 41.54           C  \\nATOM     31  HD2 PRO A   2      34.493  -8.972  46.107  1.00 41.54           H  \\nATOM     32  HD3 PRO A   2      35.893  -7.940  46.454  1.00 41.54           H  \\nATOM     33  N   VAL A   3      30.848  -7.474  44.941  1.00 40.19           N  \\nATOM     34  H   VAL A   3      30.300  -8.316  44.831  1.00 40.19           H  \\nATOM     35  CA  VAL A   3      30.251  -6.338  44.219  1.00 40.19           C  \\nATOM     36  HA  VAL A   3      30.525  -5.393  44.687  1.00 40.19           H  \\nATOM     37  C   VAL A   3      28.733  -6.578  44.318  1.00 40.19           C  \\nATOM     38  CB  VAL A   3      30.580  -6.315  42.698  1.00 40.19           C  \\nATOM     39  HB  VAL A   3      29.870  -6.972  42.195  1.00 40.19           H  \\nATOM     40  O   VAL A   3      28.286  -7.721  44.233  1.00 40.19           O  \\nATOM     41  CG1 VAL A   3      30.389  -4.904  42.129  1.00 40.19           C  \\nATOM     42 HG11 VAL A   3      29.360  -4.574  42.271  1.00 40.19           H  \\nATOM     43 HG12 VAL A   3      31.071  -4.199  42.605  1.00 40.19           H  \\nATOM     44 HG13 VAL A   3      30.579  -4.914  41.056  1.00 40.19           H  \\nATOM     45  CG2 VAL A   3      31.962  -6.800  42.240  1.00 40.19           C  \\nATOM     46 HG21 VAL A   3      31.865  -7.147  41.211  1.00 40.19           H  \\nATOM     47 HG22 VAL A   3      32.309  -7.650  42.827  1.00 40.19           H  \\nATOM     48 HG23 VAL A   3      32.692  -5.991  42.210  1.00 40.19           H  \\nATOM     49  N   ASN A   4      27.952  -5.519  44.501  1.00 42.11           N  \\nATOM     50  H   ASN A   4      28.379  -4.604  44.470  1.00 42.11           H  \\nATOM     51  CA  ASN A   4      26.494  -5.559  44.400  1.00 42.11           C  \\nATOM     52  HA  ASN A   4      26.116  -6.300  45.106  1.00 42.11           H  \\nATOM     53  C   ASN A   4      26.105  -5.946  42.955  1.00 42.11           C  \\nATOM     54  CB  ASN A   4      25.986  -4.143  44.763  1.00 42.11           C  \\nATOM     55  HB2 ASN A   4      25.812  -3.578  43.848  1.00 42.11           H  \\nATOM     56  HB3 ASN A   4      26.739  -3.606  45.340  1.00 42.11           H  \\nATOM     57  O   ASN A   4      26.673  -5.347  42.040  1.00 42.11           O  \\nATOM     58  CG  ASN A   4      24.728  -4.117  45.604  1.00 42.11           C  \\nATOM     59  ND2 ASN A   4      24.038  -3.002  45.624  1.00 42.11           N  \\nATOM     60 HD21 ASN A   4      24.264  -2.251  44.988  1.00 42.11           H  \\nATOM     61 HD22 ASN A   4      23.178  -3.029  46.153  1.00 42.11           H  \\nATOM     62  OD1 ASN A   4      24.388  -5.051  46.307  1.00 42.11           O  \\nATOM     63  N   PRO A   5      25.188  -6.900  42.714  1.00 42.39           N  \\nATOM     64  CA  PRO A   5      24.717  -7.167  41.362  1.00 42.39           C  \\nATOM     65  HA  PRO A   5      25.566  -7.322  40.696  1.00 42.39           H  \\nATOM     66  C   PRO A   5      23.911  -5.962  40.864  1.00 42.39           C  \\nATOM     67  CB  PRO A   5      23.912  -8.466  41.441  1.00 42.39           C  \\nATOM     68  HB2 PRO A   5      23.071  -8.471  40.747  1.00 42.39           H  \\nATOM     69  HB3 PRO A   5      24.572  -9.310  41.240  1.00 42.39           H  \\nATOM     70  O   PRO A   5      22.948  -5.533  41.504  1.00 42.39           O  \\nATOM     71  CG  PRO A   5      23.450  -8.520  42.897  1.00 42.39           C  \\nATOM     72  HG2 PRO A   5      22.504  -7.985  42.992  1.00 42.39           H  \\nATOM     73  HG3 PRO A   5      23.340  -9.546  43.247  1.00 42.39           H  \\nATOM     74  CD  PRO A   5      24.532  -7.769  43.675  1.00 42.39           C  \\nATOM     75  HD2 PRO A   5      24.080  -7.194  44.484  1.00 42.39           H  \\nATOM     76  HD3 PRO A   5      25.259  -8.477  44.074  1.00 42.39           H  \\nATOM     77  N   GLU A   6      24.357  -5.389  39.746  1.00 41.49           N  \\nATOM     78  H   GLU A   6      25.171  -5.776  39.291  1.00 41.49           H  \\nATOM     79  CA  GLU A   6      23.604  -4.383  39.002  1.00 41.49           C  \\nATOM     80  HA  GLU A   6      23.469  -3.516  39.649  1.00 41.49           H  \\nATOM     81  C   GLU A   6      22.232  -4.955  38.616  1.00 41.49           C  \\nATOM     82  CB  GLU A   6      24.367  -3.945  37.742  1.00 41.49           C  \\nATOM     83  HB2 GLU A   6      24.714  -4.830  37.209  1.00 41.49           H  \\nATOM     84  HB3 GLU A   6      23.686  -3.393  37.094  1.00 41.49           H  \\nATOM     85  O   GLU A   6      22.136  -6.130  38.252  1.00 41.49           O  \\nATOM     86  CG  GLU A   6      25.551  -3.028  38.079  1.00 41.49           C  \\nATOM     87  HG2 GLU A   6      25.171  -2.157  38.612  1.00 41.49           H  \\nATOM     88  HG3 GLU A   6      26.229  -3.557  38.749  1.00 41.49           H  \\nATOM     89  CD  GLU A   6      26.338  -2.574  36.838  1.00 41.49           C  \\nATOM     90  OE1 GLU A   6      26.865  -1.438  36.883  1.00 41.49           O  \\nATOM     91  OE2 GLU A   6      26.449  -3.368  35.879  1.00 41.49           O  \\nATOM     92  N   PRO A   7      21.156  -4.158  38.723  1.00 42.58           N  \\nATOM     93  CA  PRO A   7      19.829  -4.625  38.379  1.00 42.58           C  \\nATOM     94  HA  PRO A   7      19.631  -5.579  38.867  1.00 42.58           H  \\nATOM     95  C   PRO A   7      19.735  -4.802  36.866  1.00 42.58           C  \\nATOM     96  CB  PRO A   7      18.869  -3.567  38.927  1.00 42.58           C  \\nATOM     97  HB2 PRO A   7      18.610  -3.816  39.956  1.00 42.58           H  \\nATOM     98  HB3 PRO A   7      17.968  -3.476  38.320  1.00 42.58           H  \\nATOM     99  O   PRO A   7      20.021  -3.871  36.109  1.00 42.58           O  \\nATOM    100  CG  PRO A   7      19.700  -2.283  38.910  1.00 42.58           C  \\nATOM    101  HG2 PRO A   7      19.384  -1.588  39.688  1.00 42.58           H  \\nATOM    102  HG3 PRO A   7      19.623  -1.814  37.929  1.00 42.58           H  \\nATOM    103  CD  PRO A   7      21.133  -2.764  39.129  1.00 42.58           C  \\nATOM    104  HD2 PRO A   7      21.823  -2.169  38.531  1.00 42.58           H  \\nATOM    105  HD3 PRO A   7      21.385  -2.693  40.187  1.00 42.58           H  \\nATOM    106  N   ASP A   8      19.295  -5.991  36.453  1.00 40.36           N  \\nATOM    107  H   ASP A   8      19.207  -6.736  37.128  1.00 40.36           H  \\nATOM    108  CA  ASP A   8      18.903  -6.287  35.082  1.00 40.36           C  \\nATOM    109  HA  ASP A   8      19.789  -6.277  34.446  1.00 40.36           H  \\nATOM    110  C   ASP A   8      17.941  -5.203  34.596  1.00 40.36           C  \\nATOM    111  CB  ASP A   8      18.237  -7.673  34.990  1.00 40.36           C  \\nATOM    112  HB2 ASP A   8      17.594  -7.689  34.109  1.00 40.36           H  \\nATOM    113  HB3 ASP A   8      17.603  -7.834  35.862  1.00 40.36           H  \\nATOM    114  O   ASP A   8      16.776  -5.124  34.998  1.00 40.36           O  \\nATOM    115  CG  ASP A   8      19.233  -8.825  34.835  1.00 40.36           C  \\nATOM    116  OD1 ASP A   8      20.206  -8.650  34.070  1.00 40.36           O  \\nATOM    117  OD2 ASP A   8      18.973  -9.890  35.438  1.00 40.36           O  \\nATOM    118  N   ALA A   9      18.451  -4.345  33.716  1.00 40.70           N  \\nATOM    119  H   ALA A   9      19.444  -4.395  33.539  1.00 40.70           H  \\nATOM    120  CA  ALA A   9      17.648  -3.463  32.904  1.00 40.70           C  \\nATOM    121  HA  ALA A   9      16.956  -2.915  33.544  1.00 40.70           H  \\nATOM    122  C   ALA A   9      16.840  -4.347  31.949  1.00 40.70           C  \\nATOM    123  CB  ALA A   9      18.570  -2.462  32.194  1.00 40.70           C  \\nATOM    124  HB1 ALA A   9      17.975  -1.799  31.565  1.00 40.70           H  \\nATOM    125  HB2 ALA A   9      19.290  -2.992  31.572  1.00 40.70           H  \\nATOM    126  HB3 ALA A   9      19.106  -1.868  32.935  1.00 40.70           H  \\nATOM    127  O   ALA A   9      17.230  -4.577  30.805  1.00 40.70           O  \\nATOM    128  N   THR A  10      15.705  -4.865  32.421  1.00 39.44           N  \\nATOM    129  H   THR A  10      15.523  -4.803  33.413  1.00 39.44           H  \\nATOM    130  CA  THR A  10      14.665  -5.376  31.538  1.00 39.44           C  \\nATOM    131  HA  THR A  10      15.058  -6.215  30.964  1.00 39.44           H  \\nATOM    132  C   THR A  10      14.315  -4.243  30.595  1.00 39.44           C  \\nATOM    133  CB  THR A  10      13.410  -5.836  32.293  1.00 39.44           C  \\nATOM    134  HB  THR A  10      12.581  -5.903  31.588  1.00 39.44           H  \\nATOM    135  O   THR A  10      13.726  -3.240  31.005  1.00 39.44           O  \\nATOM    136  CG2 THR A  10      13.611  -7.210  32.923  1.00 39.44           C  \\nATOM    137 HG21 THR A  10      12.704  -7.502  33.452  1.00 39.44           H  \\nATOM    138 HG22 THR A  10      13.830  -7.946  32.149  1.00 39.44           H  \\nATOM    139 HG23 THR A  10      14.441  -7.173  33.630  1.00 39.44           H  \\nATOM    140  OG1 THR A  10      13.070  -4.923  33.311  1.00 39.44           O  \\nATOM    141  HG1 THR A  10      13.120  -4.049  32.916  1.00 39.44           H  \\nATOM    142  N   SER A  11      14.729  -4.390  29.340  1.00 40.63           N  \\nATOM    143  H   SER A  11      15.296  -5.192  29.108  1.00 40.63           H  \\nATOM    144  CA  SER A  11      14.310  -3.551  28.236  1.00 40.63           C  \\nATOM    145  HA  SER A  11      14.615  -2.520  28.415  1.00 40.63           H  \\nATOM    146  C   SER A  11      12.793  -3.631  28.169  1.00 40.63           C  \\nATOM    147  CB  SER A  11      14.958  -4.045  26.937  1.00 40.63           C  \\nATOM    148  HB2 SER A  11      14.525  -3.513  26.090  1.00 40.63           H  \\nATOM    149  HB3 SER A  11      16.029  -3.842  26.968  1.00 40.63           H  \\nATOM    150  O   SER A  11      12.226  -4.579  27.626  1.00 40.63           O  \\nATOM    151  OG  SER A  11      14.763  -5.439  26.780  1.00 40.63           O  \\nATOM    152  HG  SER A  11      13.817  -5.593  26.732  1.00 40.63           H  \\nATOM    153  N   VAL A  12      12.129  -2.651  28.773  1.00 41.68           N  \\nATOM    154  H   VAL A  12      12.647  -2.004  29.350  1.00 41.68           H  \\nATOM    155  CA  VAL A  12      10.720  -2.401  28.524  1.00 41.68           C  \\nATOM    156  HA  VAL A  12      10.173  -3.341  28.591  1.00 41.68           H  \\nATOM    157  C   VAL A  12      10.690  -1.906  27.087  1.00 41.68           C  \\nATOM    158  CB  VAL A  12      10.102  -1.401  29.528  1.00 41.68           C  \\nATOM    159  HB  VAL A  12      10.367  -0.379  29.255  1.00 41.68           H  \\nATOM    160  O   VAL A  12      10.948  -0.734  26.812  1.00 41.68           O  \\nATOM    161  CG1 VAL A  12       8.575  -1.538  29.518  1.00 41.68           C  \\nATOM    162 HG11 VAL A  12       8.198  -1.356  28.511  1.00 41.68           H  \\nATOM    163 HG12 VAL A  12       8.134  -0.800  30.188  1.00 41.68           H  \\nATOM    164 HG13 VAL A  12       8.281  -2.538  29.838  1.00 41.68           H  \\nATOM    165  CG2 VAL A  12      10.582  -1.626  30.970  1.00 41.68           C  \\nATOM    166 HG21 VAL A  12      11.642  -1.388  31.058  1.00 41.68           H  \\nATOM    167 HG22 VAL A  12      10.422  -2.664  31.260  1.00 41.68           H  \\nATOM    168 HG23 VAL A  12      10.034  -0.974  31.650  1.00 41.68           H  \\nATOM    169  N   GLU A  13      10.495  -2.828  26.144  1.00 42.60           N  \\nATOM    170  H   GLU A  13      10.466  -3.801  26.415  1.00 42.60           H  \\nATOM    171  CA  GLU A  13      10.066  -2.469  24.803  1.00 42.60           C  \\nATOM    172  HA  GLU A  13      10.838  -1.881  24.306  1.00 42.60           H  \\nATOM    173  C   GLU A  13       8.820  -1.608  24.997  1.00 42.60           C  \\nATOM    174  CB  GLU A  13       9.750  -3.711  23.951  1.00 42.60           C  \\nATOM    175  HB2 GLU A  13       9.167  -4.416  24.543  1.00 42.60           H  \\nATOM    176  HB3 GLU A  13       9.145  -3.389  23.103  1.00 42.60           H  \\nATOM    177  O   GLU A  13       7.762  -2.098  25.394  1.00 42.60           O  \\nATOM    178  CG  GLU A  13      11.002  -4.408  23.394  1.00 42.60           C  \\nATOM    179  HG2 GLU A  13      11.609  -3.664  22.878  1.00 42.60           H  \\nATOM    180  HG3 GLU A  13      11.586  -4.800  24.227  1.00 42.60           H  \\nATOM    181  CD  GLU A  13      10.658  -5.554  22.420  1.00 42.60           C  \\nATOM    182  OE1 GLU A  13      11.542  -5.907  21.606  1.00 42.60           O  \\nATOM    183  OE2 GLU A  13       9.516  -6.067  22.475  1.00 42.60           O  \\nATOM    184  N   ASN A  14       8.973  -0.297  24.802  1.00 38.11           N  \\nATOM    185  H   ASN A  14       9.914   0.045  24.669  1.00 38.11           H  \\nATOM    186  CA  ASN A  14       7.879   0.660  24.783  1.00 38.11           C  \\nATOM    187  HA  ASN A  14       7.242   0.497  25.651  1.00 38.11           H  \\nATOM    188  C   ASN A  14       7.031   0.379  23.537  1.00 38.11           C  \\nATOM    189  CB  ASN A  14       8.435   2.099  24.828  1.00 38.11           C  \\nATOM    190  HB2 ASN A  14       9.329   2.173  24.209  1.00 38.11           H  \\nATOM    191  HB3 ASN A  14       7.691   2.783  24.420  1.00 38.11           H  \\nATOM    192  O   ASN A  14       7.023   1.144  22.574  1.00 38.11           O  \\nATOM    193  CG  ASN A  14       8.719   2.596  26.233  1.00 38.11           C  \\nATOM    194  ND2 ASN A  14       9.809   3.298  26.433  1.00 38.11           N  \\nATOM    195 HD21 ASN A  14      10.489   3.415  25.696  1.00 38.11           H  \\nATOM    196 HD22 ASN A  14       9.979   3.566  27.392  1.00 38.11           H  \\nATOM    197  OD1 ASN A  14       7.943   2.420  27.154  1.00 38.11           O  \\nATOM    198  N   VAL A  15       6.327  -0.750  23.535  1.00 48.38           N  \\nATOM    199  H   VAL A  15       6.467  -1.387  24.307  1.00 48.38           H  \\nATOM    200  CA  VAL A  15       5.195  -0.975  22.654  1.00 48.38           C  \\nATOM    201  HA  VAL A  15       5.479  -0.737  21.629  1.00 48.38           H  \\nATOM    202  C   VAL A  15       4.148   0.011  23.142  1.00 48.38           C  \\nATOM    203  CB  VAL A  15       4.697  -2.432  22.710  1.00 48.38           C  \\nATOM    204  HB  VAL A  15       4.300  -2.653  23.701  1.00 48.38           H  \\nATOM    205  O   VAL A  15       3.560  -0.181  24.206  1.00 48.38           O  \\nATOM    206  CG1 VAL A  15       3.587  -2.647  21.672  1.00 48.38           C  \\nATOM    207 HG11 VAL A  15       3.969  -2.450  20.670  1.00 48.38           H  \\nATOM    208 HG12 VAL A  15       3.232  -3.676  21.726  1.00 48.38           H  \\nATOM    209 HG13 VAL A  15       2.750  -1.980  21.876  1.00 48.38           H  \\nATOM    210  CG2 VAL A  15       5.824  -3.432  22.415  1.00 48.38           C  \\nATOM    211 HG21 VAL A  15       6.293  -3.207  21.457  1.00 48.38           H  \\nATOM    212 HG22 VAL A  15       5.429  -4.448  22.391  1.00 48.38           H  \\nATOM    213 HG23 VAL A  15       6.582  -3.389  23.197  1.00 48.38           H  \\nATOM    214  N   ALA A  16       3.982   1.114  22.411  1.00 55.85           N  \\nATOM    215  H   ALA A  16       4.594   1.265  21.621  1.00 55.85           H  \\nATOM    216  CA  ALA A  16       2.934   2.087  22.676  1.00 55.85           C  \\nATOM    217  HA  ALA A  16       3.193   2.646  23.575  1.00 55.85           H  \\nATOM    218  C   ALA A  16       1.619   1.329  22.912  1.00 55.85           C  \\nATOM    219  CB  ALA A  16       2.848   3.059  21.493  1.00 55.85           C  \\nATOM    220  HB1 ALA A  16       2.608   2.517  20.578  1.00 55.85           H  \\nATOM    221  HB2 ALA A  16       3.800   3.575  21.363  1.00 55.85           H  \\nATOM    222  HB3 ALA A  16       2.068   3.797  21.680  1.00 55.85           H  \\nATOM    223  O   ALA A  16       1.166   0.566  22.053  1.00 55.85           O  \\nATOM    224  N   LEU A  17       1.071   1.461  24.123  1.00 55.23           N  \\nATOM    225  H   LEU A  17       1.538   2.047  24.800  1.00 55.23           H  \\nATOM    226  CA  LEU A  17      -0.118   0.729  24.536  1.00 55.23           C  \\nATOM    227  HA  LEU A  17       0.104  -0.338  24.490  1.00 55.23           H  \\nATOM    228  C   LEU A  17      -1.253   1.028  23.557  1.00 55.23           C  \\nATOM    229  CB  LEU A  17      -0.509   1.113  25.975  1.00 55.23           C  \\nATOM    230  HB2 LEU A  17      -0.491   2.200  26.052  1.00 55.23           H  \\nATOM    231  HB3 LEU A  17      -1.535   0.794  26.162  1.00 55.23           H  \\nATOM    232  O   LEU A  17      -1.531   2.185  23.237  1.00 55.23           O  \\nATOM    233  CG  LEU A  17       0.398   0.507  27.062  1.00 55.23           C  \\nATOM    234  HG  LEU A  17       1.444   0.630  26.779  1.00 55.23           H  \\nATOM    235  CD1 LEU A  17       0.171   1.230  28.388  1.00 55.23           C  \\nATOM    236 HD11 LEU A  17      -0.869   1.128  28.701  1.00 55.23           H  \\nATOM    237 HD12 LEU A  17       0.412   2.288  28.277  1.00 55.23           H  \\nATOM    238 HD13 LEU A  17       0.822   0.808  29.153  1.00 55.23           H  \\nATOM    239  CD2 LEU A  17       0.102  -0.981  27.278  1.00 55.23           C  \\nATOM    240 HD21 LEU A  17       0.326  -1.539  26.369  1.00 55.23           H  \\nATOM    241 HD22 LEU A  17      -0.943  -1.122  27.555  1.00 55.23           H  \\nATOM    242 HD23 LEU A  17       0.741  -1.363  28.074  1.00 55.23           H  \\nATOM    243  N   LYS A  18      -1.919  -0.034  23.103  1.00 69.25           N  \\nATOM    244  H   LYS A  18      -1.588  -0.948  23.379  1.00 69.25           H  \\nATOM    245  CA  LYS A  18      -3.141   0.076  22.310  1.00 69.25           C  \\nATOM    246  HA  LYS A  18      -2.900   0.530  21.349  1.00 69.25           H  \\nATOM    247  C   LYS A  18      -4.152   0.951  23.030  1.00 69.25           C  \\nATOM    248  CB  LYS A  18      -3.775  -1.293  22.106  1.00 69.25           C  \\nATOM    249  HB2 LYS A  18      -4.759  -1.165  21.653  1.00 69.25           H  \\nATOM    250  HB3 LYS A  18      -3.902  -1.772  23.077  1.00 69.25           H  \\nATOM    251  O   LYS A  18      -4.375   0.783  24.228  1.00 69.25           O  \\nATOM    252  CG  LYS A  18      -2.925  -2.172  21.202  1.00 69.25           C  \\nATOM    253  HG2 LYS A  18      -1.907  -2.235  21.587  1.00 69.25           H  \\nATOM    254  HG3 LYS A  18      -2.900  -1.726  20.207  1.00 69.25           H  \\nATOM    255  CD  LYS A  18      -3.566  -3.556  21.126  1.00 69.25           C  \\nATOM    256  HD2 LYS A  18      -4.440  -3.617  21.774  1.00 69.25           H  \\nATOM    257  HD3 LYS A  18      -3.893  -3.738  20.102  1.00 69.25           H  \\nATOM    258  CE  LYS A  18      -2.546  -4.600  21.543  1.00 69.25           C  \\nATOM    259  HE2 LYS A  18      -1.741  -4.618  20.808  1.00 69.25           H  \\nATOM    260  HE3 LYS A  18      -2.104  -4.320  22.499  1.00 69.25           H  \\nATOM    261  NZ  LYS A  18      -3.217  -5.911  21.642  1.00 69.25           N  \\nATOM    262  HZ1 LYS A  18      -3.666  -6.003  22.542  1.00 69.25           H  \\nATOM    263  HZ2 LYS A  18      -3.911  -5.990  20.913  1.00 69.25           H  \\nATOM    264  HZ3 LYS A  18      -2.517  -6.629  21.514  1.00 69.25           H  \\nATOM    265  N   THR A  19      -4.825   1.806  22.274  1.00 85.36           N  \\nATOM    266  H   THR A  19      -4.600   1.847  21.290  1.00 85.36           H  \\nATOM    267  CA  THR A  19      -6.026   2.502  22.746  1.00 85.36           C  \\nATOM    268  HA  THR A  19      -6.276   2.147  23.746  1.00 85.36           H  \\nATOM    269  C   THR A  19      -7.215   2.149  21.861  1.00 85.36           C  \\nATOM    270  CB  THR A  19      -5.812   4.018  22.863  1.00 85.36           C  \\nATOM    271  HB  THR A  19      -6.740   4.479  23.199  1.00 85.36           H  \\nATOM    272  O   THR A  19      -7.031   1.762  20.710  1.00 85.36           O  \\nATOM    273  CG2 THR A  19      -4.705   4.368  23.858  1.00 85.36           C  \\nATOM    274 HG21 THR A  19      -4.936   3.930  24.829  1.00 85.36           H  \\nATOM    275 HG22 THR A  19      -3.743   3.986  23.518  1.00 85.36           H  \\nATOM    276 HG23 THR A  19      -4.640   5.450  23.967  1.00 85.36           H  \\nATOM    277  OG1 THR A  19      -5.440   4.589  21.634  1.00 85.36           O  \\nATOM    278  HG1 THR A  19      -4.545   4.288  21.461  1.00 85.36           H  \\nATOM    279  N   GLY A  20      -8.421   2.258  22.413  1.00 90.73           N  \\nATOM    280  H   GLY A  20      -8.488   2.558  23.376  1.00 90.73           H  \\nATOM    281  CA  GLY A  20      -9.677   1.904  21.750  1.00 90.73           C  \\nATOM    282  HA2 GLY A  20     -10.473   2.452  22.253  1.00 90.73           H  \\nATOM    283  HA3 GLY A  20      -9.665   2.217  20.706  1.00 90.73           H  \\nATOM    284  C   GLY A  20     -10.029   0.415  21.817  1.00 90.73           C  \\nATOM    285  O   GLY A  20      -9.157  -0.452  21.910  1.00 90.73           O  \\nATOM    286  N   SER A  21     -11.329   0.118  21.816  1.00 93.49           N  \\nATOM    287  H   SER A  21     -11.994   0.877  21.762  1.00 93.49           H  \\nATOM    288  CA  SER A  21     -11.857  -1.249  21.947  1.00 93.49           C  \\nATOM    289  HA  SER A  21     -11.307  -1.764  22.735  1.00 93.49           H  \\nATOM    290  C   SER A  21     -11.710  -2.081  20.672  1.00 93.49           C  \\nATOM    291  CB  SER A  21     -13.337  -1.208  22.347  1.00 93.49           C  \\nATOM    292  HB2 SER A  21     -13.699  -2.228  22.477  1.00 93.49           H  \\nATOM    293  HB3 SER A  21     -13.449  -0.672  23.290  1.00 93.49           H  \\nATOM    294  O   SER A  21     -11.608  -3.305  20.738  1.00 93.49           O  \\nATOM    295  OG  SER A  21     -14.104  -0.566  21.348  1.00 93.49           O  \\nATOM    296  HG  SER A  21     -13.995   0.377  21.492  1.00 93.49           H  \\nATOM    297  N   GLY A  22     -11.710  -1.433  19.505  1.00 95.33           N  \\nATOM    298  H   GLY A  22     -11.842  -0.432  19.525  1.00 95.33           H  \\nATOM    299  CA  GLY A  22     -11.819  -2.096  18.208  1.00 95.33           C  \\nATOM    300  HA2 GLY A  22     -11.172  -2.973  18.200  1.00 95.33           H  \\nATOM    301  HA3 GLY A  22     -11.496  -1.405  17.429  1.00 95.33           H  \\nATOM    302  C   GLY A  22     -13.228  -2.569  17.861  1.00 95.33           C  \\nATOM    303  O   GLY A  22     -13.411  -3.198  16.818  1.00 95.33           O  \\nATOM    304  N   ASP A  23     -14.225  -2.278  18.698  1.00 94.12           N  \\nATOM    305  H   ASP A  23     -14.011  -1.770  19.544  1.00 94.12           H  \\nATOM    306  CA  ASP A  23     -15.595  -2.762  18.515  1.00 94.12           C  \\nATOM    307  HA  ASP A  23     -15.565  -3.794  18.167  1.00 94.12           H  \\nATOM    308  C   ASP A  23     -16.373  -1.975  17.459  1.00 94.12           C  \\nATOM    309  CB  ASP A  23     -16.335  -2.706  19.858  1.00 94.12           C  \\nATOM    310  HB2 ASP A  23     -17.404  -2.778  19.661  1.00 94.12           H  \\nATOM    311  HB3 ASP A  23     -16.163  -1.746  20.345  1.00 94.12           H  \\nATOM    312  O   ASP A  23     -17.347  -2.477  16.896  1.00 94.12           O  \\nATOM    313  CG  ASP A  23     -15.981  -3.857  20.799  1.00 94.12           C  \\nATOM    314  OD1 ASP A  23     -15.345  -4.838  20.350  1.00 94.12           O  \\nATOM    315  OD2 ASP A  23     -16.635  -3.965  21.854  1.00 94.12           O  \\nATOM    316  N   SER A  24     -15.940  -0.749  17.169  1.00 95.40           N  \\nATOM    317  H   SER A  24     -15.090  -0.429  17.610  1.00 95.40           H  \\nATOM    318  CA  SER A  24     -16.583   0.135  16.202  1.00 95.40           C  \\nATOM    319  HA  SER A  24     -17.167  -0.466  15.505  1.00 95.40           H  \\nATOM    320  C   SER A  24     -15.552   0.910  15.392  1.00 95.40           C  \\nATOM    321  CB  SER A  24     -17.541   1.095  16.917  1.00 95.40           C  \\nATOM    322  HB2 SER A  24     -18.212   0.531  17.565  1.00 95.40           H  \\nATOM    323  HB3 SER A  24     -18.141   1.619  16.173  1.00 95.40           H  \\nATOM    324  O   SER A  24     -14.416   1.105  15.818  1.00 95.40           O  \\nATOM    325  OG  SER A  24     -16.829   2.053  17.678  1.00 95.40           O  \\nATOM    326  HG  SER A  24     -16.436   1.610  18.433  1.00 95.40           H  \\nATOM    327  N   GLN A  25     -15.972   1.416  14.235  1.00 96.15           N  \\nATOM    328  H   GLN A  25     -16.903   1.196  13.910  1.00 96.15           H  \\nATOM    329  CA  GLN A  25     -15.136   2.283  13.409  1.00 96.15           C  \\nATOM    330  HA  GLN A  25     -14.209   1.757  13.180  1.00 96.15           H  \\nATOM    331  C   GLN A  25     -14.764   3.597  14.120  1.00 96.15           C  \\nATOM    332  CB  GLN A  25     -15.892   2.550  12.104  1.00 96.15           C  \\nATOM    333  HB2 GLN A  25     -16.197   1.596  11.674  1.00 96.15           H  \\nATOM    334  HB3 GLN A  25     -16.787   3.136  12.315  1.00 96.15           H  \\nATOM    335  O   GLN A  25     -13.702   4.152  13.857  1.00 96.15           O  \\nATOM    336  CG  GLN A  25     -15.030   3.290  11.074  1.00 96.15           C  \\nATOM    337  HG2 GLN A  25     -14.754   4.273  11.457  1.00 96.15           H  \\nATOM    338  HG3 GLN A  25     -14.119   2.721  10.894  1.00 96.15           H  \\nATOM    339  CD  GLN A  25     -15.740   3.493   9.743  1.00 96.15           C  \\nATOM    340  NE2 GLN A  25     -15.088   4.114   8.786  1.00 96.15           N  \\nATOM    341 HE21 GLN A  25     -14.149   4.453   8.942  1.00 96.15           H  \\nATOM    342 HE22 GLN A  25     -15.604   4.270   7.932  1.00 96.15           H  \\nATOM    343  OE1 GLN A  25     -16.879   3.114   9.526  1.00 96.15           O  \\nATOM    344  N   SER A  26     -15.617   4.095  15.023  1.00 96.88           N  \\nATOM    345  H   SER A  26     -16.479   3.603  15.210  1.00 96.88           H  \\nATOM    346  CA  SER A  26     -15.353   5.292  15.836  1.00 96.88           C  \\nATOM    347  HA  SER A  26     -14.879   6.047  15.210  1.00 96.88           H  \\nATOM    348  C   SER A  26     -14.406   5.041  17.013  1.00 96.88           C  \\nATOM    349  CB  SER A  26     -16.670   5.875  16.361  1.00 96.88           C  \\nATOM    350  HB2 SER A  26     -17.186   6.370  15.538  1.00 96.88           H  \\nATOM    351  HB3 SER A  26     -16.466   6.617  17.132  1.00 96.88           H  \\nATOM    352  O   SER A  26     -13.843   5.995  17.543  1.00 96.88           O  \\nATOM    353  OG  SER A  26     -17.519   4.865  16.878  1.00 96.88           O  \\nATOM    354  HG  SER A  26     -17.069   4.362  17.561  1.00 96.88           H  \\nATOM    355  N   ASP A  27     -14.216   3.780  17.406  1.00 96.80           N  \\nATOM    356  H   ASP A  27     -14.675   3.039  16.896  1.00 96.80           H  \\nATOM    357  CA  ASP A  27     -13.320   3.372  18.490  1.00 96.80           C  \\nATOM    358  HA  ASP A  27     -12.660   4.204  18.736  1.00 96.80           H  \\nATOM    359  C   ASP A  27     -12.410   2.204  18.054  1.00 96.80           C  \\nATOM    360  CB  ASP A  27     -14.153   3.068  19.745  1.00 96.80           C  \\nATOM    361  HB2 ASP A  27     -14.784   2.197  19.565  1.00 96.80           H  \\nATOM    362  HB3 ASP A  27     -14.797   3.919  19.962  1.00 96.80           H  \\nATOM    363  O   ASP A  27     -12.517   1.094  18.584  1.00 96.80           O  \\nATOM    364  CG  ASP A  27     -13.254   2.818  20.954  1.00 96.80           C  \\nATOM    365  OD1 ASP A  27     -12.219   3.505  21.070  1.00 96.80           O  \\nATOM    366  OD2 ASP A  27     -13.574   1.905  21.747  1.00 96.80           O  \\nATOM    367  N   PRO A  28     -11.549   2.405  17.036  1.00 97.78           N  \\nATOM    368  CA  PRO A  28     -10.664   1.361  16.538  1.00 97.78           C  \\nATOM    369  HA  PRO A  28     -11.220   0.437  16.381  1.00 97.78           H  \\nATOM    370  C   PRO A  28      -9.506   1.111  17.509  1.00 97.78           C  \\nATOM    371  CB  PRO A  28     -10.160   1.888  15.195  1.00 97.78           C  \\nATOM    372  HB2 PRO A  28     -10.900   1.662  14.427  1.00 97.78           H  \\nATOM    373  HB3 PRO A  28      -9.186   1.480  14.925  1.00 97.78           H  \\nATOM    374  O   PRO A  28      -9.089   2.020  18.226  1.00 97.78           O  \\nATOM    375  CG  PRO A  28     -10.097   3.399  15.415  1.00 97.78           C  \\nATOM    376  HG2 PRO A  28     -10.182   3.949  14.478  1.00 97.78           H  \\nATOM    377  HG3 PRO A  28      -9.173   3.660  15.930  1.00 97.78           H  \\nATOM    378  CD  PRO A  28     -11.293   3.658  16.332  1.00 97.78           C  \\nATOM    379  HD2 PRO A  28     -12.160   3.918  15.725  1.00 97.78           H  \\nATOM    380  HD3 PRO A  28     -11.067   4.465  17.030  1.00 97.78           H  \\nATOM    381  N   ILE A  29      -8.902  -0.079  17.450  1.00 97.52           N  \\nATOM    382  H   ILE A  29      -9.287  -0.794  16.849  1.00 97.52           H  \\nATOM    383  CA  ILE A  29      -7.587  -0.307  18.062  1.00 97.52           C  \\nATOM    384  HA  ILE A  29      -7.646  -0.029  19.114  1.00 97.52           H  \\nATOM    385  C   ILE A  29      -6.569   0.598  17.369  1.00 97.52           C  \\nATOM    386  CB  ILE A  29      -7.143  -1.784  17.981  1.00 97.52           C  \\nATOM    387  HB  ILE A  29      -7.152  -2.101  16.938  1.00 97.52           H  \\nATOM    388  O   ILE A  29      -6.280   0.423  16.185  1.00 97.52           O  \\nATOM    389  CG1 ILE A  29      -8.135  -2.666  18.765  1.00 97.52           C  \\nATOM    390 HG12 ILE A  29      -9.092  -2.629  18.243  1.00 97.52           H  \\nATOM    391 HG13 ILE A  29      -8.278  -2.263  19.768  1.00 97.52           H  \\nATOM    392  CG2 ILE A  29      -5.707  -1.947  18.519  1.00 97.52           C  \\nATOM    393 HG21 ILE A  29      -5.394  -2.989  18.442  1.00 97.52           H  \\nATOM    394 HG22 ILE A  29      -4.998  -1.358  17.937  1.00 97.52           H  \\nATOM    395 HG23 ILE A  29      -5.670  -1.628  19.561  1.00 97.52           H  \\nATOM    396  CD1 ILE A  29      -7.738  -4.140  18.881  1.00 97.52           C  \\nATOM    397 HD11 ILE A  29      -7.411  -4.520  17.913  1.00 97.52           H  \\nATOM    398 HD12 ILE A  29      -6.933  -4.247  19.609  1.00 97.52           H  \\nATOM    399 HD13 ILE A  29      -8.593  -4.716  19.236  1.00 97.52           H  \\nATOM    400  N   LYS A  30      -5.999   1.538  18.111  1.00 97.27           N  \\nATOM    401  H   LYS A  30      -6.349   1.647  19.052  1.00 97.27           H  \\nATOM    402  CA  LYS A  30      -4.934   2.435  17.663  1.00 97.27           C  \\nATOM    403  HA  LYS A  30      -4.919   2.491  16.574  1.00 97.27           H  \\nATOM    404  C   LYS A  30      -3.596   1.902  18.137  1.00 97.27           C  \\nATOM    405  CB  LYS A  30      -5.195   3.830  18.227  1.00 97.27           C  \\nATOM    406  HB2 LYS A  30      -5.355   3.736  19.301  1.00 97.27           H  \\nATOM    407  HB3 LYS A  30      -4.322   4.462  18.063  1.00 97.27           H  \\nATOM    408  O   LYS A  30      -3.406   1.724  19.341  1.00 97.27           O  \\nATOM    409  CG  LYS A  30      -6.423   4.476  17.584  1.00 97.27           C  \\nATOM    410  HG2 LYS A  30      -6.220   4.674  16.531  1.00 97.27           H  \\nATOM    411  HG3 LYS A  30      -7.278   3.804  17.659  1.00 97.27           H  \\nATOM    412  CD  LYS A  30      -6.767   5.783  18.294  1.00 97.27           C  \\nATOM    413  HD2 LYS A  30      -5.956   6.499  18.161  1.00 97.27           H  \\nATOM    414  HD3 LYS A  30      -6.908   5.583  19.356  1.00 97.27           H  \\nATOM    415  CE  LYS A  30      -8.065   6.325  17.697  1.00 97.27           C  \\nATOM    416  HE2 LYS A  30      -8.818   5.538  17.754  1.00 97.27           H  \\nATOM    417  HE3 LYS A  30      -7.902   6.549  16.643  1.00 97.27           H  \\nATOM    418  NZ  LYS A  30      -8.533   7.524  18.429  1.00 97.27           N  \\nATOM    419  HZ1 LYS A  30      -8.703   7.280  19.395  1.00 97.27           H  \\nATOM    420  HZ2 LYS A  30      -7.841   8.259  18.390  1.00 97.27           H  \\nATOM    421  HZ3 LYS A  30      -9.403   7.851  18.035  1.00 97.27           H  \\nATOM    422  N   ALA A  31      -2.695   1.642  17.199  1.00 96.27           N  \\nATOM    423  H   ALA A  31      -2.926   1.820  16.232  1.00 96.27           H  \\nATOM    424  CA  ALA A  31      -1.361   1.146  17.496  1.00 96.27           C  \\nATOM    425  HA  ALA A  31      -1.033   1.548  18.455  1.00 96.27           H  \\nATOM    426  C   ALA A  31      -0.352   1.594  16.438  1.00 96.27           C  \\nATOM    427  CB  ALA A  31      -1.413  -0.382  17.598  1.00 96.27           C  \\nATOM    428  HB1 ALA A  31      -2.059  -0.668  18.428  1.00 96.27           H  \\nATOM    429  HB2 ALA A  31      -1.812  -0.805  16.677  1.00 96.27           H  \\nATOM    430  HB3 ALA A  31      -0.413  -0.777  17.778  1.00 96.27           H  \\nATOM    431  O   ALA A  31      -0.685   1.752  15.261  1.00 96.27           O  \\nATOM    432  N   ASP A  32       0.898   1.738  16.860  1.00 95.56           N  \\nATOM    433  H   ASP A  32       1.108   1.588  17.837  1.00 95.56           H  \\nATOM    434  CA  ASP A  32       2.011   2.000  15.958  1.00 95.56           C  \\nATOM    435  HA  ASP A  32       1.672   2.669  15.167  1.00 95.56           H  \\nATOM    436  C   ASP A  32       2.478   0.702  15.281  1.00 95.56           C  \\nATOM    437  CB  ASP A  32       3.143   2.713  16.713  1.00 95.56           C  \\nATOM    438  HB2 ASP A  32       4.022   2.742  16.070  1.00 95.56           H  \\nATOM    439  HB3 ASP A  32       3.399   2.139  17.604  1.00 95.56           H  \\nATOM    440  O   ASP A  32       2.321  -0.402  15.812  1.00 95.56           O  \\nATOM    441  CG  ASP A  32       2.810   4.158  17.115  1.00 95.56           C  \\nATOM    442  OD1 ASP A  32       1.749   4.684  16.695  1.00 95.56           O  \\nATOM    443  OD2 ASP A  32       3.673   4.776  17.775  1.00 95.56           O  \\nATOM    444  N   LEU A  33       3.061   0.840  14.092  1.00 97.05           N  \\nATOM    445  H   LEU A  33       3.151   1.772  13.714  1.00 97.05           H  \\nATOM    446  CA  LEU A  33       3.641  -0.256  13.325  1.00 97.05           C  \\nATOM    447  HA  LEU A  33       3.576  -1.173  13.911  1.00 97.05           H  \\nATOM    448  C   LEU A  33       5.123   0.024  13.073  1.00 97.05           C  \\nATOM    449  CB  LEU A  33       2.809  -0.465  12.047  1.00 97.05           C  \\nATOM    450  HB2 LEU A  33       1.778  -0.630  12.360  1.00 97.05           H  \\nATOM    451  HB3 LEU A  33       2.833   0.441  11.442  1.00 97.05           H  \\nATOM    452  O   LEU A  33       5.487   1.019  12.447  1.00 97.05           O  \\nATOM    453  CG  LEU A  33       3.267  -1.667  11.198  1.00 97.05           C  \\nATOM    454  HG  LEU A  33       3.678  -2.440  11.848  1.00 97.05           H  \\nATOM    455  CD1 LEU A  33       2.086  -2.271  10.434  1.00 97.05           C  \\nATOM    456 HD11 LEU A  33       1.284  -2.529  11.126  1.00 97.05           H  \\nATOM    457 HD12 LEU A  33       2.409  -3.175   9.916  1.00 97.05           H  \\nATOM    458 HD13 LEU A  33       1.721  -1.556   9.698  1.00 97.05           H  \\nATOM    459  CD2 LEU A  33       4.311  -1.266  10.155  1.00 97.05           C  \\nATOM    460 HD21 LEU A  33       3.907  -0.493   9.501  1.00 97.05           H  \\nATOM    461 HD22 LEU A  33       4.575  -2.128   9.542  1.00 97.05           H  \\nATOM    462 HD23 LEU A  33       5.221  -0.897  10.629  1.00 97.05           H  \\nATOM    463  N   GLU A  34       5.981  -0.880  13.536  1.00 97.46           N  \\nATOM    464  H   GLU A  34       5.631  -1.667  14.063  1.00 97.46           H  \\nATOM    465  CA  GLU A  34       7.415  -0.824  13.267  1.00 97.46           C  \\nATOM    466  HA  GLU A  34       7.738   0.217  13.277  1.00 97.46           H  \\nATOM    467  C   GLU A  34       7.719  -1.410  11.881  1.00 97.46           C  \\nATOM    468  CB  GLU A  34       8.175  -1.557  14.378  1.00 97.46           C  \\nATOM    469  HB2 GLU A  34       7.810  -1.226  15.350  1.00 97.46           H  \\nATOM    470  HB3 GLU A  34       7.983  -2.625  14.279  1.00 97.46           H  \\nATOM    471  O   GLU A  34       7.550  -2.606  11.634  1.00 97.46           O  \\nATOM    472  CG  GLU A  34       9.683  -1.277  14.300  1.00 97.46           C  \\nATOM    473  HG2 GLU A  34      10.009  -1.370  13.264  1.00 97.46           H  \\nATOM    474  HG3 GLU A  34       9.868  -0.250  14.616  1.00 97.46           H  \\nATOM    475  CD  GLU A  34      10.522  -2.243  15.148  1.00 97.46           C  \\nATOM    476  OE1 GLU A  34      11.758  -2.199  14.982  1.00 97.46           O  \\nATOM    477  OE2 GLU A  34       9.960  -3.165  15.789  1.00 97.46           O  \\nATOM    478  N   VAL A  35       8.178  -0.559  10.965  1.00 98.17           N  \\nATOM    479  H   VAL A  35       8.313   0.406  11.231  1.00 98.17           H  \\nATOM    480  CA  VAL A  35       8.574  -0.938   9.608  1.00 98.17           C  \\nATOM    481  HA  VAL A  35       7.866  -1.674   9.227  1.00 98.17           H  \\nATOM    482  C   VAL A  35       9.966  -1.564   9.637  1.00 98.17           C  \\nATOM    483  CB  VAL A  35       8.539   0.288   8.681  1.00 98.17           C  \\nATOM    484  HB  VAL A  35       9.202   1.059   9.074  1.00 98.17           H  \\nATOM    485  O   VAL A  35      10.909  -0.984  10.180  1.00 98.17           O  \\nATOM    486  CG1 VAL A  35       9.021  -0.041   7.271  1.00 98.17           C  \\nATOM    487 HG11 VAL A  35      10.065  -0.354   7.279  1.00 98.17           H  \\nATOM    488 HG12 VAL A  35       8.956   0.870   6.676  1.00 98.17           H  \\nATOM    489 HG13 VAL A  35       8.406  -0.825   6.827  1.00 98.17           H  \\nATOM    490  CG2 VAL A  35       7.121   0.865   8.578  1.00 98.17           C  \\nATOM    491 HG21 VAL A  35       7.109   1.710   7.890  1.00 98.17           H  \\nATOM    492 HG22 VAL A  35       6.788   1.214   9.556  1.00 98.17           H  \\nATOM    493 HG23 VAL A  35       6.430   0.101   8.220  1.00 98.17           H  \\nATOM    494  N   LYS A  36      10.114  -2.736   9.007  1.00 95.80           N  \\nATOM    495  H   LYS A  36       9.303  -3.120   8.543  1.00 95.80           H  \\nATOM    496  CA  LYS A  36      11.365  -3.515   9.003  1.00 95.80           C  \\nATOM    497  HA  LYS A  36      12.149  -2.914   9.462  1.00 95.80           H  \\nATOM    498  C   LYS A  36      11.825  -3.847   7.590  1.00 95.80           C  \\nATOM    499  CB  LYS A  36      11.212  -4.793   9.854  1.00 95.80           C  \\nATOM    500  HB2 LYS A  36      10.402  -5.406   9.458  1.00 95.80           H  \\nATOM    501  HB3 LYS A  36      12.139  -5.364   9.808  1.00 95.80           H  \\nATOM    502  O   LYS A  36      11.052  -4.370   6.787  1.00 95.80           O  \\nATOM    503  CG  LYS A  36      10.921  -4.440  11.317  1.00 95.80           C  \\nATOM    504  HG2 LYS A  36      11.686  -3.742  11.656  1.00 95.80           H  \\nATOM    505  HG3 LYS A  36       9.957  -3.933  11.365  1.00 95.80           H  \\nATOM    506  CD  LYS A  36      10.882  -5.633  12.286  1.00 95.80           C  \\nATOM    507  HD2 LYS A  36      11.855  -6.123  12.324  1.00 95.80           H  \\nATOM    508  HD3 LYS A  36      10.097  -6.334  12.004  1.00 95.80           H  \\nATOM    509  CE  LYS A  36      10.559  -4.957  13.621  1.00 95.80           C  \\nATOM    510  HE2 LYS A  36       9.591  -4.462  13.538  1.00 95.80           H  \\nATOM    511  HE3 LYS A  36      11.286  -4.160  13.770  1.00 95.80           H  \\nATOM    512  NZ  LYS A  36      10.566  -5.789  14.843  1.00 95.80           N  \\nATOM    513  HZ1 LYS A  36       9.846  -6.496  14.851  1.00 95.80           H  \\nATOM    514  HZ2 LYS A  36      10.367  -5.132  15.584  1.00 95.80           H  \\nATOM    515  HZ3 LYS A  36      11.484  -6.160  15.045  1.00 95.80           H  \\nATOM    516  N   GLY A  37      13.103  -3.604   7.307  1.00 92.85           N  \\nATOM    517  H   GLY A  37      13.660  -3.127   8.001  1.00 92.85           H  \\nATOM    518  CA  GLY A  37      13.760  -4.027   6.069  1.00 92.85           C  \\nATOM    519  HA2 GLY A  37      13.884  -5.110   6.091  1.00 92.85           H  \\nATOM    520  HA3 GLY A  37      14.751  -3.575   6.014  1.00 92.85           H  \\nATOM    521  C   GLY A  37      12.993  -3.631   4.803  1.00 92.85           C  \\nATOM    522  O   GLY A  37      12.586  -2.485   4.631  1.00 92.85           O  \\nATOM    523  N   GLN A  38      12.769  -4.599   3.913  1.00 95.67           N  \\nATOM    524  H   GLN A  38      13.084  -5.535   4.122  1.00 95.67           H  \\nATOM    525  CA  GLN A  38      12.152  -4.351   2.607  1.00 95.67           C  \\nATOM    526  HA  GLN A  38      12.588  -3.437   2.206  1.00 95.67           H  \\nATOM    527  C   GLN A  38      10.639  -4.079   2.651  1.00 95.67           C  \\nATOM    528  CB  GLN A  38      12.509  -5.489   1.640  1.00 95.67           C  \\nATOM    529  HB2 GLN A  38      11.923  -5.382   0.727  1.00 95.67           H  \\nATOM    530  HB3 GLN A  38      12.271  -6.453   2.090  1.00 95.67           H  \\nATOM    531  O   GLN A  38      10.088  -3.691   1.618  1.00 95.67           O  \\nATOM    532  CG  GLN A  38      13.995  -5.427   1.254  1.00 95.67           C  \\nATOM    533  HG2 GLN A  38      14.621  -5.642   2.120  1.00 95.67           H  \\nATOM    534  HG3 GLN A  38      14.230  -4.421   0.905  1.00 95.67           H  \\nATOM    535  CD  GLN A  38      14.345  -6.411   0.148  1.00 95.67           C  \\nATOM    536  NE2 GLN A  38      15.236  -6.074  -0.758  1.00 95.67           N  \\nATOM    537 HE21 GLN A  38      15.770  -5.220  -0.686  1.00 95.67           H  \\nATOM    538 HE22 GLN A  38      15.498  -6.795  -1.415  1.00 95.67           H  \\nATOM    539  OE1 GLN A  38      13.834  -7.512   0.082  1.00 95.67           O  \\nATOM    540  N   SER A  39       9.975  -4.203   3.810  1.00 97.70           N  \\nATOM    541  H   SER A  39      10.465  -4.547   4.623  1.00 97.70           H  \\nATOM    542  CA  SER A  39       8.582  -3.751   3.979  1.00 97.70           C  \\nATOM    543  HA  SER A  39       7.986  -4.181   3.174  1.00 97.70           H  \\nATOM    544  C   SER A  39       8.425  -2.228   3.882  1.00 97.70           C  \\nATOM    545  CB  SER A  39       7.973  -4.238   5.298  1.00 97.70           C  \\nATOM    546  HB2 SER A  39       6.911  -3.994   5.294  1.00 97.70           H  \\nATOM    547  HB3 SER A  39       8.070  -5.321   5.373  1.00 97.70           H  \\nATOM    548  O   SER A  39       7.309  -1.744   3.721  1.00 97.70           O  \\nATOM    549  OG  SER A  39       8.579  -3.633   6.425  1.00 97.70           O  \\nATOM    550  HG  SER A  39       9.470  -3.983   6.502  1.00 97.70           H  \\nATOM    551  N   ALA A  40       9.528  -1.470   3.924  1.00 98.30           N  \\nATOM    552  H   ALA A  40      10.416  -1.911   4.118  1.00 98.30           H  \\nATOM    553  CA  ALA A  40       9.503  -0.014   3.836  1.00 98.30           C  \\nATOM    554  HA  ALA A  40       8.927   0.376   4.675  1.00 98.30           H  \\nATOM    555  C   ALA A  40       8.805   0.511   2.579  1.00 98.30           C  \\nATOM    556  CB  ALA A  40      10.942   0.501   3.938  1.00 98.30           C  \\nATOM    557  HB1 ALA A  40      11.536   0.147   3.096  1.00 98.30           H  \\nATOM    558  HB2 ALA A  40      11.395   0.152   4.866  1.00 98.30           H  \\nATOM    559  HB3 ALA A  40      10.940   1.591   3.942  1.00 98.30           H  \\nATOM    560  O   ALA A  40       7.970   1.399   2.688  1.00 98.30           O  \\nATOM    561  N   LEU A  41       9.103  -0.056   1.407  1.00 98.54           N  \\nATOM    562  H   LEU A  41       9.761  -0.821   1.402  1.00 98.54           H  \\nATOM    563  CA  LEU A  41       8.507   0.378   0.142  1.00 98.54           C  \\nATOM    564  HA  LEU A  41       8.696   1.446   0.037  1.00 98.54           H  \\nATOM    565  C   LEU A  41       6.973   0.222   0.113  1.00 98.54           C  \\nATOM    566  CB  LEU A  41       9.222  -0.334  -1.021  1.00 98.54           C  \\nATOM    567  HB2 LEU A  41      10.237   0.056  -1.095  1.00 98.54           H  \\nATOM    568  HB3 LEU A  41       9.285  -1.398  -0.792  1.00 98.54           H  \\nATOM    569  O   LEU A  41       6.300   1.216  -0.149  1.00 98.54           O  \\nATOM    570  CG  LEU A  41       8.526  -0.184  -2.384  1.00 98.54           C  \\nATOM    571  HG  LEU A  41       7.534  -0.628  -2.305  1.00 98.54           H  \\nATOM    572  CD1 LEU A  41       8.378   1.255  -2.869  1.00 98.54           C  \\nATOM    573 HD11 LEU A  41       7.768   1.270  -3.773  1.00 98.54           H  \\nATOM    574 HD12 LEU A  41       9.358   1.672  -3.100  1.00 98.54           H  \\nATOM    575 HD13 LEU A  41       7.889   1.875  -2.118  1.00 98.54           H  \\nATOM    576  CD2 LEU A  41       9.314  -0.953  -3.444  1.00 98.54           C  \\nATOM    577 HD21 LEU A  41       9.367  -2.007  -3.170  1.00 98.54           H  \\nATOM    578 HD22 LEU A  41       8.817  -0.852  -4.409  1.00 98.54           H  \\nATOM    579 HD23 LEU A  41      10.319  -0.537  -3.517  1.00 98.54           H  \\nATOM    580  N   PRO A  42       6.392  -0.968   0.361  1.00 98.55           N  \\nATOM    581  CA  PRO A  42       4.945  -1.127   0.300  1.00 98.55           C  \\nATOM    582  HA  PRO A  42       4.584  -0.780  -0.669  1.00 98.55           H  \\nATOM    583  C   PRO A  42       4.224  -0.308   1.375  1.00 98.55           C  \\nATOM    584  CB  PRO A  42       4.695  -2.630   0.401  1.00 98.55           C  \\nATOM    585  HB2 PRO A  42       4.639  -3.051  -0.603  1.00 98.55           H  \\nATOM    586  HB3 PRO A  42       3.790  -2.857   0.965  1.00 98.55           H  \\nATOM    587  O   PRO A  42       3.261   0.376   1.052  1.00 98.55           O  \\nATOM    588  CG  PRO A  42       5.932  -3.173   1.092  1.00 98.55           C  \\nATOM    589  HG2 PRO A  42       6.107  -4.219   0.838  1.00 98.55           H  \\nATOM    590  HG3 PRO A  42       5.829  -3.041   2.169  1.00 98.55           H  \\nATOM    591  CD  PRO A  42       7.023  -2.261   0.545  1.00 98.55           C  \\nATOM    592  HD2 PRO A  42       7.338  -2.613  -0.437  1.00 98.55           H  \\nATOM    593  HD3 PRO A  42       7.869  -2.232   1.232  1.00 98.55           H  \\nATOM    594  N   PHE A  43       4.748  -0.230   2.606  1.00 98.63           N  \\nATOM    595  H   PHE A  43       5.562  -0.775   2.852  1.00 98.63           H  \\nATOM    596  CA  PHE A  43       4.164   0.680   3.597  1.00 98.63           C  \\nATOM    597  HA  PHE A  43       3.101   0.453   3.679  1.00 98.63           H  \\nATOM    598  C   PHE A  43       4.227   2.147   3.162  1.00 98.63           C  \\nATOM    599  CB  PHE A  43       4.813   0.485   4.972  1.00 98.63           C  \\nATOM    600  HB2 PHE A  43       4.521   1.319   5.610  1.00 98.63           H  \\nATOM    601  HB3 PHE A  43       5.897   0.521   4.864  1.00 98.63           H  \\nATOM    602  O   PHE A  43       3.289   2.895   3.413  1.00 98.63           O  \\nATOM    603  CG  PHE A  43       4.397  -0.787   5.688  1.00 98.63           C  \\nATOM    604  CD1 PHE A  43       3.034  -1.085   5.875  1.00 98.63           C  \\nATOM    605  HD1 PHE A  43       2.273  -0.430   5.478  1.00 98.63           H  \\nATOM    606  CD2 PHE A  43       5.370  -1.662   6.202  1.00 98.63           C  \\nATOM    607  HD2 PHE A  43       6.418  -1.439   6.066  1.00 98.63           H  \\nATOM    608  CE1 PHE A  43       2.650  -2.242   6.573  1.00 98.63           C  \\nATOM    609  HE1 PHE A  43       1.600  -2.465   6.696  1.00 98.63           H  \\nATOM    610  CE2 PHE A  43       4.989  -2.822   6.894  1.00 98.63           C  \\nATOM    611  HE2 PHE A  43       5.747  -3.483   7.288  1.00 98.63           H  \\nATOM    612  CZ  PHE A  43       3.629  -3.111   7.085  1.00 98.63           C  \\nATOM    613  HZ  PHE A  43       3.340  -4.001   7.624  1.00 98.63           H  \\nATOM    614  N   ASP A  44       5.280   2.579   2.467  1.00 98.66           N  \\nATOM    615  H   ASP A  44       6.053   1.954   2.287  1.00 98.66           H  \\nATOM    616  CA  ASP A  44       5.356   3.954   1.968  1.00 98.66           C  \\nATOM    617  HA  ASP A  44       5.123   4.620   2.799  1.00 98.66           H  \\nATOM    618  C   ASP A  44       4.324   4.248   0.866  1.00 98.66           C  \\nATOM    619  CB  ASP A  44       6.784   4.254   1.509  1.00 98.66           C  \\nATOM    620  HB2 ASP A  44       7.488   3.713   2.141  1.00 98.66           H  \\nATOM    621  HB3 ASP A  44       6.924   3.931   0.477  1.00 98.66           H  \\nATOM    622  O   ASP A  44       3.785   5.356   0.812  1.00 98.66           O  \\nATOM    623  CG  ASP A  44       7.126   5.728   1.670  1.00 98.66           C  \\nATOM    624  OD1 ASP A  44       7.004   6.235   2.809  1.00 98.66           O  \\nATOM    625  OD2 ASP A  44       7.626   6.345   0.709  1.00 98.66           O  \\nATOM    626  N   VAL A  45       3.980   3.247   0.042  1.00 98.61           N  \\nATOM    627  H   VAL A  45       4.439   2.357   0.173  1.00 98.61           H  \\nATOM    628  CA  VAL A  45       2.832   3.312  -0.884  1.00 98.61           C  \\nATOM    629  HA  VAL A  45       2.957   4.172  -1.541  1.00 98.61           H  \\nATOM    630  C   VAL A  45       1.533   3.513  -0.098  1.00 98.61           C  \\nATOM    631  CB  VAL A  45       2.715   2.039  -1.753  1.00 98.61           C  \\nATOM    632  HB  VAL A  45       2.613   1.169  -1.103  1.00 98.61           H  \\nATOM    633  O   VAL A  45       0.757   4.412  -0.433  1.00 98.61           O  \\nATOM    634  CG1 VAL A  45       1.475   2.065  -2.659  1.00 98.61           C  \\nATOM    635 HG11 VAL A  45       1.510   2.920  -3.335  1.00 98.61           H  \\nATOM    636 HG12 VAL A  45       1.420   1.135  -3.225  1.00 98.61           H  \\nATOM    637 HG13 VAL A  45       0.568   2.114  -2.057  1.00 98.61           H  \\nATOM    638  CG2 VAL A  45       3.934   1.808  -2.656  1.00 98.61           C  \\nATOM    639 HG21 VAL A  45       3.933   0.771  -2.992  1.00 98.61           H  \\nATOM    640 HG22 VAL A  45       4.861   1.995  -2.115  1.00 98.61           H  \\nATOM    641 HG23 VAL A  45       3.873   2.459  -3.528  1.00 98.61           H  \\nATOM    642  N   ASP A  46       1.314   2.729   0.960  1.00 98.60           N  \\nATOM    643  H   ASP A  46       1.942   1.954   1.117  1.00 98.60           H  \\nATOM    644  CA  ASP A  46       0.122   2.817   1.817  1.00 98.60           C  \\nATOM    645  HA  ASP A  46      -0.765   2.623   1.214  1.00 98.60           H  \\nATOM    646  C   ASP A  46      -0.000   4.210   2.445  1.00 98.60           C  \\nATOM    647  CB  ASP A  46       0.180   1.776   2.952  1.00 98.60           C  \\nATOM    648  HB2 ASP A  46       0.975   2.033   3.651  1.00 98.60           H  \\nATOM    649  HB3 ASP A  46      -0.754   1.831   3.512  1.00 98.60           H  \\nATOM    650  O   ASP A  46      -1.045   4.863   2.388  1.00 98.60           O  \\nATOM    651  CG  ASP A  46       0.383   0.334   2.492  1.00 98.60           C  \\nATOM    652  OD1 ASP A  46      -0.039   0.029   1.361  1.00 98.60           O  \\nATOM    653  OD2 ASP A  46       0.910  -0.452   3.316  1.00 98.60           O  \\nATOM    654  N   CYS A  47       1.104   4.709   2.999  1.00 98.65           N  \\nATOM    655  H   CYS A  47       1.914   4.108   3.039  1.00 98.65           H  \\nATOM    656  CA  CYS A  47       1.178   6.024   3.615  1.00 98.65           C  \\nATOM    657  HA  CYS A  47       0.421   6.085   4.397  1.00 98.65           H  \\nATOM    658  C   CYS A  47       0.906   7.146   2.608  1.00 98.65           C  \\nATOM    659  CB  CYS A  47       2.556   6.178   4.253  1.00 98.65           C  \\nATOM    660  HB2 CYS A  47       3.314   5.973   3.497  1.00 98.65           H  \\nATOM    661  HB3 CYS A  47       2.671   7.210   4.585  1.00 98.65           H  \\nATOM    662  O   CYS A  47       0.199   8.099   2.941  1.00 98.65           O  \\nATOM    663  SG  CYS A  47       2.837   5.080   5.669  1.00 98.65           S  \\nATOM    664  N   TRP A  48       1.399   7.034   1.370  1.00 98.62           N  \\nATOM    665  H   TRP A  48       1.989   6.244   1.150  1.00 98.62           H  \\nATOM    666  CA  TRP A  48       1.050   7.968   0.300  1.00 98.62           C  \\nATOM    667  HA  TRP A  48       1.273   8.983   0.629  1.00 98.62           H  \\nATOM    668  C   TRP A  48      -0.442   7.916  -0.043  1.00 98.62           C  \\nATOM    669  CB  TRP A  48       1.878   7.679  -0.954  1.00 98.62           C  \\nATOM    670  HB2 TRP A  48       2.928   7.890  -0.753  1.00 98.62           H  \\nATOM    671  HB3 TRP A  48       1.804   6.620  -1.204  1.00 98.62           H  \\nATOM    672  O   TRP A  48      -1.080   8.964  -0.166  1.00 98.62           O  \\nATOM    673  CG  TRP A  48       1.432   8.471  -2.142  1.00 98.62           C  \\nATOM    674  CD1 TRP A  48       1.646   9.789  -2.336  1.00 98.62           C  \\nATOM    675  HD1 TRP A  48       2.201  10.415  -1.653  1.00 98.62           H  \\nATOM    676  CD2 TRP A  48       0.584   8.039  -3.243  1.00 98.62           C  \\nATOM    677  CE2 TRP A  48       0.341   9.162  -4.091  1.00 98.62           C  \\nATOM    678  CE3 TRP A  48      -0.024   6.820  -3.600  1.00 98.62           C  \\nATOM    679  HE3 TRP A  48       0.129   5.950  -2.979  1.00 98.62           H  \\nATOM    680  NE1 TRP A  48       1.008  10.204  -3.487  1.00 98.62           N  \\nATOM    681  HE1 TRP A  48       1.003  11.160  -3.811  1.00 98.62           H  \\nATOM    682  CH2 TRP A  48      -1.022   7.853  -5.585  1.00 98.62           C  \\nATOM    683  HH2 TRP A  48      -1.631   7.764  -6.473  1.00 98.62           H  \\nATOM    684  CZ2 TRP A  48      -0.432   9.080  -5.256  1.00 98.62           C  \\nATOM    685  HZ2 TRP A  48      -0.573   9.946  -5.886  1.00 98.62           H  \\nATOM    686  CZ3 TRP A  48      -0.818   6.737  -4.757  1.00 98.62           C  \\nATOM    687  HZ3 TRP A  48      -1.263   5.793  -5.035  1.00 98.62           H  \\nATOM    688  N   ALA A  49      -1.020   6.719  -0.165  1.00 98.56           N  \\nATOM    689  H   ALA A  49      -0.449   5.893  -0.050  1.00 98.56           H  \\nATOM    690  CA  ALA A  49      -2.440   6.543  -0.449  1.00 98.56           C  \\nATOM    691  HA  ALA A  49      -2.667   7.023  -1.401  1.00 98.56           H  \\nATOM    692  C   ALA A  49      -3.313   7.208   0.632  1.00 98.56           C  \\nATOM    693  CB  ALA A  49      -2.720   5.041  -0.584  1.00 98.56           C  \\nATOM    694  HB1 ALA A  49      -3.786   4.869  -0.730  1.00 98.56           H  \\nATOM    695  HB2 ALA A  49      -2.167   4.635  -1.431  1.00 98.56           H  \\nATOM    696  HB3 ALA A  49      -2.412   4.510   0.316  1.00 98.56           H  \\nATOM    697  O   ALA A  49      -4.234   7.967   0.316  1.00 98.56           O  \\nATOM    698  N   ILE A  50      -2.986   7.001   1.910  1.00 98.42           N  \\nATOM    699  H   ILE A  50      -2.248   6.340   2.105  1.00 98.42           H  \\nATOM    700  CA  ILE A  50      -3.678   7.637   3.040  1.00 98.42           C  \\nATOM    701  HA  ILE A  50      -4.748   7.439   2.965  1.00 98.42           H  \\nATOM    702  C   ILE A  50      -3.484   9.157   3.000  1.00 98.42           C  \\nATOM    703  CB  ILE A  50      -3.170   7.049   4.376  1.00 98.42           C  \\nATOM    704  HB  ILE A  50      -2.089   7.180   4.410  1.00 98.42           H  \\nATOM    705  O   ILE A  50      -4.450   9.923   2.994  1.00 98.42           O  \\nATOM    706  CG1 ILE A  50      -3.482   5.540   4.499  1.00 98.42           C  \\nATOM    707 HG12 ILE A  50      -4.538   5.392   4.725  1.00 98.42           H  \\nATOM    708 HG13 ILE A  50      -3.275   5.037   3.555  1.00 98.42           H  \\nATOM    709  CG2 ILE A  50      -3.787   7.806   5.567  1.00 98.42           C  \\nATOM    710 HG21 ILE A  50      -3.439   8.839   5.589  1.00 98.42           H  \\nATOM    711 HG22 ILE A  50      -4.873   7.799   5.479  1.00 98.42           H  \\nATOM    712 HG23 ILE A  50      -3.491   7.341   6.508  1.00 98.42           H  \\nATOM    713  CD1 ILE A  50      -2.623   4.863   5.577  1.00 98.42           C  \\nATOM    714 HD11 ILE A  50      -2.970   5.137   6.573  1.00 98.42           H  \\nATOM    715 HD12 ILE A  50      -1.579   5.155   5.462  1.00 98.42           H  \\nATOM    716 HD13 ILE A  50      -2.675   3.780   5.470  1.00 98.42           H  \\nATOM    717  N   LEU A  51      -2.229   9.615   2.960  1.00 98.12           N  \\nATOM    718  H   LEU A  51      -1.475   8.944   2.926  1.00 98.12           H  \\nATOM    719  CA  LEU A  51      -1.891  11.026   3.124  1.00 98.12           C  \\nATOM    720  HA  LEU A  51      -2.415  11.380   4.011  1.00 98.12           H  \\nATOM    721  C   LEU A  51      -2.349  11.875   1.933  1.00 98.12           C  \\nATOM    722  CB  LEU A  51      -0.372  11.149   3.346  1.00 98.12           C  \\nATOM    723  HB2 LEU A  51       0.133  10.801   2.445  1.00 98.12           H  \\nATOM    724  HB3 LEU A  51      -0.069  10.493   4.162  1.00 98.12           H  \\nATOM    725  O   LEU A  51      -2.881  12.967   2.129  1.00 98.12           O  \\nATOM    726  CG  LEU A  51       0.119  12.573   3.657  1.00 98.12           C  \\nATOM    727  HG  LEU A  51      -0.235  13.260   2.888  1.00 98.12           H  \\nATOM    728  CD1 LEU A  51      -0.358  13.079   5.021  1.00 98.12           C  \\nATOM    729 HD11 LEU A  51       0.183  12.545   5.803  1.00 98.12           H  \\nATOM    730 HD12 LEU A  51      -0.123  14.136   5.141  1.00 98.12           H  \\nATOM    731 HD13 LEU A  51      -1.416  12.880   5.193  1.00 98.12           H  \\nATOM    732  CD2 LEU A  51       1.643  12.584   3.656  1.00 98.12           C  \\nATOM    733 HD21 LEU A  51       2.011  12.333   2.661  1.00 98.12           H  \\nATOM    734 HD22 LEU A  51       2.030  11.856   4.369  1.00 98.12           H  \\nATOM    735 HD23 LEU A  51       2.009  13.574   3.927  1.00 98.12           H  \\nATOM    736  N   CYS A  52      -2.141  11.397   0.710  1.00 98.22           N  \\nATOM    737  H   CYS A  52      -1.706  10.489   0.626  1.00 98.22           H  \\nATOM    738  CA  CYS A  52      -2.271  12.202  -0.502  1.00 98.22           C  \\nATOM    739  HA  CYS A  52      -2.374  13.252  -0.230  1.00 98.22           H  \\nATOM    740  C   CYS A  52      -3.501  11.860  -1.346  1.00 98.22           C  \\nATOM    741  CB  CYS A  52      -0.984  12.063  -1.318  1.00 98.22           C  \\nATOM    742  HB2 CYS A  52      -1.073  12.686  -2.207  1.00 98.22           H  \\nATOM    743  HB3 CYS A  52      -0.885  11.027  -1.641  1.00 98.22           H  \\nATOM    744  O   CYS A  52      -3.909  12.681  -2.165  1.00 98.22           O  \\nATOM    745  SG  CYS A  52       0.534  12.537  -0.440  1.00 98.22           S  \\nATOM    746  N   LYS A  53      -4.102  10.677  -1.165  1.00 97.65           N  \\nATOM    747  H   LYS A  53      -3.688  10.019  -0.520  1.00 97.65           H  \\nATOM    748  CA  LYS A  53      -5.299  10.252  -1.914  1.00 97.65           C  \\nATOM    749  HA  LYS A  53      -5.581  11.039  -2.614  1.00 97.65           H  \\nATOM    750  C   LYS A  53      -6.541  10.087  -1.042  1.00 97.65           C  \\nATOM    751  CB  LYS A  53      -4.997   8.983  -2.732  1.00 97.65           C  \\nATOM    752  HB2 LYS A  53      -5.889   8.703  -3.294  1.00 97.65           H  \\nATOM    753  HB3 LYS A  53      -4.784   8.155  -2.057  1.00 97.65           H  \\nATOM    754  O   LYS A  53      -7.606   9.808  -1.582  1.00 97.65           O  \\nATOM    755  CG  LYS A  53      -3.816   9.124  -3.707  1.00 97.65           C  \\nATOM    756  HG2 LYS A  53      -2.909   9.361  -3.151  1.00 97.65           H  \\nATOM    757  HG3 LYS A  53      -3.651   8.158  -4.186  1.00 97.65           H  \\nATOM    758  CD  LYS A  53      -4.011  10.171  -4.812  1.00 97.65           C  \\nATOM    759  HD2 LYS A  53      -3.112  10.165  -5.429  1.00 97.65           H  \\nATOM    760  HD3 LYS A  53      -4.117  11.165  -4.378  1.00 97.65           H  \\nATOM    761  CE  LYS A  53      -5.225   9.847  -5.690  1.00 97.65           C  \\nATOM    762  HE2 LYS A  53      -6.134  10.094  -5.142  1.00 97.65           H  \\nATOM    763  HE3 LYS A  53      -5.231   8.775  -5.889  1.00 97.65           H  \\nATOM    764  NZ  LYS A  53      -5.161  10.585  -6.974  1.00 97.65           N  \\nATOM    765  HZ1 LYS A  53      -5.069  11.578  -6.819  1.00 97.65           H  \\nATOM    766  HZ2 LYS A  53      -5.980  10.402  -7.537  1.00 97.65           H  \\nATOM    767  HZ3 LYS A  53      -4.356  10.272  -7.498  1.00 97.65           H  \\nATOM    768  N   GLY A  54      -6.425  10.288   0.273  1.00 96.51           N  \\nATOM    769  H   GLY A  54      -5.505  10.445   0.660  1.00 96.51           H  \\nATOM    770  CA  GLY A  54      -7.548  10.159   1.201  1.00 96.51           C  \\nATOM    771  HA2 GLY A  54      -8.382  10.772   0.859  1.00 96.51           H  \\nATOM    772  HA3 GLY A  54      -7.233  10.506   2.185  1.00 96.51           H  \\nATOM    773  C   GLY A  54      -8.036   8.719   1.350  1.00 96.51           C  \\nATOM    774  O   GLY A  54      -9.211   8.504   1.640  1.00 96.51           O  \\nATOM    775  N   ALA A  55      -7.159   7.734   1.121  1.00 97.99           N  \\nATOM    776  H   ALA A  55      -6.202   7.967   0.899  1.00 97.99           H  \\nATOM    777  CA  ALA A  55      -7.492   6.335   1.350  1.00 97.99           C  \\nATOM    778  HA  ALA A  55      -8.363   6.104   0.737  1.00 97.99           H  \\nATOM    779  C   ALA A  55      -7.855   6.099   2.833  1.00 97.99           C  \\nATOM    780  CB  ALA A  55      -6.328   5.456   0.883  1.00 97.99           C  \\nATOM    781  HB1 ALA A  55      -6.059   5.710  -0.142  1.00 97.99           H  \\nATOM    782  HB2 ALA A  55      -5.467   5.601   1.536  1.00 97.99           H  \\nATOM    783  HB3 ALA A  55      -6.617   4.405   0.918  1.00 97.99           H  \\nATOM    784  O   ALA A  55      -7.325   6.787   3.714  1.00 97.99           O  \\nATOM    785  N   PRO A  56      -8.754   5.143   3.130  1.00 97.71           N  \\nATOM    786  CA  PRO A  56      -9.218   4.912   4.493  1.00 97.71           C  \\nATOM    787  HA  PRO A  56      -9.638   5.832   4.899  1.00 97.71           H  \\nATOM    788  C   PRO A  56      -8.068   4.444   5.388  1.00 97.71           C  \\nATOM    789  CB  PRO A  56     -10.319   3.854   4.379  1.00 97.71           C  \\nATOM    790  HB2 PRO A  56     -10.343   3.189   5.243  1.00 97.71           H  \\nATOM    791  HB3 PRO A  56     -11.283   4.347   4.255  1.00 97.71           H  \\nATOM    792  O   PRO A  56      -7.343   3.529   5.023  1.00 97.71           O  \\nATOM    793  CG  PRO A  56      -9.967   3.104   3.095  1.00 97.71           C  \\nATOM    794  HG2 PRO A  56     -10.851   2.670   2.629  1.00 97.71           H  \\nATOM    795  HG3 PRO A  56      -9.222   2.338   3.313  1.00 97.71           H  \\nATOM    796  CD  PRO A  56      -9.357   4.188   2.211  1.00 97.71           C  \\nATOM    797  HD2 PRO A  56     -10.141   4.682   1.639  1.00 97.71           H  \\nATOM    798  HD3 PRO A  56      -8.621   3.743   1.541  1.00 97.71           H  \\nATOM    799  N   ASN A  57      -7.939   5.041   6.574  1.00 97.24           N  \\nATOM    800  H   ASN A  57      -8.538   5.826   6.787  1.00 97.24           H  \\nATOM    801  CA  ASN A  57      -6.915   4.699   7.570  1.00 97.24           C  \\nATOM    802  HA  ASN A  57      -6.176   4.047   7.104  1.00 97.24           H  \\nATOM    803  C   ASN A  57      -7.463   3.931   8.786  1.00 97.24           C  \\nATOM    804  CB  ASN A  57      -6.194   5.986   7.998  1.00 97.24           C  \\nATOM    805  HB2 ASN A  57      -5.808   6.487   7.110  1.00 97.24           H  \\nATOM    806  HB3 ASN A  57      -5.349   5.716   8.630  1.00 97.24           H  \\nATOM    807  O   ASN A  57      -6.689   3.477   9.626  1.00 97.24           O  \\nATOM    808  CG  ASN A  57      -7.076   6.963   8.760  1.00 97.24           C  \\nATOM    809  ND2 ASN A  57      -6.486   7.954   9.384  1.00 97.24           N  \\nATOM    810 HD21 ASN A  57      -5.485   8.072   9.320  1.00 97.24           H  \\nATOM    811 HD22 ASN A  57      -7.097   8.598   9.864  1.00 97.24           H  \\nATOM    812  OD1 ASN A  57      -8.293   6.877   8.789  1.00 97.24           O  \\nATOM    813  N   VAL A  58      -8.786   3.783   8.879  1.00 98.30           N  \\nATOM    814  H   VAL A  58      -9.353   4.212   8.162  1.00 98.30           H  \\nATOM    815  CA  VAL A  58      -9.461   2.919   9.848  1.00 98.30           C  \\nATOM    816  HA  VAL A  58      -8.735   2.563  10.579  1.00 98.30           H  \\nATOM    817  C   VAL A  58     -10.010   1.719   9.090  1.00 98.30           C  \\nATOM    818  CB  VAL A  58     -10.562   3.663  10.622  1.00 98.30           C  \\nATOM    819  HB  VAL A  58     -11.338   3.997   9.933  1.00 98.30           H  \\nATOM    820  O   VAL A  58     -10.854   1.864   8.203  1.00 98.30           O  \\nATOM    821  CG1 VAL A  58     -11.186   2.724  11.656  1.00 98.30           C  \\nATOM    822 HG11 VAL A  58     -10.403   2.265  12.260  1.00 98.30           H  \\nATOM    823 HG12 VAL A  58     -11.758   1.950  11.144  1.00 98.30           H  \\nATOM    824 HG13 VAL A  58     -11.864   3.278  12.305  1.00 98.30           H  \\nATOM    825  CG2 VAL A  58     -10.013   4.886  11.364  1.00 98.30           C  \\nATOM    826 HG21 VAL A  58     -10.808   5.355  11.944  1.00 98.30           H  \\nATOM    827 HG22 VAL A  58      -9.209   4.590  12.038  1.00 98.30           H  \\nATOM    828 HG23 VAL A  58      -9.632   5.619  10.654  1.00 98.30           H  \\nATOM    829  N   LEU A  59      -9.495   0.541   9.420  1.00 98.45           N  \\nATOM    830  H   LEU A  59      -8.861   0.503  10.205  1.00 98.45           H  \\nATOM    831  CA  LEU A  59      -9.694  -0.687   8.664  1.00 98.45           C  \\nATOM    832  HA  LEU A  59     -10.381  -0.495   7.840  1.00 98.45           H  \\nATOM    833  C   LEU A  59     -10.314  -1.775   9.537  1.00 98.45           C  \\nATOM    834  CB  LEU A  59      -8.344  -1.148   8.089  1.00 98.45           C  \\nATOM    835  HB2 LEU A  59      -8.495  -2.086   7.555  1.00 98.45           H  \\nATOM    836  HB3 LEU A  59      -7.681  -1.339   8.933  1.00 98.45           H  \\nATOM    837  O   LEU A  59     -10.194  -1.750  10.759  1.00 98.45           O  \\nATOM    838  CG  LEU A  59      -7.633  -0.154   7.151  1.00 98.45           C  \\nATOM    839  HG  LEU A  59      -7.394   0.762   7.691  1.00 98.45           H  \\nATOM    840  CD1 LEU A  59      -6.330  -0.790   6.667  1.00 98.45           C  \\nATOM    841 HD11 LEU A  59      -6.551  -1.698   6.106  1.00 98.45           H  \\nATOM    842 HD12 LEU A  59      -5.809  -0.093   6.011  1.00 98.45           H  \\nATOM    843 HD13 LEU A  59      -5.691  -1.017   7.520  1.00 98.45           H  \\nATOM    844  CD2 LEU A  59      -8.466   0.192   5.920  1.00 98.45           C  \\nATOM    845 HD21 LEU A  59      -8.733  -0.721   5.388  1.00 98.45           H  \\nATOM    846 HD22 LEU A  59      -7.891   0.843   5.261  1.00 98.45           H  \\nATOM    847 HD23 LEU A  59      -9.368   0.726   6.217  1.00 98.45           H  \\nATOM    848  N   GLN A  60     -10.936  -2.768   8.906  1.00 97.65           N  \\nATOM    849  H   GLN A  60     -11.038  -2.706   7.904  1.00 97.65           H  \\nATOM    850  CA  GLN A  60     -11.417  -3.969   9.592  1.00 97.65           C  \\nATOM    851  HA  GLN A  60     -11.028  -3.983  10.610  1.00 97.65           H  \\nATOM    852  C   GLN A  60     -10.901  -5.209   8.869  1.00 97.65           C  \\nATOM    853  CB  GLN A  60     -12.948  -3.942   9.663  1.00 97.65           C  \\nATOM    854  HB2 GLN A  60     -13.366  -4.270   8.712  1.00 97.65           H  \\nATOM    855  HB3 GLN A  60     -13.283  -2.921   9.848  1.00 97.65           H  \\nATOM    856  O   GLN A  60     -11.083  -5.329   7.660  1.00 97.65           O  \\nATOM    857  CG  GLN A  60     -13.470  -4.819  10.809  1.00 97.65           C  \\nATOM    858  HG2 GLN A  60     -13.102  -4.410  11.750  1.00 97.65           H  \\nATOM    859  HG3 GLN A  60     -13.100  -5.840  10.708  1.00 97.65           H  \\nATOM    860  CD  GLN A  60     -14.992  -4.849  10.875  1.00 97.65           C  \\nATOM    861  NE2 GLN A  60     -15.575  -5.147  12.016  1.00 97.65           N  \\nATOM    862 HE21 GLN A  60     -15.060  -5.225  12.881  1.00 97.65           H  \\nATOM    863 HE22 GLN A  60     -16.584  -5.143  11.992  1.00 97.65           H  \\nATOM    864  OE1 GLN A  60     -15.695  -4.618   9.906  1.00 97.65           O  \\nATOM    865  N   ARG A  61     -10.235  -6.127   9.576  1.00 97.32           N  \\nATOM    866  H   ARG A  61     -10.171  -6.010  10.577  1.00 97.32           H  \\nATOM    867  CA  ARG A  61      -9.587  -7.288   8.945  1.00 97.32           C  \\nATOM    868  HA  ARG A  61      -9.070  -6.935   8.052  1.00 97.32           H  \\nATOM    869  C   ARG A  61     -10.618  -8.328   8.503  1.00 97.32           C  \\nATOM    870  CB  ARG A  61      -8.543  -7.883   9.898  1.00 97.32           C  \\nATOM    871  HB2 ARG A  61      -7.862  -7.092  10.213  1.00 97.32           H  \\nATOM    872  HB3 ARG A  61      -9.048  -8.275  10.781  1.00 97.32           H  \\nATOM    873  O   ARG A  61     -11.447  -8.739   9.303  1.00 97.32           O  \\nATOM    874  CG  ARG A  61      -7.724  -9.001   9.239  1.00 97.32           C  \\nATOM    875  HG2 ARG A  61      -8.384  -9.797   8.895  1.00 97.32           H  \\nATOM    876  HG3 ARG A  61      -7.175  -8.595   8.389  1.00 97.32           H  \\nATOM    877  CD  ARG A  61      -6.736  -9.585  10.250  1.00 97.32           C  \\nATOM    878  HD2 ARG A  61      -7.296  -9.965  11.104  1.00 97.32           H  \\nATOM    879  HD3 ARG A  61      -6.069  -8.794  10.592  1.00 97.32           H  \\nATOM    880  NE  ARG A  61      -5.944 -10.671   9.648  1.00 97.32           N  \\nATOM    881  HE  ARG A  61      -6.061 -10.826   8.657  1.00 97.32           H  \\nATOM    882  NH1 ARG A  61      -4.921 -11.371  11.579  1.00 97.32           N  \\nATOM    883 HH11 ARG A  61      -5.500 -10.737  12.111  1.00 97.32           H  \\nATOM    884 HH12 ARG A  61      -4.257 -11.966  12.053  1.00 97.32           H  \\nATOM    885  NH2 ARG A  61      -4.328 -12.274   9.624  1.00 97.32           N  \\nATOM    886 HH21 ARG A  61      -4.406 -12.339   8.619  1.00 97.32           H  \\nATOM    887 HH22 ARG A  61      -3.597 -12.789  10.094  1.00 97.32           H  \\nATOM    888  CZ  ARG A  61      -5.072 -11.432  10.284  1.00 97.32           C  \\nATOM    889  N   VAL A  62     -10.535  -8.818   7.271  1.00 97.85           N  \\nATOM    890  H   VAL A  62      -9.786  -8.488   6.681  1.00 97.85           H  \\nATOM    891  CA  VAL A  62     -11.456  -9.830   6.728  1.00 97.85           C  \\nATOM    892  HA  VAL A  62     -12.301  -9.933   7.409  1.00 97.85           H  \\nATOM    893  C   VAL A  62     -10.819 -11.213   6.636  1.00 97.85           C  \\nATOM    894  CB  VAL A  62     -12.061  -9.404   5.380  1.00 97.85           C  \\nATOM    895  HB  VAL A  62     -12.740 -10.192   5.055  1.00 97.85           H  \\nATOM    896  O   VAL A  62      -9.597 -11.365   6.679  1.00 97.85           O  \\nATOM    897  CG1 VAL A  62     -12.902  -8.140   5.557  1.00 97.85           C  \\nATOM    898 HG11 VAL A  62     -12.282  -7.323   5.926  1.00 97.85           H  \\nATOM    899 HG12 VAL A  62     -13.352  -7.858   4.605  1.00 97.85           H  \\nATOM    900 HG13 VAL A  62     -13.693  -8.328   6.282  1.00 97.85           H  \\nATOM    901  CG2 VAL A  62     -11.012  -9.182   4.285  1.00 97.85           C  \\nATOM    902 HG21 VAL A  62     -10.395 -10.072   4.158  1.00 97.85           H  \\nATOM    903 HG22 VAL A  62     -10.382  -8.331   4.541  1.00 97.85           H  \\nATOM    904 HG23 VAL A  62     -11.505  -8.960   3.339  1.00 97.85           H  \\nATOM    905  N   ASN A  63     -11.669 -12.231   6.517  1.00 96.74           N  \\nATOM    906  H   ASN A  63     -12.656 -12.027   6.583  1.00 96.74           H  \\nATOM    907  CA  ASN A  63     -11.267 -13.616   6.308  1.00 96.74           C  \\nATOM    908  HA  ASN A  63     -10.521 -13.855   7.066  1.00 96.74           H  \\nATOM    909  C   ASN A  63     -10.595 -13.848   4.940  1.00 96.74           C  \\nATOM    910  CB  ASN A  63     -12.489 -14.534   6.535  1.00 96.74           C  \\nATOM    911  HB2 ASN A  63     -12.960 -14.269   7.482  1.00 96.74           H  \\nATOM    912  HB3 ASN A  63     -12.144 -15.564   6.620  1.00 96.74           H  \\nATOM    913  O   ASN A  63     -10.656 -13.028   4.022  1.00 96.74           O  \\nATOM    914  CG  ASN A  63     -13.560 -14.487   5.447  1.00 96.74           C  \\nATOM    915  ND2 ASN A  63     -14.588 -15.291   5.569  1.00 96.74           N  \\nATOM    916 HD21 ASN A  63     -14.713 -15.851   6.400  1.00 96.74           H  \\nATOM    917 HD22 ASN A  63     -15.370 -15.147   4.947  1.00 96.74           H  \\nATOM    918  OD1 ASN A  63     -13.496 -13.768   4.463  1.00 96.74           O  \\nATOM    919  N   GLU A  64      -9.999 -15.030   4.789  1.00 97.09           N  \\nATOM    920  H   GLU A  64      -9.912 -15.623   5.602  1.00 97.09           H  \\nATOM    921  CA  GLU A  64      -9.263 -15.435   3.587  1.00 97.09           C  \\nATOM    922  HA  GLU A  64      -8.419 -14.757   3.456  1.00 97.09           H  \\nATOM    923  C   GLU A  64     -10.105 -15.383   2.303  1.00 97.09           C  \\nATOM    924  CB  GLU A  64      -8.745 -16.864   3.825  1.00 97.09           C  \\nATOM    925  HB2 GLU A  64      -8.246 -16.908   4.793  1.00 97.09           H  \\nATOM    926  HB3 GLU A  64      -9.594 -17.546   3.868  1.00 97.09           H  \\nATOM    927  O   GLU A  64      -9.640 -14.911   1.267  1.00 97.09           O  \\nATOM    928  CG  GLU A  64      -7.774 -17.360   2.747  1.00 97.09           C  \\nATOM    929  HG2 GLU A  64      -8.286 -17.384   1.785  1.00 97.09           H  \\nATOM    930  HG3 GLU A  64      -7.484 -18.382   2.991  1.00 97.09           H  \\nATOM    931  CD  GLU A  64      -6.514 -16.497   2.651  1.00 97.09           C  \\nATOM    932  OE1 GLU A  64      -5.862 -16.499   1.583  1.00 97.09           O  \\nATOM    933  OE2 GLU A  64      -6.186 -15.768   3.619  1.00 97.09           O  \\nATOM    934  N   LYS A  65     -11.367 -15.824   2.374  1.00 97.72           N  \\nATOM    935  H   LYS A  65     -11.697 -16.160   3.268  1.00 97.72           H  \\nATOM    936  CA  LYS A  65     -12.283 -15.817   1.225  1.00 97.72           C  \\nATOM    937  HA  LYS A  65     -11.831 -16.385   0.412  1.00 97.72           H  \\nATOM    938  C   LYS A  65     -12.494 -14.398   0.697  1.00 97.72           C  \\nATOM    939  CB  LYS A  65     -13.611 -16.478   1.628  1.00 97.72           C  \\nATOM    940  HB2 LYS A  65     -13.413 -17.491   1.977  1.00 97.72           H  \\nATOM    941  HB3 LYS A  65     -14.061 -15.913   2.444  1.00 97.72           H  \\nATOM    942  O   LYS A  65     -12.369 -14.162  -0.503  1.00 97.72           O  \\nATOM    943  CG  LYS A  65     -14.598 -16.540   0.451  1.00 97.72           C  \\nATOM    944  HG2 LYS A  65     -14.790 -15.534   0.077  1.00 97.72           H  \\nATOM    945  HG3 LYS A  65     -14.167 -17.137  -0.353  1.00 97.72           H  \\nATOM    946  CD  LYS A  65     -15.935 -17.152   0.881  1.00 97.72           C  \\nATOM    947  HD2 LYS A  65     -16.341 -16.571   1.709  1.00 97.72           H  \\nATOM    948  HD3 LYS A  65     -15.777 -18.181   1.206  1.00 97.72           H  \\nATOM    949  CE  LYS A  65     -16.915 -17.118  -0.298  1.00 97.72           C  \\nATOM    950  HE2 LYS A  65     -16.518 -17.735  -1.104  1.00 97.72           H  \\nATOM    951  HE3 LYS A  65     -16.974 -16.093  -0.664  1.00 97.72           H  \\nATOM    952  NZ  LYS A  65     -18.265 -17.588   0.100  1.00 97.72           N  \\nATOM    953  HZ1 LYS A  65     -18.233 -18.545   0.421  1.00 97.72           H  \\nATOM    954  HZ2 LYS A  65     -18.901 -17.532  -0.683  1.00 97.72           H  \\nATOM    955  HZ3 LYS A  65     -18.632 -17.009   0.841  1.00 97.72           H  \\nATOM    956  N   THR A  66     -12.787 -13.457   1.593  1.00 98.08           N  \\nATOM    957  H   THR A  66     -12.852 -13.711   2.568  1.00 98.08           H  \\nATOM    958  CA  THR A  66     -12.984 -12.052   1.224  1.00 98.08           C  \\nATOM    959  HA  THR A  66     -13.698 -11.991   0.403  1.00 98.08           H  \\nATOM    960  C   THR A  66     -11.676 -11.431   0.758  1.00 98.08           C  \\nATOM    961  CB  THR A  66     -13.538 -11.239   2.392  1.00 98.08           C  \\nATOM    962  HB  THR A  66     -12.778 -11.183   3.171  1.00 98.08           H  \\nATOM    963  O   THR A  66     -11.669 -10.746  -0.256  1.00 98.08           O  \\nATOM    964  CG2 THR A  66     -13.939  -9.829   1.964  1.00 98.08           C  \\nATOM    965 HG21 THR A  66     -13.089  -9.302   1.531  1.00 98.08           H  \\nATOM    966 HG22 THR A  66     -14.743  -9.874   1.230  1.00 98.08           H  \\nATOM    967 HG23 THR A  66     -14.276  -9.265   2.835  1.00 98.08           H  \\nATOM    968  OG1 THR A  66     -14.680 -11.879   2.905  1.00 98.08           O  \\nATOM    969  HG1 THR A  66     -14.368 -12.548   3.519  1.00 98.08           H  \\nATOM    970  N   LYS A  67     -10.553 -11.733   1.423  1.00 98.10           N  \\nATOM    971  H   LYS A  67     -10.624 -12.286   2.265  1.00 98.10           H  \\nATOM    972  CA  LYS A  67      -9.226 -11.276   0.996  1.00 98.10           C  \\nATOM    973  HA  LYS A  67      -9.212 -10.187   1.048  1.00 98.10           H  \\nATOM    974  C   LYS A  67      -8.937 -11.641  -0.458  1.00 98.10           C  \\nATOM    975  CB  LYS A  67      -8.160 -11.851   1.938  1.00 98.10           C  \\nATOM    976  HB2 LYS A  67      -8.410 -11.591   2.967  1.00 98.10           H  \\nATOM    977  HB3 LYS A  67      -8.168 -12.938   1.853  1.00 98.10           H  \\nATOM    978  O   LYS A  67      -8.499 -10.790  -1.224  1.00 98.10           O  \\nATOM    979  CG  LYS A  67      -6.746 -11.328   1.628  1.00 98.10           C  \\nATOM    980  HG2 LYS A  67      -6.543 -11.350   0.557  1.00 98.10           H  \\nATOM    981  HG3 LYS A  67      -6.678 -10.291   1.958  1.00 98.10           H  \\nATOM    982  CD  LYS A  67      -5.663 -12.138   2.344  1.00 98.10           C  \\nATOM    983  HD2 LYS A  67      -4.737 -11.562   2.350  1.00 98.10           H  \\nATOM    984  HD3 LYS A  67      -5.968 -12.337   3.372  1.00 98.10           H  \\nATOM    985  CE  LYS A  67      -5.442 -13.448   1.586  1.00 98.10           C  \\nATOM    986  HE2 LYS A  67      -6.389 -13.982   1.513  1.00 98.10           H  \\nATOM    987  HE3 LYS A  67      -5.126 -13.239   0.564  1.00 98.10           H  \\nATOM    988  NZ  LYS A  67      -4.469 -14.334   2.255  1.00 98.10           N  \\nATOM    989  HZ1 LYS A  67      -4.498 -15.218   1.766  1.00 98.10           H  \\nATOM    990  HZ2 LYS A  67      -3.531 -13.975   2.356  1.00 98.10           H  \\nATOM    991  HZ3 LYS A  67      -4.869 -14.646   3.128  1.00 98.10           H  \\nATOM    992  N   ASN A  68      -9.207 -12.881  -0.857  1.00 98.26           N  \\nATOM    993  H   ASN A  68      -9.525 -13.551  -0.171  1.00 98.26           H  \\nATOM    994  CA  ASN A  68      -8.971 -13.320  -2.232  1.00 98.26           C  \\nATOM    995  HA  ASN A  68      -7.964 -13.024  -2.525  1.00 98.26           H  \\nATOM    996  C   ASN A  68      -9.905 -12.615  -3.224  1.00 98.26           C  \\nATOM    997  CB  ASN A  68      -9.045 -14.854  -2.277  1.00 98.26           C  \\nATOM    998  HB2 ASN A  68      -9.991 -15.191  -1.853  1.00 98.26           H  \\nATOM    999  HB3 ASN A  68      -8.987 -15.189  -3.313  1.00 98.26           H  \\nATOM   1000  O   ASN A  68      -9.431 -12.129  -4.246  1.00 98.26           O  \\nATOM   1001  CG  ASN A  68      -7.886 -15.483  -1.515  1.00 98.26           C  \\nATOM   1002  ND2 ASN A  68      -8.020 -16.704  -1.060  1.00 98.26           N  \\nATOM   1003 HD21 ASN A  68      -7.266 -17.020  -0.467  1.00 98.26           H  \\nATOM   1004 HD22 ASN A  68      -8.874 -17.225  -1.201  1.00 98.26           H  \\nATOM   1005  OD1 ASN A  68      -6.843 -14.878  -1.305  1.00 98.26           O  \\nATOM   1006  N   SER A  69     -11.182 -12.433  -2.869  1.00 98.23           N  \\nATOM   1007  H   SER A  69     -11.511 -12.845  -2.008  1.00 98.23           H  \\nATOM   1008  CA  SER A  69     -12.106 -11.611  -3.664  1.00 98.23           C  \\nATOM   1009  HA  SER A  69     -12.151 -12.024  -4.672  1.00 98.23           H  \\nATOM   1010  C   SER A  69     -11.643 -10.153  -3.784  1.00 98.23           C  \\nATOM   1011  CB  SER A  69     -13.506 -11.665  -3.047  1.00 98.23           C  \\nATOM   1012  HB2 SER A  69     -13.858 -12.696  -3.053  1.00 98.23           H  \\nATOM   1013  HB3 SER A  69     -13.468 -11.313  -2.016  1.00 98.23           H  \\nATOM   1014  O   SER A  69     -11.788  -9.543  -4.841  1.00 98.23           O  \\nATOM   1015  OG  SER A  69     -14.411 -10.856  -3.773  1.00 98.23           O  \\nATOM   1016  HG  SER A  69     -14.308 -11.046  -4.708  1.00 98.23           H  \\nATOM   1017  N   ASN A  70     -11.040  -9.588  -2.738  1.00 98.55           N  \\nATOM   1018  H   ASN A  70     -11.011 -10.094  -1.864  1.00 98.55           H  \\nATOM   1019  CA  ASN A  70     -10.481  -8.241  -2.784  1.00 98.55           C  \\nATOM   1020  HA  ASN A  70     -11.258  -7.557  -3.126  1.00 98.55           H  \\nATOM   1021  C   ASN A  70      -9.345  -8.153  -3.809  1.00 98.55           C  \\nATOM   1022  CB  ASN A  70     -10.035  -7.798  -1.380  1.00 98.55           C  \\nATOM   1023  HB2 ASN A  70      -9.527  -6.838  -1.462  1.00 98.55           H  \\nATOM   1024  HB3 ASN A  70      -9.325  -8.510  -0.960  1.00 98.55           H  \\nATOM   1025  O   ASN A  70      -9.348  -7.231  -4.614  1.00 98.55           O  \\nATOM   1026  CG  ASN A  70     -11.182  -7.623  -0.405  1.00 98.55           C  \\nATOM   1027  ND2 ASN A  70     -10.894  -7.310   0.827  1.00 98.55           N  \\nATOM   1028 HD21 ASN A  70     -11.628  -6.984   1.440  1.00 98.55           H  \\nATOM   1029 HD22 ASN A  70      -9.939  -7.321   1.155  1.00 98.55           H  \\nATOM   1030  OD1 ASN A  70     -12.348  -7.744  -0.733  1.00 98.55           O  \\nATOM   1031  N   ARG A  71      -8.439  -9.137  -3.861  1.00 98.31           N  \\nATOM   1032  H   ARG A  71      -8.511  -9.888  -3.190  1.00 98.31           H  \\nATOM   1033  CA  ARG A  71      -7.352  -9.163  -4.860  1.00 98.31           C  \\nATOM   1034  HA  ARG A  71      -6.789  -8.233  -4.792  1.00 98.31           H  \\nATOM   1035  C   ARG A  71      -7.846  -9.255  -6.306  1.00 98.31           C  \\nATOM   1036  CB  ARG A  71      -6.405 -10.336  -4.597  1.00 98.31           C  \\nATOM   1037  HB2 ARG A  71      -5.638 -10.315  -5.371  1.00 98.31           H  \\nATOM   1038  HB3 ARG A  71      -6.949 -11.278  -4.669  1.00 98.31           H  \\nATOM   1039  O   ARG A  71      -7.167  -8.786  -7.221  1.00 98.31           O  \\nATOM   1040  CG  ARG A  71      -5.713 -10.258  -3.239  1.00 98.31           C  \\nATOM   1041  HG2 ARG A  71      -6.419 -10.493  -2.443  1.00 98.31           H  \\nATOM   1042  HG3 ARG A  71      -5.335  -9.247  -3.092  1.00 98.31           H  \\nATOM   1043  CD  ARG A  71      -4.554 -11.253  -3.195  1.00 98.31           C  \\nATOM   1044  HD2 ARG A  71      -4.948 -12.256  -3.360  1.00 98.31           H  \\nATOM   1045  HD3 ARG A  71      -3.863 -11.029  -4.007  1.00 98.31           H  \\nATOM   1046  NE  ARG A  71      -3.838 -11.216  -1.906  1.00 98.31           N  \\nATOM   1047  HE  ARG A  71      -4.037 -11.974  -1.269  1.00 98.31           H  \\nATOM   1048  NH1 ARG A  71      -2.526  -9.350  -2.302  1.00 98.31           N  \\nATOM   1049 HH11 ARG A  71      -3.253  -8.860  -2.805  1.00 98.31           H  \\nATOM   1050 HH12 ARG A  71      -1.791  -8.719  -2.017  1.00 98.31           H  \\nATOM   1051  NH2 ARG A  71      -2.158 -10.564  -0.504  1.00 98.31           N  \\nATOM   1052 HH21 ARG A  71      -2.285 -11.388   0.065  1.00 98.31           H  \\nATOM   1053 HH22 ARG A  71      -1.429  -9.896  -0.297  1.00 98.31           H  \\nATOM   1054  CZ  ARG A  71      -2.861 -10.378  -1.585  1.00 98.31           C  \\nATOM   1055  N   ASP A  72      -9.008  -9.865  -6.525  1.00 98.03           N  \\nATOM   1056  H   ASP A  72      -9.461 -10.332  -5.753  1.00 98.03           H  \\nATOM   1057  CA  ASP A  72      -9.653  -9.879  -7.841  1.00 98.03           C  \\nATOM   1058  HA  ASP A  72      -8.909 -10.110  -8.603  1.00 98.03           H  \\nATOM   1059  C   ASP A  72     -10.233  -8.501  -8.180  1.00 98.03           C  \\nATOM   1060  CB  ASP A  72     -10.743 -10.960  -7.899  1.00 98.03           C  \\nATOM   1061  HB2 ASP A  72     -11.278 -10.862  -8.844  1.00 98.03           H  \\nATOM   1062  HB3 ASP A  72     -11.460 -10.804  -7.093  1.00 98.03           H  \\nATOM   1063  O   ASP A  72      -9.939  -7.949  -9.243  1.00 98.03           O  \\nATOM   1064  CG  ASP A  72     -10.182 -12.380  -7.820  1.00 98.03           C  \\nATOM   1065  OD1 ASP A  72      -9.076 -12.602  -8.368  1.00 98.03           O  \\nATOM   1066  OD2 ASP A  72     -10.861 -13.240  -7.219  1.00 98.03           O  \\nATOM   1067  N   ARG A  73     -10.998  -7.913  -7.249  1.00 97.96           N  \\nATOM   1068  H   ARG A  73     -11.168  -8.432  -6.399  1.00 97.96           H  \\nATOM   1069  CA  ARG A  73     -11.666  -6.610  -7.425  1.00 97.96           C  \\nATOM   1070  HA  ARG A  73     -12.215  -6.617  -8.367  1.00 97.96           H  \\nATOM   1071  C   ARG A  73     -10.700  -5.430  -7.522  1.00 97.96           C  \\nATOM   1072  CB  ARG A  73     -12.657  -6.369  -6.279  1.00 97.96           C  \\nATOM   1073  HB2 ARG A  73     -12.143  -6.490  -5.325  1.00 97.96           H  \\nATOM   1074  HB3 ARG A  73     -13.018  -5.343  -6.351  1.00 97.96           H  \\nATOM   1075  O   ARG A  73     -11.031  -4.428  -8.148  1.00 97.96           O  \\nATOM   1076  CG  ARG A  73     -13.874  -7.304  -6.343  1.00 97.96           C  \\nATOM   1077  HG2 ARG A  73     -13.564  -8.337  -6.182  1.00 97.96           H  \\nATOM   1078  HG3 ARG A  73     -14.318  -7.236  -7.336  1.00 97.96           H  \\nATOM   1079  CD  ARG A  73     -14.950  -6.930  -5.314  1.00 97.96           C  \\nATOM   1080  HD2 ARG A  73     -15.350  -5.950  -5.576  1.00 97.96           H  \\nATOM   1081  HD3 ARG A  73     -15.769  -7.643  -5.412  1.00 97.96           H  \\nATOM   1082  NE  ARG A  73     -14.462  -6.954  -3.915  1.00 97.96           N  \\nATOM   1083  HE  ARG A  73     -14.234  -7.860  -3.530  1.00 97.96           H  \\nATOM   1084  NH1 ARG A  73     -14.559  -4.682  -3.476  1.00 97.96           N  \\nATOM   1085 HH11 ARG A  73     -14.838  -4.472  -4.423  1.00 97.96           H  \\nATOM   1086 HH12 ARG A  73     -14.528  -3.950  -2.781  1.00 97.96           H  \\nATOM   1087  NH2 ARG A  73     -13.956  -6.032  -1.879  1.00 97.96           N  \\nATOM   1088 HH21 ARG A  73     -13.647  -6.925  -1.522  1.00 97.96           H  \\nATOM   1089 HH22 ARG A  73     -13.643  -5.225  -1.358  1.00 97.96           H  \\nATOM   1090  CZ  ARG A  73     -14.337  -5.910  -3.108  1.00 97.96           C  \\nATOM   1091  N   SER A  74      -9.520  -5.527  -6.918  1.00 98.12           N  \\nATOM   1092  H   SER A  74      -9.328  -6.344  -6.356  1.00 98.12           H  \\nATOM   1093  CA  SER A  74      -8.446  -4.535  -7.056  1.00 98.12           C  \\nATOM   1094  HA  SER A  74      -8.871  -3.532  -7.053  1.00 98.12           H  \\nATOM   1095  C   SER A  74      -7.642  -4.705  -8.348  1.00 98.12           C  \\nATOM   1096  CB  SER A  74      -7.492  -4.659  -5.870  1.00 98.12           C  \\nATOM   1097  HB2 SER A  74      -8.049  -4.638  -4.933  1.00 98.12           H  \\nATOM   1098  HB3 SER A  74      -6.794  -3.823  -5.873  1.00 98.12           H  \\nATOM   1099  O   SER A  74      -6.851  -3.839  -8.712  1.00 98.12           O  \\nATOM   1100  OG  SER A  74      -6.770  -5.866  -5.969  1.00 98.12           O  \\nATOM   1101  HG  SER A  74      -6.198  -5.893  -5.198  1.00 98.12           H  \\nATOM   1102  N   GLY A  75      -7.795  -5.838  -9.038  1.00 98.02           N  \\nATOM   1103  H   GLY A  75      -8.477  -6.509  -8.712  1.00 98.02           H  \\nATOM   1104  CA  GLY A  75      -6.964  -6.210 -10.175  1.00 98.02           C  \\nATOM   1105  HA2 GLY A  75      -6.851  -5.348 -10.832  1.00 98.02           H  \\nATOM   1106  HA3 GLY A  75      -7.475  -7.000 -10.724  1.00 98.02           H  \\nATOM   1107  C   GLY A  75      -5.566  -6.716  -9.800  1.00 98.02           C  \\nATOM   1108  O   GLY A  75      -4.801  -7.048 -10.705  1.00 98.02           O  \\nATOM   1109  N   ALA A  76      -5.211  -6.848  -8.516  1.00 97.89           N  \\nATOM   1110  H   ALA A  76      -5.872  -6.596  -7.795  1.00 97.89           H  \\nATOM   1111  CA  ALA A  76      -3.901  -7.360  -8.100  1.00 97.89           C  \\nATOM   1112  HA  ALA A  76      -3.130  -6.702  -8.501  1.00 97.89           H  \\nATOM   1113  C   ALA A  76      -3.625  -8.775  -8.645  1.00 97.89           C  \\nATOM   1114  CB  ALA A  76      -3.815  -7.315  -6.572  1.00 97.89           C  \\nATOM   1115  HB1 ALA A  76      -2.851  -7.696  -6.236  1.00 97.89           H  \\nATOM   1116  HB2 ALA A  76      -3.918  -6.285  -6.230  1.00 97.89           H  \\nATOM   1117  HB3 ALA A  76      -4.610  -7.913  -6.127  1.00 97.89           H  \\nATOM   1118  O   ALA A  76      -2.535  -9.041  -9.161  1.00 97.89           O  \\nATOM   1119  N   ASN A  77      -4.630  -9.659  -8.650  1.00 97.40           N  \\nATOM   1120  H   ASN A  77      -5.498  -9.389  -8.209  1.00 97.40           H  \\nATOM   1121  CA  ASN A  77      -4.502 -11.018  -9.197  1.00 97.40           C  \\nATOM   1122  HA  ASN A  77      -3.627 -11.488  -8.748  1.00 97.40           H  \\nATOM   1123  C   ASN A  77      -4.248 -11.056 -10.716  1.00 97.40           C  \\nATOM   1124  CB  ASN A  77      -5.745 -11.841  -8.813  1.00 97.40           C  \\nATOM   1125  HB2 ASN A  77      -5.832 -12.704  -9.473  1.00 97.40           H  \\nATOM   1126  HB3 ASN A  77      -6.648 -11.244  -8.931  1.00 97.40           H  \\nATOM   1127  O   ASN A  77      -3.739 -12.051 -11.232  1.00 97.40           O  \\nATOM   1128  CG  ASN A  77      -5.692 -12.387  -7.396  1.00 97.40           C  \\nATOM   1129  ND2 ASN A  77      -6.792 -12.895  -6.908  1.00 97.40           N  \\nATOM   1130 HD21 ASN A  77      -6.801 -13.256  -5.965  1.00 97.40           H  \\nATOM   1131 HD22 ASN A  77      -7.641 -12.874  -7.456  1.00 97.40           H  \\nATOM   1132  OD1 ASN A  77      -4.672 -12.383  -6.722  1.00 97.40           O  \\nATOM   1133  N   LYS A  78      -4.513  -9.961 -11.442  1.00 97.17           N  \\nATOM   1134  H   LYS A  78      -4.869  -9.146 -10.962  1.00 97.17           H  \\nATOM   1135  CA  LYS A  78      -4.176  -9.843 -12.872  1.00 97.17           C  \\nATOM   1136  HA  LYS A  78      -4.436 -10.776 -13.371  1.00 97.17           H  \\nATOM   1137  C   LYS A  78      -2.678  -9.647 -13.105  1.00 97.17           C  \\nATOM   1138  CB  LYS A  78      -4.974  -8.702 -13.519  1.00 97.17           C  \\nATOM   1139  HB2 LYS A  78      -4.735  -7.754 -13.037  1.00 97.17           H  \\nATOM   1140  HB3 LYS A  78      -4.685  -8.617 -14.567  1.00 97.17           H  \\nATOM   1141  O   LYS A  78      -2.211  -9.828 -14.227  1.00 97.17           O  \\nATOM   1142  CG  LYS A  78      -6.478  -8.978 -13.451  1.00 97.17           C  \\nATOM   1143  HG2 LYS A  78      -6.764  -9.199 -12.422  1.00 97.17           H  \\nATOM   1144  HG3 LYS A  78      -6.703  -9.849 -14.067  1.00 97.17           H  \\nATOM   1145  CD  LYS A  78      -7.318  -7.790 -13.920  1.00 97.17           C  \\nATOM   1146  HD2 LYS A  78      -7.128  -6.943 -13.261  1.00 97.17           H  \\nATOM   1147  HD3 LYS A  78      -7.073  -7.501 -14.941  1.00 97.17           H  \\nATOM   1148  CE  LYS A  78      -8.786  -8.202 -13.782  1.00 97.17           C  \\nATOM   1149  HE2 LYS A  78      -8.859  -9.028 -13.075  1.00 97.17           H  \\nATOM   1150  HE3 LYS A  78      -9.328  -7.372 -13.330  1.00 97.17           H  \\nATOM   1151  NZ  LYS A  78      -9.413  -8.572 -15.073  1.00 97.17           N  \\nATOM   1152  HZ1 LYS A  78      -8.935  -9.333 -15.533  1.00 97.17           H  \\nATOM   1153  HZ2 LYS A  78      -9.435  -7.769 -15.685  1.00 97.17           H  \\nATOM   1154  HZ3 LYS A  78     -10.383  -8.812 -14.927  1.00 97.17           H  \\nATOM   1155  N   GLY A  79      -1.931  -9.284 -12.059  1.00 97.17           N  \\nATOM   1156  H   GLY A  79      -2.394  -9.170 -11.169  1.00 97.17           H  \\nATOM   1157  CA  GLY A  79      -0.497  -9.021 -12.118  1.00 97.17           C  \\nATOM   1158  HA2 GLY A  79       0.022  -9.931 -12.420  1.00 97.17           H  \\nATOM   1159  HA3 GLY A  79      -0.151  -8.731 -11.126  1.00 97.17           H  \\nATOM   1160  C   GLY A  79      -0.137  -7.907 -13.105  1.00 97.17           C  \\nATOM   1161  O   GLY A  79       0.672  -8.168 -13.996  1.00 97.17           O  \\nATOM   1162  N   PRO A  80      -0.695  -6.685 -12.970  1.00 97.69           N  \\nATOM   1163  CA  PRO A  80      -0.499  -5.607 -13.945  1.00 97.69           C  \\nATOM   1164  HA  PRO A  80      -0.958  -5.894 -14.891  1.00 97.69           H  \\nATOM   1165  C   PRO A  80       0.974  -5.281 -14.203  1.00 97.69           C  \\nATOM   1166  CB  PRO A  80      -1.249  -4.397 -13.382  1.00 97.69           C  \\nATOM   1167  HB2 PRO A  80      -0.726  -3.457 -13.561  1.00 97.69           H  \\nATOM   1168  HB3 PRO A  80      -2.239  -4.358 -13.837  1.00 97.69           H  \\nATOM   1169  O   PRO A  80       1.368  -5.002 -15.328  1.00 97.69           O  \\nATOM   1170  CG  PRO A  80      -1.374  -4.689 -11.887  1.00 97.69           C  \\nATOM   1171  HG2 PRO A  80      -2.242  -4.199 -11.444  1.00 97.69           H  \\nATOM   1172  HG3 PRO A  80      -0.461  -4.383 -11.376  1.00 97.69           H  \\nATOM   1173  CD  PRO A  80      -1.499  -6.208 -11.850  1.00 97.69           C  \\nATOM   1174  HD2 PRO A  80      -2.541  -6.491 -12.003  1.00 97.69           H  \\nATOM   1175  HD3 PRO A  80      -1.142  -6.590 -10.893  1.00 97.69           H  \\nATOM   1176  N   PHE A  81       1.812  -5.402 -13.174  1.00 98.05           N  \\nATOM   1177  H   PHE A  81       1.412  -5.582 -12.264  1.00 98.05           H  \\nATOM   1178  CA  PHE A  81       3.238  -5.091 -13.237  1.00 98.05           C  \\nATOM   1179  HA  PHE A  81       3.380  -4.263 -13.931  1.00 98.05           H  \\nATOM   1180  C   PHE A  81       4.112  -6.241 -13.766  1.00 98.05           C  \\nATOM   1181  CB  PHE A  81       3.676  -4.614 -11.847  1.00 98.05           C  \\nATOM   1182  HB2 PHE A  81       3.649  -5.462 -11.164  1.00 98.05           H  \\nATOM   1183  HB3 PHE A  81       4.704  -4.253 -11.891  1.00 98.05           H  \\nATOM   1184  O   PHE A  81       5.312  -6.054 -13.946  1.00 98.05           O  \\nATOM   1185  CG  PHE A  81       2.809  -3.508 -11.282  1.00 98.05           C  \\nATOM   1186  CD1 PHE A  81       2.750  -2.259 -11.923  1.00 98.05           C  \\nATOM   1187  HD1 PHE A  81       3.361  -2.074 -12.794  1.00 98.05           H  \\nATOM   1188  CD2 PHE A  81       2.041  -3.735 -10.127  1.00 98.05           C  \\nATOM   1189  HD2 PHE A  81       2.101  -4.687  -9.620  1.00 98.05           H  \\nATOM   1190  CE1 PHE A  81       1.906  -1.250 -11.428  1.00 98.05           C  \\nATOM   1191  HE1 PHE A  81       1.857  -0.291 -11.923  1.00 98.05           H  \\nATOM   1192  CE2 PHE A  81       1.209  -2.720  -9.622  1.00 98.05           C  \\nATOM   1193  HE2 PHE A  81       0.633  -2.879  -8.722  1.00 98.05           H  \\nATOM   1194  CZ  PHE A  81       1.135  -1.480 -10.277  1.00 98.05           C  \\nATOM   1195  HZ  PHE A  81       0.501  -0.698  -9.887  1.00 98.05           H  \\nATOM   1196  N   LYS A  82       3.543  -7.426 -14.037  1.00 97.38           N  \\nATOM   1197  H   LYS A  82       2.545  -7.514 -13.914  1.00 97.38           H  \\nATOM   1198  CA  LYS A  82       4.271  -8.524 -14.703  1.00 97.38           C  \\nATOM   1199  HA  LYS A  82       5.288  -8.573 -14.314  1.00 97.38           H  \\nATOM   1200  C   LYS A  82       4.416  -8.285 -16.204  1.00 97.38           C  \\nATOM   1201  CB  LYS A  82       3.576  -9.871 -14.467  1.00 97.38           C  \\nATOM   1202  HB2 LYS A  82       2.550  -9.815 -14.829  1.00 97.38           H  \\nATOM   1203  HB3 LYS A  82       4.095 -10.629 -15.055  1.00 97.38           H  \\nATOM   1204  O   LYS A  82       5.411  -8.709 -16.780  1.00 97.38           O  \\nATOM   1205  CG  LYS A  82       3.585 -10.310 -12.997  1.00 97.38           C  \\nATOM   1206  HG2 LYS A  82       4.614 -10.377 -12.643  1.00 97.38           H  \\nATOM   1207  HG3 LYS A  82       3.046  -9.581 -12.392  1.00 97.38           H  \\nATOM   1208  CD  LYS A  82       2.904 -11.679 -12.865  1.00 97.38           C  \\nATOM   1209  HD2 LYS A  82       1.878 -11.593 -13.224  1.00 97.38           H  \\nATOM   1210  HD3 LYS A  82       3.433 -12.406 -13.480  1.00 97.38           H  \\nATOM   1211  CE  LYS A  82       2.898 -12.143 -11.405  1.00 97.38           C  \\nATOM   1212  HE2 LYS A  82       2.442 -11.361 -10.798  1.00 97.38           H  \\nATOM   1213  HE3 LYS A  82       3.928 -12.263 -11.070  1.00 97.38           H  \\nATOM   1214  NZ  LYS A  82       2.145 -13.414 -11.240  1.00 97.38           N  \\nATOM   1215  HZ1 LYS A  82       2.547 -14.152 -11.799  1.00 97.38           H  \\nATOM   1216  HZ2 LYS A  82       2.139 -13.705 -10.272  1.00 97.38           H  \\nATOM   1217  HZ3 LYS A  82       1.184 -13.291 -11.525  1.00 97.38           H  \\nATOM   1218  N   ASP A  83       3.432  -7.619 -16.802  1.00 95.39           N  \\nATOM   1219  H   ASP A  83       2.663  -7.296 -16.233  1.00 95.39           H  \\nATOM   1220  CA  ASP A  83       3.415  -7.229 -18.211  1.00 95.39           C  \\nATOM   1221  HA  ASP A  83       4.433  -7.190 -18.600  1.00 95.39           H  \\nATOM   1222  C   ASP A  83       2.797  -5.825 -18.344  1.00 95.39           C  \\nATOM   1223  CB  ASP A  83       2.665  -8.294 -19.020  1.00 95.39           C  \\nATOM   1224  HB2 ASP A  83       3.131  -9.262 -18.835  1.00 95.39           H  \\nATOM   1225  HB3 ASP A  83       1.632  -8.352 -18.680  1.00 95.39           H  \\nATOM   1226  O   ASP A  83       1.602  -5.687 -18.626  1.00 95.39           O  \\nATOM   1227  CG  ASP A  83       2.686  -8.031 -20.529  1.00 95.39           C  \\nATOM   1228  OD1 ASP A  83       3.119  -6.930 -20.953  1.00 95.39           O  \\nATOM   1229  OD2 ASP A  83       2.203  -8.943 -21.235  1.00 95.39           O  \\nATOM   1230  N   PRO A  84       3.591  -4.768 -18.084  1.00 97.10           N  \\nATOM   1231  CA  PRO A  84       3.106  -3.396 -18.118  1.00 97.10           C  \\nATOM   1232  HA  PRO A  84       2.382  -3.307 -17.308  1.00 97.10           H  \\nATOM   1233  C   PRO A  84       2.438  -2.995 -19.439  1.00 97.10           C  \\nATOM   1234  CB  PRO A  84       4.353  -2.541 -17.841  1.00 97.10           C  \\nATOM   1235  HB2 PRO A  84       4.903  -2.370 -18.767  1.00 97.10           H  \\nATOM   1236  HB3 PRO A  84       4.149  -1.584 -17.361  1.00 97.10           H  \\nATOM   1237  O   PRO A  84       1.416  -2.305 -19.448  1.00 97.10           O  \\nATOM   1238  CG  PRO A  84       5.181  -3.432 -16.931  1.00 97.10           C  \\nATOM   1239  HG2 PRO A  84       4.792  -3.393 -15.913  1.00 97.10           H  \\nATOM   1240  HG3 PRO A  84       6.236  -3.160 -16.969  1.00 97.10           H  \\nATOM   1241  CD  PRO A  84       4.940  -4.811 -17.532  1.00 97.10           C  \\nATOM   1242  HD2 PRO A  84       5.653  -4.997 -18.335  1.00 97.10           H  \\nATOM   1243  HD3 PRO A  84       5.046  -5.566 -16.753  1.00 97.10           H  \\nATOM   1244  N   GLN A  85       2.996  -3.463 -20.559  1.00 96.84           N  \\nATOM   1245  H   GLN A  85       3.722  -4.160 -20.477  1.00 96.84           H  \\nATOM   1246  CA  GLN A  85       2.545  -3.100 -21.900  1.00 96.84           C  \\nATOM   1247  HA  GLN A  85       2.482  -2.015 -21.974  1.00 96.84           H  \\nATOM   1248  C   GLN A  85       1.151  -3.651 -22.192  1.00 96.84           C  \\nATOM   1249  CB  GLN A  85       3.551  -3.612 -22.939  1.00 96.84           C  \\nATOM   1250  HB2 GLN A  85       3.726  -4.676 -22.780  1.00 96.84           H  \\nATOM   1251  HB3 GLN A  85       3.125  -3.478 -23.933  1.00 96.84           H  \\nATOM   1252  O   GLN A  85       0.302  -2.906 -22.680  1.00 96.84           O  \\nATOM   1253  CG  GLN A  85       4.883  -2.850 -22.878  1.00 96.84           C  \\nATOM   1254  HG2 GLN A  85       4.690  -1.789 -23.039  1.00 96.84           H  \\nATOM   1255  HG3 GLN A  85       5.337  -2.973 -21.894  1.00 96.84           H  \\nATOM   1256  CD  GLN A  85       5.880  -3.333 -23.926  1.00 96.84           C  \\nATOM   1257  NE2 GLN A  85       6.895  -2.556 -24.234  1.00 96.84           N  \\nATOM   1258 HE21 GLN A  85       7.511  -2.924 -24.945  1.00 96.84           H  \\nATOM   1259 HE22 GLN A  85       7.002  -1.641 -23.821  1.00 96.84           H  \\nATOM   1260  OE1 GLN A  85       5.789  -4.408 -24.487  1.00 96.84           O  \\nATOM   1261  N   LYS A  86       0.878  -4.908 -21.815  1.00 97.07           N  \\nATOM   1262  H   LYS A  86       1.630  -5.475 -21.450  1.00 97.07           H  \\nATOM   1263  CA  LYS A  86      -0.455  -5.520 -21.947  1.00 97.07           C  \\nATOM   1264  HA  LYS A  86      -0.706  -5.603 -23.005  1.00 97.07           H  \\nATOM   1265  C   LYS A  86      -1.562  -4.693 -21.297  1.00 97.07           C  \\nATOM   1266  CB  LYS A  86      -0.393  -6.921 -21.330  1.00 97.07           C  \\nATOM   1267  HB2 LYS A  86       0.252  -7.532 -21.962  1.00 97.07           H  \\nATOM   1268  HB3 LYS A  86       0.044  -6.858 -20.334  1.00 97.07           H  \\nATOM   1269  O   LYS A  86      -2.689  -4.684 -21.786  1.00 97.07           O  \\nATOM   1270  CG  LYS A  86      -1.760  -7.603 -21.216  1.00 97.07           C  \\nATOM   1271  HG2 LYS A  86      -2.405  -7.038 -20.543  1.00 97.07           H  \\nATOM   1272  HG3 LYS A  86      -2.222  -7.659 -22.201  1.00 97.07           H  \\nATOM   1273  CD  LYS A  86      -1.596  -9.007 -20.643  1.00 97.07           C  \\nATOM   1274  HD2 LYS A  86      -0.971  -9.594 -21.317  1.00 97.07           H  \\nATOM   1275  HD3 LYS A  86      -1.113  -8.946 -19.668  1.00 97.07           H  \\nATOM   1276  CE  LYS A  86      -2.982  -9.636 -20.510  1.00 97.07           C  \\nATOM   1277  HE2 LYS A  86      -3.478  -9.578 -21.479  1.00 97.07           H  \\nATOM   1278  HE3 LYS A  86      -3.563  -9.048 -19.799  1.00 97.07           H  \\nATOM   1279  NZ  LYS A  86      -2.872 -11.044 -20.071  1.00 97.07           N  \\nATOM   1280  HZ1 LYS A  86      -3.784 -11.473 -20.007  1.00 97.07           H  \\nATOM   1281  HZ2 LYS A  86      -2.406 -11.091 -19.176  1.00 97.07           H  \\nATOM   1282  HZ3 LYS A  86      -2.315 -11.558 -20.739  1.00 97.07           H  \\nATOM   1283  N   TRP A  87      -1.253  -4.028 -20.189  1.00 96.58           N  \\nATOM   1284  H   TRP A  87      -0.304  -4.097 -19.850  1.00 96.58           H  \\nATOM   1285  CA  TRP A  87      -2.227  -3.276 -19.400  1.00 96.58           C  \\nATOM   1286  HA  TRP A  87      -3.230  -3.604 -19.672  1.00 96.58           H  \\nATOM   1287  C   TRP A  87      -2.213  -1.770 -19.669  1.00 96.58           C  \\nATOM   1288  CB  TRP A  87      -2.020  -3.627 -17.927  1.00 96.58           C  \\nATOM   1289  HB2 TRP A  87      -0.987  -3.414 -17.650  1.00 96.58           H  \\nATOM   1290  HB3 TRP A  87      -2.662  -2.996 -17.312  1.00 96.58           H  \\nATOM   1291  O   TRP A  87      -2.955  -1.038 -19.014  1.00 96.58           O  \\nATOM   1292  CG  TRP A  87      -2.332  -5.055 -17.611  1.00 96.58           C  \\nATOM   1293  CD1 TRP A  87      -1.424  -6.008 -17.315  1.00 96.58           C  \\nATOM   1294  HD1 TRP A  87      -0.359  -5.835 -17.251  1.00 96.58           H  \\nATOM   1295  CD2 TRP A  87      -3.630  -5.721 -17.598  1.00 96.58           C  \\nATOM   1296  CE2 TRP A  87      -3.428  -7.088 -17.240  1.00 96.58           C  \\nATOM   1297  CE3 TRP A  87      -4.958  -5.311 -17.851  1.00 96.58           C  \\nATOM   1298  HE3 TRP A  87      -5.150  -4.287 -18.136  1.00 96.58           H  \\nATOM   1299  NE1 TRP A  87      -2.067  -7.201 -17.051  1.00 96.58           N  \\nATOM   1300  HE1 TRP A  87      -1.581  -8.018 -16.708  1.00 96.58           H  \\nATOM   1301  CH2 TRP A  87      -5.797  -7.559 -17.394  1.00 96.58           C  \\nATOM   1302  HH2 TRP A  87      -6.628  -8.245 -17.326  1.00 96.58           H  \\nATOM   1303  CZ2 TRP A  87      -4.487  -8.002 -17.142  1.00 96.58           C  \\nATOM   1304  HZ2 TRP A  87      -4.291  -9.027 -16.862  1.00 96.58           H  \\nATOM   1305  CZ3 TRP A  87      -6.031  -6.216 -17.741  1.00 96.58           C  \\nATOM   1306  HZ3 TRP A  87      -7.035  -5.873 -17.940  1.00 96.58           H  \\nATOM   1307  N   GLY A  88      -1.386  -1.296 -20.608  1.00 95.66           N  \\nATOM   1308  H   GLY A  88      -0.821  -1.948 -21.134  1.00 95.66           H  \\nATOM   1309  CA  GLY A  88      -1.234   0.133 -20.889  1.00 95.66           C  \\nATOM   1310  HA2 GLY A  88      -2.201   0.555 -21.162  1.00 95.66           H  \\nATOM   1311  HA3 GLY A  88      -0.548   0.253 -21.728  1.00 95.66           H  \\nATOM   1312  C   GLY A  88      -0.678   0.928 -19.704  1.00 95.66           C  \\nATOM   1313  O   GLY A  88      -0.892   2.136 -19.617  1.00 95.66           O  \\nATOM   1314  N   ILE A  89       0.012   0.262 -18.775  1.00 96.52           N  \\nATOM   1315  H   ILE A  89       0.237  -0.706 -18.957  1.00 96.52           H  \\nATOM   1316  CA  ILE A  89       0.690   0.926 -17.666  1.00 96.52           C  \\nATOM   1317  HA  ILE A  89       0.189   1.876 -17.483  1.00 96.52           H  \\nATOM   1318  C   ILE A  89       2.125   1.264 -18.078  1.00 96.52           C  \\nATOM   1319  CB  ILE A  89       0.589   0.133 -16.346  1.00 96.52           C  \\nATOM   1320  HB  ILE A  89       1.076   0.731 -15.576  1.00 96.52           H  \\nATOM   1321  O   ILE A  89       2.719   0.643 -18.963  1.00 96.52           O  \\nATOM   1322  CG1 ILE A  89       1.297  -1.217 -16.431  1.00 96.52           C  \\nATOM   1323 HG12 ILE A  89       0.661  -1.942 -16.939  1.00 96.52           H  \\nATOM   1324 HG13 ILE A  89       2.203  -1.071 -17.020  1.00 96.52           H  \\nATOM   1325  CG2 ILE A  89      -0.873  -0.081 -15.925  1.00 96.52           C  \\nATOM   1326 HG21 ILE A  89      -1.404   0.870 -15.929  1.00 96.52           H  \\nATOM   1327 HG22 ILE A  89      -1.375  -0.755 -16.620  1.00 96.52           H  \\nATOM   1328 HG23 ILE A  89      -0.919  -0.501 -14.920  1.00 96.52           H  \\nATOM   1329  CD1 ILE A  89       1.717  -1.770 -15.072  1.00 96.52           C  \\nATOM   1330 HD11 ILE A  89       2.287  -1.009 -14.539  1.00 96.52           H  \\nATOM   1331 HD12 ILE A  89       0.839  -2.043 -14.486  1.00 96.52           H  \\nATOM   1332 HD13 ILE A  89       2.355  -2.643 -15.206  1.00 96.52           H  \\nATOM   1333  N   LYS A  90       2.701   2.279 -17.439  1.00 95.85           N  \\nATOM   1334  H   LYS A  90       2.223   2.690 -16.649  1.00 95.85           H  \\nATOM   1335  CA  LYS A  90       4.046   2.737 -17.782  1.00 95.85           C  \\nATOM   1336  HA  LYS A  90       4.083   2.838 -18.866  1.00 95.85           H  \\nATOM   1337  C   LYS A  90       5.099   1.691 -17.401  1.00 95.85           C  \\nATOM   1338  CB  LYS A  90       4.278   4.121 -17.171  1.00 95.85           C  \\nATOM   1339  HB2 LYS A  90       5.122   4.589 -17.678  1.00 95.85           H  \\nATOM   1340  HB3 LYS A  90       3.397   4.738 -17.345  1.00 95.85           H  \\nATOM   1341  O   LYS A  90       4.868   0.852 -16.529  1.00 95.85           O  \\nATOM   1342  CG  LYS A  90       4.582   4.059 -15.674  1.00 95.85           C  \\nATOM   1343  HG2 LYS A  90       5.591   3.663 -15.562  1.00 95.85           H  \\nATOM   1344  HG3 LYS A  90       3.898   3.394 -15.147  1.00 95.85           H  \\nATOM   1345  CD  LYS A  90       4.512   5.468 -15.080  1.00 95.85           C  \\nATOM   1346  HD2 LYS A  90       4.974   6.174 -15.769  1.00 95.85           H  \\nATOM   1347  HD3 LYS A  90       3.481   5.759 -14.883  1.00 95.85           H  \\nATOM   1348  CE  LYS A  90       5.312   5.415 -13.800  1.00 95.85           C  \\nATOM   1349  HE2 LYS A  90       6.198   4.821 -14.028  1.00 95.85           H  \\nATOM   1350  HE3 LYS A  90       4.735   4.915 -13.022  1.00 95.85           H  \\nATOM   1351  NZ  LYS A  90       5.779   6.734 -13.353  1.00 95.85           N  \\nATOM   1352  HZ1 LYS A  90       6.194   7.259 -14.109  1.00 95.85           H  \\nATOM   1353  HZ2 LYS A  90       4.981   7.231 -12.984  1.00 95.85           H  \\nATOM   1354  HZ3 LYS A  90       6.441   6.612 -12.600  1.00 95.85           H  \\nATOM   1355  N   ALA A  91       6.270   1.776 -18.032  1.00 97.39           N  \\nATOM   1356  H   ALA A  91       6.390   2.490 -18.736  1.00 97.39           H  \\nATOM   1357  CA  ALA A  91       7.397   0.895 -17.736  1.00 97.39           C  \\nATOM   1358  HA  ALA A  91       7.123  -0.118 -18.028  1.00 97.39           H  \\nATOM   1359  C   ALA A  91       7.730   0.886 -16.235  1.00 97.39           C  \\nATOM   1360  CB  ALA A  91       8.606   1.331 -18.571  1.00 97.39           C  \\nATOM   1361  HB1 ALA A  91       8.362   1.295 -19.632  1.00 97.39           H  \\nATOM   1362  HB2 ALA A  91       8.900   2.346 -18.300  1.00 97.39           H  \\nATOM   1363  HB3 ALA A  91       9.445   0.661 -18.380  1.00 97.39           H  \\nATOM   1364  O   ALA A  91       7.590   1.909 -15.559  1.00 97.39           O  \\nATOM   1365  N   LEU A  92       8.166  -0.269 -15.725  1.00 97.75           N  \\nATOM   1366  H   LEU A  92       8.299  -1.054 -16.347  1.00 97.75           H  \\nATOM   1367  CA  LEU A  92       8.520  -0.442 -14.315  1.00 97.75           C  \\nATOM   1368  HA  LEU A  92       7.628  -0.240 -13.721  1.00 97.75           H  \\nATOM   1369  C   LEU A  92       9.601   0.557 -13.876  1.00 97.75           C  \\nATOM   1370  CB  LEU A  92       8.991  -1.886 -14.054  1.00 97.75           C  \\nATOM   1371  HB2 LEU A  92       9.326  -1.967 -13.020  1.00 97.75           H  \\nATOM   1372  HB3 LEU A  92       9.855  -2.091 -14.686  1.00 97.75           H  \\nATOM   1373  O   LEU A  92      10.411   0.988 -14.702  1.00 97.75           O  \\nATOM   1374  CG  LEU A  92       7.931  -2.968 -14.301  1.00 97.75           C  \\nATOM   1375  HG  LEU A  92       7.726  -3.009 -15.371  1.00 97.75           H  \\nATOM   1376  CD1 LEU A  92       8.481  -4.330 -13.879  1.00 97.75           C  \\nATOM   1377 HD11 LEU A  92       7.758  -5.107 -14.125  1.00 97.75           H  \\nATOM   1378 HD12 LEU A  92       8.665  -4.339 -12.804  1.00 97.75           H  \\nATOM   1379 HD13 LEU A  92       9.411  -4.534 -14.409  1.00 97.75           H  \\nATOM   1380  CD2 LEU A  92       6.621  -2.692 -13.561  1.00 97.75           C  \\nATOM   1381 HD21 LEU A  92       6.112  -1.835 -14.000  1.00 97.75           H  \\nATOM   1382 HD22 LEU A  92       5.968  -3.559 -13.658  1.00 97.75           H  \\nATOM   1383 HD23 LEU A  92       6.817  -2.502 -12.505  1.00 97.75           H  \\nATOM   1384  N   PRO A  93       9.625   0.937 -12.587  1.00 96.92           N  \\nATOM   1385  CA  PRO A  93      10.683   1.794 -12.078  1.00 96.92           C  \\nATOM   1386  HA  PRO A  93      10.691   2.722 -12.650  1.00 96.92           H  \\nATOM   1387  C   PRO A  93      12.054   1.111 -12.210  1.00 96.92           C  \\nATOM   1388  CB  PRO A  93      10.318   2.097 -10.626  1.00 96.92           C  \\nATOM   1389  HB2 PRO A  93       9.753   3.027 -10.572  1.00 96.92           H  \\nATOM   1390  HB3 PRO A  93      11.205   2.156  -9.995  1.00 96.92           H  \\nATOM   1391  O   PRO A  93      12.137  -0.123 -12.172  1.00 96.92           O  \\nATOM   1392  CG  PRO A  93       9.417   0.935 -10.214  1.00 96.92           C  \\nATOM   1393  HG2 PRO A  93       8.676   1.239  -9.474  1.00 96.92           H  \\nATOM   1394  HG3 PRO A  93      10.032   0.123  -9.826  1.00 96.92           H  \\nATOM   1395  CD  PRO A  93       8.747   0.495 -11.510  1.00 96.92           C  \\nATOM   1396  HD2 PRO A  93       7.775   0.981 -11.596  1.00 96.92           H  \\nATOM   1397  HD3 PRO A  93       8.616  -0.587 -11.502  1.00 96.92           H  \\nATOM   1398  N   PRO A  94      13.137   1.896 -12.355  1.00 94.10           N  \\nATOM   1399  CA  PRO A  94      14.483   1.344 -12.380  1.00 94.10           C  \\nATOM   1400  HA  PRO A  94      14.574   0.625 -13.194  1.00 94.10           H  \\nATOM   1401  C   PRO A  94      14.793   0.651 -11.052  1.00 94.10           C  \\nATOM   1402  CB  PRO A  94      15.413   2.533 -12.636  1.00 94.10           C  \\nATOM   1403  HB2 PRO A  94      16.388   2.405 -12.165  1.00 94.10           H  \\nATOM   1404  HB3 PRO A  94      15.527   2.681 -13.710  1.00 94.10           H  \\nATOM   1405  O   PRO A  94      14.376   1.101  -9.985  1.00 94.10           O  \\nATOM   1406  CG  PRO A  94      14.643   3.710 -12.038  1.00 94.10           C  \\nATOM   1407  HG2 PRO A  94      14.804   3.745 -10.960  1.00 94.10           H  \\nATOM   1408  HG3 PRO A  94      14.927   4.657 -12.498  1.00 94.10           H  \\nATOM   1409  CD  PRO A  94      13.186   3.352 -12.328  1.00 94.10           C  \\nATOM   1410  HD2 PRO A  94      12.544   3.762 -11.549  1.00 94.10           H  \\nATOM   1411  HD3 PRO A  94      12.896   3.742 -13.304  1.00 94.10           H  \\nATOM   1412  N   LYS A  95      15.557  -0.441 -11.118  1.00 93.95           N  \\nATOM   1413  H   LYS A  95      15.888  -0.734 -12.026  1.00 93.95           H  \\nATOM   1414  CA  LYS A  95      16.084  -1.110  -9.924  1.00 93.95           C  \\nATOM   1415  HA  LYS A  95      15.262  -1.344  -9.248  1.00 93.95           H  \\nATOM   1416  C   LYS A  95      17.049  -0.189  -9.186  1.00 93.95           C  \\nATOM   1417  CB  LYS A  95      16.786  -2.421 -10.288  1.00 93.95           C  \\nATOM   1418  HB2 LYS A  95      17.210  -2.857  -9.383  1.00 93.95           H  \\nATOM   1419  HB3 LYS A  95      17.598  -2.215 -10.985  1.00 93.95           H  \\nATOM   1420  O   LYS A  95      17.809   0.538  -9.820  1.00 93.95           O  \\nATOM   1421  CG  LYS A  95      15.814  -3.427 -10.908  1.00 93.95           C  \\nATOM   1422  HG2 LYS A  95      15.001  -3.624 -10.209  1.00 93.95           H  \\nATOM   1423  HG3 LYS A  95      15.400  -3.016 -11.829  1.00 93.95           H  \\nATOM   1424  CD  LYS A  95      16.545  -4.733 -11.225  1.00 93.95           C  \\nATOM   1425  HD2 LYS A  95      17.393  -4.544 -11.884  1.00 93.95           H  \\nATOM   1426  HD3 LYS A  95      16.887  -5.193 -10.297  1.00 93.95           H  \\nATOM   1427  CE  LYS A  95      15.530  -5.620 -11.923  1.00 93.95           C  \\nATOM   1428  HE2 LYS A  95      15.216  -5.146 -12.853  1.00 93.95           H  \\nATOM   1429  HE3 LYS A  95      14.645  -5.675 -11.290  1.00 93.95           H  \\nATOM   1430  NZ  LYS A  95      15.991  -6.998 -12.204  1.00 93.95           N  \\nATOM   1431  HZ1 LYS A  95      16.724  -7.004 -12.899  1.00 93.95           H  \\nATOM   1432  HZ2 LYS A  95      15.202  -7.509 -12.572  1.00 93.95           H  \\nATOM   1433  HZ3 LYS A  95      16.302  -7.443 -11.353  1.00 93.95           H  \\nATOM   1434  N   ASN A  96      17.099  -0.329  -7.866  1.00 94.49           N  \\nATOM   1435  H   ASN A  96      16.415  -0.920  -7.415  1.00 94.49           H  \\nATOM   1436  CA  ASN A  96      18.237   0.112  -7.070  1.00 94.49           C  \\nATOM   1437  HA  ASN A  96      18.942   0.657  -7.697  1.00 94.49           H  \\nATOM   1438  C   ASN A  96      18.952  -1.123  -6.496  1.00 94.49           C  \\nATOM   1439  CB  ASN A  96      17.783   1.095  -5.980  1.00 94.49           C  \\nATOM   1440  HB2 ASN A  96      17.281   1.944  -6.444  1.00 94.49           H  \\nATOM   1441  HB3 ASN A  96      17.092   0.611  -5.291  1.00 94.49           H  \\nATOM   1442  O   ASN A  96      18.536  -1.609  -5.442  1.00 94.49           O  \\nATOM   1443  CG  ASN A  96      18.980   1.618  -5.201  1.00 94.49           C  \\nATOM   1444  ND2 ASN A  96      18.782   2.408  -4.183  1.00 94.49           N  \\nATOM   1445 HD21 ASN A  96      17.871   2.534  -3.765  1.00 94.49           H  \\nATOM   1446 HD22 ASN A  96      19.601   2.730  -3.686  1.00 94.49           H  \\nATOM   1447  OD1 ASN A  96      20.127   1.316  -5.493  1.00 94.49           O  \\nATOM   1448  N   PRO A  97      20.006  -1.644  -7.156  1.00 92.85           N  \\nATOM   1449  CA  PRO A  97      20.705  -2.854  -6.718  1.00 92.85           C  \\nATOM   1450  HA  PRO A  97      20.013  -3.694  -6.778  1.00 92.85           H  \\nATOM   1451  C   PRO A  97      21.260  -2.783  -5.291  1.00 92.85           C  \\nATOM   1452  CB  PRO A  97      21.834  -3.064  -7.730  1.00 92.85           C  \\nATOM   1453  HB2 PRO A  97      22.737  -2.549  -7.400  1.00 92.85           H  \\nATOM   1454  HB3 PRO A  97      22.038  -4.123  -7.890  1.00 92.85           H  \\nATOM   1455  O   PRO A  97      21.342  -3.817  -4.635  1.00 92.85           O  \\nATOM   1456  CG  PRO A  97      21.306  -2.391  -8.993  1.00 92.85           C  \\nATOM   1457  HG2 PRO A  97      22.113  -2.078  -9.656  1.00 92.85           H  \\nATOM   1458  HG3 PRO A  97      20.621  -3.064  -9.509  1.00 92.85           H  \\nATOM   1459  CD  PRO A  97      20.535  -1.200  -8.437  1.00 92.85           C  \\nATOM   1460  HD2 PRO A  97      19.745  -0.923  -9.136  1.00 92.85           H  \\nATOM   1461  HD3 PRO A  97      21.210  -0.359  -8.279  1.00 92.85           H  \\nATOM   1462  N   SER A  98      21.577  -1.583  -4.791  1.00 94.02           N  \\nATOM   1463  H   SER A  98      21.519  -0.764  -5.380  1.00 94.02           H  \\nATOM   1464  CA  SER A  98      22.026  -1.375  -3.407  1.00 94.02           C  \\nATOM   1465  HA  SER A  98      22.860  -2.046  -3.200  1.00 94.02           H  \\nATOM   1466  C   SER A  98      20.934  -1.663  -2.370  1.00 94.02           C  \\nATOM   1467  CB  SER A  98      22.512   0.065  -3.223  1.00 94.02           C  \\nATOM   1468  HB2 SER A  98      22.920   0.182  -2.219  1.00 94.02           H  \\nATOM   1469  HB3 SER A  98      21.673   0.750  -3.345  1.00 94.02           H  \\nATOM   1470  O   SER A  98      21.243  -1.891  -1.206  1.00 94.02           O  \\nATOM   1471  OG  SER A  98      23.512   0.374  -4.176  1.00 94.02           O  \\nATOM   1472  HG  SER A  98      23.820   1.269  -4.017  1.00 94.02           H  \\nATOM   1473  N   TRP A  99      19.662  -1.659  -2.780  1.00 94.17           N  \\nATOM   1474  H   TRP A  99      19.487  -1.464  -3.755  1.00 94.17           H  \\nATOM   1475  CA  TRP A  99      18.512  -1.945  -1.920  1.00 94.17           C  \\nATOM   1476  HA  TRP A  99      18.838  -2.026  -0.883  1.00 94.17           H  \\nATOM   1477  C   TRP A  99      17.821  -3.272  -2.272  1.00 94.17           C  \\nATOM   1478  CB  TRP A  99      17.549  -0.759  -2.014  1.00 94.17           C  \\nATOM   1479  HB2 TRP A  99      17.204  -0.653  -3.043  1.00 94.17           H  \\nATOM   1480  HB3 TRP A  99      18.092   0.150  -1.754  1.00 94.17           H  \\nATOM   1481  O   TRP A  99      17.395  -4.021  -1.389  1.00 94.17           O  \\nATOM   1482  CG  TRP A  99      16.361  -0.862  -1.117  1.00 94.17           C  \\nATOM   1483  CD1 TRP A  99      16.334  -0.530   0.191  1.00 94.17           C  \\nATOM   1484  HD1 TRP A  99      17.172  -0.128   0.741  1.00 94.17           H  \\nATOM   1485  CD2 TRP A  99      15.025  -1.341  -1.432  1.00 94.17           C  \\nATOM   1486  CE2 TRP A  99      14.221  -1.250  -0.256  1.00 94.17           C  \\nATOM   1487  CE3 TRP A  99      14.419  -1.843  -2.600  1.00 94.17           C  \\nATOM   1488  HE3 TRP A  99      15.005  -1.921  -3.504  1.00 94.17           H  \\nATOM   1489  NE1 TRP A  99      15.072  -0.750   0.704  1.00 94.17           N  \\nATOM   1490  HE1 TRP A  99      14.821  -0.508   1.652  1.00 94.17           H  \\nATOM   1491  CH2 TRP A  99      12.291  -2.095  -1.428  1.00 94.17           C  \\nATOM   1492  HH2 TRP A  99      11.243  -2.355  -1.455  1.00 94.17           H  \\nATOM   1493  CZ2 TRP A  99      12.867  -1.610  -0.244  1.00 94.17           C  \\nATOM   1494  HZ2 TRP A  99      12.278  -1.498   0.655  1.00 94.17           H  \\nATOM   1495  CZ3 TRP A  99      13.071  -2.231  -2.586  1.00 94.17           C  \\nATOM   1496  HZ3 TRP A  99      12.629  -2.638  -3.483  1.00 94.17           H  \\nATOM   1497  N   SER A 100      17.697  -3.588  -3.565  1.00 93.61           N  \\nATOM   1498  H   SER A 100      18.050  -2.932  -4.247  1.00 93.61           H  \\nATOM   1499  CA  SER A 100      17.191  -4.871  -4.058  1.00 93.61           C  \\nATOM   1500  HA  SER A 100      17.604  -5.668  -3.439  1.00 93.61           H  \\nATOM   1501  C   SER A 100      17.579  -5.112  -5.516  1.00 93.61           C  \\nATOM   1502  CB  SER A 100      15.664  -4.911  -3.955  1.00 93.61           C  \\nATOM   1503  HB2 SER A 100      15.235  -4.190  -4.651  1.00 93.61           H  \\nATOM   1504  HB3 SER A 100      15.359  -4.635  -2.946  1.00 93.61           H  \\nATOM   1505  O   SER A 100      17.405  -4.249  -6.376  1.00 93.61           O  \\nATOM   1506  OG  SER A 100      15.171  -6.208  -4.253  1.00 93.61           O  \\nATOM   1507  HG  SER A 100      14.245  -6.214  -3.998  1.00 93.61           H  \\nATOM   1508  N   ALA A 101      18.011  -6.336  -5.825  1.00 93.43           N  \\nATOM   1509  H   ALA A 101      18.183  -6.986  -5.072  1.00 93.43           H  \\nATOM   1510  CA  ALA A 101      18.237  -6.785  -7.201  1.00 93.43           C  \\nATOM   1511  HA  ALA A 101      18.656  -5.957  -7.774  1.00 93.43           H  \\nATOM   1512  C   ALA A 101      16.942  -7.228  -7.927  1.00 93.43           C  \\nATOM   1513  CB  ALA A 101      19.283  -7.905  -7.160  1.00 93.43           C  \\nATOM   1514  HB1 ALA A 101      19.522  -8.222  -8.175  1.00 93.43           H  \\nATOM   1515  HB2 ALA A 101      18.901  -8.759  -6.600  1.00 93.43           H  \\nATOM   1516  HB3 ALA A 101      20.195  -7.542  -6.686  1.00 93.43           H  \\nATOM   1517  O   ALA A 101      16.956  -7.496  -9.134  1.00 93.43           O  \\nATOM   1518  N   GLN A 102      15.819  -7.341  -7.206  1.00 95.66           N  \\nATOM   1519  H   GLN A 102      15.827  -7.011  -6.251  1.00 95.66           H  \\nATOM   1520  CA  GLN A 102      14.559  -7.844  -7.756  1.00 95.66           C  \\nATOM   1521  HA  GLN A 102      14.785  -8.677  -8.422  1.00 95.66           H  \\nATOM   1522  C   GLN A 102      13.812  -6.787  -8.577  1.00 95.66           C  \\nATOM   1523  CB  GLN A 102      13.644  -8.370  -6.642  1.00 95.66           C  \\nATOM   1524  HB2 GLN A 102      12.697  -8.662  -7.096  1.00 95.66           H  \\nATOM   1525  HB3 GLN A 102      13.455  -7.569  -5.928  1.00 95.66           H  \\nATOM   1526  O   GLN A 102      13.802  -5.607  -8.241  1.00 95.66           O  \\nATOM   1527  CG  GLN A 102      14.216  -9.582  -5.894  1.00 95.66           C  \\nATOM   1528  HG2 GLN A 102      15.103  -9.279  -5.338  1.00 95.66           H  \\nATOM   1529  HG3 GLN A 102      14.510 -10.345  -6.616  1.00 95.66           H  \\nATOM   1530  CD  GLN A 102      13.217 -10.207  -4.919  1.00 95.66           C  \\nATOM   1531  NE2 GLN A 102      13.637 -11.203  -4.171  1.00 95.66           N  \\nATOM   1532 HE21 GLN A 102      12.980 -11.584  -3.507  1.00 95.66           H  \\nATOM   1533 HE22 GLN A 102      14.602 -11.499  -4.198  1.00 95.66           H  \\nATOM   1534  OE1 GLN A 102      12.050  -9.844  -4.826  1.00 95.66           O  \\nATOM   1535  N   ASP A 103      13.126  -7.236  -9.630  1.00 95.83           N  \\nATOM   1536  H   ASP A 103      13.195  -8.211  -9.883  1.00 95.83           H  \\nATOM   1537  CA  ASP A 103      12.181  -6.408 -10.383  1.00 95.83           C  \\nATOM   1538  HA  ASP A 103      12.684  -5.492 -10.693  1.00 95.83           H  \\nATOM   1539  C   ASP A 103      10.961  -6.013  -9.545  1.00 95.83           C  \\nATOM   1540  CB  ASP A 103      11.708  -7.148 -11.641  1.00 95.83           C  \\nATOM   1541  HB2 ASP A 103      11.262  -8.102 -11.359  1.00 95.83           H  \\nATOM   1542  HB3 ASP A 103      10.945  -6.554 -12.144  1.00 95.83           H  \\nATOM   1543  O   ASP A 103      10.408  -6.819  -8.789  1.00 95.83           O  \\nATOM   1544  CG  ASP A 103      12.864  -7.378 -12.602  1.00 95.83           C  \\nATOM   1545  OD1 ASP A 103      13.306  -6.400 -13.225  1.00 95.83           O  \\nATOM   1546  OD2 ASP A 103      13.424  -8.494 -12.608  1.00 95.83           O  \\nATOM   1547  N   PHE A 104      10.480  -4.791  -9.764  1.00 97.73           N  \\nATOM   1548  H   PHE A 104      10.992  -4.179 -10.383  1.00 97.73           H  \\nATOM   1549  CA  PHE A 104       9.274  -4.244  -9.147  1.00 97.73           C  \\nATOM   1550  HA  PHE A 104       9.209  -4.630  -8.130  1.00 97.73           H  \\nATOM   1551  C   PHE A 104       8.006  -4.690  -9.889  1.00 97.73           C  \\nATOM   1552  CB  PHE A 104       9.409  -2.723  -9.058  1.00 97.73           C  \\nATOM   1553  HB2 PHE A 104       8.509  -2.319  -8.594  1.00 97.73           H  \\nATOM   1554  HB3 PHE A 104       9.489  -2.308 -10.063  1.00 97.73           H  \\nATOM   1555  O   PHE A 104       7.292  -3.886 -10.475  1.00 97.73           O  \\nATOM   1556  CG  PHE A 104      10.586  -2.273  -8.226  1.00 97.73           C  \\nATOM   1557  CD1 PHE A 104      10.532  -2.430  -6.834  1.00 97.73           C  \\nATOM   1558  HD1 PHE A 104       9.655  -2.859  -6.375  1.00 97.73           H  \\nATOM   1559  CD2 PHE A 104      11.704  -1.656  -8.815  1.00 97.73           C  \\nATOM   1560  HD2 PHE A 104      11.747  -1.494  -9.882  1.00 97.73           H  \\nATOM   1561  CE1 PHE A 104      11.581  -1.972  -6.030  1.00 97.73           C  \\nATOM   1562  HE1 PHE A 104      11.464  -2.008  -4.957  1.00 97.73           H  \\nATOM   1563  CE2 PHE A 104      12.747  -1.186  -8.000  1.00 97.73           C  \\nATOM   1564  HE2 PHE A 104      13.554  -0.616  -8.436  1.00 97.73           H  \\nATOM   1565  CZ  PHE A 104      12.707  -1.368  -6.609  1.00 97.73           C  \\nATOM   1566  HZ  PHE A 104      13.492  -0.962  -5.988  1.00 97.73           H  \\nATOM   1567  N   LYS A 105       7.759  -6.001  -9.932  1.00 97.47           N  \\nATOM   1568  H   LYS A 105       8.412  -6.595  -9.442  1.00 97.47           H  \\nATOM   1569  CA  LYS A 105       6.662  -6.609 -10.711  1.00 97.47           C  \\nATOM   1570  HA  LYS A 105       6.230  -5.839 -11.351  1.00 97.47           H  \\nATOM   1571  C   LYS A 105       5.491  -7.130  -9.871  1.00 97.47           C  \\nATOM   1572  CB  LYS A 105       7.233  -7.667 -11.658  1.00 97.47           C  \\nATOM   1573  HB2 LYS A 105       6.444  -8.001 -12.332  1.00 97.47           H  \\nATOM   1574  HB3 LYS A 105       8.014  -7.205 -12.262  1.00 97.47           H  \\nATOM   1575  O   LYS A 105       4.568  -7.741 -10.413  1.00 97.47           O  \\nATOM   1576  CG  LYS A 105       7.805  -8.881 -10.913  1.00 97.47           C  \\nATOM   1577  HG2 LYS A 105       7.011  -9.391 -10.369  1.00 97.47           H  \\nATOM   1578  HG3 LYS A 105       8.571  -8.558 -10.208  1.00 97.47           H  \\nATOM   1579  CD  LYS A 105       8.431  -9.847 -11.920  1.00 97.47           C  \\nATOM   1580  HD2 LYS A 105       7.667 -10.191 -12.617  1.00 97.47           H  \\nATOM   1581  HD3 LYS A 105       9.203  -9.316 -12.478  1.00 97.47           H  \\nATOM   1582  CE  LYS A 105       9.055 -11.040 -11.193  1.00 97.47           C  \\nATOM   1583  HE2 LYS A 105       8.265 -11.610 -10.704  1.00 97.47           H  \\nATOM   1584  HE3 LYS A 105       9.724 -10.659 -10.421  1.00 97.47           H  \\nATOM   1585  NZ  LYS A 105       9.815 -11.894 -12.140  1.00 97.47           N  \\nATOM   1586  HZ1 LYS A 105       9.225 -12.218 -12.893  1.00 97.47           H  \\nATOM   1587  HZ2 LYS A 105      10.222 -12.690 -11.670  1.00 97.47           H  \\nATOM   1588  HZ3 LYS A 105      10.568 -11.362 -12.552  1.00 97.47           H  \\nATOM   1589  N   SER A 106       5.539  -6.941  -8.557  1.00 97.62           N  \\nATOM   1590  H   SER A 106       6.279  -6.364  -8.182  1.00 97.62           H  \\nATOM   1591  CA  SER A 106       4.496  -7.364  -7.622  1.00 97.62           C  \\nATOM   1592  HA  SER A 106       3.932  -8.184  -8.066  1.00 97.62           H  \\nATOM   1593  C   SER A 106       3.567  -6.185  -7.322  1.00 97.62           C  \\nATOM   1594  CB  SER A 106       5.106  -7.887  -6.320  1.00 97.62           C  \\nATOM   1595  HB2 SER A 106       5.653  -7.086  -5.823  1.00 97.62           H  \\nATOM   1596  HB3 SER A 106       4.307  -8.224  -5.659  1.00 97.62           H  \\nATOM   1597  O   SER A 106       4.068  -5.094  -7.062  1.00 97.62           O  \\nATOM   1598  OG  SER A 106       5.978  -8.975  -6.577  1.00 97.62           O  \\nATOM   1599  HG  SER A 106       6.512  -9.092  -5.788  1.00 97.62           H  \\nATOM   1600  N   PRO A 107       2.236  -6.359  -7.352  1.00 97.59           N  \\nATOM   1601  CA  PRO A 107       1.314  -5.287  -6.998  1.00 97.59           C  \\nATOM   1602  HA  PRO A 107       1.649  -4.334  -7.408  1.00 97.59           H  \\nATOM   1603  C   PRO A 107       1.220  -5.139  -5.479  1.00 97.59           C  \\nATOM   1604  CB  PRO A 107      -0.020  -5.686  -7.638  1.00 97.59           C  \\nATOM   1605  HB2 PRO A 107      -0.060  -5.318  -8.663  1.00 97.59           H  \\nATOM   1606  HB3 PRO A 107      -0.872  -5.314  -7.070  1.00 97.59           H  \\nATOM   1607  O   PRO A 107       0.711  -6.036  -4.813  1.00 97.59           O  \\nATOM   1608  CG  PRO A 107       0.028  -7.214  -7.653  1.00 97.59           C  \\nATOM   1609  HG2 PRO A 107      -0.543  -7.616  -8.491  1.00 97.59           H  \\nATOM   1610  HG3 PRO A 107      -0.353  -7.605  -6.710  1.00 97.59           H  \\nATOM   1611  CD  PRO A 107       1.513  -7.545  -7.788  1.00 97.59           C  \\nATOM   1612  HD2 PRO A 107       1.748  -7.749  -8.833  1.00 97.59           H  \\nATOM   1613  HD3 PRO A 107       1.761  -8.406  -7.167  1.00 97.59           H  \\nATOM   1614  N   GLU A 108       1.712  -4.026  -4.939  1.00 98.03           N  \\nATOM   1615  H   GLU A 108       2.139  -3.345  -5.550  1.00 98.03           H  \\nATOM   1616  CA  GLU A 108       1.227  -3.504  -3.656  1.00 98.03           C  \\nATOM   1617  HA  GLU A 108       1.193  -4.315  -2.929  1.00 98.03           H  \\nATOM   1618  C   GLU A 108      -0.196  -2.993  -3.828  1.00 98.03           C  \\nATOM   1619  CB  GLU A 108       2.151  -2.392  -3.131  1.00 98.03           C  \\nATOM   1620  HB2 GLU A 108       2.185  -1.571  -3.847  1.00 98.03           H  \\nATOM   1621  HB3 GLU A 108       3.155  -2.806  -3.034  1.00 98.03           H  \\nATOM   1622  O   GLU A 108      -0.501  -2.484  -4.905  1.00 98.03           O  \\nATOM   1623  CG  GLU A 108       1.727  -1.842  -1.759  1.00 98.03           C  \\nATOM   1624  HG2 GLU A 108       2.512  -1.180  -1.394  1.00 98.03           H  \\nATOM   1625  HG3 GLU A 108       0.817  -1.254  -1.875  1.00 98.03           H  \\nATOM   1626  CD  GLU A 108       1.506  -2.987  -0.763  1.00 98.03           C  \\nATOM   1627  OE1 GLU A 108       0.487  -2.988  -0.048  1.00 98.03           O  \\nATOM   1628  OE2 GLU A 108       2.279  -3.967  -0.848  1.00 98.03           O  \\nATOM   1629  N   GLU A 109      -1.056  -3.136  -2.817  1.00 96.89           N  \\nATOM   1630  H   GLU A 109      -0.718  -3.405  -1.904  1.00 96.89           H  \\nATOM   1631  CA  GLU A 109      -2.437  -2.664  -2.894  1.00 96.89           C  \\nATOM   1632  HA  GLU A 109      -2.458  -1.864  -3.634  1.00 96.89           H  \\nATOM   1633  C   GLU A 109      -2.936  -2.066  -1.572  1.00 96.89           C  \\nATOM   1634  CB  GLU A 109      -3.377  -3.756  -3.437  1.00 96.89           C  \\nATOM   1635  HB2 GLU A 109      -2.952  -4.151  -4.360  1.00 96.89           H  \\nATOM   1636  HB3 GLU A 109      -4.333  -3.294  -3.684  1.00 96.89           H  \\nATOM   1637  O   GLU A 109      -2.950  -2.733  -0.535  1.00 96.89           O  \\nATOM   1638  CG  GLU A 109      -3.628  -4.924  -2.469  1.00 96.89           C  \\nATOM   1639  HG2 GLU A 109      -4.230  -4.583  -1.626  1.00 96.89           H  \\nATOM   1640  HG3 GLU A 109      -2.674  -5.290  -2.090  1.00 96.89           H  \\nATOM   1641  CD  GLU A 109      -4.378  -6.080  -3.123  1.00 96.89           C  \\nATOM   1642  OE1 GLU A 109      -4.073  -7.245  -2.782  1.00 96.89           O  \\nATOM   1643  OE2 GLU A 109      -5.295  -5.844  -3.936  1.00 96.89           O  \\nATOM   1644  N   TYR A 110      -3.433  -0.826  -1.634  1.00 98.70           N  \\nATOM   1645  H   TYR A 110      -3.350  -0.321  -2.505  1.00 98.70           H  \\nATOM   1646  CA  TYR A 110      -4.021  -0.141  -0.484  1.00 98.70           C  \\nATOM   1647  HA  TYR A 110      -4.096  -0.873   0.320  1.00 98.70           H  \\nATOM   1648  C   TYR A 110      -5.412   0.446  -0.782  1.00 98.70           C  \\nATOM   1649  CB  TYR A 110      -3.085   0.940   0.050  1.00 98.70           C  \\nATOM   1650  HB2 TYR A 110      -2.067   0.549   0.067  1.00 98.70           H  \\nATOM   1651  HB3 TYR A 110      -3.095   1.806  -0.612  1.00 98.70           H  \\nATOM   1652  O   TYR A 110      -5.584   1.173  -1.754  1.00 98.70           O  \\nATOM   1653  CG  TYR A 110      -3.488   1.342   1.453  1.00 98.70           C  \\nATOM   1654  CD1 TYR A 110      -4.359   2.428   1.663  1.00 98.70           C  \\nATOM   1655  HD1 TYR A 110      -4.686   3.018   0.820  1.00 98.70           H  \\nATOM   1656  CD2 TYR A 110      -3.076   0.550   2.541  1.00 98.70           C  \\nATOM   1657  HD2 TYR A 110      -2.407  -0.280   2.369  1.00 98.70           H  \\nATOM   1658  CE1 TYR A 110      -4.801   2.734   2.965  1.00 98.70           C  \\nATOM   1659  HE1 TYR A 110      -5.467   3.565   3.150  1.00 98.70           H  \\nATOM   1660  CE2 TYR A 110      -3.497   0.865   3.844  1.00 98.70           C  \\nATOM   1661  HE2 TYR A 110      -3.136   0.275   4.674  1.00 98.70           H  \\nATOM   1662  OH  TYR A 110      -4.730   2.276   5.325  1.00 98.70           O  \\nATOM   1663  HH  TYR A 110      -4.292   1.707   5.963  1.00 98.70           H  \\nATOM   1664  CZ  TYR A 110      -4.353   1.964   4.059  1.00 98.70           C  \\nATOM   1665  N   ALA A 111      -6.442   0.249   0.042  1.00 98.44           N  \\nATOM   1666  H   ALA A 111      -7.323   0.645  -0.254  1.00 98.44           H  \\nATOM   1667  CA  ALA A 111      -6.422  -0.468   1.315  1.00 98.44           C  \\nATOM   1668  HA  ALA A 111      -5.647  -0.037   1.949  1.00 98.44           H  \\nATOM   1669  C   ALA A 111      -6.115  -1.962   1.150  1.00 98.44           C  \\nATOM   1670  CB  ALA A 111      -7.759  -0.245   2.023  1.00 98.44           C  \\nATOM   1671  HB1 ALA A 111      -8.570  -0.642   1.414  1.00 98.44           H  \\nATOM   1672  HB2 ALA A 111      -7.911   0.823   2.182  1.00 98.44           H  \\nATOM   1673  HB3 ALA A 111      -7.754  -0.748   2.990  1.00 98.44           H  \\nATOM   1674  O   ALA A 111      -6.421  -2.558   0.116  1.00 98.44           O  \\nATOM   1675  N   PHE A 112      -5.517  -2.553   2.189  1.00 97.95           N  \\nATOM   1676  H   PHE A 112      -5.227  -1.976   2.966  1.00 97.95           H  \\nATOM   1677  CA  PHE A 112      -5.054  -3.937   2.163  1.00 97.95           C  \\nATOM   1678  HA  PHE A 112      -4.198  -4.001   1.491  1.00 97.95           H  \\nATOM   1679  C   PHE A 112      -6.145  -4.886   1.674  1.00 97.95           C  \\nATOM   1680  CB  PHE A 112      -4.640  -4.384   3.576  1.00 97.95           C  \\nATOM   1681  HB2 PHE A 112      -5.525  -4.406   4.213  1.00 97.95           H  \\nATOM   1682  HB3 PHE A 112      -4.255  -5.402   3.511  1.00 97.95           H  \\nATOM   1683  O   PHE A 112      -7.282  -4.820   2.156  1.00 97.95           O  \\nATOM   1684  CG  PHE A 112      -3.578  -3.538   4.239  1.00 97.95           C  \\nATOM   1685  CD1 PHE A 112      -2.354  -3.327   3.583  1.00 97.95           C  \\nATOM   1686  HD1 PHE A 112      -2.157  -3.740   2.605  1.00 97.95           H  \\nATOM   1687  CD2 PHE A 112      -3.800  -2.969   5.509  1.00 97.95           C  \\nATOM   1688  HD2 PHE A 112      -4.736  -3.140   6.020  1.00 97.95           H  \\nATOM   1689  CE1 PHE A 112      -1.368  -2.542   4.190  1.00 97.95           C  \\nATOM   1690  HE1 PHE A 112      -0.440  -2.372   3.664  1.00 97.95           H  \\nATOM   1691  CE2 PHE A 112      -2.807  -2.176   6.114  1.00 97.95           C  \\nATOM   1692  HE2 PHE A 112      -2.976  -1.725   7.081  1.00 97.95           H  \\nATOM   1693  CZ  PHE A 112      -1.588  -1.959   5.448  1.00 97.95           C  \\nATOM   1694  HZ  PHE A 112      -0.819  -1.337   5.881  1.00 97.95           H  \\nATOM   1695  N   ALA A 113      -5.797  -5.861   0.831  1.00 97.70           N  \\nATOM   1696  H   ALA A 113      -4.903  -5.810   0.362  1.00 97.70           H  \\nATOM   1697  CA  ALA A 113      -6.735  -6.929   0.487  1.00 97.70           C  \\nATOM   1698  HA  ALA A 113      -7.599  -6.482  -0.006  1.00 97.70           H  \\nATOM   1699  C   ALA A 113      -7.257  -7.670   1.724  1.00 97.70           C  \\nATOM   1700  CB  ALA A 113      -6.087  -7.908  -0.488  1.00 97.70           C  \\nATOM   1701  HB1 ALA A 113      -5.065  -8.146  -0.191  1.00 97.70           H  \\nATOM   1702  HB2 ALA A 113      -6.678  -8.822  -0.546  1.00 97.70           H  \\nATOM   1703  HB3 ALA A 113      -6.096  -7.451  -1.478  1.00 97.70           H  \\nATOM   1704  O   ALA A 113      -8.386  -8.143   1.718  1.00 97.70           O  \\nATOM   1705  N   SER A 114      -6.489  -7.737   2.814  1.00 97.06           N  \\nATOM   1706  H   SER A 114      -5.590  -7.275   2.801  1.00 97.06           H  \\nATOM   1707  CA  SER A 114      -6.924  -8.317   4.089  1.00 97.06           C  \\nATOM   1708  HA  SER A 114      -7.458  -9.245   3.885  1.00 97.06           H  \\nATOM   1709  C   SER A 114      -7.896  -7.435   4.884  1.00 97.06           C  \\nATOM   1710  CB  SER A 114      -5.703  -8.657   4.949  1.00 97.06           C  \\nATOM   1711  HB2 SER A 114      -5.106  -9.419   4.448  1.00 97.06           H  \\nATOM   1712  HB3 SER A 114      -6.021  -9.039   5.919  1.00 97.06           H  \\nATOM   1713  O   SER A 114      -8.224  -7.798   6.011  1.00 97.06           O  \\nATOM   1714  OG  SER A 114      -4.923  -7.490   5.119  1.00 97.06           O  \\nATOM   1715  HG  SER A 114      -4.450  -7.496   5.954  1.00 97.06           H  \\nATOM   1716  N   SER A 115      -8.370  -6.310   4.341  1.00 98.21           N  \\nATOM   1717  H   SER A 115      -8.082  -6.069   3.404  1.00 98.21           H  \\nATOM   1718  CA  SER A 115      -9.331  -5.407   4.987  1.00 98.21           C  \\nATOM   1719  HA  SER A 115      -9.517  -5.780   5.994  1.00 98.21           H  \\nATOM   1720  C   SER A 115     -10.681  -5.349   4.264  1.00 98.21           C  \\nATOM   1721  CB  SER A 115      -8.737  -4.004   5.137  1.00 98.21           C  \\nATOM   1722  HB2 SER A 115      -9.359  -3.433   5.827  1.00 98.21           H  \\nATOM   1723  HB3 SER A 115      -7.732  -4.073   5.554  1.00 98.21           H  \\nATOM   1724  O   SER A 115     -10.764  -5.612   3.068  1.00 98.21           O  \\nATOM   1725  OG  SER A 115      -8.698  -3.317   3.903  1.00 98.21           O  \\nATOM   1726  HG  SER A 115      -8.184  -3.819   3.266  1.00 98.21           H  \\nATOM   1727  N   LEU A 116     -11.745  -4.961   4.971  1.00 97.74           N  \\nATOM   1728  H   LEU A 116     -11.624  -4.805   5.962  1.00 97.74           H  \\nATOM   1729  CA  LEU A 116     -13.079  -4.742   4.397  1.00 97.74           C  \\nATOM   1730  HA  LEU A 116     -13.411  -5.660   3.913  1.00 97.74           H  \\nATOM   1731  C   LEU A 116     -13.061  -3.640   3.327  1.00 97.74           C  \\nATOM   1732  CB  LEU A 116     -14.043  -4.383   5.546  1.00 97.74           C  \\nATOM   1733  HB2 LEU A 116     -14.034  -5.186   6.283  1.00 97.74           H  \\nATOM   1734  HB3 LEU A 116     -13.675  -3.481   6.035  1.00 97.74           H  \\nATOM   1735  O   LEU A 116     -13.811  -3.696   2.358  1.00 97.74           O  \\nATOM   1736  CG  LEU A 116     -15.499  -4.133   5.100  1.00 97.74           C  \\nATOM   1737  HG  LEU A 116     -15.525  -3.380   4.312  1.00 97.74           H  \\nATOM   1738  CD1 LEU A 116     -16.150  -5.415   4.575  1.00 97.74           C  \\nATOM   1739 HD11 LEU A 116     -15.650  -5.738   3.662  1.00 97.74           H  \\nATOM   1740 HD12 LEU A 116     -17.197  -5.222   4.340  1.00 97.74           H  \\nATOM   1741 HD13 LEU A 116     -16.087  -6.201   5.328  1.00 97.74           H  \\nATOM   1742  CD2 LEU A 116     -16.317  -3.580   6.266  1.00 97.74           C  \\nATOM   1743 HD21 LEU A 116     -16.265  -4.240   7.133  1.00 97.74           H  \\nATOM   1744 HD22 LEU A 116     -15.925  -2.605   6.556  1.00 97.74           H  \\nATOM   1745 HD23 LEU A 116     -17.358  -3.458   5.967  1.00 97.74           H  \\nATOM   1746  N   GLN A 117     -12.184  -2.655   3.507  1.00 98.02           N  \\nATOM   1747  H   GLN A 117     -11.618  -2.688   4.343  1.00 98.02           H  \\nATOM   1748  CA  GLN A 117     -11.989  -1.507   2.628  1.00 98.02           C  \\nATOM   1749  HA  GLN A 117     -12.959  -1.143   2.289  1.00 98.02           H  \\nATOM   1750  C   GLN A 117     -11.170  -1.849   1.375  1.00 98.02           C  \\nATOM   1751  CB  GLN A 117     -11.283  -0.399   3.427  1.00 98.02           C  \\nATOM   1752  HB2 GLN A 117     -10.321  -0.768   3.781  1.00 98.02           H  \\nATOM   1753  HB3 GLN A 117     -11.085   0.425   2.742  1.00 98.02           H  \\nATOM   1754  O   GLN A 117     -11.069  -1.023   0.470  1.00 98.02           O  \\nATOM   1755  CG  GLN A 117     -12.095   0.161   4.610  1.00 98.02           C  \\nATOM   1756  HG2 GLN A 117     -11.603   1.066   4.969  1.00 98.02           H  \\nATOM   1757  HG3 GLN A 117     -13.083   0.447   4.250  1.00 98.02           H  \\nATOM   1758  CD  GLN A 117     -12.249  -0.779   5.810  1.00 98.02           C  \\nATOM   1759  NE2 GLN A 117     -13.295  -0.612   6.587  1.00 98.02           N  \\nATOM   1760 HE21 GLN A 117     -13.378  -1.203   7.403  1.00 98.02           H  \\nATOM   1761 HE22 GLN A 117     -13.932   0.153   6.421  1.00 98.02           H  \\nATOM   1762  OE1 GLN A 117     -11.469  -1.690   6.068  1.00 98.02           O  \\nATOM   1763  N   GLY A 118     -10.560  -3.036   1.330  1.00 97.99           N  \\nATOM   1764  H   GLY A 118     -10.689  -3.674   2.102  1.00 97.99           H  \\nATOM   1765  CA  GLY A 118      -9.782  -3.499   0.192  1.00 97.99           C  \\nATOM   1766  HA2 GLY A 118      -9.090  -2.714  -0.113  1.00 97.99           H  \\nATOM   1767  HA3 GLY A 118      -9.194  -4.368   0.490  1.00 97.99           H  \\nATOM   1768  C   GLY A 118     -10.640  -3.890  -1.013  1.00 97.99           C  \\nATOM   1769  O   GLY A 118     -11.863  -4.071  -0.946  1.00 97.99           O  \\nATOM   1770  N   GLY A 119      -9.961  -4.095  -2.138  1.00 97.62           N  \\nATOM   1771  H   GLY A 119      -8.972  -3.891  -2.145  1.00 97.62           H  \\nATOM   1772  CA  GLY A 119     -10.579  -4.496  -3.395  1.00 97.62           C  \\nATOM   1773  HA2 GLY A 119      -9.877  -5.142  -3.921  1.00 97.62           H  \\nATOM   1774  HA3 GLY A 119     -11.499  -5.054  -3.218  1.00 97.62           H  \\nATOM   1775  C   GLY A 119     -10.844  -3.319  -4.318  1.00 97.62           C  \\nATOM   1776  O   GLY A 119      -9.908  -2.647  -4.740  1.00 97.62           O  \\nATOM   1777  N   THR A 120     -12.105  -3.097  -4.683  1.00 97.23           N  \\nATOM   1778  H   THR A 120     -12.834  -3.692  -4.316  1.00 97.23           H  \\nATOM   1779  CA  THR A 120     -12.495  -2.025  -5.611  1.00 97.23           C  \\nATOM   1780  HA  THR A 120     -12.092  -2.261  -6.596  1.00 97.23           H  \\nATOM   1781  C   THR A 120     -11.932  -0.676  -5.156  1.00 97.23           C  \\nATOM   1782  CB  THR A 120     -14.024  -1.929  -5.716  1.00 97.23           C  \\nATOM   1783  HB  THR A 120     -14.422  -1.581  -4.763  1.00 97.23           H  \\nATOM   1784  O   THR A 120     -12.043  -0.321  -3.985  1.00 97.23           O  \\nATOM   1785  CG2 THR A 120     -14.486  -0.983  -6.822  1.00 97.23           C  \\nATOM   1786 HG21 THR A 120     -14.197   0.041  -6.587  1.00 97.23           H  \\nATOM   1787 HG22 THR A 120     -14.043  -1.271  -7.775  1.00 97.23           H  \\nATOM   1788 HG23 THR A 120     -15.572  -1.022  -6.905  1.00 97.23           H  \\nATOM   1789  OG1 THR A 120     -14.580  -3.203  -5.988  1.00 97.23           O  \\nATOM   1790  HG1 THR A 120     -14.623  -3.302  -6.942  1.00 97.23           H  \\nATOM   1791  N   ASN A 121     -11.357   0.085  -6.091  1.00 96.14           N  \\nATOM   1792  H   ASN A 121     -11.336  -0.268  -7.037  1.00 96.14           H  \\nATOM   1793  CA  ASN A 121     -10.685   1.371  -5.856  1.00 96.14           C  \\nATOM   1794  HA  ASN A 121     -10.353   1.721  -6.834  1.00 96.14           H  \\nATOM   1795  C   ASN A 121      -9.403   1.313  -5.008  1.00 96.14           C  \\nATOM   1796  CB  ASN A 121     -11.674   2.427  -5.325  1.00 96.14           C  \\nATOM   1797  HB2 ASN A 121     -11.960   2.179  -4.303  1.00 96.14           H  \\nATOM   1798  HB3 ASN A 121     -11.183   3.399  -5.300  1.00 96.14           H  \\nATOM   1799  O   ASN A 121      -8.845   2.373  -4.705  1.00 96.14           O  \\nATOM   1800  CG  ASN A 121     -12.911   2.572  -6.181  1.00 96.14           C  \\nATOM   1801  ND2 ASN A 121     -14.066   2.714  -5.575  1.00 96.14           N  \\nATOM   1802 HD21 ASN A 121     -14.873   2.824  -6.171  1.00 96.14           H  \\nATOM   1803 HD22 ASN A 121     -14.113   2.699  -4.566  1.00 96.14           H  \\nATOM   1804  OD1 ASN A 121     -12.864   2.559  -7.397  1.00 96.14           O  \\nATOM   1805  N   ALA A 122      -8.893   0.126  -4.662  1.00 98.45           N  \\nATOM   1806  H   ALA A 122      -9.382  -0.730  -4.883  1.00 98.45           H  \\nATOM   1807  CA  ALA A 122      -7.557   0.026  -4.093  1.00 98.45           C  \\nATOM   1808  HA  ALA A 122      -7.546   0.616  -3.177  1.00 98.45           H  \\nATOM   1809  C   ALA A 122      -6.528   0.627  -5.063  1.00 98.45           C  \\nATOM   1810  CB  ALA A 122      -7.226  -1.420  -3.718  1.00 98.45           C  \\nATOM   1811  HB1 ALA A 122      -6.237  -1.465  -3.262  1.00 98.45           H  \\nATOM   1812  HB2 ALA A 122      -7.959  -1.795  -3.004  1.00 98.45           H  \\nATOM   1813  HB3 ALA A 122      -7.232  -2.041  -4.614  1.00 98.45           H  \\nATOM   1814  O   ALA A 122      -6.595   0.452  -6.282  1.00 98.45           O  \\nATOM   1815  N   ILE A 123      -5.593   1.375  -4.496  1.00 98.62           N  \\nATOM   1816  H   ILE A 123      -5.607   1.436  -3.488  1.00 98.62           H  \\nATOM   1817  CA  ILE A 123      -4.429   1.928  -5.165  1.00 98.62           C  \\nATOM   1818  HA  ILE A 123      -4.704   2.275  -6.161  1.00 98.62           H  \\nATOM   1819  C   ILE A 123      -3.401   0.821  -5.293  1.00 98.62           C  \\nATOM   1820  CB  ILE A 123      -3.899   3.112  -4.336  1.00 98.62           C  \\nATOM   1821  HB  ILE A 123      -3.928   2.838  -3.281  1.00 98.62           H  \\nATOM   1822  O   ILE A 123      -3.010   0.235  -4.290  1.00 98.62           O  \\nATOM   1823  CG1 ILE A 123      -4.832   4.320  -4.572  1.00 98.62           C  \\nATOM   1824 HG12 ILE A 123      -4.672   4.701  -5.580  1.00 98.62           H  \\nATOM   1825 HG13 ILE A 123      -5.871   4.001  -4.500  1.00 98.62           H  \\nATOM   1826  CG2 ILE A 123      -2.444   3.467  -4.681  1.00 98.62           C  \\nATOM   1827 HG21 ILE A 123      -2.094   4.215  -3.971  1.00 98.62           H  \\nATOM   1828 HG22 ILE A 123      -2.369   3.818  -5.710  1.00 98.62           H  \\nATOM   1829 HG23 ILE A 123      -1.783   2.611  -4.540  1.00 98.62           H  \\nATOM   1830  CD1 ILE A 123      -4.645   5.464  -3.576  1.00 98.62           C  \\nATOM   1831 HD11 ILE A 123      -3.613   5.815  -3.565  1.00 98.62           H  \\nATOM   1832 HD12 ILE A 123      -5.300   6.287  -3.863  1.00 98.62           H  \\nATOM   1833 HD13 ILE A 123      -4.938   5.125  -2.582  1.00 98.62           H  \\nATOM   1834  N   LEU A 124      -2.945   0.577  -6.516  1.00 98.50           N  \\nATOM   1835  H   LEU A 124      -3.350   1.093  -7.284  1.00 98.50           H  \\nATOM   1836  CA  LEU A 124      -1.891  -0.371  -6.819  1.00 98.50           C  \\nATOM   1837  HA  LEU A 124      -1.701  -0.941  -5.910  1.00 98.50           H  \\nATOM   1838  C   LEU A 124      -0.601   0.342  -7.196  1.00 98.50           C  \\nATOM   1839  CB  LEU A 124      -2.308  -1.350  -7.926  1.00 98.50           C  \\nATOM   1840  HB2 LEU A 124      -1.396  -1.796  -8.324  1.00 98.50           H  \\nATOM   1841  HB3 LEU A 124      -2.794  -0.810  -8.738  1.00 98.50           H  \\nATOM   1842  O   LEU A 124      -0.628   1.266  -8.008  1.00 98.50           O  \\nATOM   1843  CG  LEU A 124      -3.203  -2.497  -7.437  1.00 98.50           C  \\nATOM   1844  HG  LEU A 124      -2.792  -2.927  -6.524  1.00 98.50           H  \\nATOM   1845  CD1 LEU A 124      -4.638  -2.051  -7.186  1.00 98.50           C  \\nATOM   1846 HD11 LEU A 124      -4.679  -1.403  -6.311  1.00 98.50           H  \\nATOM   1847 HD12 LEU A 124      -5.047  -1.533  -8.054  1.00 98.50           H  \\nATOM   1848 HD13 LEU A 124      -5.246  -2.928  -6.967  1.00 98.50           H  \\nATOM   1849  CD2 LEU A 124      -3.227  -3.590  -8.505  1.00 98.50           C  \\nATOM   1850 HD21 LEU A 124      -2.223  -3.985  -8.657  1.00 98.50           H  \\nATOM   1851 HD22 LEU A 124      -3.885  -4.394  -8.174  1.00 98.50           H  \\nATOM   1852 HD23 LEU A 124      -3.607  -3.173  -9.437  1.00 98.50           H  \\nATOM   1853  N   ALA A 125       0.526  -0.150  -6.689  1.00 98.50           N  \\nATOM   1854  H   ALA A 125       0.459  -0.911  -6.028  1.00 98.50           H  \\nATOM   1855  CA  ALA A 125       1.855   0.301  -7.091  1.00 98.50           C  \\nATOM   1856  HA  ALA A 125       1.766   0.792  -8.060  1.00 98.50           H  \\nATOM   1857  C   ALA A 125       2.830  -0.882  -7.242  1.00 98.50           C  \\nATOM   1858  CB  ALA A 125       2.365   1.337  -6.085  1.00 98.50           C  \\nATOM   1859  HB1 ALA A 125       3.292   1.788  -6.439  1.00 98.50           H  \\nATOM   1860  HB2 ALA A 125       2.558   0.844  -5.132  1.00 98.50           H  \\nATOM   1861  HB3 ALA A 125       1.622   2.123  -5.947  1.00 98.50           H  \\nATOM   1862  O   ALA A 125       2.664  -1.912  -6.585  1.00 98.50           O  \\nATOM   1863  N   PRO A 126       3.851  -0.768  -8.106  1.00 98.19           N  \\nATOM   1864  CA  PRO A 126       4.847  -1.806  -8.298  1.00 98.19           C  \\nATOM   1865  HA  PRO A 126       4.362  -2.777  -8.405  1.00 98.19           H  \\nATOM   1866  C   PRO A 126       5.826  -1.849  -7.126  1.00 98.19           C  \\nATOM   1867  CB  PRO A 126       5.556  -1.454  -9.604  1.00 98.19           C  \\nATOM   1868  HB2 PRO A 126       5.043  -1.920 -10.445  1.00 98.19           H  \\nATOM   1869  HB3 PRO A 126       6.609  -1.735  -9.592  1.00 98.19           H  \\nATOM   1870  O   PRO A 126       6.538  -0.886  -6.840  1.00 98.19           O  \\nATOM   1871  CG  PRO A 126       5.431   0.058  -9.692  1.00 98.19           C  \\nATOM   1872  HG2 PRO A 126       5.434   0.388 -10.730  1.00 98.19           H  \\nATOM   1873  HG3 PRO A 126       6.240   0.532  -9.135  1.00 98.19           H  \\nATOM   1874  CD  PRO A 126       4.100   0.343  -9.003  1.00 98.19           C  \\nATOM   1875  HD2 PRO A 126       4.158   1.282  -8.453  1.00 98.19           H  \\nATOM   1876  HD3 PRO A 126       3.305   0.387  -9.747  1.00 98.19           H  \\nATOM   1877  N   VAL A 127       5.922  -3.017  -6.506  1.00 98.19           N  \\nATOM   1878  H   VAL A 127       5.265  -3.738  -6.767  1.00 98.19           H  \\nATOM   1879  CA  VAL A 127       6.939  -3.366  -5.513  1.00 98.19           C  \\nATOM   1880  HA  VAL A 127       7.713  -2.599  -5.527  1.00 98.19           H  \\nATOM   1881  C   VAL A 127       7.595  -4.690  -5.906  1.00 98.19           C  \\nATOM   1882  CB  VAL A 127       6.356  -3.372  -4.089  1.00 98.19           C  \\nATOM   1883  HB  VAL A 127       7.175  -3.525  -3.386  1.00 98.19           H  \\nATOM   1884  O   VAL A 127       7.198  -5.331  -6.883  1.00 98.19           O  \\nATOM   1885  CG1 VAL A 127       5.698  -2.028  -3.764  1.00 98.19           C  \\nATOM   1886 HG11 VAL A 127       5.467  -1.969  -2.701  1.00 98.19           H  \\nATOM   1887 HG12 VAL A 127       6.356  -1.206  -4.045  1.00 98.19           H  \\nATOM   1888 HG13 VAL A 127       4.774  -1.910  -4.332  1.00 98.19           H  \\nATOM   1889  CG2 VAL A 127       5.333  -4.482  -3.880  1.00 98.19           C  \\nATOM   1890 HG21 VAL A 127       5.796  -5.455  -4.043  1.00 98.19           H  \\nATOM   1891 HG22 VAL A 127       4.955  -4.422  -2.859  1.00 98.19           H  \\nATOM   1892 HG23 VAL A 127       4.492  -4.356  -4.562  1.00 98.19           H  \\nATOM   1893  N   ASN A 128       8.639  -5.115  -5.197  1.00 97.05           N  \\nATOM   1894  H   ASN A 128       8.905  -4.609  -4.365  1.00 97.05           H  \\nATOM   1895  CA  ASN A 128       9.236  -6.430  -5.437  1.00 97.05           C  \\nATOM   1896  HA  ASN A 128       8.997  -6.743  -6.454  1.00 97.05           H  \\nATOM   1897  C   ASN A 128       8.608  -7.503  -4.525  1.00 97.05           C  \\nATOM   1898  CB  ASN A 128      10.768  -6.329  -5.413  1.00 97.05           C  \\nATOM   1899  HB2 ASN A 128      11.087  -5.544  -6.099  1.00 97.05           H  \\nATOM   1900  HB3 ASN A 128      11.181  -7.270  -5.774  1.00 97.05           H  \\nATOM   1901  O   ASN A 128       7.882  -7.202  -3.575  1.00 97.05           O  \\nATOM   1902  CG  ASN A 128      11.342  -6.073  -4.039  1.00 97.05           C  \\nATOM   1903  ND2 ASN A 128      12.268  -5.158  -3.901  1.00 97.05           N  \\nATOM   1904 HD21 ASN A 128      12.479  -4.906  -2.945  1.00 97.05           H  \\nATOM   1905 HD22 ASN A 128      12.523  -4.575  -4.685  1.00 97.05           H  \\nATOM   1906  OD1 ASN A 128      10.986  -6.719  -3.080  1.00 97.05           O  \\nATOM   1907  N   LEU A 129       8.844  -8.778  -4.837  1.00 95.72           N  \\nATOM   1908  H   LEU A 129       9.518  -8.983  -5.561  1.00 95.72           H  \\nATOM   1909  CA  LEU A 129       8.254  -9.892  -4.087  1.00 95.72           C  \\nATOM   1910  HA  LEU A 129       7.175  -9.746  -4.036  1.00 95.72           H  \\nATOM   1911  C   LEU A 129       8.758  -9.946  -2.639  1.00 95.72           C  \\nATOM   1912  CB  LEU A 129       8.552 -11.198  -4.841  1.00 95.72           C  \\nATOM   1913  HB2 LEU A 129       8.121 -11.130  -5.841  1.00 95.72           H  \\nATOM   1914  HB3 LEU A 129       9.632 -11.301  -4.944  1.00 95.72           H  \\nATOM   1915  O   LEU A 129       7.980 -10.227  -1.730  1.00 95.72           O  \\nATOM   1916  CG  LEU A 129       8.009 -12.466  -4.152  1.00 95.72           C  \\nATOM   1917  HG  LEU A 129       8.468 -12.575  -3.169  1.00 95.72           H  \\nATOM   1918  CD1 LEU A 129       6.488 -12.437  -3.995  1.00 95.72           C  \\nATOM   1919 HD11 LEU A 129       6.142 -13.386  -3.585  1.00 95.72           H  \\nATOM   1920 HD12 LEU A 129       6.006 -12.259  -4.956  1.00 95.72           H  \\nATOM   1921 HD13 LEU A 129       6.199 -11.658  -3.289  1.00 95.72           H  \\nATOM   1922  CD2 LEU A 129       8.388 -13.691  -4.982  1.00 95.72           C  \\nATOM   1923 HD21 LEU A 129       7.929 -13.635  -5.969  1.00 95.72           H  \\nATOM   1924 HD22 LEU A 129       8.043 -14.594  -4.478  1.00 95.72           H  \\nATOM   1925 HD23 LEU A 129       9.472 -13.746  -5.085  1.00 95.72           H  \\nATOM   1926  N   ALA A 130      10.040  -9.662  -2.414  1.00 95.72           N  \\nATOM   1927  H   ALA A 130      10.635  -9.402  -3.187  1.00 95.72           H  \\nATOM   1928  CA  ALA A 130      10.610  -9.677  -1.073  1.00 95.72           C  \\nATOM   1929  HA  ALA A 130      10.386 -10.638  -0.611  1.00 95.72           H  \\nATOM   1930  C   ALA A 130      10.001  -8.593  -0.163  1.00 95.72           C  \\nATOM   1931  CB  ALA A 130      12.124  -9.566  -1.213  1.00 95.72           C  \\nATOM   1932  HB1 ALA A 130      12.502 -10.394  -1.812  1.00 95.72           H  \\nATOM   1933  HB2 ALA A 130      12.390  -8.622  -1.689  1.00 95.72           H  \\nATOM   1934  HB3 ALA A 130      12.575  -9.627  -0.223  1.00 95.72           H  \\nATOM   1935  O   ALA A 130       9.683  -8.871   0.993  1.00 95.72           O  \\nATOM   1936  N   SER A 131       9.728  -7.406  -0.706  1.00 97.47           N  \\nATOM   1937  H   SER A 131      10.074  -7.220  -1.636  1.00 97.47           H  \\nATOM   1938  CA  SER A 131       8.980  -6.340  -0.043  1.00 97.47           C  \\nATOM   1939  HA  SER A 131       9.503  -6.047   0.867  1.00 97.47           H  \\nATOM   1940  C   SER A 131       7.580  -6.781   0.362  1.00 97.47           C  \\nATOM   1941  CB  SER A 131       8.859  -5.131  -0.968  1.00 97.47           C  \\nATOM   1942  HB2 SER A 131       8.553  -5.444  -1.966  1.00 97.47           H  \\nATOM   1943  HB3 SER A 131       8.100  -4.460  -0.566  1.00 97.47           H  \\nATOM   1944  O   SER A 131       7.211  -6.603   1.520  1.00 97.47           O  \\nATOM   1945  OG  SER A 131      10.088  -4.449  -1.035  1.00 97.47           O  \\nATOM   1946  HG  SER A 131      10.235  -4.093  -0.156  1.00 97.47           H  \\nATOM   1947  N   GLN A 132       6.836  -7.410  -0.554  1.00 95.22           N  \\nATOM   1948  H   GLN A 132       7.201  -7.519  -1.489  1.00 95.22           H  \\nATOM   1949  CA  GLN A 132       5.491  -7.913  -0.261  1.00 95.22           C  \\nATOM   1950  HA  GLN A 132       4.906  -7.076   0.120  1.00 95.22           H  \\nATOM   1951  C   GLN A 132       5.491  -8.974   0.848  1.00 95.22           C  \\nATOM   1952  CB  GLN A 132       4.852  -8.488  -1.546  1.00 95.22           C  \\nATOM   1953  HB2 GLN A 132       5.628  -8.908  -2.186  1.00 95.22           H  \\nATOM   1954  HB3 GLN A 132       4.176  -9.301  -1.281  1.00 95.22           H  \\nATOM   1955  O   GLN A 132       4.715  -8.909   1.801  1.00 95.22           O  \\nATOM   1956  CG  GLN A 132       4.044  -7.455  -2.345  1.00 95.22           C  \\nATOM   1957  HG2 GLN A 132       4.671  -6.578  -2.505  1.00 95.22           H  \\nATOM   1958  HG3 GLN A 132       3.765  -7.880  -3.309  1.00 95.22           H  \\nATOM   1959  CD  GLN A 132       2.779  -7.020  -1.609  1.00 95.22           C  \\nATOM   1960  NE2 GLN A 132       1.726  -6.600  -2.251  1.00 95.22           N  \\nATOM   1961 HE21 GLN A 132       1.180  -6.027  -1.623  1.00 95.22           H  \\nATOM   1962 HE22 GLN A 132       1.705  -6.399  -3.241  1.00 95.22           H  \\nATOM   1963  OE1 GLN A 132       2.672  -7.138  -0.411  1.00 95.22           O  \\nATOM   1964  N   ASN A 133       6.415  -9.934   0.774  1.00 96.35           N  \\nATOM   1965  H   ASN A 133       7.009  -9.968  -0.042  1.00 96.35           H  \\nATOM   1966  CA  ASN A 133       6.554 -10.963   1.805  1.00 96.35           C  \\nATOM   1967  HA  ASN A 133       5.602 -11.481   1.923  1.00 96.35           H  \\nATOM   1968  C   ASN A 133       6.906 -10.349   3.170  1.00 96.35           C  \\nATOM   1969  CB  ASN A 133       7.627 -11.968   1.360  1.00 96.35           C  \\nATOM   1970  HB2 ASN A 133       7.848 -12.640   2.189  1.00 96.35           H  \\nATOM   1971  HB3 ASN A 133       8.541 -11.432   1.103  1.00 96.35           H  \\nATOM   1972  O   ASN A 133       6.361 -10.754   4.198  1.00 96.35           O  \\nATOM   1973  CG  ASN A 133       7.206 -12.826   0.181  1.00 96.35           C  \\nATOM   1974  ND2 ASN A 133       8.144 -13.503  -0.437  1.00 96.35           N  \\nATOM   1975 HD21 ASN A 133       9.104 -13.444  -0.127  1.00 96.35           H  \\nATOM   1976 HD22 ASN A 133       7.830 -14.095  -1.192  1.00 96.35           H  \\nATOM   1977  OD1 ASN A 133       6.049 -12.944  -0.184  1.00 96.35           O  \\nATOM   1978  N   SER A 134       7.795  -9.351   3.177  1.00 97.41           N  \\nATOM   1979  H   SER A 134       8.206  -9.071   2.298  1.00 97.41           H  \\nATOM   1980  CA  SER A 134       8.192  -8.635   4.387  1.00 97.41           C  \\nATOM   1981  HA  SER A 134       8.515  -9.369   5.126  1.00 97.41           H  \\nATOM   1982  C   SER A 134       7.022  -7.855   4.999  1.00 97.41           C  \\nATOM   1983  CB  SER A 134       9.382  -7.725   4.066  1.00 97.41           C  \\nATOM   1984  HB2 SER A 134       9.081  -6.964   3.346  1.00 97.41           H  \\nATOM   1985  HB3 SER A 134      10.178  -8.324   3.623  1.00 97.41           H  \\nATOM   1986  O   SER A 134       6.790  -7.979   6.201  1.00 97.41           O  \\nATOM   1987  OG  SER A 134       9.882  -7.096   5.230  1.00 97.41           O  \\nATOM   1988  HG  SER A 134       9.956  -7.744   5.935  1.00 97.41           H  \\nATOM   1989  N   GLN A 135       6.241  -7.124   4.192  1.00 97.82           N  \\nATOM   1990  H   GLN A 135       6.465  -7.074   3.209  1.00 97.82           H  \\nATOM   1991  CA  GLN A 135       5.044  -6.404   4.651  1.00 97.82           C  \\nATOM   1992  HA  GLN A 135       5.330  -5.695   5.428  1.00 97.82           H  \\nATOM   1993  C   GLN A 135       4.015  -7.371   5.249  1.00 97.82           C  \\nATOM   1994  CB  GLN A 135       4.433  -5.623   3.476  1.00 97.82           C  \\nATOM   1995  HB2 GLN A 135       5.231  -5.085   2.963  1.00 97.82           H  \\nATOM   1996  HB3 GLN A 135       4.028  -6.346   2.767  1.00 97.82           H  \\nATOM   1997  O   GLN A 135       3.521  -7.133   6.350  1.00 97.82           O  \\nATOM   1998  CG  GLN A 135       3.347  -4.596   3.887  1.00 97.82           C  \\nATOM   1999  HG2 GLN A 135       3.827  -3.645   4.118  1.00 97.82           H  \\nATOM   2000  HG3 GLN A 135       2.823  -4.935   4.781  1.00 97.82           H  \\nATOM   2001  CD  GLN A 135       2.293  -4.383   2.798  1.00 97.82           C  \\nATOM   2002  NE2 GLN A 135       1.799  -3.190   2.546  1.00 97.82           N  \\nATOM   2003 HE21 GLN A 135       1.203  -3.127   1.733  1.00 97.82           H  \\nATOM   2004 HE22 GLN A 135       1.976  -2.332   3.049  1.00 97.82           H  \\nATOM   2005  OE1 GLN A 135       1.859  -5.346   2.194  1.00 97.82           O  \\nATOM   2006  N   GLY A 136       3.763  -8.509   4.592  1.00 96.60           N  \\nATOM   2007  H   GLY A 136       4.149  -8.617   3.665  1.00 96.60           H  \\nATOM   2008  CA  GLY A 136       2.874  -9.549   5.111  1.00 96.60           C  \\nATOM   2009  HA2 GLY A 136       2.854 -10.375   4.399  1.00 96.60           H  \\nATOM   2010  HA3 GLY A 136       1.867  -9.143   5.208  1.00 96.60           H  \\nATOM   2011  C   GLY A 136       3.317 -10.095   6.474  1.00 96.60           C  \\nATOM   2012  O   GLY A 136       2.487 -10.271   7.367  1.00 96.60           O  \\nATOM   2013  N   GLY A 137       4.622 -10.313   6.669  1.00 97.41           N  \\nATOM   2014  H   GLY A 137       5.251 -10.159   5.894  1.00 97.41           H  \\nATOM   2015  CA  GLY A 137       5.187 -10.713   7.961  1.00 97.41           C  \\nATOM   2016  HA2 GLY A 137       4.733 -11.648   8.286  1.00 97.41           H  \\nATOM   2017  HA3 GLY A 137       6.259 -10.866   7.842  1.00 97.41           H  \\nATOM   2018  C   GLY A 137       4.976  -9.659   9.053  1.00 97.41           C  \\nATOM   2019  O   GLY A 137       4.493  -9.986  10.139  1.00 97.41           O  \\nATOM   2020  N   VAL A 138       5.268  -8.391   8.749  1.00 98.10           N  \\nATOM   2021  H   VAL A 138       5.638  -8.191   7.831  1.00 98.10           H  \\nATOM   2022  CA  VAL A 138       5.087  -7.269   9.686  1.00 98.10           C  \\nATOM   2023  HA  VAL A 138       5.604  -7.507  10.616  1.00 98.10           H  \\nATOM   2024  C   VAL A 138       3.610  -7.075  10.043  1.00 98.10           C  \\nATOM   2025  CB  VAL A 138       5.707  -5.975   9.123  1.00 98.10           C  \\nATOM   2026  HB  VAL A 138       5.297  -5.779   8.132  1.00 98.10           H  \\nATOM   2027  O   VAL A 138       3.280  -6.976  11.223  1.00 98.10           O  \\nATOM   2028  CG1 VAL A 138       5.418  -4.771  10.024  1.00 98.10           C  \\nATOM   2029 HG11 VAL A 138       5.721  -4.983  11.050  1.00 98.10           H  \\nATOM   2030 HG12 VAL A 138       4.357  -4.526  10.006  1.00 98.10           H  \\nATOM   2031 HG13 VAL A 138       5.980  -3.904   9.677  1.00 98.10           H  \\nATOM   2032  CG2 VAL A 138       7.234  -6.089   9.015  1.00 98.10           C  \\nATOM   2033 HG21 VAL A 138       7.637  -5.184   8.561  1.00 98.10           H  \\nATOM   2034 HG22 VAL A 138       7.514  -6.940   8.393  1.00 98.10           H  \\nATOM   2035 HG23 VAL A 138       7.670  -6.213  10.007  1.00 98.10           H  \\nATOM   2036  N   LEU A 139       2.704  -7.097   9.060  1.00 97.39           N  \\nATOM   2037  H   LEU A 139       3.030  -7.173   8.108  1.00 97.39           H  \\nATOM   2038  CA  LEU A 139       1.260  -6.989   9.293  1.00 97.39           C  \\nATOM   2039  HA  LEU A 139       1.066  -6.082   9.865  1.00 97.39           H  \\nATOM   2040  C   LEU A 139       0.724  -8.150  10.134  1.00 97.39           C  \\nATOM   2041  CB  LEU A 139       0.507  -6.925   7.953  1.00 97.39           C  \\nATOM   2042  HB2 LEU A 139       0.926  -7.669   7.276  1.00 97.39           H  \\nATOM   2043  HB3 LEU A 139      -0.536  -7.191   8.125  1.00 97.39           H  \\nATOM   2044  O   LEU A 139      -0.077  -7.929  11.037  1.00 97.39           O  \\nATOM   2045  CG  LEU A 139       0.530  -5.547   7.270  1.00 97.39           C  \\nATOM   2046  HG  LEU A 139       1.563  -5.223   7.142  1.00 97.39           H  \\nATOM   2047  CD1 LEU A 139      -0.118  -5.657   5.892  1.00 97.39           C  \\nATOM   2048 HD11 LEU A 139      -0.048  -4.698   5.379  1.00 97.39           H  \\nATOM   2049 HD12 LEU A 139      -1.168  -5.937   5.981  1.00 97.39           H  \\nATOM   2050 HD13 LEU A 139       0.414  -6.391   5.288  1.00 97.39           H  \\nATOM   2051  CD2 LEU A 139      -0.240  -4.497   8.080  1.00 97.39           C  \\nATOM   2052 HD21 LEU A 139       0.282  -4.272   9.010  1.00 97.39           H  \\nATOM   2053 HD22 LEU A 139      -1.247  -4.855   8.299  1.00 97.39           H  \\nATOM   2054 HD23 LEU A 139      -0.312  -3.576   7.503  1.00 97.39           H  \\nATOM   2055  N   ASN A 140       1.180  -9.382   9.892  1.00 96.70           N  \\nATOM   2056  H   ASN A 140       1.818  -9.522   9.122  1.00 96.70           H  \\nATOM   2057  CA  ASN A 140       0.786 -10.527  10.715  1.00 96.70           C  \\nATOM   2058  HA  ASN A 140      -0.302 -10.579  10.746  1.00 96.70           H  \\nATOM   2059  C   ASN A 140       1.245 -10.371  12.172  1.00 96.70           C  \\nATOM   2060  CB  ASN A 140       1.330 -11.818  10.089  1.00 96.70           C  \\nATOM   2061  HB2 ASN A 140       2.368 -11.670   9.790  1.00 96.70           H  \\nATOM   2062  HB3 ASN A 140       1.306 -12.616  10.830  1.00 96.70           H  \\nATOM   2063  O   ASN A 140       0.459 -10.637  13.086  1.00 96.70           O  \\nATOM   2064  CG  ASN A 140       0.517 -12.292   8.898  1.00 96.70           C  \\nATOM   2065  ND2 ASN A 140       1.117 -13.079   8.038  1.00 96.70           N  \\nATOM   2066 HD21 ASN A 140       2.114 -13.224   8.113  1.00 96.70           H  \\nATOM   2067 HD22 ASN A 140       0.586 -13.355   7.224  1.00 96.70           H  \\nATOM   2068  OD1 ASN A 140      -0.672 -12.032   8.758  1.00 96.70           O  \\nATOM   2069  N   GLY A 141       2.475  -9.893  12.388  1.00 96.80           N  \\nATOM   2070  H   GLY A 141       3.070  -9.713  11.592  1.00 96.80           H  \\nATOM   2071  CA  GLY A 141       2.981  -9.540  13.715  1.00 96.80           C  \\nATOM   2072  HA2 GLY A 141       3.999  -9.164  13.618  1.00 96.80           H  \\nATOM   2073  HA3 GLY A 141       2.988 -10.423  14.355  1.00 96.80           H  \\nATOM   2074  C   GLY A 141       2.132  -8.455  14.378  1.00 96.80           C  \\nATOM   2075  O   GLY A 141       1.679  -8.640  15.508  1.00 96.80           O  \\nATOM   2076  N   PHE A 142       1.823  -7.381  13.643  1.00 97.09           N  \\nATOM   2077  H   PHE A 142       2.245  -7.290  12.730  1.00 97.09           H  \\nATOM   2078  CA  PHE A 142       0.941  -6.308  14.099  1.00 97.09           C  \\nATOM   2079  HA  PHE A 142       1.382  -5.849  14.984  1.00 97.09           H  \\nATOM   2080  C   PHE A 142      -0.433  -6.847  14.502  1.00 97.09           C  \\nATOM   2081  CB  PHE A 142       0.813  -5.226  13.013  1.00 97.09           C  \\nATOM   2082  HB2 PHE A 142       0.502  -5.685  12.075  1.00 97.09           H  \\nATOM   2083  HB3 PHE A 142       1.793  -4.780  12.844  1.00 97.09           H  \\nATOM   2084  O   PHE A 142      -0.897  -6.553  15.597  1.00 97.09           O  \\nATOM   2085  CG  PHE A 142      -0.179  -4.127  13.354  1.00 97.09           C  \\nATOM   2086  CD1 PHE A 142      -1.539  -4.268  13.014  1.00 97.09           C  \\nATOM   2087  HD1 PHE A 142      -1.867  -5.145  12.475  1.00 97.09           H  \\nATOM   2088  CD2 PHE A 142       0.253  -2.964  14.018  1.00 97.09           C  \\nATOM   2089  HD2 PHE A 142       1.295  -2.828  14.270  1.00 97.09           H  \\nATOM   2090  CE1 PHE A 142      -2.468  -3.271  13.364  1.00 97.09           C  \\nATOM   2091  HE1 PHE A 142      -3.510  -3.382  13.104  1.00 97.09           H  \\nATOM   2092  CE2 PHE A 142      -0.670  -1.958  14.347  1.00 97.09           C  \\nATOM   2093  HE2 PHE A 142      -0.325  -1.058  14.834  1.00 97.09           H  \\nATOM   2094  CZ  PHE A 142      -2.031  -2.113  14.033  1.00 97.09           C  \\nATOM   2095  HZ  PHE A 142      -2.731  -1.331  14.287  1.00 97.09           H  \\nATOM   2096  N   TYR A 143      -1.087  -7.662  13.672  1.00 96.69           N  \\nATOM   2097  H   TYR A 143      -0.686  -7.854  12.765  1.00 96.69           H  \\nATOM   2098  CA  TYR A 143      -2.410  -8.195  13.997  1.00 96.69           C  \\nATOM   2099  HA  TYR A 143      -3.074  -7.358  14.217  1.00 96.69           H  \\nATOM   2100  C   TYR A 143      -2.394  -9.081  15.243  1.00 96.69           C  \\nATOM   2101  CB  TYR A 143      -2.976  -8.998  12.821  1.00 96.69           C  \\nATOM   2102  HB2 TYR A 143      -3.937  -9.404  13.137  1.00 96.69           H  \\nATOM   2103  HB3 TYR A 143      -2.309  -9.837  12.626  1.00 96.69           H  \\nATOM   2104  O   TYR A 143      -3.296  -8.967  16.073  1.00 96.69           O  \\nATOM   2105  CG  TYR A 143      -3.208  -8.244  11.525  1.00 96.69           C  \\nATOM   2106  CD1 TYR A 143      -3.720  -6.931  11.531  1.00 96.69           C  \\nATOM   2107  HD1 TYR A 143      -3.939  -6.426  12.460  1.00 96.69           H  \\nATOM   2108  CD2 TYR A 143      -2.926  -8.877  10.299  1.00 96.69           C  \\nATOM   2109  HD2 TYR A 143      -2.495  -9.867  10.283  1.00 96.69           H  \\nATOM   2110  CE1 TYR A 143      -3.925  -6.245  10.321  1.00 96.69           C  \\nATOM   2111  HE1 TYR A 143      -4.291  -5.229  10.336  1.00 96.69           H  \\nATOM   2112  CE2 TYR A 143      -3.156  -8.204   9.086  1.00 96.69           C  \\nATOM   2113  HE2 TYR A 143      -2.908  -8.672   8.144  1.00 96.69           H  \\nATOM   2114  OH  TYR A 143      -3.857  -6.249   7.913  1.00 96.69           O  \\nATOM   2115  HH  TYR A 143      -4.028  -5.312   8.032  1.00 96.69           H  \\nATOM   2116  CZ  TYR A 143      -3.648  -6.884   9.094  1.00 96.69           C  \\nATOM   2117  N   SER A 144      -1.379  -9.940  15.381  1.00 96.30           N  \\nATOM   2118  H   SER A 144      -0.668  -9.961  14.664  1.00 96.30           H  \\nATOM   2119  CA  SER A 144      -1.225 -10.829  16.537  1.00 96.30           C  \\nATOM   2120  HA  SER A 144      -2.135 -11.416  16.661  1.00 96.30           H  \\nATOM   2121  C   SER A 144      -1.002 -10.031  17.820  1.00 96.30           C  \\nATOM   2122  CB  SER A 144      -0.062 -11.795  16.298  1.00 96.30           C  \\nATOM   2123  HB2 SER A 144      -0.212 -12.317  15.353  1.00 96.30           H  \\nATOM   2124  HB3 SER A 144       0.877 -11.244  16.252  1.00 96.30           H  \\nATOM   2125  O   SER A 144      -1.800 -10.124  18.757  1.00 96.30           O  \\nATOM   2126  OG  SER A 144      -0.017 -12.735  17.350  1.00 96.30           O  \\nATOM   2127  HG  SER A 144       0.760 -13.290  17.249  1.00 96.30           H  \\nATOM   2128  N   ALA A 145       0.005  -9.151  17.818  1.00 96.20           N  \\nATOM   2129  H   ALA A 145       0.592  -9.077  17.000  1.00 96.20           H  \\nATOM   2130  CA  ALA A 145       0.307  -8.271  18.941  1.00 96.20           C  \\nATOM   2131  HA  ALA A 145       0.571  -8.876  19.808  1.00 96.20           H  \\nATOM   2132  C   ALA A 145      -0.908  -7.415  19.306  1.00 96.20           C  \\nATOM   2133  CB  ALA A 145       1.503  -7.390  18.567  1.00 96.20           C  \\nATOM   2134  HB1 ALA A 145       2.368  -8.013  18.339  1.00 96.20           H  \\nATOM   2135  HB2 ALA A 145       1.755  -6.737  19.402  1.00 96.20           H  \\nATOM   2136  HB3 ALA A 145       1.266  -6.778  17.697  1.00 96.20           H  \\nATOM   2137  O   ALA A 145      -1.187  -7.207  20.488  1.00 96.20           O  \\nATOM   2138  N   ASN A 146      -1.681  -6.984  18.300  1.00 95.55           N  \\nATOM   2139  H   ASN A 146      -1.357  -7.147  17.358  1.00 95.55           H  \\nATOM   2140  CA  ASN A 146      -2.826  -6.108  18.475  1.00 95.55           C  \\nATOM   2141  HA  ASN A 146      -2.615  -5.570  19.399  1.00 95.55           H  \\nATOM   2142  C   ASN A 146      -4.168  -6.796  18.750  1.00 95.55           C  \\nATOM   2143  CB  ASN A 146      -2.828  -4.952  17.473  1.00 95.55           C  \\nATOM   2144  HB2 ASN A 146      -2.809  -5.315  16.446  1.00 95.55           H  \\nATOM   2145  HB3 ASN A 146      -3.727  -4.349  17.599  1.00 95.55           H  \\nATOM   2146  O   ASN A 146      -5.096  -6.104  19.151  1.00 95.55           O  \\nATOM   2147  CG  ASN A 146      -1.635  -4.053  17.755  1.00 95.55           C  \\nATOM   2148  ND2 ASN A 146      -0.662  -4.002  16.889  1.00 95.55           N  \\nATOM   2149 HD21 ASN A 146       0.088  -3.352  17.076  1.00 95.55           H  \\nATOM   2150 HD22 ASN A 146      -0.714  -4.543  16.038  1.00 95.55           H  \\nATOM   2151  OD1 ASN A 146      -1.536  -3.455  18.809  1.00 95.55           O  \\nATOM   2152  N   LYS A 147      -4.247  -8.134  18.714  1.00 94.61           N  \\nATOM   2153  H   LYS A 147      -3.421  -8.641  18.431  1.00 94.61           H  \\nATOM   2154  CA  LYS A 147      -5.510  -8.896  18.823  1.00 94.61           C  \\nATOM   2155  HA  LYS A 147      -5.275  -9.934  18.588  1.00 94.61           H  \\nATOM   2156  C   LYS A 147      -6.543  -8.474  17.765  1.00 94.61           C  \\nATOM   2157  CB  LYS A 147      -6.096  -8.869  20.252  1.00 94.61           C  \\nATOM   2158  HB2 LYS A 147      -6.429  -7.858  20.488  1.00 94.61           H  \\nATOM   2159  HB3 LYS A 147      -6.975  -9.513  20.267  1.00 94.61           H  \\nATOM   2160  O   LYS A 147      -7.740  -8.448  18.030  1.00 94.61           O  \\nATOM   2161  CG  LYS A 147      -5.138  -9.353  21.351  1.00 94.61           C  \\nATOM   2162  HG2 LYS A 147      -4.268  -8.699  21.399  1.00 94.61           H  \\nATOM   2163  HG3 LYS A 147      -4.809 -10.366  21.121  1.00 94.61           H  \\nATOM   2164  CD  LYS A 147      -5.855  -9.343  22.711  1.00 94.61           C  \\nATOM   2165  HD2 LYS A 147      -6.729  -9.993  22.652  1.00 94.61           H  \\nATOM   2166  HD3 LYS A 147      -6.188  -8.329  22.934  1.00 94.61           H  \\nATOM   2167  CE  LYS A 147      -4.925  -9.840  23.825  1.00 94.61           C  \\nATOM   2168  HE2 LYS A 147      -4.056  -9.185  23.880  1.00 94.61           H  \\nATOM   2169  HE3 LYS A 147      -4.568 -10.835  23.557  1.00 94.61           H  \\nATOM   2170  NZ  LYS A 147      -5.615  -9.888  25.142  1.00 94.61           N  \\nATOM   2171  HZ1 LYS A 147      -4.989 -10.221  25.861  1.00 94.61           H  \\nATOM   2172  HZ2 LYS A 147      -5.950  -8.975  25.412  1.00 94.61           H  \\nATOM   2173  HZ3 LYS A 147      -6.406 -10.516  25.107  1.00 94.61           H  \\nATOM   2174  N   VAL A 148      -6.079  -8.139  16.559  1.00 95.99           N  \\nATOM   2175  H   VAL A 148      -5.088  -8.244  16.394  1.00 95.99           H  \\nATOM   2176  CA  VAL A 148      -6.953  -7.768  15.437  1.00 95.99           C  \\nATOM   2177  HA  VAL A 148      -7.753  -7.133  15.816  1.00 95.99           H  \\nATOM   2178  C   VAL A 148      -7.585  -9.032  14.859  1.00 95.99           C  \\nATOM   2179  CB  VAL A 148      -6.199  -6.964  14.364  1.00 95.99           C  \\nATOM   2180  HB  VAL A 148      -5.368  -7.561  13.992  1.00 95.99           H  \\nATOM   2181  O   VAL A 148      -6.947  -9.778  14.104  1.00 95.99           O  \\nATOM   2182  CG1 VAL A 148      -7.106  -6.600  13.182  1.00 95.99           C  \\nATOM   2183 HG11 VAL A 148      -6.537  -6.033  12.445  1.00 95.99           H  \\nATOM   2184 HG12 VAL A 148      -7.946  -5.995  13.524  1.00 95.99           H  \\nATOM   2185 HG13 VAL A 148      -7.489  -7.500  12.701  1.00 95.99           H  \\nATOM   2186  CG2 VAL A 148      -5.644  -5.658  14.946  1.00 95.99           C  \\nATOM   2187 HG21 VAL A 148      -5.133  -5.086  14.171  1.00 95.99           H  \\nATOM   2188 HG22 VAL A 148      -6.455  -5.055  15.354  1.00 95.99           H  \\nATOM   2189 HG23 VAL A 148      -4.935  -5.878  15.743  1.00 95.99           H  \\nATOM   2190  N   ALA A 149      -8.846  -9.271  15.218  1.00 95.62           N  \\nATOM   2191  H   ALA A 149      -9.306  -8.639  15.858  1.00 95.62           H  \\nATOM   2192  CA  ALA A 149      -9.611 -10.409  14.743  1.00 95.62           C  \\nATOM   2193  HA  ALA A 149      -8.953 -11.279  14.748  1.00 95.62           H  \\nATOM   2194  C   ALA A 149     -10.109 -10.191  13.310  1.00 95.62           C  \\nATOM   2195  CB  ALA A 149     -10.763 -10.705  15.708  1.00 95.62           C  \\nATOM   2196  HB1 ALA A 149     -11.264 -11.628  15.415  1.00 95.62           H  \\nATOM   2197  HB2 ALA A 149     -10.383 -10.811  16.724  1.00 95.62           H  \\nATOM   2198  HB3 ALA A 149     -11.493  -9.896  15.681  1.00 95.62           H  \\nATOM   2199  O   ALA A 149     -10.575  -9.111  12.939  1.00 95.62           O  \\nATOM   2200  N   GLN A 150     -10.026 -11.258  12.518  1.00 95.95           N  \\nATOM   2201  H   GLN A 150      -9.695 -12.120  12.927  1.00 95.95           H  \\nATOM   2202  CA  GLN A 150     -10.679 -11.334  11.217  1.00 95.95           C  \\nATOM   2203  HA  GLN A 150     -10.581 -10.360  10.738  1.00 95.95           H  \\nATOM   2204  C   GLN A 150     -12.171 -11.644  11.378  1.00 95.95           C  \\nATOM   2205  CB  GLN A 150      -9.971 -12.366  10.324  1.00 95.95           C  \\nATOM   2206  HB2 GLN A 150     -10.391 -12.305   9.320  1.00 95.95           H  \\nATOM   2207  HB3 GLN A 150      -8.914 -12.106  10.264  1.00 95.95           H  \\nATOM   2208  O   GLN A 150     -12.558 -12.353  12.307  1.00 95.95           O  \\nATOM   2209  CG  GLN A 150     -10.115 -13.814  10.832  1.00 95.95           C  \\nATOM   2210  HG2 GLN A 150     -11.164 -14.106  10.804  1.00 95.95           H  \\nATOM   2211  HG3 GLN A 150      -9.770 -13.880  11.864  1.00 95.95           H  \\nATOM   2212  CD  GLN A 150      -9.326 -14.825  10.009  1.00 95.95           C  \\nATOM   2213  NE2 GLN A 150      -9.467 -16.101  10.294  1.00 95.95           N  \\nATOM   2214 HE21 GLN A 150      -8.899 -16.744   9.762  1.00 95.95           H  \\nATOM   2215 HE22 GLN A 150     -10.063 -16.399  11.053  1.00 95.95           H  \\nATOM   2216  OE1 GLN A 150      -8.575 -14.504   9.104  1.00 95.95           O  \\nATOM   2217  N   PHE A 151     -12.992 -11.205  10.428  1.00 96.13           N  \\nATOM   2218  H   PHE A 151     -12.633 -10.578   9.722  1.00 96.13           H  \\nATOM   2219  CA  PHE A 151     -14.394 -11.616  10.335  1.00 96.13           C  \\nATOM   2220  HA  PHE A 151     -14.534 -12.517  10.933  1.00 96.13           H  \\nATOM   2221  C   PHE A 151     -14.795 -11.984   8.909  1.00 96.13           C  \\nATOM   2222  CB  PHE A 151     -15.306 -10.541  10.931  1.00 96.13           C  \\nATOM   2223  HB2 PHE A 151     -16.266 -10.999  11.173  1.00 96.13           H  \\nATOM   2224  HB3 PHE A 151     -14.863 -10.232  11.878  1.00 96.13           H  \\nATOM   2225  O   PHE A 151     -14.073 -11.707   7.951  1.00 96.13           O  \\nATOM   2226  CG  PHE A 151     -15.565  -9.335  10.050  1.00 96.13           C  \\nATOM   2227  CD1 PHE A 151     -14.519  -8.453   9.737  1.00 96.13           C  \\nATOM   2228  HD1 PHE A 151     -13.556  -8.590  10.205  1.00 96.13           H  \\nATOM   2229  CD2 PHE A 151     -16.850  -9.100   9.527  1.00 96.13           C  \\nATOM   2230  HD2 PHE A 151     -17.670  -9.758   9.777  1.00 96.13           H  \\nATOM   2231  CE1 PHE A 151     -14.721  -7.393   8.841  1.00 96.13           C  \\nATOM   2232  HE1 PHE A 151     -13.906  -6.736   8.576  1.00 96.13           H  \\nATOM   2233  CE2 PHE A 151     -17.073  -7.996   8.688  1.00 96.13           C  \\nATOM   2234  HE2 PHE A 151     -18.065  -7.787   8.316  1.00 96.13           H  \\nATOM   2235  CZ  PHE A 151     -16.005  -7.161   8.328  1.00 96.13           C  \\nATOM   2236  HZ  PHE A 151     -16.181  -6.315   7.679  1.00 96.13           H  \\nATOM   2237  N   ASP A 152     -15.941 -12.645   8.780  1.00 96.66           N  \\nATOM   2238  H   ASP A 152     -16.498 -12.811   9.606  1.00 96.66           H  \\nATOM   2239  CA  ASP A 152     -16.601 -12.923   7.514  1.00 96.66           C  \\nATOM   2240  HA  ASP A 152     -15.875 -12.881   6.702  1.00 96.66           H  \\nATOM   2241  C   ASP A 152     -17.703 -11.876   7.278  1.00 96.66           C  \\nATOM   2242  CB  ASP A 152     -17.155 -14.355   7.541  1.00 96.66           C  \\nATOM   2243  HB2 ASP A 152     -17.942 -14.434   8.292  1.00 96.66           H  \\nATOM   2244  HB3 ASP A 152     -16.352 -15.032   7.832  1.00 96.66           H  \\nATOM   2245  O   ASP A 152     -18.758 -11.941   7.918  1.00 96.66           O  \\nATOM   2246  CG  ASP A 152     -17.699 -14.808   6.183  1.00 96.66           C  \\nATOM   2247  OD1 ASP A 152     -17.927 -13.955   5.293  1.00 96.66           O  \\nATOM   2248  OD2 ASP A 152     -17.825 -16.036   5.989  1.00 96.66           O  \\nATOM   2249  N   PRO A 153     -17.513 -10.921   6.349  1.00 95.35           N  \\nATOM   2250  CA  PRO A 153     -18.533  -9.925   6.034  1.00 95.35           C  \\nATOM   2251  HA  PRO A 153     -18.766  -9.365   6.940  1.00 95.35           H  \\nATOM   2252  C   PRO A 153     -19.835 -10.523   5.495  1.00 95.35           C  \\nATOM   2253  CB  PRO A 153     -17.900  -8.987   5.003  1.00 95.35           C  \\nATOM   2254  HB2 PRO A 153     -18.162  -9.304   3.993  1.00 95.35           H  \\nATOM   2255  HB3 PRO A 153     -18.199  -7.951   5.164  1.00 95.35           H  \\nATOM   2256  O   PRO A 153     -20.872  -9.874   5.575  1.00 95.35           O  \\nATOM   2257  CG  PRO A 153     -16.402  -9.178   5.208  1.00 95.35           C  \\nATOM   2258  HG2 PRO A 153     -16.059  -8.559   6.037  1.00 95.35           H  \\nATOM   2259  HG3 PRO A 153     -15.844  -8.950   4.300  1.00 95.35           H  \\nATOM   2260  CD  PRO A 153     -16.301 -10.647   5.595  1.00 95.35           C  \\nATOM   2261  HD2 PRO A 153     -15.395 -10.804   6.179  1.00 95.35           H  \\nATOM   2262  HD3 PRO A 153     -16.298 -11.269   4.700  1.00 95.35           H  \\nATOM   2263  N   SER A 154     -19.814 -11.758   4.976  1.00 95.31           N  \\nATOM   2264  H   SER A 154     -18.948 -12.277   4.987  1.00 95.31           H  \\nATOM   2265  CA  SER A 154     -21.033 -12.459   4.553  1.00 95.31           C  \\nATOM   2266  HA  SER A 154     -21.691 -11.730   4.080  1.00 95.31           H  \\nATOM   2267  C   SER A 154     -21.829 -13.050   5.724  1.00 95.31           C  \\nATOM   2268  CB  SER A 154     -20.713 -13.518   3.491  1.00 95.31           C  \\nATOM   2269  HB2 SER A 154     -20.092 -13.067   2.718  1.00 95.31           H  \\nATOM   2270  HB3 SER A 154     -21.650 -13.831   3.029  1.00 95.31           H  \\nATOM   2271  O   SER A 154     -22.958 -13.502   5.537  1.00 95.31           O  \\nATOM   2272  OG  SER A 154     -20.063 -14.669   4.001  1.00 95.31           O  \\nATOM   2273  HG  SER A 154     -19.257 -14.411   4.454  1.00 95.31           H  \\nATOM   2274  N   LYS A 155     -21.263 -13.040   6.940  1.00 93.44           N  \\nATOM   2275  H   LYS A 155     -20.341 -12.635   7.019  1.00 93.44           H  \\nATOM   2276  CA  LYS A 155     -21.862 -13.584   8.167  1.00 93.44           C  \\nATOM   2277  HA  LYS A 155     -22.937 -13.692   8.021  1.00 93.44           H  \\nATOM   2278  C   LYS A 155     -21.660 -12.637   9.366  1.00 93.44           C  \\nATOM   2279  CB  LYS A 155     -21.275 -14.975   8.439  1.00 93.44           C  \\nATOM   2280  HB2 LYS A 155     -20.203 -14.877   8.611  1.00 93.44           H  \\nATOM   2281  HB3 LYS A 155     -21.736 -15.371   9.344  1.00 93.44           H  \\nATOM   2282  O   LYS A 155     -21.136 -13.058  10.401  1.00 93.44           O  \\nATOM   2283  CG  LYS A 155     -21.518 -15.986   7.314  1.00 93.44           C  \\nATOM   2284  HG2 LYS A 155     -21.125 -15.617   6.366  1.00 93.44           H  \\nATOM   2285  HG3 LYS A 155     -22.588 -16.171   7.218  1.00 93.44           H  \\nATOM   2286  CD  LYS A 155     -20.788 -17.280   7.659  1.00 93.44           C  \\nATOM   2287  HD2 LYS A 155     -21.085 -17.604   8.656  1.00 93.44           H  \\nATOM   2288  HD3 LYS A 155     -19.714 -17.094   7.660  1.00 93.44           H  \\nATOM   2289  CE  LYS A 155     -21.119 -18.366   6.643  1.00 93.44           C  \\nATOM   2290  HE2 LYS A 155     -20.697 -18.077   5.681  1.00 93.44           H  \\nATOM   2291  HE3 LYS A 155     -22.202 -18.429   6.543  1.00 93.44           H  \\nATOM   2292  NZ  LYS A 155     -20.566 -19.659   7.104  1.00 93.44           N  \\nATOM   2293  HZ1 LYS A 155     -19.562 -19.587   7.182  1.00 93.44           H  \\nATOM   2294  HZ2 LYS A 155     -20.933 -19.871   8.022  1.00 93.44           H  \\nATOM   2295  HZ3 LYS A 155     -20.808 -20.407   6.470  1.00 93.44           H  \\nATOM   2296  N   PRO A 156     -22.110 -11.370   9.287  1.00 87.07           N  \\nATOM   2297  CA  PRO A 156     -21.802 -10.348  10.295  1.00 87.07           C  \\nATOM   2298  HA  PRO A 156     -20.722 -10.229  10.376  1.00 87.07           H  \\nATOM   2299  C   PRO A 156     -22.344 -10.675  11.695  1.00 87.07           C  \\nATOM   2300  CB  PRO A 156     -22.409  -9.053   9.741  1.00 87.07           C  \\nATOM   2301  HB2 PRO A 156     -22.768  -8.393  10.531  1.00 87.07           H  \\nATOM   2302  HB3 PRO A 156     -21.664  -8.539   9.133  1.00 87.07           H  \\nATOM   2303  O   PRO A 156     -21.780 -10.233  12.687  1.00 87.07           O  \\nATOM   2304  CG  PRO A 156     -23.548  -9.535   8.843  1.00 87.07           C  \\nATOM   2305  HG2 PRO A 156     -24.428  -9.757   9.447  1.00 87.07           H  \\nATOM   2306  HG3 PRO A 156     -23.786  -8.809   8.066  1.00 87.07           H  \\nATOM   2307  CD  PRO A 156     -22.984 -10.825   8.256  1.00 87.07           C  \\nATOM   2308  HD2 PRO A 156     -22.398 -10.590   7.367  1.00 87.07           H  \\nATOM   2309  HD3 PRO A 156     -23.792 -11.510   8.001  1.00 87.07           H  \\nATOM   2310  N   GLN A 157     -23.402 -11.487  11.790  1.00 88.01           N  \\nATOM   2311  H   GLN A 157     -23.835 -11.808  10.936  1.00 88.01           H  \\nATOM   2312  CA  GLN A 157     -23.974 -11.930  13.070  1.00 88.01           C  \\nATOM   2313  HA  GLN A 157     -23.919 -11.108  13.784  1.00 88.01           H  \\nATOM   2314  C   GLN A 157     -23.211 -13.102  13.716  1.00 88.01           C  \\nATOM   2315  CB  GLN A 157     -25.455 -12.302  12.864  1.00 88.01           C  \\nATOM   2316  HB2 GLN A 157     -25.874 -12.623  13.818  1.00 88.01           H  \\nATOM   2317  HB3 GLN A 157     -25.518 -13.143  12.174  1.00 88.01           H  \\nATOM   2318  O   GLN A 157     -23.451 -13.422  14.873  1.00 88.01           O  \\nATOM   2319  CG  GLN A 157     -26.318 -11.150  12.317  1.00 88.01           C  \\nATOM   2320  HG2 GLN A 157     -27.348 -11.495  12.224  1.00 88.01           H  \\nATOM   2321  HG3 GLN A 157     -25.972 -10.876  11.320  1.00 88.01           H  \\nATOM   2322  CD  GLN A 157     -26.314  -9.909  13.207  1.00 88.01           C  \\nATOM   2323  NE2 GLN A 157     -26.559  -8.739  12.661  1.00 88.01           N  \\nATOM   2324 HE21 GLN A 157     -26.800  -8.646  11.684  1.00 88.01           H  \\nATOM   2325 HE22 GLN A 157     -26.559  -7.960  13.305  1.00 88.01           H  \\nATOM   2326  OE1 GLN A 157     -26.093  -9.957  14.401  1.00 88.01           O  \\nATOM   2327  N   GLN A 158     -22.318 -13.770  12.977  1.00 88.94           N  \\nATOM   2328  H   GLN A 158     -22.111 -13.428  12.050  1.00 88.94           H  \\nATOM   2329  CA  GLN A 158     -21.595 -14.965  13.444  1.00 88.94           C  \\nATOM   2330  HA  GLN A 158     -22.045 -15.333  14.366  1.00 88.94           H  \\nATOM   2331  C   GLN A 158     -20.124 -14.675  13.754  1.00 88.94           C  \\nATOM   2332  CB  GLN A 158     -21.689 -16.082  12.392  1.00 88.94           C  \\nATOM   2333  HB2 GLN A 158     -21.193 -15.751  11.480  1.00 88.94           H  \\nATOM   2334  HB3 GLN A 158     -21.159 -16.958  12.767  1.00 88.94           H  \\nATOM   2335  O   GLN A 158     -19.499 -15.399  14.524  1.00 88.94           O  \\nATOM   2336  CG  GLN A 158     -23.137 -16.485  12.075  1.00 88.94           C  \\nATOM   2337  HG2 GLN A 158     -23.613 -16.858  12.982  1.00 88.94           H  \\nATOM   2338  HG3 GLN A 158     -23.693 -15.610  11.738  1.00 88.94           H  \\nATOM   2339  CD  GLN A 158     -23.254 -17.553  10.991  1.00 88.94           C  \\nATOM   2340  NE2 GLN A 158     -24.425 -17.706  10.412  1.00 88.94           N  \\nATOM   2341 HE21 GLN A 158     -25.222 -17.178  10.736  1.00 88.94           H  \\nATOM   2342 HE22 GLN A 158     -24.516 -18.471   9.758  1.00 88.94           H  \\nATOM   2343  OE1 GLN A 158     -22.326 -18.257  10.621  1.00 88.94           O  \\nATOM   2344  N   THR A 159     -19.556 -13.630  13.150  1.00 88.51           N  \\nATOM   2345  H   THR A 159     -20.112 -13.087  12.505  1.00 88.51           H  \\nATOM   2346  CA  THR A 159     -18.152 -13.252  13.335  1.00 88.51           C  \\nATOM   2347  HA  THR A 159     -17.742 -13.775  14.199  1.00 88.51           H  \\nATOM   2348  C   THR A 159     -18.041 -11.756  13.597  1.00 88.51           C  \\nATOM   2349  CB  THR A 159     -17.301 -13.637  12.116  1.00 88.51           C  \\nATOM   2350  HB  THR A 159     -16.282 -13.290  12.286  1.00 88.51           H  \\nATOM   2351  O   THR A 159     -18.427 -10.959  12.742  1.00 88.51           O  \\nATOM   2352  CG2 THR A 159     -17.262 -15.142  11.854  1.00 88.51           C  \\nATOM   2353 HG21 THR A 159     -18.254 -15.503  11.581  1.00 88.51           H  \\nATOM   2354 HG22 THR A 159     -16.924 -15.662  12.750  1.00 88.51           H  \\nATOM   2355 HG23 THR A 159     -16.571 -15.349  11.036  1.00 88.51           H  \\nATOM   2356  OG1 THR A 159     -17.797 -13.042  10.941  1.00 88.51           O  \\nATOM   2357  HG1 THR A 159     -18.293 -12.262  11.198  1.00 88.51           H  \\nATOM   2358  N   LYS A 160     -17.475 -11.372  14.747  1.00 88.42           N  \\nATOM   2359  H   LYS A 160     -17.173 -12.082  15.398  1.00 88.42           H  \\nATOM   2360  CA  LYS A 160     -17.197  -9.971  15.087  1.00 88.42           C  \\nATOM   2361  HA  LYS A 160     -17.903  -9.336  14.552  1.00 88.42           H  \\nATOM   2362  C   LYS A 160     -15.783  -9.605  14.638  1.00 88.42           C  \\nATOM   2363  CB  LYS A 160     -17.405  -9.741  16.595  1.00 88.42           C  \\nATOM   2364  HB2 LYS A 160     -18.358 -10.179  16.892  1.00 88.42           H  \\nATOM   2365  HB3 LYS A 160     -16.610 -10.240  17.150  1.00 88.42           H  \\nATOM   2366  O   LYS A 160     -14.815 -10.202  15.100  1.00 88.42           O  \\nATOM   2367  CG  LYS A 160     -17.427  -8.244  16.957  1.00 88.42           C  \\nATOM   2368  HG2 LYS A 160     -16.523  -7.759  16.587  1.00 88.42           H  \\nATOM   2369  HG3 LYS A 160     -18.291  -7.781  16.481  1.00 88.42           H  \\nATOM   2370  CD  LYS A 160     -17.510  -8.031  18.479  1.00 88.42           C  \\nATOM   2371  HD2 LYS A 160     -18.321  -8.634  18.887  1.00 88.42           H  \\nATOM   2372  HD3 LYS A 160     -16.569  -8.355  18.924  1.00 88.42           H  \\nATOM   2373  CE  LYS A 160     -17.759  -6.555  18.829  1.00 88.42           C  \\nATOM   2374  HE2 LYS A 160     -18.766  -6.286  18.509  1.00 88.42           H  \\nATOM   2375  HE3 LYS A 160     -17.057  -5.932  18.275  1.00 88.42           H  \\nATOM   2376  NZ  LYS A 160     -17.596  -6.281  20.281  1.00 88.42           N  \\nATOM   2377  HZ1 LYS A 160     -16.620  -6.126  20.489  1.00 88.42           H  \\nATOM   2378  HZ2 LYS A 160     -17.964  -6.998  20.890  1.00 88.42           H  \\nATOM   2379  HZ3 LYS A 160     -17.966  -5.382  20.555  1.00 88.42           H  \\nATOM   2380  N   GLY A 161     -15.672  -8.639  13.732  1.00 90.51           N  \\nATOM   2381  H   GLY A 161     -16.513  -8.195  13.392  1.00 90.51           H  \\nATOM   2382  CA  GLY A 161     -14.381  -8.128  13.271  1.00 90.51           C  \\nATOM   2383  HA2 GLY A 161     -13.649  -8.934  13.234  1.00 90.51           H  \\nATOM   2384  HA3 GLY A 161     -14.489  -7.713  12.269  1.00 90.51           H  \\nATOM   2385  C   GLY A 161     -13.847  -7.032  14.174  1.00 90.51           C  \\nATOM   2386  O   GLY A 161     -14.626  -6.275  14.748  1.00 90.51           O  \\nATOM   2387  N   THR A 162     -12.526  -6.900  14.232  1.00 95.20           N  \\nATOM   2388  H   THR A 162     -11.940  -7.575  13.762  1.00 95.20           H  \\nATOM   2389  CA  THR A 162     -11.868  -5.819  14.971  1.00 95.20           C  \\nATOM   2390  HA  THR A 162     -12.540  -5.440  15.740  1.00 95.20           H  \\nATOM   2391  C   THR A 162     -11.511  -4.674  14.035  1.00 95.20           C  \\nATOM   2392  CB  THR A 162     -10.610  -6.333  15.671  1.00 95.20           C  \\nATOM   2393  HB  THR A 162      -9.879  -6.629  14.919  1.00 95.20           H  \\nATOM   2394  O   THR A 162     -10.818  -4.872  13.031  1.00 95.20           O  \\nATOM   2395  CG2 THR A 162      -9.981  -5.313  16.605  1.00 95.20           C  \\nATOM   2396 HG21 THR A 162      -9.572  -4.478  16.035  1.00 95.20           H  \\nATOM   2397 HG22 THR A 162      -9.172  -5.794  17.154  1.00 95.20           H  \\nATOM   2398 HG23 THR A 162     -10.723  -4.945  17.314  1.00 95.20           H  \\nATOM   2399  OG1 THR A 162     -10.908  -7.452  16.469  1.00 95.20           O  \\nATOM   2400  HG1 THR A 162     -11.419  -7.142  17.220  1.00 95.20           H  \\nATOM   2401  N   TRP A 163     -11.966  -3.473  14.379  1.00 97.62           N  \\nATOM   2402  H   TRP A 163     -12.535  -3.393  15.209  1.00 97.62           H  \\nATOM   2403  CA  TRP A 163     -11.527  -2.233  13.754  1.00 97.62           C  \\nATOM   2404  HA  TRP A 163     -11.451  -2.390  12.678  1.00 97.62           H  \\nATOM   2405  C   TRP A 163     -10.155  -1.830  14.287  1.00 97.62           C  \\nATOM   2406  CB  TRP A 163     -12.563  -1.137  13.996  1.00 97.62           C  \\nATOM   2407  HB2 TRP A 163     -12.743  -1.036  15.066  1.00 97.62           H  \\nATOM   2408  HB3 TRP A 163     -12.160  -0.190  13.638  1.00 97.62           H  \\nATOM   2409  O   TRP A 163      -9.899  -1.913  15.488  1.00 97.62           O  \\nATOM   2410  CG  TRP A 163     -13.861  -1.370  13.296  1.00 97.62           C  \\nATOM   2411  CD1 TRP A 163     -14.941  -2.001  13.809  1.00 97.62           C  \\nATOM   2412  HD1 TRP A 163     -14.994  -2.419  14.804  1.00 97.62           H  \\nATOM   2413  CD2 TRP A 163     -14.227  -0.977  11.943  1.00 97.62           C  \\nATOM   2414  CE2 TRP A 163     -15.573  -1.379  11.704  1.00 97.62           C  \\nATOM   2415  CE3 TRP A 163     -13.552  -0.315  10.898  1.00 97.62           C  \\nATOM   2416  HE3 TRP A 163     -12.517  -0.042  11.041  1.00 97.62           H  \\nATOM   2417  NE1 TRP A 163     -15.957  -2.002  12.874  1.00 97.62           N  \\nATOM   2418  HE1 TRP A 163     -16.862  -2.419  13.043  1.00 97.62           H  \\nATOM   2419  CH2 TRP A 163     -15.541  -0.407   9.486  1.00 97.62           C  \\nATOM   2420  HH2 TRP A 163     -16.046  -0.162   8.564  1.00 97.62           H  \\nATOM   2421  CZ2 TRP A 163     -16.225  -1.114  10.491  1.00 97.62           C  \\nATOM   2422  HZ2 TRP A 163     -17.246  -1.431  10.339  1.00 97.62           H  \\nATOM   2423  CZ3 TRP A 163     -14.207  -0.006   9.692  1.00 97.62           C  \\nATOM   2424  HZ3 TRP A 163     -13.683   0.557   8.934  1.00 97.62           H  \\nATOM   2425  N   PHE A 164      -9.269  -1.381  13.406  1.00 97.99           N  \\nATOM   2426  H   PHE A 164      -9.528  -1.339  12.430  1.00 97.99           H  \\nATOM   2427  CA  PHE A 164      -7.923  -0.949  13.763  1.00 97.99           C  \\nATOM   2428  HA  PHE A 164      -7.936  -0.626  14.804  1.00 97.99           H  \\nATOM   2429  C   PHE A 164      -7.470   0.248  12.925  1.00 97.99           C  \\nATOM   2430  CB  PHE A 164      -6.938  -2.124  13.654  1.00 97.99           C  \\nATOM   2431  HB2 PHE A 164      -6.001  -1.830  14.127  1.00 97.99           H  \\nATOM   2432  HB3 PHE A 164      -7.329  -2.966  14.224  1.00 97.99           H  \\nATOM   2433  O   PHE A 164      -7.944   0.473  11.813  1.00 97.99           O  \\nATOM   2434  CG  PHE A 164      -6.643  -2.579  12.235  1.00 97.99           C  \\nATOM   2435  CD1 PHE A 164      -7.496  -3.497  11.593  1.00 97.99           C  \\nATOM   2436  HD1 PHE A 164      -8.371  -3.867  12.107  1.00 97.99           H  \\nATOM   2437  CD2 PHE A 164      -5.525  -2.069  11.547  1.00 97.99           C  \\nATOM   2438  HD2 PHE A 164      -4.876  -1.351  12.026  1.00 97.99           H  \\nATOM   2439  CE1 PHE A 164      -7.227  -3.910  10.276  1.00 97.99           C  \\nATOM   2440  HE1 PHE A 164      -7.895  -4.598   9.779  1.00 97.99           H  \\nATOM   2441  CE2 PHE A 164      -5.258  -2.478  10.228  1.00 97.99           C  \\nATOM   2442  HE2 PHE A 164      -4.407  -2.070   9.703  1.00 97.99           H  \\nATOM   2443  CZ  PHE A 164      -6.109  -3.400   9.592  1.00 97.99           C  \\nATOM   2444  HZ  PHE A 164      -5.920  -3.695   8.571  1.00 97.99           H  \\nATOM   2445  N   GLN A 165      -6.520   0.996  13.468  1.00 97.82           N  \\nATOM   2446  H   GLN A 165      -6.220   0.769  14.405  1.00 97.82           H  \\nATOM   2447  CA  GLN A 165      -5.852   2.118  12.833  1.00 97.82           C  \\nATOM   2448  HA  GLN A 165      -5.989   2.071  11.753  1.00 97.82           H  \\nATOM   2449  C   GLN A 165      -4.360   2.032  13.154  1.00 97.82           C  \\nATOM   2450  CB  GLN A 165      -6.471   3.422  13.354  1.00 97.82           C  \\nATOM   2451  HB2 GLN A 165      -7.527   3.441  13.085  1.00 97.82           H  \\nATOM   2452  HB3 GLN A 165      -6.393   3.446  14.441  1.00 97.82           H  \\nATOM   2453  O   GLN A 165      -3.974   1.978  14.323  1.00 97.82           O  \\nATOM   2454  CG  GLN A 165      -5.779   4.665  12.775  1.00 97.82           C  \\nATOM   2455  HG2 GLN A 165      -5.839   4.643  11.687  1.00 97.82           H  \\nATOM   2456  HG3 GLN A 165      -4.725   4.661  13.055  1.00 97.82           H  \\nATOM   2457  CD  GLN A 165      -6.372   5.980  13.264  1.00 97.82           C  \\nATOM   2458  NE2 GLN A 165      -5.875   7.088  12.760  1.00 97.82           N  \\nATOM   2459 HE21 GLN A 165      -6.272   7.954  13.096  1.00 97.82           H  \\nATOM   2460 HE22 GLN A 165      -5.124   7.039  12.086  1.00 97.82           H  \\nATOM   2461  OE1 GLN A 165      -7.247   6.058  14.114  1.00 97.82           O  \\nATOM   2462  N   ILE A 166      -3.525   2.062  12.116  1.00 96.90           N  \\nATOM   2463  H   ILE A 166      -3.906   2.161  11.186  1.00 96.90           H  \\nATOM   2464  CA  ILE A 166      -2.087   2.270  12.294  1.00 96.90           C  \\nATOM   2465  HA  ILE A 166      -1.768   1.750  13.198  1.00 96.90           H  \\nATOM   2466  C   ILE A 166      -1.873   3.771  12.487  1.00 96.90           C  \\nATOM   2467  CB  ILE A 166      -1.264   1.685  11.129  1.00 96.90           C  \\nATOM   2468  HB  ILE A 166      -1.590   2.152  10.200  1.00 96.90           H  \\nATOM   2469  O   ILE A 166      -2.141   4.558  11.578  1.00 96.90           O  \\nATOM   2470  CG1 ILE A 166      -1.465   0.154  11.027  1.00 96.90           C  \\nATOM   2471 HG12 ILE A 166      -2.513  -0.103  11.178  1.00 96.90           H  \\nATOM   2472 HG13 ILE A 166      -0.888  -0.340  11.809  1.00 96.90           H  \\nATOM   2473  CG2 ILE A 166       0.226   2.006  11.357  1.00 96.90           C  \\nATOM   2474 HG21 ILE A 166       0.400   3.081  11.303  1.00 96.90           H  \\nATOM   2475 HG22 ILE A 166       0.548   1.644  12.333  1.00 96.90           H  \\nATOM   2476 HG23 ILE A 166       0.842   1.542  10.587  1.00 96.90           H  \\nATOM   2477  CD1 ILE A 166      -1.057  -0.415   9.664  1.00 96.90           C  \\nATOM   2478 HD11 ILE A 166      -1.160  -1.500   9.677  1.00 96.90           H  \\nATOM   2479 HD12 ILE A 166      -1.703  -0.006   8.887  1.00 96.90           H  \\nATOM   2480 HD13 ILE A 166      -0.023  -0.159   9.431  1.00 96.90           H  \\nATOM   2481  N   THR A 167      -1.456   4.180  13.683  1.00 93.30           N  \\nATOM   2482  H   THR A 167      -1.273   3.492  14.400  1.00 93.30           H  \\nATOM   2483  CA  THR A 167      -1.315   5.600  14.048  1.00 93.30           C  \\nATOM   2484  HA  THR A 167      -2.115   6.170  13.575  1.00 93.30           H  \\nATOM   2485  C   THR A 167      -0.017   6.202  13.546  1.00 93.30           C  \\nATOM   2486  CB  THR A 167      -1.412   5.797  15.567  1.00 93.30           C  \\nATOM   2487  HB  THR A 167      -0.971   6.750  15.859  1.00 93.30           H  \\nATOM   2488  O   THR A 167      -0.003   7.338  13.070  1.00 93.30           O  \\nATOM   2489  CG2 THR A 167      -2.884   5.782  15.979  1.00 93.30           C  \\nATOM   2490 HG21 THR A 167      -3.428   6.564  15.451  1.00 93.30           H  \\nATOM   2491 HG22 THR A 167      -3.327   4.812  15.752  1.00 93.30           H  \\nATOM   2492 HG23 THR A 167      -2.947   5.973  17.050  1.00 93.30           H  \\nATOM   2493  OG1 THR A 167      -0.819   4.741  16.286  1.00 93.30           O  \\nATOM   2494  HG1 THR A 167       0.123   4.871  16.419  1.00 93.30           H  \\nATOM   2495  N   LYS A 168       1.068   5.434  13.619  1.00 93.75           N  \\nATOM   2496  H   LYS A 168       0.989   4.546  14.093  1.00 93.75           H  \\nATOM   2497  CA  LYS A 168       2.384   5.844  13.151  1.00 93.75           C  \\nATOM   2498  HA  LYS A 168       2.256   6.575  12.353  1.00 93.75           H  \\nATOM   2499  C   LYS A 168       3.147   4.651  12.603  1.00 93.75           C  \\nATOM   2500  CB  LYS A 168       3.129   6.499  14.319  1.00 93.75           C  \\nATOM   2501  HB2 LYS A 168       2.461   7.208  14.810  1.00 93.75           H  \\nATOM   2502  HB3 LYS A 168       3.404   5.730  15.040  1.00 93.75           H  \\nATOM   2503  O   LYS A 168       3.053   3.540  13.117  1.00 93.75           O  \\nATOM   2504  CG  LYS A 168       4.399   7.238  13.893  1.00 93.75           C  \\nATOM   2505  HG2 LYS A 168       4.154   7.995  13.148  1.00 93.75           H  \\nATOM   2506  HG3 LYS A 168       5.118   6.534  13.476  1.00 93.75           H  \\nATOM   2507  CD  LYS A 168       5.010   7.906  15.126  1.00 93.75           C  \\nATOM   2508  HD2 LYS A 168       4.295   8.620  15.534  1.00 93.75           H  \\nATOM   2509  HD3 LYS A 168       5.213   7.144  15.878  1.00 93.75           H  \\nATOM   2510  CE  LYS A 168       6.305   8.625  14.753  1.00 93.75           C  \\nATOM   2511  HE2 LYS A 168       7.001   7.895  14.339  1.00 93.75           H  \\nATOM   2512  HE3 LYS A 168       6.086   9.361  13.979  1.00 93.75           H  \\nATOM   2513  NZ  LYS A 168       6.887   9.284  15.946  1.00 93.75           N  \\nATOM   2514  HZ1 LYS A 168       6.238   9.964  16.315  1.00 93.75           H  \\nATOM   2515  HZ2 LYS A 168       7.056   8.600  16.669  1.00 93.75           H  \\nATOM   2516  HZ3 LYS A 168       7.755   9.749  15.720  1.00 93.75           H  \\nATOM   2517  N   PHE A 169       3.946   4.928  11.585  1.00 97.29           N  \\nATOM   2518  H   PHE A 169       3.962   5.875  11.235  1.00 97.29           H  \\nATOM   2519  CA  PHE A 169       4.922   4.003  11.033  1.00 97.29           C  \\nATOM   2520  HA  PHE A 169       4.697   2.985  11.348  1.00 97.29           H  \\nATOM   2521  C   PHE A 169       6.304   4.402  11.560  1.00 97.29           C  \\nATOM   2522  CB  PHE A 169       4.823   4.055   9.510  1.00 97.29           C  \\nATOM   2523  HB2 PHE A 169       4.931   5.098   9.212  1.00 97.29           H  \\nATOM   2524  HB3 PHE A 169       5.646   3.487   9.076  1.00 97.29           H  \\nATOM   2525  O   PHE A 169       6.750   5.534  11.353  1.00 97.29           O  \\nATOM   2526  CG  PHE A 169       3.507   3.532   8.962  1.00 97.29           C  \\nATOM   2527  CD1 PHE A 169       3.412   2.206   8.503  1.00 97.29           C  \\nATOM   2528  HD1 PHE A 169       4.277   1.560   8.526  1.00 97.29           H  \\nATOM   2529  CD2 PHE A 169       2.374   4.368   8.910  1.00 97.29           C  \\nATOM   2530  HD2 PHE A 169       2.438   5.393   9.245  1.00 97.29           H  \\nATOM   2531  CE1 PHE A 169       2.195   1.722   7.992  1.00 97.29           C  \\nATOM   2532  HE1 PHE A 169       2.127   0.706   7.633  1.00 97.29           H  \\nATOM   2533  CE2 PHE A 169       1.157   3.883   8.399  1.00 97.29           C  \\nATOM   2534  HE2 PHE A 169       0.292   4.529   8.350  1.00 97.29           H  \\nATOM   2535  CZ  PHE A 169       1.070   2.562   7.930  1.00 97.29           C  \\nATOM   2536  HZ  PHE A 169       0.144   2.193   7.516  1.00 97.29           H  \\nATOM   2537  N   THR A 170       6.959   3.504  12.288  1.00 96.26           N  \\nATOM   2538  H   THR A 170       6.530   2.600  12.428  1.00 96.26           H  \\nATOM   2539  CA  THR A 170       8.263   3.731  12.935  1.00 96.26           C  \\nATOM   2540  HA  THR A 170       8.586   4.756  12.752  1.00 96.26           H  \\nATOM   2541  C   THR A 170       9.340   2.825  12.327  1.00 96.26           C  \\nATOM   2542  CB  THR A 170       8.153   3.554  14.460  1.00 96.26           C  \\nATOM   2543  HB  THR A 170       9.119   3.777  14.915  1.00 96.26           H  \\nATOM   2544  O   THR A 170       9.073   2.110  11.364  1.00 96.26           O  \\nATOM   2545  CG2 THR A 170       7.107   4.479  15.087  1.00 96.26           C  \\nATOM   2546 HG21 THR A 170       7.309   5.510  14.800  1.00 96.26           H  \\nATOM   2547 HG22 THR A 170       7.148   4.396  16.173  1.00 96.26           H  \\nATOM   2548 HG23 THR A 170       6.107   4.200  14.757  1.00 96.26           H  \\nATOM   2549  OG1 THR A 170       7.803   2.232  14.767  1.00 96.26           O  \\nATOM   2550  HG1 THR A 170       7.546   2.184  15.690  1.00 96.26           H  \\nATOM   2551  N   GLY A 171      10.577   2.870  12.831  1.00 96.19           N  \\nATOM   2552  H   GLY A 171      10.767   3.456  13.631  1.00 96.19           H  \\nATOM   2553  CA  GLY A 171      11.678   2.051  12.309  1.00 96.19           C  \\nATOM   2554  HA2 GLY A 171      11.349   1.014  12.240  1.00 96.19           H  \\nATOM   2555  HA3 GLY A 171      12.522   2.086  12.998  1.00 96.19           H  \\nATOM   2556  C   GLY A 171      12.158   2.520  10.933  1.00 96.19           C  \\nATOM   2557  O   GLY A 171      12.394   3.712  10.725  1.00 96.19           O  \\nATOM   2558  N   ALA A 172      12.277   1.594   9.980  1.00 96.38           N  \\nATOM   2559  H   ALA A 172      11.978   0.654  10.199  1.00 96.38           H  \\nATOM   2560  CA  ALA A 172      12.730   1.850   8.608  1.00 96.38           C  \\nATOM   2561  HA  ALA A 172      13.571   2.542   8.659  1.00 96.38           H  \\nATOM   2562  C   ALA A 172      11.659   2.538   7.730  1.00 96.38           C  \\nATOM   2563  CB  ALA A 172      13.254   0.535   8.012  1.00 96.38           C  \\nATOM   2564  HB1 ALA A 172      12.441  -0.185   7.925  1.00 96.38           H  \\nATOM   2565  HB2 ALA A 172      14.035   0.125   8.652  1.00 96.38           H  \\nATOM   2566  HB3 ALA A 172      13.672   0.720   7.023  1.00 96.38           H  \\nATOM   2567  O   ALA A 172      11.520   2.237   6.547  1.00 96.38           O  \\nATOM   2568  N   ALA A 173      10.863   3.440   8.308  1.00 97.63           N  \\nATOM   2569  H   ALA A 173      11.042   3.671   9.275  1.00 97.63           H  \\nATOM   2570  CA  ALA A 173       9.774   4.116   7.619  1.00 97.63           C  \\nATOM   2571  HA  ALA A 173       9.099   3.352   7.234  1.00 97.63           H  \\nATOM   2572  C   ALA A 173      10.281   4.950   6.428  1.00 97.63           C  \\nATOM   2573  CB  ALA A 173       8.993   4.964   8.630  1.00 97.63           C  \\nATOM   2574  HB1 ALA A 173       8.163   5.464   8.131  1.00 97.63           H  \\nATOM   2575  HB2 ALA A 173       8.596   4.324   9.417  1.00 97.63           H  \\nATOM   2576  HB3 ALA A 173       9.647   5.715   9.073  1.00 97.63           H  \\nATOM   2577  O   ALA A 173      11.243   5.726   6.540  1.00 97.63           O  \\nATOM   2578  N   GLY A 174       9.589   4.810   5.297  1.00 98.28           N  \\nATOM   2579  H   GLY A 174       8.824   4.151   5.270  1.00 98.28           H  \\nATOM   2580  CA  GLY A 174       9.815   5.614   4.104  1.00 98.28           C  \\nATOM   2581  HA2 GLY A 174       9.247   5.180   3.281  1.00 98.28           H  \\nATOM   2582  HA3 GLY A 174      10.872   5.578   3.841  1.00 98.28           H  \\nATOM   2583  C   GLY A 174       9.389   7.083   4.272  1.00 98.28           C  \\nATOM   2584  O   GLY A 174       8.844   7.474   5.314  1.00 98.28           O  \\nATOM   2585  N   PRO A 175       9.684   7.934   3.276  1.00 98.33           N  \\nATOM   2586  CA  PRO A 175       9.400   9.365   3.337  1.00 98.33           C  \\nATOM   2587  HA  PRO A 175       9.904   9.796   4.202  1.00 98.33           H  \\nATOM   2588  C   PRO A 175       7.907   9.714   3.449  1.00 98.33           C  \\nATOM   2589  CB  PRO A 175      10.027   9.944   2.067  1.00 98.33           C  \\nATOM   2590  HB2 PRO A 175       9.455  10.790   1.688  1.00 98.33           H  \\nATOM   2591  HB3 PRO A 175      11.052  10.250   2.276  1.00 98.33           H  \\nATOM   2592  O   PRO A 175       7.573  10.609   4.226  1.00 98.33           O  \\nATOM   2593  CG  PRO A 175      10.043   8.764   1.093  1.00 98.33           C  \\nATOM   2594  HG2 PRO A 175       9.057   8.635   0.646  1.00 98.33           H  \\nATOM   2595  HG3 PRO A 175      10.807   8.876   0.324  1.00 98.33           H  \\nATOM   2596  CD  PRO A 175      10.334   7.592   2.019  1.00 98.33           C  \\nATOM   2597  HD2 PRO A 175      11.409   7.500   2.174  1.00 98.33           H  \\nATOM   2598  HD3 PRO A 175       9.937   6.673   1.588  1.00 98.33           H  \\nATOM   2599  N   TYR A 176       6.997   9.025   2.751  1.00 98.61           N  \\nATOM   2600  H   TYR A 176       7.281   8.206   2.233  1.00 98.61           H  \\nATOM   2601  CA  TYR A 176       5.557   9.273   2.898  1.00 98.61           C  \\nATOM   2602  HA  TYR A 176       5.378  10.346   2.822  1.00 98.61           H  \\nATOM   2603  C   TYR A 176       5.036   8.829   4.258  1.00 98.61           C  \\nATOM   2604  CB  TYR A 176       4.748   8.574   1.804  1.00 98.61           C  \\nATOM   2605  HB2 TYR A 176       3.691   8.645   2.059  1.00 98.61           H  \\nATOM   2606  HB3 TYR A 176       4.999   7.514   1.778  1.00 98.61           H  \\nATOM   2607  O   TYR A 176       4.200   9.513   4.844  1.00 98.61           O  \\nATOM   2608  CG  TYR A 176       4.922   9.183   0.435  1.00 98.61           C  \\nATOM   2609  CD1 TYR A 176       4.435  10.479   0.184  1.00 98.61           C  \\nATOM   2610  HD1 TYR A 176       3.947  11.026   0.977  1.00 98.61           H  \\nATOM   2611  CD2 TYR A 176       5.529   8.447  -0.596  1.00 98.61           C  \\nATOM   2612  HD2 TYR A 176       5.870   7.439  -0.413  1.00 98.61           H  \\nATOM   2613  CE1 TYR A 176       4.577  11.045  -1.097  1.00 98.61           C  \\nATOM   2614  HE1 TYR A 176       4.207  12.040  -1.296  1.00 98.61           H  \\nATOM   2615  CE2 TYR A 176       5.665   9.004  -1.877  1.00 98.61           C  \\nATOM   2616  HE2 TYR A 176       6.139   8.439  -2.666  1.00 98.61           H  \\nATOM   2617  OH  TYR A 176       5.297  10.813  -3.379  1.00 98.61           O  \\nATOM   2618  HH  TYR A 176       5.770  10.206  -3.954  1.00 98.61           H  \\nATOM   2619  CZ  TYR A 176       5.184  10.298  -2.130  1.00 98.61           C  \\nATOM   2620  N   CYS A 177       5.544   7.727   4.801  1.00 98.53           N  \\nATOM   2621  H   CYS A 177       6.160   7.158   4.238  1.00 98.53           H  \\nATOM   2622  CA  CYS A 177       5.188   7.295   6.148  1.00 98.53           C  \\nATOM   2623  HA  CYS A 177       4.103   7.278   6.249  1.00 98.53           H  \\nATOM   2624  C   CYS A 177       5.670   8.260   7.230  1.00 98.53           C  \\nATOM   2625  CB  CYS A 177       5.736   5.892   6.372  1.00 98.53           C  \\nATOM   2626  HB2 CYS A 177       5.722   5.685   7.442  1.00 98.53           H  \\nATOM   2627  HB3 CYS A 177       6.774   5.852   6.042  1.00 98.53           H  \\nATOM   2628  O   CYS A 177       4.920   8.576   8.157  1.00 98.53           O  \\nATOM   2629  SG  CYS A 177       4.820   4.580   5.541  1.00 98.53           S  \\nATOM   2630  N   LYS A 178       6.890   8.791   7.087  1.00 97.95           N  \\nATOM   2631  H   LYS A 178       7.459   8.467   6.318  1.00 97.95           H  \\nATOM   2632  CA  LYS A 178       7.395   9.871   7.946  1.00 97.95           C  \\nATOM   2633  HA  LYS A 178       7.355   9.551   8.988  1.00 97.95           H  \\nATOM   2634  C   LYS A 178       6.502  11.111   7.850  1.00 97.95           C  \\nATOM   2635  CB  LYS A 178       8.849  10.189   7.570  1.00 97.95           C  \\nATOM   2636  HB2 LYS A 178       8.927  10.277   6.486  1.00 97.95           H  \\nATOM   2637  HB3 LYS A 178       9.135  11.145   8.009  1.00 97.95           H  \\nATOM   2638  O   LYS A 178       6.108  11.644   8.884  1.00 97.95           O  \\nATOM   2639  CG  LYS A 178       9.819   9.112   8.078  1.00 97.95           C  \\nATOM   2640  HG2 LYS A 178       9.417   8.119   7.873  1.00 97.95           H  \\nATOM   2641  HG3 LYS A 178       9.934   9.222   9.157  1.00 97.95           H  \\nATOM   2642  CD  LYS A 178      11.186   9.234   7.395  1.00 97.95           C  \\nATOM   2643  HD2 LYS A 178      11.566  10.251   7.488  1.00 97.95           H  \\nATOM   2644  HD3 LYS A 178      11.054   8.996   6.339  1.00 97.95           H  \\nATOM   2645  CE  LYS A 178      12.175   8.251   8.031  1.00 97.95           C  \\nATOM   2646  HE2 LYS A 178      11.624   7.367   8.351  1.00 97.95           H  \\nATOM   2647  HE3 LYS A 178      12.614   8.713   8.916  1.00 97.95           H  \\nATOM   2648  NZ  LYS A 178      13.226   7.825   7.078  1.00 97.95           N  \\nATOM   2649  HZ1 LYS A 178      13.933   7.274   7.545  1.00 97.95           H  \\nATOM   2650  HZ2 LYS A 178      13.656   8.604   6.602  1.00 97.95           H  \\nATOM   2651  HZ3 LYS A 178      12.796   7.207   6.405  1.00 97.95           H  \\nATOM   2652  N   ALA A 179       6.127  11.512   6.634  1.00 97.97           N  \\nATOM   2653  H   ALA A 179       6.517  11.041   5.830  1.00 97.97           H  \\nATOM   2654  CA  ALA A 179       5.238  12.647   6.393  1.00 97.97           C  \\nATOM   2655  HA  ALA A 179       5.645  13.523   6.898  1.00 97.97           H  \\nATOM   2656  C   ALA A 179       3.815  12.425   6.932  1.00 97.97           C  \\nATOM   2657  CB  ALA A 179       5.222  12.928   4.890  1.00 97.97           C  \\nATOM   2658  HB1 ALA A 179       4.603  13.803   4.690  1.00 97.97           H  \\nATOM   2659  HB2 ALA A 179       6.235  13.128   4.542  1.00 97.97           H  \\nATOM   2660  HB3 ALA A 179       4.813  12.072   4.354  1.00 97.97           H  \\nATOM   2661  O   ALA A 179       3.204  13.356   7.448  1.00 97.97           O  \\nATOM   2662  N   LEU A 180       3.285  11.199   6.878  1.00 97.55           N  \\nATOM   2663  H   LEU A 180       3.785  10.482   6.372  1.00 97.55           H  \\nATOM   2664  CA  LEU A 180       1.994  10.860   7.477  1.00 97.55           C  \\nATOM   2665  HA  LEU A 180       1.255  11.571   7.108  1.00 97.55           H  \\nATOM   2666  C   LEU A 180       2.015  11.051   8.994  1.00 97.55           C  \\nATOM   2667  CB  LEU A 180       1.572   9.442   7.050  1.00 97.55           C  \\nATOM   2668  HB2 LEU A 180       1.516   9.404   5.962  1.00 97.55           H  \\nATOM   2669  HB3 LEU A 180       2.343   8.740   7.369  1.00 97.55           H  \\nATOM   2670  O   LEU A 180       1.101  11.665   9.539  1.00 97.55           O  \\nATOM   2671  CG  LEU A 180       0.223   8.978   7.637  1.00 97.55           C  \\nATOM   2672  HG  LEU A 180       0.275   8.999   8.726  1.00 97.55           H  \\nATOM   2673  CD1 LEU A 180      -0.958   9.841   7.181  1.00 97.55           C  \\nATOM   2674 HD11 LEU A 180      -1.877   9.449   7.616  1.00 97.55           H  \\nATOM   2675 HD12 LEU A 180      -1.038   9.821   6.094  1.00 97.55           H  \\nATOM   2676 HD13 LEU A 180      -0.827  10.865   7.531  1.00 97.55           H  \\nATOM   2677  CD2 LEU A 180      -0.055   7.539   7.209  1.00 97.55           C  \\nATOM   2678 HD21 LEU A 180      -0.163   7.485   6.125  1.00 97.55           H  \\nATOM   2679 HD22 LEU A 180       0.771   6.897   7.515  1.00 97.55           H  \\nATOM   2680 HD23 LEU A 180      -0.968   7.179   7.681  1.00 97.55           H  \\nATOM   2681  N   GLY A 181       3.085  10.603   9.655  1.00 93.71           N  \\nATOM   2682  H   GLY A 181       3.799  10.117   9.132  1.00 93.71           H  \\nATOM   2683  CA  GLY A 181       3.265  10.776  11.097  1.00 93.71           C  \\nATOM   2684  HA2 GLY A 181       2.416  10.332  11.619  1.00 93.71           H  \\nATOM   2685  HA3 GLY A 181       4.173  10.254  11.397  1.00 93.71           H  \\nATOM   2686  C   GLY A 181       3.389  12.236  11.552  1.00 93.71           C  \\nATOM   2687  O   GLY A 181       3.064  12.535  12.698  1.00 93.71           O  \\nATOM   2688  N   SER A 182       3.833  13.144  10.678  1.00 94.94           N  \\nATOM   2689  H   SER A 182       4.159  12.830   9.775  1.00 94.94           H  \\nATOM   2690  CA  SER A 182       3.905  14.591  10.940  1.00 94.94           C  \\nATOM   2691  HA  SER A 182       3.814  14.762  12.012  1.00 94.94           H  \\nATOM   2692  C   SER A 182       2.780  15.401  10.281  1.00 94.94           C  \\nATOM   2693  CB  SER A 182       5.273  15.130  10.512  1.00 94.94           C  \\nATOM   2694  HB2 SER A 182       5.324  16.198  10.724  1.00 94.94           H  \\nATOM   2695  HB3 SER A 182       6.054  14.623  11.079  1.00 94.94           H  \\nATOM   2696  O   SER A 182       2.767  16.625  10.396  1.00 94.94           O  \\nATOM   2697  OG  SER A 182       5.483  14.910   9.131  1.00 94.94           O  \\nATOM   2698  HG  SER A 182       6.073  15.588   8.793  1.00 94.94           H  \\nATOM   2699  N   ASN A 183       1.842  14.742   9.590  1.00 94.78           N  \\nATOM   2700  H   ASN A 183       1.925  13.737   9.544  1.00 94.78           H  \\nATOM   2701  CA  ASN A 183       0.819  15.363   8.742  1.00 94.78           C  \\nATOM   2702  HA  ASN A 183       0.398  14.557   8.141  1.00 94.78           H  \\nATOM   2703  C   ASN A 183       1.387  16.378   7.718  1.00 94.78           C  \\nATOM   2704  CB  ASN A 183      -0.310  15.908   9.633  1.00 94.78           C  \\nATOM   2705  HB2 ASN A 183       0.062  16.779  10.172  1.00 94.78           H  \\nATOM   2706  HB3 ASN A 183      -0.595  15.151  10.364  1.00 94.78           H  \\nATOM   2707  O   ASN A 183       0.740  17.370   7.375  1.00 94.78           O  \\nATOM   2708  CG  ASN A 183      -1.557  16.293   8.854  1.00 94.78           C  \\nATOM   2709  ND2 ASN A 183      -2.221  17.347   9.270  1.00 94.78           N  \\nATOM   2710 HD21 ASN A 183      -1.844  17.895  10.030  1.00 94.78           H  \\nATOM   2711 HD22 ASN A 183      -3.042  17.620   8.750  1.00 94.78           H  \\nATOM   2712  OD1 ASN A 183      -1.977  15.649   7.897  1.00 94.78           O  \\nATOM   2713  N   ASP A 184       2.595  16.135   7.204  1.00 97.47           N  \\nATOM   2714  H   ASP A 184       3.069  15.289   7.484  1.00 97.47           H  \\nATOM   2715  CA  ASP A 184       3.228  16.982   6.193  1.00 97.47           C  \\nATOM   2716  HA  ASP A 184       3.018  18.024   6.435  1.00 97.47           H  \\nATOM   2717  C   ASP A 184       2.643  16.699   4.801  1.00 97.47           C  \\nATOM   2718  CB  ASP A 184       4.758  16.828   6.222  1.00 97.47           C  \\nATOM   2719  HB2 ASP A 184       5.126  17.144   7.199  1.00 97.47           H  \\nATOM   2720  HB3 ASP A 184       5.020  15.780   6.083  1.00 97.47           H  \\nATOM   2721  O   ASP A 184       3.099  15.827   4.066  1.00 97.47           O  \\nATOM   2722  CG  ASP A 184       5.448  17.669   5.135  1.00 97.47           C  \\nATOM   2723  OD1 ASP A 184       4.784  18.524   4.498  1.00 97.47           O  \\nATOM   2724  OD2 ASP A 184       6.649  17.454   4.884  1.00 97.47           O  \\nATOM   2725  N   LYS A 185       1.632  17.474   4.399  1.00 97.59           N  \\nATOM   2726  H   LYS A 185       1.260  18.124   5.077  1.00 97.59           H  \\nATOM   2727  CA  LYS A 185       1.024  17.380   3.059  1.00 97.59           C  \\nATOM   2728  HA  LYS A 185       0.866  16.327   2.827  1.00 97.59           H  \\nATOM   2729  C   LYS A 185       1.924  17.903   1.940  1.00 97.59           C  \\nATOM   2730  CB  LYS A 185      -0.322  18.120   3.035  1.00 97.59           C  \\nATOM   2731  HB2 LYS A 185      -0.153  19.162   3.307  1.00 97.59           H  \\nATOM   2732  HB3 LYS A 185      -0.726  18.104   2.023  1.00 97.59           H  \\nATOM   2733  O   LYS A 185       1.594  17.715   0.769  1.00 97.59           O  \\nATOM   2734  CG  LYS A 185      -1.376  17.537   3.983  1.00 97.59           C  \\nATOM   2735  HG2 LYS A 185      -0.980  17.499   4.998  1.00 97.59           H  \\nATOM   2736  HG3 LYS A 185      -2.233  18.211   3.985  1.00 97.59           H  \\nATOM   2737  CD  LYS A 185      -1.858  16.142   3.570  1.00 97.59           C  \\nATOM   2738  HD2 LYS A 185      -2.276  16.175   2.564  1.00 97.59           H  \\nATOM   2739  HD3 LYS A 185      -1.023  15.441   3.590  1.00 97.59           H  \\nATOM   2740  CE  LYS A 185      -2.936  15.719   4.573  1.00 97.59           C  \\nATOM   2741  HE2 LYS A 185      -2.526  15.798   5.580  1.00 97.59           H  \\nATOM   2742  HE3 LYS A 185      -3.768  16.420   4.497  1.00 97.59           H  \\nATOM   2743  NZ  LYS A 185      -3.412  14.342   4.336  1.00 97.59           N  \\nATOM   2744  HZ1 LYS A 185      -3.606  14.173   3.359  1.00 97.59           H  \\nATOM   2745  HZ2 LYS A 185      -4.225  14.126   4.895  1.00 97.59           H  \\nATOM   2746  HZ3 LYS A 185      -2.677  13.689   4.571  1.00 97.59           H  \\nATOM   2747  N   SER A 186       3.052  18.543   2.249  1.00 97.63           N  \\nATOM   2748  H   SER A 186       3.361  18.575   3.210  1.00 97.63           H  \\nATOM   2749  CA  SER A 186       3.924  19.115   1.225  1.00 97.63           C  \\nATOM   2750  HA  SER A 186       3.315  19.763   0.594  1.00 97.63           H  \\nATOM   2751  C   SER A 186       4.529  18.039   0.323  1.00 97.63           C  \\nATOM   2752  CB  SER A 186       5.012  19.981   1.866  1.00 97.63           C  \\nATOM   2753  HB2 SER A 186       4.618  20.493   2.744  1.00 97.63           H  \\nATOM   2754  HB3 SER A 186       5.315  20.740   1.145  1.00 97.63           H  \\nATOM   2755  O   SER A 186       4.853  18.337  -0.826  1.00 97.63           O  \\nATOM   2756  OG  SER A 186       6.143  19.212   2.219  1.00 97.63           O  \\nATOM   2757  HG  SER A 186       5.982  18.866   3.100  1.00 97.63           H  \\nATOM   2758  N   VAL A 187       4.653  16.794   0.801  1.00 97.99           N  \\nATOM   2759  H   VAL A 187       4.444  16.635   1.777  1.00 97.99           H  \\nATOM   2760  CA  VAL A 187       5.100  15.655  -0.011  1.00 97.99           C  \\nATOM   2761  HA  VAL A 187       5.985  15.966  -0.566  1.00 97.99           H  \\nATOM   2762  C   VAL A 187       4.064  15.230  -1.048  1.00 97.99           C  \\nATOM   2763  CB  VAL A 187       5.513  14.439   0.843  1.00 97.99           C  \\nATOM   2764  HB  VAL A 187       5.938  13.686   0.179  1.00 97.99           H  \\nATOM   2765  O   VAL A 187       4.413  14.459  -1.927  1.00 97.99           O  \\nATOM   2766  CG1 VAL A 187       6.600  14.805   1.857  1.00 97.99           C  \\nATOM   2767 HG11 VAL A 187       6.969  13.899   2.339  1.00 97.99           H  \\nATOM   2768 HG12 VAL A 187       6.206  15.470   2.625  1.00 97.99           H  \\nATOM   2769 HG13 VAL A 187       7.434  15.293   1.353  1.00 97.99           H  \\nATOM   2770  CG2 VAL A 187       4.325  13.789   1.554  1.00 97.99           C  \\nATOM   2771 HG21 VAL A 187       4.680  12.943   2.144  1.00 97.99           H  \\nATOM   2772 HG22 VAL A 187       3.843  14.506   2.217  1.00 97.99           H  \\nATOM   2773 HG23 VAL A 187       3.593  13.423   0.834  1.00 97.99           H  \\nATOM   2774  N   CYS A 188       2.814  15.703  -0.998  1.00 98.19           N  \\nATOM   2775  H   CYS A 188       2.560  16.332  -0.249  1.00 98.19           H  \\nATOM   2776  CA  CYS A 188       1.762  15.321  -1.946  1.00 98.19           C  \\nATOM   2777  HA  CYS A 188       1.898  14.270  -2.202  1.00 98.19           H  \\nATOM   2778  C   CYS A 188       1.801  16.094  -3.272  1.00 98.19           C  \\nATOM   2779  CB  CYS A 188       0.393  15.458  -1.274  1.00 98.19           C  \\nATOM   2780  HB2 CYS A 188       0.229  16.503  -1.009  1.00 98.19           H  \\nATOM   2781  HB3 CYS A 188      -0.376  15.177  -1.994  1.00 98.19           H  \\nATOM   2782  O   CYS A 188       1.099  15.713  -4.207  1.00 98.19           O  \\nATOM   2783  SG  CYS A 188       0.174  14.440   0.208  1.00 98.19           S  \\nATOM   2784  N   ASP A 189       2.624  17.140  -3.378  1.00 97.19           N  \\nATOM   2785  H   ASP A 189       3.179  17.403  -2.577  1.00 97.19           H  \\nATOM   2786  CA  ASP A 189       2.802  17.903  -4.616  1.00 97.19           C  \\nATOM   2787  HA  ASP A 189       1.839  18.326  -4.899  1.00 97.19           H  \\nATOM   2788  C   ASP A 189       3.286  16.992  -5.762  1.00 97.19           C  \\nATOM   2789  CB  ASP A 189       3.765  19.071  -4.348  1.00 97.19           C  \\nATOM   2790  HB2 ASP A 189       3.403  19.619  -3.478  1.00 97.19           H  \\nATOM   2791  HB3 ASP A 189       4.746  18.666  -4.098  1.00 97.19           H  \\nATOM   2792  O   ASP A 189       4.241  16.220  -5.609  1.00 97.19           O  \\nATOM   2793  CG  ASP A 189       3.912  20.064  -5.511  1.00 97.19           C  \\nATOM   2794  OD1 ASP A 189       3.581  19.712  -6.668  1.00 97.19           O  \\nATOM   2795  OD2 ASP A 189       4.398  21.178  -5.244  1.00 97.19           O  \\nATOM   2796  N   LYS A 190       2.618  17.098  -6.917  1.00 94.69           N  \\nATOM   2797  H   LYS A 190       1.929  17.835  -6.972  1.00 94.69           H  \\nATOM   2798  CA  LYS A 190       2.934  16.357  -8.146  1.00 94.69           C  \\nATOM   2799  HA  LYS A 190       2.952  15.294  -7.905  1.00 94.69           H  \\nATOM   2800  C   LYS A 190       4.313  16.687  -8.720  1.00 94.69           C  \\nATOM   2801  CB  LYS A 190       1.830  16.582  -9.192  1.00 94.69           C  \\nATOM   2802  HB2 LYS A 190       0.874  16.264  -8.776  1.00 94.69           H  \\nATOM   2803  HB3 LYS A 190       2.050  15.948 -10.051  1.00 94.69           H  \\nATOM   2804  O   LYS A 190       4.925  15.851  -9.376  1.00 94.69           O  \\nATOM   2805  CG  LYS A 190       1.722  18.037  -9.680  1.00 94.69           C  \\nATOM   2806  HG2 LYS A 190       1.478  18.693  -8.844  1.00 94.69           H  \\nATOM   2807  HG3 LYS A 190       2.674  18.349 -10.108  1.00 94.69           H  \\nATOM   2808  CD  LYS A 190       0.644  18.168 -10.758  1.00 94.69           C  \\nATOM   2809  HD2 LYS A 190      -0.327  17.916 -10.330  1.00 94.69           H  \\nATOM   2810  HD3 LYS A 190       0.869  17.474 -11.567  1.00 94.69           H  \\nATOM   2811  CE  LYS A 190       0.636  19.603 -11.291  1.00 94.69           C  \\nATOM   2812  HE2 LYS A 190       0.413  20.281 -10.467  1.00 94.69           H  \\nATOM   2813  HE3 LYS A 190       1.638  19.835 -11.652  1.00 94.69           H  \\nATOM   2814  NZ  LYS A 190      -0.354  19.763 -12.383  1.00 94.69           N  \\nATOM   2815  HZ1 LYS A 190      -1.284  19.554 -12.050  1.00 94.69           H  \\nATOM   2816  HZ2 LYS A 190      -0.335  20.707 -12.743  1.00 94.69           H  \\nATOM   2817  HZ3 LYS A 190      -0.133  19.127 -13.136  1.00 94.69           H  \\nATOM   2818  N   ASN A 191       4.823  17.886  -8.445  1.00 95.55           N  \\nATOM   2819  H   ASN A 191       4.273  18.520  -7.882  1.00 95.55           H  \\nATOM   2820  CA  ASN A 191       6.120  18.346  -8.932  1.00 95.55           C  \\nATOM   2821  HA  ASN A 191       6.280  17.960  -9.939  1.00 95.55           H  \\nATOM   2822  C   ASN A 191       7.286  17.846  -8.066  1.00 95.55           C  \\nATOM   2823  CB  ASN A 191       6.098  19.877  -9.002  1.00 95.55           C  \\nATOM   2824  HB2 ASN A 191       7.055  20.237  -9.380  1.00 95.55           H  \\nATOM   2825  HB3 ASN A 191       5.961  20.277  -7.997  1.00 95.55           H  \\nATOM   2826  O   ASN A 191       8.443  17.943  -8.475  1.00 95.55           O  \\nATOM   2827  CG  ASN A 191       5.011  20.420  -9.912  1.00 95.55           C  \\nATOM   2828  ND2 ASN A 191       4.317  21.440  -9.468  1.00 95.55           N  \\nATOM   2829 HD21 ASN A 191       3.634  21.855 -10.086  1.00 95.55           H  \\nATOM   2830 HD22 ASN A 191       4.407  21.700  -8.496  1.00 95.55           H  \\nATOM   2831  OD1 ASN A 191       4.760  19.947 -11.010  1.00 95.55           O  \\nATOM   2832  N   LYS A 192       7.010  17.302  -6.873  1.00 96.66           N  \\nATOM   2833  H   LYS A 192       6.041  17.182  -6.614  1.00 96.66           H  \\nATOM   2834  CA  LYS A 192       8.044  16.753  -5.993  1.00 96.66           C  \\nATOM   2835  HA  LYS A 192       8.975  17.285  -6.183  1.00 96.66           H  \\nATOM   2836  C   LYS A 192       8.295  15.281  -6.288  1.00 96.66           C  \\nATOM   2837  CB  LYS A 192       7.686  16.959  -4.521  1.00 96.66           C  \\nATOM   2838  HB2 LYS A 192       6.692  16.558  -4.321  1.00 96.66           H  \\nATOM   2839  HB3 LYS A 192       8.409  16.406  -3.923  1.00 96.66           H  \\nATOM   2840  O   LYS A 192       7.365  14.489  -6.401  1.00 96.66           O  \\nATOM   2841  CG  LYS A 192       7.750  18.437  -4.114  1.00 96.66           C  \\nATOM   2842  HG2 LYS A 192       6.953  18.994  -4.607  1.00 96.66           H  \\nATOM   2843  HG3 LYS A 192       8.712  18.861  -4.403  1.00 96.66           H  \\nATOM   2844  CD  LYS A 192       7.599  18.525  -2.596  1.00 96.66           C  \\nATOM   2845  HD2 LYS A 192       8.464  18.062  -2.121  1.00 96.66           H  \\nATOM   2846  HD3 LYS A 192       6.709  17.961  -2.315  1.00 96.66           H  \\nATOM   2847  CE  LYS A 192       7.471  19.972  -2.117  1.00 96.66           C  \\nATOM   2848  HE2 LYS A 192       8.382  20.516  -2.365  1.00 96.66           H  \\nATOM   2849  HE3 LYS A 192       6.642  20.438  -2.651  1.00 96.66           H  \\nATOM   2850  NZ  LYS A 192       7.216  20.011  -0.657  1.00 96.66           N  \\nATOM   2851  HZ1 LYS A 192       7.961  19.587  -0.123  1.00 96.66           H  \\nATOM   2852  HZ2 LYS A 192       7.081  20.956  -0.327  1.00 96.66           H  \\nATOM   2853  HZ3 LYS A 192       6.378  19.484  -0.455  1.00 96.66           H  \\nATOM   2854  N   ASN A 193       9.575  14.921  -6.342  1.00 96.82           N  \\nATOM   2855  H   ASN A 193      10.272  15.640  -6.208  1.00 96.82           H  \\nATOM   2856  CA  ASN A 193      10.034  13.543  -6.449  1.00 96.82           C  \\nATOM   2857  HA  ASN A 193       9.222  12.909  -6.807  1.00 96.82           H  \\nATOM   2858  C   ASN A 193      10.457  13.029  -5.070  1.00 96.82           C  \\nATOM   2859  CB  ASN A 193      11.182  13.498  -7.465  1.00 96.82           C  \\nATOM   2860  HB2 ASN A 193      10.856  13.934  -8.410  1.00 96.82           H  \\nATOM   2861  HB3 ASN A 193      12.025  14.083  -7.097  1.00 96.82           H  \\nATOM   2862  O   ASN A 193      11.546  13.351  -4.600  1.00 96.82           O  \\nATOM   2863  CG  ASN A 193      11.626  12.073  -7.722  1.00 96.82           C  \\nATOM   2864  ND2 ASN A 193      12.825  11.708  -7.344  1.00 96.82           N  \\nATOM   2865 HD21 ASN A 193      13.102  10.750  -7.499  1.00 96.82           H  \\nATOM   2866 HD22 ASN A 193      13.391  12.333  -6.788  1.00 96.82           H  \\nATOM   2867  OD1 ASN A 193      10.893  11.292  -8.312  1.00 96.82           O  \\nATOM   2868  N   ILE A 194       9.594  12.245  -4.434  1.00 98.19           N  \\nATOM   2869  H   ILE A 194       8.727  12.000  -4.890  1.00 98.19           H  \\nATOM   2870  CA  ILE A 194       9.796  11.746  -3.073  1.00 98.19           C  \\nATOM   2871  HA  ILE A 194      10.578  12.323  -2.579  1.00 98.19           H  \\nATOM   2872  C   ILE A 194      10.279  10.295  -3.145  1.00 98.19           C  \\nATOM   2873  CB  ILE A 194       8.498  11.929  -2.261  1.00 98.19           C  \\nATOM   2874  HB  ILE A 194       7.743  11.273  -2.694  1.00 98.19           H  \\nATOM   2875  O   ILE A 194       9.488   9.358  -3.082  1.00 98.19           O  \\nATOM   2876  CG1 ILE A 194       7.935  13.368  -2.317  1.00 98.19           C  \\nATOM   2877 HG12 ILE A 194       7.671  13.618  -3.344  1.00 98.19           H  \\nATOM   2878 HG13 ILE A 194       7.005  13.393  -1.749  1.00 98.19           H  \\nATOM   2879  CG2 ILE A 194       8.703  11.502  -0.803  1.00 98.19           C  \\nATOM   2880 HG21 ILE A 194       8.878  10.426  -0.772  1.00 98.19           H  \\nATOM   2881 HG22 ILE A 194       9.553  12.021  -0.359  1.00 98.19           H  \\nATOM   2882 HG23 ILE A 194       7.801  11.701  -0.224  1.00 98.19           H  \\nATOM   2883  CD1 ILE A 194       8.866  14.462  -1.774  1.00 98.19           C  \\nATOM   2884 HD11 ILE A 194       9.132  14.259  -0.737  1.00 98.19           H  \\nATOM   2885 HD12 ILE A 194       9.772  14.525  -2.377  1.00 98.19           H  \\nATOM   2886 HD13 ILE A 194       8.349  15.421  -1.808  1.00 98.19           H  \\nATOM   2887  N   ALA A 195      11.581  10.108  -3.357  1.00 97.43           N  \\nATOM   2888  H   ALA A 195      12.192  10.912  -3.365  1.00 97.43           H  \\nATOM   2889  CA  ALA A 195      12.186   8.780  -3.429  1.00 97.43           C  \\nATOM   2890  HA  ALA A 195      11.428   8.078  -3.774  1.00 97.43           H  \\nATOM   2891  C   ALA A 195      12.654   8.294  -2.049  1.00 97.43           C  \\nATOM   2892  CB  ALA A 195      13.315   8.790  -4.466  1.00 97.43           C  \\nATOM   2893  HB1 ALA A 195      14.093   9.489  -4.159  1.00 97.43           H  \\nATOM   2894  HB2 ALA A 195      13.748   7.793  -4.538  1.00 97.43           H  \\nATOM   2895  HB3 ALA A 195      12.921   9.082  -5.439  1.00 97.43           H  \\nATOM   2896  O   ALA A 195      13.210   9.070  -1.273  1.00 97.43           O  \\nATOM   2897  N   GLY A 196      12.444   7.007  -1.763  1.00 97.18           N  \\nATOM   2898  H   GLY A 196      11.954   6.442  -2.442  1.00 97.18           H  \\nATOM   2899  CA  GLY A 196      13.068   6.309  -0.635  1.00 97.18           C  \\nATOM   2900  HA2 GLY A 196      12.348   5.617  -0.200  1.00 97.18           H  \\nATOM   2901  HA3 GLY A 196      13.361   7.019   0.139  1.00 97.18           H  \\nATOM   2902  C   GLY A 196      14.309   5.520  -1.060  1.00 97.18           C  \\nATOM   2903  O   GLY A 196      14.778   5.641  -2.194  1.00 97.18           O  \\nATOM   2904  N   ASP A 197      14.797   4.647  -0.176  1.00 96.42           N  \\nATOM   2905  H   ASP A 197      14.439   4.666   0.768  1.00 96.42           H  \\nATOM   2906  CA  ASP A 197      16.004   3.833  -0.406  1.00 96.42           C  \\nATOM   2907  HA  ASP A 197      16.852   4.499  -0.568  1.00 96.42           H  \\nATOM   2908  C   ASP A 197      15.896   2.913  -1.633  1.00 96.42           C  \\nATOM   2909  CB  ASP A 197      16.297   2.974   0.834  1.00 96.42           C  \\nATOM   2910  HB2 ASP A 197      17.155   2.336   0.620  1.00 96.42           H  \\nATOM   2911  HB3 ASP A 197      15.439   2.330   1.027  1.00 96.42           H  \\nATOM   2912  O   ASP A 197      16.898   2.530  -2.227  1.00 96.42           O  \\nATOM   2913  CG  ASP A 197      16.617   3.779   2.096  1.00 96.42           C  \\nATOM   2914  OD1 ASP A 197      16.955   4.976   1.969  1.00 96.42           O  \\nATOM   2915  OD2 ASP A 197      16.470   3.186   3.186  1.00 96.42           O  \\nATOM   2916  N   TRP A 198      14.683   2.590  -2.079  1.00 96.97           N  \\nATOM   2917  H   TRP A 198      13.890   2.946  -1.564  1.00 96.97           H  \\nATOM   2918  CA  TRP A 198      14.424   1.836  -3.310  1.00 96.97           C  \\nATOM   2919  HA  TRP A 198      15.090   0.973  -3.335  1.00 96.97           H  \\nATOM   2920  C   TRP A 198      14.686   2.631  -4.605  1.00 96.97           C  \\nATOM   2921  CB  TRP A 198      12.984   1.330  -3.260  1.00 96.97           C  \\nATOM   2922  HB2 TRP A 198      12.739   0.821  -4.193  1.00 96.97           H  \\nATOM   2923  HB3 TRP A 198      12.900   0.602  -2.453  1.00 96.97           H  \\nATOM   2924  O   TRP A 198      14.674   2.038  -5.681  1.00 96.97           O  \\nATOM   2925  CG  TRP A 198      11.988   2.409  -3.006  1.00 96.97           C  \\nATOM   2926  CD1 TRP A 198      11.478   3.240  -3.932  1.00 96.97           C  \\nATOM   2927  HD1 TRP A 198      11.712   3.214  -4.987  1.00 96.97           H  \\nATOM   2928  CD2 TRP A 198      11.429   2.841  -1.732  1.00 96.97           C  \\nATOM   2929  CE2 TRP A 198      10.546   3.934  -1.976  1.00 96.97           C  \\nATOM   2930  CE3 TRP A 198      11.597   2.423  -0.397  1.00 96.97           C  \\nATOM   2931  HE3 TRP A 198      12.263   1.603  -0.175  1.00 96.97           H  \\nATOM   2932  NE1 TRP A 198      10.626   4.132  -3.330  1.00 96.97           N  \\nATOM   2933  HE1 TRP A 198      10.202   4.895  -3.839  1.00 96.97           H  \\nATOM   2934  CH2 TRP A 198      10.018   4.116   0.361  1.00 96.97           C  \\nATOM   2935  HH2 TRP A 198       9.476   4.586   1.169  1.00 96.97           H  \\nATOM   2936  CZ2 TRP A 198       9.842   4.572  -0.953  1.00 96.97           C  \\nATOM   2937  HZ2 TRP A 198       9.170   5.392  -1.159  1.00 96.97           H  \\nATOM   2938  CZ3 TRP A 198      10.894   3.056   0.642  1.00 96.97           C  \\nATOM   2939  HZ3 TRP A 198      11.015   2.723   1.662  1.00 96.97           H  \\nATOM   2940  N   GLY A 199      14.945   3.942  -4.529  1.00 96.97           N  \\nATOM   2941  H   GLY A 199      14.932   4.369  -3.614  1.00 96.97           H  \\nATOM   2942  CA  GLY A 199      15.490   4.745  -5.632  1.00 96.97           C  \\nATOM   2943  HA2 GLY A 199      16.141   4.119  -6.243  1.00 96.97           H  \\nATOM   2944  HA3 GLY A 199      16.105   5.534  -5.199  1.00 96.97           H  \\nATOM   2945  C   GLY A 199      14.481   5.418  -6.573  1.00 96.97           C  \\nATOM   2946  O   GLY A 199      14.886   5.995  -7.579  1.00 96.97           O  \\nATOM   2947  N   PHE A 200      13.179   5.388  -6.279  1.00 97.32           N  \\nATOM   2948  H   PHE A 200      12.891   4.939  -5.421  1.00 97.32           H  \\nATOM   2949  CA  PHE A 200      12.151   6.069  -7.083  1.00 97.32           C  \\nATOM   2950  HA  PHE A 200      12.603   6.942  -7.555  1.00 97.32           H  \\nATOM   2951  C   PHE A 200      10.994   6.578  -6.216  1.00 97.32           C  \\nATOM   2952  CB  PHE A 200      11.650   5.137  -8.201  1.00 97.32           C  \\nATOM   2953  HB2 PHE A 200      11.023   5.714  -8.879  1.00 97.32           H  \\nATOM   2954  HB3 PHE A 200      12.505   4.791  -8.782  1.00 97.32           H  \\nATOM   2955  O   PHE A 200      10.845   6.158  -5.076  1.00 97.32           O  \\nATOM   2956  CG  PHE A 200      10.863   3.930  -7.726  1.00 97.32           C  \\nATOM   2957  CD1 PHE A 200      11.534   2.740  -7.389  1.00 97.32           C  \\nATOM   2958  HD1 PHE A 200      12.609   2.675  -7.474  1.00 97.32           H  \\nATOM   2959  CD2 PHE A 200       9.458   3.983  -7.630  1.00 97.32           C  \\nATOM   2960  HD2 PHE A 200       8.929   4.882  -7.912  1.00 97.32           H  \\nATOM   2961  CE1 PHE A 200      10.806   1.637  -6.912  1.00 97.32           C  \\nATOM   2962  HE1 PHE A 200      11.326   0.738  -6.617  1.00 97.32           H  \\nATOM   2963  CE2 PHE A 200       8.734   2.864  -7.182  1.00 97.32           C  \\nATOM   2964  HE2 PHE A 200       7.655   2.879  -7.137  1.00 97.32           H  \\nATOM   2965  CZ  PHE A 200       9.409   1.695  -6.803  1.00 97.32           C  \\nATOM   2966  HZ  PHE A 200       8.854   0.839  -6.449  1.00 97.32           H  \\nATOM   2967  N   ASP A 201      10.155   7.475  -6.737  1.00 98.05           N  \\nATOM   2968  H   ASP A 201      10.310   7.809  -7.677  1.00 98.05           H  \\nATOM   2969  CA  ASP A 201       8.926   7.898  -6.048  1.00 98.05           C  \\nATOM   2970  HA  ASP A 201       9.107   7.903  -4.973  1.00 98.05           H  \\nATOM   2971  C   ASP A 201       7.774   6.926  -6.367  1.00 98.05           C  \\nATOM   2972  CB  ASP A 201       8.589   9.341  -6.448  1.00 98.05           C  \\nATOM   2973  HB2 ASP A 201       8.405   9.385  -7.522  1.00 98.05           H  \\nATOM   2974  HB3 ASP A 201       9.457   9.965  -6.236  1.00 98.05           H  \\nATOM   2975  O   ASP A 201       7.321   6.906  -7.519  1.00 98.05           O  \\nATOM   2976  CG  ASP A 201       7.379   9.931  -5.711  1.00 98.05           C  \\nATOM   2977  OD1 ASP A 201       6.573   9.183  -5.114  1.00 98.05           O  \\nATOM   2978  OD2 ASP A 201       7.237  11.173  -5.760  1.00 98.05           O  \\nATOM   2979  N   PRO A 202       7.278   6.124  -5.402  1.00 97.85           N  \\nATOM   2980  CA  PRO A 202       6.229   5.142  -5.668  1.00 97.85           C  \\nATOM   2981  HA  PRO A 202       6.552   4.462  -6.456  1.00 97.85           H  \\nATOM   2982  C   PRO A 202       4.915   5.775  -6.132  1.00 97.85           C  \\nATOM   2983  CB  PRO A 202       6.058   4.363  -4.363  1.00 97.85           C  \\nATOM   2984  HB2 PRO A 202       6.693   3.478  -4.378  1.00 97.85           H  \\nATOM   2985  HB3 PRO A 202       5.014   4.092  -4.204  1.00 97.85           H  \\nATOM   2986  O   PRO A 202       4.198   5.173  -6.934  1.00 97.85           O  \\nATOM   2987  CG  PRO A 202       6.540   5.323  -3.279  1.00 97.85           C  \\nATOM   2988  HG2 PRO A 202       6.919   4.792  -2.405  1.00 97.85           H  \\nATOM   2989  HG3 PRO A 202       5.730   5.995  -2.996  1.00 97.85           H  \\nATOM   2990  CD  PRO A 202       7.640   6.108  -3.988  1.00 97.85           C  \\nATOM   2991  HD2 PRO A 202       8.592   5.594  -3.864  1.00 97.85           H  \\nATOM   2992  HD3 PRO A 202       7.704   7.114  -3.572  1.00 97.85           H  \\nATOM   2993  N   ALA A 203       4.637   7.022  -5.739  1.00 98.09           N  \\nATOM   2994  H   ALA A 203       5.312   7.519  -5.175  1.00 98.09           H  \\nATOM   2995  CA  ALA A 203       3.440   7.747  -6.155  1.00 98.09           C  \\nATOM   2996  HA  ALA A 203       2.558   7.175  -5.868  1.00 98.09           H  \\nATOM   2997  C   ALA A 203       3.385   7.987  -7.666  1.00 98.09           C  \\nATOM   2998  CB  ALA A 203       3.414   9.089  -5.431  1.00 98.09           C  \\nATOM   2999  HB1 ALA A 203       2.492   9.611  -5.685  1.00 98.09           H  \\nATOM   3000  HB2 ALA A 203       3.467   8.927  -4.355  1.00 98.09           H  \\nATOM   3001  HB3 ALA A 203       4.253   9.705  -5.754  1.00 98.09           H  \\nATOM   3002  O   ALA A 203       2.311   8.197  -8.224  1.00 98.09           O  \\nATOM   3003  N   LYS A 204       4.528   7.942  -8.360  1.00 97.78           N  \\nATOM   3004  H   LYS A 204       5.384   7.764  -7.854  1.00 97.78           H  \\nATOM   3005  CA  LYS A 204       4.559   8.054  -9.818  1.00 97.78           C  \\nATOM   3006  HA  LYS A 204       3.858   8.831 -10.125  1.00 97.78           H  \\nATOM   3007  C   LYS A 204       4.071   6.780 -10.506  1.00 97.78           C  \\nATOM   3008  CB  LYS A 204       5.960   8.472 -10.281  1.00 97.78           C  \\nATOM   3009  HB2 LYS A 204       6.002   8.426 -11.369  1.00 97.78           H  \\nATOM   3010  HB3 LYS A 204       6.701   7.791  -9.864  1.00 97.78           H  \\nATOM   3011  O   LYS A 204       3.803   6.835 -11.695  1.00 97.78           O  \\nATOM   3012  CG  LYS A 204       6.313   9.902  -9.866  1.00 97.78           C  \\nATOM   3013  HG2 LYS A 204       5.572  10.585 -10.280  1.00 97.78           H  \\nATOM   3014  HG3 LYS A 204       6.304   9.989  -8.779  1.00 97.78           H  \\nATOM   3015  CD  LYS A 204       7.703  10.276 -10.392  1.00 97.78           C  \\nATOM   3016  HD2 LYS A 204       8.452   9.648  -9.911  1.00 97.78           H  \\nATOM   3017  HD3 LYS A 204       7.739  10.127 -11.472  1.00 97.78           H  \\nATOM   3018  CE  LYS A 204       7.967  11.747 -10.072  1.00 97.78           C  \\nATOM   3019  HE2 LYS A 204       7.826  11.900  -9.002  1.00 97.78           H  \\nATOM   3020  HE3 LYS A 204       7.227  12.357 -10.589  1.00 97.78           H  \\nATOM   3021  NZ  LYS A 204       9.337  12.149 -10.462  1.00 97.78           N  \\nATOM   3022  HZ1 LYS A 204       9.528  11.925 -11.428  1.00 97.78           H  \\nATOM   3023  HZ2 LYS A 204       9.998  11.676  -9.862  1.00 97.78           H  \\nATOM   3024  HZ3 LYS A 204       9.451  13.143 -10.323  1.00 97.78           H  \\nATOM   3025  N   TRP A 205       3.971   5.632  -9.836  1.00 97.75           N  \\nATOM   3026  H   TRP A 205       4.171   5.633  -8.846  1.00 97.75           H  \\nATOM   3027  CA  TRP A 205       3.513   4.365 -10.431  1.00 97.75           C  \\nATOM   3028  HA  TRP A 205       3.273   4.519 -11.483  1.00 97.75           H  \\nATOM   3029  C   TRP A 205       2.197   3.868  -9.816  1.00 97.75           C  \\nATOM   3030  CB  TRP A 205       4.618   3.302 -10.378  1.00 97.75           C  \\nATOM   3031  HB2 TRP A 205       4.155   2.350 -10.636  1.00 97.75           H  \\nATOM   3032  HB3 TRP A 205       5.007   3.236  -9.362  1.00 97.75           H  \\nATOM   3033  O   TRP A 205       1.960   2.665  -9.754  1.00 97.75           O  \\nATOM   3034  CG  TRP A 205       5.758   3.420 -11.335  1.00 97.75           C  \\nATOM   3035  CD1 TRP A 205       5.903   2.685 -12.462  1.00 97.75           C  \\nATOM   3036  HD1 TRP A 205       5.213   1.928 -12.806  1.00 97.75           H  \\nATOM   3037  CD2 TRP A 205       6.933   4.276 -11.263  1.00 97.75           C  \\nATOM   3038  CE2 TRP A 205       7.734   4.042 -12.420  1.00 97.75           C  \\nATOM   3039  CE3 TRP A 205       7.390   5.250 -10.357  1.00 97.75           C  \\nATOM   3040  HE3 TRP A 205       6.837   5.410  -9.442  1.00 97.75           H  \\nATOM   3041  NE1 TRP A 205       7.083   3.029 -13.086  1.00 97.75           N  \\nATOM   3042  HE1 TRP A 205       7.399   2.599 -13.943  1.00 97.75           H  \\nATOM   3043  CH2 TRP A 205       9.308   5.760 -11.782  1.00 97.75           C  \\nATOM   3044  HH2 TRP A 205      10.223   6.306 -11.955  1.00 97.75           H  \\nATOM   3045  CZ2 TRP A 205       8.884   4.787 -12.702  1.00 97.75           C  \\nATOM   3046  HZ2 TRP A 205       9.460   4.577 -13.591  1.00 97.75           H  \\nATOM   3047  CZ3 TRP A 205       8.567   5.980 -10.608  1.00 97.75           C  \\nATOM   3048  HZ3 TRP A 205       8.912   6.696  -9.876  1.00 97.75           H  \\nATOM   3049  N   ALA A 206       1.336   4.782  -9.371  1.00 98.26           N  \\nATOM   3050  H   ALA A 206       1.580   5.761  -9.415  1.00 98.26           H  \\nATOM   3051  CA  ALA A 206       0.067   4.422  -8.757  1.00 98.26           C  \\nATOM   3052  HA  ALA A 206       0.186   3.455  -8.268  1.00 98.26           H  \\nATOM   3053  C   ALA A 206      -1.082   4.309  -9.771  1.00 98.26           C  \\nATOM   3054  CB  ALA A 206      -0.236   5.419  -7.652  1.00 98.26           C  \\nATOM   3055  HB1 ALA A 206      -0.281   6.427  -8.066  1.00 98.26           H  \\nATOM   3056  HB2 ALA A 206      -1.199   5.171  -7.207  1.00 98.26           H  \\nATOM   3057  HB3 ALA A 206       0.543   5.374  -6.891  1.00 98.26           H  \\nATOM   3058  O   ALA A 206      -1.307   5.206 -10.592  1.00 98.26           O  \\nATOM   3059  N   TYR A 207      -1.851   3.228  -9.660  1.00 98.58           N  \\nATOM   3060  H   TYR A 207      -1.598   2.544  -8.961  1.00 98.58           H  \\nATOM   3061  CA  TYR A 207      -2.975   2.896 -10.533  1.00 98.58           C  \\nATOM   3062  HA  TYR A 207      -3.249   3.781 -11.107  1.00 98.58           H  \\nATOM   3063  C   TYR A 207      -4.205   2.462  -9.733  1.00 98.58           C  \\nATOM   3064  CB  TYR A 207      -2.552   1.796 -11.516  1.00 98.58           C  \\nATOM   3065  HB2 TYR A 207      -3.428   1.480 -12.082  1.00 98.58           H  \\nATOM   3066  HB3 TYR A 207      -2.195   0.932 -10.955  1.00 98.58           H  \\nATOM   3067  O   TYR A 207      -4.086   1.993  -8.609  1.00 98.58           O  \\nATOM   3068  CG  TYR A 207      -1.482   2.240 -12.492  1.00 98.58           C  \\nATOM   3069  CD1 TYR A 207      -1.857   2.819 -13.717  1.00 98.58           C  \\nATOM   3070  HD1 TYR A 207      -2.902   2.892 -13.981  1.00 98.58           H  \\nATOM   3071  CD2 TYR A 207      -0.119   2.115 -12.160  1.00 98.58           C  \\nATOM   3072  HD2 TYR A 207       0.163   1.669 -11.217  1.00 98.58           H  \\nATOM   3073  CE1 TYR A 207      -0.878   3.308 -14.597  1.00 98.58           C  \\nATOM   3074  HE1 TYR A 207      -1.178   3.750 -15.536  1.00 98.58           H  \\nATOM   3075  CE2 TYR A 207       0.870   2.598 -13.039  1.00 98.58           C  \\nATOM   3076  HE2 TYR A 207       1.916   2.518 -12.782  1.00 98.58           H  \\nATOM   3077  OH  TYR A 207       1.407   3.728 -15.099  1.00 98.58           O  \\nATOM   3078  HH  TYR A 207       0.965   4.224 -15.792  1.00 98.58           H  \\nATOM   3079  CZ  TYR A 207       0.485   3.208 -14.252  1.00 98.58           C  \\nATOM   3080  N   GLN A 208      -5.388   2.577 -10.328  1.00 98.39           N  \\nATOM   3081  H   GLN A 208      -5.420   3.023 -11.233  1.00 98.39           H  \\nATOM   3082  CA  GLN A 208      -6.619   1.946  -9.847  1.00 98.39           C  \\nATOM   3083  HA  GLN A 208      -6.402   1.297  -8.998  1.00 98.39           H  \\nATOM   3084  C   GLN A 208      -7.205   1.075 -10.952  1.00 98.39           C  \\nATOM   3085  CB  GLN A 208      -7.635   3.004  -9.397  1.00 98.39           C  \\nATOM   3086  HB2 GLN A 208      -7.616   3.846 -10.089  1.00 98.39           H  \\nATOM   3087  HB3 GLN A 208      -8.635   2.570  -9.412  1.00 98.39           H  \\nATOM   3088  O   GLN A 208      -7.228   1.474 -12.119  1.00 98.39           O  \\nATOM   3089  CG  GLN A 208      -7.350   3.479  -7.968  1.00 98.39           C  \\nATOM   3090  HG2 GLN A 208      -7.460   2.632  -7.291  1.00 98.39           H  \\nATOM   3091  HG3 GLN A 208      -6.323   3.836  -7.891  1.00 98.39           H  \\nATOM   3092  CD  GLN A 208      -8.294   4.583  -7.507  1.00 98.39           C  \\nATOM   3093  NE2 GLN A 208      -8.474   4.754  -6.218  1.00 98.39           N  \\nATOM   3094 HE21 GLN A 208      -8.197   4.023  -5.578  1.00 98.39           H  \\nATOM   3095 HE22 GLN A 208      -9.164   5.442  -5.954  1.00 98.39           H  \\nATOM   3096  OE1 GLN A 208      -8.882   5.322  -8.278  1.00 98.39           O  \\nATOM   3097  N   TYR A 209      -7.679  -0.113 -10.593  1.00 98.41           N  \\nATOM   3098  H   TYR A 209      -7.651  -0.392  -9.622  1.00 98.41           H  \\nATOM   3099  CA  TYR A 209      -8.332  -1.001 -11.543  1.00 98.41           C  \\nATOM   3100  HA  TYR A 209      -7.814  -0.928 -12.498  1.00 98.41           H  \\nATOM   3101  C   TYR A 209      -9.790  -0.592 -11.765  1.00 98.41           C  \\nATOM   3102  CB  TYR A 209      -8.201  -2.445 -11.069  1.00 98.41           C  \\nATOM   3103  HB2 TYR A 209      -7.142  -2.683 -10.975  1.00 98.41           H  \\nATOM   3104  HB3 TYR A 209      -8.659  -2.523 -10.082  1.00 98.41           H  \\nATOM   3105  O   TYR A 209     -10.580  -0.473 -10.830  1.00 98.41           O  \\nATOM   3106  CG  TYR A 209      -8.834  -3.461 -11.994  1.00 98.41           C  \\nATOM   3107  CD1 TYR A 209      -9.853  -4.296 -11.505  1.00 98.41           C  \\nATOM   3108  HD1 TYR A 209     -10.164  -4.222 -10.473  1.00 98.41           H  \\nATOM   3109  CD2 TYR A 209      -8.426  -3.560 -13.339  1.00 98.41           C  \\nATOM   3110  HD2 TYR A 209      -7.625  -2.936 -13.709  1.00 98.41           H  \\nATOM   3111  CE1 TYR A 209     -10.508  -5.190 -12.364  1.00 98.41           C  \\nATOM   3112  HE1 TYR A 209     -11.307  -5.806 -11.977  1.00 98.41           H  \\nATOM   3113  CE2 TYR A 209      -9.078  -4.456 -14.207  1.00 98.41           C  \\nATOM   3114  HE2 TYR A 209      -8.786  -4.522 -15.245  1.00 98.41           H  \\nATOM   3115  OH  TYR A 209     -10.846  -6.051 -14.559  1.00 98.41           O  \\nATOM   3116  HH  TYR A 209     -11.700  -6.273 -14.181  1.00 98.41           H  \\nATOM   3117  CZ  TYR A 209     -10.152  -5.234 -13.726  1.00 98.41           C  \\nATOM   3118  N   ASP A 210     -10.135  -0.389 -13.029  1.00 97.33           N  \\nATOM   3119  H   ASP A 210      -9.427  -0.521 -13.737  1.00 97.33           H  \\nATOM   3120  CA  ASP A 210     -11.491  -0.172 -13.507  1.00 97.33           C  \\nATOM   3121  HA  ASP A 210     -12.116   0.229 -12.709  1.00 97.33           H  \\nATOM   3122  C   ASP A 210     -12.060  -1.516 -13.957  1.00 97.33           C  \\nATOM   3123  CB  ASP A 210     -11.420   0.861 -14.629  1.00 97.33           C  \\nATOM   3124  HB2 ASP A 210     -10.984   1.763 -14.198  1.00 97.33           H  \\nATOM   3125  HB3 ASP A 210     -10.751   0.503 -15.411  1.00 97.33           H  \\nATOM   3126  O   ASP A 210     -11.778  -1.996 -15.057  1.00 97.33           O  \\nATOM   3127  CG  ASP A 210     -12.752   1.238 -15.271  1.00 97.33           C  \\nATOM   3128  OD1 ASP A 210     -13.757   0.505 -15.142  1.00 97.33           O  \\nATOM   3129  OD2 ASP A 210     -12.731   2.265 -15.992  1.00 97.33           O  \\nATOM   3130  N   GLU A 211     -12.841  -2.139 -13.075  1.00 94.77           N  \\nATOM   3131  H   GLU A 211     -12.949  -1.722 -12.161  1.00 94.77           H  \\nATOM   3132  CA  GLU A 211     -13.420  -3.464 -13.296  1.00 94.77           C  \\nATOM   3133  HA  GLU A 211     -12.614  -4.168 -13.500  1.00 94.77           H  \\nATOM   3134  C   GLU A 211     -14.354  -3.497 -14.509  1.00 94.77           C  \\nATOM   3135  CB  GLU A 211     -14.140  -3.894 -12.010  1.00 94.77           C  \\nATOM   3136  HB2 GLU A 211     -14.973  -3.215 -11.827  1.00 94.77           H  \\nATOM   3137  HB3 GLU A 211     -13.437  -3.812 -11.182  1.00 94.77           H  \\nATOM   3138  O   GLU A 211     -14.328  -4.456 -15.279  1.00 94.77           O  \\nATOM   3139  CG  GLU A 211     -14.658  -5.338 -12.066  1.00 94.77           C  \\nATOM   3140  HG2 GLU A 211     -13.830  -5.991 -12.340  1.00 94.77           H  \\nATOM   3141  HG3 GLU A 211     -15.421  -5.413 -12.841  1.00 94.77           H  \\nATOM   3142  CD  GLU A 211     -15.260  -5.807 -10.730  1.00 94.77           C  \\nATOM   3143  OE1 GLU A 211     -15.359  -7.041 -10.558  1.00 94.77           O  \\nATOM   3144  OE2 GLU A 211     -15.596  -4.946  -9.884  1.00 94.77           O  \\nATOM   3145  N   LYS A 212     -15.117  -2.418 -14.723  1.00 95.05           N  \\nATOM   3146  H   LYS A 212     -14.960  -1.616 -14.129  1.00 95.05           H  \\nATOM   3147  CA  LYS A 212     -16.083  -2.306 -15.819  1.00 95.05           C  \\nATOM   3148  HA  LYS A 212     -16.728  -3.184 -15.808  1.00 95.05           H  \\nATOM   3149  C   LYS A 212     -15.389  -2.291 -17.178  1.00 95.05           C  \\nATOM   3150  CB  LYS A 212     -16.928  -1.042 -15.603  1.00 95.05           C  \\nATOM   3151  HB2 LYS A 212     -16.276  -0.169 -15.587  1.00 95.05           H  \\nATOM   3152  HB3 LYS A 212     -17.430  -1.117 -14.638  1.00 95.05           H  \\nATOM   3153  O   LYS A 212     -15.867  -2.925 -18.113  1.00 95.05           O  \\nATOM   3154  CG  LYS A 212     -17.982  -0.860 -16.704  1.00 95.05           C  \\nATOM   3155  HG2 LYS A 212     -18.629  -1.736 -16.733  1.00 95.05           H  \\nATOM   3156  HG3 LYS A 212     -17.490  -0.750 -17.671  1.00 95.05           H  \\nATOM   3157  CD  LYS A 212     -18.828   0.390 -16.453  1.00 95.05           C  \\nATOM   3158  HD2 LYS A 212     -18.174   1.259 -16.396  1.00 95.05           H  \\nATOM   3159  HD3 LYS A 212     -19.365   0.277 -15.510  1.00 95.05           H  \\nATOM   3160  CE  LYS A 212     -19.819   0.557 -17.608  1.00 95.05           C  \\nATOM   3161  HE2 LYS A 212     -20.434  -0.341 -17.670  1.00 95.05           H  \\nATOM   3162  HE3 LYS A 212     -19.254   0.635 -18.537  1.00 95.05           H  \\nATOM   3163  NZ  LYS A 212     -20.673   1.755 -17.422  1.00 95.05           N  \\nATOM   3164  HZ1 LYS A 212     -21.205   1.678 -16.566  1.00 95.05           H  \\nATOM   3165  HZ2 LYS A 212     -20.103   2.587 -17.375  1.00 95.05           H  \\nATOM   3166  HZ3 LYS A 212     -21.318   1.849 -18.194  1.00 95.05           H  \\nATOM   3167  N   ASN A 213     -14.284  -1.557 -17.288  1.00 96.35           N  \\nATOM   3168  H   ASN A 213     -13.987  -1.035 -16.476  1.00 96.35           H  \\nATOM   3169  CA  ASN A 213     -13.555  -1.398 -18.547  1.00 96.35           C  \\nATOM   3170  HA  ASN A 213     -14.204  -1.690 -19.373  1.00 96.35           H  \\nATOM   3171  C   ASN A 213     -12.318  -2.307 -18.656  1.00 96.35           C  \\nATOM   3172  CB  ASN A 213     -13.226   0.088 -18.738  1.00 96.35           C  \\nATOM   3173  HB2 ASN A 213     -12.602   0.420 -17.908  1.00 96.35           H  \\nATOM   3174  HB3 ASN A 213     -12.671   0.217 -19.667  1.00 96.35           H  \\nATOM   3175  O   ASN A 213     -11.599  -2.229 -19.649  1.00 96.35           O  \\nATOM   3176  CG  ASN A 213     -14.470   0.954 -18.818  1.00 96.35           C  \\nATOM   3177  ND2 ASN A 213     -14.582   1.962 -17.991  1.00 96.35           N  \\nATOM   3178 HD21 ASN A 213     -15.437   2.498 -18.029  1.00 96.35           H  \\nATOM   3179 HD22 ASN A 213     -13.953   2.035 -17.204  1.00 96.35           H  \\nATOM   3180  OD1 ASN A 213     -15.352   0.765 -19.636  1.00 96.35           O  \\nATOM   3181  N   ASN A 214     -12.058  -3.148 -17.648  1.00 95.45           N  \\nATOM   3182  H   ASN A 214     -12.662  -3.103 -16.841  1.00 95.45           H  \\nATOM   3183  CA  ASN A 214     -10.912  -4.056 -17.562  1.00 95.45           C  \\nATOM   3184  HA  ASN A 214     -10.846  -4.363 -16.518  1.00 95.45           H  \\nATOM   3185  C   ASN A 214      -9.566  -3.369 -17.878  1.00 95.45           C  \\nATOM   3186  CB  ASN A 214     -11.214  -5.317 -18.390  1.00 95.45           C  \\nATOM   3187  HB2 ASN A 214     -11.311  -5.029 -19.437  1.00 95.45           H  \\nATOM   3188  HB3 ASN A 214     -12.164  -5.743 -18.067  1.00 95.45           H  \\nATOM   3189  O   ASN A 214      -8.762  -3.870 -18.666  1.00 95.45           O  \\nATOM   3190  CG  ASN A 214     -10.152  -6.397 -18.265  1.00 95.45           C  \\nATOM   3191  ND2 ASN A 214      -9.725  -6.954 -19.372  1.00 95.45           N  \\nATOM   3192 HD21 ASN A 214      -8.938  -7.585 -19.311  1.00 95.45           H  \\nATOM   3193 HD22 ASN A 214     -10.027  -6.558 -20.251  1.00 95.45           H  \\nATOM   3194  OD1 ASN A 214      -9.703  -6.789 -17.193  1.00 95.45           O  \\nATOM   3195  N   LYS A 215      -9.318  -2.209 -17.258  1.00 97.03           N  \\nATOM   3196  H   LYS A 215      -9.998  -1.889 -16.583  1.00 97.03           H  \\nATOM   3197  CA  LYS A 215      -8.100  -1.405 -17.462  1.00 97.03           C  \\nATOM   3198  HA  LYS A 215      -7.318  -2.057 -17.850  1.00 97.03           H  \\nATOM   3199  C   LYS A 215      -7.573  -0.820 -16.155  1.00 97.03           C  \\nATOM   3200  CB  LYS A 215      -8.356  -0.312 -18.516  1.00 97.03           C  \\nATOM   3201  HB2 LYS A 215      -7.413   0.183 -18.746  1.00 97.03           H  \\nATOM   3202  HB3 LYS A 215      -8.721  -0.782 -19.429  1.00 97.03           H  \\nATOM   3203  O   LYS A 215      -8.303  -0.717 -15.175  1.00 97.03           O  \\nATOM   3204  CG  LYS A 215      -9.372   0.740 -18.047  1.00 97.03           C  \\nATOM   3205  HG2 LYS A 215      -9.024   1.198 -17.121  1.00 97.03           H  \\nATOM   3206  HG3 LYS A 215     -10.330   0.254 -17.861  1.00 97.03           H  \\nATOM   3207  CD  LYS A 215      -9.558   1.844 -19.088  1.00 97.03           C  \\nATOM   3208  HD2 LYS A 215      -9.914   1.414 -20.024  1.00 97.03           H  \\nATOM   3209  HD3 LYS A 215      -8.600   2.337 -19.250  1.00 97.03           H  \\nATOM   3210  CE  LYS A 215     -10.577   2.850 -18.547  1.00 97.03           C  \\nATOM   3211  HE2 LYS A 215     -11.560   2.381 -18.507  1.00 97.03           H  \\nATOM   3212  HE3 LYS A 215     -10.305   3.100 -17.521  1.00 97.03           H  \\nATOM   3213  NZ  LYS A 215     -10.611   4.081 -19.366  1.00 97.03           N  \\nATOM   3214  HZ1 LYS A 215      -9.713   4.540 -19.306  1.00 97.03           H  \\nATOM   3215  HZ2 LYS A 215     -11.312   4.708 -18.997  1.00 97.03           H  \\nATOM   3216  HZ3 LYS A 215     -10.825   3.864 -20.329  1.00 97.03           H  \\nATOM   3217  N   PHE A 216      -6.311  -0.400 -16.156  1.00 98.25           N  \\nATOM   3218  H   PHE A 216      -5.774  -0.476 -17.008  1.00 98.25           H  \\nATOM   3219  CA  PHE A 216      -5.684   0.276 -15.022  1.00 98.25           C  \\nATOM   3220  HA  PHE A 216      -6.298   0.150 -14.130  1.00 98.25           H  \\nATOM   3221  C   PHE A 216      -5.582   1.775 -15.305  1.00 98.25           C  \\nATOM   3222  CB  PHE A 216      -4.327  -0.374 -14.727  1.00 98.25           C  \\nATOM   3223  HB2 PHE A 216      -3.787   0.241 -14.007  1.00 98.25           H  \\nATOM   3224  HB3 PHE A 216      -3.735  -0.404 -15.642  1.00 98.25           H  \\nATOM   3225  O   PHE A 216      -4.862   2.203 -16.204  1.00 98.25           O  \\nATOM   3226  CG  PHE A 216      -4.461  -1.777 -14.166  1.00 98.25           C  \\nATOM   3227  CD1 PHE A 216      -4.560  -1.977 -12.776  1.00 98.25           C  \\nATOM   3228  HD1 PHE A 216      -4.513  -1.136 -12.100  1.00 98.25           H  \\nATOM   3229  CD2 PHE A 216      -4.559  -2.879 -15.036  1.00 98.25           C  \\nATOM   3230  HD2 PHE A 216      -4.510  -2.729 -16.104  1.00 98.25           H  \\nATOM   3231  CE1 PHE A 216      -4.753  -3.272 -12.265  1.00 98.25           C  \\nATOM   3232  HE1 PHE A 216      -4.865  -3.424 -11.201  1.00 98.25           H  \\nATOM   3233  CE2 PHE A 216      -4.751  -4.173 -14.521  1.00 98.25           C  \\nATOM   3234  HE2 PHE A 216      -4.835  -5.019 -15.188  1.00 98.25           H  \\nATOM   3235  CZ  PHE A 216      -4.843  -4.370 -13.134  1.00 98.25           C  \\nATOM   3236  HZ  PHE A 216      -5.000  -5.361 -12.734  1.00 98.25           H  \\nATOM   3237  N   ASN A 217      -6.310   2.582 -14.538  1.00 98.01           N  \\nATOM   3238  H   ASN A 217      -6.836   2.176 -13.777  1.00 98.01           H  \\nATOM   3239  CA  ASN A 217      -6.286   4.035 -14.652  1.00 98.01           C  \\nATOM   3240  HA  ASN A 217      -6.114   4.311 -15.693  1.00 98.01           H  \\nATOM   3241  C   ASN A 217      -5.142   4.599 -13.812  1.00 98.01           C  \\nATOM   3242  CB  ASN A 217      -7.641   4.608 -14.216  1.00 98.01           C  \\nATOM   3243  HB2 ASN A 217      -7.862   4.259 -13.207  1.00 98.01           H  \\nATOM   3244  HB3 ASN A 217      -7.585   5.696 -14.189  1.00 98.01           H  \\nATOM   3245  O   ASN A 217      -4.996   4.248 -12.642  1.00 98.01           O  \\nATOM   3246  CG  ASN A 217      -8.781   4.228 -15.146  1.00 98.01           C  \\nATOM   3247  ND2 ASN A 217      -9.952   4.010 -14.598  1.00 98.01           N  \\nATOM   3248 HD21 ASN A 217     -10.048   4.032 -13.593  1.00 98.01           H  \\nATOM   3249 HD22 ASN A 217     -10.707   3.673 -15.177  1.00 98.01           H  \\nATOM   3250  OD1 ASN A 217      -8.651   4.154 -16.361  1.00 98.01           O  \\nATOM   3251  N   TYR A 218      -4.340   5.485 -14.399  1.00 98.06           N  \\nATOM   3252  H   TYR A 218      -4.514   5.747 -15.359  1.00 98.06           H  \\nATOM   3253  CA  TYR A 218      -3.300   6.195 -13.664  1.00 98.06           C  \\nATOM   3254  HA  TYR A 218      -2.686   5.466 -13.137  1.00 98.06           H  \\nATOM   3255  C   TYR A 218      -3.920   7.142 -12.631  1.00 98.06           C  \\nATOM   3256  CB  TYR A 218      -2.415   6.969 -14.647  1.00 98.06           C  \\nATOM   3257  HB2 TYR A 218      -3.018   7.712 -15.168  1.00 98.06           H  \\nATOM   3258  HB3 TYR A 218      -2.018   6.278 -15.390  1.00 98.06           H  \\nATOM   3259  O   TYR A 218      -4.766   7.964 -12.978  1.00 98.06           O  \\nATOM   3260  CG  TYR A 218      -1.251   7.650 -13.964  1.00 98.06           C  \\nATOM   3261  CD1 TYR A 218      -1.152   9.056 -13.892  1.00 98.06           C  \\nATOM   3262  HD1 TYR A 218      -1.916   9.679 -14.335  1.00 98.06           H  \\nATOM   3263  CD2 TYR A 218      -0.281   6.846 -13.345  1.00 98.06           C  \\nATOM   3264  HD2 TYR A 218      -0.377   5.770 -13.345  1.00 98.06           H  \\nATOM   3265  CE1 TYR A 218      -0.056   9.657 -13.233  1.00 98.06           C  \\nATOM   3266  HE1 TYR A 218       0.032  10.731 -13.165  1.00 98.06           H  \\nATOM   3267  CE2 TYR A 218       0.798   7.442 -12.685  1.00 98.06           C  \\nATOM   3268  HE2 TYR A 218       1.519   6.816 -12.182  1.00 98.06           H  \\nATOM   3269  OH  TYR A 218       2.022   9.341 -12.009  1.00 98.06           O  \\nATOM   3270  HH  TYR A 218       2.074  10.298 -11.966  1.00 98.06           H  \\nATOM   3271  CZ  TYR A 218       0.934   8.841 -12.645  1.00 98.06           C  \\nATOM   3272  N   VAL A 219      -3.491   7.040 -11.372  1.00 97.74           N  \\nATOM   3273  H   VAL A 219      -2.820   6.316 -11.156  1.00 97.74           H  \\nATOM   3274  CA  VAL A 219      -3.950   7.913 -10.269  1.00 97.74           C  \\nATOM   3275  HA  VAL A 219      -4.498   8.750 -10.699  1.00 97.74           H  \\nATOM   3276  C   VAL A 219      -2.804   8.541  -9.480  1.00 97.74           C  \\nATOM   3277  CB  VAL A 219      -4.909   7.178  -9.317  1.00 97.74           C  \\nATOM   3278  HB  VAL A 219      -5.259   7.888  -8.568  1.00 97.74           H  \\nATOM   3279  O   VAL A 219      -3.048   9.206  -8.463  1.00 97.74           O  \\nATOM   3280  CG1 VAL A 219      -6.144   6.674 -10.064  1.00 97.74           C  \\nATOM   3281 HG11 VAL A 219      -6.589   7.483 -10.643  1.00 97.74           H  \\nATOM   3282 HG12 VAL A 219      -5.882   5.856 -10.735  1.00 97.74           H  \\nATOM   3283 HG13 VAL A 219      -6.878   6.316  -9.343  1.00 97.74           H  \\nATOM   3284  CG2 VAL A 219      -4.236   6.013  -8.583  1.00 97.74           C  \\nATOM   3285 HG21 VAL A 219      -3.940   5.243  -9.296  1.00 97.74           H  \\nATOM   3286 HG22 VAL A 219      -3.358   6.355  -8.035  1.00 97.74           H  \\nATOM   3287 HG23 VAL A 219      -4.937   5.586  -7.866  1.00 97.74           H  \\nATOM   3288  N   GLY A 220      -1.577   8.289  -9.937  1.00 96.13           N  \\nATOM   3289  H   GLY A 220      -1.493   7.734 -10.777  1.00 96.13           H  \\nATOM   3290  CA  GLY A 220      -0.348   8.825  -9.385  1.00 96.13           C  \\nATOM   3291  HA2 GLY A 220      -0.306   8.635  -8.313  1.00 96.13           H  \\nATOM   3292  HA3 GLY A 220       0.488   8.314  -9.862  1.00 96.13           H  \\nATOM   3293  C   GLY A 220      -0.175  10.325  -9.600  1.00 96.13           C  \\nATOM   3294  O   GLY A 220      -1.114  11.042  -9.949  1.00 96.13           O  \\nATOM   3295  N   LYS A 221       1.046  10.784  -9.341  1.00 93.80           N  \\nATOM   3296  H   LYS A 221       1.751  10.098  -9.111  1.00 93.80           H  \\nATOM   3297  CA  LYS A 221       1.462  12.185  -9.469  1.00 93.80           C  \\nATOM   3298  HA  LYS A 221       0.636  12.840  -9.192  1.00 93.80           H  \\nATOM   3299  C   LYS A 221       1.824  12.601 -10.888  1.00 93.80           C  \\nATOM   3300  CB  LYS A 221       2.641  12.435  -8.544  1.00 93.80           C  \\nATOM   3301  HB2 LYS A 221       3.377  11.643  -8.685  1.00 93.80           H  \\nATOM   3302  HB3 LYS A 221       3.111  13.374  -8.834  1.00 93.80           H  \\nATOM   3303  O   LYS A 221       2.440  11.781 -11.603  1.00 93.80           O  \\nATOM   3304  CG  LYS A 221       2.208  12.507  -7.077  1.00 93.80           C  \\nATOM   3305  HG2 LYS A 221       1.512  13.333  -6.931  1.00 93.80           H  \\nATOM   3306  HG3 LYS A 221       1.752  11.570  -6.759  1.00 93.80           H  \\nATOM   3307  CD  LYS A 221       3.492  12.761  -6.306  1.00 93.80           C  \\nATOM   3308  HD2 LYS A 221       4.176  11.928  -6.465  1.00 93.80           H  \\nATOM   3309  HD3 LYS A 221       3.965  13.658  -6.706  1.00 93.80           H  \\nATOM   3310  CE  LYS A 221       3.292  12.952  -4.819  1.00 93.80           C  \\nATOM   3311  HE2 LYS A 221       2.573  13.754  -4.648  1.00 93.80           H  \\nATOM   3312  HE3 LYS A 221       2.949  12.023  -4.364  1.00 93.80           H  \\nATOM   3313  NZ  LYS A 221       4.616  13.323  -4.301  1.00 93.80           N  \\nATOM   3314  HZ1 LYS A 221       5.270  12.565  -4.432  1.00 93.80           H  \\nATOM   3315  HZ2 LYS A 221       4.585  13.517  -3.310  1.00 93.80           H  \\nATOM   3316  HZ3 LYS A 221       4.965  14.143  -4.776  1.00 93.80           H  \\nATOM   3317  OXT LYS A 221       1.511  13.763 -11.203  1.00 93.80           O  \\nTER    3318      LYS A 221                                                       \\nEND   \\n\");\n",
              "\tviewer_16704658566527212.setStyle({\"model\": -1},{\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {}});\n",
              "\tviewer_16704658566527212.zoom(0.12);\n",
              "\tviewer_16704658566527212.rotate(235,{\"x\": 0, \"y\": 1, \"z\": 1});\n",
              "\tviewer_16704658566527212.spin({\"x\": -0.2, \"y\": 0.5, \"z\": 1},1);\n",
              "viewer_16704658566527212.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        }
      ],
      "source": [
        "!pip install py3Dmol -q\n",
        "import py3Dmol \n",
        "with open(\"../input/novozymes-enzyme-stability-prediction/wildtype_structure_prediction_af2.pdb\") as ifile:\n",
        "    protein = \"\".join([x for x in ifile])\n",
        "#view = py3Dmol.view(query='pdb:1DIV', width=800, height=600)\n",
        "view = py3Dmol.view(width=800, height=600) \n",
        "view.addModelsAsFrames(protein)\n",
        "style = {'cartoon': {'color': 'spectrum'},'stick':{}}\n",
        "view.setStyle({'model': -1},style) \n",
        "view.zoom(0.12)\n",
        "view.rotate(235, {'x':0,'y':1,'z':1})\n",
        "view.spin({'x':-0.2,'y':0.5,'z':1},1)\n",
        "view.show()"
      ]
    },
    {
      "cell_type": "markdown",
      "id": "df1a845d",
      "metadata": {
        "papermill": {
          "duration": 0.016382,
          "end_time": "2022-12-08T02:17:36.707812",
          "exception": false,
          "start_time": "2022-12-08T02:17:36.691430",
          "status": "completed"
        },
        "tags": [],
        "id": "df1a845d"
      },
      "source": [
        "## Amino Acid Features\n",
        "For amino acid features, we use the CSV provided by Moth @alejopaullier [here][1]. Lucas Morin shows their usefulness [here][2]\n",
        "\n",
        "[1]: https://www.kaggle.com/datasets/alejopaullier/aminoacids-physical-and-chemical-properties\n",
        "[2]: https://www.kaggle.com/code/lucasmorin/nesp-changes-eda-and-baseline"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "fba30a9d",
      "metadata": {
        "_kg_hide-output": false,
        "execution": {
          "iopub.execute_input": "2022-12-08T02:17:36.742023Z",
          "iopub.status.busy": "2022-12-08T02:17:36.741706Z",
          "iopub.status.idle": "2022-12-08T02:17:47.556250Z",
          "shell.execute_reply": "2022-12-08T02:17:47.554067Z"
        },
        "papermill": {
          "duration": 10.83517,
          "end_time": "2022-12-08T02:17:47.559298",
          "exception": false,
          "start_time": "2022-12-08T02:17:36.724128",
          "status": "completed"
        },
        "tags": [],
        "id": "fba30a9d",
        "outputId": "cb0d9511-5dfc-4207-ac66-9f4e5c08f032"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\r\n",
            "\u001b[0mAmino Acid properties dataframe. Shape: (22, 21)\n"
          ]
        },
        {
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>Name</th>\n",
              "      <th>Abbr</th>\n",
              "      <th>Molecular Weight</th>\n",
              "      <th>Molecular Formula</th>\n",
              "      <th>Residue Formula</th>\n",
              "      <th>Residue Weight</th>\n",
              "      <th>pKa1</th>\n",
              "      <th>pKb2</th>\n",
              "      <th>pKx3</th>\n",
              "      <th>pl4</th>\n",
              "      <th>H</th>\n",
              "      <th>VSC</th>\n",
              "      <th>P1</th>\n",
              "      <th>P2</th>\n",
              "      <th>SASA</th>\n",
              "      <th>NCISC</th>\n",
              "      <th>carbon</th>\n",
              "      <th>hydrogen</th>\n",
              "      <th>nitrogen</th>\n",
              "      <th>oxygen</th>\n",
              "      <th>sulfur</th>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>Letter</th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "      <th></th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>A</th>\n",
              "      <td>Alanine</td>\n",
              "      <td>Ala</td>\n",
              "      <td>89.10</td>\n",
              "      <td>C3H7NO2</td>\n",
              "      <td>C3H5NO</td>\n",
              "      <td>71.08</td>\n",
              "      <td>2.34</td>\n",
              "      <td>9.69</td>\n",
              "      <td>NaN</td>\n",
              "      <td>6.00</td>\n",
              "      <td>0.62</td>\n",
              "      <td>27.5</td>\n",
              "      <td>8.1</td>\n",
              "      <td>0.046</td>\n",
              "      <td>1.181</td>\n",
              "      <td>0.007187</td>\n",
              "      <td>3</td>\n",
              "      <td>7</td>\n",
              "      <td>1</td>\n",
              "      <td>2</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>C</th>\n",
              "      <td>Cysteine</td>\n",
              "      <td>Cys</td>\n",
              "      <td>121.16</td>\n",
              "      <td>C3H7NO2S</td>\n",
              "      <td>C3H5NOS</td>\n",
              "      <td>103.15</td>\n",
              "      <td>1.96</td>\n",
              "      <td>10.28</td>\n",
              "      <td>8.18</td>\n",
              "      <td>5.07</td>\n",
              "      <td>0.29</td>\n",
              "      <td>44.6</td>\n",
              "      <td>5.5</td>\n",
              "      <td>0.128</td>\n",
              "      <td>1.461</td>\n",
              "      <td>-0.036610</td>\n",
              "      <td>3</td>\n",
              "      <td>7</td>\n",
              "      <td>1</td>\n",
              "      <td>2</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>D</th>\n",
              "      <td>Aspartic acid</td>\n",
              "      <td>Asp</td>\n",
              "      <td>133.11</td>\n",
              "      <td>C4H7NO4</td>\n",
              "      <td>C4H5NO3</td>\n",
              "      <td>115.09</td>\n",
              "      <td>1.88</td>\n",
              "      <td>9.60</td>\n",
              "      <td>3.65</td>\n",
              "      <td>2.77</td>\n",
              "      <td>-0.90</td>\n",
              "      <td>40.0</td>\n",
              "      <td>13.0</td>\n",
              "      <td>0.105</td>\n",
              "      <td>1.587</td>\n",
              "      <td>-0.023820</td>\n",
              "      <td>4</td>\n",
              "      <td>7</td>\n",
              "      <td>1</td>\n",
              "      <td>4</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>E</th>\n",
              "      <td>Glutamic acid</td>\n",
              "      <td>Glu</td>\n",
              "      <td>147.13</td>\n",
              "      <td>C5H9NO4</td>\n",
              "      <td>C5H7NO3</td>\n",
              "      <td>129.12</td>\n",
              "      <td>2.19</td>\n",
              "      <td>9.67</td>\n",
              "      <td>4.25</td>\n",
              "      <td>3.22</td>\n",
              "      <td>-0.74</td>\n",
              "      <td>62.0</td>\n",
              "      <td>12.3</td>\n",
              "      <td>0.151</td>\n",
              "      <td>1.862</td>\n",
              "      <td>0.006802</td>\n",
              "      <td>5</td>\n",
              "      <td>9</td>\n",
              "      <td>1</td>\n",
              "      <td>4</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>F</th>\n",
              "      <td>Phenylalanine</td>\n",
              "      <td>Phe</td>\n",
              "      <td>165.19</td>\n",
              "      <td>C9H11NO2</td>\n",
              "      <td>C9H9NO</td>\n",
              "      <td>147.18</td>\n",
              "      <td>1.83</td>\n",
              "      <td>9.13</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.48</td>\n",
              "      <td>1.19</td>\n",
              "      <td>115.5</td>\n",
              "      <td>5.2</td>\n",
              "      <td>0.290</td>\n",
              "      <td>2.228</td>\n",
              "      <td>0.037552</td>\n",
              "      <td>9</td>\n",
              "      <td>11</td>\n",
              "      <td>1</td>\n",
              "      <td>2</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>G</th>\n",
              "      <td>Glycine</td>\n",
              "      <td>Gly</td>\n",
              "      <td>75.07</td>\n",
              "      <td>C2H5NO2</td>\n",
              "      <td>C2H3NO</td>\n",
              "      <td>57.05</td>\n",
              "      <td>2.34</td>\n",
              "      <td>9.60</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.97</td>\n",
              "      <td>0.48</td>\n",
              "      <td>0.0</td>\n",
              "      <td>9.0</td>\n",
              "      <td>0.000</td>\n",
              "      <td>0.881</td>\n",
              "      <td>0.179052</td>\n",
              "      <td>2</td>\n",
              "      <td>5</td>\n",
              "      <td>1</td>\n",
              "      <td>2</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>H</th>\n",
              "      <td>Histidine</td>\n",
              "      <td>His</td>\n",
              "      <td>155.16</td>\n",
              "      <td>C6H9N3O2</td>\n",
              "      <td>C6H7N3O</td>\n",
              "      <td>137.14</td>\n",
              "      <td>1.82</td>\n",
              "      <td>9.17</td>\n",
              "      <td>6.00</td>\n",
              "      <td>7.59</td>\n",
              "      <td>-0.40</td>\n",
              "      <td>79.0</td>\n",
              "      <td>10.4</td>\n",
              "      <td>0.230</td>\n",
              "      <td>2.025</td>\n",
              "      <td>-0.010690</td>\n",
              "      <td>6</td>\n",
              "      <td>9</td>\n",
              "      <td>3</td>\n",
              "      <td>2</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>I</th>\n",
              "      <td>Isoleucine</td>\n",
              "      <td>Ile</td>\n",
              "      <td>131.18</td>\n",
              "      <td>C6H13NO2</td>\n",
              "      <td>C6H11NO</td>\n",
              "      <td>113.16</td>\n",
              "      <td>2.36</td>\n",
              "      <td>9.60</td>\n",
              "      <td>NaN</td>\n",
              "      <td>6.02</td>\n",
              "      <td>1.38</td>\n",
              "      <td>93.5</td>\n",
              "      <td>5.2</td>\n",
              "      <td>0.186</td>\n",
              "      <td>1.810</td>\n",
              "      <td>0.021631</td>\n",
              "      <td>6</td>\n",
              "      <td>13</td>\n",
              "      <td>1</td>\n",
              "      <td>2</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>K</th>\n",
              "      <td>Lysine</td>\n",
              "      <td>Lys</td>\n",
              "      <td>146.19</td>\n",
              "      <td>C6H14N2O2</td>\n",
              "      <td>C6H12N2O</td>\n",
              "      <td>128.18</td>\n",
              "      <td>2.18</td>\n",
              "      <td>8.95</td>\n",
              "      <td>10.53</td>\n",
              "      <td>9.74</td>\n",
              "      <td>-1.50</td>\n",
              "      <td>100.0</td>\n",
              "      <td>11.3</td>\n",
              "      <td>0.219</td>\n",
              "      <td>2.258</td>\n",
              "      <td>0.017708</td>\n",
              "      <td>6</td>\n",
              "      <td>14</td>\n",
              "      <td>2</td>\n",
              "      <td>2</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>L</th>\n",
              "      <td>Leucine</td>\n",
              "      <td>Leu</td>\n",
              "      <td>131.18</td>\n",
              "      <td>C6H13NO2</td>\n",
              "      <td>C6H11NO</td>\n",
              "      <td>113.16</td>\n",
              "      <td>2.36</td>\n",
              "      <td>9.60</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.98</td>\n",
              "      <td>1.06</td>\n",
              "      <td>93.5</td>\n",
              "      <td>4.9</td>\n",
              "      <td>0.186</td>\n",
              "      <td>1.931</td>\n",
              "      <td>0.051672</td>\n",
              "      <td>6</td>\n",
              "      <td>13</td>\n",
              "      <td>1</td>\n",
              "      <td>2</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>M</th>\n",
              "      <td>Methionine</td>\n",
              "      <td>Met</td>\n",
              "      <td>149.21</td>\n",
              "      <td>C5H11NO2S</td>\n",
              "      <td>C5H9NOS</td>\n",
              "      <td>131.20</td>\n",
              "      <td>2.28</td>\n",
              "      <td>9.21</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.74</td>\n",
              "      <td>0.64</td>\n",
              "      <td>94.1</td>\n",
              "      <td>5.7</td>\n",
              "      <td>0.221</td>\n",
              "      <td>2.034</td>\n",
              "      <td>0.002683</td>\n",
              "      <td>5</td>\n",
              "      <td>11</td>\n",
              "      <td>1</td>\n",
              "      <td>2</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>N</th>\n",
              "      <td>Asparagine</td>\n",
              "      <td>Asn</td>\n",
              "      <td>132.12</td>\n",
              "      <td>C4H8N2O3</td>\n",
              "      <td>C4H6N2O2</td>\n",
              "      <td>114.11</td>\n",
              "      <td>2.02</td>\n",
              "      <td>8.80</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.41</td>\n",
              "      <td>-0.78</td>\n",
              "      <td>58.7</td>\n",
              "      <td>11.6</td>\n",
              "      <td>0.134</td>\n",
              "      <td>1.655</td>\n",
              "      <td>0.005392</td>\n",
              "      <td>4</td>\n",
              "      <td>8</td>\n",
              "      <td>2</td>\n",
              "      <td>3</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>O</th>\n",
              "      <td>Hydroxyproline</td>\n",
              "      <td>Hyp</td>\n",
              "      <td>131.13</td>\n",
              "      <td>C5H9NO3</td>\n",
              "      <td>C5H7NO2</td>\n",
              "      <td>113.11</td>\n",
              "      <td>1.82</td>\n",
              "      <td>9.65</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5</td>\n",
              "      <td>9</td>\n",
              "      <td>1</td>\n",
              "      <td>3</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>P</th>\n",
              "      <td>Proline</td>\n",
              "      <td>Pro</td>\n",
              "      <td>115.13</td>\n",
              "      <td>C5H9NO2</td>\n",
              "      <td>C5H7NO</td>\n",
              "      <td>97.12</td>\n",
              "      <td>1.99</td>\n",
              "      <td>10.60</td>\n",
              "      <td>NaN</td>\n",
              "      <td>6.30</td>\n",
              "      <td>0.12</td>\n",
              "      <td>41.9</td>\n",
              "      <td>8.0</td>\n",
              "      <td>0.131</td>\n",
              "      <td>1.468</td>\n",
              "      <td>0.239531</td>\n",
              "      <td>5</td>\n",
              "      <td>9</td>\n",
              "      <td>1</td>\n",
              "      <td>2</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>Q</th>\n",
              "      <td>Glutamine</td>\n",
              "      <td>Gln</td>\n",
              "      <td>146.15</td>\n",
              "      <td>C5H10N2O3</td>\n",
              "      <td>C5H8N2O2</td>\n",
              "      <td>128.13</td>\n",
              "      <td>2.17</td>\n",
              "      <td>9.13</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.65</td>\n",
              "      <td>-0.85</td>\n",
              "      <td>80.7</td>\n",
              "      <td>10.5</td>\n",
              "      <td>0.180</td>\n",
              "      <td>1.932</td>\n",
              "      <td>0.049211</td>\n",
              "      <td>5</td>\n",
              "      <td>10</td>\n",
              "      <td>2</td>\n",
              "      <td>3</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>R</th>\n",
              "      <td>Arginine</td>\n",
              "      <td>Arg</td>\n",
              "      <td>174.20</td>\n",
              "      <td>C6H14N4O2</td>\n",
              "      <td>C6H12N4O</td>\n",
              "      <td>156.19</td>\n",
              "      <td>2.17</td>\n",
              "      <td>9.04</td>\n",
              "      <td>12.48</td>\n",
              "      <td>10.76</td>\n",
              "      <td>-2.53</td>\n",
              "      <td>105.0</td>\n",
              "      <td>10.5</td>\n",
              "      <td>0.291</td>\n",
              "      <td>2.560</td>\n",
              "      <td>0.043587</td>\n",
              "      <td>6</td>\n",
              "      <td>14</td>\n",
              "      <td>4</td>\n",
              "      <td>2</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>S</th>\n",
              "      <td>Serine</td>\n",
              "      <td>Ser</td>\n",
              "      <td>105.09</td>\n",
              "      <td>C3H7NO3</td>\n",
              "      <td>C3H5NO2</td>\n",
              "      <td>87.08</td>\n",
              "      <td>2.21</td>\n",
              "      <td>9.15</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.68</td>\n",
              "      <td>-0.18</td>\n",
              "      <td>29.3</td>\n",
              "      <td>9.2</td>\n",
              "      <td>0.062</td>\n",
              "      <td>1.298</td>\n",
              "      <td>0.004627</td>\n",
              "      <td>3</td>\n",
              "      <td>7</td>\n",
              "      <td>1</td>\n",
              "      <td>3</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>T</th>\n",
              "      <td>Threonine</td>\n",
              "      <td>Thr</td>\n",
              "      <td>119.12</td>\n",
              "      <td>C4H9NO3</td>\n",
              "      <td>C4H7NO2</td>\n",
              "      <td>101.11</td>\n",
              "      <td>2.09</td>\n",
              "      <td>9.10</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.60</td>\n",
              "      <td>-0.05</td>\n",
              "      <td>51.3</td>\n",
              "      <td>8.6</td>\n",
              "      <td>0.108</td>\n",
              "      <td>1.525</td>\n",
              "      <td>0.003352</td>\n",
              "      <td>4</td>\n",
              "      <td>9</td>\n",
              "      <td>1</td>\n",
              "      <td>3</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>U</th>\n",
              "      <td>Pyroglutamatic</td>\n",
              "      <td>Glp</td>\n",
              "      <td>139.11</td>\n",
              "      <td>C5H7NO3</td>\n",
              "      <td>C5H5NO2</td>\n",
              "      <td>121.09</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.68</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5</td>\n",
              "      <td>7</td>\n",
              "      <td>1</td>\n",
              "      <td>3</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>V</th>\n",
              "      <td>Valine</td>\n",
              "      <td>Val</td>\n",
              "      <td>117.15</td>\n",
              "      <td>C5H11NO2</td>\n",
              "      <td>C5H9NO</td>\n",
              "      <td>99.13</td>\n",
              "      <td>2.32</td>\n",
              "      <td>9.62</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.96</td>\n",
              "      <td>1.08</td>\n",
              "      <td>71.5</td>\n",
              "      <td>5.9</td>\n",
              "      <td>0.140</td>\n",
              "      <td>1.645</td>\n",
              "      <td>0.057004</td>\n",
              "      <td>5</td>\n",
              "      <td>11</td>\n",
              "      <td>1</td>\n",
              "      <td>2</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>W</th>\n",
              "      <td>Tryptophan</td>\n",
              "      <td>Trp</td>\n",
              "      <td>204.23</td>\n",
              "      <td>C11H12N2O2</td>\n",
              "      <td>C11H10N2O</td>\n",
              "      <td>186.22</td>\n",
              "      <td>2.83</td>\n",
              "      <td>9.39</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.89</td>\n",
              "      <td>0.81</td>\n",
              "      <td>145.5</td>\n",
              "      <td>5.4</td>\n",
              "      <td>0.409</td>\n",
              "      <td>2.663</td>\n",
              "      <td>0.037977</td>\n",
              "      <td>11</td>\n",
              "      <td>12</td>\n",
              "      <td>2</td>\n",
              "      <td>2</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>Y</th>\n",
              "      <td>Tyrosine</td>\n",
              "      <td>Tyr</td>\n",
              "      <td>181.19</td>\n",
              "      <td>C9H11NO3</td>\n",
              "      <td>C9H9NO2</td>\n",
              "      <td>163.18</td>\n",
              "      <td>2.20</td>\n",
              "      <td>9.11</td>\n",
              "      <td>10.07</td>\n",
              "      <td>5.66</td>\n",
              "      <td>0.26</td>\n",
              "      <td>117.3</td>\n",
              "      <td>6.2</td>\n",
              "      <td>0.298</td>\n",
              "      <td>2.368</td>\n",
              "      <td>0.023599</td>\n",
              "      <td>9</td>\n",
              "      <td>11</td>\n",
              "      <td>1</td>\n",
              "      <td>3</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "                  Name Abbr  Molecular Weight Molecular Formula  \\\n",
              "Letter                                                            \n",
              "A              Alanine  Ala             89.10           C3H7NO2   \n",
              "C             Cysteine  Cys            121.16          C3H7NO2S   \n",
              "D        Aspartic acid  Asp            133.11           C4H7NO4   \n",
              "E        Glutamic acid  Glu            147.13           C5H9NO4   \n",
              "F        Phenylalanine  Phe            165.19          C9H11NO2   \n",
              "G              Glycine  Gly             75.07           C2H5NO2   \n",
              "H            Histidine  His            155.16          C6H9N3O2   \n",
              "I           Isoleucine  Ile            131.18          C6H13NO2   \n",
              "K               Lysine  Lys            146.19         C6H14N2O2   \n",
              "L              Leucine  Leu            131.18          C6H13NO2   \n",
              "M           Methionine  Met            149.21         C5H11NO2S   \n",
              "N           Asparagine  Asn            132.12          C4H8N2O3   \n",
              "O       Hydroxyproline  Hyp            131.13           C5H9NO3   \n",
              "P              Proline  Pro            115.13           C5H9NO2   \n",
              "Q            Glutamine  Gln            146.15         C5H10N2O3   \n",
              "R             Arginine  Arg            174.20         C6H14N4O2   \n",
              "S               Serine  Ser            105.09           C3H7NO3   \n",
              "T            Threonine  Thr            119.12           C4H9NO3   \n",
              "U       Pyroglutamatic  Glp            139.11           C5H7NO3   \n",
              "V               Valine  Val            117.15          C5H11NO2   \n",
              "W           Tryptophan  Trp            204.23        C11H12N2O2   \n",
              "Y             Tyrosine  Tyr            181.19          C9H11NO3   \n",
              "\n",
              "       Residue Formula  Residue Weight  pKa1   pKb2   pKx3    pl4     H  \\\n",
              "Letter                                                                    \n",
              "A               C3H5NO           71.08  2.34   9.69    NaN   6.00  0.62   \n",
              "C              C3H5NOS          103.15  1.96  10.28   8.18   5.07  0.29   \n",
              "D              C4H5NO3          115.09  1.88   9.60   3.65   2.77 -0.90   \n",
              "E              C5H7NO3          129.12  2.19   9.67   4.25   3.22 -0.74   \n",
              "F               C9H9NO          147.18  1.83   9.13    NaN   5.48  1.19   \n",
              "G               C2H3NO           57.05  2.34   9.60    NaN   5.97  0.48   \n",
              "H              C6H7N3O          137.14  1.82   9.17   6.00   7.59 -0.40   \n",
              "I              C6H11NO          113.16  2.36   9.60    NaN   6.02  1.38   \n",
              "K             C6H12N2O          128.18  2.18   8.95  10.53   9.74 -1.50   \n",
              "L              C6H11NO          113.16  2.36   9.60    NaN   5.98  1.06   \n",
              "M              C5H9NOS          131.20  2.28   9.21    NaN   5.74  0.64   \n",
              "N             C4H6N2O2          114.11  2.02   8.80    NaN   5.41 -0.78   \n",
              "O              C5H7NO2          113.11  1.82   9.65    NaN    NaN   NaN   \n",
              "P               C5H7NO           97.12  1.99  10.60    NaN   6.30  0.12   \n",
              "Q             C5H8N2O2          128.13  2.17   9.13    NaN   5.65 -0.85   \n",
              "R             C6H12N4O          156.19  2.17   9.04  12.48  10.76 -2.53   \n",
              "S              C3H5NO2           87.08  2.21   9.15    NaN   5.68 -0.18   \n",
              "T              C4H7NO2          101.11  2.09   9.10    NaN   5.60 -0.05   \n",
              "U              C5H5NO2          121.09   NaN    NaN    NaN   5.68   NaN   \n",
              "V               C5H9NO           99.13  2.32   9.62    NaN   5.96  1.08   \n",
              "W            C11H10N2O          186.22  2.83   9.39    NaN   5.89  0.81   \n",
              "Y              C9H9NO2          163.18  2.20   9.11  10.07   5.66  0.26   \n",
              "\n",
              "          VSC    P1     P2   SASA     NCISC  carbon  hydrogen  nitrogen  \\\n",
              "Letter                                                                    \n",
              "A        27.5   8.1  0.046  1.181  0.007187       3         7         1   \n",
              "C        44.6   5.5  0.128  1.461 -0.036610       3         7         1   \n",
              "D        40.0  13.0  0.105  1.587 -0.023820       4         7         1   \n",
              "E        62.0  12.3  0.151  1.862  0.006802       5         9         1   \n",
              "F       115.5   5.2  0.290  2.228  0.037552       9        11         1   \n",
              "G         0.0   9.0  0.000  0.881  0.179052       2         5         1   \n",
              "H        79.0  10.4  0.230  2.025 -0.010690       6         9         3   \n",
              "I        93.5   5.2  0.186  1.810  0.021631       6        13         1   \n",
              "K       100.0  11.3  0.219  2.258  0.017708       6        14         2   \n",
              "L        93.5   4.9  0.186  1.931  0.051672       6        13         1   \n",
              "M        94.1   5.7  0.221  2.034  0.002683       5        11         1   \n",
              "N        58.7  11.6  0.134  1.655  0.005392       4         8         2   \n",
              "O         NaN   NaN    NaN    NaN       NaN       5         9         1   \n",
              "P        41.9   8.0  0.131  1.468  0.239531       5         9         1   \n",
              "Q        80.7  10.5  0.180  1.932  0.049211       5        10         2   \n",
              "R       105.0  10.5  0.291  2.560  0.043587       6        14         4   \n",
              "S        29.3   9.2  0.062  1.298  0.004627       3         7         1   \n",
              "T        51.3   8.6  0.108  1.525  0.003352       4         9         1   \n",
              "U         NaN   NaN    NaN    NaN       NaN       5         7         1   \n",
              "V        71.5   5.9  0.140  1.645  0.057004       5        11         1   \n",
              "W       145.5   5.4  0.409  2.663  0.037977      11        12         2   \n",
              "Y       117.3   6.2  0.298  2.368  0.023599       9        11         1   \n",
              "\n",
              "        oxygen  sulfur  \n",
              "Letter                  \n",
              "A            2       0  \n",
              "C            2       1  \n",
              "D            4       0  \n",
              "E            4       0  \n",
              "F            2       0  \n",
              "G            2       0  \n",
              "H            2       0  \n",
              "I            2       0  \n",
              "K            2       0  \n",
              "L            2       0  \n",
              "M            2       1  \n",
              "N            3       0  \n",
              "O            3       0  \n",
              "P            2       0  \n",
              "Q            3       0  \n",
              "R            2       0  \n",
              "S            3       0  \n",
              "T            3       0  \n",
              "U            3       0  \n",
              "V            2       0  \n",
              "W            2       0  \n",
              "Y            3       0  "
            ]
          },
          "execution_count": 10,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# BIOPANDAS PDB READER\n",
        "!pip install biopandas -q\n",
        "from biopandas.pdb import PandasPdb\n",
        "from biopandas.mmcif import PandasMmcif\n",
        "aa_map = {'VAL': 'V', 'PRO': 'P', 'ASN': 'N', 'GLU': 'E', 'ASP': 'D', 'ALA': 'A', 'THR': 'T', 'SER': 'S',\n",
        "          'LEU': 'L', 'LYS': 'K', 'GLY': 'G', 'GLN': 'Q', 'ILE': 'I', 'PHE': 'F', 'CYS': 'C', 'TRP': 'W',\n",
        "          'ARG': 'R', 'TYR': 'Y', 'HIS': 'H', 'MET': 'M'}\n",
        "aa_map_2 = {x:y for x,y in zip(np.sort(list(aa_map.values())),np.arange(20))}\n",
        "aa_map_2['X'] = 20\n",
        "\n",
        "# https://www.kaggle.com/datasets/alejopaullier/aminoacids-physical-and-chemical-properties\n",
        "aa_props = pd.read_csv('../input/aminoacids-physical-and-chemical-properties/aminoacids.csv').set_index('Letter')\n",
        "PROPS = ['Molecular Weight', 'Residue Weight', 'pKa1', 'pKb2', 'pKx3', 'pl4', \n",
        "         'H', 'VSC', 'P1', 'P2', 'SASA', 'NCISC']\n",
        "print('Amino Acid properties dataframe. Shape:', aa_props.shape )\n",
        "aa_props.head(22)"
      ]
    },
    {
      "cell_type": "markdown",
      "id": "8ba4e530",
      "metadata": {
        "papermill": {
          "duration": 0.01892,
          "end_time": "2022-12-08T02:17:47.598349",
          "exception": false,
          "start_time": "2022-12-08T02:17:47.579429",
          "status": "completed"
        },
        "tags": [],
        "id": "8ba4e530"
      },
      "source": [
        "## Transformer ESM Features\n",
        "In order to convert amino acid sequences aka proteins into meaningful features, we will use embeddings from SOTA protein transformer. We use Facebook's pretrained protein transformer ESM (Evolutionary Scale Modeling) with research paper [here][1] and GitHub [here][2]. Kaggleqrdl provided a starter notebook [here][3]. In future notebook versions, we will show how to extract more feature engineering information from ESM including mutation probabilties and entropy!\n",
        "\n",
        "[1]: https://www.biorxiv.org/content/10.1101/622803v4\n",
        "[2]: https://github.com/facebookresearch/esm\n",
        "[3]: https://www.kaggle.com/code/kaggleqrdl/esm-quick-start-lb237"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "ef141d9d",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:17:47.637362Z",
          "iopub.status.busy": "2022-12-08T02:17:47.635840Z",
          "iopub.status.idle": "2022-12-08T02:19:13.295415Z",
          "shell.execute_reply": "2022-12-08T02:19:13.294399Z"
        },
        "papermill": {
          "duration": 85.681031,
          "end_time": "2022-12-08T02:19:13.297643",
          "exception": false,
          "start_time": "2022-12-08T02:17:47.616612",
          "status": "completed"
        },
        "tags": [],
        "id": "ef141d9d",
        "outputId": "2067514a-999a-4eb2-d8e9-2c7752a8e93c"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\r\n",
            "\u001b[0m"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "Downloading: \"https://dl.fbaipublicfiles.com/fair-esm/models/esm2_t33_650M_UR50D.pt\" to /root/.cache/torch/hub/checkpoints/esm2_t33_650M_UR50D.pt\n",
            "Downloading: \"https://dl.fbaipublicfiles.com/fair-esm/regression/esm2_t33_650M_UR50D-contact-regression.pt\" to /root/.cache/torch/hub/checkpoints/esm2_t33_650M_UR50D-contact-regression.pt\n"
          ]
        },
        {
          "data": {
            "text/plain": [
              "ESM2(\n",
              "  (embed_tokens): Embedding(33, 1280, padding_idx=1)\n",
              "  (layers): ModuleList(\n",
              "    (0): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (1): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (2): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (3): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (4): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (5): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (6): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (7): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (8): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (9): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (10): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (11): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (12): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (13): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (14): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (15): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (16): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (17): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (18): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (19): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (20): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (21): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (22): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (23): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (24): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (25): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (26): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (27): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (28): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (29): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (30): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (31): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "    (32): TransformerLayer(\n",
              "      (self_attn): MultiheadAttention(\n",
              "        (k_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (v_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (q_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (out_proj): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "        (rot_emb): RotaryEmbedding()\n",
              "      )\n",
              "      (self_attn_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "      (fc1): Linear(in_features=1280, out_features=5120, bias=True)\n",
              "      (fc2): Linear(in_features=5120, out_features=1280, bias=True)\n",
              "      (final_layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "    )\n",
              "  )\n",
              "  (contact_head): ContactPredictionHead(\n",
              "    (regression): Linear(in_features=660, out_features=1, bias=True)\n",
              "    (activation): Sigmoid()\n",
              "  )\n",
              "  (emb_layer_norm_after): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "  (lm_head): RobertaLMHead(\n",
              "    (dense): Linear(in_features=1280, out_features=1280, bias=True)\n",
              "    (layer_norm): LayerNorm((1280,), eps=1e-05, elementwise_affine=True)\n",
              "  )\n",
              ")"
            ]
          },
          "execution_count": 11,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# https://github.com/facebookresearch/esm\n",
        "!pip install fair-esm -q\n",
        "\n",
        "# https://www.kaggle.com/code/kaggleqrdl/esm-quick-start-lb237\n",
        "import torch, esm\n",
        "t_model, alphabet = esm.pretrained.esm2_t33_650M_UR50D()\n",
        "batch_converter = alphabet.get_batch_converter()\n",
        "t_model.eval()  # disables dropout for deterministic results\n",
        "device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')\n",
        "t_model.to(device)"
      ]
    },
    {
      "cell_type": "markdown",
      "id": "aece1d44",
      "metadata": {
        "papermill": {
          "duration": 0.01785,
          "end_time": "2022-12-08T02:19:13.334004",
          "exception": false,
          "start_time": "2022-12-08T02:19:13.316154",
          "status": "completed"
        },
        "tags": [],
        "id": "aece1d44"
      },
      "source": [
        "### Embeddings\n",
        "We input each train and test wildtype into our transformer and extract the last hidden layers activations. For each protein, this has shape `(1, len_protein_seq, 1280)`. We will save the full embeddings and the pooled embeddings for use later."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "6e0b9fae",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:19:13.372055Z",
          "iopub.status.busy": "2022-12-08T02:19:13.371445Z",
          "iopub.status.idle": "2022-12-08T02:19:30.156849Z",
          "shell.execute_reply": "2022-12-08T02:19:30.155759Z"
        },
        "papermill": {
          "duration": 16.80677,
          "end_time": "2022-12-08T02:19:30.158988",
          "exception": false,
          "start_time": "2022-12-08T02:19:13.352218",
          "status": "completed"
        },
        "tags": [],
        "id": "6e0b9fae",
        "outputId": "357c3c13-094f-4fc4-a5d4-901cffe33ed2"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Extracting embeddings from proteins...\n",
            "GP01 , GP02 , GP03 , GP04 , GP06 , GP07 , GP08 , GP09 , GP10 , GP11 , GP12 , GP13 , GP14 , GP15 , GP16 , GP17 , GP18 , GP19 , GP20 , GP21 , GP22 , GP23 , GP24 , GP25 , GP26 , GP27 , GP28 , GP29 , GP30 , GP31 , GP32 , GP33 , GP34 , GP35 , GP36 , GP37 , GP38 , GP39 , GP40 , GP41 , GP42 , GP43 , GP44 , GP45 , GP46 , GP48 , GP49 , GP50 , GP51 , GP52 , GP53 , GP54 , GP55 , GP57 , GP58 , GP59 , GP60 , GP62 , GP63 , GP64 , GP65 , GP66 , GP67 , GP68 , GP69 , GP70 , GP71 , GP72 , GP73 , GP74 , GP75 , GP76 , GP77 , TEST , "
          ]
        }
      ],
      "source": [
        "# TRAIN AND TEST WILDTYPES\n",
        "all_pdb = [f for f in kaggle.PDB.unique() if f not in bad]\n",
        "base = 'VPVNPEPDATSVENVALKTGSGDSQSDPIKADLEVKGQSALPFDVDCWAILCKGAPNVLQRVNEKTKNSNRDRSGANKGPFKDPQKWGIKALPPKNPSWSAQDFKSPEEYAFASSLQGGTNAILAPVNLASQNSQGGVLNGFYSANKVAQFDPSKPQQTKGTWFQITKFTGAAGPYCKALGSNDKSVCDKNKNIAGDWGFDPAKWAYQYDEKNNKFNYVGK'\n",
        "all_pdb_embed_pool = np.zeros((len(all_pdb)+1,1280))\n",
        "all_pdb_embed_local = []\n",
        "all_pdb_embed_tmp = []\n",
        "\n",
        "# EXTRACT TRANSFORMER EMBEDDINGS FOR TRAIN AND TEST WILDTYPES\n",
        "print('Extracting embeddings from proteins...')\n",
        "for i,p in enumerate(all_pdb+['TEST']):\n",
        "    \n",
        "    # WILDTYPE SEQUENCE\n",
        "    print(p,', ',end='')\n",
        "    if p=='TEST': seq = base\n",
        "    else: seq = kaggle.loc[kaggle.PDB==p,'sequence'].iloc[0]\n",
        "        \n",
        "    # EXTRACT EMBEDDINGS\n",
        "    data = [(\"protein1\", seq)]\n",
        "    batch_labels, batch_strs, batch_tokens = batch_converter(data)\n",
        "    batch_tokens = batch_tokens.to(device)\n",
        "    with torch.no_grad():\n",
        "        results = t_model(batch_tokens, repr_layers=[33])\n",
        "    results = results[\"representations\"][33].detach().cpu().numpy()\n",
        "    \n",
        "    # SAVE EMBEDDINGS\n",
        "    all_pdb_embed_local.append(results)\n",
        "    all_pdb_embed_pool[i,] = np.mean( results[0,:,:],axis=0 )\n",
        "    \n",
        "    # TEMPORARILY SAVE LOCAL MUTATION EMBEDDINGS\n",
        "    tmp = kaggle.loc[kaggle.PDB==p,'position'].unique()\n",
        "    if p=='TEST': tmp = np.random.choice(range(20,200),8,replace=False)\n",
        "    if len(tmp)>8: tmp = np.random.choice(tmp,8,replace=False)\n",
        "    for j in tmp: all_pdb_embed_tmp.append( results[0,j,:] )\n",
        "        \n",
        "    del batch_tokens, results\n",
        "    gc.collect(); torch.cuda.empty_cache()\n",
        "\n",
        "all_pdb_embed_tmp = np.stack(all_pdb_embed_tmp)"
      ]
    },
    {
      "cell_type": "markdown",
      "id": "59444c6d",
      "metadata": {
        "papermill": {
          "duration": 0.019912,
          "end_time": "2022-12-08T02:19:30.199695",
          "exception": false,
          "start_time": "2022-12-08T02:19:30.179783",
          "status": "completed"
        },
        "tags": [],
        "id": "59444c6d"
      },
      "source": [
        "### RAPIDS PCA\n",
        "The transformer embeddings have dimension 1280. Since we only have a few thousand rows of train data, that is too many features to include all of them in our XGB model. Furthermore, we want to use local, pooling, and delta embeddings. Which would be 3x1280. To prevent our model from overfitting as a result of the \"curse of dimensionality\", we reduce the dimension of embeddings using RAPIDS PCA. "
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "2345ace5",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:19:30.242924Z",
          "iopub.status.busy": "2022-12-08T02:19:30.241892Z",
          "iopub.status.idle": "2022-12-08T02:19:39.717894Z",
          "shell.execute_reply": "2022-12-08T02:19:39.716850Z"
        },
        "papermill": {
          "duration": 9.500433,
          "end_time": "2022-12-08T02:19:39.720305",
          "exception": false,
          "start_time": "2022-12-08T02:19:30.219872",
          "status": "completed"
        },
        "tags": [],
        "id": "2345ace5"
      },
      "outputs": [],
      "source": [
        "# REDUCE EMBEDDING DIM FROM 1280 TO 32 OR 16 WITH PCA\n",
        "from cuml import PCA\n",
        "pca_pool = PCA(n_components=32)\n",
        "pca_embeds = pca_pool.fit_transform(all_pdb_embed_pool.astype('float32'))\n",
        "pca_local = PCA(n_components=16)\n",
        "pca_local.fit(all_pdb_embed_tmp.astype('float32'))\n",
        "pdb_map = {x:y for x,y in zip(all_pdb,range(len(all_pdb)))}\n",
        "pdb_map['kaggle'] = len(all_pdb)\n",
        "del all_pdb_embed_tmp\n",
        "_ = gc.collect()"
      ]
    },
    {
      "cell_type": "markdown",
      "id": "31968626",
      "metadata": {
        "papermill": {
          "duration": 0.019801,
          "end_time": "2022-12-08T02:19:39.761677",
          "exception": false,
          "start_time": "2022-12-08T02:19:39.741876",
          "status": "completed"
        },
        "tags": [],
        "id": "31968626"
      },
      "source": [
        "## Feature Engineering Function"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "4a7410fe",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:19:39.803304Z",
          "iopub.status.busy": "2022-12-08T02:19:39.802972Z",
          "iopub.status.idle": "2022-12-08T02:19:39.825439Z",
          "shell.execute_reply": "2022-12-08T02:19:39.824560Z"
        },
        "papermill": {
          "duration": 0.045871,
          "end_time": "2022-12-08T02:19:39.827456",
          "exception": false,
          "start_time": "2022-12-08T02:19:39.781585",
          "status": "completed"
        },
        "tags": [],
        "id": "4a7410fe"
      },
      "outputs": [],
      "source": [
        "# FEATURE ENGINEER FUNCTION\n",
        "def get_new_row(atom_df, j, row):\n",
        "    ##################\n",
        "    # ATOM_DF - IS PDB FILE'S ATOM_DF\n",
        "    # J - IS RESIDUE NUMBER WHICH IS TRAIN CSV POSITION PLUS OFFSET\n",
        "    # ROW - IS ROW FROM DOWNLOADED TRAIN CSV\n",
        "    ##################\n",
        "        \n",
        "    dd = None\n",
        "    tmp = atom_df.loc[(atom_df.residue_number==j)].reset_index(drop=True)\n",
        "    prev = atom_df.loc[(atom_df.residue_number==j-1)].reset_index(drop=True)\n",
        "    post = atom_df.loc[(atom_df.residue_number==j+1)].reset_index(drop=True)\n",
        "    \n",
        "    # FEATURE ENGINEER\n",
        "    if len(tmp)>0:\n",
        "        \n",
        "        # GET MUTANT EMBEDDINGS\n",
        "        data = [(\"protein1\", row.mutant_seq)]\n",
        "        batch_labels, batch_strs, batch_tokens = batch_converter(data)\n",
        "        batch_tokens = batch_tokens.to(device)\n",
        "        with torch.no_grad():\n",
        "            results = t_model(batch_tokens, repr_layers=[33]) \n",
        "        results = results[\"representations\"][33].cpu().numpy()\n",
        "        mutant_local = pca_local.transform(results[:1,row.position,:])[0,]\n",
        "        mutant_pool = np.mean( results[:1,:,:],axis=1 )\n",
        "        mutant_pool = pca_pool.transform(mutant_pool)[0,]\n",
        "    \n",
        "        # MUTATION AND POSITION\n",
        "        dd = {}\n",
        "        dd['WT'] = row.wildtype\n",
        "        dd['WT2'] = tmp.residue_name.map(aa_map)[0]\n",
        "        dd['MUT'] = row.mutation\n",
        "        dd['position'] = row.position\n",
        "        dd['relative_position'] = row.position / len(row.sequence)\n",
        "\n",
        "        # B_FACTOR\n",
        "        dd['b_factor'] = tmp.b_factor.mean()\n",
        "        \n",
        "        # ANIMO ACID PROPERTIES AND DELTAS\n",
        "        for c in PROPS:\n",
        "            dd[f'{c}_1'] = aa_props.loc[row.wildtype,c]\n",
        "            dd[f'{c}_2'] = aa_props.loc[row.mutation,c]\n",
        "            dd[f'{c}_delta'] = dd[f'{c}_2']-dd[f'{c}_1']\n",
        "\n",
        "        # PREVIOUS AND POST AMINO ACID INFO\n",
        "        if (len(prev)>0):\n",
        "            dd['prev'] = prev.residue_name.map(aa_map)[0]\n",
        "            dd['b_factor_prev'] = prev.b_factor.mean()\n",
        "        else:\n",
        "            dd['prev'] = 'X'\n",
        "            dd['b_factor_prev'] = -999            \n",
        "            \n",
        "        if (len(post)>0):\n",
        "            dd['post'] = post.residue_name.map(aa_map)[0]\n",
        "            dd['b_factor_post'] = post.b_factor.mean() \n",
        "        else:\n",
        "            dd['post'] = 'X'\n",
        "            dd['b_factor_post'] = -999 \n",
        "            \n",
        "        # ANGLE BETWEEN MUTATION AND NEIGHBORS\n",
        "        if (len(prev)>0)&(len(post)>0):\n",
        "            # BACKBONE ATOMS\n",
        "            atm = ['N','H','CA','O']\n",
        "            prev = prev.loc[prev.atom_name.isin(atm)]\n",
        "            tmp = tmp.loc[tmp.atom_name.isin(atm)]\n",
        "            post = post.loc[post.atom_name.isin(atm)]\n",
        "            # VECTORS\n",
        "            c_prev = np.array( [prev.x_coord.mean(),prev.y_coord.mean(),prev.z_coord.mean()] )\n",
        "            c_tmp = np.array( [tmp.x_coord.mean(),tmp.y_coord.mean(),tmp.z_coord.mean()] )\n",
        "            c_post = np.array( [post.x_coord.mean(),post.y_coord.mean(),post.z_coord.mean()] )\n",
        "            vec_a = c_prev - c_tmp\n",
        "            vec_b = c_post - c_tmp\n",
        "            # COMPUTE ANGLE\n",
        "            norm_a = np.sqrt(vec_a.dot(vec_a))\n",
        "            norm_b = np.sqrt(vec_b.dot(vec_b))\n",
        "            dd['cos_angle'] = vec_a.dot(vec_b)/norm_a/norm_b\n",
        "        else:\n",
        "            dd['cos_angle'] = -2\n",
        "            \n",
        "        # 3D LOCATION OF MUTATION\n",
        "        atm = ['N','H','CA','O']\n",
        "        atoms = atom_df.loc[atom_df.atom_name.isin(atm)]\n",
        "        centroid1 = np.array( [atoms.x_coord.mean(),atoms.y_coord.mean(),atoms.z_coord.mean()] )\n",
        "        tmp = tmp.loc[tmp.atom_name.isin(atm)]\n",
        "        centroid2 = np.array( [tmp.x_coord.mean(),tmp.y_coord.mean(),tmp.z_coord.mean()] )\n",
        "        dist = centroid2 - centroid1\n",
        "        dd['location3d'] = dist.dot(dist)\n",
        "        \n",
        "        # TRANSFORMER ESM EMBEDDINGS\n",
        "        wt_local = pca_local.transform(all_pdb_embed_local[pdb_map[row.PDB]][:1,row.position,:])[0,]\n",
        "        wt_pool = pca_embeds[pdb_map[row.PDB],]\n",
        "        for kk in range(32):\n",
        "            dd[f'pca_pool_{kk}'] = mutant_pool[kk] - wt_pool[kk]\n",
        "            if kk>=16: continue\n",
        "            dd[f'pca_wt_{kk}'] = wt_local[kk]\n",
        "            dd[f'pca_mutant_{kk}'] = mutant_local[kk]\n",
        "            dd[f'pca_local_{kk}'] = mutant_local[kk] - wt_local[kk]\n",
        "        \n",
        "        # LABEL ENCODE AMINO ACIDS\n",
        "        dd['AA1'] = aa_map_2[dd['WT']]\n",
        "        dd['AA2'] = aa_map_2[dd['MUT']]\n",
        "        dd['AA3'] = aa_map_2[dd['prev']]\n",
        "        dd['AA4'] = aa_map_2[dd['post']]\n",
        "        \n",
        "        # TARGETS AND SOURCES\n",
        "        dd['ddG'] = row.ddG\n",
        "        dd['dTm'] = row.dTm\n",
        "        dd['pdb'] = row.PDB\n",
        "        dd['source'] = row.source\n",
        "        \n",
        "        del batch_tokens, results, mutant_local, mutant_pool, wt_local, wt_pool\n",
        "        gc.collect(); torch.cuda.empty_cache()\n",
        "\n",
        "    return dd"
      ]
    },
    {
      "cell_type": "markdown",
      "id": "3bc0122e",
      "metadata": {
        "papermill": {
          "duration": 0.019659,
          "end_time": "2022-12-08T02:19:39.867024",
          "exception": false,
          "start_time": "2022-12-08T02:19:39.847365",
          "status": "completed"
        },
        "tags": [],
        "id": "3bc0122e"
      },
      "source": [
        "# Transform Train Data"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "da3bc940",
      "metadata": {
        "_kg_hide-output": true,
        "execution": {
          "iopub.execute_input": "2022-12-08T02:19:39.908408Z",
          "iopub.status.busy": "2022-12-08T02:19:39.908131Z",
          "iopub.status.idle": "2022-12-08T02:30:07.985683Z",
          "shell.execute_reply": "2022-12-08T02:30:07.984675Z"
        },
        "papermill": {
          "duration": 628.101759,
          "end_time": "2022-12-08T02:30:07.988577",
          "exception": false,
          "start_time": "2022-12-08T02:19:39.886818",
          "status": "completed"
        },
        "scrolled": true,
        "tags": [],
        "id": "da3bc940",
        "outputId": "95930c51-aa10-41e2-d1ad-554cbccf4387"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "GP01 PDB residue_number equals 0 added to position in CSV\n",
            "GP02 PDB residue_number equals 0 added to position in CSV\n",
            "GP03 PDB residue_number equals 0 added to position in CSV\n",
            "GP04 PDB residue_number equals 0 added to position in CSV\n",
            "GP06 PDB residue_number equals 0 added to position in CSV\n",
            "GP07 PDB residue_number equals 0 added to position in CSV\n",
            "GP08 PDB residue_number equals 0 added to position in CSV\n",
            "GP09 PDB residue_number equals 0 added to position in CSV\n",
            "GP10 PDB residue_number equals 0 added to position in CSV\n",
            "GP11 PDB residue_number equals 0 added to position in CSV\n",
            "GP12 PDB residue_number equals 0 added to position in CSV\n",
            "GP13 PDB residue_number equals 0 added to position in CSV\n",
            "GP14 PDB residue_number equals 0 added to position in CSV\n",
            "GP15 PDB residue_number equals 0 added to position in CSV\n",
            "GP16 PDB residue_number equals 0 added to position in CSV\n",
            "GP17 PDB residue_number equals 0 added to position in CSV\n",
            "GP18 PDB residue_number equals 0 added to position in CSV\n",
            "GP19 PDB residue_number equals 0 added to position in CSV\n",
            "GP20 PDB residue_number equals 0 added to position in CSV\n",
            "GP21 PDB residue_number equals 0 added to position in CSV\n",
            "GP22 PDB residue_number equals 0 added to position in CSV\n",
            "GP23 PDB residue_number equals 0 added to position in CSV\n",
            "GP24 PDB residue_number equals 0 added to position in CSV\n",
            "GP25 PDB residue_number equals 0 added to position in CSV\n",
            "GP26 PDB residue_number equals 0 added to position in CSV\n",
            "GP27 PDB residue_number equals 0 added to position in CSV\n",
            "GP28 PDB residue_number equals 0 added to position in CSV\n",
            "GP29 PDB residue_number equals 0 added to position in CSV\n",
            "GP30 PDB residue_number equals 0 added to position in CSV\n",
            "GP31 PDB residue_number equals 0 added to position in CSV\n",
            "GP32 PDB residue_number equals 0 added to position in CSV\n",
            "GP33 PDB residue_number equals 0 added to position in CSV\n",
            "GP34 PDB residue_number equals 0 added to position in CSV\n",
            "GP35 PDB residue_number equals 0 added to position in CSV\n",
            "GP36 PDB residue_number equals 0 added to position in CSV\n",
            "GP37 PDB residue_number equals 0 added to position in CSV\n",
            "GP38 PDB residue_number equals 0 added to position in CSV\n",
            "GP39 PDB residue_number equals 0 added to position in CSV\n",
            "GP40 PDB residue_number equals 0 added to position in CSV\n",
            "GP41 PDB residue_number equals 0 added to position in CSV\n",
            "GP42 PDB residue_number equals 0 added to position in CSV\n",
            "GP43 PDB residue_number equals 0 added to position in CSV\n",
            "GP44 PDB residue_number equals 0 added to position in CSV\n",
            "GP45 PDB residue_number equals 0 added to position in CSV\n",
            "GP46 PDB residue_number equals 0 added to position in CSV\n",
            "GP48 PDB residue_number equals 0 added to position in CSV\n",
            "GP49 PDB residue_number equals 0 added to position in CSV\n",
            "GP50 PDB residue_number equals 0 added to position in CSV\n",
            "GP51 PDB residue_number equals 0 added to position in CSV\n",
            "GP52 PDB residue_number equals 0 added to position in CSV\n",
            "GP53 PDB residue_number equals 0 added to position in CSV\n",
            "GP54 PDB residue_number equals 0 added to position in CSV\n",
            "GP55 PDB residue_number equals 0 added to position in CSV\n",
            "GP57 PDB residue_number equals 0 added to position in CSV\n",
            "GP58 PDB residue_number equals 0 added to position in CSV\n",
            "GP59 PDB residue_number equals 0 added to position in CSV\n",
            "GP60 PDB residue_number equals 0 added to position in CSV\n",
            "GP62 PDB residue_number equals 0 added to position in CSV\n",
            "GP63 PDB residue_number equals 0 added to position in CSV\n",
            "GP64 PDB residue_number equals 0 added to position in CSV\n",
            "GP65 PDB residue_number equals 0 added to position in CSV\n",
            "GP66 PDB residue_number equals 0 added to position in CSV\n",
            "GP67 PDB residue_number equals 0 added to position in CSV\n",
            "GP68 PDB residue_number equals 0 added to position in CSV\n",
            "GP69 PDB residue_number equals 0 added to position in CSV\n",
            "GP70 PDB residue_number equals 0 added to position in CSV\n",
            "GP71 PDB residue_number equals 0 added to position in CSV\n",
            "GP72 PDB residue_number equals 0 added to position in CSV\n",
            "GP73 PDB residue_number equals 0 added to position in CSV\n",
            "GP74 PDB residue_number equals 0 added to position in CSV\n",
            "GP75 PDB residue_number equals 0 added to position in CSV\n",
            "GP76 PDB residue_number equals 0 added to position in CSV\n",
            "GP77 PDB residue_number equals 0 added to position in CSV\n"
          ]
        }
      ],
      "source": [
        "pdb = None\n",
        "rows = []\n",
        "offsets = []\n",
        "\n",
        "for index,row in kaggle.iterrows():\n",
        "    if row.PDB in bad: continue\n",
        "        \n",
        "    # READ PDB FILE WHICH CONTAINS MORE INFO ABOUT PROTEIN\n",
        "    first = False\n",
        "    if row.PDB != pdb:\n",
        "        pdb = row.PDB\n",
        "        if row.CIF:\n",
        "            atom_df = PandasMmcif().read_mmcif(f'../input/train-wildtypes-af/cif/{row.CIF}-model_v3.cif')\n",
        "            atom_df = atom_df.df['ATOM']\n",
        "            atom_df = atom_df.rename({'label_seq_id':'residue_number','label_comp_id':'residue_name'},axis=1)\n",
        "            atom_df = atom_df.rename({'Cartn_x':'x_coord','Cartn_y':'y_coord','Cartn_z':'z_coord'},axis=1)\n",
        "            atom_df = atom_df.rename({'B_iso_or_equiv':'b_factor','label_atom_id':'atom_name'},axis=1)\n",
        "        else:\n",
        "            atom_df = PandasPdb().read_pdb(f'downloaded_pdb/{row.PDB}.pdb')\n",
        "            atom_df = atom_df.df['ATOM']\n",
        "        first = True\n",
        "\n",
        "    # VERY IMPORTANT - ALIGN SEQUENCES\n",
        "    # THE RESIDUE NUMBERS IN PDB FILES DONT MATCH THE POSTION NUMBERS IN CSV FILE!\n",
        "    tmp = atom_df.drop_duplicates(['residue_name','residue_number']).sort_values('residue_number')\n",
        "    tmp = tmp.iloc[20:36].reset_index(drop=True)\n",
        "    d = (tmp.residue_number.diff()!=1.0).sum()\n",
        "    if d>1: print(f'=> ERROR missing consecutive amino acids in PDB file {row.PDB}')\n",
        "    tmp['letter'] = tmp.residue_name.map(aa_map)  \n",
        "    pdb_seq = (''.join( tmp.letter.values ))\n",
        "    csv_seq = kaggle.loc[kaggle.PDB==row.PDB,'sequence'].values[0]\n",
        "    i = csv_seq.find(pdb_seq)\n",
        "    if i==-1: print('=> ERROR cannot find PDB sequence in CSV sequence for {row.PDB}')\n",
        "    x = tmp.loc[0,'residue_number']\n",
        "    offset = (x-i)-1\n",
        "    if first: \n",
        "        print(f'{row.PDB} PDB residue_number equals {offset} added to position in CSV')\n",
        "        dd = {}\n",
        "        dd['pdb'] = row.PDB\n",
        "        dd['offset'] = offset\n",
        "        offsets.append(dd)\n",
        "    \n",
        "    # FEATURE ENGINEER\n",
        "    j = row.position + offset\n",
        "    dd = get_new_row(atom_df, j, row)\n",
        "    if dd is not None:\n",
        "        rows.append(dd)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "69565677",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:30:08.040123Z",
          "iopub.status.busy": "2022-12-08T02:30:08.039800Z",
          "iopub.status.idle": "2022-12-08T02:30:08.052596Z",
          "shell.execute_reply": "2022-12-08T02:30:08.050548Z"
        },
        "papermill": {
          "duration": 0.042205,
          "end_time": "2022-12-08T02:30:08.056376",
          "exception": false,
          "start_time": "2022-12-08T02:30:08.014171",
          "status": "completed"
        },
        "tags": [],
        "id": "69565677",
        "outputId": "3c3a39af-eba8-41c7-93db-73f8b0b5285c"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Add these offsets to CSV position to get PDB residue number\n"
          ]
        },
        {
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pdb</th>\n",
              "      <th>offset</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "Empty DataFrame\n",
              "Columns: [pdb, offset]\n",
              "Index: []"
            ]
          },
          "execution_count": 16,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# ADD THESE OFFSETS TO CSV'S POSITION TO GET PDB'S RESIDUE NUMBER\n",
        "offsets = pd.DataFrame(offsets)\n",
        "offsets = offsets.loc[offsets.pdb.str[:2]!='GP'] # drop kaggle CIF\n",
        "#offsets.to_csv('downloaded_csv/PDB_offset_from_CSV.csv',index=False)\n",
        "print('Add these offsets to CSV position to get PDB residue number')\n",
        "offsets.head()"
      ]
    },
    {
      "cell_type": "markdown",
      "id": "3dbee061",
      "metadata": {
        "papermill": {
          "duration": 0.023805,
          "end_time": "2022-12-08T02:30:08.104736",
          "exception": false,
          "start_time": "2022-12-08T02:30:08.080931",
          "status": "completed"
        },
        "tags": [],
        "id": "3dbee061"
      },
      "source": [
        "# Create Train and Holdout\n",
        "We create train and holdout datasets based on variables in code cell #1. We will rank normalize the targets so that the model so the model can train with data from different sources and all targets will look similar to the model."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "f7a050d7",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:30:08.156156Z",
          "iopub.status.busy": "2022-12-08T02:30:08.154315Z",
          "iopub.status.idle": "2022-12-08T02:30:08.283918Z",
          "shell.execute_reply": "2022-12-08T02:30:08.282263Z"
        },
        "papermill": {
          "duration": 0.157666,
          "end_time": "2022-12-08T02:30:08.286821",
          "exception": false,
          "start_time": "2022-12-08T02:30:08.129155",
          "status": "completed"
        },
        "tags": [],
        "id": "f7a050d7",
        "outputId": "ff655757-ee58-41fc-c93c-f300b68aa620"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Train plus Holdout data shape (2556, 136)\n",
            "Data shape after removing small mutation groups (2056, 135)\n"
          ]
        }
      ],
      "source": [
        "# CREATE EXTERNAL TRAIN DATAFRAME\n",
        "train = pd.DataFrame(rows)\n",
        "train = train.loc[train.WT == train.WT2].reset_index(drop=True)\n",
        "print('Train plus Holdout data shape', train.shape )\n",
        "train['ct'] = train.groupby('pdb').WT.transform('count')\n",
        "train = train.loc[train.ct>EXCLUDE_CT_UNDER].reset_index(drop=True)\n",
        "train = train.drop(['WT2','ct'],axis=1)\n",
        "print('Data shape after removing small mutation groups', train.shape )"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "df74c884",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:30:08.338195Z",
          "iopub.status.busy": "2022-12-08T02:30:08.337866Z",
          "iopub.status.idle": "2022-12-08T02:30:08.518108Z",
          "shell.execute_reply": "2022-12-08T02:30:08.516918Z"
        },
        "papermill": {
          "duration": 0.209532,
          "end_time": "2022-12-08T02:30:08.521640",
          "exception": false,
          "start_time": "2022-12-08T02:30:08.312108",
          "status": "completed"
        },
        "tags": [],
        "id": "df74c884",
        "outputId": "4021ec60-bf97-4e42-8438-874297299481"
      },
      "outputs": [
        {
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>WT</th>\n",
              "      <th>MUT</th>\n",
              "      <th>position</th>\n",
              "      <th>relative_position</th>\n",
              "      <th>b_factor</th>\n",
              "      <th>Molecular Weight_1</th>\n",
              "      <th>Molecular Weight_2</th>\n",
              "      <th>Molecular Weight_delta</th>\n",
              "      <th>Residue Weight_1</th>\n",
              "      <th>Residue Weight_2</th>\n",
              "      <th>Residue Weight_delta</th>\n",
              "      <th>pKa1_1</th>\n",
              "      <th>pKa1_2</th>\n",
              "      <th>pKa1_delta</th>\n",
              "      <th>pKb2_1</th>\n",
              "      <th>pKb2_2</th>\n",
              "      <th>pKb2_delta</th>\n",
              "      <th>pKx3_1</th>\n",
              "      <th>pKx3_2</th>\n",
              "      <th>pKx3_delta</th>\n",
              "      <th>pl4_1</th>\n",
              "      <th>pl4_2</th>\n",
              "      <th>pl4_delta</th>\n",
              "      <th>H_1</th>\n",
              "      <th>H_2</th>\n",
              "      <th>H_delta</th>\n",
              "      <th>VSC_1</th>\n",
              "      <th>VSC_2</th>\n",
              "      <th>VSC_delta</th>\n",
              "      <th>P1_1</th>\n",
              "      <th>P1_2</th>\n",
              "      <th>P1_delta</th>\n",
              "      <th>P2_1</th>\n",
              "      <th>P2_2</th>\n",
              "      <th>P2_delta</th>\n",
              "      <th>SASA_1</th>\n",
              "      <th>SASA_2</th>\n",
              "      <th>SASA_delta</th>\n",
              "      <th>NCISC_1</th>\n",
              "      <th>NCISC_2</th>\n",
              "      <th>NCISC_delta</th>\n",
              "      <th>prev</th>\n",
              "      <th>b_factor_prev</th>\n",
              "      <th>post</th>\n",
              "      <th>b_factor_post</th>\n",
              "      <th>cos_angle</th>\n",
              "      <th>location3d</th>\n",
              "      <th>pca_pool_0</th>\n",
              "      <th>pca_wt_0</th>\n",
              "      <th>pca_mutant_0</th>\n",
              "      <th>pca_local_0</th>\n",
              "      <th>pca_pool_1</th>\n",
              "      <th>pca_wt_1</th>\n",
              "      <th>pca_mutant_1</th>\n",
              "      <th>pca_local_1</th>\n",
              "      <th>pca_pool_2</th>\n",
              "      <th>pca_wt_2</th>\n",
              "      <th>pca_mutant_2</th>\n",
              "      <th>pca_local_2</th>\n",
              "      <th>pca_pool_3</th>\n",
              "      <th>pca_wt_3</th>\n",
              "      <th>pca_mutant_3</th>\n",
              "      <th>pca_local_3</th>\n",
              "      <th>pca_pool_4</th>\n",
              "      <th>pca_wt_4</th>\n",
              "      <th>pca_mutant_4</th>\n",
              "      <th>pca_local_4</th>\n",
              "      <th>pca_pool_5</th>\n",
              "      <th>pca_wt_5</th>\n",
              "      <th>pca_mutant_5</th>\n",
              "      <th>pca_local_5</th>\n",
              "      <th>pca_pool_6</th>\n",
              "      <th>pca_wt_6</th>\n",
              "      <th>pca_mutant_6</th>\n",
              "      <th>pca_local_6</th>\n",
              "      <th>pca_pool_7</th>\n",
              "      <th>pca_wt_7</th>\n",
              "      <th>pca_mutant_7</th>\n",
              "      <th>pca_local_7</th>\n",
              "      <th>pca_pool_8</th>\n",
              "      <th>pca_wt_8</th>\n",
              "      <th>pca_mutant_8</th>\n",
              "      <th>pca_local_8</th>\n",
              "      <th>pca_pool_9</th>\n",
              "      <th>pca_wt_9</th>\n",
              "      <th>pca_mutant_9</th>\n",
              "      <th>pca_local_9</th>\n",
              "      <th>pca_pool_10</th>\n",
              "      <th>pca_wt_10</th>\n",
              "      <th>pca_mutant_10</th>\n",
              "      <th>pca_local_10</th>\n",
              "      <th>pca_pool_11</th>\n",
              "      <th>pca_wt_11</th>\n",
              "      <th>pca_mutant_11</th>\n",
              "      <th>pca_local_11</th>\n",
              "      <th>pca_pool_12</th>\n",
              "      <th>pca_wt_12</th>\n",
              "      <th>pca_mutant_12</th>\n",
              "      <th>pca_local_12</th>\n",
              "      <th>pca_pool_13</th>\n",
              "      <th>pca_wt_13</th>\n",
              "      <th>pca_mutant_13</th>\n",
              "      <th>pca_local_13</th>\n",
              "      <th>pca_pool_14</th>\n",
              "      <th>pca_wt_14</th>\n",
              "      <th>pca_mutant_14</th>\n",
              "      <th>pca_local_14</th>\n",
              "      <th>pca_pool_15</th>\n",
              "      <th>pca_wt_15</th>\n",
              "      <th>pca_mutant_15</th>\n",
              "      <th>pca_local_15</th>\n",
              "      <th>pca_pool_16</th>\n",
              "      <th>pca_pool_17</th>\n",
              "      <th>pca_pool_18</th>\n",
              "      <th>pca_pool_19</th>\n",
              "      <th>pca_pool_20</th>\n",
              "      <th>pca_pool_21</th>\n",
              "      <th>pca_pool_22</th>\n",
              "      <th>pca_pool_23</th>\n",
              "      <th>pca_pool_24</th>\n",
              "      <th>pca_pool_25</th>\n",
              "      <th>pca_pool_26</th>\n",
              "      <th>pca_pool_27</th>\n",
              "      <th>pca_pool_28</th>\n",
              "      <th>pca_pool_29</th>\n",
              "      <th>pca_pool_30</th>\n",
              "      <th>pca_pool_31</th>\n",
              "      <th>AA1</th>\n",
              "      <th>AA2</th>\n",
              "      <th>AA3</th>\n",
              "      <th>AA4</th>\n",
              "      <th>ddG</th>\n",
              "      <th>dTm</th>\n",
              "      <th>pdb</th>\n",
              "      <th>source</th>\n",
              "      <th>target</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>L</td>\n",
              "      <td>A</td>\n",
              "      <td>89</td>\n",
              "      <td>0.385281</td>\n",
              "      <td>95.07</td>\n",
              "      <td>131.18</td>\n",
              "      <td>89.10</td>\n",
              "      <td>-42.08</td>\n",
              "      <td>113.16</td>\n",
              "      <td>71.08</td>\n",
              "      <td>-42.08</td>\n",
              "      <td>2.36</td>\n",
              "      <td>2.34</td>\n",
              "      <td>-0.02</td>\n",
              "      <td>9.6</td>\n",
              "      <td>9.69</td>\n",
              "      <td>0.09</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.98</td>\n",
              "      <td>6.00</td>\n",
              "      <td>0.02</td>\n",
              "      <td>1.06</td>\n",
              "      <td>0.62</td>\n",
              "      <td>-0.44</td>\n",
              "      <td>93.5</td>\n",
              "      <td>27.5</td>\n",
              "      <td>-66.0</td>\n",
              "      <td>4.9</td>\n",
              "      <td>8.1</td>\n",
              "      <td>3.2</td>\n",
              "      <td>0.186</td>\n",
              "      <td>0.046</td>\n",
              "      <td>-0.140</td>\n",
              "      <td>1.931</td>\n",
              "      <td>1.181</td>\n",
              "      <td>-0.750</td>\n",
              "      <td>0.051672</td>\n",
              "      <td>0.007187</td>\n",
              "      <td>-0.044485</td>\n",
              "      <td>K</td>\n",
              "      <td>88.59</td>\n",
              "      <td>H</td>\n",
              "      <td>95.97</td>\n",
              "      <td>-0.663945</td>\n",
              "      <td>137.210421</td>\n",
              "      <td>-0.014211</td>\n",
              "      <td>3.765163</td>\n",
              "      <td>3.986665</td>\n",
              "      <td>0.221501</td>\n",
              "      <td>-0.007354</td>\n",
              "      <td>0.923303</td>\n",
              "      <td>0.430142</td>\n",
              "      <td>-0.493161</td>\n",
              "      <td>0.015841</td>\n",
              "      <td>-0.535758</td>\n",
              "      <td>-0.248253</td>\n",
              "      <td>0.287504</td>\n",
              "      <td>0.014395</td>\n",
              "      <td>1.357306</td>\n",
              "      <td>1.130861</td>\n",
              "      <td>-0.226445</td>\n",
              "      <td>-0.021402</td>\n",
              "      <td>-0.888456</td>\n",
              "      <td>-1.839115</td>\n",
              "      <td>-0.950659</td>\n",
              "      <td>0.014413</td>\n",
              "      <td>-0.999351</td>\n",
              "      <td>-0.681300</td>\n",
              "      <td>0.318051</td>\n",
              "      <td>-0.019543</td>\n",
              "      <td>-0.605538</td>\n",
              "      <td>-0.249450</td>\n",
              "      <td>0.356088</td>\n",
              "      <td>0.001984</td>\n",
              "      <td>-0.512543</td>\n",
              "      <td>0.409721</td>\n",
              "      <td>0.922264</td>\n",
              "      <td>-0.002915</td>\n",
              "      <td>0.075837</td>\n",
              "      <td>1.274776</td>\n",
              "      <td>1.198939</td>\n",
              "      <td>0.002151</td>\n",
              "      <td>0.217916</td>\n",
              "      <td>0.350949</td>\n",
              "      <td>0.133032</td>\n",
              "      <td>0.014058</td>\n",
              "      <td>0.331017</td>\n",
              "      <td>0.224792</td>\n",
              "      <td>-0.106225</td>\n",
              "      <td>-0.004729</td>\n",
              "      <td>-0.482276</td>\n",
              "      <td>0.149871</td>\n",
              "      <td>0.632147</td>\n",
              "      <td>-0.029064</td>\n",
              "      <td>-0.884008</td>\n",
              "      <td>-0.740836</td>\n",
              "      <td>0.143172</td>\n",
              "      <td>-0.008013</td>\n",
              "      <td>-0.009869</td>\n",
              "      <td>0.011051</td>\n",
              "      <td>0.020920</td>\n",
              "      <td>-0.001556</td>\n",
              "      <td>0.749459</td>\n",
              "      <td>-0.283267</td>\n",
              "      <td>-1.032727</td>\n",
              "      <td>-0.014077</td>\n",
              "      <td>0.041021</td>\n",
              "      <td>-0.543452</td>\n",
              "      <td>-0.584473</td>\n",
              "      <td>0.000626</td>\n",
              "      <td>-0.003769</td>\n",
              "      <td>0.002321</td>\n",
              "      <td>-0.004178</td>\n",
              "      <td>-0.019689</td>\n",
              "      <td>-0.005612</td>\n",
              "      <td>0.005915</td>\n",
              "      <td>-0.006020</td>\n",
              "      <td>0.002419</td>\n",
              "      <td>-0.009607</td>\n",
              "      <td>0.002067</td>\n",
              "      <td>-0.005116</td>\n",
              "      <td>-0.011324</td>\n",
              "      <td>-0.004637</td>\n",
              "      <td>-0.007803</td>\n",
              "      <td>-0.001802</td>\n",
              "      <td>9</td>\n",
              "      <td>0</td>\n",
              "      <td>8</td>\n",
              "      <td>6</td>\n",
              "      <td>NaN</td>\n",
              "      <td>2.28642</td>\n",
              "      <td>GP01</td>\n",
              "      <td>kaggle.csv</td>\n",
              "      <td>0.578189</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>T</td>\n",
              "      <td>C</td>\n",
              "      <td>95</td>\n",
              "      <td>0.411255</td>\n",
              "      <td>98.41</td>\n",
              "      <td>119.12</td>\n",
              "      <td>121.16</td>\n",
              "      <td>2.04</td>\n",
              "      <td>101.11</td>\n",
              "      <td>103.15</td>\n",
              "      <td>2.04</td>\n",
              "      <td>2.09</td>\n",
              "      <td>1.96</td>\n",
              "      <td>-0.13</td>\n",
              "      <td>9.1</td>\n",
              "      <td>10.28</td>\n",
              "      <td>1.18</td>\n",
              "      <td>NaN</td>\n",
              "      <td>8.18</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.60</td>\n",
              "      <td>5.07</td>\n",
              "      <td>-0.53</td>\n",
              "      <td>-0.05</td>\n",
              "      <td>0.29</td>\n",
              "      <td>0.34</td>\n",
              "      <td>51.3</td>\n",
              "      <td>44.6</td>\n",
              "      <td>-6.7</td>\n",
              "      <td>8.6</td>\n",
              "      <td>5.5</td>\n",
              "      <td>-3.1</td>\n",
              "      <td>0.108</td>\n",
              "      <td>0.128</td>\n",
              "      <td>0.020</td>\n",
              "      <td>1.525</td>\n",
              "      <td>1.461</td>\n",
              "      <td>-0.064</td>\n",
              "      <td>0.003352</td>\n",
              "      <td>-0.036610</td>\n",
              "      <td>-0.039962</td>\n",
              "      <td>A</td>\n",
              "      <td>98.61</td>\n",
              "      <td>L</td>\n",
              "      <td>98.69</td>\n",
              "      <td>-0.828820</td>\n",
              "      <td>328.740432</td>\n",
              "      <td>0.048098</td>\n",
              "      <td>2.922032</td>\n",
              "      <td>2.992943</td>\n",
              "      <td>0.070911</td>\n",
              "      <td>-0.001994</td>\n",
              "      <td>0.084846</td>\n",
              "      <td>0.343217</td>\n",
              "      <td>0.258371</td>\n",
              "      <td>-0.027369</td>\n",
              "      <td>-0.214586</td>\n",
              "      <td>-0.026981</td>\n",
              "      <td>0.187606</td>\n",
              "      <td>-0.003642</td>\n",
              "      <td>0.112732</td>\n",
              "      <td>-0.079620</td>\n",
              "      <td>-0.192352</td>\n",
              "      <td>0.059643</td>\n",
              "      <td>0.439916</td>\n",
              "      <td>0.603866</td>\n",
              "      <td>0.163950</td>\n",
              "      <td>-0.024526</td>\n",
              "      <td>-0.176911</td>\n",
              "      <td>-0.227224</td>\n",
              "      <td>-0.050313</td>\n",
              "      <td>0.039543</td>\n",
              "      <td>-0.382759</td>\n",
              "      <td>-0.420848</td>\n",
              "      <td>-0.038089</td>\n",
              "      <td>-0.004431</td>\n",
              "      <td>0.399533</td>\n",
              "      <td>0.204947</td>\n",
              "      <td>-0.194586</td>\n",
              "      <td>0.017648</td>\n",
              "      <td>-0.557062</td>\n",
              "      <td>-0.308782</td>\n",
              "      <td>0.248280</td>\n",
              "      <td>0.027573</td>\n",
              "      <td>0.309789</td>\n",
              "      <td>0.185098</td>\n",
              "      <td>-0.124691</td>\n",
              "      <td>-0.037391</td>\n",
              "      <td>0.140752</td>\n",
              "      <td>0.234206</td>\n",
              "      <td>0.093454</td>\n",
              "      <td>0.019958</td>\n",
              "      <td>-0.623229</td>\n",
              "      <td>-0.043937</td>\n",
              "      <td>0.579291</td>\n",
              "      <td>0.032968</td>\n",
              "      <td>0.201464</td>\n",
              "      <td>-0.107320</td>\n",
              "      <td>-0.308784</td>\n",
              "      <td>-0.007294</td>\n",
              "      <td>-0.580688</td>\n",
              "      <td>-0.081087</td>\n",
              "      <td>0.499601</td>\n",
              "      <td>0.004310</td>\n",
              "      <td>-0.285651</td>\n",
              "      <td>-0.230316</td>\n",
              "      <td>0.055335</td>\n",
              "      <td>0.020032</td>\n",
              "      <td>-0.323325</td>\n",
              "      <td>-0.097982</td>\n",
              "      <td>0.225342</td>\n",
              "      <td>-0.015060</td>\n",
              "      <td>-0.002613</td>\n",
              "      <td>0.012312</td>\n",
              "      <td>0.008467</td>\n",
              "      <td>0.041541</td>\n",
              "      <td>0.010030</td>\n",
              "      <td>-0.008755</td>\n",
              "      <td>0.016984</td>\n",
              "      <td>-0.021218</td>\n",
              "      <td>-0.022134</td>\n",
              "      <td>0.015226</td>\n",
              "      <td>0.011769</td>\n",
              "      <td>0.020131</td>\n",
              "      <td>-0.004492</td>\n",
              "      <td>-0.007832</td>\n",
              "      <td>-0.015564</td>\n",
              "      <td>16</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>9</td>\n",
              "      <td>NaN</td>\n",
              "      <td>1.48642</td>\n",
              "      <td>GP01</td>\n",
              "      <td>kaggle.csv</td>\n",
              "      <td>0.512346</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>T</td>\n",
              "      <td>C</td>\n",
              "      <td>95</td>\n",
              "      <td>0.411255</td>\n",
              "      <td>98.41</td>\n",
              "      <td>119.12</td>\n",
              "      <td>121.16</td>\n",
              "      <td>2.04</td>\n",
              "      <td>101.11</td>\n",
              "      <td>103.15</td>\n",
              "      <td>2.04</td>\n",
              "      <td>2.09</td>\n",
              "      <td>1.96</td>\n",
              "      <td>-0.13</td>\n",
              "      <td>9.1</td>\n",
              "      <td>10.28</td>\n",
              "      <td>1.18</td>\n",
              "      <td>NaN</td>\n",
              "      <td>8.18</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.60</td>\n",
              "      <td>5.07</td>\n",
              "      <td>-0.53</td>\n",
              "      <td>-0.05</td>\n",
              "      <td>0.29</td>\n",
              "      <td>0.34</td>\n",
              "      <td>51.3</td>\n",
              "      <td>44.6</td>\n",
              "      <td>-6.7</td>\n",
              "      <td>8.6</td>\n",
              "      <td>5.5</td>\n",
              "      <td>-3.1</td>\n",
              "      <td>0.108</td>\n",
              "      <td>0.128</td>\n",
              "      <td>0.020</td>\n",
              "      <td>1.525</td>\n",
              "      <td>1.461</td>\n",
              "      <td>-0.064</td>\n",
              "      <td>0.003352</td>\n",
              "      <td>-0.036610</td>\n",
              "      <td>-0.039962</td>\n",
              "      <td>A</td>\n",
              "      <td>98.61</td>\n",
              "      <td>L</td>\n",
              "      <td>98.69</td>\n",
              "      <td>-0.828820</td>\n",
              "      <td>328.740432</td>\n",
              "      <td>0.048098</td>\n",
              "      <td>2.922032</td>\n",
              "      <td>2.992943</td>\n",
              "      <td>0.070911</td>\n",
              "      <td>-0.001994</td>\n",
              "      <td>0.084846</td>\n",
              "      <td>0.343217</td>\n",
              "      <td>0.258371</td>\n",
              "      <td>-0.027369</td>\n",
              "      <td>-0.214586</td>\n",
              "      <td>-0.026981</td>\n",
              "      <td>0.187606</td>\n",
              "      <td>-0.003642</td>\n",
              "      <td>0.112732</td>\n",
              "      <td>-0.079620</td>\n",
              "      <td>-0.192352</td>\n",
              "      <td>0.059643</td>\n",
              "      <td>0.439916</td>\n",
              "      <td>0.603866</td>\n",
              "      <td>0.163950</td>\n",
              "      <td>-0.024526</td>\n",
              "      <td>-0.176911</td>\n",
              "      <td>-0.227224</td>\n",
              "      <td>-0.050313</td>\n",
              "      <td>0.039543</td>\n",
              "      <td>-0.382759</td>\n",
              "      <td>-0.420848</td>\n",
              "      <td>-0.038089</td>\n",
              "      <td>-0.004431</td>\n",
              "      <td>0.399533</td>\n",
              "      <td>0.204947</td>\n",
              "      <td>-0.194586</td>\n",
              "      <td>0.017648</td>\n",
              "      <td>-0.557062</td>\n",
              "      <td>-0.308782</td>\n",
              "      <td>0.248280</td>\n",
              "      <td>0.027573</td>\n",
              "      <td>0.309789</td>\n",
              "      <td>0.185098</td>\n",
              "      <td>-0.124691</td>\n",
              "      <td>-0.037391</td>\n",
              "      <td>0.140752</td>\n",
              "      <td>0.234206</td>\n",
              "      <td>0.093454</td>\n",
              "      <td>0.019958</td>\n",
              "      <td>-0.623229</td>\n",
              "      <td>-0.043937</td>\n",
              "      <td>0.579291</td>\n",
              "      <td>0.032968</td>\n",
              "      <td>0.201464</td>\n",
              "      <td>-0.107320</td>\n",
              "      <td>-0.308784</td>\n",
              "      <td>-0.007294</td>\n",
              "      <td>-0.580688</td>\n",
              "      <td>-0.081087</td>\n",
              "      <td>0.499601</td>\n",
              "      <td>0.004310</td>\n",
              "      <td>-0.285651</td>\n",
              "      <td>-0.230316</td>\n",
              "      <td>0.055335</td>\n",
              "      <td>0.020032</td>\n",
              "      <td>-0.323325</td>\n",
              "      <td>-0.097982</td>\n",
              "      <td>0.225342</td>\n",
              "      <td>-0.015060</td>\n",
              "      <td>-0.002613</td>\n",
              "      <td>0.012312</td>\n",
              "      <td>0.008467</td>\n",
              "      <td>0.041541</td>\n",
              "      <td>0.010030</td>\n",
              "      <td>-0.008755</td>\n",
              "      <td>0.016984</td>\n",
              "      <td>-0.021218</td>\n",
              "      <td>-0.022134</td>\n",
              "      <td>0.015226</td>\n",
              "      <td>0.011769</td>\n",
              "      <td>0.020131</td>\n",
              "      <td>-0.004492</td>\n",
              "      <td>-0.007832</td>\n",
              "      <td>-0.015564</td>\n",
              "      <td>16</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>9</td>\n",
              "      <td>NaN</td>\n",
              "      <td>0.28642</td>\n",
              "      <td>GP01</td>\n",
              "      <td>kaggle.csv</td>\n",
              "      <td>0.425926</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>T</td>\n",
              "      <td>S</td>\n",
              "      <td>95</td>\n",
              "      <td>0.411255</td>\n",
              "      <td>98.41</td>\n",
              "      <td>119.12</td>\n",
              "      <td>105.09</td>\n",
              "      <td>-14.03</td>\n",
              "      <td>101.11</td>\n",
              "      <td>87.08</td>\n",
              "      <td>-14.03</td>\n",
              "      <td>2.09</td>\n",
              "      <td>2.21</td>\n",
              "      <td>0.12</td>\n",
              "      <td>9.1</td>\n",
              "      <td>9.15</td>\n",
              "      <td>0.05</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.60</td>\n",
              "      <td>5.68</td>\n",
              "      <td>0.08</td>\n",
              "      <td>-0.05</td>\n",
              "      <td>-0.18</td>\n",
              "      <td>-0.13</td>\n",
              "      <td>51.3</td>\n",
              "      <td>29.3</td>\n",
              "      <td>-22.0</td>\n",
              "      <td>8.6</td>\n",
              "      <td>9.2</td>\n",
              "      <td>0.6</td>\n",
              "      <td>0.108</td>\n",
              "      <td>0.062</td>\n",
              "      <td>-0.046</td>\n",
              "      <td>1.525</td>\n",
              "      <td>1.298</td>\n",
              "      <td>-0.227</td>\n",
              "      <td>0.003352</td>\n",
              "      <td>0.004627</td>\n",
              "      <td>0.001275</td>\n",
              "      <td>A</td>\n",
              "      <td>98.61</td>\n",
              "      <td>L</td>\n",
              "      <td>98.69</td>\n",
              "      <td>-0.828820</td>\n",
              "      <td>328.740432</td>\n",
              "      <td>0.021657</td>\n",
              "      <td>2.922032</td>\n",
              "      <td>2.949984</td>\n",
              "      <td>0.027951</td>\n",
              "      <td>-0.005534</td>\n",
              "      <td>0.084846</td>\n",
              "      <td>0.028643</td>\n",
              "      <td>-0.056203</td>\n",
              "      <td>-0.007399</td>\n",
              "      <td>-0.214586</td>\n",
              "      <td>-0.112081</td>\n",
              "      <td>0.102506</td>\n",
              "      <td>0.003737</td>\n",
              "      <td>0.112732</td>\n",
              "      <td>0.140168</td>\n",
              "      <td>0.027436</td>\n",
              "      <td>0.012031</td>\n",
              "      <td>0.439916</td>\n",
              "      <td>0.472333</td>\n",
              "      <td>0.032416</td>\n",
              "      <td>-0.003811</td>\n",
              "      <td>-0.176911</td>\n",
              "      <td>-0.100895</td>\n",
              "      <td>0.076016</td>\n",
              "      <td>0.003911</td>\n",
              "      <td>-0.382759</td>\n",
              "      <td>-0.426468</td>\n",
              "      <td>-0.043709</td>\n",
              "      <td>0.011039</td>\n",
              "      <td>0.399533</td>\n",
              "      <td>0.330352</td>\n",
              "      <td>-0.069180</td>\n",
              "      <td>0.012612</td>\n",
              "      <td>-0.557062</td>\n",
              "      <td>-0.326145</td>\n",
              "      <td>0.230917</td>\n",
              "      <td>0.004261</td>\n",
              "      <td>0.309789</td>\n",
              "      <td>0.462552</td>\n",
              "      <td>0.152764</td>\n",
              "      <td>0.005646</td>\n",
              "      <td>0.140752</td>\n",
              "      <td>0.189888</td>\n",
              "      <td>0.049135</td>\n",
              "      <td>-0.003867</td>\n",
              "      <td>-0.623229</td>\n",
              "      <td>-0.267794</td>\n",
              "      <td>0.355435</td>\n",
              "      <td>0.003348</td>\n",
              "      <td>0.201464</td>\n",
              "      <td>-0.110686</td>\n",
              "      <td>-0.312150</td>\n",
              "      <td>0.003065</td>\n",
              "      <td>-0.580688</td>\n",
              "      <td>-0.348110</td>\n",
              "      <td>0.232577</td>\n",
              "      <td>0.009396</td>\n",
              "      <td>-0.285651</td>\n",
              "      <td>-0.328500</td>\n",
              "      <td>-0.042849</td>\n",
              "      <td>0.002500</td>\n",
              "      <td>-0.323325</td>\n",
              "      <td>-0.273896</td>\n",
              "      <td>0.049428</td>\n",
              "      <td>-0.010602</td>\n",
              "      <td>-0.006238</td>\n",
              "      <td>-0.003419</td>\n",
              "      <td>-0.003106</td>\n",
              "      <td>0.000572</td>\n",
              "      <td>0.000522</td>\n",
              "      <td>0.006822</td>\n",
              "      <td>0.004235</td>\n",
              "      <td>-0.007185</td>\n",
              "      <td>0.001336</td>\n",
              "      <td>0.006135</td>\n",
              "      <td>-0.006449</td>\n",
              "      <td>-0.000727</td>\n",
              "      <td>0.006524</td>\n",
              "      <td>0.011033</td>\n",
              "      <td>0.015508</td>\n",
              "      <td>16</td>\n",
              "      <td>15</td>\n",
              "      <td>0</td>\n",
              "      <td>9</td>\n",
              "      <td>NaN</td>\n",
              "      <td>2.48642</td>\n",
              "      <td>GP01</td>\n",
              "      <td>kaggle.csv</td>\n",
              "      <td>0.592593</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>T</td>\n",
              "      <td>S</td>\n",
              "      <td>95</td>\n",
              "      <td>0.411255</td>\n",
              "      <td>98.41</td>\n",
              "      <td>119.12</td>\n",
              "      <td>105.09</td>\n",
              "      <td>-14.03</td>\n",
              "      <td>101.11</td>\n",
              "      <td>87.08</td>\n",
              "      <td>-14.03</td>\n",
              "      <td>2.09</td>\n",
              "      <td>2.21</td>\n",
              "      <td>0.12</td>\n",
              "      <td>9.1</td>\n",
              "      <td>9.15</td>\n",
              "      <td>0.05</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.60</td>\n",
              "      <td>5.68</td>\n",
              "      <td>0.08</td>\n",
              "      <td>-0.05</td>\n",
              "      <td>-0.18</td>\n",
              "      <td>-0.13</td>\n",
              "      <td>51.3</td>\n",
              "      <td>29.3</td>\n",
              "      <td>-22.0</td>\n",
              "      <td>8.6</td>\n",
              "      <td>9.2</td>\n",
              "      <td>0.6</td>\n",
              "      <td>0.108</td>\n",
              "      <td>0.062</td>\n",
              "      <td>-0.046</td>\n",
              "      <td>1.525</td>\n",
              "      <td>1.298</td>\n",
              "      <td>-0.227</td>\n",
              "      <td>0.003352</td>\n",
              "      <td>0.004627</td>\n",
              "      <td>0.001275</td>\n",
              "      <td>A</td>\n",
              "      <td>98.61</td>\n",
              "      <td>L</td>\n",
              "      <td>98.69</td>\n",
              "      <td>-0.828820</td>\n",
              "      <td>328.740432</td>\n",
              "      <td>0.021657</td>\n",
              "      <td>2.922032</td>\n",
              "      <td>2.949984</td>\n",
              "      <td>0.027951</td>\n",
              "      <td>-0.005534</td>\n",
              "      <td>0.084846</td>\n",
              "      <td>0.028643</td>\n",
              "      <td>-0.056203</td>\n",
              "      <td>-0.007399</td>\n",
              "      <td>-0.214586</td>\n",
              "      <td>-0.112081</td>\n",
              "      <td>0.102506</td>\n",
              "      <td>0.003737</td>\n",
              "      <td>0.112732</td>\n",
              "      <td>0.140168</td>\n",
              "      <td>0.027436</td>\n",
              "      <td>0.012031</td>\n",
              "      <td>0.439916</td>\n",
              "      <td>0.472333</td>\n",
              "      <td>0.032416</td>\n",
              "      <td>-0.003811</td>\n",
              "      <td>-0.176911</td>\n",
              "      <td>-0.100895</td>\n",
              "      <td>0.076016</td>\n",
              "      <td>0.003911</td>\n",
              "      <td>-0.382759</td>\n",
              "      <td>-0.426468</td>\n",
              "      <td>-0.043709</td>\n",
              "      <td>0.011039</td>\n",
              "      <td>0.399533</td>\n",
              "      <td>0.330352</td>\n",
              "      <td>-0.069180</td>\n",
              "      <td>0.012612</td>\n",
              "      <td>-0.557062</td>\n",
              "      <td>-0.326145</td>\n",
              "      <td>0.230917</td>\n",
              "      <td>0.004261</td>\n",
              "      <td>0.309789</td>\n",
              "      <td>0.462552</td>\n",
              "      <td>0.152764</td>\n",
              "      <td>0.005646</td>\n",
              "      <td>0.140752</td>\n",
              "      <td>0.189888</td>\n",
              "      <td>0.049135</td>\n",
              "      <td>-0.003867</td>\n",
              "      <td>-0.623229</td>\n",
              "      <td>-0.267794</td>\n",
              "      <td>0.355435</td>\n",
              "      <td>0.003348</td>\n",
              "      <td>0.201464</td>\n",
              "      <td>-0.110686</td>\n",
              "      <td>-0.312150</td>\n",
              "      <td>0.003065</td>\n",
              "      <td>-0.580688</td>\n",
              "      <td>-0.348110</td>\n",
              "      <td>0.232577</td>\n",
              "      <td>0.009396</td>\n",
              "      <td>-0.285651</td>\n",
              "      <td>-0.328500</td>\n",
              "      <td>-0.042849</td>\n",
              "      <td>0.002500</td>\n",
              "      <td>-0.323325</td>\n",
              "      <td>-0.273896</td>\n",
              "      <td>0.049428</td>\n",
              "      <td>-0.010602</td>\n",
              "      <td>-0.006238</td>\n",
              "      <td>-0.003419</td>\n",
              "      <td>-0.003106</td>\n",
              "      <td>0.000572</td>\n",
              "      <td>0.000522</td>\n",
              "      <td>0.006822</td>\n",
              "      <td>0.004235</td>\n",
              "      <td>-0.007185</td>\n",
              "      <td>0.001336</td>\n",
              "      <td>0.006135</td>\n",
              "      <td>-0.006449</td>\n",
              "      <td>-0.000727</td>\n",
              "      <td>0.006524</td>\n",
              "      <td>0.011033</td>\n",
              "      <td>0.015508</td>\n",
              "      <td>16</td>\n",
              "      <td>15</td>\n",
              "      <td>0</td>\n",
              "      <td>9</td>\n",
              "      <td>NaN</td>\n",
              "      <td>3.88642</td>\n",
              "      <td>GP01</td>\n",
              "      <td>kaggle.csv</td>\n",
              "      <td>0.769547</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "  WT MUT  position  relative_position  b_factor  Molecular Weight_1  \\\n",
              "0  L   A        89           0.385281     95.07              131.18   \n",
              "1  T   C        95           0.411255     98.41              119.12   \n",
              "2  T   C        95           0.411255     98.41              119.12   \n",
              "3  T   S        95           0.411255     98.41              119.12   \n",
              "4  T   S        95           0.411255     98.41              119.12   \n",
              "\n",
              "   Molecular Weight_2  Molecular Weight_delta  Residue Weight_1  \\\n",
              "0               89.10                  -42.08            113.16   \n",
              "1              121.16                    2.04            101.11   \n",
              "2              121.16                    2.04            101.11   \n",
              "3              105.09                  -14.03            101.11   \n",
              "4              105.09                  -14.03            101.11   \n",
              "\n",
              "   Residue Weight_2  Residue Weight_delta  pKa1_1  pKa1_2  pKa1_delta  pKb2_1  \\\n",
              "0             71.08                -42.08    2.36    2.34       -0.02     9.6   \n",
              "1            103.15                  2.04    2.09    1.96       -0.13     9.1   \n",
              "2            103.15                  2.04    2.09    1.96       -0.13     9.1   \n",
              "3             87.08                -14.03    2.09    2.21        0.12     9.1   \n",
              "4             87.08                -14.03    2.09    2.21        0.12     9.1   \n",
              "\n",
              "   pKb2_2  pKb2_delta  pKx3_1  pKx3_2  pKx3_delta  pl4_1  pl4_2  pl4_delta  \\\n",
              "0    9.69        0.09     NaN     NaN         NaN   5.98   6.00       0.02   \n",
              "1   10.28        1.18     NaN    8.18         NaN   5.60   5.07      -0.53   \n",
              "2   10.28        1.18     NaN    8.18         NaN   5.60   5.07      -0.53   \n",
              "3    9.15        0.05     NaN     NaN         NaN   5.60   5.68       0.08   \n",
              "4    9.15        0.05     NaN     NaN         NaN   5.60   5.68       0.08   \n",
              "\n",
              "    H_1   H_2  H_delta  VSC_1  VSC_2  VSC_delta  P1_1  P1_2  P1_delta   P2_1  \\\n",
              "0  1.06  0.62    -0.44   93.5   27.5      -66.0   4.9   8.1       3.2  0.186   \n",
              "1 -0.05  0.29     0.34   51.3   44.6       -6.7   8.6   5.5      -3.1  0.108   \n",
              "2 -0.05  0.29     0.34   51.3   44.6       -6.7   8.6   5.5      -3.1  0.108   \n",
              "3 -0.05 -0.18    -0.13   51.3   29.3      -22.0   8.6   9.2       0.6  0.108   \n",
              "4 -0.05 -0.18    -0.13   51.3   29.3      -22.0   8.6   9.2       0.6  0.108   \n",
              "\n",
              "    P2_2  P2_delta  SASA_1  SASA_2  SASA_delta   NCISC_1   NCISC_2  \\\n",
              "0  0.046    -0.140   1.931   1.181      -0.750  0.051672  0.007187   \n",
              "1  0.128     0.020   1.525   1.461      -0.064  0.003352 -0.036610   \n",
              "2  0.128     0.020   1.525   1.461      -0.064  0.003352 -0.036610   \n",
              "3  0.062    -0.046   1.525   1.298      -0.227  0.003352  0.004627   \n",
              "4  0.062    -0.046   1.525   1.298      -0.227  0.003352  0.004627   \n",
              "\n",
              "   NCISC_delta prev  b_factor_prev post  b_factor_post  cos_angle  location3d  \\\n",
              "0    -0.044485    K          88.59    H          95.97  -0.663945  137.210421   \n",
              "1    -0.039962    A          98.61    L          98.69  -0.828820  328.740432   \n",
              "2    -0.039962    A          98.61    L          98.69  -0.828820  328.740432   \n",
              "3     0.001275    A          98.61    L          98.69  -0.828820  328.740432   \n",
              "4     0.001275    A          98.61    L          98.69  -0.828820  328.740432   \n",
              "\n",
              "   pca_pool_0  pca_wt_0  pca_mutant_0  pca_local_0  pca_pool_1  pca_wt_1  \\\n",
              "0   -0.014211  3.765163      3.986665     0.221501   -0.007354  0.923303   \n",
              "1    0.048098  2.922032      2.992943     0.070911   -0.001994  0.084846   \n",
              "2    0.048098  2.922032      2.992943     0.070911   -0.001994  0.084846   \n",
              "3    0.021657  2.922032      2.949984     0.027951   -0.005534  0.084846   \n",
              "4    0.021657  2.922032      2.949984     0.027951   -0.005534  0.084846   \n",
              "\n",
              "   pca_mutant_1  pca_local_1  pca_pool_2  pca_wt_2  pca_mutant_2  pca_local_2  \\\n",
              "0      0.430142    -0.493161    0.015841 -0.535758     -0.248253     0.287504   \n",
              "1      0.343217     0.258371   -0.027369 -0.214586     -0.026981     0.187606   \n",
              "2      0.343217     0.258371   -0.027369 -0.214586     -0.026981     0.187606   \n",
              "3      0.028643    -0.056203   -0.007399 -0.214586     -0.112081     0.102506   \n",
              "4      0.028643    -0.056203   -0.007399 -0.214586     -0.112081     0.102506   \n",
              "\n",
              "   pca_pool_3  pca_wt_3  pca_mutant_3  pca_local_3  pca_pool_4  pca_wt_4  \\\n",
              "0    0.014395  1.357306      1.130861    -0.226445   -0.021402 -0.888456   \n",
              "1   -0.003642  0.112732     -0.079620    -0.192352    0.059643  0.439916   \n",
              "2   -0.003642  0.112732     -0.079620    -0.192352    0.059643  0.439916   \n",
              "3    0.003737  0.112732      0.140168     0.027436    0.012031  0.439916   \n",
              "4    0.003737  0.112732      0.140168     0.027436    0.012031  0.439916   \n",
              "\n",
              "   pca_mutant_4  pca_local_4  pca_pool_5  pca_wt_5  pca_mutant_5  pca_local_5  \\\n",
              "0     -1.839115    -0.950659    0.014413 -0.999351     -0.681300     0.318051   \n",
              "1      0.603866     0.163950   -0.024526 -0.176911     -0.227224    -0.050313   \n",
              "2      0.603866     0.163950   -0.024526 -0.176911     -0.227224    -0.050313   \n",
              "3      0.472333     0.032416   -0.003811 -0.176911     -0.100895     0.076016   \n",
              "4      0.472333     0.032416   -0.003811 -0.176911     -0.100895     0.076016   \n",
              "\n",
              "   pca_pool_6  pca_wt_6  pca_mutant_6  pca_local_6  pca_pool_7  pca_wt_7  \\\n",
              "0   -0.019543 -0.605538     -0.249450     0.356088    0.001984 -0.512543   \n",
              "1    0.039543 -0.382759     -0.420848    -0.038089   -0.004431  0.399533   \n",
              "2    0.039543 -0.382759     -0.420848    -0.038089   -0.004431  0.399533   \n",
              "3    0.003911 -0.382759     -0.426468    -0.043709    0.011039  0.399533   \n",
              "4    0.003911 -0.382759     -0.426468    -0.043709    0.011039  0.399533   \n",
              "\n",
              "   pca_mutant_7  pca_local_7  pca_pool_8  pca_wt_8  pca_mutant_8  pca_local_8  \\\n",
              "0      0.409721     0.922264   -0.002915  0.075837      1.274776     1.198939   \n",
              "1      0.204947    -0.194586    0.017648 -0.557062     -0.308782     0.248280   \n",
              "2      0.204947    -0.194586    0.017648 -0.557062     -0.308782     0.248280   \n",
              "3      0.330352    -0.069180    0.012612 -0.557062     -0.326145     0.230917   \n",
              "4      0.330352    -0.069180    0.012612 -0.557062     -0.326145     0.230917   \n",
              "\n",
              "   pca_pool_9  pca_wt_9  pca_mutant_9  pca_local_9  pca_pool_10  pca_wt_10  \\\n",
              "0    0.002151  0.217916      0.350949     0.133032     0.014058   0.331017   \n",
              "1    0.027573  0.309789      0.185098    -0.124691    -0.037391   0.140752   \n",
              "2    0.027573  0.309789      0.185098    -0.124691    -0.037391   0.140752   \n",
              "3    0.004261  0.309789      0.462552     0.152764     0.005646   0.140752   \n",
              "4    0.004261  0.309789      0.462552     0.152764     0.005646   0.140752   \n",
              "\n",
              "   pca_mutant_10  pca_local_10  pca_pool_11  pca_wt_11  pca_mutant_11  \\\n",
              "0       0.224792     -0.106225    -0.004729  -0.482276       0.149871   \n",
              "1       0.234206      0.093454     0.019958  -0.623229      -0.043937   \n",
              "2       0.234206      0.093454     0.019958  -0.623229      -0.043937   \n",
              "3       0.189888      0.049135    -0.003867  -0.623229      -0.267794   \n",
              "4       0.189888      0.049135    -0.003867  -0.623229      -0.267794   \n",
              "\n",
              "   pca_local_11  pca_pool_12  pca_wt_12  pca_mutant_12  pca_local_12  \\\n",
              "0      0.632147    -0.029064  -0.884008      -0.740836      0.143172   \n",
              "1      0.579291     0.032968   0.201464      -0.107320     -0.308784   \n",
              "2      0.579291     0.032968   0.201464      -0.107320     -0.308784   \n",
              "3      0.355435     0.003348   0.201464      -0.110686     -0.312150   \n",
              "4      0.355435     0.003348   0.201464      -0.110686     -0.312150   \n",
              "\n",
              "   pca_pool_13  pca_wt_13  pca_mutant_13  pca_local_13  pca_pool_14  \\\n",
              "0    -0.008013  -0.009869       0.011051      0.020920    -0.001556   \n",
              "1    -0.007294  -0.580688      -0.081087      0.499601     0.004310   \n",
              "2    -0.007294  -0.580688      -0.081087      0.499601     0.004310   \n",
              "3     0.003065  -0.580688      -0.348110      0.232577     0.009396   \n",
              "4     0.003065  -0.580688      -0.348110      0.232577     0.009396   \n",
              "\n",
              "   pca_wt_14  pca_mutant_14  pca_local_14  pca_pool_15  pca_wt_15  \\\n",
              "0   0.749459      -0.283267     -1.032727    -0.014077   0.041021   \n",
              "1  -0.285651      -0.230316      0.055335     0.020032  -0.323325   \n",
              "2  -0.285651      -0.230316      0.055335     0.020032  -0.323325   \n",
              "3  -0.285651      -0.328500     -0.042849     0.002500  -0.323325   \n",
              "4  -0.285651      -0.328500     -0.042849     0.002500  -0.323325   \n",
              "\n",
              "   pca_mutant_15  pca_local_15  pca_pool_16  pca_pool_17  pca_pool_18  \\\n",
              "0      -0.543452     -0.584473     0.000626    -0.003769     0.002321   \n",
              "1      -0.097982      0.225342    -0.015060    -0.002613     0.012312   \n",
              "2      -0.097982      0.225342    -0.015060    -0.002613     0.012312   \n",
              "3      -0.273896      0.049428    -0.010602    -0.006238    -0.003419   \n",
              "4      -0.273896      0.049428    -0.010602    -0.006238    -0.003419   \n",
              "\n",
              "   pca_pool_19  pca_pool_20  pca_pool_21  pca_pool_22  pca_pool_23  \\\n",
              "0    -0.004178    -0.019689    -0.005612     0.005915    -0.006020   \n",
              "1     0.008467     0.041541     0.010030    -0.008755     0.016984   \n",
              "2     0.008467     0.041541     0.010030    -0.008755     0.016984   \n",
              "3    -0.003106     0.000572     0.000522     0.006822     0.004235   \n",
              "4    -0.003106     0.000572     0.000522     0.006822     0.004235   \n",
              "\n",
              "   pca_pool_24  pca_pool_25  pca_pool_26  pca_pool_27  pca_pool_28  \\\n",
              "0     0.002419    -0.009607     0.002067    -0.005116    -0.011324   \n",
              "1    -0.021218    -0.022134     0.015226     0.011769     0.020131   \n",
              "2    -0.021218    -0.022134     0.015226     0.011769     0.020131   \n",
              "3    -0.007185     0.001336     0.006135    -0.006449    -0.000727   \n",
              "4    -0.007185     0.001336     0.006135    -0.006449    -0.000727   \n",
              "\n",
              "   pca_pool_29  pca_pool_30  pca_pool_31  AA1  AA2  AA3  AA4  ddG      dTm  \\\n",
              "0    -0.004637    -0.007803    -0.001802    9    0    8    6  NaN  2.28642   \n",
              "1    -0.004492    -0.007832    -0.015564   16    1    0    9  NaN  1.48642   \n",
              "2    -0.004492    -0.007832    -0.015564   16    1    0    9  NaN  0.28642   \n",
              "3     0.006524     0.011033     0.015508   16   15    0    9  NaN  2.48642   \n",
              "4     0.006524     0.011033     0.015508   16   15    0    9  NaN  3.88642   \n",
              "\n",
              "    pdb      source    target  \n",
              "0  GP01  kaggle.csv  0.578189  \n",
              "1  GP01  kaggle.csv  0.512346  \n",
              "2  GP01  kaggle.csv  0.425926  \n",
              "3  GP01  kaggle.csv  0.592593  \n",
              "4  GP01  kaggle.csv  0.769547  "
            ]
          },
          "execution_count": 18,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# RANK NORMALIZE ddG AND dTm TARGETS SO ALL CAN BE MIXED TOGETHER\n",
        "train['target'] = 0.5\n",
        "for g in train.pdb.unique():\n",
        "    target = 'dTm'\n",
        "    tmp = train.loc[train.pdb==g,'dTm']\n",
        "    if tmp.isna().sum()>len(tmp)/2: target = 'ddG'\n",
        "    train.loc[train.pdb==g,'target'] =\\\n",
        "        rankdata(train.loc[train.pdb==g,target])/len(train.loc[train.pdb==g,target])\n",
        "train.head()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "df4f364e",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:30:08.577770Z",
          "iopub.status.busy": "2022-12-08T02:30:08.577426Z",
          "iopub.status.idle": "2022-12-08T02:30:08.588830Z",
          "shell.execute_reply": "2022-12-08T02:30:08.587990Z"
        },
        "papermill": {
          "duration": 0.040526,
          "end_time": "2022-12-08T02:30:08.590800",
          "exception": false,
          "start_time": "2022-12-08T02:30:08.550274",
          "status": "completed"
        },
        "tags": [],
        "id": "df4f364e"
      },
      "outputs": [],
      "source": [
        "# USE some sources TO TRAIN/VALIDATE AND other sources TO HOLDOUT VALIDATE\n",
        "holdout = train.loc[train.source.isin(HOLDOUT_SOURCES)].reset_index(drop=True)\n",
        "train = train.loc[train.source.isin(KFOLD_SOURCES)].reset_index(drop=True)\n",
        "\n",
        "# LABEL ENCODE GROUPS FOR GROUP K FOLD\n",
        "train['group'],_ = train.pdb.factorize()\n",
        "holdout['group'],_ = holdout.pdb.factorize()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "d742d17a",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:30:08.645271Z",
          "iopub.status.busy": "2022-12-08T02:30:08.644345Z",
          "iopub.status.idle": "2022-12-08T02:30:08.652021Z",
          "shell.execute_reply": "2022-12-08T02:30:08.650386Z"
        },
        "papermill": {
          "duration": 0.037762,
          "end_time": "2022-12-08T02:30:08.654779",
          "exception": false,
          "start_time": "2022-12-08T02:30:08.617017",
          "status": "completed"
        },
        "tags": [],
        "id": "d742d17a",
        "outputId": "3f0d1b18-206e-400a-9a20-11cc80fa068d"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "We have 127 features for our model:\n",
            "['position', 'relative_position', 'b_factor', 'Molecular Weight_1', 'Molecular Weight_2', 'Molecular Weight_delta', 'Residue Weight_1', 'Residue Weight_2', 'Residue Weight_delta', 'pKa1_1', 'pKa1_2', 'pKa1_delta', 'pKb2_1', 'pKb2_2', 'pKb2_delta', 'pKx3_1', 'pKx3_2', 'pKx3_delta', 'pl4_1', 'pl4_2', 'pl4_delta', 'H_1', 'H_2', 'H_delta', 'VSC_1', 'VSC_2', 'VSC_delta', 'P1_1', 'P1_2', 'P1_delta', 'P2_1', 'P2_2', 'P2_delta', 'SASA_1', 'SASA_2', 'SASA_delta', 'NCISC_1', 'NCISC_2', 'NCISC_delta', 'b_factor_prev', 'b_factor_post', 'cos_angle', 'location3d', 'pca_pool_0', 'pca_wt_0', 'pca_mutant_0', 'pca_local_0', 'pca_pool_1', 'pca_wt_1', 'pca_mutant_1', 'pca_local_1', 'pca_pool_2', 'pca_wt_2', 'pca_mutant_2', 'pca_local_2', 'pca_pool_3', 'pca_wt_3', 'pca_mutant_3', 'pca_local_3', 'pca_pool_4', 'pca_wt_4', 'pca_mutant_4', 'pca_local_4', 'pca_pool_5', 'pca_wt_5', 'pca_mutant_5', 'pca_local_5', 'pca_pool_6', 'pca_wt_6', 'pca_mutant_6', 'pca_local_6', 'pca_pool_7', 'pca_wt_7', 'pca_mutant_7', 'pca_local_7', 'pca_pool_8', 'pca_wt_8', 'pca_mutant_8', 'pca_local_8', 'pca_pool_9', 'pca_wt_9', 'pca_mutant_9', 'pca_local_9', 'pca_pool_10', 'pca_wt_10', 'pca_mutant_10', 'pca_local_10', 'pca_pool_11', 'pca_wt_11', 'pca_mutant_11', 'pca_local_11', 'pca_pool_12', 'pca_wt_12', 'pca_mutant_12', 'pca_local_12', 'pca_pool_13', 'pca_wt_13', 'pca_mutant_13', 'pca_local_13', 'pca_pool_14', 'pca_wt_14', 'pca_mutant_14', 'pca_local_14', 'pca_pool_15', 'pca_wt_15', 'pca_mutant_15', 'pca_local_15', 'pca_pool_16', 'pca_pool_17', 'pca_pool_18', 'pca_pool_19', 'pca_pool_20', 'pca_pool_21', 'pca_pool_22', 'pca_pool_23', 'pca_pool_24', 'pca_pool_25', 'pca_pool_26', 'pca_pool_27', 'pca_pool_28', 'pca_pool_29', 'pca_pool_30', 'pca_pool_31', 'AA1', 'AA2', 'AA3', 'AA4']\n"
          ]
        }
      ],
      "source": [
        "EXCLUDE = ['WT','MUT','prev','post','ddG','dTm','pdb','source','target','group','oof']\n",
        "FEATURES = [c for c in train.columns if c not in EXCLUDE]\n",
        "print(f'We have {len(FEATURES)} features for our model:')\n",
        "print( FEATURES )"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "5f994ab3",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:30:08.708789Z",
          "iopub.status.busy": "2022-12-08T02:30:08.707267Z",
          "iopub.status.idle": "2022-12-08T02:30:08.806874Z",
          "shell.execute_reply": "2022-12-08T02:30:08.805717Z"
        },
        "papermill": {
          "duration": 0.129741,
          "end_time": "2022-12-08T02:30:08.810325",
          "exception": false,
          "start_time": "2022-12-08T02:30:08.680584",
          "status": "completed"
        },
        "tags": [],
        "id": "5f994ab3",
        "outputId": "fba88746-3092-4eb8-ee17-f31b8daafe38"
      },
      "outputs": [
        {
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>WT</th>\n",
              "      <th>MUT</th>\n",
              "      <th>position</th>\n",
              "      <th>relative_position</th>\n",
              "      <th>b_factor</th>\n",
              "      <th>Molecular Weight_1</th>\n",
              "      <th>Molecular Weight_2</th>\n",
              "      <th>Molecular Weight_delta</th>\n",
              "      <th>Residue Weight_1</th>\n",
              "      <th>Residue Weight_2</th>\n",
              "      <th>Residue Weight_delta</th>\n",
              "      <th>pKa1_1</th>\n",
              "      <th>pKa1_2</th>\n",
              "      <th>pKa1_delta</th>\n",
              "      <th>pKb2_1</th>\n",
              "      <th>pKb2_2</th>\n",
              "      <th>pKb2_delta</th>\n",
              "      <th>pKx3_1</th>\n",
              "      <th>pKx3_2</th>\n",
              "      <th>pKx3_delta</th>\n",
              "      <th>pl4_1</th>\n",
              "      <th>pl4_2</th>\n",
              "      <th>pl4_delta</th>\n",
              "      <th>H_1</th>\n",
              "      <th>H_2</th>\n",
              "      <th>H_delta</th>\n",
              "      <th>VSC_1</th>\n",
              "      <th>VSC_2</th>\n",
              "      <th>VSC_delta</th>\n",
              "      <th>P1_1</th>\n",
              "      <th>P1_2</th>\n",
              "      <th>P1_delta</th>\n",
              "      <th>P2_1</th>\n",
              "      <th>P2_2</th>\n",
              "      <th>P2_delta</th>\n",
              "      <th>SASA_1</th>\n",
              "      <th>SASA_2</th>\n",
              "      <th>SASA_delta</th>\n",
              "      <th>NCISC_1</th>\n",
              "      <th>NCISC_2</th>\n",
              "      <th>NCISC_delta</th>\n",
              "      <th>prev</th>\n",
              "      <th>b_factor_prev</th>\n",
              "      <th>post</th>\n",
              "      <th>b_factor_post</th>\n",
              "      <th>cos_angle</th>\n",
              "      <th>location3d</th>\n",
              "      <th>pca_pool_0</th>\n",
              "      <th>pca_wt_0</th>\n",
              "      <th>pca_mutant_0</th>\n",
              "      <th>pca_local_0</th>\n",
              "      <th>pca_pool_1</th>\n",
              "      <th>pca_wt_1</th>\n",
              "      <th>pca_mutant_1</th>\n",
              "      <th>pca_local_1</th>\n",
              "      <th>pca_pool_2</th>\n",
              "      <th>pca_wt_2</th>\n",
              "      <th>pca_mutant_2</th>\n",
              "      <th>pca_local_2</th>\n",
              "      <th>pca_pool_3</th>\n",
              "      <th>pca_wt_3</th>\n",
              "      <th>pca_mutant_3</th>\n",
              "      <th>pca_local_3</th>\n",
              "      <th>pca_pool_4</th>\n",
              "      <th>pca_wt_4</th>\n",
              "      <th>pca_mutant_4</th>\n",
              "      <th>pca_local_4</th>\n",
              "      <th>pca_pool_5</th>\n",
              "      <th>pca_wt_5</th>\n",
              "      <th>pca_mutant_5</th>\n",
              "      <th>pca_local_5</th>\n",
              "      <th>pca_pool_6</th>\n",
              "      <th>pca_wt_6</th>\n",
              "      <th>pca_mutant_6</th>\n",
              "      <th>pca_local_6</th>\n",
              "      <th>pca_pool_7</th>\n",
              "      <th>pca_wt_7</th>\n",
              "      <th>pca_mutant_7</th>\n",
              "      <th>pca_local_7</th>\n",
              "      <th>pca_pool_8</th>\n",
              "      <th>pca_wt_8</th>\n",
              "      <th>pca_mutant_8</th>\n",
              "      <th>pca_local_8</th>\n",
              "      <th>pca_pool_9</th>\n",
              "      <th>pca_wt_9</th>\n",
              "      <th>pca_mutant_9</th>\n",
              "      <th>pca_local_9</th>\n",
              "      <th>pca_pool_10</th>\n",
              "      <th>pca_wt_10</th>\n",
              "      <th>pca_mutant_10</th>\n",
              "      <th>pca_local_10</th>\n",
              "      <th>pca_pool_11</th>\n",
              "      <th>pca_wt_11</th>\n",
              "      <th>pca_mutant_11</th>\n",
              "      <th>pca_local_11</th>\n",
              "      <th>pca_pool_12</th>\n",
              "      <th>pca_wt_12</th>\n",
              "      <th>pca_mutant_12</th>\n",
              "      <th>pca_local_12</th>\n",
              "      <th>pca_pool_13</th>\n",
              "      <th>pca_wt_13</th>\n",
              "      <th>pca_mutant_13</th>\n",
              "      <th>pca_local_13</th>\n",
              "      <th>pca_pool_14</th>\n",
              "      <th>pca_wt_14</th>\n",
              "      <th>pca_mutant_14</th>\n",
              "      <th>pca_local_14</th>\n",
              "      <th>pca_pool_15</th>\n",
              "      <th>pca_wt_15</th>\n",
              "      <th>pca_mutant_15</th>\n",
              "      <th>pca_local_15</th>\n",
              "      <th>pca_pool_16</th>\n",
              "      <th>pca_pool_17</th>\n",
              "      <th>pca_pool_18</th>\n",
              "      <th>pca_pool_19</th>\n",
              "      <th>pca_pool_20</th>\n",
              "      <th>pca_pool_21</th>\n",
              "      <th>pca_pool_22</th>\n",
              "      <th>pca_pool_23</th>\n",
              "      <th>pca_pool_24</th>\n",
              "      <th>pca_pool_25</th>\n",
              "      <th>pca_pool_26</th>\n",
              "      <th>pca_pool_27</th>\n",
              "      <th>pca_pool_28</th>\n",
              "      <th>pca_pool_29</th>\n",
              "      <th>pca_pool_30</th>\n",
              "      <th>pca_pool_31</th>\n",
              "      <th>AA1</th>\n",
              "      <th>AA2</th>\n",
              "      <th>AA3</th>\n",
              "      <th>AA4</th>\n",
              "      <th>ddG</th>\n",
              "      <th>dTm</th>\n",
              "      <th>pdb</th>\n",
              "      <th>source</th>\n",
              "      <th>target</th>\n",
              "      <th>group</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>L</td>\n",
              "      <td>A</td>\n",
              "      <td>89</td>\n",
              "      <td>0.385281</td>\n",
              "      <td>95.07</td>\n",
              "      <td>131.18</td>\n",
              "      <td>89.10</td>\n",
              "      <td>-42.08</td>\n",
              "      <td>113.16</td>\n",
              "      <td>71.08</td>\n",
              "      <td>-42.08</td>\n",
              "      <td>2.36</td>\n",
              "      <td>2.34</td>\n",
              "      <td>-0.02</td>\n",
              "      <td>9.6</td>\n",
              "      <td>9.69</td>\n",
              "      <td>0.09</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.98</td>\n",
              "      <td>6.00</td>\n",
              "      <td>0.02</td>\n",
              "      <td>1.06</td>\n",
              "      <td>0.62</td>\n",
              "      <td>-0.44</td>\n",
              "      <td>93.5</td>\n",
              "      <td>27.5</td>\n",
              "      <td>-66.0</td>\n",
              "      <td>4.9</td>\n",
              "      <td>8.1</td>\n",
              "      <td>3.2</td>\n",
              "      <td>0.186</td>\n",
              "      <td>0.046</td>\n",
              "      <td>-0.140</td>\n",
              "      <td>1.931</td>\n",
              "      <td>1.181</td>\n",
              "      <td>-0.750</td>\n",
              "      <td>0.051672</td>\n",
              "      <td>0.007187</td>\n",
              "      <td>-0.044485</td>\n",
              "      <td>K</td>\n",
              "      <td>88.59</td>\n",
              "      <td>H</td>\n",
              "      <td>95.97</td>\n",
              "      <td>-0.663945</td>\n",
              "      <td>137.210421</td>\n",
              "      <td>-0.014211</td>\n",
              "      <td>3.765163</td>\n",
              "      <td>3.986665</td>\n",
              "      <td>0.221501</td>\n",
              "      <td>-0.007354</td>\n",
              "      <td>0.923303</td>\n",
              "      <td>0.430142</td>\n",
              "      <td>-0.493161</td>\n",
              "      <td>0.015841</td>\n",
              "      <td>-0.535758</td>\n",
              "      <td>-0.248253</td>\n",
              "      <td>0.287504</td>\n",
              "      <td>0.014395</td>\n",
              "      <td>1.357306</td>\n",
              "      <td>1.130861</td>\n",
              "      <td>-0.226445</td>\n",
              "      <td>-0.021402</td>\n",
              "      <td>-0.888456</td>\n",
              "      <td>-1.839115</td>\n",
              "      <td>-0.950659</td>\n",
              "      <td>0.014413</td>\n",
              "      <td>-0.999351</td>\n",
              "      <td>-0.681300</td>\n",
              "      <td>0.318051</td>\n",
              "      <td>-0.019543</td>\n",
              "      <td>-0.605538</td>\n",
              "      <td>-0.249450</td>\n",
              "      <td>0.356088</td>\n",
              "      <td>0.001984</td>\n",
              "      <td>-0.512543</td>\n",
              "      <td>0.409721</td>\n",
              "      <td>0.922264</td>\n",
              "      <td>-0.002915</td>\n",
              "      <td>0.075837</td>\n",
              "      <td>1.274776</td>\n",
              "      <td>1.198939</td>\n",
              "      <td>0.002151</td>\n",
              "      <td>0.217916</td>\n",
              "      <td>0.350949</td>\n",
              "      <td>0.133032</td>\n",
              "      <td>0.014058</td>\n",
              "      <td>0.331017</td>\n",
              "      <td>0.224792</td>\n",
              "      <td>-0.106225</td>\n",
              "      <td>-0.004729</td>\n",
              "      <td>-0.482276</td>\n",
              "      <td>0.149871</td>\n",
              "      <td>0.632147</td>\n",
              "      <td>-0.029064</td>\n",
              "      <td>-0.884008</td>\n",
              "      <td>-0.740836</td>\n",
              "      <td>0.143172</td>\n",
              "      <td>-0.008013</td>\n",
              "      <td>-0.009869</td>\n",
              "      <td>0.011051</td>\n",
              "      <td>0.020920</td>\n",
              "      <td>-0.001556</td>\n",
              "      <td>0.749459</td>\n",
              "      <td>-0.283267</td>\n",
              "      <td>-1.032727</td>\n",
              "      <td>-0.014077</td>\n",
              "      <td>0.041021</td>\n",
              "      <td>-0.543452</td>\n",
              "      <td>-0.584473</td>\n",
              "      <td>0.000626</td>\n",
              "      <td>-0.003769</td>\n",
              "      <td>0.002321</td>\n",
              "      <td>-0.004178</td>\n",
              "      <td>-0.019689</td>\n",
              "      <td>-0.005612</td>\n",
              "      <td>0.005915</td>\n",
              "      <td>-0.006020</td>\n",
              "      <td>0.002419</td>\n",
              "      <td>-0.009607</td>\n",
              "      <td>0.002067</td>\n",
              "      <td>-0.005116</td>\n",
              "      <td>-0.011324</td>\n",
              "      <td>-0.004637</td>\n",
              "      <td>-0.007803</td>\n",
              "      <td>-0.001802</td>\n",
              "      <td>9</td>\n",
              "      <td>0</td>\n",
              "      <td>8</td>\n",
              "      <td>6</td>\n",
              "      <td>NaN</td>\n",
              "      <td>2.28642</td>\n",
              "      <td>GP01</td>\n",
              "      <td>kaggle.csv</td>\n",
              "      <td>0.578189</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>T</td>\n",
              "      <td>C</td>\n",
              "      <td>95</td>\n",
              "      <td>0.411255</td>\n",
              "      <td>98.41</td>\n",
              "      <td>119.12</td>\n",
              "      <td>121.16</td>\n",
              "      <td>2.04</td>\n",
              "      <td>101.11</td>\n",
              "      <td>103.15</td>\n",
              "      <td>2.04</td>\n",
              "      <td>2.09</td>\n",
              "      <td>1.96</td>\n",
              "      <td>-0.13</td>\n",
              "      <td>9.1</td>\n",
              "      <td>10.28</td>\n",
              "      <td>1.18</td>\n",
              "      <td>NaN</td>\n",
              "      <td>8.18</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.60</td>\n",
              "      <td>5.07</td>\n",
              "      <td>-0.53</td>\n",
              "      <td>-0.05</td>\n",
              "      <td>0.29</td>\n",
              "      <td>0.34</td>\n",
              "      <td>51.3</td>\n",
              "      <td>44.6</td>\n",
              "      <td>-6.7</td>\n",
              "      <td>8.6</td>\n",
              "      <td>5.5</td>\n",
              "      <td>-3.1</td>\n",
              "      <td>0.108</td>\n",
              "      <td>0.128</td>\n",
              "      <td>0.020</td>\n",
              "      <td>1.525</td>\n",
              "      <td>1.461</td>\n",
              "      <td>-0.064</td>\n",
              "      <td>0.003352</td>\n",
              "      <td>-0.036610</td>\n",
              "      <td>-0.039962</td>\n",
              "      <td>A</td>\n",
              "      <td>98.61</td>\n",
              "      <td>L</td>\n",
              "      <td>98.69</td>\n",
              "      <td>-0.828820</td>\n",
              "      <td>328.740432</td>\n",
              "      <td>0.048098</td>\n",
              "      <td>2.922032</td>\n",
              "      <td>2.992943</td>\n",
              "      <td>0.070911</td>\n",
              "      <td>-0.001994</td>\n",
              "      <td>0.084846</td>\n",
              "      <td>0.343217</td>\n",
              "      <td>0.258371</td>\n",
              "      <td>-0.027369</td>\n",
              "      <td>-0.214586</td>\n",
              "      <td>-0.026981</td>\n",
              "      <td>0.187606</td>\n",
              "      <td>-0.003642</td>\n",
              "      <td>0.112732</td>\n",
              "      <td>-0.079620</td>\n",
              "      <td>-0.192352</td>\n",
              "      <td>0.059643</td>\n",
              "      <td>0.439916</td>\n",
              "      <td>0.603866</td>\n",
              "      <td>0.163950</td>\n",
              "      <td>-0.024526</td>\n",
              "      <td>-0.176911</td>\n",
              "      <td>-0.227224</td>\n",
              "      <td>-0.050313</td>\n",
              "      <td>0.039543</td>\n",
              "      <td>-0.382759</td>\n",
              "      <td>-0.420848</td>\n",
              "      <td>-0.038089</td>\n",
              "      <td>-0.004431</td>\n",
              "      <td>0.399533</td>\n",
              "      <td>0.204947</td>\n",
              "      <td>-0.194586</td>\n",
              "      <td>0.017648</td>\n",
              "      <td>-0.557062</td>\n",
              "      <td>-0.308782</td>\n",
              "      <td>0.248280</td>\n",
              "      <td>0.027573</td>\n",
              "      <td>0.309789</td>\n",
              "      <td>0.185098</td>\n",
              "      <td>-0.124691</td>\n",
              "      <td>-0.037391</td>\n",
              "      <td>0.140752</td>\n",
              "      <td>0.234206</td>\n",
              "      <td>0.093454</td>\n",
              "      <td>0.019958</td>\n",
              "      <td>-0.623229</td>\n",
              "      <td>-0.043937</td>\n",
              "      <td>0.579291</td>\n",
              "      <td>0.032968</td>\n",
              "      <td>0.201464</td>\n",
              "      <td>-0.107320</td>\n",
              "      <td>-0.308784</td>\n",
              "      <td>-0.007294</td>\n",
              "      <td>-0.580688</td>\n",
              "      <td>-0.081087</td>\n",
              "      <td>0.499601</td>\n",
              "      <td>0.004310</td>\n",
              "      <td>-0.285651</td>\n",
              "      <td>-0.230316</td>\n",
              "      <td>0.055335</td>\n",
              "      <td>0.020032</td>\n",
              "      <td>-0.323325</td>\n",
              "      <td>-0.097982</td>\n",
              "      <td>0.225342</td>\n",
              "      <td>-0.015060</td>\n",
              "      <td>-0.002613</td>\n",
              "      <td>0.012312</td>\n",
              "      <td>0.008467</td>\n",
              "      <td>0.041541</td>\n",
              "      <td>0.010030</td>\n",
              "      <td>-0.008755</td>\n",
              "      <td>0.016984</td>\n",
              "      <td>-0.021218</td>\n",
              "      <td>-0.022134</td>\n",
              "      <td>0.015226</td>\n",
              "      <td>0.011769</td>\n",
              "      <td>0.020131</td>\n",
              "      <td>-0.004492</td>\n",
              "      <td>-0.007832</td>\n",
              "      <td>-0.015564</td>\n",
              "      <td>16</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>9</td>\n",
              "      <td>NaN</td>\n",
              "      <td>1.48642</td>\n",
              "      <td>GP01</td>\n",
              "      <td>kaggle.csv</td>\n",
              "      <td>0.512346</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>T</td>\n",
              "      <td>C</td>\n",
              "      <td>95</td>\n",
              "      <td>0.411255</td>\n",
              "      <td>98.41</td>\n",
              "      <td>119.12</td>\n",
              "      <td>121.16</td>\n",
              "      <td>2.04</td>\n",
              "      <td>101.11</td>\n",
              "      <td>103.15</td>\n",
              "      <td>2.04</td>\n",
              "      <td>2.09</td>\n",
              "      <td>1.96</td>\n",
              "      <td>-0.13</td>\n",
              "      <td>9.1</td>\n",
              "      <td>10.28</td>\n",
              "      <td>1.18</td>\n",
              "      <td>NaN</td>\n",
              "      <td>8.18</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.60</td>\n",
              "      <td>5.07</td>\n",
              "      <td>-0.53</td>\n",
              "      <td>-0.05</td>\n",
              "      <td>0.29</td>\n",
              "      <td>0.34</td>\n",
              "      <td>51.3</td>\n",
              "      <td>44.6</td>\n",
              "      <td>-6.7</td>\n",
              "      <td>8.6</td>\n",
              "      <td>5.5</td>\n",
              "      <td>-3.1</td>\n",
              "      <td>0.108</td>\n",
              "      <td>0.128</td>\n",
              "      <td>0.020</td>\n",
              "      <td>1.525</td>\n",
              "      <td>1.461</td>\n",
              "      <td>-0.064</td>\n",
              "      <td>0.003352</td>\n",
              "      <td>-0.036610</td>\n",
              "      <td>-0.039962</td>\n",
              "      <td>A</td>\n",
              "      <td>98.61</td>\n",
              "      <td>L</td>\n",
              "      <td>98.69</td>\n",
              "      <td>-0.828820</td>\n",
              "      <td>328.740432</td>\n",
              "      <td>0.048098</td>\n",
              "      <td>2.922032</td>\n",
              "      <td>2.992943</td>\n",
              "      <td>0.070911</td>\n",
              "      <td>-0.001994</td>\n",
              "      <td>0.084846</td>\n",
              "      <td>0.343217</td>\n",
              "      <td>0.258371</td>\n",
              "      <td>-0.027369</td>\n",
              "      <td>-0.214586</td>\n",
              "      <td>-0.026981</td>\n",
              "      <td>0.187606</td>\n",
              "      <td>-0.003642</td>\n",
              "      <td>0.112732</td>\n",
              "      <td>-0.079620</td>\n",
              "      <td>-0.192352</td>\n",
              "      <td>0.059643</td>\n",
              "      <td>0.439916</td>\n",
              "      <td>0.603866</td>\n",
              "      <td>0.163950</td>\n",
              "      <td>-0.024526</td>\n",
              "      <td>-0.176911</td>\n",
              "      <td>-0.227224</td>\n",
              "      <td>-0.050313</td>\n",
              "      <td>0.039543</td>\n",
              "      <td>-0.382759</td>\n",
              "      <td>-0.420848</td>\n",
              "      <td>-0.038089</td>\n",
              "      <td>-0.004431</td>\n",
              "      <td>0.399533</td>\n",
              "      <td>0.204947</td>\n",
              "      <td>-0.194586</td>\n",
              "      <td>0.017648</td>\n",
              "      <td>-0.557062</td>\n",
              "      <td>-0.308782</td>\n",
              "      <td>0.248280</td>\n",
              "      <td>0.027573</td>\n",
              "      <td>0.309789</td>\n",
              "      <td>0.185098</td>\n",
              "      <td>-0.124691</td>\n",
              "      <td>-0.037391</td>\n",
              "      <td>0.140752</td>\n",
              "      <td>0.234206</td>\n",
              "      <td>0.093454</td>\n",
              "      <td>0.019958</td>\n",
              "      <td>-0.623229</td>\n",
              "      <td>-0.043937</td>\n",
              "      <td>0.579291</td>\n",
              "      <td>0.032968</td>\n",
              "      <td>0.201464</td>\n",
              "      <td>-0.107320</td>\n",
              "      <td>-0.308784</td>\n",
              "      <td>-0.007294</td>\n",
              "      <td>-0.580688</td>\n",
              "      <td>-0.081087</td>\n",
              "      <td>0.499601</td>\n",
              "      <td>0.004310</td>\n",
              "      <td>-0.285651</td>\n",
              "      <td>-0.230316</td>\n",
              "      <td>0.055335</td>\n",
              "      <td>0.020032</td>\n",
              "      <td>-0.323325</td>\n",
              "      <td>-0.097982</td>\n",
              "      <td>0.225342</td>\n",
              "      <td>-0.015060</td>\n",
              "      <td>-0.002613</td>\n",
              "      <td>0.012312</td>\n",
              "      <td>0.008467</td>\n",
              "      <td>0.041541</td>\n",
              "      <td>0.010030</td>\n",
              "      <td>-0.008755</td>\n",
              "      <td>0.016984</td>\n",
              "      <td>-0.021218</td>\n",
              "      <td>-0.022134</td>\n",
              "      <td>0.015226</td>\n",
              "      <td>0.011769</td>\n",
              "      <td>0.020131</td>\n",
              "      <td>-0.004492</td>\n",
              "      <td>-0.007832</td>\n",
              "      <td>-0.015564</td>\n",
              "      <td>16</td>\n",
              "      <td>1</td>\n",
              "      <td>0</td>\n",
              "      <td>9</td>\n",
              "      <td>NaN</td>\n",
              "      <td>0.28642</td>\n",
              "      <td>GP01</td>\n",
              "      <td>kaggle.csv</td>\n",
              "      <td>0.425926</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>T</td>\n",
              "      <td>S</td>\n",
              "      <td>95</td>\n",
              "      <td>0.411255</td>\n",
              "      <td>98.41</td>\n",
              "      <td>119.12</td>\n",
              "      <td>105.09</td>\n",
              "      <td>-14.03</td>\n",
              "      <td>101.11</td>\n",
              "      <td>87.08</td>\n",
              "      <td>-14.03</td>\n",
              "      <td>2.09</td>\n",
              "      <td>2.21</td>\n",
              "      <td>0.12</td>\n",
              "      <td>9.1</td>\n",
              "      <td>9.15</td>\n",
              "      <td>0.05</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.60</td>\n",
              "      <td>5.68</td>\n",
              "      <td>0.08</td>\n",
              "      <td>-0.05</td>\n",
              "      <td>-0.18</td>\n",
              "      <td>-0.13</td>\n",
              "      <td>51.3</td>\n",
              "      <td>29.3</td>\n",
              "      <td>-22.0</td>\n",
              "      <td>8.6</td>\n",
              "      <td>9.2</td>\n",
              "      <td>0.6</td>\n",
              "      <td>0.108</td>\n",
              "      <td>0.062</td>\n",
              "      <td>-0.046</td>\n",
              "      <td>1.525</td>\n",
              "      <td>1.298</td>\n",
              "      <td>-0.227</td>\n",
              "      <td>0.003352</td>\n",
              "      <td>0.004627</td>\n",
              "      <td>0.001275</td>\n",
              "      <td>A</td>\n",
              "      <td>98.61</td>\n",
              "      <td>L</td>\n",
              "      <td>98.69</td>\n",
              "      <td>-0.828820</td>\n",
              "      <td>328.740432</td>\n",
              "      <td>0.021657</td>\n",
              "      <td>2.922032</td>\n",
              "      <td>2.949984</td>\n",
              "      <td>0.027951</td>\n",
              "      <td>-0.005534</td>\n",
              "      <td>0.084846</td>\n",
              "      <td>0.028643</td>\n",
              "      <td>-0.056203</td>\n",
              "      <td>-0.007399</td>\n",
              "      <td>-0.214586</td>\n",
              "      <td>-0.112081</td>\n",
              "      <td>0.102506</td>\n",
              "      <td>0.003737</td>\n",
              "      <td>0.112732</td>\n",
              "      <td>0.140168</td>\n",
              "      <td>0.027436</td>\n",
              "      <td>0.012031</td>\n",
              "      <td>0.439916</td>\n",
              "      <td>0.472333</td>\n",
              "      <td>0.032416</td>\n",
              "      <td>-0.003811</td>\n",
              "      <td>-0.176911</td>\n",
              "      <td>-0.100895</td>\n",
              "      <td>0.076016</td>\n",
              "      <td>0.003911</td>\n",
              "      <td>-0.382759</td>\n",
              "      <td>-0.426468</td>\n",
              "      <td>-0.043709</td>\n",
              "      <td>0.011039</td>\n",
              "      <td>0.399533</td>\n",
              "      <td>0.330352</td>\n",
              "      <td>-0.069180</td>\n",
              "      <td>0.012612</td>\n",
              "      <td>-0.557062</td>\n",
              "      <td>-0.326145</td>\n",
              "      <td>0.230917</td>\n",
              "      <td>0.004261</td>\n",
              "      <td>0.309789</td>\n",
              "      <td>0.462552</td>\n",
              "      <td>0.152764</td>\n",
              "      <td>0.005646</td>\n",
              "      <td>0.140752</td>\n",
              "      <td>0.189888</td>\n",
              "      <td>0.049135</td>\n",
              "      <td>-0.003867</td>\n",
              "      <td>-0.623229</td>\n",
              "      <td>-0.267794</td>\n",
              "      <td>0.355435</td>\n",
              "      <td>0.003348</td>\n",
              "      <td>0.201464</td>\n",
              "      <td>-0.110686</td>\n",
              "      <td>-0.312150</td>\n",
              "      <td>0.003065</td>\n",
              "      <td>-0.580688</td>\n",
              "      <td>-0.348110</td>\n",
              "      <td>0.232577</td>\n",
              "      <td>0.009396</td>\n",
              "      <td>-0.285651</td>\n",
              "      <td>-0.328500</td>\n",
              "      <td>-0.042849</td>\n",
              "      <td>0.002500</td>\n",
              "      <td>-0.323325</td>\n",
              "      <td>-0.273896</td>\n",
              "      <td>0.049428</td>\n",
              "      <td>-0.010602</td>\n",
              "      <td>-0.006238</td>\n",
              "      <td>-0.003419</td>\n",
              "      <td>-0.003106</td>\n",
              "      <td>0.000572</td>\n",
              "      <td>0.000522</td>\n",
              "      <td>0.006822</td>\n",
              "      <td>0.004235</td>\n",
              "      <td>-0.007185</td>\n",
              "      <td>0.001336</td>\n",
              "      <td>0.006135</td>\n",
              "      <td>-0.006449</td>\n",
              "      <td>-0.000727</td>\n",
              "      <td>0.006524</td>\n",
              "      <td>0.011033</td>\n",
              "      <td>0.015508</td>\n",
              "      <td>16</td>\n",
              "      <td>15</td>\n",
              "      <td>0</td>\n",
              "      <td>9</td>\n",
              "      <td>NaN</td>\n",
              "      <td>2.48642</td>\n",
              "      <td>GP01</td>\n",
              "      <td>kaggle.csv</td>\n",
              "      <td>0.592593</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>T</td>\n",
              "      <td>S</td>\n",
              "      <td>95</td>\n",
              "      <td>0.411255</td>\n",
              "      <td>98.41</td>\n",
              "      <td>119.12</td>\n",
              "      <td>105.09</td>\n",
              "      <td>-14.03</td>\n",
              "      <td>101.11</td>\n",
              "      <td>87.08</td>\n",
              "      <td>-14.03</td>\n",
              "      <td>2.09</td>\n",
              "      <td>2.21</td>\n",
              "      <td>0.12</td>\n",
              "      <td>9.1</td>\n",
              "      <td>9.15</td>\n",
              "      <td>0.05</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.60</td>\n",
              "      <td>5.68</td>\n",
              "      <td>0.08</td>\n",
              "      <td>-0.05</td>\n",
              "      <td>-0.18</td>\n",
              "      <td>-0.13</td>\n",
              "      <td>51.3</td>\n",
              "      <td>29.3</td>\n",
              "      <td>-22.0</td>\n",
              "      <td>8.6</td>\n",
              "      <td>9.2</td>\n",
              "      <td>0.6</td>\n",
              "      <td>0.108</td>\n",
              "      <td>0.062</td>\n",
              "      <td>-0.046</td>\n",
              "      <td>1.525</td>\n",
              "      <td>1.298</td>\n",
              "      <td>-0.227</td>\n",
              "      <td>0.003352</td>\n",
              "      <td>0.004627</td>\n",
              "      <td>0.001275</td>\n",
              "      <td>A</td>\n",
              "      <td>98.61</td>\n",
              "      <td>L</td>\n",
              "      <td>98.69</td>\n",
              "      <td>-0.828820</td>\n",
              "      <td>328.740432</td>\n",
              "      <td>0.021657</td>\n",
              "      <td>2.922032</td>\n",
              "      <td>2.949984</td>\n",
              "      <td>0.027951</td>\n",
              "      <td>-0.005534</td>\n",
              "      <td>0.084846</td>\n",
              "      <td>0.028643</td>\n",
              "      <td>-0.056203</td>\n",
              "      <td>-0.007399</td>\n",
              "      <td>-0.214586</td>\n",
              "      <td>-0.112081</td>\n",
              "      <td>0.102506</td>\n",
              "      <td>0.003737</td>\n",
              "      <td>0.112732</td>\n",
              "      <td>0.140168</td>\n",
              "      <td>0.027436</td>\n",
              "      <td>0.012031</td>\n",
              "      <td>0.439916</td>\n",
              "      <td>0.472333</td>\n",
              "      <td>0.032416</td>\n",
              "      <td>-0.003811</td>\n",
              "      <td>-0.176911</td>\n",
              "      <td>-0.100895</td>\n",
              "      <td>0.076016</td>\n",
              "      <td>0.003911</td>\n",
              "      <td>-0.382759</td>\n",
              "      <td>-0.426468</td>\n",
              "      <td>-0.043709</td>\n",
              "      <td>0.011039</td>\n",
              "      <td>0.399533</td>\n",
              "      <td>0.330352</td>\n",
              "      <td>-0.069180</td>\n",
              "      <td>0.012612</td>\n",
              "      <td>-0.557062</td>\n",
              "      <td>-0.326145</td>\n",
              "      <td>0.230917</td>\n",
              "      <td>0.004261</td>\n",
              "      <td>0.309789</td>\n",
              "      <td>0.462552</td>\n",
              "      <td>0.152764</td>\n",
              "      <td>0.005646</td>\n",
              "      <td>0.140752</td>\n",
              "      <td>0.189888</td>\n",
              "      <td>0.049135</td>\n",
              "      <td>-0.003867</td>\n",
              "      <td>-0.623229</td>\n",
              "      <td>-0.267794</td>\n",
              "      <td>0.355435</td>\n",
              "      <td>0.003348</td>\n",
              "      <td>0.201464</td>\n",
              "      <td>-0.110686</td>\n",
              "      <td>-0.312150</td>\n",
              "      <td>0.003065</td>\n",
              "      <td>-0.580688</td>\n",
              "      <td>-0.348110</td>\n",
              "      <td>0.232577</td>\n",
              "      <td>0.009396</td>\n",
              "      <td>-0.285651</td>\n",
              "      <td>-0.328500</td>\n",
              "      <td>-0.042849</td>\n",
              "      <td>0.002500</td>\n",
              "      <td>-0.323325</td>\n",
              "      <td>-0.273896</td>\n",
              "      <td>0.049428</td>\n",
              "      <td>-0.010602</td>\n",
              "      <td>-0.006238</td>\n",
              "      <td>-0.003419</td>\n",
              "      <td>-0.003106</td>\n",
              "      <td>0.000572</td>\n",
              "      <td>0.000522</td>\n",
              "      <td>0.006822</td>\n",
              "      <td>0.004235</td>\n",
              "      <td>-0.007185</td>\n",
              "      <td>0.001336</td>\n",
              "      <td>0.006135</td>\n",
              "      <td>-0.006449</td>\n",
              "      <td>-0.000727</td>\n",
              "      <td>0.006524</td>\n",
              "      <td>0.011033</td>\n",
              "      <td>0.015508</td>\n",
              "      <td>16</td>\n",
              "      <td>15</td>\n",
              "      <td>0</td>\n",
              "      <td>9</td>\n",
              "      <td>NaN</td>\n",
              "      <td>3.88642</td>\n",
              "      <td>GP01</td>\n",
              "      <td>kaggle.csv</td>\n",
              "      <td>0.769547</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "  WT MUT  position  relative_position  b_factor  Molecular Weight_1  \\\n",
              "0  L   A        89           0.385281     95.07              131.18   \n",
              "1  T   C        95           0.411255     98.41              119.12   \n",
              "2  T   C        95           0.411255     98.41              119.12   \n",
              "3  T   S        95           0.411255     98.41              119.12   \n",
              "4  T   S        95           0.411255     98.41              119.12   \n",
              "\n",
              "   Molecular Weight_2  Molecular Weight_delta  Residue Weight_1  \\\n",
              "0               89.10                  -42.08            113.16   \n",
              "1              121.16                    2.04            101.11   \n",
              "2              121.16                    2.04            101.11   \n",
              "3              105.09                  -14.03            101.11   \n",
              "4              105.09                  -14.03            101.11   \n",
              "\n",
              "   Residue Weight_2  Residue Weight_delta  pKa1_1  pKa1_2  pKa1_delta  pKb2_1  \\\n",
              "0             71.08                -42.08    2.36    2.34       -0.02     9.6   \n",
              "1            103.15                  2.04    2.09    1.96       -0.13     9.1   \n",
              "2            103.15                  2.04    2.09    1.96       -0.13     9.1   \n",
              "3             87.08                -14.03    2.09    2.21        0.12     9.1   \n",
              "4             87.08                -14.03    2.09    2.21        0.12     9.1   \n",
              "\n",
              "   pKb2_2  pKb2_delta  pKx3_1  pKx3_2  pKx3_delta  pl4_1  pl4_2  pl4_delta  \\\n",
              "0    9.69        0.09     NaN     NaN         NaN   5.98   6.00       0.02   \n",
              "1   10.28        1.18     NaN    8.18         NaN   5.60   5.07      -0.53   \n",
              "2   10.28        1.18     NaN    8.18         NaN   5.60   5.07      -0.53   \n",
              "3    9.15        0.05     NaN     NaN         NaN   5.60   5.68       0.08   \n",
              "4    9.15        0.05     NaN     NaN         NaN   5.60   5.68       0.08   \n",
              "\n",
              "    H_1   H_2  H_delta  VSC_1  VSC_2  VSC_delta  P1_1  P1_2  P1_delta   P2_1  \\\n",
              "0  1.06  0.62    -0.44   93.5   27.5      -66.0   4.9   8.1       3.2  0.186   \n",
              "1 -0.05  0.29     0.34   51.3   44.6       -6.7   8.6   5.5      -3.1  0.108   \n",
              "2 -0.05  0.29     0.34   51.3   44.6       -6.7   8.6   5.5      -3.1  0.108   \n",
              "3 -0.05 -0.18    -0.13   51.3   29.3      -22.0   8.6   9.2       0.6  0.108   \n",
              "4 -0.05 -0.18    -0.13   51.3   29.3      -22.0   8.6   9.2       0.6  0.108   \n",
              "\n",
              "    P2_2  P2_delta  SASA_1  SASA_2  SASA_delta   NCISC_1   NCISC_2  \\\n",
              "0  0.046    -0.140   1.931   1.181      -0.750  0.051672  0.007187   \n",
              "1  0.128     0.020   1.525   1.461      -0.064  0.003352 -0.036610   \n",
              "2  0.128     0.020   1.525   1.461      -0.064  0.003352 -0.036610   \n",
              "3  0.062    -0.046   1.525   1.298      -0.227  0.003352  0.004627   \n",
              "4  0.062    -0.046   1.525   1.298      -0.227  0.003352  0.004627   \n",
              "\n",
              "   NCISC_delta prev  b_factor_prev post  b_factor_post  cos_angle  location3d  \\\n",
              "0    -0.044485    K          88.59    H          95.97  -0.663945  137.210421   \n",
              "1    -0.039962    A          98.61    L          98.69  -0.828820  328.740432   \n",
              "2    -0.039962    A          98.61    L          98.69  -0.828820  328.740432   \n",
              "3     0.001275    A          98.61    L          98.69  -0.828820  328.740432   \n",
              "4     0.001275    A          98.61    L          98.69  -0.828820  328.740432   \n",
              "\n",
              "   pca_pool_0  pca_wt_0  pca_mutant_0  pca_local_0  pca_pool_1  pca_wt_1  \\\n",
              "0   -0.014211  3.765163      3.986665     0.221501   -0.007354  0.923303   \n",
              "1    0.048098  2.922032      2.992943     0.070911   -0.001994  0.084846   \n",
              "2    0.048098  2.922032      2.992943     0.070911   -0.001994  0.084846   \n",
              "3    0.021657  2.922032      2.949984     0.027951   -0.005534  0.084846   \n",
              "4    0.021657  2.922032      2.949984     0.027951   -0.005534  0.084846   \n",
              "\n",
              "   pca_mutant_1  pca_local_1  pca_pool_2  pca_wt_2  pca_mutant_2  pca_local_2  \\\n",
              "0      0.430142    -0.493161    0.015841 -0.535758     -0.248253     0.287504   \n",
              "1      0.343217     0.258371   -0.027369 -0.214586     -0.026981     0.187606   \n",
              "2      0.343217     0.258371   -0.027369 -0.214586     -0.026981     0.187606   \n",
              "3      0.028643    -0.056203   -0.007399 -0.214586     -0.112081     0.102506   \n",
              "4      0.028643    -0.056203   -0.007399 -0.214586     -0.112081     0.102506   \n",
              "\n",
              "   pca_pool_3  pca_wt_3  pca_mutant_3  pca_local_3  pca_pool_4  pca_wt_4  \\\n",
              "0    0.014395  1.357306      1.130861    -0.226445   -0.021402 -0.888456   \n",
              "1   -0.003642  0.112732     -0.079620    -0.192352    0.059643  0.439916   \n",
              "2   -0.003642  0.112732     -0.079620    -0.192352    0.059643  0.439916   \n",
              "3    0.003737  0.112732      0.140168     0.027436    0.012031  0.439916   \n",
              "4    0.003737  0.112732      0.140168     0.027436    0.012031  0.439916   \n",
              "\n",
              "   pca_mutant_4  pca_local_4  pca_pool_5  pca_wt_5  pca_mutant_5  pca_local_5  \\\n",
              "0     -1.839115    -0.950659    0.014413 -0.999351     -0.681300     0.318051   \n",
              "1      0.603866     0.163950   -0.024526 -0.176911     -0.227224    -0.050313   \n",
              "2      0.603866     0.163950   -0.024526 -0.176911     -0.227224    -0.050313   \n",
              "3      0.472333     0.032416   -0.003811 -0.176911     -0.100895     0.076016   \n",
              "4      0.472333     0.032416   -0.003811 -0.176911     -0.100895     0.076016   \n",
              "\n",
              "   pca_pool_6  pca_wt_6  pca_mutant_6  pca_local_6  pca_pool_7  pca_wt_7  \\\n",
              "0   -0.019543 -0.605538     -0.249450     0.356088    0.001984 -0.512543   \n",
              "1    0.039543 -0.382759     -0.420848    -0.038089   -0.004431  0.399533   \n",
              "2    0.039543 -0.382759     -0.420848    -0.038089   -0.004431  0.399533   \n",
              "3    0.003911 -0.382759     -0.426468    -0.043709    0.011039  0.399533   \n",
              "4    0.003911 -0.382759     -0.426468    -0.043709    0.011039  0.399533   \n",
              "\n",
              "   pca_mutant_7  pca_local_7  pca_pool_8  pca_wt_8  pca_mutant_8  pca_local_8  \\\n",
              "0      0.409721     0.922264   -0.002915  0.075837      1.274776     1.198939   \n",
              "1      0.204947    -0.194586    0.017648 -0.557062     -0.308782     0.248280   \n",
              "2      0.204947    -0.194586    0.017648 -0.557062     -0.308782     0.248280   \n",
              "3      0.330352    -0.069180    0.012612 -0.557062     -0.326145     0.230917   \n",
              "4      0.330352    -0.069180    0.012612 -0.557062     -0.326145     0.230917   \n",
              "\n",
              "   pca_pool_9  pca_wt_9  pca_mutant_9  pca_local_9  pca_pool_10  pca_wt_10  \\\n",
              "0    0.002151  0.217916      0.350949     0.133032     0.014058   0.331017   \n",
              "1    0.027573  0.309789      0.185098    -0.124691    -0.037391   0.140752   \n",
              "2    0.027573  0.309789      0.185098    -0.124691    -0.037391   0.140752   \n",
              "3    0.004261  0.309789      0.462552     0.152764     0.005646   0.140752   \n",
              "4    0.004261  0.309789      0.462552     0.152764     0.005646   0.140752   \n",
              "\n",
              "   pca_mutant_10  pca_local_10  pca_pool_11  pca_wt_11  pca_mutant_11  \\\n",
              "0       0.224792     -0.106225    -0.004729  -0.482276       0.149871   \n",
              "1       0.234206      0.093454     0.019958  -0.623229      -0.043937   \n",
              "2       0.234206      0.093454     0.019958  -0.623229      -0.043937   \n",
              "3       0.189888      0.049135    -0.003867  -0.623229      -0.267794   \n",
              "4       0.189888      0.049135    -0.003867  -0.623229      -0.267794   \n",
              "\n",
              "   pca_local_11  pca_pool_12  pca_wt_12  pca_mutant_12  pca_local_12  \\\n",
              "0      0.632147    -0.029064  -0.884008      -0.740836      0.143172   \n",
              "1      0.579291     0.032968   0.201464      -0.107320     -0.308784   \n",
              "2      0.579291     0.032968   0.201464      -0.107320     -0.308784   \n",
              "3      0.355435     0.003348   0.201464      -0.110686     -0.312150   \n",
              "4      0.355435     0.003348   0.201464      -0.110686     -0.312150   \n",
              "\n",
              "   pca_pool_13  pca_wt_13  pca_mutant_13  pca_local_13  pca_pool_14  \\\n",
              "0    -0.008013  -0.009869       0.011051      0.020920    -0.001556   \n",
              "1    -0.007294  -0.580688      -0.081087      0.499601     0.004310   \n",
              "2    -0.007294  -0.580688      -0.081087      0.499601     0.004310   \n",
              "3     0.003065  -0.580688      -0.348110      0.232577     0.009396   \n",
              "4     0.003065  -0.580688      -0.348110      0.232577     0.009396   \n",
              "\n",
              "   pca_wt_14  pca_mutant_14  pca_local_14  pca_pool_15  pca_wt_15  \\\n",
              "0   0.749459      -0.283267     -1.032727    -0.014077   0.041021   \n",
              "1  -0.285651      -0.230316      0.055335     0.020032  -0.323325   \n",
              "2  -0.285651      -0.230316      0.055335     0.020032  -0.323325   \n",
              "3  -0.285651      -0.328500     -0.042849     0.002500  -0.323325   \n",
              "4  -0.285651      -0.328500     -0.042849     0.002500  -0.323325   \n",
              "\n",
              "   pca_mutant_15  pca_local_15  pca_pool_16  pca_pool_17  pca_pool_18  \\\n",
              "0      -0.543452     -0.584473     0.000626    -0.003769     0.002321   \n",
              "1      -0.097982      0.225342    -0.015060    -0.002613     0.012312   \n",
              "2      -0.097982      0.225342    -0.015060    -0.002613     0.012312   \n",
              "3      -0.273896      0.049428    -0.010602    -0.006238    -0.003419   \n",
              "4      -0.273896      0.049428    -0.010602    -0.006238    -0.003419   \n",
              "\n",
              "   pca_pool_19  pca_pool_20  pca_pool_21  pca_pool_22  pca_pool_23  \\\n",
              "0    -0.004178    -0.019689    -0.005612     0.005915    -0.006020   \n",
              "1     0.008467     0.041541     0.010030    -0.008755     0.016984   \n",
              "2     0.008467     0.041541     0.010030    -0.008755     0.016984   \n",
              "3    -0.003106     0.000572     0.000522     0.006822     0.004235   \n",
              "4    -0.003106     0.000572     0.000522     0.006822     0.004235   \n",
              "\n",
              "   pca_pool_24  pca_pool_25  pca_pool_26  pca_pool_27  pca_pool_28  \\\n",
              "0     0.002419    -0.009607     0.002067    -0.005116    -0.011324   \n",
              "1    -0.021218    -0.022134     0.015226     0.011769     0.020131   \n",
              "2    -0.021218    -0.022134     0.015226     0.011769     0.020131   \n",
              "3    -0.007185     0.001336     0.006135    -0.006449    -0.000727   \n",
              "4    -0.007185     0.001336     0.006135    -0.006449    -0.000727   \n",
              "\n",
              "   pca_pool_29  pca_pool_30  pca_pool_31  AA1  AA2  AA3  AA4  ddG      dTm  \\\n",
              "0    -0.004637    -0.007803    -0.001802    9    0    8    6  NaN  2.28642   \n",
              "1    -0.004492    -0.007832    -0.015564   16    1    0    9  NaN  1.48642   \n",
              "2    -0.004492    -0.007832    -0.015564   16    1    0    9  NaN  0.28642   \n",
              "3     0.006524     0.011033     0.015508   16   15    0    9  NaN  2.48642   \n",
              "4     0.006524     0.011033     0.015508   16   15    0    9  NaN  3.88642   \n",
              "\n",
              "    pdb      source    target  group  \n",
              "0  GP01  kaggle.csv  0.578189      0  \n",
              "1  GP01  kaggle.csv  0.512346      0  \n",
              "2  GP01  kaggle.csv  0.425926      0  \n",
              "3  GP01  kaggle.csv  0.592593      0  \n",
              "4  GP01  kaggle.csv  0.769547      0  "
            ]
          },
          "execution_count": 21,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "train.head()"
      ]
    },
    {
      "cell_type": "markdown",
      "id": "a5faa3b6",
      "metadata": {
        "papermill": {
          "duration": 0.025882,
          "end_time": "2022-12-08T02:30:08.863580",
          "exception": false,
          "start_time": "2022-12-08T02:30:08.837698",
          "status": "completed"
        },
        "tags": [],
        "id": "a5faa3b6"
      },
      "source": [
        "# LGBM Model"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "fbfd1822",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:30:08.918620Z",
          "iopub.status.busy": "2022-12-08T02:30:08.917628Z",
          "iopub.status.idle": "2022-12-08T02:30:09.592643Z",
          "shell.execute_reply": "2022-12-08T02:30:09.591621Z"
        },
        "papermill": {
          "duration": 0.705689,
          "end_time": "2022-12-08T02:30:09.595462",
          "exception": false,
          "start_time": "2022-12-08T02:30:08.889773",
          "status": "completed"
        },
        "tags": [],
        "id": "fbfd1822",
        "outputId": "c54fe7eb-78ab-4fc9-b435-25086178ecc2"
      },
      "outputs": [
        {
          "data": {
            "text/html": [
              "<style type='text/css'>\n",
              ".datatable table.frame { margin-bottom: 0; }\n",
              ".datatable table.frame thead { border-bottom: none; }\n",
              ".datatable table.frame tr.coltypes td {  color: #FFFFFF;  line-height: 6px;  padding: 0 0.5em;}\n",
              ".datatable .bool    { background: #DDDD99; }\n",
              ".datatable .object  { background: #565656; }\n",
              ".datatable .int     { background: #5D9E5D; }\n",
              ".datatable .float   { background: #4040CC; }\n",
              ".datatable .str     { background: #CC4040; }\n",
              ".datatable .time    { background: #40CC40; }\n",
              ".datatable .row_index {  background: var(--jp-border-color3);  border-right: 1px solid var(--jp-border-color0);  color: var(--jp-ui-font-color3);  font-size: 9px;}\n",
              ".datatable .frame tbody td { text-align: left; }\n",
              ".datatable .frame tr.coltypes .row_index {  background: var(--jp-border-color0);}\n",
              ".datatable th:nth-child(2) { padding-left: 12px; }\n",
              ".datatable .hellipsis {  color: var(--jp-cell-editor-border-color);}\n",
              ".datatable .vellipsis {  background: var(--jp-layout-color0);  color: var(--jp-cell-editor-border-color);}\n",
              ".datatable .na {  color: var(--jp-cell-editor-border-color);  font-size: 80%;}\n",
              ".datatable .sp {  opacity: 0.25;}\n",
              ".datatable .footer { font-size: 9px; }\n",
              ".datatable .frame_dimensions {  background: var(--jp-border-color3);  border-top: 1px solid var(--jp-border-color0);  color: var(--jp-ui-font-color3);  display: inline-block;  opacity: 0.6;  padding: 1px 10px 1px 5px;}\n",
              ".datatable .frame thead tr.colnames {  background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABwAAAA4CAYAAADuMJi0AAAGR0lEQVR42rVZ21IbRxBtCbQrkIR2dQVjsLmDLBsET3nTQ8ouYRkQVf6e/E9+Im958qMfkgoXAaKSSj6C9Jnd2R2NeiRSRaZqitVOT5+Z6dNnWoKGlN94JFp8Ipofkb/7SOXjGyp8wF+z35K3f0uUp/GW4XfLQ8v2gefj3ZCCzojoNfue+43o1Q3l3xB/yA3JO7jnF2pCLnI+pNyx/qw7L+SQ7T2N9p2f8c60QcfcK6KGXsAd+ZvA4LlZYuSSAoOhMs5vwJkEGDlbPMaJoA+FcQ0IH38QLWkbAFLkOOhoMF5tU6/eBRhNjro0ZgKiPRAt3FLhCO/vqdgmNTm32LkmKpvBmQY4q5uAaAgbwDBG2BVv3bfI8KKAMWj2kfw9+pkZREIbEql4ST1x7hgHIANkbJ//MF8mAH/ilTCJ2tIi4ASr1IC3VNqXHKOxjy34mgoImnOQtx1g81fkqTiMOBVGcTogNhiT5iBHET8R8C+iApJUmgim3SQAXhsLQz7ee2G8gOAQNtJckBEplADiAxtX+G9NmhDl0qJKnTvyWlAMPYZnvIviGXRg6/Dh824DBXhP/tbfREXJEIvQ+aaPGjG7pvw6r3xdx+9hqb4dgZaP2XmdHO2K/B0c1+oUph6k8kShBryl/Ft0DYgjTlOieOACHFFpVyUl72T9V3cM1jUoYvxIC2vpCSys/ck70mDYuYvdvKjlMdKAUThneWVU1aAsyjv6PURDiwNsHGBZzY+JtAAgE2TFxdRHJdyIp/f+zqu09M5cDP2F08Ukkpj4YNSdX950HY2pNCCUK/Hhx5ZMBfjNSEzdsIihVzzAMdn9dz4eDYhnyQb9SSCiAryiJcQk82LiTbJ4x2FZJaUenpKnzP95WyDf4Y+QN9EFHHSeDLGdBjjKNQ5vKHf4XMA7KrY0y0GEObBOO/8e1ywuQExOHXktuQyJALEBpcEqhwtHqgiDuCK5b6i0p2MQpcckIIoh+6hYgTZtO8xlMi6O4tKCF/kOGHEg/W0UUpHW0ZoGNZ1ExZWcn7EErgwt4uj50E/sFBjXXIayWvh7WryjasxarZKssXon0zxvvkc32Q0bqbBCuZiKt9dWFysfQefeL29JYFaeztX6tePaZdz5mYx8+6Zq3Mk0wXECQxlhdzgS2wjBHju3j1RIgKyOMdNUE8X0+RAdbSapS11MRCv1SzUXmO6wGZe2SQYrv2MvCSWEv2VODE6DN7bz8ufypgQKW7uQskFTQHULLKyaEyrnlZbgOGLrV5qrn9U79jjm2HJmgkaVN98AfBub91lGPLZBqdroN5LYgjSu4zYZDDHXZOIPC691HqrWI1900I8qLzgKP4ft8DxEWigprPfrO+KcXno9gZz4jjGewWdUcpGCj0qVFuGPYbl2VturndZ2qRvlL8acDO6lF/DY/VjsFesiUK+ypJ+r/ep+cJkSQxEK4PG4WozgA75TYrDDqStE69K8/mzGEM+JXTeqvmedEElMmwCMm2SLd6bNNF9su02zEtoW6nAQtpMj5Gd7fKa//wqonF7UdtHFsVn+6hf1o7AfriPH7M6EeIUEF5zKVxXbYo7kS/OEtOqDYZKPoBsETIixn0uYrasThmzDkhdKPkz2EnaX0HdQbIgr59vAdGYDqjHrxkjS7WOxkTD8sqEqhiwcJETgBYigrBqF08KyDaje9SZ/I1A7MzaTzMGDEulPtZUkuKcyIRAjxEJPVrnVlb/9wkfij31D/pQt1IN+iL8bGJcstBIO7Y5VI/cwDqURbXhMuJxBqD0KLoK3esWFs0Jz5i5ZvJUAfFJMFb9XmGIOnzGpijpcWYCaMqXSQWp8EnCABepQ0Elyi4wfKfsw78ikIqif1pe1AGPlLmojl1SKxHHXp1L+Ut7AmDQHvhI5xHGi4EooO2BR7k78PEkJOdL7cAxQUZ/Tyclu9gnfwGgOmm2lNHGNmZXsq4Pqgc1EG1ATrvKl8s4R9ywwnqulGUnaRLVhxy8v3ieUwy2hbooT68uscW++DCDH0WSzuoyN2D4LUJ/tLECbcSKznwMIFs0ChF4mRTCnQbIIfk4SHJo6A9BMuTnXTs3Ku/KxsgZWqzuSe+Os8cEUfnMBY6UF5gi3SUbd5K7vDjq5WW0UENJlRsWn4sy21Er/E/AvPQSFHy1p4fgAAAAASUVORK5CYII=');  background-repeat: repeat-x;  background-size: 14px;  height: 28px;}\n",
              "</style>\n"
            ],
            "text/plain": [
              "<IPython.core.display.HTML object>"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        }
      ],
      "source": [
        "# LOAD XGB LIBRARY\n",
        "from sklearn.model_selection import GroupKFold\n",
        "from sklearn.metrics import mean_squared_error\n",
        "import lightgbm as lgb\n",
        "\n",
        "FOLDS = 10\n",
        "SEED = 123\n",
        "\n",
        "# MODEL PARAMETERS\n",
        "params = {'learning_rate': 0.001, \n",
        "              'max_depth': 4, \n",
        "              'boosting': 'gbdt', \n",
        "              'objective': 'regression', \n",
        "              'metric': 'rmse', \n",
        "              'is_training_metric': True, \n",
        "              'num_leaves': 144, \n",
        "              'feature_fraction': 0.9, \n",
        "              'bagging_fraction': 0.7, \n",
        "              'bagging_freq': 5, \n",
        "              'seed':2018}"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "a83ecf03",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:30:09.652392Z",
          "iopub.status.busy": "2022-12-08T02:30:09.651492Z",
          "iopub.status.idle": "2022-12-08T02:30:19.559514Z",
          "shell.execute_reply": "2022-12-08T02:30:19.558270Z"
        },
        "papermill": {
          "duration": 9.939808,
          "end_time": "2022-12-08T02:30:19.561954",
          "exception": false,
          "start_time": "2022-12-08T02:30:09.622146",
          "status": "completed"
        },
        "tags": [],
        "id": "a83ecf03",
        "outputId": "51d97cc2-dc88-4ae7-a49b-0ee590cd38fb"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Requirement already satisfied: joblib in /opt/conda/lib/python3.7/site-packages (1.0.1)\r\n",
            "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\r\n",
            "\u001b[0m"
          ]
        }
      ],
      "source": [
        "!pip install joblib"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "c568c5eb",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:30:19.617803Z",
          "iopub.status.busy": "2022-12-08T02:30:19.617421Z",
          "iopub.status.idle": "2022-12-08T02:30:19.622944Z",
          "shell.execute_reply": "2022-12-08T02:30:19.621833Z"
        },
        "papermill": {
          "duration": 0.035706,
          "end_time": "2022-12-08T02:30:19.624949",
          "exception": false,
          "start_time": "2022-12-08T02:30:19.589243",
          "status": "completed"
        },
        "tags": [],
        "id": "c568c5eb"
      },
      "outputs": [],
      "source": [
        "from joblib import dump, load"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "265d038d",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:30:19.682948Z",
          "iopub.status.busy": "2022-12-08T02:30:19.682579Z",
          "iopub.status.idle": "2022-12-08T02:30:19.687032Z",
          "shell.execute_reply": "2022-12-08T02:30:19.686031Z"
        },
        "papermill": {
          "duration": 0.035554,
          "end_time": "2022-12-08T02:30:19.689035",
          "exception": false,
          "start_time": "2022-12-08T02:30:19.653481",
          "status": "completed"
        },
        "tags": [],
        "id": "265d038d"
      },
      "outputs": [],
      "source": [
        "import joblib"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "df127182",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:30:19.746060Z",
          "iopub.status.busy": "2022-12-08T02:30:19.745696Z",
          "iopub.status.idle": "2022-12-08T02:30:26.166186Z",
          "shell.execute_reply": "2022-12-08T02:30:26.164626Z"
        },
        "papermill": {
          "duration": 6.452214,
          "end_time": "2022-12-08T02:30:26.168254",
          "exception": false,
          "start_time": "2022-12-08T02:30:19.716040",
          "status": "completed"
        },
        "tags": [],
        "id": "df127182",
        "outputId": "2b51f45b-4e22-430b-ffdc-fbeb2cfbf0a8"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "#########################\n",
            "### Fold 1\n",
            "### Train size 1813 Valid size 243\n",
            "#########################\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Warning] Auto-choosing col-wise multi-threading, the overhead of testing was 0.004164 seconds.\n",
            "You can set `force_col_wise=true` to remove the overhead.\n",
            "[LightGBM] [Info] Total Bins 24390\n",
            "[LightGBM] [Info] Number of data points in the train set: 1813, number of used features: 127\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Info] Start training from score 0.508274\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[1]\ttraining's rmse: 0.282311\tvalid_1's rmse: 0.288694\n",
            "Training until validation scores don't improve for 10 rounds\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[2]\ttraining's rmse: 0.282262\tvalid_1's rmse: 0.288645\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[3]\ttraining's rmse: 0.282216\tvalid_1's rmse: 0.2886\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[4]\ttraining's rmse: 0.282173\tvalid_1's rmse: 0.288585\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[5]\ttraining's rmse: 0.282127\tvalid_1's rmse: 0.288543\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[6]\ttraining's rmse: 0.282089\tvalid_1's rmse: 0.288504\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[7]\ttraining's rmse: 0.282052\tvalid_1's rmse: 0.288472\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[8]\ttraining's rmse: 0.282013\tvalid_1's rmse: 0.28844\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[9]\ttraining's rmse: 0.281974\tvalid_1's rmse: 0.288408\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[10]\ttraining's rmse: 0.281935\tvalid_1's rmse: 0.288376\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[11]\ttraining's rmse: 0.281894\tvalid_1's rmse: 0.288346\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[12]\ttraining's rmse: 0.281853\tvalid_1's rmse: 0.28831\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[13]\ttraining's rmse: 0.281812\tvalid_1's rmse: 0.288279\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[14]\ttraining's rmse: 0.281764\tvalid_1's rmse: 0.288222\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[15]\ttraining's rmse: 0.281723\tvalid_1's rmse: 0.288184\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[16]\ttraining's rmse: 0.281682\tvalid_1's rmse: 0.288149\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[17]\ttraining's rmse: 0.28164\tvalid_1's rmse: 0.288114\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[18]\ttraining's rmse: 0.281599\tvalid_1's rmse: 0.288079\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[19]\ttraining's rmse: 0.281558\tvalid_1's rmse: 0.288038\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[20]\ttraining's rmse: 0.281511\tvalid_1's rmse: 0.288012\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[21]\ttraining's rmse: 0.281466\tvalid_1's rmse: 0.287957\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[22]\ttraining's rmse: 0.281424\tvalid_1's rmse: 0.287906\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[23]\ttraining's rmse: 0.281384\tvalid_1's rmse: 0.28789\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[24]\ttraining's rmse: 0.28134\tvalid_1's rmse: 0.287846\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[25]\ttraining's rmse: 0.281297\tvalid_1's rmse: 0.287804\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[26]\ttraining's rmse: 0.281253\tvalid_1's rmse: 0.28778\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[27]\ttraining's rmse: 0.281212\tvalid_1's rmse: 0.287766\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[28]\ttraining's rmse: 0.281167\tvalid_1's rmse: 0.287738\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/opt/conda/lib/python3.7/site-packages/lightgbm/engine.py:181: UserWarning: 'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. Pass 'early_stopping()' callback via 'callbacks' argument instead.\n",
            "  _log_warning(\"'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. \"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[29]\ttraining's rmse: 0.281122\tvalid_1's rmse: 0.287706\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[30]\ttraining's rmse: 0.281078\tvalid_1's rmse: 0.287658\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[31]\ttraining's rmse: 0.281031\tvalid_1's rmse: 0.28762\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[32]\ttraining's rmse: 0.280986\tvalid_1's rmse: 0.287587\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[33]\ttraining's rmse: 0.280938\tvalid_1's rmse: 0.287555\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[34]\ttraining's rmse: 0.280892\tvalid_1's rmse: 0.287517\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[35]\ttraining's rmse: 0.280846\tvalid_1's rmse: 0.287478\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[36]\ttraining's rmse: 0.2808\tvalid_1's rmse: 0.28746\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[37]\ttraining's rmse: 0.280756\tvalid_1's rmse: 0.287423\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[38]\ttraining's rmse: 0.280709\tvalid_1's rmse: 0.287396\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[39]\ttraining's rmse: 0.28066\tvalid_1's rmse: 0.28736\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[40]\ttraining's rmse: 0.280613\tvalid_1's rmse: 0.287327\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[41]\ttraining's rmse: 0.280573\tvalid_1's rmse: 0.287293\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[42]\ttraining's rmse: 0.280536\tvalid_1's rmse: 0.287262\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[43]\ttraining's rmse: 0.280494\tvalid_1's rmse: 0.287228\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[44]\ttraining's rmse: 0.280452\tvalid_1's rmse: 0.287197\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[45]\ttraining's rmse: 0.280413\tvalid_1's rmse: 0.287163\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[46]\ttraining's rmse: 0.280376\tvalid_1's rmse: 0.287156\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[47]\ttraining's rmse: 0.28033\tvalid_1's rmse: 0.287121\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[48]\ttraining's rmse: 0.280283\tvalid_1's rmse: 0.287084\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[49]\ttraining's rmse: 0.280237\tvalid_1's rmse: 0.287046\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[50]\ttraining's rmse: 0.28019\tvalid_1's rmse: 0.287009\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[51]\ttraining's rmse: 0.280144\tvalid_1's rmse: 0.286982\n",
            "[52]\ttraining's rmse: 0.280098\tvalid_1's rmse: 0.286941\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[53]\ttraining's rmse: 0.280051\tvalid_1's rmse: 0.286915\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[54]\ttraining's rmse: 0.280004\tvalid_1's rmse: 0.286889\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[55]\ttraining's rmse: 0.279958\tvalid_1's rmse: 0.286862\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[56]\ttraining's rmse: 0.279912\tvalid_1's rmse: 0.286824\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[57]\ttraining's rmse: 0.27987\tvalid_1's rmse: 0.286787\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[58]\ttraining's rmse: 0.279832\tvalid_1's rmse: 0.286759\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[59]\ttraining's rmse: 0.27979\tvalid_1's rmse: 0.28671\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[60]\ttraining's rmse: 0.279744\tvalid_1's rmse: 0.286668\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[61]\ttraining's rmse: 0.279697\tvalid_1's rmse: 0.286622\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[62]\ttraining's rmse: 0.279659\tvalid_1's rmse: 0.286587\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[63]\ttraining's rmse: 0.279614\tvalid_1's rmse: 0.286547\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[64]\ttraining's rmse: 0.279568\tvalid_1's rmse: 0.286504\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[65]\ttraining's rmse: 0.279521\tvalid_1's rmse: 0.286458\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[66]\ttraining's rmse: 0.279483\tvalid_1's rmse: 0.286443\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[67]\ttraining's rmse: 0.279447\tvalid_1's rmse: 0.286442\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[68]\ttraining's rmse: 0.27941\tvalid_1's rmse: 0.286427\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[69]\ttraining's rmse: 0.279372\tvalid_1's rmse: 0.286419\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[70]\ttraining's rmse: 0.279337\tvalid_1's rmse: 0.286417\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[71]\ttraining's rmse: 0.279298\tvalid_1's rmse: 0.2864\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[72]\ttraining's rmse: 0.279248\tvalid_1's rmse: 0.286358\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[73]\ttraining's rmse: 0.27921\tvalid_1's rmse: 0.28633\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[74]\ttraining's rmse: 0.279169\tvalid_1's rmse: 0.286317\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[75]\ttraining's rmse: 0.279129\tvalid_1's rmse: 0.286305\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[76]\ttraining's rmse: 0.279086\tvalid_1's rmse: 0.286271\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[77]\ttraining's rmse: 0.279044\tvalid_1's rmse: 0.286228\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[78]\ttraining's rmse: 0.279001\tvalid_1's rmse: 0.2862\n",
            "[79]\ttraining's rmse: 0.278955\tvalid_1's rmse: 0.286189\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[80]\ttraining's rmse: 0.278914\tvalid_1's rmse: 0.286142\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[81]\ttraining's rmse: 0.278865\tvalid_1's rmse: 0.286119\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[82]\ttraining's rmse: 0.278816\tvalid_1's rmse: 0.286098\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[83]\ttraining's rmse: 0.27877\tvalid_1's rmse: 0.286079\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[84]\ttraining's rmse: 0.278722\tvalid_1's rmse: 0.286056\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[85]\ttraining's rmse: 0.278673\tvalid_1's rmse: 0.286034\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[86]\ttraining's rmse: 0.27863\tvalid_1's rmse: 0.286008\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[87]\ttraining's rmse: 0.278585\tvalid_1's rmse: 0.285963\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[88]\ttraining's rmse: 0.278538\tvalid_1's rmse: 0.285915\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[89]\ttraining's rmse: 0.27849\tvalid_1's rmse: 0.285867\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[90]\ttraining's rmse: 0.278442\tvalid_1's rmse: 0.285819\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[91]\ttraining's rmse: 0.278404\tvalid_1's rmse: 0.285801\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[92]\ttraining's rmse: 0.278363\tvalid_1's rmse: 0.28577\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[93]\ttraining's rmse: 0.278322\tvalid_1's rmse: 0.285739\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[94]\ttraining's rmse: 0.278281\tvalid_1's rmse: 0.285702\n",
            "[95]\ttraining's rmse: 0.278239\tvalid_1's rmse: 0.285671\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[96]\ttraining's rmse: 0.278198\tvalid_1's rmse: 0.285638\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[97]\ttraining's rmse: 0.278158\tvalid_1's rmse: 0.285613\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[98]\ttraining's rmse: 0.278117\tvalid_1's rmse: 0.285591\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[99]\ttraining's rmse: 0.278079\tvalid_1's rmse: 0.285569\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[100]\ttraining's rmse: 0.278038\tvalid_1's rmse: 0.285539\n",
            "Did not meet early stopping. Best iteration is:\n",
            "[100]\ttraining's rmse: 0.278038\tvalid_1's rmse: 0.285539\n",
            "RMSE = 0.2855391963264229 \n",
            "\n",
            "#########################\n",
            "### Fold 2\n",
            "### Train size 1845 Valid size 211\n",
            "#########################\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Warning] Auto-choosing col-wise multi-threading, the overhead of testing was 0.004225 seconds.\n",
            "You can set `force_col_wise=true` to remove the overhead.\n",
            "[LightGBM] [Info] Total Bins 24295\n",
            "[LightGBM] [Info] Number of data points in the train set: 1845, number of used features: 127\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Info] Start training from score 0.508130\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[1]\ttraining's rmse: 0.282826\tvalid_1's rmse: 0.285209\n",
            "Training until validation scores don't improve for 10 rounds\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[2]\ttraining's rmse: 0.28277\tvalid_1's rmse: 0.285224\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[3]\ttraining's rmse: 0.282718\tvalid_1's rmse: 0.285232\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[4]\ttraining's rmse: 0.282664\tvalid_1's rmse: 0.285238\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[5]\ttraining's rmse: 0.282611\tvalid_1's rmse: 0.285246\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[6]\ttraining's rmse: 0.282554\tvalid_1's rmse: 0.285238\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[7]\ttraining's rmse: 0.2825\tvalid_1's rmse: 0.285251\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[8]\ttraining's rmse: 0.282446\tvalid_1's rmse: 0.285254\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[9]\ttraining's rmse: 0.282391\tvalid_1's rmse: 0.285254\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[10]\ttraining's rmse: 0.282337\tvalid_1's rmse: 0.285266\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[11]\ttraining's rmse: 0.282282\tvalid_1's rmse: 0.285265\n",
            "Early stopping, best iteration is:\n",
            "[1]\ttraining's rmse: 0.282826\tvalid_1's rmse: 0.285209\n",
            "RMSE = 0.28520863949331093 \n",
            "\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/opt/conda/lib/python3.7/site-packages/lightgbm/engine.py:181: UserWarning: 'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. Pass 'early_stopping()' callback via 'callbacks' argument instead.\n",
            "  _log_warning(\"'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. \"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "#########################\n",
            "### Fold 3\n",
            "### Train size 1871 Valid size 185\n",
            "#########################\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Warning] Auto-choosing col-wise multi-threading, the overhead of testing was 0.004198 seconds.\n",
            "You can set `force_col_wise=true` to remove the overhead.\n",
            "[LightGBM] [Info] Total Bins 24431\n",
            "[LightGBM] [Info] Number of data points in the train set: 1871, number of used features: 127\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Info] Start training from score 0.507750\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[1]\ttraining's rmse: 0.28252\tvalid_1's rmse: 0.288553\n",
            "Training until validation scores don't improve for 10 rounds\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[2]\ttraining's rmse: 0.282467\tvalid_1's rmse: 0.288508\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[3]\ttraining's rmse: 0.282426\tvalid_1's rmse: 0.288491\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[4]\ttraining's rmse: 0.282385\tvalid_1's rmse: 0.288474\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[5]\ttraining's rmse: 0.282345\tvalid_1's rmse: 0.288444\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[6]\ttraining's rmse: 0.282294\tvalid_1's rmse: 0.28841\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[7]\ttraining's rmse: 0.282243\tvalid_1's rmse: 0.288372\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[8]\ttraining's rmse: 0.282192\tvalid_1's rmse: 0.288338\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[9]\ttraining's rmse: 0.28214\tvalid_1's rmse: 0.288302\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[10]\ttraining's rmse: 0.282089\tvalid_1's rmse: 0.288269\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[11]\ttraining's rmse: 0.282036\tvalid_1's rmse: 0.288251\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[12]\ttraining's rmse: 0.281984\tvalid_1's rmse: 0.288221\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[13]\ttraining's rmse: 0.28193\tvalid_1's rmse: 0.288203\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[14]\ttraining's rmse: 0.281877\tvalid_1's rmse: 0.288184\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[15]\ttraining's rmse: 0.281836\tvalid_1's rmse: 0.288161\n",
            "[16]\ttraining's rmse: 0.281779\tvalid_1's rmse: 0.288125\n",
            "[17]\ttraining's rmse: 0.281725\tvalid_1's rmse: 0.288085\n",
            "[18]\ttraining's rmse: 0.281669\tvalid_1's rmse: 0.288049\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[19]\ttraining's rmse: 0.281615\tvalid_1's rmse: 0.288017\n",
            "[20]\ttraining's rmse: 0.28156\tvalid_1's rmse: 0.287991\n",
            "[21]\ttraining's rmse: 0.281509\tvalid_1's rmse: 0.287959\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[22]\ttraining's rmse: 0.28146\tvalid_1's rmse: 0.287931\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[23]\ttraining's rmse: 0.281414\tvalid_1's rmse: 0.287917\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[24]\ttraining's rmse: 0.281364\tvalid_1's rmse: 0.287892\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[25]\ttraining's rmse: 0.281313\tvalid_1's rmse: 0.287859\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[26]\ttraining's rmse: 0.281262\tvalid_1's rmse: 0.287816\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[27]\ttraining's rmse: 0.281224\tvalid_1's rmse: 0.287807\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[28]\ttraining's rmse: 0.28118\tvalid_1's rmse: 0.287779\n",
            "[29]\ttraining's rmse: 0.281127\tvalid_1's rmse: 0.287738\n",
            "[30]\ttraining's rmse: 0.281075\tvalid_1's rmse: 0.287693\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[31]\ttraining's rmse: 0.281028\tvalid_1's rmse: 0.287651\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[32]\ttraining's rmse: 0.280979\tvalid_1's rmse: 0.287612\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[33]\ttraining's rmse: 0.280932\tvalid_1's rmse: 0.287582\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/opt/conda/lib/python3.7/site-packages/lightgbm/engine.py:181: UserWarning: 'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. Pass 'early_stopping()' callback via 'callbacks' argument instead.\n",
            "  _log_warning(\"'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. \"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[34]\ttraining's rmse: 0.280881\tvalid_1's rmse: 0.287564\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[35]\ttraining's rmse: 0.280831\tvalid_1's rmse: 0.28755\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[36]\ttraining's rmse: 0.280783\tvalid_1's rmse: 0.287513\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[37]\ttraining's rmse: 0.280735\tvalid_1's rmse: 0.287495\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[38]\ttraining's rmse: 0.280687\tvalid_1's rmse: 0.287472\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[39]\ttraining's rmse: 0.28064\tvalid_1's rmse: 0.287438\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[40]\ttraining's rmse: 0.280591\tvalid_1's rmse: 0.287407\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[41]\ttraining's rmse: 0.28054\tvalid_1's rmse: 0.287387\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[42]\ttraining's rmse: 0.280494\tvalid_1's rmse: 0.28734\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[43]\ttraining's rmse: 0.280447\tvalid_1's rmse: 0.2873\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[44]\ttraining's rmse: 0.2804\tvalid_1's rmse: 0.287262\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[45]\ttraining's rmse: 0.280348\tvalid_1's rmse: 0.287256\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[46]\ttraining's rmse: 0.280309\tvalid_1's rmse: 0.28725\n",
            "[47]\ttraining's rmse: 0.280268\tvalid_1's rmse: 0.287241\n",
            "[48]\ttraining's rmse: 0.280226\tvalid_1's rmse: 0.287235\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[49]\ttraining's rmse: 0.28018\tvalid_1's rmse: 0.287217\n",
            "[50]\ttraining's rmse: 0.280138\tvalid_1's rmse: 0.287207\n",
            "[51]\ttraining's rmse: 0.280085\tvalid_1's rmse: 0.287171\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[52]\ttraining's rmse: 0.280032\tvalid_1's rmse: 0.287145\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[53]\ttraining's rmse: 0.27998\tvalid_1's rmse: 0.28711\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[54]\ttraining's rmse: 0.279927\tvalid_1's rmse: 0.287075\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[55]\ttraining's rmse: 0.279875\tvalid_1's rmse: 0.28704\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[56]\ttraining's rmse: 0.279823\tvalid_1's rmse: 0.287018\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[57]\ttraining's rmse: 0.27978\tvalid_1's rmse: 0.287016\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[58]\ttraining's rmse: 0.279739\tvalid_1's rmse: 0.286993\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[59]\ttraining's rmse: 0.279687\tvalid_1's rmse: 0.286982\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[60]\ttraining's rmse: 0.279635\tvalid_1's rmse: 0.28696\n",
            "[61]\ttraining's rmse: 0.279582\tvalid_1's rmse: 0.286904\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[62]\ttraining's rmse: 0.279544\tvalid_1's rmse: 0.286889\n",
            "[63]\ttraining's rmse: 0.279491\tvalid_1's rmse: 0.286833\n",
            "[64]\ttraining's rmse: 0.279439\tvalid_1's rmse: 0.286786\n",
            "[65]\ttraining's rmse: 0.279387\tvalid_1's rmse: 0.286728\n",
            "[66]\ttraining's rmse: 0.279335\tvalid_1's rmse: 0.286691\n",
            "[67]\ttraining's rmse: 0.279284\tvalid_1's rmse: 0.286649\n",
            "[68]\ttraining's rmse: 0.279233\tvalid_1's rmse: 0.286609\n",
            "[69]\ttraining's rmse: 0.279183\tvalid_1's rmse: 0.286571\n",
            "[70]\ttraining's rmse: 0.279133\tvalid_1's rmse: 0.286522\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[71]\ttraining's rmse: 0.279087\tvalid_1's rmse: 0.286494\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[72]\ttraining's rmse: 0.27904\tvalid_1's rmse: 0.286448\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[73]\ttraining's rmse: 0.278993\tvalid_1's rmse: 0.286411\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[74]\ttraining's rmse: 0.278946\tvalid_1's rmse: 0.286365\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[75]\ttraining's rmse: 0.278899\tvalid_1's rmse: 0.286337\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[76]\ttraining's rmse: 0.278851\tvalid_1's rmse: 0.286316\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[77]\ttraining's rmse: 0.278802\tvalid_1's rmse: 0.286294\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[78]\ttraining's rmse: 0.278754\tvalid_1's rmse: 0.286282\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[79]\ttraining's rmse: 0.278714\tvalid_1's rmse: 0.286226\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[80]\ttraining's rmse: 0.278666\tvalid_1's rmse: 0.286205\n",
            "[81]\ttraining's rmse: 0.278618\tvalid_1's rmse: 0.28616\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[82]\ttraining's rmse: 0.278571\tvalid_1's rmse: 0.286114\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[83]\ttraining's rmse: 0.278525\tvalid_1's rmse: 0.286073\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[84]\ttraining's rmse: 0.278479\tvalid_1's rmse: 0.286034\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[85]\ttraining's rmse: 0.278434\tvalid_1's rmse: 0.286012\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[86]\ttraining's rmse: 0.278394\tvalid_1's rmse: 0.285989\n",
            "[87]\ttraining's rmse: 0.278343\tvalid_1's rmse: 0.285962\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[88]\ttraining's rmse: 0.278295\tvalid_1's rmse: 0.285936\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[89]\ttraining's rmse: 0.278247\tvalid_1's rmse: 0.285911\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[90]\ttraining's rmse: 0.278201\tvalid_1's rmse: 0.285876\n",
            "[91]\ttraining's rmse: 0.278158\tvalid_1's rmse: 0.285853\n",
            "[92]\ttraining's rmse: 0.278108\tvalid_1's rmse: 0.285816\n",
            "[93]\ttraining's rmse: 0.278058\tvalid_1's rmse: 0.285782\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[94]\ttraining's rmse: 0.278013\tvalid_1's rmse: 0.285757\n",
            "[95]\ttraining's rmse: 0.277964\tvalid_1's rmse: 0.28572\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[96]\ttraining's rmse: 0.27792\tvalid_1's rmse: 0.285701\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[97]\ttraining's rmse: 0.277876\tvalid_1's rmse: 0.285673\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[98]\ttraining's rmse: 0.277832\tvalid_1's rmse: 0.285671\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[99]\ttraining's rmse: 0.277787\tvalid_1's rmse: 0.285647\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[100]\ttraining's rmse: 0.277743\tvalid_1's rmse: 0.285625\n",
            "Did not meet early stopping. Best iteration is:\n",
            "[100]\ttraining's rmse: 0.277743\tvalid_1's rmse: 0.285625\n",
            "RMSE = 0.2856246237814212 \n",
            "\n",
            "#########################\n",
            "### Fold 4\n",
            "### Train size 1854 Valid size 202\n",
            "#########################\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Warning] Auto-choosing col-wise multi-threading, the overhead of testing was 0.004180 seconds.\n",
            "You can set `force_col_wise=true` to remove the overhead.\n",
            "[LightGBM] [Info] Total Bins 24480\n",
            "[LightGBM] [Info] Number of data points in the train set: 1854, number of used features: 127\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Info] Start training from score 0.507551\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[1]\ttraining's rmse: 0.287069\tvalid_1's rmse: 0.243243\n",
            "Training until validation scores don't improve for 10 rounds\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[2]\ttraining's rmse: 0.287011\tvalid_1's rmse: 0.243228\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[3]\ttraining's rmse: 0.286953\tvalid_1's rmse: 0.243207\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[4]\ttraining's rmse: 0.286899\tvalid_1's rmse: 0.243185\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[5]\ttraining's rmse: 0.286845\tvalid_1's rmse: 0.24317\n",
            "[6]\ttraining's rmse: 0.286787\tvalid_1's rmse: 0.243152\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[7]\ttraining's rmse: 0.28673\tvalid_1's rmse: 0.243136\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[8]\ttraining's rmse: 0.286672\tvalid_1's rmse: 0.243121\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[9]\ttraining's rmse: 0.286614\tvalid_1's rmse: 0.243105\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[10]\ttraining's rmse: 0.286557\tvalid_1's rmse: 0.243089\n",
            "[11]\ttraining's rmse: 0.286497\tvalid_1's rmse: 0.243078\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[12]\ttraining's rmse: 0.286438\tvalid_1's rmse: 0.24306\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[13]\ttraining's rmse: 0.286378\tvalid_1's rmse: 0.243043\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[14]\ttraining's rmse: 0.286323\tvalid_1's rmse: 0.243029\n",
            "[15]\ttraining's rmse: 0.286266\tvalid_1's rmse: 0.243011\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[16]\ttraining's rmse: 0.286208\tvalid_1's rmse: 0.242994\n",
            "[17]\ttraining's rmse: 0.286147\tvalid_1's rmse: 0.242982\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[18]\ttraining's rmse: 0.286087\tvalid_1's rmse: 0.242972\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[19]\ttraining's rmse: 0.28603\tvalid_1's rmse: 0.242966\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[20]\ttraining's rmse: 0.285971\tvalid_1's rmse: 0.242957\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[21]\ttraining's rmse: 0.285914\tvalid_1's rmse: 0.242935\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[22]\ttraining's rmse: 0.285858\tvalid_1's rmse: 0.242917\n",
            "[23]\ttraining's rmse: 0.285804\tvalid_1's rmse: 0.242904\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[24]\ttraining's rmse: 0.285748\tvalid_1's rmse: 0.24288\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[25]\ttraining's rmse: 0.285692\tvalid_1's rmse: 0.242857\n",
            "[26]\ttraining's rmse: 0.285636\tvalid_1's rmse: 0.242839\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[27]\ttraining's rmse: 0.285589\tvalid_1's rmse: 0.242822\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[28]\ttraining's rmse: 0.285541\tvalid_1's rmse: 0.242806\n",
            "[29]\ttraining's rmse: 0.285486\tvalid_1's rmse: 0.242791\n",
            "[30]\ttraining's rmse: 0.28543\tvalid_1's rmse: 0.24277\n",
            "[31]\ttraining's rmse: 0.285372\tvalid_1's rmse: 0.242752\n",
            "[32]\ttraining's rmse: 0.285314\tvalid_1's rmse: 0.242733\n",
            "[33]\ttraining's rmse: 0.285261\tvalid_1's rmse: 0.242718\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/opt/conda/lib/python3.7/site-packages/lightgbm/engine.py:181: UserWarning: 'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. Pass 'early_stopping()' callback via 'callbacks' argument instead.\n",
            "  _log_warning(\"'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. \"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "[34]\ttraining's rmse: 0.285205\tvalid_1's rmse: 0.242696\n",
            "[35]\ttraining's rmse: 0.28515\tvalid_1's rmse: 0.242682\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[36]\ttraining's rmse: 0.285094\tvalid_1's rmse: 0.242669\n",
            "[37]\ttraining's rmse: 0.285038\tvalid_1's rmse: 0.242656\n",
            "[38]\ttraining's rmse: 0.28498\tvalid_1's rmse: 0.242645\n",
            "[39]\ttraining's rmse: 0.284928\tvalid_1's rmse: 0.242629\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[40]\ttraining's rmse: 0.284871\tvalid_1's rmse: 0.242618\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[41]\ttraining's rmse: 0.284817\tvalid_1's rmse: 0.242606\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[42]\ttraining's rmse: 0.284763\tvalid_1's rmse: 0.242591\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[43]\ttraining's rmse: 0.284706\tvalid_1's rmse: 0.242582\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[44]\ttraining's rmse: 0.284648\tvalid_1's rmse: 0.242573\n",
            "[45]\ttraining's rmse: 0.284589\tvalid_1's rmse: 0.242564\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[46]\ttraining's rmse: 0.284534\tvalid_1's rmse: 0.242551\n",
            "[47]\ttraining's rmse: 0.284473\tvalid_1's rmse: 0.242539\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[48]\ttraining's rmse: 0.284416\tvalid_1's rmse: 0.242526\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[49]\ttraining's rmse: 0.284361\tvalid_1's rmse: 0.242518\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[50]\ttraining's rmse: 0.284306\tvalid_1's rmse: 0.242509\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[51]\ttraining's rmse: 0.284253\tvalid_1's rmse: 0.242485\n",
            "[52]\ttraining's rmse: 0.2842\tvalid_1's rmse: 0.242457\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[53]\ttraining's rmse: 0.284147\tvalid_1's rmse: 0.242433\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[54]\ttraining's rmse: 0.284096\tvalid_1's rmse: 0.242409\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[55]\ttraining's rmse: 0.284041\tvalid_1's rmse: 0.242391\n",
            "[56]\ttraining's rmse: 0.283979\tvalid_1's rmse: 0.242378\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[57]\ttraining's rmse: 0.28393\tvalid_1's rmse: 0.242358\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[58]\ttraining's rmse: 0.283879\tvalid_1's rmse: 0.242338\n",
            "[59]\ttraining's rmse: 0.283817\tvalid_1's rmse: 0.242326\n",
            "[60]\ttraining's rmse: 0.283757\tvalid_1's rmse: 0.24231\n",
            "[61]\ttraining's rmse: 0.283701\tvalid_1's rmse: 0.242286\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[62]\ttraining's rmse: 0.283646\tvalid_1's rmse: 0.242268\n",
            "[63]\ttraining's rmse: 0.283591\tvalid_1's rmse: 0.242245\n",
            "[64]\ttraining's rmse: 0.283536\tvalid_1's rmse: 0.242221\n",
            "[65]\ttraining's rmse: 0.283481\tvalid_1's rmse: 0.242202\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[66]\ttraining's rmse: 0.283425\tvalid_1's rmse: 0.242186\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[67]\ttraining's rmse: 0.28337\tvalid_1's rmse: 0.242172\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[68]\ttraining's rmse: 0.283314\tvalid_1's rmse: 0.242153\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[69]\ttraining's rmse: 0.283259\tvalid_1's rmse: 0.242136\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[70]\ttraining's rmse: 0.283204\tvalid_1's rmse: 0.242121\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[71]\ttraining's rmse: 0.283146\tvalid_1's rmse: 0.242107\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[72]\ttraining's rmse: 0.283088\tvalid_1's rmse: 0.242092\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[73]\ttraining's rmse: 0.283033\tvalid_1's rmse: 0.242083\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[74]\ttraining's rmse: 0.282975\tvalid_1's rmse: 0.242069\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[75]\ttraining's rmse: 0.282918\tvalid_1's rmse: 0.242055\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[76]\ttraining's rmse: 0.282868\tvalid_1's rmse: 0.242034\n",
            "[77]\ttraining's rmse: 0.282814\tvalid_1's rmse: 0.24202\n",
            "[78]\ttraining's rmse: 0.282761\tvalid_1's rmse: 0.242011\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[79]\ttraining's rmse: 0.282712\tvalid_1's rmse: 0.242001\n",
            "[80]\ttraining's rmse: 0.282658\tvalid_1's rmse: 0.241988\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[81]\ttraining's rmse: 0.282609\tvalid_1's rmse: 0.241976\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[82]\ttraining's rmse: 0.28256\tvalid_1's rmse: 0.241966\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[83]\ttraining's rmse: 0.28251\tvalid_1's rmse: 0.241955\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[84]\ttraining's rmse: 0.282461\tvalid_1's rmse: 0.241945\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[85]\ttraining's rmse: 0.282411\tvalid_1's rmse: 0.241936\n",
            "[86]\ttraining's rmse: 0.28236\tvalid_1's rmse: 0.241918\n",
            "[87]\ttraining's rmse: 0.282304\tvalid_1's rmse: 0.241902\n",
            "[88]\ttraining's rmse: 0.282248\tvalid_1's rmse: 0.241889\n",
            "[89]\ttraining's rmse: 0.282189\tvalid_1's rmse: 0.241877\n",
            "[90]\ttraining's rmse: 0.282131\tvalid_1's rmse: 0.241862\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[91]\ttraining's rmse: 0.282078\tvalid_1's rmse: 0.241845\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[92]\ttraining's rmse: 0.28202\tvalid_1's rmse: 0.241828\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[93]\ttraining's rmse: 0.281964\tvalid_1's rmse: 0.241816\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[94]\ttraining's rmse: 0.281908\tvalid_1's rmse: 0.241804\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[95]\ttraining's rmse: 0.281851\tvalid_1's rmse: 0.241789\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[96]\ttraining's rmse: 0.2818\tvalid_1's rmse: 0.24177\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[97]\ttraining's rmse: 0.281759\tvalid_1's rmse: 0.241751\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[98]\ttraining's rmse: 0.281718\tvalid_1's rmse: 0.241732\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[99]\ttraining's rmse: 0.281669\tvalid_1's rmse: 0.241716\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[100]\ttraining's rmse: 0.281618\tvalid_1's rmse: 0.241697\n",
            "Did not meet early stopping. Best iteration is:\n",
            "[100]\ttraining's rmse: 0.281618\tvalid_1's rmse: 0.241697\n",
            "RMSE = 0.24169659751958172 \n",
            "\n",
            "#########################\n",
            "### Fold 5\n",
            "### Train size 1847 Valid size 209\n",
            "#########################\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Warning] Auto-choosing col-wise multi-threading, the overhead of testing was 0.006491 seconds.\n",
            "You can set `force_col_wise=true` to remove the overhead.\n",
            "[LightGBM] [Info] Total Bins 24450\n",
            "[LightGBM] [Info] Number of data points in the train set: 1847, number of used features: 127\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Info] Start training from score 0.507309\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[1]\ttraining's rmse: 0.282665\tvalid_1's rmse: 0.286598\n",
            "Training until validation scores don't improve for 10 rounds\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[2]\ttraining's rmse: 0.282611\tvalid_1's rmse: 0.286581\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[3]\ttraining's rmse: 0.282558\tvalid_1's rmse: 0.286576\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[4]\ttraining's rmse: 0.282502\tvalid_1's rmse: 0.28658\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[5]\ttraining's rmse: 0.282449\tvalid_1's rmse: 0.286575\n",
            "[6]\ttraining's rmse: 0.282392\tvalid_1's rmse: 0.286539\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[7]\ttraining's rmse: 0.282339\tvalid_1's rmse: 0.28651\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[8]\ttraining's rmse: 0.282286\tvalid_1's rmse: 0.286479\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[9]\ttraining's rmse: 0.282232\tvalid_1's rmse: 0.286445\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[10]\ttraining's rmse: 0.282179\tvalid_1's rmse: 0.286418\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[11]\ttraining's rmse: 0.282128\tvalid_1's rmse: 0.286389\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[12]\ttraining's rmse: 0.282079\tvalid_1's rmse: 0.286356\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[13]\ttraining's rmse: 0.282031\tvalid_1's rmse: 0.286322\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[14]\ttraining's rmse: 0.281983\tvalid_1's rmse: 0.28629\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[15]\ttraining's rmse: 0.281931\tvalid_1's rmse: 0.286262\n",
            "[16]\ttraining's rmse: 0.281882\tvalid_1's rmse: 0.286245\n",
            "[17]\ttraining's rmse: 0.281829\tvalid_1's rmse: 0.286219\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/opt/conda/lib/python3.7/site-packages/lightgbm/engine.py:181: UserWarning: 'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. Pass 'early_stopping()' callback via 'callbacks' argument instead.\n",
            "  _log_warning(\"'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. \"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "[18]\ttraining's rmse: 0.281775\tvalid_1's rmse: 0.286187\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[19]\ttraining's rmse: 0.281728\tvalid_1's rmse: 0.286175\n",
            "[20]\ttraining's rmse: 0.281676\tvalid_1's rmse: 0.286149\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[21]\ttraining's rmse: 0.281626\tvalid_1's rmse: 0.286119\n",
            "[22]\ttraining's rmse: 0.281573\tvalid_1's rmse: 0.286088\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[23]\ttraining's rmse: 0.281524\tvalid_1's rmse: 0.286072\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[24]\ttraining's rmse: 0.281474\tvalid_1's rmse: 0.286034\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[25]\ttraining's rmse: 0.281423\tvalid_1's rmse: 0.285996\n",
            "[26]\ttraining's rmse: 0.281369\tvalid_1's rmse: 0.285992\n",
            "[27]\ttraining's rmse: 0.281319\tvalid_1's rmse: 0.285994\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[28]\ttraining's rmse: 0.281272\tvalid_1's rmse: 0.285969\n",
            "[29]\ttraining's rmse: 0.281218\tvalid_1's rmse: 0.285965\n",
            "[30]\ttraining's rmse: 0.281166\tvalid_1's rmse: 0.285962\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[31]\ttraining's rmse: 0.281115\tvalid_1's rmse: 0.285966\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[32]\ttraining's rmse: 0.281063\tvalid_1's rmse: 0.285971\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[33]\ttraining's rmse: 0.281015\tvalid_1's rmse: 0.285954\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[34]\ttraining's rmse: 0.280966\tvalid_1's rmse: 0.28595\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[35]\ttraining's rmse: 0.280915\tvalid_1's rmse: 0.285955\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[36]\ttraining's rmse: 0.280866\tvalid_1's rmse: 0.285935\n",
            "[37]\ttraining's rmse: 0.280814\tvalid_1's rmse: 0.285923\n",
            "[38]\ttraining's rmse: 0.280762\tvalid_1's rmse: 0.285899\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[39]\ttraining's rmse: 0.280714\tvalid_1's rmse: 0.285882\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[40]\ttraining's rmse: 0.280665\tvalid_1's rmse: 0.285861\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[41]\ttraining's rmse: 0.280614\tvalid_1's rmse: 0.285825\n",
            "[42]\ttraining's rmse: 0.280562\tvalid_1's rmse: 0.285795\n",
            "[43]\ttraining's rmse: 0.280513\tvalid_1's rmse: 0.285746\n",
            "[44]\ttraining's rmse: 0.280463\tvalid_1's rmse: 0.285696\n",
            "[45]\ttraining's rmse: 0.280412\tvalid_1's rmse: 0.28565\n",
            "[46]\ttraining's rmse: 0.28036\tvalid_1's rmse: 0.285635\n",
            "[47]\ttraining's rmse: 0.28031\tvalid_1's rmse: 0.285622\n",
            "[48]\ttraining's rmse: 0.280261\tvalid_1's rmse: 0.285612\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[49]\ttraining's rmse: 0.280212\tvalid_1's rmse: 0.2856\n",
            "[50]\ttraining's rmse: 0.280163\tvalid_1's rmse: 0.285593\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[51]\ttraining's rmse: 0.280111\tvalid_1's rmse: 0.285573\n",
            "[52]\ttraining's rmse: 0.28006\tvalid_1's rmse: 0.285561\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[53]\ttraining's rmse: 0.280007\tvalid_1's rmse: 0.28554\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[54]\ttraining's rmse: 0.279955\tvalid_1's rmse: 0.285521\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[55]\ttraining's rmse: 0.279903\tvalid_1's rmse: 0.285503\n",
            "[56]\ttraining's rmse: 0.279852\tvalid_1's rmse: 0.28548\n",
            "[57]\ttraining's rmse: 0.2798\tvalid_1's rmse: 0.285464\n",
            "[58]\ttraining's rmse: 0.279747\tvalid_1's rmse: 0.285452\n",
            "[59]\ttraining's rmse: 0.279696\tvalid_1's rmse: 0.285432\n",
            "[60]\ttraining's rmse: 0.279645\tvalid_1's rmse: 0.285406\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[61]\ttraining's rmse: 0.279592\tvalid_1's rmse: 0.28541\n",
            "[62]\ttraining's rmse: 0.279543\tvalid_1's rmse: 0.285395\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[63]\ttraining's rmse: 0.279492\tvalid_1's rmse: 0.285394\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[64]\ttraining's rmse: 0.27944\tvalid_1's rmse: 0.285397\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[65]\ttraining's rmse: 0.279388\tvalid_1's rmse: 0.285401\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[66]\ttraining's rmse: 0.279336\tvalid_1's rmse: 0.285392\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[67]\ttraining's rmse: 0.279288\tvalid_1's rmse: 0.285384\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[68]\ttraining's rmse: 0.279239\tvalid_1's rmse: 0.285373\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[69]\ttraining's rmse: 0.279188\tvalid_1's rmse: 0.285365\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[70]\ttraining's rmse: 0.279138\tvalid_1's rmse: 0.285356\n",
            "[71]\ttraining's rmse: 0.279088\tvalid_1's rmse: 0.28535\n",
            "[72]\ttraining's rmse: 0.279039\tvalid_1's rmse: 0.285339\n",
            "[73]\ttraining's rmse: 0.278988\tvalid_1's rmse: 0.285336\n",
            "[74]\ttraining's rmse: 0.278939\tvalid_1's rmse: 0.285335\n",
            "[75]\ttraining's rmse: 0.278889\tvalid_1's rmse: 0.285319\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[76]\ttraining's rmse: 0.278842\tvalid_1's rmse: 0.285304\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[77]\ttraining's rmse: 0.278795\tvalid_1's rmse: 0.285289\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[78]\ttraining's rmse: 0.278747\tvalid_1's rmse: 0.28527\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[79]\ttraining's rmse: 0.278703\tvalid_1's rmse: 0.28524\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[80]\ttraining's rmse: 0.278656\tvalid_1's rmse: 0.285221\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[81]\ttraining's rmse: 0.278606\tvalid_1's rmse: 0.285195\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[82]\ttraining's rmse: 0.278556\tvalid_1's rmse: 0.285169\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[83]\ttraining's rmse: 0.278505\tvalid_1's rmse: 0.285146\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[84]\ttraining's rmse: 0.278461\tvalid_1's rmse: 0.285123\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[85]\ttraining's rmse: 0.278417\tvalid_1's rmse: 0.285114\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[86]\ttraining's rmse: 0.278368\tvalid_1's rmse: 0.285113\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[87]\ttraining's rmse: 0.278316\tvalid_1's rmse: 0.28511\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[88]\ttraining's rmse: 0.278264\tvalid_1's rmse: 0.285107\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[89]\ttraining's rmse: 0.278212\tvalid_1's rmse: 0.285104\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[90]\ttraining's rmse: 0.278163\tvalid_1's rmse: 0.285094\n",
            "[91]\ttraining's rmse: 0.278113\tvalid_1's rmse: 0.285076\n",
            "[92]\ttraining's rmse: 0.27806\tvalid_1's rmse: 0.285053\n",
            "[93]\ttraining's rmse: 0.278005\tvalid_1's rmse: 0.285052\n",
            "[94]\ttraining's rmse: 0.277952\tvalid_1's rmse: 0.285018\n",
            "[95]\ttraining's rmse: 0.277899\tvalid_1's rmse: 0.284995\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[96]\ttraining's rmse: 0.277849\tvalid_1's rmse: 0.284984\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[97]\ttraining's rmse: 0.277802\tvalid_1's rmse: 0.284971\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[98]\ttraining's rmse: 0.277756\tvalid_1's rmse: 0.284952\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[99]\ttraining's rmse: 0.277706\tvalid_1's rmse: 0.284949\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[100]\ttraining's rmse: 0.277657\tvalid_1's rmse: 0.284938\n",
            "Did not meet early stopping. Best iteration is:\n",
            "[100]\ttraining's rmse: 0.277657\tvalid_1's rmse: 0.284938\n",
            "RMSE = 0.284938233534118 \n",
            "\n",
            "#########################\n",
            "### Fold 6\n",
            "### Train size 1854 Valid size 202\n",
            "#########################\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Warning] Auto-choosing col-wise multi-threading, the overhead of testing was 0.004517 seconds.\n",
            "You can set `force_col_wise=true` to remove the overhead.\n",
            "[LightGBM] [Info] Total Bins 24430\n",
            "[LightGBM] [Info] Number of data points in the train set: 1854, number of used features: 127\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Info] Start training from score 0.507282\n",
            "[1]\ttraining's rmse: 0.282698\tvalid_1's rmse: 0.286454\n",
            "Training until validation scores don't improve for 10 rounds\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[2]\ttraining's rmse: 0.282647\tvalid_1's rmse: 0.286426\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[3]\ttraining's rmse: 0.282601\tvalid_1's rmse: 0.286394\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[4]\ttraining's rmse: 0.282553\tvalid_1's rmse: 0.286359\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[5]\ttraining's rmse: 0.282505\tvalid_1's rmse: 0.286334\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[6]\ttraining's rmse: 0.282454\tvalid_1's rmse: 0.286282\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[7]\ttraining's rmse: 0.282405\tvalid_1's rmse: 0.286228\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[8]\ttraining's rmse: 0.282354\tvalid_1's rmse: 0.286169\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[9]\ttraining's rmse: 0.282304\tvalid_1's rmse: 0.286118\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[10]\ttraining's rmse: 0.282254\tvalid_1's rmse: 0.286056\n",
            "[11]\ttraining's rmse: 0.282201\tvalid_1's rmse: 0.286007\n",
            "[12]\ttraining's rmse: 0.282147\tvalid_1's rmse: 0.285961\n",
            "[13]\ttraining's rmse: 0.282094\tvalid_1's rmse: 0.285916\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[14]\ttraining's rmse: 0.282044\tvalid_1's rmse: 0.285872\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[15]\ttraining's rmse: 0.281997\tvalid_1's rmse: 0.285839\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[16]\ttraining's rmse: 0.28195\tvalid_1's rmse: 0.285803\n",
            "[17]\ttraining's rmse: 0.281895\tvalid_1's rmse: 0.285759\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[18]\ttraining's rmse: 0.281849\tvalid_1's rmse: 0.285718\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[19]\ttraining's rmse: 0.281805\tvalid_1's rmse: 0.285679\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[20]\ttraining's rmse: 0.281759\tvalid_1's rmse: 0.285644\n",
            "[21]\ttraining's rmse: 0.281706\tvalid_1's rmse: 0.285592\n",
            "[22]\ttraining's rmse: 0.281656\tvalid_1's rmse: 0.28554\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[23]\ttraining's rmse: 0.281609\tvalid_1's rmse: 0.285513\n",
            "[24]\ttraining's rmse: 0.281559\tvalid_1's rmse: 0.285462\n",
            "[25]\ttraining's rmse: 0.281509\tvalid_1's rmse: 0.285408\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[26]\ttraining's rmse: 0.281458\tvalid_1's rmse: 0.285371\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[27]\ttraining's rmse: 0.281409\tvalid_1's rmse: 0.285339\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[28]\ttraining's rmse: 0.281362\tvalid_1's rmse: 0.285304\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[29]\ttraining's rmse: 0.281315\tvalid_1's rmse: 0.285256\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[30]\ttraining's rmse: 0.281267\tvalid_1's rmse: 0.285222\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[31]\ttraining's rmse: 0.281215\tvalid_1's rmse: 0.285188\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[32]\ttraining's rmse: 0.281163\tvalid_1's rmse: 0.285154\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[33]\ttraining's rmse: 0.281115\tvalid_1's rmse: 0.285111\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[34]\ttraining's rmse: 0.281064\tvalid_1's rmse: 0.285076\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/opt/conda/lib/python3.7/site-packages/lightgbm/engine.py:181: UserWarning: 'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. Pass 'early_stopping()' callback via 'callbacks' argument instead.\n",
            "  _log_warning(\"'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. \"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "[35]\ttraining's rmse: 0.281013\tvalid_1's rmse: 0.285036\n",
            "[36]\ttraining's rmse: 0.280959\tvalid_1's rmse: 0.284983\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[37]\ttraining's rmse: 0.280907\tvalid_1's rmse: 0.284938\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[38]\ttraining's rmse: 0.280854\tvalid_1's rmse: 0.284898\n",
            "[39]\ttraining's rmse: 0.280801\tvalid_1's rmse: 0.284853\n",
            "[40]\ttraining's rmse: 0.280751\tvalid_1's rmse: 0.284803\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[41]\ttraining's rmse: 0.280704\tvalid_1's rmse: 0.284768\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[42]\ttraining's rmse: 0.280659\tvalid_1's rmse: 0.284728\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[43]\ttraining's rmse: 0.280614\tvalid_1's rmse: 0.284687\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[44]\ttraining's rmse: 0.280568\tvalid_1's rmse: 0.284634\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[45]\ttraining's rmse: 0.280523\tvalid_1's rmse: 0.284588\n",
            "[46]\ttraining's rmse: 0.28047\tvalid_1's rmse: 0.284549\n",
            "[47]\ttraining's rmse: 0.280419\tvalid_1's rmse: 0.284512\n",
            "[48]\ttraining's rmse: 0.28037\tvalid_1's rmse: 0.284469\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[49]\ttraining's rmse: 0.280321\tvalid_1's rmse: 0.284426\n",
            "[50]\ttraining's rmse: 0.28027\tvalid_1's rmse: 0.28438\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[51]\ttraining's rmse: 0.280223\tvalid_1's rmse: 0.284342\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[52]\ttraining's rmse: 0.280174\tvalid_1's rmse: 0.284301\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[53]\ttraining's rmse: 0.280131\tvalid_1's rmse: 0.284267\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[54]\ttraining's rmse: 0.280088\tvalid_1's rmse: 0.284232\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[55]\ttraining's rmse: 0.280046\tvalid_1's rmse: 0.284196\n",
            "[56]\ttraining's rmse: 0.27999\tvalid_1's rmse: 0.284149\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[57]\ttraining's rmse: 0.279935\tvalid_1's rmse: 0.284114\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[58]\ttraining's rmse: 0.279881\tvalid_1's rmse: 0.284081\n",
            "[59]\ttraining's rmse: 0.279824\tvalid_1's rmse: 0.284034\n",
            "[60]\ttraining's rmse: 0.279767\tvalid_1's rmse: 0.283986\n",
            "[61]\ttraining's rmse: 0.279717\tvalid_1's rmse: 0.283958\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[62]\ttraining's rmse: 0.279676\tvalid_1's rmse: 0.283926\n",
            "[63]\ttraining's rmse: 0.279626\tvalid_1's rmse: 0.283898\n",
            "[64]\ttraining's rmse: 0.279577\tvalid_1's rmse: 0.28387\n",
            "[65]\ttraining's rmse: 0.279527\tvalid_1's rmse: 0.283841\n",
            "[66]\ttraining's rmse: 0.279477\tvalid_1's rmse: 0.283805\n",
            "[67]\ttraining's rmse: 0.279428\tvalid_1's rmse: 0.283774\n",
            "[68]\ttraining's rmse: 0.279378\tvalid_1's rmse: 0.283737\n",
            "[69]\ttraining's rmse: 0.279328\tvalid_1's rmse: 0.283701\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[70]\ttraining's rmse: 0.279284\tvalid_1's rmse: 0.283665\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[71]\ttraining's rmse: 0.279241\tvalid_1's rmse: 0.283622\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[72]\ttraining's rmse: 0.279193\tvalid_1's rmse: 0.283588\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[73]\ttraining's rmse: 0.279145\tvalid_1's rmse: 0.283555\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[74]\ttraining's rmse: 0.279098\tvalid_1's rmse: 0.283527\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[75]\ttraining's rmse: 0.279055\tvalid_1's rmse: 0.283484\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[76]\ttraining's rmse: 0.279012\tvalid_1's rmse: 0.283451\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[77]\ttraining's rmse: 0.27897\tvalid_1's rmse: 0.283417\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[78]\ttraining's rmse: 0.278928\tvalid_1's rmse: 0.283385\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[79]\ttraining's rmse: 0.278883\tvalid_1's rmse: 0.283346\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[80]\ttraining's rmse: 0.27884\tvalid_1's rmse: 0.283314\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[81]\ttraining's rmse: 0.27879\tvalid_1's rmse: 0.283276\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[82]\ttraining's rmse: 0.278739\tvalid_1's rmse: 0.283239\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[83]\ttraining's rmse: 0.278692\tvalid_1's rmse: 0.283212\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[84]\ttraining's rmse: 0.278643\tvalid_1's rmse: 0.283177\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[85]\ttraining's rmse: 0.278596\tvalid_1's rmse: 0.283144\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[86]\ttraining's rmse: 0.278554\tvalid_1's rmse: 0.283108\n",
            "[87]\ttraining's rmse: 0.278501\tvalid_1's rmse: 0.283067\n",
            "[88]\ttraining's rmse: 0.278449\tvalid_1's rmse: 0.283033\n",
            "[89]\ttraining's rmse: 0.278396\tvalid_1's rmse: 0.282997\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[90]\ttraining's rmse: 0.278345\tvalid_1's rmse: 0.282963\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[91]\ttraining's rmse: 0.278297\tvalid_1's rmse: 0.28294\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[92]\ttraining's rmse: 0.278249\tvalid_1's rmse: 0.282894\n",
            "[93]\ttraining's rmse: 0.278199\tvalid_1's rmse: 0.282843\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[94]\ttraining's rmse: 0.278157\tvalid_1's rmse: 0.282803\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[95]\ttraining's rmse: 0.278108\tvalid_1's rmse: 0.282755\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[96]\ttraining's rmse: 0.278056\tvalid_1's rmse: 0.282726\n",
            "[97]\ttraining's rmse: 0.278005\tvalid_1's rmse: 0.282705\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[98]\ttraining's rmse: 0.277957\tvalid_1's rmse: 0.282677\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[99]\ttraining's rmse: 0.277906\tvalid_1's rmse: 0.282647\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[100]\ttraining's rmse: 0.277854\tvalid_1's rmse: 0.282619\n",
            "Did not meet early stopping. Best iteration is:\n",
            "[100]\ttraining's rmse: 0.277854\tvalid_1's rmse: 0.282619\n",
            "RMSE = 0.28261889410633684 \n",
            "\n",
            "#########################\n",
            "### Fold 7\n",
            "### Train size 1855 Valid size 201\n",
            "#########################\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Warning] Auto-choosing col-wise multi-threading, the overhead of testing was 0.004137 seconds.\n",
            "You can set `force_col_wise=true` to remove the overhead.\n",
            "[LightGBM] [Info] Total Bins 24333\n",
            "[LightGBM] [Info] Number of data points in the train set: 1855, number of used features: 127\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Info] Start training from score 0.507278\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[1]\ttraining's rmse: 0.282564\tvalid_1's rmse: 0.28771\n",
            "Training until validation scores don't improve for 10 rounds\n",
            "[2]\ttraining's rmse: 0.282507\tvalid_1's rmse: 0.287696\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[3]\ttraining's rmse: 0.282455\tvalid_1's rmse: 0.28768\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[4]\ttraining's rmse: 0.282408\tvalid_1's rmse: 0.287657\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[5]\ttraining's rmse: 0.282357\tvalid_1's rmse: 0.287641\n",
            "[6]\ttraining's rmse: 0.282301\tvalid_1's rmse: 0.287624\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[7]\ttraining's rmse: 0.282245\tvalid_1's rmse: 0.287605\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[8]\ttraining's rmse: 0.282189\tvalid_1's rmse: 0.287593\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[9]\ttraining's rmse: 0.282132\tvalid_1's rmse: 0.287582\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[10]\ttraining's rmse: 0.282078\tvalid_1's rmse: 0.28757\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[11]\ttraining's rmse: 0.282026\tvalid_1's rmse: 0.287551\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[12]\ttraining's rmse: 0.281972\tvalid_1's rmse: 0.287534\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[13]\ttraining's rmse: 0.281918\tvalid_1's rmse: 0.287515\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[14]\ttraining's rmse: 0.281866\tvalid_1's rmse: 0.287501\n",
            "[15]\ttraining's rmse: 0.281811\tvalid_1's rmse: 0.287498\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[16]\ttraining's rmse: 0.281756\tvalid_1's rmse: 0.287478\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[17]\ttraining's rmse: 0.2817\tvalid_1's rmse: 0.287458\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[18]\ttraining's rmse: 0.281644\tvalid_1's rmse: 0.287435\n",
            "[19]\ttraining's rmse: 0.28159\tvalid_1's rmse: 0.287419\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[20]\ttraining's rmse: 0.281534\tvalid_1's rmse: 0.287396\n",
            "[21]\ttraining's rmse: 0.28148\tvalid_1's rmse: 0.287372\n",
            "[22]\ttraining's rmse: 0.281427\tvalid_1's rmse: 0.287357\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[23]\ttraining's rmse: 0.281377\tvalid_1's rmse: 0.28732\n",
            "[24]\ttraining's rmse: 0.281323\tvalid_1's rmse: 0.287299\n",
            "[25]\ttraining's rmse: 0.28127\tvalid_1's rmse: 0.287279\n",
            "[26]\ttraining's rmse: 0.281215\tvalid_1's rmse: 0.287278\n",
            "[27]\ttraining's rmse: 0.281161\tvalid_1's rmse: 0.287269\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[28]\ttraining's rmse: 0.281109\tvalid_1's rmse: 0.287249\n",
            "[29]\ttraining's rmse: 0.281057\tvalid_1's rmse: 0.287242\n",
            "[30]\ttraining's rmse: 0.281004\tvalid_1's rmse: 0.287237\n",
            "[31]\ttraining's rmse: 0.28095\tvalid_1's rmse: 0.287218\n",
            "[32]\ttraining's rmse: 0.280895\tvalid_1's rmse: 0.287201\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[33]\ttraining's rmse: 0.280851\tvalid_1's rmse: 0.287194\n",
            "[34]\ttraining's rmse: 0.280795\tvalid_1's rmse: 0.287174\n",
            "[35]\ttraining's rmse: 0.28074\tvalid_1's rmse: 0.287152\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/opt/conda/lib/python3.7/site-packages/lightgbm/engine.py:181: UserWarning: 'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. Pass 'early_stopping()' callback via 'callbacks' argument instead.\n",
            "  _log_warning(\"'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. \"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[36]\ttraining's rmse: 0.280689\tvalid_1's rmse: 0.287133\n",
            "[37]\ttraining's rmse: 0.280635\tvalid_1's rmse: 0.287118\n",
            "[38]\ttraining's rmse: 0.28058\tvalid_1's rmse: 0.287098\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[39]\ttraining's rmse: 0.280531\tvalid_1's rmse: 0.287084\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[40]\ttraining's rmse: 0.28048\tvalid_1's rmse: 0.287064\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[41]\ttraining's rmse: 0.280429\tvalid_1's rmse: 0.287052\n",
            "[42]\ttraining's rmse: 0.280381\tvalid_1's rmse: 0.287025\n",
            "[43]\ttraining's rmse: 0.280326\tvalid_1's rmse: 0.287001\n",
            "[44]\ttraining's rmse: 0.280272\tvalid_1's rmse: 0.286976\n",
            "[45]\ttraining's rmse: 0.280218\tvalid_1's rmse: 0.28695\n",
            "[46]\ttraining's rmse: 0.280164\tvalid_1's rmse: 0.286929\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[47]\ttraining's rmse: 0.280113\tvalid_1's rmse: 0.286905\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[48]\ttraining's rmse: 0.280064\tvalid_1's rmse: 0.286892\n",
            "[49]\ttraining's rmse: 0.280011\tvalid_1's rmse: 0.286878\n",
            "[50]\ttraining's rmse: 0.279957\tvalid_1's rmse: 0.286863\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[51]\ttraining's rmse: 0.2799\tvalid_1's rmse: 0.286847\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[52]\ttraining's rmse: 0.279845\tvalid_1's rmse: 0.286832\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[53]\ttraining's rmse: 0.279789\tvalid_1's rmse: 0.286818\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[54]\ttraining's rmse: 0.279732\tvalid_1's rmse: 0.286796\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[55]\ttraining's rmse: 0.279675\tvalid_1's rmse: 0.286778\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[56]\ttraining's rmse: 0.279621\tvalid_1's rmse: 0.286751\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[57]\ttraining's rmse: 0.279572\tvalid_1's rmse: 0.286735\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[58]\ttraining's rmse: 0.279525\tvalid_1's rmse: 0.286727\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[59]\ttraining's rmse: 0.27947\tvalid_1's rmse: 0.286706\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[60]\ttraining's rmse: 0.279414\tvalid_1's rmse: 0.286687\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[61]\ttraining's rmse: 0.279364\tvalid_1's rmse: 0.286671\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[62]\ttraining's rmse: 0.279317\tvalid_1's rmse: 0.286648\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[63]\ttraining's rmse: 0.279268\tvalid_1's rmse: 0.286625\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[64]\ttraining's rmse: 0.279217\tvalid_1's rmse: 0.28661\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[65]\ttraining's rmse: 0.279167\tvalid_1's rmse: 0.286591\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[66]\ttraining's rmse: 0.279114\tvalid_1's rmse: 0.286576\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[67]\ttraining's rmse: 0.279063\tvalid_1's rmse: 0.28656\n",
            "[68]\ttraining's rmse: 0.279012\tvalid_1's rmse: 0.286553\n",
            "[69]\ttraining's rmse: 0.278959\tvalid_1's rmse: 0.286543\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[70]\ttraining's rmse: 0.27891\tvalid_1's rmse: 0.286535\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[71]\ttraining's rmse: 0.278866\tvalid_1's rmse: 0.286512\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[72]\ttraining's rmse: 0.278822\tvalid_1's rmse: 0.286488\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[73]\ttraining's rmse: 0.278778\tvalid_1's rmse: 0.286468\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[74]\ttraining's rmse: 0.278734\tvalid_1's rmse: 0.286444\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[75]\ttraining's rmse: 0.27869\tvalid_1's rmse: 0.286421\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[76]\ttraining's rmse: 0.278643\tvalid_1's rmse: 0.286403\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[77]\ttraining's rmse: 0.2786\tvalid_1's rmse: 0.286379\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[78]\ttraining's rmse: 0.278553\tvalid_1's rmse: 0.286362\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[79]\ttraining's rmse: 0.278507\tvalid_1's rmse: 0.286345\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[80]\ttraining's rmse: 0.278462\tvalid_1's rmse: 0.286313\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[81]\ttraining's rmse: 0.278413\tvalid_1's rmse: 0.286293\n",
            "[82]\ttraining's rmse: 0.278362\tvalid_1's rmse: 0.286267\n",
            "[83]\ttraining's rmse: 0.278312\tvalid_1's rmse: 0.286241\n",
            "[84]\ttraining's rmse: 0.278261\tvalid_1's rmse: 0.286214\n",
            "[85]\ttraining's rmse: 0.278214\tvalid_1's rmse: 0.286192\n",
            "[86]\ttraining's rmse: 0.278165\tvalid_1's rmse: 0.286177\n",
            "[87]\ttraining's rmse: 0.278109\tvalid_1's rmse: 0.286147\n",
            "[88]\ttraining's rmse: 0.278055\tvalid_1's rmse: 0.286127\n",
            "[89]\ttraining's rmse: 0.278\tvalid_1's rmse: 0.2861\n",
            "[90]\ttraining's rmse: 0.277948\tvalid_1's rmse: 0.286081\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[91]\ttraining's rmse: 0.277897\tvalid_1's rmse: 0.28606\n",
            "[92]\ttraining's rmse: 0.277844\tvalid_1's rmse: 0.286028\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[93]\ttraining's rmse: 0.277793\tvalid_1's rmse: 0.285999\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[94]\ttraining's rmse: 0.277748\tvalid_1's rmse: 0.285978\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[95]\ttraining's rmse: 0.277696\tvalid_1's rmse: 0.28595\n",
            "[96]\ttraining's rmse: 0.277643\tvalid_1's rmse: 0.285937\n",
            "[97]\ttraining's rmse: 0.277595\tvalid_1's rmse: 0.285929\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[98]\ttraining's rmse: 0.27755\tvalid_1's rmse: 0.285913\n",
            "[99]\ttraining's rmse: 0.277497\tvalid_1's rmse: 0.2859\n",
            "[100]\ttraining's rmse: 0.277445\tvalid_1's rmse: 0.28589\n",
            "Did not meet early stopping. Best iteration is:\n",
            "[100]\ttraining's rmse: 0.277445\tvalid_1's rmse: 0.28589\n",
            "RMSE = 0.28588995673747 \n",
            "\n",
            "#########################\n",
            "### Fold 8\n",
            "### Train size 1855 Valid size 201\n",
            "#########################\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Warning] Auto-choosing col-wise multi-threading, the overhead of testing was 0.004523 seconds.\n",
            "You can set `force_col_wise=true` to remove the overhead.\n",
            "[LightGBM] [Info] Total Bins 24368\n",
            "[LightGBM] [Info] Number of data points in the train set: 1855, number of used features: 127\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Info] Start training from score 0.507278\n",
            "[1]\ttraining's rmse: 0.282479\tvalid_1's rmse: 0.288418\n",
            "Training until validation scores don't improve for 10 rounds\n",
            "[2]\ttraining's rmse: 0.282427\tvalid_1's rmse: 0.288373\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[3]\ttraining's rmse: 0.282379\tvalid_1's rmse: 0.288332\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[4]\ttraining's rmse: 0.282332\tvalid_1's rmse: 0.288292\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[5]\ttraining's rmse: 0.28229\tvalid_1's rmse: 0.288255\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[6]\ttraining's rmse: 0.282236\tvalid_1's rmse: 0.288214\n",
            "[7]\ttraining's rmse: 0.282182\tvalid_1's rmse: 0.288187\n",
            "[8]\ttraining's rmse: 0.282126\tvalid_1's rmse: 0.288149\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[9]\ttraining's rmse: 0.28207\tvalid_1's rmse: 0.28811\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[10]\ttraining's rmse: 0.282015\tvalid_1's rmse: 0.288076\n",
            "[11]\ttraining's rmse: 0.281959\tvalid_1's rmse: 0.288028\n",
            "[12]\ttraining's rmse: 0.281904\tvalid_1's rmse: 0.287982\n",
            "[13]\ttraining's rmse: 0.281848\tvalid_1's rmse: 0.287933\n",
            "[14]\ttraining's rmse: 0.281793\tvalid_1's rmse: 0.287892\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[15]\ttraining's rmse: 0.281741\tvalid_1's rmse: 0.28786\n",
            "[16]\ttraining's rmse: 0.281685\tvalid_1's rmse: 0.287823\n",
            "[17]\ttraining's rmse: 0.281629\tvalid_1's rmse: 0.287777\n",
            "[18]\ttraining's rmse: 0.281572\tvalid_1's rmse: 0.287727\n",
            "[19]\ttraining's rmse: 0.281515\tvalid_1's rmse: 0.287687\n",
            "[20]\ttraining's rmse: 0.281458\tvalid_1's rmse: 0.287628\n",
            "[21]\ttraining's rmse: 0.281404\tvalid_1's rmse: 0.287601\n",
            "[22]\ttraining's rmse: 0.281354\tvalid_1's rmse: 0.287555\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[23]\ttraining's rmse: 0.281309\tvalid_1's rmse: 0.287518\n",
            "[24]\ttraining's rmse: 0.281259\tvalid_1's rmse: 0.287473\n",
            "[25]\ttraining's rmse: 0.28121\tvalid_1's rmse: 0.287432\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[26]\ttraining's rmse: 0.281161\tvalid_1's rmse: 0.287399\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[27]\ttraining's rmse: 0.281123\tvalid_1's rmse: 0.287383\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[28]\ttraining's rmse: 0.281075\tvalid_1's rmse: 0.287343\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[29]\ttraining's rmse: 0.281025\tvalid_1's rmse: 0.287328\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[30]\ttraining's rmse: 0.280974\tvalid_1's rmse: 0.287298\n",
            "[31]\ttraining's rmse: 0.280919\tvalid_1's rmse: 0.287257\n",
            "[32]\ttraining's rmse: 0.280865\tvalid_1's rmse: 0.287215\n",
            "[33]\ttraining's rmse: 0.280813\tvalid_1's rmse: 0.287186\n",
            "[34]\ttraining's rmse: 0.28076\tvalid_1's rmse: 0.287146\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/opt/conda/lib/python3.7/site-packages/lightgbm/engine.py:181: UserWarning: 'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. Pass 'early_stopping()' callback via 'callbacks' argument instead.\n",
            "  _log_warning(\"'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. \"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "[35]\ttraining's rmse: 0.280706\tvalid_1's rmse: 0.287107\n",
            "[36]\ttraining's rmse: 0.28066\tvalid_1's rmse: 0.287092\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[37]\ttraining's rmse: 0.280613\tvalid_1's rmse: 0.287075\n",
            "[38]\ttraining's rmse: 0.280565\tvalid_1's rmse: 0.287057\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[39]\ttraining's rmse: 0.28052\tvalid_1's rmse: 0.287038\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[40]\ttraining's rmse: 0.280475\tvalid_1's rmse: 0.287025\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[41]\ttraining's rmse: 0.280431\tvalid_1's rmse: 0.286994\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[42]\ttraining's rmse: 0.280386\tvalid_1's rmse: 0.286954\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[43]\ttraining's rmse: 0.280342\tvalid_1's rmse: 0.286923\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[44]\ttraining's rmse: 0.280298\tvalid_1's rmse: 0.286892\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[45]\ttraining's rmse: 0.280249\tvalid_1's rmse: 0.286858\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[46]\ttraining's rmse: 0.280198\tvalid_1's rmse: 0.286824\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[47]\ttraining's rmse: 0.280149\tvalid_1's rmse: 0.286785\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[48]\ttraining's rmse: 0.280102\tvalid_1's rmse: 0.286748\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[49]\ttraining's rmse: 0.280051\tvalid_1's rmse: 0.28671\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[50]\ttraining's rmse: 0.280001\tvalid_1's rmse: 0.286678\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[51]\ttraining's rmse: 0.279953\tvalid_1's rmse: 0.286647\n",
            "[52]\ttraining's rmse: 0.279898\tvalid_1's rmse: 0.286609\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[53]\ttraining's rmse: 0.279845\tvalid_1's rmse: 0.286574\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[54]\ttraining's rmse: 0.279796\tvalid_1's rmse: 0.286538\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[55]\ttraining's rmse: 0.279743\tvalid_1's rmse: 0.286512\n",
            "[56]\ttraining's rmse: 0.279685\tvalid_1's rmse: 0.28648\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[57]\ttraining's rmse: 0.279636\tvalid_1's rmse: 0.286452\n",
            "[58]\ttraining's rmse: 0.279587\tvalid_1's rmse: 0.286427\n",
            "[59]\ttraining's rmse: 0.279529\tvalid_1's rmse: 0.286393\n",
            "[60]\ttraining's rmse: 0.279472\tvalid_1's rmse: 0.286361\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[61]\ttraining's rmse: 0.279423\tvalid_1's rmse: 0.286314\n",
            "[62]\ttraining's rmse: 0.279376\tvalid_1's rmse: 0.286297\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[63]\ttraining's rmse: 0.279327\tvalid_1's rmse: 0.28625\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[64]\ttraining's rmse: 0.279279\tvalid_1's rmse: 0.286201\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[65]\ttraining's rmse: 0.279231\tvalid_1's rmse: 0.286155\n",
            "[66]\ttraining's rmse: 0.279184\tvalid_1's rmse: 0.286111\n",
            "[67]\ttraining's rmse: 0.279137\tvalid_1's rmse: 0.286073\n",
            "[68]\ttraining's rmse: 0.27909\tvalid_1's rmse: 0.286028\n",
            "[69]\ttraining's rmse: 0.279043\tvalid_1's rmse: 0.285982\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[70]\ttraining's rmse: 0.278998\tvalid_1's rmse: 0.285956\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[71]\ttraining's rmse: 0.278952\tvalid_1's rmse: 0.285925\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[72]\ttraining's rmse: 0.278906\tvalid_1's rmse: 0.28589\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[73]\ttraining's rmse: 0.278862\tvalid_1's rmse: 0.285863\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[74]\ttraining's rmse: 0.278816\tvalid_1's rmse: 0.285827\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[75]\ttraining's rmse: 0.278771\tvalid_1's rmse: 0.2858\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[76]\ttraining's rmse: 0.278729\tvalid_1's rmse: 0.285771\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[77]\ttraining's rmse: 0.278688\tvalid_1's rmse: 0.285739\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[78]\ttraining's rmse: 0.278647\tvalid_1's rmse: 0.285713\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[79]\ttraining's rmse: 0.278604\tvalid_1's rmse: 0.285684\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[80]\ttraining's rmse: 0.278563\tvalid_1's rmse: 0.285653\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[81]\ttraining's rmse: 0.278515\tvalid_1's rmse: 0.285611\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[82]\ttraining's rmse: 0.278466\tvalid_1's rmse: 0.28558\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[83]\ttraining's rmse: 0.278417\tvalid_1's rmse: 0.285549\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[84]\ttraining's rmse: 0.278373\tvalid_1's rmse: 0.285521\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[85]\ttraining's rmse: 0.278325\tvalid_1's rmse: 0.28549\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[86]\ttraining's rmse: 0.278279\tvalid_1's rmse: 0.285481\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[87]\ttraining's rmse: 0.278225\tvalid_1's rmse: 0.285445\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[88]\ttraining's rmse: 0.278173\tvalid_1's rmse: 0.285417\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[89]\ttraining's rmse: 0.278122\tvalid_1's rmse: 0.285388\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[90]\ttraining's rmse: 0.278071\tvalid_1's rmse: 0.285359\n",
            "[91]\ttraining's rmse: 0.278022\tvalid_1's rmse: 0.28533\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[92]\ttraining's rmse: 0.277967\tvalid_1's rmse: 0.28531\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[93]\ttraining's rmse: 0.277912\tvalid_1's rmse: 0.285283\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[94]\ttraining's rmse: 0.277865\tvalid_1's rmse: 0.285255\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[95]\ttraining's rmse: 0.277814\tvalid_1's rmse: 0.285223\n",
            "[96]\ttraining's rmse: 0.277757\tvalid_1's rmse: 0.285191\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[97]\ttraining's rmse: 0.277711\tvalid_1's rmse: 0.285163\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[98]\ttraining's rmse: 0.277667\tvalid_1's rmse: 0.285131\n",
            "[99]\ttraining's rmse: 0.277609\tvalid_1's rmse: 0.2851\n",
            "[100]\ttraining's rmse: 0.277551\tvalid_1's rmse: 0.28507\n",
            "Did not meet early stopping. Best iteration is:\n",
            "[100]\ttraining's rmse: 0.277551\tvalid_1's rmse: 0.28507\n",
            "RMSE = 0.28506966860704036 \n",
            "\n",
            "#########################\n",
            "### Fold 9\n",
            "### Train size 1855 Valid size 201\n",
            "#########################\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Warning] Auto-choosing col-wise multi-threading, the overhead of testing was 0.004314 seconds.\n",
            "You can set `force_col_wise=true` to remove the overhead.\n",
            "[LightGBM] [Info] Total Bins 24350\n",
            "[LightGBM] [Info] Number of data points in the train set: 1855, number of used features: 127\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Info] Start training from score 0.507278\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[1]\ttraining's rmse: 0.282574\tvalid_1's rmse: 0.287549\n",
            "Training until validation scores don't improve for 10 rounds\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[2]\ttraining's rmse: 0.282518\tvalid_1's rmse: 0.287503\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[3]\ttraining's rmse: 0.282466\tvalid_1's rmse: 0.287489\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[4]\ttraining's rmse: 0.282414\tvalid_1's rmse: 0.287475\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[5]\ttraining's rmse: 0.282362\tvalid_1's rmse: 0.287464\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[6]\ttraining's rmse: 0.282311\tvalid_1's rmse: 0.287422\n",
            "[7]\ttraining's rmse: 0.282257\tvalid_1's rmse: 0.287388\n",
            "[8]\ttraining's rmse: 0.282203\tvalid_1's rmse: 0.287347\n",
            "[9]\ttraining's rmse: 0.282147\tvalid_1's rmse: 0.287307\n",
            "[10]\ttraining's rmse: 0.282091\tvalid_1's rmse: 0.287274\n",
            "[11]\ttraining's rmse: 0.282036\tvalid_1's rmse: 0.287234\n",
            "[12]\ttraining's rmse: 0.28198\tvalid_1's rmse: 0.287199\n",
            "[13]\ttraining's rmse: 0.281927\tvalid_1's rmse: 0.287166\n",
            "[14]\ttraining's rmse: 0.281874\tvalid_1's rmse: 0.287133\n",
            "[15]\ttraining's rmse: 0.281817\tvalid_1's rmse: 0.287123\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[16]\ttraining's rmse: 0.281762\tvalid_1's rmse: 0.287098\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[17]\ttraining's rmse: 0.281706\tvalid_1's rmse: 0.287076\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[18]\ttraining's rmse: 0.281651\tvalid_1's rmse: 0.287058\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[19]\ttraining's rmse: 0.281598\tvalid_1's rmse: 0.287038\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[20]\ttraining's rmse: 0.281543\tvalid_1's rmse: 0.28702\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[21]\ttraining's rmse: 0.281493\tvalid_1's rmse: 0.286997\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[22]\ttraining's rmse: 0.281441\tvalid_1's rmse: 0.286961\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[23]\ttraining's rmse: 0.28139\tvalid_1's rmse: 0.286958\n",
            "[24]\ttraining's rmse: 0.281337\tvalid_1's rmse: 0.286925\n",
            "[25]\ttraining's rmse: 0.281283\tvalid_1's rmse: 0.286886\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[26]\ttraining's rmse: 0.281226\tvalid_1's rmse: 0.286851\n",
            "[27]\ttraining's rmse: 0.281172\tvalid_1's rmse: 0.286835\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[28]\ttraining's rmse: 0.281119\tvalid_1's rmse: 0.286811\n",
            "[29]\ttraining's rmse: 0.28106\tvalid_1's rmse: 0.286768\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[30]\ttraining's rmse: 0.281005\tvalid_1's rmse: 0.286737\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[31]\ttraining's rmse: 0.280951\tvalid_1's rmse: 0.286714\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[32]\ttraining's rmse: 0.280896\tvalid_1's rmse: 0.28669\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[33]\ttraining's rmse: 0.28084\tvalid_1's rmse: 0.286676\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/opt/conda/lib/python3.7/site-packages/lightgbm/engine.py:181: UserWarning: 'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. Pass 'early_stopping()' callback via 'callbacks' argument instead.\n",
            "  _log_warning(\"'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. \"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[34]\ttraining's rmse: 0.280789\tvalid_1's rmse: 0.28666\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[35]\ttraining's rmse: 0.280735\tvalid_1's rmse: 0.286638\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[36]\ttraining's rmse: 0.280681\tvalid_1's rmse: 0.286619\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[37]\ttraining's rmse: 0.280627\tvalid_1's rmse: 0.286606\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[38]\ttraining's rmse: 0.280573\tvalid_1's rmse: 0.286593\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[39]\ttraining's rmse: 0.280518\tvalid_1's rmse: 0.286577\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[40]\ttraining's rmse: 0.280469\tvalid_1's rmse: 0.286576\n",
            "[41]\ttraining's rmse: 0.28042\tvalid_1's rmse: 0.286547\n",
            "[42]\ttraining's rmse: 0.280371\tvalid_1's rmse: 0.286519\n",
            "[43]\ttraining's rmse: 0.280322\tvalid_1's rmse: 0.28649\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[44]\ttraining's rmse: 0.280273\tvalid_1's rmse: 0.286465\n",
            "[45]\ttraining's rmse: 0.280223\tvalid_1's rmse: 0.286436\n",
            "[46]\ttraining's rmse: 0.280171\tvalid_1's rmse: 0.2864\n",
            "[47]\ttraining's rmse: 0.280118\tvalid_1's rmse: 0.286383\n",
            "[48]\ttraining's rmse: 0.280066\tvalid_1's rmse: 0.286371\n",
            "[49]\ttraining's rmse: 0.280013\tvalid_1's rmse: 0.286338\n",
            "[50]\ttraining's rmse: 0.27996\tvalid_1's rmse: 0.286307\n",
            "[51]\ttraining's rmse: 0.279904\tvalid_1's rmse: 0.286283\n",
            "[52]\ttraining's rmse: 0.279851\tvalid_1's rmse: 0.286247\n",
            "[53]\ttraining's rmse: 0.279798\tvalid_1's rmse: 0.286219\n",
            "[54]\ttraining's rmse: 0.279746\tvalid_1's rmse: 0.286189\n",
            "[55]\ttraining's rmse: 0.279696\tvalid_1's rmse: 0.286156\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[56]\ttraining's rmse: 0.279641\tvalid_1's rmse: 0.286137\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[57]\ttraining's rmse: 0.279595\tvalid_1's rmse: 0.286126\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[58]\ttraining's rmse: 0.279551\tvalid_1's rmse: 0.286115\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[59]\ttraining's rmse: 0.2795\tvalid_1's rmse: 0.286095\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[60]\ttraining's rmse: 0.279447\tvalid_1's rmse: 0.286064\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[61]\ttraining's rmse: 0.279402\tvalid_1's rmse: 0.286035\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[62]\ttraining's rmse: 0.279353\tvalid_1's rmse: 0.28602\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[63]\ttraining's rmse: 0.279305\tvalid_1's rmse: 0.286002\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[64]\ttraining's rmse: 0.279257\tvalid_1's rmse: 0.285987\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[65]\ttraining's rmse: 0.279209\tvalid_1's rmse: 0.285967\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[66]\ttraining's rmse: 0.279161\tvalid_1's rmse: 0.285938\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[67]\ttraining's rmse: 0.279113\tvalid_1's rmse: 0.285918\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[68]\ttraining's rmse: 0.279065\tvalid_1's rmse: 0.285889\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[69]\ttraining's rmse: 0.279017\tvalid_1's rmse: 0.285866\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[70]\ttraining's rmse: 0.278968\tvalid_1's rmse: 0.285848\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[71]\ttraining's rmse: 0.278922\tvalid_1's rmse: 0.285821\n",
            "[72]\ttraining's rmse: 0.278872\tvalid_1's rmse: 0.285786\n",
            "[73]\ttraining's rmse: 0.278824\tvalid_1's rmse: 0.28576\n",
            "[74]\ttraining's rmse: 0.278776\tvalid_1's rmse: 0.285733\n",
            "[75]\ttraining's rmse: 0.278727\tvalid_1's rmse: 0.285712\n",
            "[76]\ttraining's rmse: 0.278675\tvalid_1's rmse: 0.285685\n",
            "[77]\ttraining's rmse: 0.278622\tvalid_1's rmse: 0.285655\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[78]\ttraining's rmse: 0.278576\tvalid_1's rmse: 0.285643\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[79]\ttraining's rmse: 0.27853\tvalid_1's rmse: 0.285607\n",
            "[80]\ttraining's rmse: 0.278478\tvalid_1's rmse: 0.285576\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[81]\ttraining's rmse: 0.278426\tvalid_1's rmse: 0.285568\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[82]\ttraining's rmse: 0.278372\tvalid_1's rmse: 0.285556\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[83]\ttraining's rmse: 0.278324\tvalid_1's rmse: 0.28555\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[84]\ttraining's rmse: 0.278271\tvalid_1's rmse: 0.285546\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[85]\ttraining's rmse: 0.278221\tvalid_1's rmse: 0.285539\n",
            "[86]\ttraining's rmse: 0.278175\tvalid_1's rmse: 0.285526\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[87]\ttraining's rmse: 0.278122\tvalid_1's rmse: 0.285505\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[88]\ttraining's rmse: 0.278072\tvalid_1's rmse: 0.285486\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[89]\ttraining's rmse: 0.278021\tvalid_1's rmse: 0.285462\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[90]\ttraining's rmse: 0.277972\tvalid_1's rmse: 0.285438\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[91]\ttraining's rmse: 0.277927\tvalid_1's rmse: 0.285423\n",
            "[92]\ttraining's rmse: 0.277877\tvalid_1's rmse: 0.285401\n",
            "[93]\ttraining's rmse: 0.277826\tvalid_1's rmse: 0.285379\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[94]\ttraining's rmse: 0.277783\tvalid_1's rmse: 0.285362\n",
            "[95]\ttraining's rmse: 0.277733\tvalid_1's rmse: 0.285341\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[96]\ttraining's rmse: 0.277686\tvalid_1's rmse: 0.285318\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[97]\ttraining's rmse: 0.27764\tvalid_1's rmse: 0.285304\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[98]\ttraining's rmse: 0.277594\tvalid_1's rmse: 0.285294\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[99]\ttraining's rmse: 0.27755\tvalid_1's rmse: 0.285273\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[100]\ttraining's rmse: 0.2775\tvalid_1's rmse: 0.28525\n",
            "Did not meet early stopping. Best iteration is:\n",
            "[100]\ttraining's rmse: 0.2775\tvalid_1's rmse: 0.28525\n",
            "RMSE = 0.28524971172568736 \n",
            "\n",
            "#########################\n",
            "### Fold 10\n",
            "### Train size 1855 Valid size 201\n",
            "#########################\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Warning] Auto-choosing col-wise multi-threading, the overhead of testing was 0.004169 seconds.\n",
            "You can set `force_col_wise=true` to remove the overhead.\n",
            "[LightGBM] [Info] Total Bins 24377\n",
            "[LightGBM] [Info] Number of data points in the train set: 1855, number of used features: 127\n",
            "[LightGBM] [Warning] Found whitespace in feature_names, replace with underlines\n",
            "[LightGBM] [Info] Start training from score 0.507278\n",
            "[1]\ttraining's rmse: 0.282891\tvalid_1's rmse: 0.284632\n",
            "Training until validation scores don't improve for 10 rounds\n",
            "[2]\ttraining's rmse: 0.28283\tvalid_1's rmse: 0.284616\n",
            "[3]\ttraining's rmse: 0.282775\tvalid_1's rmse: 0.284592\n",
            "[4]\ttraining's rmse: 0.282726\tvalid_1's rmse: 0.284574\n",
            "[5]\ttraining's rmse: 0.282669\tvalid_1's rmse: 0.284547\n",
            "[6]\ttraining's rmse: 0.282616\tvalid_1's rmse: 0.284522\n",
            "[7]\ttraining's rmse: 0.282562\tvalid_1's rmse: 0.28449\n",
            "[8]\ttraining's rmse: 0.282508\tvalid_1's rmse: 0.284463\n",
            "[9]\ttraining's rmse: 0.282454\tvalid_1's rmse: 0.284435\n",
            "[10]\ttraining's rmse: 0.2824\tvalid_1's rmse: 0.28441\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[11]\ttraining's rmse: 0.282347\tvalid_1's rmse: 0.284391\n",
            "[12]\ttraining's rmse: 0.282292\tvalid_1's rmse: 0.28437\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[13]\ttraining's rmse: 0.28224\tvalid_1's rmse: 0.284351\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[14]\ttraining's rmse: 0.282188\tvalid_1's rmse: 0.284332\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[15]\ttraining's rmse: 0.282135\tvalid_1's rmse: 0.284297\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[16]\ttraining's rmse: 0.282082\tvalid_1's rmse: 0.284262\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[17]\ttraining's rmse: 0.282026\tvalid_1's rmse: 0.28424\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[18]\ttraining's rmse: 0.281969\tvalid_1's rmse: 0.284206\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[19]\ttraining's rmse: 0.281914\tvalid_1's rmse: 0.284185\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[20]\ttraining's rmse: 0.281857\tvalid_1's rmse: 0.284151\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[21]\ttraining's rmse: 0.281807\tvalid_1's rmse: 0.284132\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[22]\ttraining's rmse: 0.281755\tvalid_1's rmse: 0.284116\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[23]\ttraining's rmse: 0.281709\tvalid_1's rmse: 0.284093\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[24]\ttraining's rmse: 0.281658\tvalid_1's rmse: 0.284076\n",
            "[25]\ttraining's rmse: 0.281607\tvalid_1's rmse: 0.284056\n",
            "[26]\ttraining's rmse: 0.281553\tvalid_1's rmse: 0.284024\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[27]\ttraining's rmse: 0.281505\tvalid_1's rmse: 0.284004\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[28]\ttraining's rmse: 0.281455\tvalid_1's rmse: 0.283969\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[29]\ttraining's rmse: 0.281401\tvalid_1's rmse: 0.283927\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[30]\ttraining's rmse: 0.28135\tvalid_1's rmse: 0.283884\n",
            "[31]\ttraining's rmse: 0.281292\tvalid_1's rmse: 0.283855\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[32]\ttraining's rmse: 0.281234\tvalid_1's rmse: 0.283828\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[33]\ttraining's rmse: 0.281181\tvalid_1's rmse: 0.283809\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/opt/conda/lib/python3.7/site-packages/lightgbm/engine.py:181: UserWarning: 'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. Pass 'early_stopping()' callback via 'callbacks' argument instead.\n",
            "  _log_warning(\"'early_stopping_rounds' argument is deprecated and will be removed in a future release of LightGBM. \"\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "[34]\ttraining's rmse: 0.281126\tvalid_1's rmse: 0.283775\n",
            "[35]\ttraining's rmse: 0.281071\tvalid_1's rmse: 0.28375\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[36]\ttraining's rmse: 0.28102\tvalid_1's rmse: 0.28373\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[37]\ttraining's rmse: 0.28097\tvalid_1's rmse: 0.283706\n",
            "[38]\ttraining's rmse: 0.28092\tvalid_1's rmse: 0.28369\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[39]\ttraining's rmse: 0.280869\tvalid_1's rmse: 0.283678\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[40]\ttraining's rmse: 0.280818\tvalid_1's rmse: 0.283672\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[41]\ttraining's rmse: 0.280768\tvalid_1's rmse: 0.283651\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[42]\ttraining's rmse: 0.280718\tvalid_1's rmse: 0.283623\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[43]\ttraining's rmse: 0.280666\tvalid_1's rmse: 0.283597\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[44]\ttraining's rmse: 0.280615\tvalid_1's rmse: 0.283573\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[45]\ttraining's rmse: 0.280563\tvalid_1's rmse: 0.283548\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[46]\ttraining's rmse: 0.280508\tvalid_1's rmse: 0.283529\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[47]\ttraining's rmse: 0.280453\tvalid_1's rmse: 0.283514\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[48]\ttraining's rmse: 0.280399\tvalid_1's rmse: 0.283493\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[49]\ttraining's rmse: 0.280344\tvalid_1's rmse: 0.283479\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[50]\ttraining's rmse: 0.280289\tvalid_1's rmse: 0.283464\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[51]\ttraining's rmse: 0.280235\tvalid_1's rmse: 0.283435\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[52]\ttraining's rmse: 0.280183\tvalid_1's rmse: 0.283426\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[53]\ttraining's rmse: 0.28013\tvalid_1's rmse: 0.283415\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[54]\ttraining's rmse: 0.280077\tvalid_1's rmse: 0.283398\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[55]\ttraining's rmse: 0.280023\tvalid_1's rmse: 0.283381\n",
            "[56]\ttraining's rmse: 0.279963\tvalid_1's rmse: 0.283362\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[57]\ttraining's rmse: 0.279913\tvalid_1's rmse: 0.283331\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[58]\ttraining's rmse: 0.279864\tvalid_1's rmse: 0.283303\n",
            "[59]\ttraining's rmse: 0.279804\tvalid_1's rmse: 0.283283\n",
            "[60]\ttraining's rmse: 0.279745\tvalid_1's rmse: 0.283259\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[61]\ttraining's rmse: 0.279694\tvalid_1's rmse: 0.283237\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[62]\ttraining's rmse: 0.279646\tvalid_1's rmse: 0.283212\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[63]\ttraining's rmse: 0.279595\tvalid_1's rmse: 0.283191\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[64]\ttraining's rmse: 0.279547\tvalid_1's rmse: 0.283174\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[65]\ttraining's rmse: 0.279496\tvalid_1's rmse: 0.283152\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[66]\ttraining's rmse: 0.279445\tvalid_1's rmse: 0.283143\n",
            "[67]\ttraining's rmse: 0.279394\tvalid_1's rmse: 0.283121\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[68]\ttraining's rmse: 0.279344\tvalid_1's rmse: 0.283112\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[69]\ttraining's rmse: 0.279294\tvalid_1's rmse: 0.283104\n",
            "[70]\ttraining's rmse: 0.279243\tvalid_1's rmse: 0.283086\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[71]\ttraining's rmse: 0.279195\tvalid_1's rmse: 0.283068\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[72]\ttraining's rmse: 0.279148\tvalid_1's rmse: 0.28305\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[73]\ttraining's rmse: 0.279102\tvalid_1's rmse: 0.28302\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[74]\ttraining's rmse: 0.279056\tvalid_1's rmse: 0.282996\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[75]\ttraining's rmse: 0.279009\tvalid_1's rmse: 0.282977\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[76]\ttraining's rmse: 0.278963\tvalid_1's rmse: 0.282956\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[77]\ttraining's rmse: 0.278916\tvalid_1's rmse: 0.282931\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[78]\ttraining's rmse: 0.27887\tvalid_1's rmse: 0.282905\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[79]\ttraining's rmse: 0.278822\tvalid_1's rmse: 0.282884\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[80]\ttraining's rmse: 0.278776\tvalid_1's rmse: 0.282859\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[81]\ttraining's rmse: 0.278725\tvalid_1's rmse: 0.282837\n",
            "[82]\ttraining's rmse: 0.27867\tvalid_1's rmse: 0.282822\n",
            "[83]\ttraining's rmse: 0.278616\tvalid_1's rmse: 0.282804\n",
            "[84]\ttraining's rmse: 0.278561\tvalid_1's rmse: 0.282785\n",
            "[85]\ttraining's rmse: 0.278507\tvalid_1's rmse: 0.282762\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[86]\ttraining's rmse: 0.27846\tvalid_1's rmse: 0.282742\n",
            "[87]\ttraining's rmse: 0.278403\tvalid_1's rmse: 0.282716\n",
            "[88]\ttraining's rmse: 0.278347\tvalid_1's rmse: 0.28269\n",
            "[89]\ttraining's rmse: 0.27829\tvalid_1's rmse: 0.282667\n",
            "[90]\ttraining's rmse: 0.278234\tvalid_1's rmse: 0.282643\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[91]\ttraining's rmse: 0.278187\tvalid_1's rmse: 0.282627\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[92]\ttraining's rmse: 0.278136\tvalid_1's rmse: 0.282602\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[93]\ttraining's rmse: 0.278085\tvalid_1's rmse: 0.28258\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[94]\ttraining's rmse: 0.278034\tvalid_1's rmse: 0.282564\n",
            "[95]\ttraining's rmse: 0.277984\tvalid_1's rmse: 0.282534\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[96]\ttraining's rmse: 0.277933\tvalid_1's rmse: 0.282514\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[97]\ttraining's rmse: 0.277886\tvalid_1's rmse: 0.282493\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[98]\ttraining's rmse: 0.27784\tvalid_1's rmse: 0.282471\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[99]\ttraining's rmse: 0.277789\tvalid_1's rmse: 0.282455\n",
            "[LightGBM] [Warning] No further splits with positive gain, best gain: -inf\n",
            "[100]\ttraining's rmse: 0.277738\tvalid_1's rmse: 0.282438\n",
            "Did not meet early stopping. Best iteration is:\n",
            "[100]\ttraining's rmse: 0.277738\tvalid_1's rmse: 0.282438\n",
            "RMSE = 0.28243795110733305 \n",
            "\n",
            "#########################\n",
            "OVERALL RMSE = 0.280809149500095 \n",
            "\n",
            "CPU times: user 6.28 s, sys: 83 ms, total: 6.37 s\n",
            "Wall time: 6.41 s\n"
          ]
        }
      ],
      "source": [
        "%%time\n",
        "importances = []\n",
        "importances2 = []\n",
        "oof = np.zeros(len(train))\n",
        "preds = np.zeros(len(holdout))\n",
        "os.system('mkdir lgbm_models')\n",
        "\n",
        "skf = GroupKFold(n_splits=FOLDS)\n",
        "for fold,(train_idx, valid_idx) in enumerate(skf.split(\n",
        "            train, train.target, train.group )):\n",
        "        \n",
        "    print('#'*25)\n",
        "    print('### Fold',fold+1)\n",
        "    print('### Train size',len(train_idx),'Valid size',len(valid_idx))\n",
        "    print('#'*25)\n",
        "    \n",
        "    # TRAIN, VALID, HOLDOUT FOR FOLD K\n",
        "    X_train = train.loc[train_idx, FEATURES]\n",
        "    y_train = train.loc[train_idx,'target']\n",
        "    X_valid = train.loc[valid_idx, FEATURES]\n",
        "    y_valid = train.loc[valid_idx, 'target']\n",
        "    X_holdout = holdout[FEATURES]\n",
        "    \n",
        "    dtrain = lgb.Dataset(data=X_train, label=y_train)\n",
        "    dvalid = lgb.Dataset(data=X_valid, label=y_valid)\n",
        "    dholdout = lgb.Dataset(data=X_holdout)\n",
        "    \n",
        "    # TRAIN MODEL FOLD K\n",
        "    model = lgb.train(params, \n",
        "                dtrain,\n",
        "                valid_sets=[dtrain,dvalid],\n",
        "                early_stopping_rounds=10) \n",
        "    #model.save_model(f'lgbm_models/LGBM_fold{fold}.lgb')\n",
        "    joblib.dump(model, f'lgb_{fold}.pkl') \n",
        "            \n",
        "    # INFER OOF FOLD K\n",
        "    oof_preds = model.predict(X_valid)\n",
        "    rsme = mean_squared_error(y_valid.values, oof_preds, squared=False)\n",
        "    print('RMSE =',rsme,'\\n')\n",
        "    oof[valid_idx] = oof_preds\n",
        "    \n",
        "    # HOLDOUT PREDS - len(holdout)=0   .\n",
        "    if len(holdout)>0:\n",
        "        p = model.predict(dholdout)\n",
        "        preds += p/FOLDS\n",
        "    \n",
        "    del dtrain, X_train, y_train \n",
        "    del X_valid, y_valid, dvalid, model\n",
        "    _ = gc.collect()\n",
        "    \n",
        "print('#'*25)\n",
        "rsme = mean_squared_error(train.target.values, oof, squared=False)\n",
        "print('OVERALL RMSE =',rsme,'\\n')\n",
        "\n",
        "train['oof'] = oof\n",
        "if len(holdout)>0: holdout['preds'] = preds"
      ]
    },
    {
      "cell_type": "markdown",
      "id": "a1a04eb2",
      "metadata": {
        "papermill": {
          "duration": 0.030407,
          "end_time": "2022-12-08T02:30:26.230790",
          "exception": false,
          "start_time": "2022-12-08T02:30:26.200383",
          "status": "completed"
        },
        "tags": [],
        "id": "a1a04eb2"
      },
      "source": [
        "# Validate OOF (on either dTm or ddG)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "589dc124",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:30:26.293022Z",
          "iopub.status.busy": "2022-12-08T02:30:26.292296Z",
          "iopub.status.idle": "2022-12-08T02:30:26.374403Z",
          "shell.execute_reply": "2022-12-08T02:30:26.373075Z"
        },
        "papermill": {
          "duration": 0.117648,
          "end_time": "2022-12-08T02:30:26.378731",
          "exception": false,
          "start_time": "2022-12-08T02:30:26.261083",
          "status": "completed"
        },
        "scrolled": true,
        "tags": [],
        "id": "589dc124",
        "outputId": "e9857c65-3e09-433e-b22d-e667e0ad1eae"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Protein GP01 has mutation count = 243 and target = dTm\n",
            "Spearman Metric = 0.49961024369925666\n",
            "\n",
            "Protein GP02 has mutation count = 211 and target = dTm\n",
            "Spearman Metric = 0.10906143858413393\n",
            "\n",
            "Protein GP03 has mutation count = 155 and target = dTm\n",
            "Spearman Metric = 0.5059450569969896\n",
            "\n",
            "Protein GP04 has mutation count = 62 and target = dTm\n",
            "Spearman Metric = 0.10659863888356542\n",
            "\n",
            "Protein GP06 has mutation count = 144 and target = dTm\n",
            "Spearman Metric = 0.19287399354596216\n",
            "\n",
            "Protein GP07 has mutation count = 77 and target = dTm\n",
            "Spearman Metric = 0.08867612186461961\n",
            "\n",
            "Protein GP08 has mutation count = 88 and target = dTm\n",
            "Spearman Metric = 0.027974648937315563\n",
            "\n",
            "Protein GP09 has mutation count = 84 and target = dTm\n",
            "Spearman Metric = 0.33650246274974027\n",
            "\n",
            "Protein GP10 has mutation count = 84 and target = dTm\n",
            "Spearman Metric = 0.08289820041205338\n",
            "\n",
            "Protein GP11 has mutation count = 71 and target = dTm\n",
            "Spearman Metric = 0.06087899823838309\n",
            "\n",
            "Protein GP12 has mutation count = 78 and target = dTm\n",
            "Spearman Metric = 0.4284359264809894\n",
            "\n",
            "Protein GP13 has mutation count = 29 and target = dTm\n",
            "Spearman Metric = 0.5186737374541593\n",
            "\n",
            "Protein GP14 has mutation count = 72 and target = dTm\n",
            "Spearman Metric = 0.39014350616324\n",
            "\n",
            "Protein GP16 has mutation count = 38 and target = dTm\n",
            "Spearman Metric = 0.5394370942035949\n",
            "\n",
            "Protein GP17 has mutation count = 47 and target = dTm\n",
            "Spearman Metric = 0.09498283693498927\n",
            "\n",
            "Protein GP18 has mutation count = 60 and target = dTm\n",
            "Spearman Metric = 0.3688664736470906\n",
            "\n",
            "Protein GP19 has mutation count = 48 and target = dTm\n",
            "Spearman Metric = 0.43933510289497063\n",
            "\n",
            "Protein GP20 has mutation count = 28 and target = dTm\n",
            "Spearman Metric = 0.3481782012933116\n",
            "\n",
            "Protein GP22 has mutation count = 37 and target = dTm\n",
            "Spearman Metric = 0.3247346723547339\n",
            "\n",
            "Protein GP23 has mutation count = 28 and target = dTm\n",
            "Spearman Metric = 0.3523289786926384\n",
            "\n",
            "Protein GP24 has mutation count = 52 and target = dTm\n",
            "Spearman Metric = 0.4417093865988769\n",
            "\n",
            "Protein GP25 has mutation count = 52 and target = dTm\n",
            "Spearman Metric = 0.6181530973829141\n",
            "\n",
            "Protein GP27 has mutation count = 27 and target = dTm\n",
            "Spearman Metric = 0.41117074008847665\n",
            "\n",
            "Protein GP28 has mutation count = 33 and target = dTm\n",
            "Spearman Metric = 0.38584052712456834\n",
            "\n",
            "Protein GP29 has mutation count = 26 and target = dTm\n",
            "Spearman Metric = 0.521635034138373\n",
            "\n",
            "Protein GP30 has mutation count = 36 and target = dTm\n",
            "Spearman Metric = 0.16451806069491395\n",
            "\n",
            "Protein GP32 has mutation count = 32 and target = dTm\n",
            "Spearman Metric = 0.6605226987881994\n",
            "\n",
            "Protein GP33 has mutation count = 31 and target = dTm\n",
            "Spearman Metric = nan\n",
            "\n",
            "Protein GP34 has mutation count = 30 and target = dTm\n",
            "Spearman Metric = 0.018258746943899156\n",
            "\n",
            "Protein GP35 has mutation count = 26 and target = dTm\n",
            "Spearman Metric = 0.4252404549523418\n",
            "\n",
            "Protein GP37 has mutation count = 27 and target = dTm\n",
            "Spearman Metric = nan\n",
            "\n",
            "#########################\n",
            "Overall Spearman Metric (predicting dTm) = 0.3263167269222173\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/opt/conda/lib/python3.7/site-packages/scipy/stats/stats.py:4484: SpearmanRConstantInputWarning: An input array is constant; the correlation coefficient is not defined.\n",
            "  warnings.warn(SpearmanRConstantInputWarning())\n"
          ]
        }
      ],
      "source": [
        "sp = []; sp_dtm = []; sp_ddg = []\n",
        "for p in train.pdb.unique():\n",
        "\n",
        "    tmp = train.loc[train.pdb==p].reset_index(drop=True)\n",
        "    ttarget = 'dTm'\n",
        "    if tmp['dTm'].isna().sum()>len(tmp)/2: ttarget = 'ddG'\n",
        "    print('Protein',p,'has mutation count =',len(tmp),'and target =',ttarget)\n",
        "    r = np.abs( spearmanr(tmp.oof.values, tmp[ttarget].values).correlation )\n",
        "    print('Spearman Metric =',r)\n",
        "    sp.append(r)\n",
        "    if ttarget=='dTm': sp_dtm.append(r)\n",
        "    else: sp_ddg.append(r)\n",
        "    print()\n",
        "\n",
        "print('#'*25)\n",
        "if len(sp_dtm)>0:\n",
        "    print(f'Overall Spearman Metric (predicting dTm) =',np.nanmean(sp_dtm))\n",
        "if len(sp_ddg)>0:\n",
        "    print(f'Overall Spearman Metric (predicting ddG) =',np.nanmean(sp_ddg))"
      ]
    },
    {
      "cell_type": "markdown",
      "id": "ccf39cc0",
      "metadata": {
        "papermill": {
          "duration": 0.032095,
          "end_time": "2022-12-08T02:30:26.444419",
          "exception": false,
          "start_time": "2022-12-08T02:30:26.412324",
          "status": "completed"
        },
        "tags": [],
        "id": "ccf39cc0"
      },
      "source": [
        "# Transform Test Data\n",
        "The PDB file provided by Kaggle does not contain real `b_factor`. The column labeled `b_factor` is actually `pLDDT` predicted by Alpha Fold. We will load a `PDF` file below of Kaggle's test protein with estimated `b_factor` downloaded from internet and provided by @kaggleqrdl and @ropeonmars described [here][1]. If we train with Alpha Fold PDB then we will load Kaggle's test PDB created by Alpha Fold.\n",
        "\n",
        "[1]: https://www.kaggle.com/competitions/novozymes-enzyme-stability-prediction/discussion/356182#1968210"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "103db544",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:30:26.510378Z",
          "iopub.status.busy": "2022-12-08T02:30:26.509366Z",
          "iopub.status.idle": "2022-12-08T02:30:26.518270Z",
          "shell.execute_reply": "2022-12-08T02:30:26.517356Z"
        },
        "papermill": {
          "duration": 0.044667,
          "end_time": "2022-12-08T02:30:26.520317",
          "exception": false,
          "start_time": "2022-12-08T02:30:26.475650",
          "status": "completed"
        },
        "tags": [],
        "id": "103db544",
        "outputId": "5a2b1850-f837-4ada-ae74-9119b0c2e20f"
      },
      "outputs": [
        {
          "data": {
            "text/plain": [
              "221"
            ]
          },
          "execution_count": 28,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# LOAD TEST WILDTYPE\n",
        "base = 'VPVNPEPDATSVENVALKTGSGDSQSDPIKADLEVKGQSALPFDVDCWAILCKGAPNVLQRVNEKTKNSNRDRSGANKGPFKDPQKWGIKALPPKNPSWSAQDFKSPEEYAFASSLQGGTNAILAPVNLASQNSQGGVLNGFYSANKVAQFDPSKPQQTKGTWFQITKFTGAAGPYCKALGSNDKSVCDKNKNIAGDWGFDPAKWAYQYDEKNNKFNYVGK'\n",
        "len(base)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "979c08aa",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:30:26.585378Z",
          "iopub.status.busy": "2022-12-08T02:30:26.583770Z",
          "iopub.status.idle": "2022-12-08T02:30:26.620588Z",
          "shell.execute_reply": "2022-12-08T02:30:26.619682Z"
        },
        "papermill": {
          "duration": 0.07118,
          "end_time": "2022-12-08T02:30:26.622754",
          "exception": false,
          "start_time": "2022-12-08T02:30:26.551574",
          "status": "completed"
        },
        "tags": [],
        "id": "979c08aa",
        "outputId": "19427bf7-fb9a-4024-d5eb-6b6658b7b711"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Test shape (2413, 4)\n"
          ]
        },
        {
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>seq_id</th>\n",
              "      <th>protein_sequence</th>\n",
              "      <th>pH</th>\n",
              "      <th>data_source</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>31390</td>\n",
              "      <td>VPVNPEPDATSVENVAEKTGSGDSQSDPIKADLEVKGQSALPFDVD...</td>\n",
              "      <td>8</td>\n",
              "      <td>Novozymes</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>31391</td>\n",
              "      <td>VPVNPEPDATSVENVAKKTGSGDSQSDPIKADLEVKGQSALPFDVD...</td>\n",
              "      <td>8</td>\n",
              "      <td>Novozymes</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>31392</td>\n",
              "      <td>VPVNPEPDATSVENVAKTGSGDSQSDPIKADLEVKGQSALPFDVDC...</td>\n",
              "      <td>8</td>\n",
              "      <td>Novozymes</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>31393</td>\n",
              "      <td>VPVNPEPDATSVENVALCTGSGDSQSDPIKADLEVKGQSALPFDVD...</td>\n",
              "      <td>8</td>\n",
              "      <td>Novozymes</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>31394</td>\n",
              "      <td>VPVNPEPDATSVENVALFTGSGDSQSDPIKADLEVKGQSALPFDVD...</td>\n",
              "      <td>8</td>\n",
              "      <td>Novozymes</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "   seq_id                                   protein_sequence  pH data_source\n",
              "0   31390  VPVNPEPDATSVENVAEKTGSGDSQSDPIKADLEVKGQSALPFDVD...   8   Novozymes\n",
              "1   31391  VPVNPEPDATSVENVAKKTGSGDSQSDPIKADLEVKGQSALPFDVD...   8   Novozymes\n",
              "2   31392  VPVNPEPDATSVENVAKTGSGDSQSDPIKADLEVKGQSALPFDVDC...   8   Novozymes\n",
              "3   31393  VPVNPEPDATSVENVALCTGSGDSQSDPIKADLEVKGQSALPFDVD...   8   Novozymes\n",
              "4   31394  VPVNPEPDATSVENVALFTGSGDSQSDPIKADLEVKGQSALPFDVD...   8   Novozymes"
            ]
          },
          "execution_count": 29,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# LOAD TEST DATA\n",
        "test = pd.read_csv('../input/novozymes-enzyme-stability-prediction/test.csv')\n",
        "deletions = test.loc[test.protein_sequence.str.len()==220,'seq_id'].values\n",
        "print('Test shape', test.shape )\n",
        "test.head()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "7db54417",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:30:26.687106Z",
          "iopub.status.busy": "2022-12-08T02:30:26.686743Z",
          "iopub.status.idle": "2022-12-08T02:30:26.773603Z",
          "shell.execute_reply": "2022-12-08T02:30:26.772469Z"
        },
        "papermill": {
          "duration": 0.12192,
          "end_time": "2022-12-08T02:30:26.775977",
          "exception": false,
          "start_time": "2022-12-08T02:30:26.654057",
          "status": "completed"
        },
        "tags": [],
        "id": "7db54417",
        "outputId": "31cba1db-2620-43ec-f7f2-5fdd275167f3"
      },
      "outputs": [
        {
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>record_name</th>\n",
              "      <th>atom_number</th>\n",
              "      <th>blank_1</th>\n",
              "      <th>atom_name</th>\n",
              "      <th>alt_loc</th>\n",
              "      <th>residue_name</th>\n",
              "      <th>blank_2</th>\n",
              "      <th>chain_id</th>\n",
              "      <th>residue_number</th>\n",
              "      <th>insertion</th>\n",
              "      <th>blank_3</th>\n",
              "      <th>x_coord</th>\n",
              "      <th>y_coord</th>\n",
              "      <th>z_coord</th>\n",
              "      <th>occupancy</th>\n",
              "      <th>b_factor</th>\n",
              "      <th>blank_4</th>\n",
              "      <th>segment_id</th>\n",
              "      <th>element_symbol</th>\n",
              "      <th>charge</th>\n",
              "      <th>line_idx</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>ATOM</td>\n",
              "      <td>1</td>\n",
              "      <td></td>\n",
              "      <td>N</td>\n",
              "      <td></td>\n",
              "      <td>VAL</td>\n",
              "      <td></td>\n",
              "      <td>A</td>\n",
              "      <td>1</td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td>34.064</td>\n",
              "      <td>-6.456</td>\n",
              "      <td>50.464</td>\n",
              "      <td>1.0</td>\n",
              "      <td>45.11</td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td>N</td>\n",
              "      <td>NaN</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>ATOM</td>\n",
              "      <td>2</td>\n",
              "      <td></td>\n",
              "      <td>H</td>\n",
              "      <td></td>\n",
              "      <td>VAL</td>\n",
              "      <td></td>\n",
              "      <td>A</td>\n",
              "      <td>1</td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td>33.576</td>\n",
              "      <td>-6.009</td>\n",
              "      <td>51.228</td>\n",
              "      <td>1.0</td>\n",
              "      <td>45.11</td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td>H</td>\n",
              "      <td>NaN</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>ATOM</td>\n",
              "      <td>3</td>\n",
              "      <td></td>\n",
              "      <td>H2</td>\n",
              "      <td></td>\n",
              "      <td>VAL</td>\n",
              "      <td></td>\n",
              "      <td>A</td>\n",
              "      <td>1</td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td>33.882</td>\n",
              "      <td>-7.449</td>\n",
              "      <td>50.477</td>\n",
              "      <td>1.0</td>\n",
              "      <td>45.11</td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td>H</td>\n",
              "      <td>NaN</td>\n",
              "      <td>2</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>ATOM</td>\n",
              "      <td>4</td>\n",
              "      <td></td>\n",
              "      <td>H3</td>\n",
              "      <td></td>\n",
              "      <td>VAL</td>\n",
              "      <td></td>\n",
              "      <td>A</td>\n",
              "      <td>1</td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td>35.060</td>\n",
              "      <td>-6.323</td>\n",
              "      <td>50.566</td>\n",
              "      <td>1.0</td>\n",
              "      <td>45.11</td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td>H</td>\n",
              "      <td>NaN</td>\n",
              "      <td>3</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>ATOM</td>\n",
              "      <td>5</td>\n",
              "      <td></td>\n",
              "      <td>CA</td>\n",
              "      <td></td>\n",
              "      <td>VAL</td>\n",
              "      <td></td>\n",
              "      <td>A</td>\n",
              "      <td>1</td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td>33.643</td>\n",
              "      <td>-5.877</td>\n",
              "      <td>49.162</td>\n",
              "      <td>1.0</td>\n",
              "      <td>45.11</td>\n",
              "      <td></td>\n",
              "      <td></td>\n",
              "      <td>C</td>\n",
              "      <td>NaN</td>\n",
              "      <td>4</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "  record_name  atom_number blank_1 atom_name alt_loc residue_name blank_2  \\\n",
              "0        ATOM            1                 N                  VAL           \n",
              "1        ATOM            2                 H                  VAL           \n",
              "2        ATOM            3                H2                  VAL           \n",
              "3        ATOM            4                H3                  VAL           \n",
              "4        ATOM            5                CA                  VAL           \n",
              "\n",
              "  chain_id  residue_number insertion blank_3  x_coord  y_coord  z_coord  \\\n",
              "0        A               1                     34.064   -6.456   50.464   \n",
              "1        A               1                     33.576   -6.009   51.228   \n",
              "2        A               1                     33.882   -7.449   50.477   \n",
              "3        A               1                     35.060   -6.323   50.566   \n",
              "4        A               1                     33.643   -5.877   49.162   \n",
              "\n",
              "   occupancy  b_factor blank_4 segment_id element_symbol  charge  line_idx  \n",
              "0        1.0     45.11                                 N     NaN         0  \n",
              "1        1.0     45.11                                 H     NaN         1  \n",
              "2        1.0     45.11                                 H     NaN         2  \n",
              "3        1.0     45.11                                 H     NaN         3  \n",
              "4        1.0     45.11                                 C     NaN         4  "
            ]
          },
          "execution_count": 30,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# LOAD TEST DATA PDB FILE \n",
        "# NOTE KAGGLE'S PDB IS GENERATED BY ALPHA FOLD AND CONTAINS PLDDT IN B_FACTOR COLUMN\n",
        "# WHEN TRAINING WITH KAGGLE.CSV WE NEED ALPHA FOLD PDB\n",
        "# WHEN TRAINING WITH JIN DATA WE NEED PROTEIN DATA BANK PDB (with real b_factor)\n",
        "if USE_PLDDT_INFER:\n",
        "    atom_df = PandasPdb().read_pdb('../input/novozymes-enzyme-stability-prediction/wildtype_structure_prediction_af2.pdb')\n",
        "else:\n",
        "    atom_df = PandasPdb().read_pdb('../input/nesp-test-wildtype-pdb/model.pdb')\n",
        "atom_df = atom_df.df['ATOM']\n",
        "atom_df.head()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "b96608cb",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:30:26.840795Z",
          "iopub.status.busy": "2022-12-08T02:30:26.840424Z",
          "iopub.status.idle": "2022-12-08T02:39:55.121443Z",
          "shell.execute_reply": "2022-12-08T02:39:55.120437Z"
        },
        "papermill": {
          "duration": 568.315748,
          "end_time": "2022-12-08T02:39:55.123767",
          "exception": false,
          "start_time": "2022-12-08T02:30:26.808019",
          "status": "completed"
        },
        "tags": [],
        "id": "b96608cb",
        "outputId": "364eefd1-47c1-46f5-acdb-175735b2c624"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Extracting embeddings and feature engineering 2413 test rows...\n",
            "0 , 10 , 20 , 30 , 40 , 50 , 60 , 70 , 80 , 90 , 100 , 110 , 120 , 130 , 140 , 150 , 160 , 170 , 180 , 190 , 200 , 210 , 220 , 230 , 240 , 250 , 260 , 270 , 280 , 290 , 300 , 310 , 320 , 330 , 340 , 350 , 360 , 370 , 380 , 390 , 400 , 410 , 420 , 430 , 440 , 450 , 460 , 470 , 480 , 490 , 500 , 510 , 520 , 530 , 540 , 550 , 560 , 570 , 580 , 590 , 600 , 610 , 620 , 630 , 640 , 650 , 660 , 670 , 680 , 690 , 700 , 710 , 720 , 730 , 740 , 750 , 760 , 770 , 780 , 790 , 800 , 810 , 820 , 830 , 840 , 850 , 860 , 870 , 880 , 890 , 900 , 910 , 920 , 930 , 940 , 950 , 960 , 970 , 980 , 990 , 1000 , 1010 , 1020 , 1030 , 1040 , 1050 , 1060 , 1070 , 1080 , 1090 , 1100 , 1110 , 1120 , 1130 , 1140 , 1150 , 1160 , 1170 , 1180 , 1190 , 1200 , 1210 , 1220 , 1230 , 1240 , 1250 , 1260 , 1270 , 1280 , 1290 , 1300 , 1310 , 1320 , 1330 , 1340 , 1350 , 1360 , 1370 , 1380 , 1390 , 1400 , 1410 , 1420 , 1430 , 1440 , 1450 , 1460 , 1470 , 1480 , 1490 , 1500 , 1510 , 1520 , 1530 , 1540 , 1550 , 1560 , 1570 , 1580 , 1590 , 1600 , 1610 , 1620 , 1630 , 1640 , 1650 , 1660 , 1670 , 1680 , 1690 , 1700 , 1710 , 1720 , 1730 , 1740 , 1750 , 1760 , 1770 , 1780 , 1790 , 1800 , 1810 , 1820 , 1830 , 1840 , 1850 , 1860 , 1870 , 1880 , 1890 , 1900 , 1910 , 1920 , 1930 , 1940 , 1950 , 1960 , 1970 , 1980 , 1990 , 2000 , 2010 , 2020 , 2030 , 2040 , 2050 , 2060 , 2070 , 2080 , 2090 , 2100 , 2110 , 2120 , 2130 , 2140 , 2150 , 2160 , 2170 , 2180 , 2190 , 2200 , 2210 , 2220 , 2230 , 2240 , 2250 , 2260 , 2270 , 2280 , 2290 , 2300 , 2310 , 2320 , 2330 , 2340 , 2350 , 2360 , 2370 , 2380 , 2390 , 2400 , 2410 , "
          ]
        },
        {
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>WT</th>\n",
              "      <th>WT2</th>\n",
              "      <th>MUT</th>\n",
              "      <th>position</th>\n",
              "      <th>relative_position</th>\n",
              "      <th>b_factor</th>\n",
              "      <th>Molecular Weight_1</th>\n",
              "      <th>Molecular Weight_2</th>\n",
              "      <th>Molecular Weight_delta</th>\n",
              "      <th>Residue Weight_1</th>\n",
              "      <th>Residue Weight_2</th>\n",
              "      <th>Residue Weight_delta</th>\n",
              "      <th>pKa1_1</th>\n",
              "      <th>pKa1_2</th>\n",
              "      <th>pKa1_delta</th>\n",
              "      <th>pKb2_1</th>\n",
              "      <th>pKb2_2</th>\n",
              "      <th>pKb2_delta</th>\n",
              "      <th>pKx3_1</th>\n",
              "      <th>pKx3_2</th>\n",
              "      <th>pKx3_delta</th>\n",
              "      <th>pl4_1</th>\n",
              "      <th>pl4_2</th>\n",
              "      <th>pl4_delta</th>\n",
              "      <th>H_1</th>\n",
              "      <th>H_2</th>\n",
              "      <th>H_delta</th>\n",
              "      <th>VSC_1</th>\n",
              "      <th>VSC_2</th>\n",
              "      <th>VSC_delta</th>\n",
              "      <th>P1_1</th>\n",
              "      <th>P1_2</th>\n",
              "      <th>P1_delta</th>\n",
              "      <th>P2_1</th>\n",
              "      <th>P2_2</th>\n",
              "      <th>P2_delta</th>\n",
              "      <th>SASA_1</th>\n",
              "      <th>SASA_2</th>\n",
              "      <th>SASA_delta</th>\n",
              "      <th>NCISC_1</th>\n",
              "      <th>NCISC_2</th>\n",
              "      <th>NCISC_delta</th>\n",
              "      <th>prev</th>\n",
              "      <th>b_factor_prev</th>\n",
              "      <th>post</th>\n",
              "      <th>b_factor_post</th>\n",
              "      <th>cos_angle</th>\n",
              "      <th>location3d</th>\n",
              "      <th>pca_pool_0</th>\n",
              "      <th>pca_wt_0</th>\n",
              "      <th>pca_mutant_0</th>\n",
              "      <th>pca_local_0</th>\n",
              "      <th>pca_pool_1</th>\n",
              "      <th>pca_wt_1</th>\n",
              "      <th>pca_mutant_1</th>\n",
              "      <th>pca_local_1</th>\n",
              "      <th>pca_pool_2</th>\n",
              "      <th>pca_wt_2</th>\n",
              "      <th>pca_mutant_2</th>\n",
              "      <th>pca_local_2</th>\n",
              "      <th>pca_pool_3</th>\n",
              "      <th>pca_wt_3</th>\n",
              "      <th>pca_mutant_3</th>\n",
              "      <th>pca_local_3</th>\n",
              "      <th>pca_pool_4</th>\n",
              "      <th>pca_wt_4</th>\n",
              "      <th>pca_mutant_4</th>\n",
              "      <th>pca_local_4</th>\n",
              "      <th>pca_pool_5</th>\n",
              "      <th>pca_wt_5</th>\n",
              "      <th>pca_mutant_5</th>\n",
              "      <th>pca_local_5</th>\n",
              "      <th>pca_pool_6</th>\n",
              "      <th>pca_wt_6</th>\n",
              "      <th>pca_mutant_6</th>\n",
              "      <th>pca_local_6</th>\n",
              "      <th>pca_pool_7</th>\n",
              "      <th>pca_wt_7</th>\n",
              "      <th>pca_mutant_7</th>\n",
              "      <th>pca_local_7</th>\n",
              "      <th>pca_pool_8</th>\n",
              "      <th>pca_wt_8</th>\n",
              "      <th>pca_mutant_8</th>\n",
              "      <th>pca_local_8</th>\n",
              "      <th>pca_pool_9</th>\n",
              "      <th>pca_wt_9</th>\n",
              "      <th>pca_mutant_9</th>\n",
              "      <th>pca_local_9</th>\n",
              "      <th>pca_pool_10</th>\n",
              "      <th>pca_wt_10</th>\n",
              "      <th>pca_mutant_10</th>\n",
              "      <th>pca_local_10</th>\n",
              "      <th>pca_pool_11</th>\n",
              "      <th>pca_wt_11</th>\n",
              "      <th>pca_mutant_11</th>\n",
              "      <th>pca_local_11</th>\n",
              "      <th>pca_pool_12</th>\n",
              "      <th>pca_wt_12</th>\n",
              "      <th>pca_mutant_12</th>\n",
              "      <th>pca_local_12</th>\n",
              "      <th>pca_pool_13</th>\n",
              "      <th>pca_wt_13</th>\n",
              "      <th>pca_mutant_13</th>\n",
              "      <th>pca_local_13</th>\n",
              "      <th>pca_pool_14</th>\n",
              "      <th>pca_wt_14</th>\n",
              "      <th>pca_mutant_14</th>\n",
              "      <th>pca_local_14</th>\n",
              "      <th>pca_pool_15</th>\n",
              "      <th>pca_wt_15</th>\n",
              "      <th>pca_mutant_15</th>\n",
              "      <th>pca_local_15</th>\n",
              "      <th>pca_pool_16</th>\n",
              "      <th>pca_pool_17</th>\n",
              "      <th>pca_pool_18</th>\n",
              "      <th>pca_pool_19</th>\n",
              "      <th>pca_pool_20</th>\n",
              "      <th>pca_pool_21</th>\n",
              "      <th>pca_pool_22</th>\n",
              "      <th>pca_pool_23</th>\n",
              "      <th>pca_pool_24</th>\n",
              "      <th>pca_pool_25</th>\n",
              "      <th>pca_pool_26</th>\n",
              "      <th>pca_pool_27</th>\n",
              "      <th>pca_pool_28</th>\n",
              "      <th>pca_pool_29</th>\n",
              "      <th>pca_pool_30</th>\n",
              "      <th>pca_pool_31</th>\n",
              "      <th>AA1</th>\n",
              "      <th>AA2</th>\n",
              "      <th>AA3</th>\n",
              "      <th>AA4</th>\n",
              "      <th>ddG</th>\n",
              "      <th>dTm</th>\n",
              "      <th>pdb</th>\n",
              "      <th>source</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>L</td>\n",
              "      <td>L</td>\n",
              "      <td>E</td>\n",
              "      <td>17</td>\n",
              "      <td>0.076923</td>\n",
              "      <td>55.23</td>\n",
              "      <td>131.18</td>\n",
              "      <td>147.13</td>\n",
              "      <td>15.95</td>\n",
              "      <td>113.16</td>\n",
              "      <td>129.12</td>\n",
              "      <td>15.96</td>\n",
              "      <td>2.36</td>\n",
              "      <td>2.19</td>\n",
              "      <td>-0.17</td>\n",
              "      <td>9.60</td>\n",
              "      <td>9.67</td>\n",
              "      <td>0.07</td>\n",
              "      <td>NaN</td>\n",
              "      <td>4.25</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.98</td>\n",
              "      <td>3.22</td>\n",
              "      <td>-2.76</td>\n",
              "      <td>1.06</td>\n",
              "      <td>-0.74</td>\n",
              "      <td>-1.80</td>\n",
              "      <td>93.5</td>\n",
              "      <td>62.0</td>\n",
              "      <td>-31.5</td>\n",
              "      <td>4.9</td>\n",
              "      <td>12.3</td>\n",
              "      <td>7.4</td>\n",
              "      <td>0.186</td>\n",
              "      <td>0.151</td>\n",
              "      <td>-0.035</td>\n",
              "      <td>1.931</td>\n",
              "      <td>1.862</td>\n",
              "      <td>-0.069</td>\n",
              "      <td>0.051672</td>\n",
              "      <td>0.006802</td>\n",
              "      <td>-0.044870</td>\n",
              "      <td>A</td>\n",
              "      <td>55.85</td>\n",
              "      <td>K</td>\n",
              "      <td>69.25</td>\n",
              "      <td>-0.507968</td>\n",
              "      <td>436.616963</td>\n",
              "      <td>0.015670</td>\n",
              "      <td>3.653698</td>\n",
              "      <td>4.109841</td>\n",
              "      <td>0.456143</td>\n",
              "      <td>-0.033623</td>\n",
              "      <td>0.732240</td>\n",
              "      <td>0.071339</td>\n",
              "      <td>-0.660901</td>\n",
              "      <td>0.021688</td>\n",
              "      <td>-1.339261</td>\n",
              "      <td>-1.104002</td>\n",
              "      <td>0.235258</td>\n",
              "      <td>0.014146</td>\n",
              "      <td>2.826937</td>\n",
              "      <td>2.226573</td>\n",
              "      <td>-0.600364</td>\n",
              "      <td>0.008607</td>\n",
              "      <td>-1.928185</td>\n",
              "      <td>-2.181124</td>\n",
              "      <td>-0.252939</td>\n",
              "      <td>0.013137</td>\n",
              "      <td>-1.528955</td>\n",
              "      <td>-1.901557</td>\n",
              "      <td>-0.372601</td>\n",
              "      <td>0.003923</td>\n",
              "      <td>-0.309841</td>\n",
              "      <td>-0.071118</td>\n",
              "      <td>0.238723</td>\n",
              "      <td>0.003649</td>\n",
              "      <td>-0.892207</td>\n",
              "      <td>0.128042</td>\n",
              "      <td>1.020249</td>\n",
              "      <td>0.005894</td>\n",
              "      <td>0.337565</td>\n",
              "      <td>0.282693</td>\n",
              "      <td>-0.054872</td>\n",
              "      <td>-0.009253</td>\n",
              "      <td>0.460667</td>\n",
              "      <td>-0.283019</td>\n",
              "      <td>-0.743686</td>\n",
              "      <td>-0.006708</td>\n",
              "      <td>0.862695</td>\n",
              "      <td>0.641422</td>\n",
              "      <td>-0.221273</td>\n",
              "      <td>0.002106</td>\n",
              "      <td>-0.774643</td>\n",
              "      <td>-0.111474</td>\n",
              "      <td>0.663168</td>\n",
              "      <td>-0.012079</td>\n",
              "      <td>-0.636301</td>\n",
              "      <td>0.437104</td>\n",
              "      <td>1.073406</td>\n",
              "      <td>-0.004603</td>\n",
              "      <td>0.016339</td>\n",
              "      <td>-0.658829</td>\n",
              "      <td>-0.675169</td>\n",
              "      <td>0.010921</td>\n",
              "      <td>1.181575</td>\n",
              "      <td>0.557593</td>\n",
              "      <td>-0.623982</td>\n",
              "      <td>-0.025972</td>\n",
              "      <td>0.869995</td>\n",
              "      <td>0.524928</td>\n",
              "      <td>-0.345066</td>\n",
              "      <td>4.377282e-02</td>\n",
              "      <td>-0.021601</td>\n",
              "      <td>-0.021052</td>\n",
              "      <td>-0.027163</td>\n",
              "      <td>-0.001233</td>\n",
              "      <td>-0.003741</td>\n",
              "      <td>0.022431</td>\n",
              "      <td>-0.023499</td>\n",
              "      <td>-0.020398</td>\n",
              "      <td>0.010647</td>\n",
              "      <td>-0.000060</td>\n",
              "      <td>-0.000336</td>\n",
              "      <td>0.015998</td>\n",
              "      <td>0.023160</td>\n",
              "      <td>0.006968</td>\n",
              "      <td>0.020548</td>\n",
              "      <td>9</td>\n",
              "      <td>3</td>\n",
              "      <td>0</td>\n",
              "      <td>8</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>kaggle</td>\n",
              "      <td>kaggle</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>L</td>\n",
              "      <td>L</td>\n",
              "      <td>K</td>\n",
              "      <td>17</td>\n",
              "      <td>0.076923</td>\n",
              "      <td>55.23</td>\n",
              "      <td>131.18</td>\n",
              "      <td>146.19</td>\n",
              "      <td>15.01</td>\n",
              "      <td>113.16</td>\n",
              "      <td>128.18</td>\n",
              "      <td>15.02</td>\n",
              "      <td>2.36</td>\n",
              "      <td>2.18</td>\n",
              "      <td>-0.18</td>\n",
              "      <td>9.60</td>\n",
              "      <td>8.95</td>\n",
              "      <td>-0.65</td>\n",
              "      <td>NaN</td>\n",
              "      <td>10.53</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.98</td>\n",
              "      <td>9.74</td>\n",
              "      <td>3.76</td>\n",
              "      <td>1.06</td>\n",
              "      <td>-1.50</td>\n",
              "      <td>-2.56</td>\n",
              "      <td>93.5</td>\n",
              "      <td>100.0</td>\n",
              "      <td>6.5</td>\n",
              "      <td>4.9</td>\n",
              "      <td>11.3</td>\n",
              "      <td>6.4</td>\n",
              "      <td>0.186</td>\n",
              "      <td>0.219</td>\n",
              "      <td>0.033</td>\n",
              "      <td>1.931</td>\n",
              "      <td>2.258</td>\n",
              "      <td>0.327</td>\n",
              "      <td>0.051672</td>\n",
              "      <td>0.017708</td>\n",
              "      <td>-0.033964</td>\n",
              "      <td>A</td>\n",
              "      <td>55.85</td>\n",
              "      <td>K</td>\n",
              "      <td>69.25</td>\n",
              "      <td>-0.507968</td>\n",
              "      <td>436.616963</td>\n",
              "      <td>-0.009102</td>\n",
              "      <td>3.653698</td>\n",
              "      <td>4.143652</td>\n",
              "      <td>0.489954</td>\n",
              "      <td>-0.003685</td>\n",
              "      <td>0.732240</td>\n",
              "      <td>0.191917</td>\n",
              "      <td>-0.540324</td>\n",
              "      <td>0.011366</td>\n",
              "      <td>-1.339261</td>\n",
              "      <td>-1.281947</td>\n",
              "      <td>0.057314</td>\n",
              "      <td>0.003272</td>\n",
              "      <td>2.826937</td>\n",
              "      <td>2.657982</td>\n",
              "      <td>-0.168955</td>\n",
              "      <td>-0.001346</td>\n",
              "      <td>-1.928185</td>\n",
              "      <td>-1.674855</td>\n",
              "      <td>0.253330</td>\n",
              "      <td>0.015566</td>\n",
              "      <td>-1.528955</td>\n",
              "      <td>-1.343973</td>\n",
              "      <td>0.184982</td>\n",
              "      <td>0.021477</td>\n",
              "      <td>-0.309841</td>\n",
              "      <td>0.451318</td>\n",
              "      <td>0.761159</td>\n",
              "      <td>0.021384</td>\n",
              "      <td>-0.892207</td>\n",
              "      <td>-0.263082</td>\n",
              "      <td>0.629125</td>\n",
              "      <td>-0.008346</td>\n",
              "      <td>0.337565</td>\n",
              "      <td>0.266066</td>\n",
              "      <td>-0.071499</td>\n",
              "      <td>-0.005846</td>\n",
              "      <td>0.460667</td>\n",
              "      <td>0.817349</td>\n",
              "      <td>0.356682</td>\n",
              "      <td>0.000513</td>\n",
              "      <td>0.862695</td>\n",
              "      <td>0.820465</td>\n",
              "      <td>-0.042230</td>\n",
              "      <td>0.009505</td>\n",
              "      <td>-0.774643</td>\n",
              "      <td>-0.059180</td>\n",
              "      <td>0.715463</td>\n",
              "      <td>-0.006734</td>\n",
              "      <td>-0.636301</td>\n",
              "      <td>0.172920</td>\n",
              "      <td>0.809222</td>\n",
              "      <td>-0.008161</td>\n",
              "      <td>0.016339</td>\n",
              "      <td>-0.445026</td>\n",
              "      <td>-0.461365</td>\n",
              "      <td>0.014918</td>\n",
              "      <td>1.181575</td>\n",
              "      <td>0.145600</td>\n",
              "      <td>-1.035975</td>\n",
              "      <td>-0.007561</td>\n",
              "      <td>0.869995</td>\n",
              "      <td>0.610329</td>\n",
              "      <td>-0.259665</td>\n",
              "      <td>-1.873773e-02</td>\n",
              "      <td>0.002600</td>\n",
              "      <td>0.008300</td>\n",
              "      <td>-0.001512</td>\n",
              "      <td>-0.001385</td>\n",
              "      <td>0.000830</td>\n",
              "      <td>0.009779</td>\n",
              "      <td>0.001686</td>\n",
              "      <td>-0.001279</td>\n",
              "      <td>-0.021260</td>\n",
              "      <td>0.008111</td>\n",
              "      <td>-0.006684</td>\n",
              "      <td>-0.006700</td>\n",
              "      <td>-0.003747</td>\n",
              "      <td>0.004089</td>\n",
              "      <td>0.004199</td>\n",
              "      <td>9</td>\n",
              "      <td>8</td>\n",
              "      <td>0</td>\n",
              "      <td>8</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>kaggle</td>\n",
              "      <td>kaggle</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>L</td>\n",
              "      <td>L</td>\n",
              "      <td>K</td>\n",
              "      <td>17</td>\n",
              "      <td>0.076923</td>\n",
              "      <td>55.23</td>\n",
              "      <td>131.18</td>\n",
              "      <td>146.19</td>\n",
              "      <td>15.01</td>\n",
              "      <td>113.16</td>\n",
              "      <td>128.18</td>\n",
              "      <td>15.02</td>\n",
              "      <td>2.36</td>\n",
              "      <td>2.18</td>\n",
              "      <td>-0.18</td>\n",
              "      <td>9.60</td>\n",
              "      <td>8.95</td>\n",
              "      <td>-0.65</td>\n",
              "      <td>NaN</td>\n",
              "      <td>10.53</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5.98</td>\n",
              "      <td>9.74</td>\n",
              "      <td>3.76</td>\n",
              "      <td>1.06</td>\n",
              "      <td>-1.50</td>\n",
              "      <td>-2.56</td>\n",
              "      <td>93.5</td>\n",
              "      <td>100.0</td>\n",
              "      <td>6.5</td>\n",
              "      <td>4.9</td>\n",
              "      <td>11.3</td>\n",
              "      <td>6.4</td>\n",
              "      <td>0.186</td>\n",
              "      <td>0.219</td>\n",
              "      <td>0.033</td>\n",
              "      <td>1.931</td>\n",
              "      <td>2.258</td>\n",
              "      <td>0.327</td>\n",
              "      <td>0.051672</td>\n",
              "      <td>0.017708</td>\n",
              "      <td>-0.033964</td>\n",
              "      <td>A</td>\n",
              "      <td>55.85</td>\n",
              "      <td>K</td>\n",
              "      <td>69.25</td>\n",
              "      <td>-0.507968</td>\n",
              "      <td>436.616963</td>\n",
              "      <td>-0.005362</td>\n",
              "      <td>3.653698</td>\n",
              "      <td>4.059747</td>\n",
              "      <td>0.406049</td>\n",
              "      <td>-0.010738</td>\n",
              "      <td>0.732240</td>\n",
              "      <td>0.245013</td>\n",
              "      <td>-0.487227</td>\n",
              "      <td>0.015509</td>\n",
              "      <td>-1.339261</td>\n",
              "      <td>-1.298038</td>\n",
              "      <td>0.041223</td>\n",
              "      <td>0.007163</td>\n",
              "      <td>2.826937</td>\n",
              "      <td>2.715610</td>\n",
              "      <td>-0.111326</td>\n",
              "      <td>-0.014605</td>\n",
              "      <td>-1.928185</td>\n",
              "      <td>-1.886105</td>\n",
              "      <td>0.042080</td>\n",
              "      <td>0.010339</td>\n",
              "      <td>-1.528955</td>\n",
              "      <td>-1.400370</td>\n",
              "      <td>0.128585</td>\n",
              "      <td>0.022345</td>\n",
              "      <td>-0.309841</td>\n",
              "      <td>0.411571</td>\n",
              "      <td>0.721412</td>\n",
              "      <td>0.010690</td>\n",
              "      <td>-0.892207</td>\n",
              "      <td>-0.269352</td>\n",
              "      <td>0.622855</td>\n",
              "      <td>0.001848</td>\n",
              "      <td>0.337565</td>\n",
              "      <td>0.310879</td>\n",
              "      <td>-0.026686</td>\n",
              "      <td>-0.005226</td>\n",
              "      <td>0.460667</td>\n",
              "      <td>0.814417</td>\n",
              "      <td>0.353750</td>\n",
              "      <td>-0.006431</td>\n",
              "      <td>0.862695</td>\n",
              "      <td>0.867996</td>\n",
              "      <td>0.005301</td>\n",
              "      <td>0.003691</td>\n",
              "      <td>-0.774643</td>\n",
              "      <td>0.043115</td>\n",
              "      <td>0.817758</td>\n",
              "      <td>-0.008198</td>\n",
              "      <td>-0.636301</td>\n",
              "      <td>0.235629</td>\n",
              "      <td>0.871930</td>\n",
              "      <td>-0.002155</td>\n",
              "      <td>0.016339</td>\n",
              "      <td>-0.427761</td>\n",
              "      <td>-0.444101</td>\n",
              "      <td>0.003016</td>\n",
              "      <td>1.181575</td>\n",
              "      <td>0.091577</td>\n",
              "      <td>-1.089998</td>\n",
              "      <td>-0.009796</td>\n",
              "      <td>0.869995</td>\n",
              "      <td>0.599038</td>\n",
              "      <td>-0.270956</td>\n",
              "      <td>4.584312e-03</td>\n",
              "      <td>-0.005657</td>\n",
              "      <td>-0.001442</td>\n",
              "      <td>-0.012316</td>\n",
              "      <td>0.000034</td>\n",
              "      <td>0.000305</td>\n",
              "      <td>0.011045</td>\n",
              "      <td>-0.008179</td>\n",
              "      <td>-0.009527</td>\n",
              "      <td>-0.003520</td>\n",
              "      <td>0.003255</td>\n",
              "      <td>-0.002284</td>\n",
              "      <td>0.004493</td>\n",
              "      <td>0.009693</td>\n",
              "      <td>0.004287</td>\n",
              "      <td>0.008953</td>\n",
              "      <td>9</td>\n",
              "      <td>8</td>\n",
              "      <td>0</td>\n",
              "      <td>8</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>kaggle</td>\n",
              "      <td>kaggle</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>K</td>\n",
              "      <td>K</td>\n",
              "      <td>C</td>\n",
              "      <td>18</td>\n",
              "      <td>0.081448</td>\n",
              "      <td>69.25</td>\n",
              "      <td>146.19</td>\n",
              "      <td>121.16</td>\n",
              "      <td>-25.03</td>\n",
              "      <td>128.18</td>\n",
              "      <td>103.15</td>\n",
              "      <td>-25.03</td>\n",
              "      <td>2.18</td>\n",
              "      <td>1.96</td>\n",
              "      <td>-0.22</td>\n",
              "      <td>8.95</td>\n",
              "      <td>10.28</td>\n",
              "      <td>1.33</td>\n",
              "      <td>10.53</td>\n",
              "      <td>8.18</td>\n",
              "      <td>-2.35</td>\n",
              "      <td>9.74</td>\n",
              "      <td>5.07</td>\n",
              "      <td>-4.67</td>\n",
              "      <td>-1.50</td>\n",
              "      <td>0.29</td>\n",
              "      <td>1.79</td>\n",
              "      <td>100.0</td>\n",
              "      <td>44.6</td>\n",
              "      <td>-55.4</td>\n",
              "      <td>11.3</td>\n",
              "      <td>5.5</td>\n",
              "      <td>-5.8</td>\n",
              "      <td>0.219</td>\n",
              "      <td>0.128</td>\n",
              "      <td>-0.091</td>\n",
              "      <td>2.258</td>\n",
              "      <td>1.461</td>\n",
              "      <td>-0.797</td>\n",
              "      <td>0.017708</td>\n",
              "      <td>-0.036610</td>\n",
              "      <td>-0.054318</td>\n",
              "      <td>L</td>\n",
              "      <td>55.23</td>\n",
              "      <td>T</td>\n",
              "      <td>85.36</td>\n",
              "      <td>-0.495687</td>\n",
              "      <td>403.053103</td>\n",
              "      <td>-0.013351</td>\n",
              "      <td>4.156505</td>\n",
              "      <td>3.706668</td>\n",
              "      <td>-0.449837</td>\n",
              "      <td>0.042808</td>\n",
              "      <td>0.218701</td>\n",
              "      <td>0.515684</td>\n",
              "      <td>0.296983</td>\n",
              "      <td>0.017336</td>\n",
              "      <td>-1.117844</td>\n",
              "      <td>-0.826530</td>\n",
              "      <td>0.291314</td>\n",
              "      <td>0.033058</td>\n",
              "      <td>2.745929</td>\n",
              "      <td>1.884782</td>\n",
              "      <td>-0.861147</td>\n",
              "      <td>0.003961</td>\n",
              "      <td>-1.823559</td>\n",
              "      <td>-1.695721</td>\n",
              "      <td>0.127837</td>\n",
              "      <td>-0.038768</td>\n",
              "      <td>-1.354425</td>\n",
              "      <td>-1.124987</td>\n",
              "      <td>0.229438</td>\n",
              "      <td>0.054527</td>\n",
              "      <td>0.338079</td>\n",
              "      <td>0.397221</td>\n",
              "      <td>0.059142</td>\n",
              "      <td>-0.020372</td>\n",
              "      <td>-0.340035</td>\n",
              "      <td>-0.311429</td>\n",
              "      <td>0.028606</td>\n",
              "      <td>-0.010099</td>\n",
              "      <td>0.347938</td>\n",
              "      <td>0.498460</td>\n",
              "      <td>0.150522</td>\n",
              "      <td>-0.005917</td>\n",
              "      <td>0.767338</td>\n",
              "      <td>0.279042</td>\n",
              "      <td>-0.488296</td>\n",
              "      <td>-0.006121</td>\n",
              "      <td>0.727083</td>\n",
              "      <td>0.361452</td>\n",
              "      <td>-0.365631</td>\n",
              "      <td>0.025275</td>\n",
              "      <td>0.015427</td>\n",
              "      <td>-0.430646</td>\n",
              "      <td>-0.446073</td>\n",
              "      <td>0.002057</td>\n",
              "      <td>0.280884</td>\n",
              "      <td>0.118037</td>\n",
              "      <td>-0.162847</td>\n",
              "      <td>0.003074</td>\n",
              "      <td>-0.344929</td>\n",
              "      <td>-0.108765</td>\n",
              "      <td>0.236164</td>\n",
              "      <td>-0.039876</td>\n",
              "      <td>0.039195</td>\n",
              "      <td>0.381782</td>\n",
              "      <td>0.342588</td>\n",
              "      <td>0.012637</td>\n",
              "      <td>0.623551</td>\n",
              "      <td>0.302342</td>\n",
              "      <td>-0.321209</td>\n",
              "      <td>4.172325e-07</td>\n",
              "      <td>-0.004923</td>\n",
              "      <td>0.029515</td>\n",
              "      <td>-0.020812</td>\n",
              "      <td>0.006318</td>\n",
              "      <td>0.025256</td>\n",
              "      <td>-0.010666</td>\n",
              "      <td>0.015329</td>\n",
              "      <td>-0.008509</td>\n",
              "      <td>-0.024626</td>\n",
              "      <td>-0.000458</td>\n",
              "      <td>-0.010051</td>\n",
              "      <td>0.004109</td>\n",
              "      <td>0.008895</td>\n",
              "      <td>-0.033582</td>\n",
              "      <td>-0.031853</td>\n",
              "      <td>8</td>\n",
              "      <td>1</td>\n",
              "      <td>9</td>\n",
              "      <td>16</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>kaggle</td>\n",
              "      <td>kaggle</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>K</td>\n",
              "      <td>K</td>\n",
              "      <td>F</td>\n",
              "      <td>18</td>\n",
              "      <td>0.081448</td>\n",
              "      <td>69.25</td>\n",
              "      <td>146.19</td>\n",
              "      <td>165.19</td>\n",
              "      <td>19.00</td>\n",
              "      <td>128.18</td>\n",
              "      <td>147.18</td>\n",
              "      <td>19.00</td>\n",
              "      <td>2.18</td>\n",
              "      <td>1.83</td>\n",
              "      <td>-0.35</td>\n",
              "      <td>8.95</td>\n",
              "      <td>9.13</td>\n",
              "      <td>0.18</td>\n",
              "      <td>10.53</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>9.74</td>\n",
              "      <td>5.48</td>\n",
              "      <td>-4.26</td>\n",
              "      <td>-1.50</td>\n",
              "      <td>1.19</td>\n",
              "      <td>2.69</td>\n",
              "      <td>100.0</td>\n",
              "      <td>115.5</td>\n",
              "      <td>15.5</td>\n",
              "      <td>11.3</td>\n",
              "      <td>5.2</td>\n",
              "      <td>-6.1</td>\n",
              "      <td>0.219</td>\n",
              "      <td>0.290</td>\n",
              "      <td>0.071</td>\n",
              "      <td>2.258</td>\n",
              "      <td>2.228</td>\n",
              "      <td>-0.030</td>\n",
              "      <td>0.017708</td>\n",
              "      <td>0.037552</td>\n",
              "      <td>0.019844</td>\n",
              "      <td>L</td>\n",
              "      <td>55.23</td>\n",
              "      <td>T</td>\n",
              "      <td>85.36</td>\n",
              "      <td>-0.495687</td>\n",
              "      <td>403.053103</td>\n",
              "      <td>-0.012155</td>\n",
              "      <td>4.156505</td>\n",
              "      <td>3.584667</td>\n",
              "      <td>-0.571838</td>\n",
              "      <td>0.023557</td>\n",
              "      <td>0.218701</td>\n",
              "      <td>0.670925</td>\n",
              "      <td>0.452224</td>\n",
              "      <td>-0.008664</td>\n",
              "      <td>-1.117844</td>\n",
              "      <td>-1.658330</td>\n",
              "      <td>-0.540486</td>\n",
              "      <td>0.005803</td>\n",
              "      <td>2.745929</td>\n",
              "      <td>1.726001</td>\n",
              "      <td>-1.019928</td>\n",
              "      <td>0.004373</td>\n",
              "      <td>-1.823559</td>\n",
              "      <td>-2.157400</td>\n",
              "      <td>-0.333842</td>\n",
              "      <td>-0.027923</td>\n",
              "      <td>-1.354425</td>\n",
              "      <td>-0.951652</td>\n",
              "      <td>0.402773</td>\n",
              "      <td>0.001244</td>\n",
              "      <td>0.338079</td>\n",
              "      <td>0.021872</td>\n",
              "      <td>-0.316207</td>\n",
              "      <td>-0.022832</td>\n",
              "      <td>-0.340035</td>\n",
              "      <td>0.000601</td>\n",
              "      <td>0.340636</td>\n",
              "      <td>0.003952</td>\n",
              "      <td>0.347938</td>\n",
              "      <td>0.454015</td>\n",
              "      <td>0.106077</td>\n",
              "      <td>-0.001414</td>\n",
              "      <td>0.767338</td>\n",
              "      <td>0.138612</td>\n",
              "      <td>-0.628725</td>\n",
              "      <td>-0.003323</td>\n",
              "      <td>0.727083</td>\n",
              "      <td>0.304173</td>\n",
              "      <td>-0.422910</td>\n",
              "      <td>0.015512</td>\n",
              "      <td>0.015427</td>\n",
              "      <td>-0.089042</td>\n",
              "      <td>-0.104469</td>\n",
              "      <td>0.003978</td>\n",
              "      <td>0.280884</td>\n",
              "      <td>0.611981</td>\n",
              "      <td>0.331096</td>\n",
              "      <td>-0.009356</td>\n",
              "      <td>-0.344929</td>\n",
              "      <td>-0.326506</td>\n",
              "      <td>0.018422</td>\n",
              "      <td>-0.032109</td>\n",
              "      <td>0.039195</td>\n",
              "      <td>0.196869</td>\n",
              "      <td>0.157674</td>\n",
              "      <td>0.011514</td>\n",
              "      <td>0.623551</td>\n",
              "      <td>0.088702</td>\n",
              "      <td>-0.534849</td>\n",
              "      <td>6.026566e-03</td>\n",
              "      <td>0.009164</td>\n",
              "      <td>0.009946</td>\n",
              "      <td>0.001949</td>\n",
              "      <td>0.010027</td>\n",
              "      <td>0.002498</td>\n",
              "      <td>-0.007427</td>\n",
              "      <td>0.012622</td>\n",
              "      <td>-0.003327</td>\n",
              "      <td>-0.002705</td>\n",
              "      <td>-0.010729</td>\n",
              "      <td>-0.015474</td>\n",
              "      <td>0.006176</td>\n",
              "      <td>0.006175</td>\n",
              "      <td>-0.010698</td>\n",
              "      <td>-0.015702</td>\n",
              "      <td>8</td>\n",
              "      <td>4</td>\n",
              "      <td>9</td>\n",
              "      <td>16</td>\n",
              "      <td>NaN</td>\n",
              "      <td>NaN</td>\n",
              "      <td>kaggle</td>\n",
              "      <td>kaggle</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "  WT WT2 MUT  position  relative_position  b_factor  Molecular Weight_1  \\\n",
              "0  L   L   E        17           0.076923     55.23              131.18   \n",
              "1  L   L   K        17           0.076923     55.23              131.18   \n",
              "2  L   L   K        17           0.076923     55.23              131.18   \n",
              "3  K   K   C        18           0.081448     69.25              146.19   \n",
              "4  K   K   F        18           0.081448     69.25              146.19   \n",
              "\n",
              "   Molecular Weight_2  Molecular Weight_delta  Residue Weight_1  \\\n",
              "0              147.13                   15.95            113.16   \n",
              "1              146.19                   15.01            113.16   \n",
              "2              146.19                   15.01            113.16   \n",
              "3              121.16                  -25.03            128.18   \n",
              "4              165.19                   19.00            128.18   \n",
              "\n",
              "   Residue Weight_2  Residue Weight_delta  pKa1_1  pKa1_2  pKa1_delta  pKb2_1  \\\n",
              "0            129.12                 15.96    2.36    2.19       -0.17    9.60   \n",
              "1            128.18                 15.02    2.36    2.18       -0.18    9.60   \n",
              "2            128.18                 15.02    2.36    2.18       -0.18    9.60   \n",
              "3            103.15                -25.03    2.18    1.96       -0.22    8.95   \n",
              "4            147.18                 19.00    2.18    1.83       -0.35    8.95   \n",
              "\n",
              "   pKb2_2  pKb2_delta  pKx3_1  pKx3_2  pKx3_delta  pl4_1  pl4_2  pl4_delta  \\\n",
              "0    9.67        0.07     NaN    4.25         NaN   5.98   3.22      -2.76   \n",
              "1    8.95       -0.65     NaN   10.53         NaN   5.98   9.74       3.76   \n",
              "2    8.95       -0.65     NaN   10.53         NaN   5.98   9.74       3.76   \n",
              "3   10.28        1.33   10.53    8.18       -2.35   9.74   5.07      -4.67   \n",
              "4    9.13        0.18   10.53     NaN         NaN   9.74   5.48      -4.26   \n",
              "\n",
              "    H_1   H_2  H_delta  VSC_1  VSC_2  VSC_delta  P1_1  P1_2  P1_delta   P2_1  \\\n",
              "0  1.06 -0.74    -1.80   93.5   62.0      -31.5   4.9  12.3       7.4  0.186   \n",
              "1  1.06 -1.50    -2.56   93.5  100.0        6.5   4.9  11.3       6.4  0.186   \n",
              "2  1.06 -1.50    -2.56   93.5  100.0        6.5   4.9  11.3       6.4  0.186   \n",
              "3 -1.50  0.29     1.79  100.0   44.6      -55.4  11.3   5.5      -5.8  0.219   \n",
              "4 -1.50  1.19     2.69  100.0  115.5       15.5  11.3   5.2      -6.1  0.219   \n",
              "\n",
              "    P2_2  P2_delta  SASA_1  SASA_2  SASA_delta   NCISC_1   NCISC_2  \\\n",
              "0  0.151    -0.035   1.931   1.862      -0.069  0.051672  0.006802   \n",
              "1  0.219     0.033   1.931   2.258       0.327  0.051672  0.017708   \n",
              "2  0.219     0.033   1.931   2.258       0.327  0.051672  0.017708   \n",
              "3  0.128    -0.091   2.258   1.461      -0.797  0.017708 -0.036610   \n",
              "4  0.290     0.071   2.258   2.228      -0.030  0.017708  0.037552   \n",
              "\n",
              "   NCISC_delta prev  b_factor_prev post  b_factor_post  cos_angle  location3d  \\\n",
              "0    -0.044870    A          55.85    K          69.25  -0.507968  436.616963   \n",
              "1    -0.033964    A          55.85    K          69.25  -0.507968  436.616963   \n",
              "2    -0.033964    A          55.85    K          69.25  -0.507968  436.616963   \n",
              "3    -0.054318    L          55.23    T          85.36  -0.495687  403.053103   \n",
              "4     0.019844    L          55.23    T          85.36  -0.495687  403.053103   \n",
              "\n",
              "   pca_pool_0  pca_wt_0  pca_mutant_0  pca_local_0  pca_pool_1  pca_wt_1  \\\n",
              "0    0.015670  3.653698      4.109841     0.456143   -0.033623  0.732240   \n",
              "1   -0.009102  3.653698      4.143652     0.489954   -0.003685  0.732240   \n",
              "2   -0.005362  3.653698      4.059747     0.406049   -0.010738  0.732240   \n",
              "3   -0.013351  4.156505      3.706668    -0.449837    0.042808  0.218701   \n",
              "4   -0.012155  4.156505      3.584667    -0.571838    0.023557  0.218701   \n",
              "\n",
              "   pca_mutant_1  pca_local_1  pca_pool_2  pca_wt_2  pca_mutant_2  pca_local_2  \\\n",
              "0      0.071339    -0.660901    0.021688 -1.339261     -1.104002     0.235258   \n",
              "1      0.191917    -0.540324    0.011366 -1.339261     -1.281947     0.057314   \n",
              "2      0.245013    -0.487227    0.015509 -1.339261     -1.298038     0.041223   \n",
              "3      0.515684     0.296983    0.017336 -1.117844     -0.826530     0.291314   \n",
              "4      0.670925     0.452224   -0.008664 -1.117844     -1.658330    -0.540486   \n",
              "\n",
              "   pca_pool_3  pca_wt_3  pca_mutant_3  pca_local_3  pca_pool_4  pca_wt_4  \\\n",
              "0    0.014146  2.826937      2.226573    -0.600364    0.008607 -1.928185   \n",
              "1    0.003272  2.826937      2.657982    -0.168955   -0.001346 -1.928185   \n",
              "2    0.007163  2.826937      2.715610    -0.111326   -0.014605 -1.928185   \n",
              "3    0.033058  2.745929      1.884782    -0.861147    0.003961 -1.823559   \n",
              "4    0.005803  2.745929      1.726001    -1.019928    0.004373 -1.823559   \n",
              "\n",
              "   pca_mutant_4  pca_local_4  pca_pool_5  pca_wt_5  pca_mutant_5  pca_local_5  \\\n",
              "0     -2.181124    -0.252939    0.013137 -1.528955     -1.901557    -0.372601   \n",
              "1     -1.674855     0.253330    0.015566 -1.528955     -1.343973     0.184982   \n",
              "2     -1.886105     0.042080    0.010339 -1.528955     -1.400370     0.128585   \n",
              "3     -1.695721     0.127837   -0.038768 -1.354425     -1.124987     0.229438   \n",
              "4     -2.157400    -0.333842   -0.027923 -1.354425     -0.951652     0.402773   \n",
              "\n",
              "   pca_pool_6  pca_wt_6  pca_mutant_6  pca_local_6  pca_pool_7  pca_wt_7  \\\n",
              "0    0.003923 -0.309841     -0.071118     0.238723    0.003649 -0.892207   \n",
              "1    0.021477 -0.309841      0.451318     0.761159    0.021384 -0.892207   \n",
              "2    0.022345 -0.309841      0.411571     0.721412    0.010690 -0.892207   \n",
              "3    0.054527  0.338079      0.397221     0.059142   -0.020372 -0.340035   \n",
              "4    0.001244  0.338079      0.021872    -0.316207   -0.022832 -0.340035   \n",
              "\n",
              "   pca_mutant_7  pca_local_7  pca_pool_8  pca_wt_8  pca_mutant_8  pca_local_8  \\\n",
              "0      0.128042     1.020249    0.005894  0.337565      0.282693    -0.054872   \n",
              "1     -0.263082     0.629125   -0.008346  0.337565      0.266066    -0.071499   \n",
              "2     -0.269352     0.622855    0.001848  0.337565      0.310879    -0.026686   \n",
              "3     -0.311429     0.028606   -0.010099  0.347938      0.498460     0.150522   \n",
              "4      0.000601     0.340636    0.003952  0.347938      0.454015     0.106077   \n",
              "\n",
              "   pca_pool_9  pca_wt_9  pca_mutant_9  pca_local_9  pca_pool_10  pca_wt_10  \\\n",
              "0   -0.009253  0.460667     -0.283019    -0.743686    -0.006708   0.862695   \n",
              "1   -0.005846  0.460667      0.817349     0.356682     0.000513   0.862695   \n",
              "2   -0.005226  0.460667      0.814417     0.353750    -0.006431   0.862695   \n",
              "3   -0.005917  0.767338      0.279042    -0.488296    -0.006121   0.727083   \n",
              "4   -0.001414  0.767338      0.138612    -0.628725    -0.003323   0.727083   \n",
              "\n",
              "   pca_mutant_10  pca_local_10  pca_pool_11  pca_wt_11  pca_mutant_11  \\\n",
              "0       0.641422     -0.221273     0.002106  -0.774643      -0.111474   \n",
              "1       0.820465     -0.042230     0.009505  -0.774643      -0.059180   \n",
              "2       0.867996      0.005301     0.003691  -0.774643       0.043115   \n",
              "3       0.361452     -0.365631     0.025275   0.015427      -0.430646   \n",
              "4       0.304173     -0.422910     0.015512   0.015427      -0.089042   \n",
              "\n",
              "   pca_local_11  pca_pool_12  pca_wt_12  pca_mutant_12  pca_local_12  \\\n",
              "0      0.663168    -0.012079  -0.636301       0.437104      1.073406   \n",
              "1      0.715463    -0.006734  -0.636301       0.172920      0.809222   \n",
              "2      0.817758    -0.008198  -0.636301       0.235629      0.871930   \n",
              "3     -0.446073     0.002057   0.280884       0.118037     -0.162847   \n",
              "4     -0.104469     0.003978   0.280884       0.611981      0.331096   \n",
              "\n",
              "   pca_pool_13  pca_wt_13  pca_mutant_13  pca_local_13  pca_pool_14  \\\n",
              "0    -0.004603   0.016339      -0.658829     -0.675169     0.010921   \n",
              "1    -0.008161   0.016339      -0.445026     -0.461365     0.014918   \n",
              "2    -0.002155   0.016339      -0.427761     -0.444101     0.003016   \n",
              "3     0.003074  -0.344929      -0.108765      0.236164    -0.039876   \n",
              "4    -0.009356  -0.344929      -0.326506      0.018422    -0.032109   \n",
              "\n",
              "   pca_wt_14  pca_mutant_14  pca_local_14  pca_pool_15  pca_wt_15  \\\n",
              "0   1.181575       0.557593     -0.623982    -0.025972   0.869995   \n",
              "1   1.181575       0.145600     -1.035975    -0.007561   0.869995   \n",
              "2   1.181575       0.091577     -1.089998    -0.009796   0.869995   \n",
              "3   0.039195       0.381782      0.342588     0.012637   0.623551   \n",
              "4   0.039195       0.196869      0.157674     0.011514   0.623551   \n",
              "\n",
              "   pca_mutant_15  pca_local_15   pca_pool_16  pca_pool_17  pca_pool_18  \\\n",
              "0       0.524928     -0.345066  4.377282e-02    -0.021601    -0.021052   \n",
              "1       0.610329     -0.259665 -1.873773e-02     0.002600     0.008300   \n",
              "2       0.599038     -0.270956  4.584312e-03    -0.005657    -0.001442   \n",
              "3       0.302342     -0.321209  4.172325e-07    -0.004923     0.029515   \n",
              "4       0.088702     -0.534849  6.026566e-03     0.009164     0.009946   \n",
              "\n",
              "   pca_pool_19  pca_pool_20  pca_pool_21  pca_pool_22  pca_pool_23  \\\n",
              "0    -0.027163    -0.001233    -0.003741     0.022431    -0.023499   \n",
              "1    -0.001512    -0.001385     0.000830     0.009779     0.001686   \n",
              "2    -0.012316     0.000034     0.000305     0.011045    -0.008179   \n",
              "3    -0.020812     0.006318     0.025256    -0.010666     0.015329   \n",
              "4     0.001949     0.010027     0.002498    -0.007427     0.012622   \n",
              "\n",
              "   pca_pool_24  pca_pool_25  pca_pool_26  pca_pool_27  pca_pool_28  \\\n",
              "0    -0.020398     0.010647    -0.000060    -0.000336     0.015998   \n",
              "1    -0.001279    -0.021260     0.008111    -0.006684    -0.006700   \n",
              "2    -0.009527    -0.003520     0.003255    -0.002284     0.004493   \n",
              "3    -0.008509    -0.024626    -0.000458    -0.010051     0.004109   \n",
              "4    -0.003327    -0.002705    -0.010729    -0.015474     0.006176   \n",
              "\n",
              "   pca_pool_29  pca_pool_30  pca_pool_31  AA1  AA2  AA3  AA4  ddG  dTm  \\\n",
              "0     0.023160     0.006968     0.020548    9    3    0    8  NaN  NaN   \n",
              "1    -0.003747     0.004089     0.004199    9    8    0    8  NaN  NaN   \n",
              "2     0.009693     0.004287     0.008953    9    8    0    8  NaN  NaN   \n",
              "3     0.008895    -0.033582    -0.031853    8    1    9   16  NaN  NaN   \n",
              "4     0.006175    -0.010698    -0.015702    8    4    9   16  NaN  NaN   \n",
              "\n",
              "      pdb  source  \n",
              "0  kaggle  kaggle  \n",
              "1  kaggle  kaggle  \n",
              "2  kaggle  kaggle  \n",
              "3  kaggle  kaggle  \n",
              "4  kaggle  kaggle  "
            ]
          },
          "execution_count": 31,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "def get_test_mutation(row):\n",
        "    for i,(a,b) in enumerate(zip(row.protein_sequence,base)):\n",
        "        if a!=b: break\n",
        "    row['wildtype'] = base[i]\n",
        "    row['mutation'] = row.protein_sequence[i]\n",
        "    row['position'] = i+1\n",
        "    return row\n",
        "\n",
        "# TRANSFORM TEST DATAFRAME TO MATCH TRAIN DATAFRAME\n",
        "test = test.apply(get_test_mutation,axis=1)\n",
        "test['ddG'] = np.nan\n",
        "test['dTm'] = np.nan\n",
        "test['CIF'] = None\n",
        "test['sequence'] = base\n",
        "test = test.rename({'protein_sequence':'mutant_seq'},axis=1)\n",
        "test['source'] = 'kaggle'\n",
        "test['PDB'] = 'kaggle'\n",
        "\n",
        "# FEATURE ENGINEER TEST DATA\n",
        "rows = []\n",
        "print(f'Extracting embeddings and feature engineering {len(test)} test rows...')\n",
        "for index,row in test.iterrows():\n",
        "    if index%10==0: print(index,', ',end='')\n",
        "    j = row.position\n",
        "    dd = get_new_row(atom_df, j, row)\n",
        "    rows.append(dd)\n",
        "test = pd.DataFrame(rows)\n",
        "test.head()"
      ]
    },
    {
      "cell_type": "markdown",
      "id": "e08f639a",
      "metadata": {
        "papermill": {
          "duration": 0.043603,
          "end_time": "2022-12-08T02:39:55.212866",
          "exception": false,
          "start_time": "2022-12-08T02:39:55.169263",
          "status": "completed"
        },
        "tags": [],
        "id": "e08f639a"
      },
      "source": [
        "# Infer Test Data"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "e784811e",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:39:55.301723Z",
          "iopub.status.busy": "2022-12-08T02:39:55.300091Z",
          "iopub.status.idle": "2022-12-08T02:39:55.305997Z",
          "shell.execute_reply": "2022-12-08T02:39:55.305140Z"
        },
        "papermill": {
          "duration": 0.052227,
          "end_time": "2022-12-08T02:39:55.307899",
          "exception": false,
          "start_time": "2022-12-08T02:39:55.255672",
          "status": "completed"
        },
        "tags": [],
        "id": "e784811e"
      },
      "outputs": [],
      "source": [
        "#lgb_model_0 = joblib.load('/kaggle/working/lgb_0.pkl')\n",
        "#lgb_model_1 = joblib.load('/kaggle/working/lgb_1.pkl')\n",
        "#lgb_model_2 = joblib.load('/kaggle/working/lgb_2.pkl')\n",
        "#lgb_model_3 = joblib.load('/kaggle/working/lgb_3.pkl')\n",
        "#lgb_model_4 = joblib.load('/kaggle/working/lgb_4.pkl')\n",
        "#lgb_model_5 = joblib.load('/kaggle/working/lgb_5.pkl')\n",
        "#lgb_model_6 = joblib.load('/kaggle/working/lgb_6.pkl')\n",
        "#lgb_model_7 = joblib.load('/kaggle/working/lgb_7.pkl')\n",
        "#lgb_model_8 = joblib.load('/kaggle/working/lgb_8.pkl')\n",
        "#lgb_model_9 = joblib.load('/kaggle/working/lgb_9.pkl')"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "5c61d97c",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:39:55.395560Z",
          "iopub.status.busy": "2022-12-08T02:39:55.395259Z",
          "iopub.status.idle": "2022-12-08T02:39:55.399953Z",
          "shell.execute_reply": "2022-12-08T02:39:55.399118Z"
        },
        "papermill": {
          "duration": 0.050966,
          "end_time": "2022-12-08T02:39:55.401864",
          "exception": false,
          "start_time": "2022-12-08T02:39:55.350898",
          "status": "completed"
        },
        "tags": [],
        "id": "5c61d97c"
      },
      "outputs": [],
      "source": [
        "#X_test = test[FEATURES]"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "8b2b2a6c",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:39:55.489050Z",
          "iopub.status.busy": "2022-12-08T02:39:55.488775Z",
          "iopub.status.idle": "2022-12-08T02:39:55.492934Z",
          "shell.execute_reply": "2022-12-08T02:39:55.491964Z"
        },
        "papermill": {
          "duration": 0.05057,
          "end_time": "2022-12-08T02:39:55.494982",
          "exception": false,
          "start_time": "2022-12-08T02:39:55.444412",
          "status": "completed"
        },
        "tags": [],
        "id": "8b2b2a6c"
      },
      "outputs": [],
      "source": [
        "#X_test.head()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "7565871f",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:39:55.581910Z",
          "iopub.status.busy": "2022-12-08T02:39:55.581601Z",
          "iopub.status.idle": "2022-12-08T02:39:55.587369Z",
          "shell.execute_reply": "2022-12-08T02:39:55.586527Z"
        },
        "papermill": {
          "duration": 0.051758,
          "end_time": "2022-12-08T02:39:55.589372",
          "exception": false,
          "start_time": "2022-12-08T02:39:55.537614",
          "status": "completed"
        },
        "tags": [],
        "id": "7565871f"
      },
      "outputs": [],
      "source": [
        "#pred_0 = lgb_model_0.predict(X_test)\n",
        "#pred_1 = lgb_model_1.predict(X_test)\n",
        "#pred_2 = lgb_model_2.predict(X_test)\n",
        "#pred_3 = lgb_model_3.predict(X_test)\n",
        "#pred_4 = lgb_model_4.predict(X_test)\n",
        "#pred_5 = lgb_model_5.predict(X_test)\n",
        "#pred_6 = lgb_model_6.predict(X_test)\n",
        "#pred_7 = lgb_model_7.predict(X_test)\n",
        "#pred_8 = lgb_model_8.predict(X_test)\n",
        "#pred_9 = lgb_model_9.predict(X_test)\n",
        "#pred = (pred_0 + pred_1 + pred_2 + pred_3 + pred_4 + pred_5 + pred_6 + pred_7 +pred_8 + pred_9) / 10"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "90846655",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:39:55.678208Z",
          "iopub.status.busy": "2022-12-08T02:39:55.676763Z",
          "iopub.status.idle": "2022-12-08T02:39:55.681914Z",
          "shell.execute_reply": "2022-12-08T02:39:55.681076Z"
        },
        "papermill": {
          "duration": 0.051215,
          "end_time": "2022-12-08T02:39:55.683920",
          "exception": false,
          "start_time": "2022-12-08T02:39:55.632705",
          "status": "completed"
        },
        "tags": [],
        "id": "90846655"
      },
      "outputs": [],
      "source": [
        "#len(pred)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "88ae4748",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:39:55.774744Z",
          "iopub.status.busy": "2022-12-08T02:39:55.773211Z",
          "iopub.status.idle": "2022-12-08T02:39:55.917134Z",
          "shell.execute_reply": "2022-12-08T02:39:55.915865Z"
        },
        "papermill": {
          "duration": 0.191131,
          "end_time": "2022-12-08T02:39:55.920121",
          "exception": false,
          "start_time": "2022-12-08T02:39:55.728990",
          "status": "completed"
        },
        "tags": [],
        "id": "88ae4748",
        "outputId": "6fb74b36-5148-41dc-a69f-480a2e825e2f"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "CPU times: user 137 ms, sys: 0 ns, total: 137 ms\n",
            "Wall time: 137 ms\n"
          ]
        }
      ],
      "source": [
        "%%time\n",
        "# TEST DATA FOR XGB\n",
        "X_test = test[FEATURES]\n",
        "\n",
        "# INFER XGB MODELS ON TEST DATA\n",
        "loaded_model = joblib.load(f'/kaggle/working/lgb_0.pkl')\n",
        "preds = loaded_model.predict(X_test)\n",
        "for f in range(1,FOLDS):\n",
        "    loaded_model = joblib.load(f'/kaggle/working/lgb_{f}.pkl')\n",
        "    preds += loaded_model.predict(X_test)\n",
        "preds /= FOLDS"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "3e386415",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:39:56.009467Z",
          "iopub.status.busy": "2022-12-08T02:39:56.009166Z",
          "iopub.status.idle": "2022-12-08T02:39:56.353620Z",
          "shell.execute_reply": "2022-12-08T02:39:56.352567Z"
        },
        "papermill": {
          "duration": 0.391137,
          "end_time": "2022-12-08T02:39:56.355855",
          "exception": false,
          "start_time": "2022-12-08T02:39:55.964718",
          "status": "completed"
        },
        "tags": [],
        "id": "3e386415",
        "outputId": "c7cf70fe-1c7d-4731-f2fa-304f1e1b6c1a"
      },
      "outputs": [
        {
          "data": {
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAXAAAAEKCAYAAAALoA6YAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjUuMywgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/NK7nSAAAACXBIWXMAAAsTAAALEwEAmpwYAAAW4ElEQVR4nO3de7hldX3f8fdHRlRGkdughIuDCUqpUdSJlxqNFRMJGsHGGPuoQWNKak2jxucpqE0lXirUxEu1XigEMNqKohbr3aBTpRHigHhBVEZAhQzOgGBREUG//WOt42w2+5yzz5m9zzm/c96v59nP3mvt31rr+1tr5ru/+7fW2idVhSSpPXdZ7gAkSYtjApekRpnAJalRJnBJapQJXJIaZQKXpEaZwBuVpMZ4XD2B7RyZ5OQk+0wg7KlLctaE+r05yQVjtDs5yYKuxU3y4iT/avHRSZ11yx2AFu3RQ9MfAr4MnDww79YJbOdI4JXAu4EfTGB9q83pwCcWuMyLgQuAD048Gq0pJvBGVdWFg9NJbgWuH57fkiR3q6pJfOgsmaq6BrhmueMYV4v7WLNzCGUVS3Jokvck2ZHk1iSXJnnaUJsHJPlQku1Jfprku0nen2RdkucCZ/ZNrxgYmtk4xzavTvLuJP8mydZ+nZck+ZdD7c5Kck2SRyf5hyS3AP+lf29DknckubaP+xtJThixraP6df80ybeT/OmINuuSvLp//6dJrk9yQZLfHHMfPrHfxk+SfG3E/rvTEEqSFyW5PMktSW5MsmVmuX54537Aswb251kDyx6d5Av9sj9M8r+SPHBo/bsleU2SbX1cn0lyeL+uk4djS/KgJJ9M8iPgff17v5PkYwPr+FqSlybZbWhbM8fzOUm+2cf1+SSHJVmf5J1Jbkjy/SR/k8SicAm5s1epJAcDFwHbgZcAO4A/BD6Q5Liq+nDf9KPAjcALgOuBA4Fj6D7cPwq8BviPwB+ws9LcNs/mHw88HHgF3TDOicDHkzykqr450O7ewHuBvwZeDtySZE+64YV70A0HXQU8CXh7Xz2+pe/fPwM+BmwBngncrW9/T+DnA9s4se//K4BLgT2BTcA4Y/q/CrwZeB3dvnkp8P4kh1fV1lELJHkW8DfAq4DP9/148MD2ntbHPTjctaNf9mi6ff4ZumN1z349FyQ5sqqu7dv/Vb+/Xg/8Pd2+njmeo5wHnAGcCvyin3d/4HzgLcBP6fbJycAG4KSh5R/X74sTgd2BNwEfAK4EttLt/8fR/Tv5NvC2OWLRJFWVj1XwAK4G3j0wfQZdYth3qN2ngUv71/sBBTx1jvU+t2/zawuI42fAwQPz7kU3fv53A/PO6td77NDyf0mXUA4bmv/f6ZLoun76Pf30+oE2B/fbvnpg3keADy5if24GbhuMA9if7sPh5QPzTu7+G/1y+q3AJQs5VgPztwBXzPSxn3doH8cb+um9gR8Bbxta9i/6/XnycGzAi+aJJ3TF3CvoPszvMhTrD4B7D8z78369pw+t5xLgs8v9f2EtPRxCWb2Opqv0ftgPI6zrv95+EnhIX+neQFdFndIPeRw2oW1fWFXfm5moqpvpKsvhE6+30SXY4bgvAq4aEfe+wBF9u0cDH6uqHw9s53vA/x1a3xeBY5K8NslvJtl9Af24oqquGFj/drpvNIfMscwXgSOTvKUfftljnA0lWQ88DDinqm4f2OZVdH36rX7WrwPrgfcPreLcOVb/oRHbO6Af/vgO3YfebXTftvai+6Aa9IWq+uHA9Df6508OtfsG3YeologJfPXaH/gjuv+Yg4/X9+/vW13Z9Nt0ld/rgG8luTLJC3Zx29+fZd6BQ/N2VNXPh+btT/d1fDjumYS1b/98wBzbGfSf6a6ieSrdkMYNSc5Mst8Y/Rh11c2twN3nWOZddMNRj6RLcD9I8sG5zhv09qarhEcNT13HziGYA/rn7UNtRu2LGXdYZ5K70A25PIUuaT8B+A3gtX2T4f7dODT9sznmz7VvNGGOga9eN9AlrFNnef+fAKrqSuCPkgR4CPBnwNuSXF1VH1/ktu8zy7xrh+aNun76Brrk9KJZ1j0zhr5tju3s3EDVbXT74NQk96VLWm8A9qAbZ56o/kPxncA7k+wN/A7dmPg5dEl9NjfS7Y/7jnjvvuz8MJlJxvsDlw20GbUvfhnW0PSv0o15P6eq3j0zM8nvzbEOrUBW4KvXJ+hOnl1WVVtGPO5wKVl1LqUbSwV4UP880+4eC9j2o/qTqAAkuRfwZOALY8Z9OPDdWeK+uW/3BbqhkfUD2zkYeMxsK66q66rqdLoTfw+ard2kVNWNVXUO3ZUfg9u7laH92Q8FXQz8weCVIEnuB/wLujF5gK8CP6Y7qTxoeHouM8M6tw1s567AsxawDq0AVuCr138C/hH4XJK30p2M2psukdy/qv44yYPprrI4h+5qgt3oTlreTnclBMDX++cXJjmb7j/9V6pq5mv0KN8HPtVf0jZzFcp64NVjxP1Gusr480neSFdxr6dL6o+tqmP7dq+hS1qfSvJ6uqsjTmZoKCHJeXRXfFxCV+U+lG6c/Z1jxLJgSU4Dbqb7gNkOPAB4DvCpgWZfBx6b5Cl0wyPXV9XVdCdwPwp8JMnb6K5C+Svgh3RVPFV1Y5I3AS9PcjPdh9HDgOf36/4F87sc+A7w2iQ/pzumL1lkl7Wclvssqo/JPBhxZQNwEN2dgtfSjU9uo7sK5dn9+/sDZwPfAn5C9zX9/wBPGlrPK/t1/Jzu6/jG+eIA/oTukrJbgS8BTxhqdxZwzSzr2JsukV/Vx72dbjjoxUPtntiv+1a6k7F/2q/36oE2LwUupBuauYXuA+Fk4K7z7M/NwAWz9O+sgemTueNVKMf3y27v47qq78ueA20O7/vzk35/Dq7vaLrkfwtd4j4PeOBQDLvRjVdf17fbTFel3+GKE3ZehbJuRD+OpLtc8yd0l4e+qj9mdzi+s/y7enzf7onjHlMf03mk3/HSRPQ3qlxQVc9e7ljWkiRPpzvR+7iq+vxyx6Ol4RCK1Jgkj6Q7p3AR3TXzD6e7+eZCuqpaa4QJXGrPj+gutXwh3Z2l2+lOlL6s/Eq9pjiEIkmN8jJCSWrUkg6h7LfffrVx48al3KQkNe/iiy++vqo2DM9f0gS+ceNGtmzZspSblKTm9b9ZcycOoUhSo0zgktQoE7gkNcoELkmNMoFLUqNM4JLUKBO4JDXKBC5JjTKBS1Kj/DVCSbPaeNJHf/n66lOevIyRaBQrcElqlBW4pF1ilb58rMAlqVEmcElqlAlckhplApekRpnAJalRJnBJapQJXJIaZQKXpEZ5I4+kqRu82Qe84WdSrMAlqVEmcElqlEMokibG30VZWmNV4ElekuSyJF9L8j+T3D3JoUkuSrI1yTlJdp92sJKkneatwJMcCPw5cERV3ZLkfcAzgWOAN1bVe5O8A3g+8PapRitp2QyfiNTyG3cMfB1wjyTrgD2AbcATgHP7988Gjpt4dJKkWc2bwKvqWuCvge/SJe4fAhcDN1XV7X2za4ADpxWkJOnO5k3gSfYGjgUOBX4FWA8cPe4GkpyQZEuSLTt27Fh0oJKkOxpnCOWJwFVVtaOqbgM+CDwG2KsfUgE4CLh21MJVdVpVbaqqTRs2bJhI0JKk8RL4d4FHJdkjSYCjgK8DnwWe3rc5HjhvOiFKkkYZZwz8IrqTlZcAX+2XOQ04EfiLJFuBfYEzphinJGnIWDfyVNUrgVcOzb4SeMTEI5IkjcVb6SWpUSZwSWqUv4UiaVn5+ymLZwUuSY0ygUtSo0zgktQoE7gkNcoELkmNMoFLUqO8jFDSkvOPQ0yGFbgkNcoELkmNMoFLUqNM4JLUKBO4JDXKBC5JjTKBS1KjTOCS1Chv5JE0ld/k9mad6bMCl6RGWYFLugMr53ZYgUtSo0zgktQoE7gkNcoELkmNMoFLUqNM4JLUKBO4JDXKBC5JjTKBS1KjvBNT0ooxjd9kWc2swCWpUVbgklYkq/H5WYFLUqNM4JLUKBO4JDXKBC5JjfIkprSGeGJwdRmrAk+yV5Jzk3wjyeVJHp1knySfTnJF/7z3tIOVJO00bgX+ZuATVfX0JLsDewAvB86vqlOSnAScBJw4pTglTZh/Oq1981bgSe4NPA44A6CqflZVNwHHAmf3zc4GjptOiJKkUcYZQjkU2AGcmeRLSU5Psh64T1Vt69tcB9xnWkFKku5snAS+DngY8PaqeijwY7rhkl+qqgJq1MJJTkiyJcmWHTt27Gq8kqTeOAn8GuCaqrqonz6XLqF/P8kBAP3z9lELV9VpVbWpqjZt2LBhEjFLkhgjgVfVdcD3kjywn3UU8HXgw8Dx/bzjgfOmEqEkaaRxr0L598B7+itQrgSeR5f835fk+cB3gGdMJ0RJ0ihjJfCquhTYNOKtoyYajSRpbN5KL0mNMoFLUqP8LRRplfOOy9XLClySGmUCl6RGmcAlqVEmcElqlAlckhplApekRnkZoaQVb7ZLIdf6n4WzApekRpnAJalRJnBJapQJXJIaZQKXpEaZwCWpUV5GKDVs8PK6tX5J3VpkBS5JjTKBS1KjTOCS1CgTuCQ1ygQuSY0ygUtSo0zgktQoE7gkNcoELkmNMoFLUqNM4JLUKBO4JDXKBC5JjfLXCCU1a63/GqMVuCQ1ygpcasBarzTHsRb3kRW4JDXKBC5JjTKBS1KjTOCS1CgTuCQ1ygQuSY0aO4En2S3Jl5J8pJ8+NMlFSbYmOSfJ7tMLU5I0bCEV+IuAywemTwXeWFW/BtwIPH+SgUmS5jZWAk9yEPBk4PR+OsATgHP7JmcDx00hPknSLMa9E/NNwH8A7tVP7wvcVFW399PXAAeOWjDJCcAJAIcccsiiA5U0vsG7ErV6zVuBJ3kKsL2qLl7MBqrqtKraVFWbNmzYsJhVSJJGGKcCfwzw1CTHAHcH9gTeDOyVZF1fhR8EXDu9MCVJw+atwKvqZVV1UFVtBJ4JfKaqngV8Fnh63+x44LypRSlJupNd+TXCE4H3JnkN8CXgjMmEJEm7Zq38MuGCEnhVbQY296+vBB4x+ZAkSePwTkxJapR/0EFaoWa7FNBLBDXDClySGmUCl6RGmcAlqVEmcElqlAlckhplApekRpnAJalRJnBJapQJXJIaZQKXpEaZwCWpUSZwSWqUCVySGuWvEUrLYBp/cMBfKVx7rMAlqVEmcElqlAlckhplApekRnkSU1oi45xk9ESkFsIKXJIaZQKXpEaZwCWpUY6BS1rVZjuvMKkbqJaTFbgkNcoELkmNcghFmsM0frNEmhQrcElqlAlckhplApekRpnAJalRJnBJapQJXJIaZQKXpEaZwCWpUd7II02Rv++tabICl6RGzVuBJzkYeBdwH6CA06rqzUn2Ac4BNgJXA8+oqhunF6o0voXeAu8t82rROBX47cBLq+oI4FHAC5McAZwEnF9VhwHn99OSpCUybwKvqm1VdUn/+mbgcuBA4Fjg7L7Z2cBxU4pRkjTCgk5iJtkIPBS4CLhPVW3r37qObohl1DInACcAHHLIIYsOVGvbShviWM1/JGCtWGn/phZj7JOYSe4JfAB4cVX9v8H3qqroxsfvpKpOq6pNVbVpw4YNuxSsJGmnsSrwJHelS97vqaoP9rO/n+SAqtqW5ABg+7SClGazlFWUlwRqpZm3Ak8S4Azg8qp6w8BbHwaO718fD5w3+fAkSbMZpwJ/DPAc4KtJLu3nvRw4BXhfkucD3wGeMZUIV5jVMG4maXWYN4FX1QVAZnn7qMmGI0kal3diSlKj/C0UrSnjDIF5slKtsAKXpEZZgU+IJze1WFb8WiwrcElqlAlckhrlEIrWrGkMXTgc0qZWh0CtwCWpUVbgu8BqS9JysgKXpEZZgWtFaXUsUloOVuCS1CgTuCQ1yiEUNWe2k8eeVNZaYwUuSY2yAteqZ2Wu1coKXJIataYrcC9ZWxkc05YWxwpckhplApekRq3pIZRxLMXX+IUO5ayGoR+HR7RStfT/ywpckhplBb5MrECllW+lV+NW4JLUKCvwEXa1Op5t+ZX4Cb4QK70akdYaK3BJapQJXJIa5RCK7mQxQyXjLOOJW2myrMAlqVGrtgLflZtjVpulro5X876UYOVcqGAFLkmNar4CX22Xtk27P6ttf0lrmRW4JDXKBC5JjWp+CGUcK+Wk2kLjWAlxr4QYpFYs9RClFbgkNWpNVOCr2XJVyFbmWmt25Rv0tKpxK3BJatQuVeBJjgbeDOwGnF5Vp0wkqhG8VXvy3F9S2xZdgSfZDfhvwO8CRwD/OskRkwpMkjS3XRlCeQSwtaqurKqfAe8Fjp1MWJKk+ezKEMqBwPcGpq8BHjncKMkJwAn95I+SfHOe9e4HXD9Xg5y6gCiX17x9acRq6QfYl5VotfQDZunLBHLW/UbNnPpVKFV1GnDauO2TbKmqTVMMacmslr6sln6AfVmJVks/YOn7sitDKNcCBw9MH9TPkyQtgV1J4F8EDktyaJLdgWcCH55MWJKk+Sx6CKWqbk/yZ8An6S4j/NuqumwCMY093NKA1dKX1dIPsC8r0WrpByxxX1JVS7k9SdKEeCemJDXKBC5JjZp6Ak9ydJJvJtma5KQ52v1+kkqyqZ/ePcmZSb6a5MtJHj/Q9uH9/K1J/muSNNqPzf06L+0f+0+7H/125+xLkucm2TEQ158MvHd8kiv6x/ED81fcMVlkP1o8Jp9IclOSjwwtc2iSi/p1ntNfbNBiP85KctXAMkdOux/9dhfVlyRHJvlCksuSfCXJHw4sM9ljUlVTe9Cd3Pw2cH9gd+DLwBEj2t0L+BxwIbCpn/dC4Mz+9f7AxcBd+ul/BB4FBPg48LuN9mPzTLuleozTF+C5wFtHLLsPcGX/vHf/eu+VekwW2Y+mjkn/3lHA7wEfGZr/PuCZ/et3AC9otB9nAU9v5ZgADwAO61//CrAN2Gsax2TaFfi4t9u/GjgV+OnAvCOAzwBU1XbgJmBTkgOAPavqwur2wruA46bWg87E+zHVaOe2Kz+B8CTg01X1g6q6Efg0cPQKPyajjOzHlOIcxy79LEVVnQ/cPDiv/wb0BODcftbZrOxjMrIfy2jRfamqb1XVFf3rfwK2AxumcUymncBH3W5/4GCDJA8DDq6q4Z/G+zLw1CTrkhwKPJzuxqED+/XMus4pmEY/ZpzZf/36y6UYdmCMvvR+v//6d26SmXhnW3ZFHpPeQvoxo6VjMpt9gZuq6vZ51jlJ0+jHjNf2y7wxyd12OdL5TaQvSR5BV8F/mykck2U9iZnkLsAbgJeOePtv6Tq4BXgT8A/Az5csuAXYhX48q6p+HXhs/3jO1IMdz/8GNlbVg+mq07OXOZ7FWkw/PCbTtZh+vAw4HPgNumGvE6cX3oLM2Zf+m+nfAc+rql9MI4BpJ/D5bre/F/AgYHOSq+nGUD+cZFNV3V5VL6mqI6vqWGAv4Fv98gfNsc5pmEY/qKpr++ebgf9B97Vt2ub9CYSquqGqbu0nT6f71jDXsivxmCymHy0ek9ncAOyVZOZmvZV+TGZVVduqcytwJg0ckyR7Ah8FXlFVF/azJ39MpnwiYB3dCaJD2Xki4J/P0X4zO0/+7QGs71//NvC5gXbDJ8yOaa0f/Tr361/flW5c7N9Osx/j9gU4YOD104AL+9f7AFfRnfjbu3+9z0o9JgvtR4vHZGDe47nzyb/3c8cTZv+u0X4c0D+H7lvsKSv5mPTtzwdePGK9Ez0mU90JfZDH0FWc36b7NAJ4FfDUEW03szPxbQS+CVwO/D1wv4F2m4Cv9et8K/0dpS31A1hPd0XKV4DL6P+y0bT7MU5fgNf1MX0Z+Cxw+MCyfwxs7R/PW8nHZKH9aPiYfB7YAdxCN1z3pH7+/ek+WLfSJY67NdqPzwBf7f99vRu450o+JsCzgduASwceR07jmHgrvSQ1yjsxJalRJnBJapQJXJIaZQKXpEaZwCWpUSZwSWqUCVySGvX/AbvrfiuA3JnoAAAAAElFTkSuQmCC\n",
            "text/plain": [
              "<Figure size 432x288 with 1 Axes>"
            ]
          },
          "metadata": {
            "needs_background": "light"
          },
          "output_type": "display_data"
        }
      ],
      "source": [
        "plt.hist(preds,bins=100)\n",
        "plt.title('Test preds histogram',size=16)\n",
        "plt.show()"
      ]
    },
    {
      "cell_type": "markdown",
      "id": "a68d1b29",
      "metadata": {
        "papermill": {
          "duration": 0.043097,
          "end_time": "2022-12-08T02:39:56.443195",
          "exception": false,
          "start_time": "2022-12-08T02:39:56.400098",
          "status": "completed"
        },
        "tags": [],
        "id": "a68d1b29"
      },
      "source": [
        "# Create Submission CSV\n",
        "There are 2413 rows in test data. Among these, 2336 that are `edit mutations` and 77 are `delete mutations`. Our trained model can only predict `edit mutations`, so will set all `delete mutations` to the mean `edit mutation` prediction below."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "ebaa12e7",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:39:56.533329Z",
          "iopub.status.busy": "2022-12-08T02:39:56.531796Z",
          "iopub.status.idle": "2022-12-08T02:39:56.558565Z",
          "shell.execute_reply": "2022-12-08T02:39:56.557566Z"
        },
        "papermill": {
          "duration": 0.073827,
          "end_time": "2022-12-08T02:39:56.560516",
          "exception": false,
          "start_time": "2022-12-08T02:39:56.486689",
          "status": "completed"
        },
        "tags": [],
        "id": "ebaa12e7",
        "outputId": "cee201b9-2333-41cf-eaae-bec30e5129bf"
      },
      "outputs": [
        {
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>seq_id</th>\n",
              "      <th>tm</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>31390</td>\n",
              "      <td>0.502892</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>31391</td>\n",
              "      <td>0.509391</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>31392</td>\n",
              "      <td>0.510849</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>31393</td>\n",
              "      <td>0.517013</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>31394</td>\n",
              "      <td>0.514814</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "   seq_id        tm\n",
              "0   31390  0.502892\n",
              "1   31391  0.509391\n",
              "2   31392  0.510849\n",
              "3   31393  0.517013\n",
              "4   31394  0.514814"
            ]
          },
          "execution_count": 39,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "sub = pd.read_csv('../input/novozymes-enzyme-stability-prediction/sample_submission.csv')\n",
        "sub.tm = preds\n",
        "sub.loc[sub.seq_id.isin(deletions),'tm'] = sub.loc[~sub.seq_id.isin(deletions),'tm'].mean()\n",
        "sub.head()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "2fc23cfb",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:39:56.652856Z",
          "iopub.status.busy": "2022-12-08T02:39:56.652465Z",
          "iopub.status.idle": "2022-12-08T02:39:56.664899Z",
          "shell.execute_reply": "2022-12-08T02:39:56.663788Z"
        },
        "papermill": {
          "duration": 0.061928,
          "end_time": "2022-12-08T02:39:56.667146",
          "exception": false,
          "start_time": "2022-12-08T02:39:56.605218",
          "status": "completed"
        },
        "tags": [],
        "id": "2fc23cfb"
      },
      "outputs": [],
      "source": [
        "sub.to_csv('submission_single_model_lgbm.csv',index=False)"
      ]
    },
    {
      "cell_type": "markdown",
      "id": "c79da4e5",
      "metadata": {
        "papermill": {
          "duration": 0.106095,
          "end_time": "2022-12-08T02:39:56.817561",
          "exception": false,
          "start_time": "2022-12-08T02:39:56.711466",
          "status": "completed"
        },
        "tags": [],
        "id": "c79da4e5"
      },
      "source": [
        "# Ensemble"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "82d429f6",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:39:56.908237Z",
          "iopub.status.busy": "2022-12-08T02:39:56.907865Z",
          "iopub.status.idle": "2022-12-08T02:39:57.184569Z",
          "shell.execute_reply": "2022-12-08T02:39:57.183625Z"
        },
        "papermill": {
          "duration": 0.323986,
          "end_time": "2022-12-08T02:39:57.186923",
          "exception": false,
          "start_time": "2022-12-08T02:39:56.862937",
          "status": "completed"
        },
        "tags": [],
        "id": "82d429f6"
      },
      "outputs": [],
      "source": [
        "import pandas as pd\n",
        "import numpy as np\n",
        "import matplotlib.pyplot as plt\n",
        "import seaborn as sns\n",
        "from scipy import stats\n",
        "from sklearn import model_selection\n",
        "from sklearn import preprocessing\n",
        "from sklearn import feature_extraction\n",
        "from sklearn.linear_model import Ridge,LinearRegression\n",
        "from xgboost import XGBRegressor\n",
        "from catboost import CatBoostRegressor\n",
        "import warnings\n",
        "import os\n",
        "warnings.simplefilter('ignore')\n",
        "import pickle\n",
        "import os\n",
        "SEED = 42\n",
        "N_FOLDS = 10"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "id": "e7522508",
      "metadata": {
        "execution": {
          "iopub.execute_input": "2022-12-08T02:39:57.282873Z",
          "iopub.status.busy": "2022-12-08T02:39:57.282448Z",
          "iopub.status.idle": "2022-12-08T02:39:57.501845Z",
          "shell.execute_reply": "2022-12-08T02:39:57.500476Z"
        },
        "papermill": {
          "duration": 0.272213,
          "end_time": "2022-12-08T02:39:57.503275",
          "exception": true,
          "start_time": "2022-12-08T02:39:57.231062",
          "status": "failed"
        },
        "tags": [],
        "id": "e7522508",
        "outputId": "16f1e1cc-5f42-4bce-b280-8e2f4868c358"
      },
      "outputs": [
        {
          "ename": "NameError",
          "evalue": "name 'config' is not defined",
          "output_type": "error",
          "traceback": [
            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
            "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
            "\u001b[0;32m/tmp/ipykernel_23/3937539682.py\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0mtrain_df\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mtrain\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0mtest_df\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mtest\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m \u001b[0mss\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mpd\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mread_csv\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mconfig\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msubmit\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
            "\u001b[0;31mNameError\u001b[0m: name 'config' is not defined"
          ]
        }
      ],
      "source": [
        "train_df = train\n",
        "test_df = test\n",
        "ss = pd.read_csv(config.submit)"
      ]
    }
  ],
  "metadata": {
    "kernelspec": {
      "display_name": "Python 3",
      "language": "python",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.7.12"
    },
    "papermill": {
      "default_parameters": {},
      "duration": 1367.948327,
      "end_time": "2022-12-08T02:39:59.848375",
      "environment_variables": {},
      "exception": true,
      "input_path": "__notebook__.ipynb",
      "output_path": "__notebook__.ipynb",
      "parameters": {},
      "start_time": "2022-12-08T02:17:11.900048",
      "version": "2.3.4"
    },
    "colab": {
      "provenance": [],
      "include_colab_link": true
    }
  },
  "nbformat": 4,
  "nbformat_minor": 5
}